<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=Generator content="Microsoft Word 14 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:"MS Mincho";
	panose-1:2 2 6 9 4 2 5 8 3 4;}
@font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"\@SimSun";
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:"\@MS Mincho";
	panose-1:2 2 6 9 4 2 5 8 3 4;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";}
.MsoChpDefault
	{font-family:"Calibri","sans-serif";}
.MsoPapDefault
	{margin-bottom:10.0pt;
	line-height:115%;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;}
div.WordSection1
	{page:WordSection1;}
-->
</style>

</head>

<body lang=EN-US>

<div class=WordSection1>

<p class=MsoNormal style='text-autospace:none'><span lang=EN>PLoS Genet. 2010
Dec; 6(12): e1001258.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Published online 2010
Dec 23. doi:� 10.1371/journal.pgen.1001258</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>PMCID: PMC3009656</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>A Role for ATF2 in
Regulating MITF and Melanoma Development</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Meera Shah,#1 Anindita
Bhoumik,#1 Vikas Goel,1 Antimone Dewing,1 Wolfgang Breitwieser,2 Harriet
Kluger,3 Stan Krajewski,1 Maryla Krajewska,1 Jason DeHart,1 Eric Lau,1 David M.
Kallenberg,4 Hyeongnam Jeong,5 Alexey Eroshkin,1 Dorothy C. Bennett,4 Lynda
Chin,5 Marcus Bosenberg,6 Nic Jones,2 and Ze'ev A. Ronai1,*</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>James M. Ford, Editor</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Author information </span><span
lang=EN style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman","serif"'>&#9658;</span><span
lang=EN style='font-size:12.0pt;line-height:115%;font-family:"Cambria","serif"'>
Article notes </span><span lang=EN style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>&#9658;</span><span lang=EN
style='font-size:12.0pt;line-height:115%;font-family:"Cambria","serif"'>
Copyright and License information </span><span lang=EN style='font-size:12.0pt;
line-height:115%;font-family:"Times New Roman","serif"'>&#9658;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>This article has been
cited by other articles in PMC.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Go to:</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Abstract</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>The transcription factor
ATF2 has been shown to attenuate melanoma susceptibility to apoptosis and to
promote its ability to form tumors in xenograft models. <span style='background:yellow'>To directly assess ATF2's role in melanoma development, we crossed a
mouse melanoma model (NrasQ61K::Ink4a&#8722;/&#8722;) with mice expressing a
transcriptionally inactive form of ATF2 in melanocytes.</span> <span
style='background:yellow'>In contrast to 7/21 of the
NrasQ61K::Ink4a&#8722;/&#8722; mice, only 1/21 mice expressing mutant ATF2 in
melanocytes developed melanoma.</span> Gene expression profiling identified
higher MITF expression in primary melanocytes expressing transcriptionally
inactive ATF2. MITF downregulation by ATF2 was confirmed in the skin of
Atf2&#8722;/&#8722; mice, in primary human melanocytes, and in 50% of human
melanoma cell lines. Inhibition of MITF transcription by MITF was shown to be
mediated by ATF2-JunB�dependent suppression of SOX10 transcription. Remarkably,
oncogenic BRAF (V600E)�dependent focus formation of melanocytes on soft agar
was inhibited by ATF2 knockdown and partially rescued upon shMITF
co-expression. On melanoma tissue microarrays, a high nuclear ATF2 to MITF
ratio in primary specimens was associated with metastatic disease and poor
prognosis. Our findings establish the importance of transcriptionally active
ATF2 in melanoma development through fine-tuning of MITF expression.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Go to:</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Author Summary</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Understanding mechanisms
underlying early stages in melanoma development is of major interest and
importance. Recent studies indicate a role for MITF, a master regulator of
melanocyte development and biogenesis, in melanoma progression. Here we
demonstrate that the transcription factor ATF2 negatively regulates MITF
transcription in melanocytes and in about 50% of melanoma cell lines. Increased
MITF expression, seen upon inhibition of ATF2, effectively attenuated the
ability of BRAFV600E-expressing melanocytes to exhibit a transformed phenotype,
an effect partially rescued when MITF expression was also blocked.
Significantly, the development of melanoma in mice carrying genetic changes
seen in human tumors was inhibited upon inactivation of ATF2 in melanocytes.
Melanocytes from mice lacking active ATF2 expressed increased levels of MITF,
confirming that ATF2 negatively regulates MITF and implicating this newly
discovered regulatory link in melanoma development. Primary melanoma specimens
that exhibit a high nuclear ATF2-to-MITF ratio were found to be associated with
metastatic disease and poor prognosis, further substantiating the significance
of MITF control by ATF2. In all, these findings provide genetic evidence for
the role of ATF2 in melanoma development and indicate an ATF2 function in
fine-tuning MITF expression, which is central to understanding MITF control at
the early phases of melanocyte transformation.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Go to:</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Introduction</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Malignant melanoma is
one of the most highly invasive and metastatic tumors [1], and its incidence
has been increasing at a higher rate than other cancers in recent years [2].
Significant advances in understanding melanoma biology have been made over the
past few years, thanks to identification of genetic changes along the MAPK
signaling pathway. Those include mutations in BRAF, NRAS, KIT and GNAQ, all of
which result in a constitutively active MAPK pathway [3]�[5]. Consequently,
corresponding transcription factor targets such as microphthalmia-associated
transcription factor (MITF) [6], AP2 [7], and C-JUN [8] and its heterodimeric
partner ATF2 [9] are activated and induce changes in cellular growth, motility
and resistance to external stress [10], [11]. In addition, constitutively
active MAPK/ERK causes rewiring of other signaling pathways [4]. Among examples
of rewired signaling is upregulation of C-JUN expression and activity [8],
which potentiates other pathways, including PDK1, AKT and PKC, and plays a
critical role in melanoma development [12].</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Activating transcription
factor 2 (ATF2), a member of the bZIP family, is activated by stress kinases
including JNK and p38 and is implicated in transcriptional regulation of
immediate early genes regulating stress and DNA damage responses [13]�[15] and
expression of cell cycle control proteins [16]. To activate transcription, ATF2
heterodimerizes with bZIP proteins, including C-JUN and CREB [17], [18], both
of which are constitutively upregulated in melanomas [8]. ATF2 is also
implicated in the DNA damage response through phosphorylation by ATM/ATR [19].
Knock-in mice expressing a form of ATF2 that cannot be phosphorylated by ATM
are more susceptible to tumor development [20]. Nuclear localization of ATF2 in
melanoma tumor cells is associated with poor prognosis [21], likely due to
transcriptional activity of constitutively active ATF2. Indeed, expression of
transcriptionally inactive ATF2 or peptides that attenuate endogenous ATF2
activity inhibits melanoma development and progression in xenograft models
[22]�[26]. These studies suggest that ATF2 is required for melanoma development
and progression.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>The transcription factor
MITF has been shown to play a central role in melanocyte biology and in
melanoma progression [27], [28]. Yet, the role of MITF in early stages of
melanoma development remains largely unexplored. Factors controlling MITF
transcription have been well documented and include transcriptional activators,
such as SOX10, CREB, PAX3, lymphoid enhancer-binding factor 1 (LEF1, also known
as TCF), onecut domain 2 (ONECUT-2) and MITF itself [29]�[33], as well as
factors that repress MITF transcription, including BRN2 and FOXD3 [34], [35].
In addition, MITF is subject to several post translational modifications which
affect its availability and activity, including acetylation, sumoylation and
ubiquitination [27], [28].</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>To directly assess the
importance of ATF2 in melanoma development, we employed a mouse melanoma model
in which ATF2 is selectively inactivated in melanocytes. We demonstrate that
melanoma development is markedly attenuated in mice expressing a
transcriptionally inactive form of ATF2 in melanocytes. Surprisingly, ATF2
control of melanoma development was mediated, in part, through its negative
regulation of SOX10 and consequently of MITF transcription. Inhibition of ATF2
abolished mutant BRAF-expressing melanocytes' ability to form foci on soft
agar, which was partially rescued when expression of MITF was attenuated. The
significance of these findings is underscored by our observation of human
melanoma tumors, in which high ratio of nuclear ATF2 to MITF expression was associated
with poor prognosis. These findings identify a novel mechanism underlying
melanocyte transformation and melanoma development.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Go to:</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Results</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Generation of
melanocyte-specific ATF2 mutant mice</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Global Atf2 knockout in
mice leads to early post-natal death [36]. Therefore, the Cre-loxP system was
utilized to disrupt Atf2 in melanocytes. Deletion of its DNA binding domain and
a portion of the leucine zipper motif results in a transcriptionally inactive
form of ATF2 (Figure 1a; [36]). To generate loss-of-function mutants, we
established mice that would allow CRE-dependent deletion of these domains. Mice
homozygous for the loxP-flanked (floxed) Atf2 gene (Atf2f/f) were born at the
expected Mendelian ratios and presented no apparent abnormalities. In addition,
in several tissues analyzed, ATF2 expression levels were comparable between WT
and Atf2f/f mice (data not shown).</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 1</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 1</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Melanoma development is
inhibited in Nras/Ink4a mice expressing a melanocyte-specific ATF2 mutation.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>To elucidate the role of
ATF2 in melanoma, Atf2f/f mice were crossed with mice harboring a
4-hydroxytamoxifen (OHT)-inducible Cre recombinase-estrogen receptor fusion
transgene under the control of the melanocyte-specific tyrosinase promoter,
designated Tyr::CreER(T2). Upon administration of OHT, we predicted that
CRE-mediated recombination would be induced in a spatially and temporally
controlled manner in embryonic melanoblasts, melanocytes, and in putative
melanocyte stem cells [37]. The resulting Atf2f/f/Tyr-CreER(T2) mice,
designated melanocyte-deleted (md) Atf2md), indeed expressed the gene encoding
the ATF2 transcriptional mutant in melanocytes. Immunoblot analysis of ATF2
protein confirmed that melanocytes prepared from wild-type
TyrCre+::Atf2+/+::Nras+::Ink4a&#8722;/&#8722; (WT) mice express a 70 kDa band
corresponding to full length ATF2, whereas melanocytes of
TyrCre+::Atf2md::Nras+::Ink4a&#8722;/&#8722; mice express only a 55 kDa band,
corresponding to the size of ATF2 lacking the DNA binding and leucine zipper
domains (Figure 1b).</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Disruption of ATF2 in
melanocytes inhibits melanoma formation</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>To address the role of
ATF2 in de novo melanoma formation Tyr::CreER::NrasQ61K::Ink4a&#8722;/&#8722;
(KO of exon 2�3 of Cdkn2a locus, which encodes for both p16Ink4a and p19Arf;
[38]) mice, which develop spontaneous melanoma (Lynda Chin, unpublished
observations), were crossed with Atf2md mice. Similar to findings reported by
Ackermann et al. [39], mutant N-Ras/Ink4a&#8722;/&#8722; mice developed
melanoma within 8�12 weeks with metastatic lesions often seen in the lymph
nodes. However, the incidence of melanoma was lower in
Tyr::CreER::NrasQ61K::Ink4a&#8722;/&#8722; mice used in the present study (50%
penetrance, of which 50% of the tumors were confirmed to be melanoma), probably
because expression of mutant NRAS was induced only after birth, as opposed to
activation of NRAS during embryogenesis, as reported in [39]). Thus,
Atf2md::N-RasQ61K::Ink4a&#8722;/&#8722; mice were used to assess changes in
melanoma incidence in the absence of functional ATF2 over a period of up to 8
months. In all cases, mouse skin was treated with Tamoxifen within 3�5 days
after birth to inactivate ATF2 (Figure 1b) and with doxycycline in their
drinking water to induce expression of the NRAS mutant transgene (See Materials
and Methods for details; Figure 1c). <span style='background:yellow'>In the
control group (Tyr::CreER::Atf2+/+::NrasQ61K::Ink4a&#8722;/&#8722;), 11/21 mice
(52%) developed tumors within 8�16 weeks (Table 1).</span> <span
style='background:yellow'>In ATF2 heterozygotes
(Tyr::CreER::Atf2&#8722;/+::NrasQ61K::Ink4a&#8722;/&#8722;), 18/44 mice (41%)
developed tumors within 8�16 weeks, and in the Tyr::CreER::Atf2md::NrasQ61K::Ink4a&#8722;/&#8722;
group only 3 of 21 animals (15%) developed tumors within 24�36 weeks (Figure 1d
and Table 1).</span> <span style='background:yellow'>To evaluate tumor type, we
examined melanoma markers including DCT and S100 in all tumors (Figure 1e,
Figure S1).</span> This analysis identified 55�63% of tumors as melanomas in
both the Atf2+/+ (7/11) and Atf2+/&#8722; (10/18) groups (Table 2). Only one of
the three tumors observed in the Atf2md group was identified as a melanoma.
Kaplan Meier curve did not reveal significant differences in survival among the
different genotypes, probably since this study was primarily designed to follow
tumor incidence. Common to all genotypes, most tumors that were not identified
as melanomas were fibrosarcomas and lymphomas, consistent with previous reports
[38]. These data suggest that transcriptionally active ATF2 is required for
melanoma development in the NrasQ61K::Ink4a&#8722;/&#8722; mouse melanoma
model.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Table 1</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Table 1</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Tumor incidence in mixed
genetic backgrounds.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Table 2</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Table 2</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'><span style='background:yellow'>Tumors positive for
melanoma markers S100 and DCT.</span></span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Identification of MITF
as an ATF2-regulated gene</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>To assess the mechanism
underlying ATF2's contribution to melanoma development, we conducted gene
profiling array analysis of primary melanocytes prepared from
Tyr::Cre+::Atf2+/+::NrasQ61K::Ink4a&#8722;/&#8722; and
Tyr::Cre+::Atf2md::NrasQ61K::Ink4a&#8722;/&#8722; mice. Analysis was limited to
melanocytes, since, as reported above, only one melanoma formed in the ATF2
mutant group. In all cases, ATF2 was inactivated and NRAS was induced in
culture within 48h of plating cells, as monitored by western blots (Figure 1b,
1c and data not shown). Melanocytes were enriched, and immunostaining with
appropriate markers confirmed that samples were free of keratinocytes and
fibroblasts (data not shown; see Materials and Methods for details). RNA was
prepared from cultures and two biological and technical replicates were used
for data analysis. As shown in Table 3, among transcripts differentially
expressed in ATF2 WT and mutant cultures were several factors that play an important
role in melanocyte pigmentation, including Mitf, Silver, Tyrp1 and Dct. qPCR
analysis, performed on independently prepared RNA samples from melanocytes
expressing WT (Tyr::Cre+::Atf2+/+::NrasQ61K::Ink4a&#8722;/&#8722;) or mutant
ATF2 (Tyr::Cre+::Atf2md::NrasQ61K::Ink4a&#8722;/&#8722;), confirmed altered
expression of pigmentation genes (Table 3). These data provide the initial
indication that ATF2 negatively regulates Mitf and several other important
pigmentation genes. As the pigmentation genes identified in this array are
known to be regulated by MITF [27], we focused on regulation of MITF by ATF2.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Table 3</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Table 3</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Microarray analysis of
ATF2+/+TyrCre+Nras+Ink4a&#8722;/&#8722; and
ATF2&#8722;/&#8722;TyrCre+Nras+Ink4a&#8722;/&#8722;melanocytes and confirmation
by qPCR.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>MITF is negatively
regulated by ATF2 in mouse and human melanocytes</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>To confirm that ATF2
negatively regulates Mitf expression, we assessed MITF transcription in primary
mouse melanocytes harboring WT (Tyr::Cre&#8722;::Atf2+/+) or mutant
(Tyr::Cre+::Atf2md) forms of ATF2. RNA prepared from whole skin of these mice
(3 mice per group) was subjected qPCR analysis. Significantly, Mitf expression
was inversely correlated to the presence of functional ATF2; samples obtained
from ATF2 mutant skin exhibited a greater than 2-fold increase in MITF
expression compared with those obtained from WT ATF2 mice (Figure 2a).
Likewise, we found that genes transcriptionally regulated by MITF, such as Dct,
Silver and Tyrp1, were upregulated in the skin of mutant ATF2 mice (Figure 2a).
The degree of altered expression of pigmentation genes was less pronounced in
whole skin samples than in cultured melanocytes (Table 3), probably due to
confounding effects of in vitro cell culture. To confirm the qPCR data, we
performed immunostaining of skin tissue samples obtained from 4 days old WT or
ATF2 mutant mice and observed increased MITF expression in melanocytes from
Atf2md mice relative to their WT counterparts (Figure 2b). Quantification of
MITF staining revealed an approximate 2-fold increase in nuclear MITF
expression in Atf2md compared to WT mice (Figure 2c). Of note, the level of
S100 staining in the hair matrix was markedly reduced in the skin of Atf2md
mice. At a later time point (2 weeks) representing an advanced stage of
melanocyte development, S100 staining was similar in both genotypes, while MITF
expression remained upregulated in Atf2md mice (not shown). In all, these data
confirm our initial observations in primary mouse melanocytes that MITF levels
are elevated in ATF2 mutant-expressing cells.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 2</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 2</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>ATF2 negatively
regulates MITF in melanocytes.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Additional assessment
was performed in melan-Ink4a-Arf1 melanocytes, a line derived from black
Ink4a-Arf null mice [40], and in primary human melanocytes. In both, ATF2
expression was inhibited by viral infection with the corresponding mouse or
human shRNA (shATF2). Infection of either primary human (Figure 3a) or
melan-Ink4a-Arf1 melanocytes (Figure 3b) with shATF2 markedly increased MITF
transcription and protein expression (Figure 3a, 3b). These findings show that
loss of transcriptionally active ATF2 allows higher expression of MITF and
strongly suggest that ATF2 negatively regulates MITF expression in melanocytes.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 3</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 3</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>ATF2 regulates MITF in
melanocytes and melanoma cells.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>MITF transcription is
negatively regulated by ATF2 in about 50% of human melanoma cells</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Given that ATF2
negatively regulates MITF in melanocytes of mouse and human tissues and in
related melanocyte cell lines, we asked whether ATF2 also regulates MITF in
human melanoma cells. Initially, we assessed changes in MITF expression in six
human melanoma lines harboring oncogenic mutations in BRAF or NRAS, and in
which ATF2 expression was effectively inhibited by corresponding shRNA
(shATF2). In all cases, shRNA specificity was confirmed using three independent
sequences (data not shown). Surprisingly, the six melanoma lines fell into two
classes based on distinct patterns of regulation of MITF by ATF2 (Table 4). The
first class comprised four of the six melanoma cultures (1205Lu, WM35, WM793
and WM1361), in which MITF expression was elevated 3�6-fold following
inhibition of ATF2 expression (Figure 3c, S2a). Conversely, a second class of
cells, including MeWo and 501Mel cells, exhibited decreased MITF expression
after ATF2 knockdown (KD), suggesting positive regulation of MITF by ATF2
(Figure 3d, S2b). Notably, this latter group showed high levels of basal MITF
expression [41], [42], suggesting that regulation of MITF expression in these
cells differs mechanistically from that of the first group. Further, in
response to stress (UV or hypoxia) the MeWo and 501Mel lines further reduced
MITF expression (Figure 3d and data not shown), providing further evidence for
differential regulation of MITF in these cells both prior to and in response to
stress stimuli. Additional analyses were performed, employing 12 more melanoma
cell lines. Inhibition of ATF2 expression revealed that 4/12 exhibited increase
in MITF expression, while 6/12 decreased MITF expression. Two of the 12 lines
did not exhibit change in MITF expression following ATF2 KD (Table 4, S5).
Collectively, out of 18 melanoma lines we found that 8 (44%) retained similar
negative regulation of MITF by ATF2 as observed in the melanocytes. However,
another 8 (44%) exhibited positive regulation of MITF by ATF2, pointing to a
transcriptional switch that occurred in the course of melanocyte
transformation. MITF was not affected by altered ATF2 expression in 2/18 cell
lines (Table 4, Figure S5). In all, in about 50% of the melanoma cell lines
ATF2 elicits negative regulation of MITF, similar to what was seen in human and
mouse melanocytes.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Table 4</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Table 4</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>MITF and SOX10 mRNA
levels in melanocytes and melanoma cell lines following ATF2 KD.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>ATF2 regulation of MITF
is mediated by discrete promoter elements</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>MITF transcription is
regulated by complex positive and negative cues [27]. For instance, while CREB
and SOX10 positively regulate MITF, BRN2 and FOXD3 have been shown to
downregulate MITF expression [29], [30], [34], [35]. Hence we used melanocytes
and representative melanoma lines to assess mechanisms underlying positive or
negative regulation of MITF. Infection of the human melanocyte line Hermes 3A
with shATF2 effectively inhibited ATF2 expression, upregulated Mitf
transcription and increased transcription of SOX10 and FOXD3 (from 7- to
10-fold) and to a lesser extent of Pax3 and Brn2 (from 1.5- to 2-fold) (Figure
4a, S3a). Similarly, inhibition of ATF2 transcription in human melanoma 1361
cells increased SOX10 and FOXD3 transcription, albeit, to a lesser degree (3-
and 1.5-fold, respectively) compared with human melanocytes (Figure S3b).
Neither BRN2 nor PAX3 transcription was elevated in melanoma cells in which
ATF2 expression was inhibited (Figure S3a). These observations suggest a role
for ATF2 in FOXD3- and SOX10-mediated regulation of MITF transcription in
melanocytes and melanoma cells.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 4</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 4</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Negative regulation of
MITF by ATF2 is mediated by SOX10.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>To assess the possible
role of FOXD3 in regulation of MITF we inhibited FOXD3 expression in
melanocytes expressing control shRNA and shATF2. Inhibition of FOXD3 expression
increased SOX10 transcription and protein expression, albeit to lower levels
compared with inhibition of ATF2 expression (Figure S4). Concomitant increase
of MITF RNA and protein levels was also lower, compared with that seen upon
inhibition of ATF2 expression. Notably, inhibition of both ATF2 and FOXD3
resulted in additive increase of SOX10 and MITF (Figure S4). These data suggest
that FOXD3 may also contribute to negative regulation of MITF in melanocytes,
independent of ATF2. Since inhibition of FOXD3 elicited a less pronounced
effect compared with ATF2, and since the effect appeared ATF2-independent and
furthermore did not appear to mediate similar changes in human melanoma cells
(Figure S3b and data not shown), we focused on assessment of direct mechanisms
underlying ATF2 effect on MITF transcription.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>To this end we first
analyzed MITF promoter sequences for ATF2/CRE elements (Cyclic AMP response
element), which can be targeted by ATF2, as well as sequences recognized by
BRN2 and SOX10 using a luciferase reporter construct (MITF-Luc) [43]. Using
either a wild-type (WT) construct or one in which the BRN2 site was mutated, we
observed increased luciferase activity following inhibition of ATF2
transcription in WM1361 melanoma (Figure 4b), as well as in LU1205 and WM35
melanoma cells (data not shown). The relative increase in luciferase activity
following ATF2 inhibition was equivalent in both constructs, suggesting that an
ATF2 effect is not mediated by BRN2 (Figure 4b, left panel). Similarly, MITF
transcriptional activities were altered to a similar degree following
inactivation of the CRE element (Figure 4b, right panel), suggesting that ATF2
down-regulation of the MITF promoter is indirect. We therefore assessed whether
SOX10, which positively regulates MITF and whose transcription markedly
increases in melanocytes and melanoma cells in which ATF2 expression is
inhibited (Figure 4a, S3b), may mediate ATF2 effect on MITF transcription.
Analysis of a MITF-Luc construct harboring a mutant SOX10 binding site revealed
that ATF2 inhibition no longer elicited increased MITF transcription in human
melanocytes or in melanoma cells (Figure 4c). In agreement, inhibition of SOX10
expression by corresponding siRNA attenuated the increase in MITF transcription
seen in shATF2-expressing human melanocytes (Figure 4d) or melanoma cells
(Figure 4e). These results suggest that ATF2 regulation of MITF transcription
is mediated by SOX10. In agreement, chromatin immunoprecipitation (ChIP) assays
confirmed increased binding of SOX10 to the MITF promoter in melanoma cells
expressing shATF2 (Figure 5a).</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 5</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 5</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>ATF2 and JunB negatively
regulate SOX10 transcription, with concomitant effect on MITF.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>A putative response
element for AP1 (which can serve as an ATF2 response element through ATF2
heterodimerization with JUN family members; [9]) has been identified in
upstream regions of the Sox10 promoter [44]. We examined potential ATF2 binding
to this element by ChIP and found that endogenous ATF2, but not ATFa, binds to
that AP1 sequence (&#8722;4797�4791) in both human melanocytes and melanoma
cells (Figure 5b). We next set to identify ATF2 heterodimeric partner, which
could mediate negative regulation of SOX10 transcription. Among members of the
JUN family implicated in transcriptional silencing is JunB. Thus, further
assessment was performed to determine if JunB functions as an ATF2
heterodimerization partner to regulate SOX10 transcription through the AP1
site. ChIP analysis confirmed that JunB binds to the AP1 site found in SOX10
promoter sequences (Figure 5c). To confirm a possible role for JunB in
regulating MITF transcription we asked whether expression of TAM67, a negative
regulator of Jun family members, could attenuate the binding and
transcriptional activities elicited by JunB. Expression of TAM67 indeed reduced
the degree of ATF2 and JunB binding to the AP1 site on SOX10 promoter. Further,
KD of ATF2 expression abolished binding of both ATF2 and JunB to the AP1 site
on the SOX10 promoter (Figure 5c). These data confirm the presence of ATF2-JunB
complex on Sox10 promoter and suggest that ATF2 recruits JunB for binding to
the AP1 site on SOX10 promoter. To assess the role of JunB on SOX10
transcription we have monitored changes in Sox10 expression at the protein and
RNA levels. Expression of TAM67 caused increased expression of SOX10 in both
human melanoma (</span><span lang=EN style='font-size:12.0pt;line-height:115%;
font-family:"Cambria","serif"'>&#8764;</span><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>2 folds; Figure 5d) and
melanocytes (</span><span lang=EN style='font-size:12.0pt;line-height:115%;
font-family:"Cambria","serif"'>&#8764;</span><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>3 folds; Figure 5e),
indicating some relief of JunB inhibition. Co-expression of TAM67 with Jun B
attenuated this increase, reducing the level of Sox10 expression to basal
levels (Figure 5d, 5e). Over-expression of JunB, but not Jun D, effectively
inhibited Sox10 expression in both the melanoma and melanocytes cells (Figure
5d, 5e). These data suggest that JunB mediates inhibition of Sox10 expression.
To further reveal the role of ATF2 in this inhibition, we assessed the effect
of JunB on Sox10 expression in cells expressing control shRNA or shATF2. While
ectopic expression of JunB reduced the expression of Sox10 in control
shRNA-expressing cells, such decrease was no longer seen in cells expressing
shATF2 (Figure 5f). Collectively, these findings suggest that ATF2, in concert
with JunB, is responsible for inhibition of Sox10 expression.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>We next assessed the
effect of ATF2 on SOX10 and MITF expression in 12 additional human melanoma
cell lines. In all cases cells were infected with shATF2 and changes in SOX10
and MITF were monitored at the level of RNA. Notably, about 4/12 melanoma lines
revealed increase in both SOX10 and MITF expression upon KD of ATF2 (Figure S5,
Table 4). In contrast, 6/12 melanoma lines revealed decrease in MITF
expression, of which 5 also shown decrease in SOX10 expression, pointing to
positive regulation of SOX10 and MITF in these melanoma cells. In two out of
the 12 melanoma lines ATF2 affected SOX10 but not MITF transcription (Figure
S5). Overall, our cohort of 18 melanoma lines revealed that about 50% of the
melanomas retained negative regulation of MITF by ATF2, as seen in the
melanocytes (primary and cell lines) (Table 4).</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>To further assess
whether ATF2 regulation of MITF is SOX10-dependent in melanocytes and melanoma
cells, we coexpressed SOX10 in shATF2-expressing cells. As seen in earlier
analysis, inhibition of ATF2 expression caused increase in MITF transcription
in the human melanocytes and 4 melanoma cell lines, (WM1361, WM793, LU1205,
WM35; Figure S6). Notably, the melanocytes and 2/4 melanoma cell lines revealed
ATF2 effect on MITF expression is SOX10-dependent (WM1361, WM793; Figure S6).
Two of the four melanoma cell lines did not reveal increased SOX10 expression,
although they retained increased MITF expression, upon inhibition of ATF2
(Lu1205, WM35; Figure S6). These findings confirm that while in melanocytes,
expression of SOX10 and MITF is negatively regulated by ATF2, this mechanism is
conserved in approximately half of melanomas surveyed.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Along these lines, the
two melanoma lines (MeWo and 501 Mel) that exhibit positive regulation of MITF
by ATF2 also exhibited positive regulation of SOX10 by ATF2 (Figure S7).
Inhibition of ATF2 expression reduced SOX10 and MITF RNA and protein levels
(Figure S7a�c). In order to determine whether JunB lost its ability to elicit
negative regulation of SOX10 and MITF in melanoma cells where ATF2 no longer
inhibited SOX10 or MITF expression, we transfected those cell lines with TAM67
and JunB alone and in combination. In these cells, whereas TAM67 effectively
attenuated Sox10 and MITF expression, JunB did not alter expression of these
genes, suggesting that positive regulation of MITF and SOX10 by ATF2 depends on
other members of the Jun family of transcription factors (Figure S7d).
Conversely, TAM67 or JunB had no effect on melanoma cells in which ATF2
inhibits MITF independently of SOX10, suggesting that in these cases, ATF2
likely cooperates with transcription factors other than JunB to elicit negative
regulation of SOX10 and MITF (Figure S7d). Consistent with this observation, ChIP
assay confirmed ATF2 and CREB, but not JunB, binding to the Sox10 promoter in
these cells (Figure S7e). These findings suggest that changes in ATF2
heterodimeric partner (from JunB to CREB) are likely to cause the switch from
negative to positive regulation of SOX10, and in turn, MITF (see below). The
possibility that altered expression of JunB may account for ATF2 positive or
negative regulation of Sox10 and MITF were excluded, as no clear correlation
between JunB expression and the ability of ATF2 to elicit negative regulation
of Sox10/MITF were seen (Figure S7f).</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Among response elements
potentially required to upregulate MITF transcription is the CRE element, which
is implicated in CREB-mediated upregulation of MITF transcription [45].
Although transcriptional activity from a CRE mutant MITF promoter was lower
compared to the WT promoter (30%), it was no longer responsive to inhibition of
ATF2 expression in the MeWo cells (Figure S8a). Pull-down assays using
biotin-tagged MITF promoter sequences harboring the CRE identified ATF2 and
CREB as CRE-bound proteins in MeWo melanoma cells (Figure S8b). In agreement,
ChIP analysis confirmed occupancy of the CRE site on MITF promoter by ATF2
(Figure S8c). These findings are consistent with the fact that ATF2 heterodimerizes
with CREB [9] and with a report that p38/MAPK14 (which phosphorylates ATF2)
plays an important role in MITF transcription dependent on the CRE site [46].
These results establish that ATF2-dependent activation of MITF transcription in
these melanoma cells is mediated through the CRE site, likely in cooperation
with CREB. Notably, MeWo and 501Mel lines are known to express high MITF levels
compared to other melanoma lines [41], [42], suggesting these cells harbor
distinct mechanisms that preclude negative regulation of MITF by ATF2.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Inhibition of MITF
expression rescues focus formation on soft agar in shATF2-expressing
melanocytes</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>To determine whether the
contribution of ATF2 to melanocyte transformation and development is
MITF-dependent, we assessed melanocytes' ability to grow and form colonies in
soft agar, which is indicative of their transformed potential. Expression of
mutant BRAFV600E in immortal melanocytes is reportedly sufficient for growth on
soft agar [47]. Thus we infected melan-Ink4a-Arf1 melanocytes with mutant BRAF
(Figure 6a) and confirmed their ability to form colonies in soft agar. Mutant
BRAF expression effectively caused formation of about 1000 colonies per 5000
cells (Figure 6b, 6c). In contrast, melanocytes infected with BRAF600E and with
shATF2 formed on average about 20 colonies, indicative of loss of
tumorigenicity (Figure 6b, 6c, 6d) and consistent with our initial observation
that the number of melanoma tumors significantly decreases in the absence of
transcriptionally functional ATF2 (Tables 1�2). To determine the importance of
MITF at this early stage of melanocyte transformation we inhibited MITF
expression (using shRNA) in melanocytes expressing mutant BRAF alone or mutant
BRAF+shATF2. Significantly, inhibition of MITF expression decreased the number
of BRAF-induced foci (from 1000 to about 100 per well). Over-expression of MITF
in BRAF-expressing melanocytes also inhibited focus formation, to a degree
similar to that seen following inhibition of MITF expression (Figure 6b, 6c,
6d). This observation implies that effective inhibition or overexpression of
MITF attenuates melanocyte transformation, consistent with previous reports
(52). Remarkably, inhibition of MITF expression in melanocytes expressing both
mutant BRAF and shATF2 rescued, at least partially, melanocytes' ability to
form foci on soft agar (400 compared with 20 seen in shATF2 cells; Figure 6b,
6c, 6d). These findings suggest that inhibition of MITF expression in
melanocytes lacking ATF2 expression can promote transformation. That MITF
inhibition in melanocytes expressing ATF2 WT can attenuate their ability to
form foci on soft agar is attributable to the relative expression of MITF RNA
and protein in each condition (Figure 6d). MITF expression levels in ATF2 KD
cells increased 7.5-fold compared with control BRAF-expressing melanocytes.
Inhibition of MITF expression in ATF2 KD melanocytes reduced MITF expression
2.5-fold relative to controls, whereas MITF KD alone resulted in lower MITF
expression (5-fold; Figure 6d). Thus, complete abrogation of MITF expression
attenuates melanocyte transformation, whereas low to moderate levels of MITF
expression are sufficient to promote growth on soft agar. Higher MITF
expression levels, as seen in ATF2 KD cells, result in a total loss of
melanocytes' ability to form foci on soft agar. These findings are in line with
the proposed rheostat model in which medium levels of MITF are optimal for
growth and melanoma development [48] and in agreement with our observations in
a mouse melanoma model.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 6</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 6</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>MITF down-regulation
partially rescues colony formation by BRAFV600E-expressing mouse melanocytes
with inhibited ATF2 expression.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>We next assessed whether
inhibition of melanocyte growth on soft agar by altered ATF2 and/or MITF
expression can be attributed to decreased proliferation or increased apoptosis.
Inhibition of ATF2 expression caused notable accumulation of cells in G2 (60%),
with significant cell death induction (22%) compared to controls (4%), (Figure
6e, 6f). Interestingly, such altered cell cycle distribution and cell death
rate were associated with a significant increase in MITF protein levels (Figure
6d). In contrast, inhibition of MITF expression did not significantly induce
cell death (6.5%) but resulted in fewer cells in G2/M-phase and more cells in
G1, compared with inhibition of ATF2 alone. These observations suggest that
MITF inhibition is sufficient to reduce the rate of cell cycle progression
through G2/M phase and that inhibited growth of BRAF600E-expressing melanocytes
on soft agar may be attributed to abrogation of distinct cell cycle-regulatory
mechanisms. Combined inhibition of ATF2 and MITF restored cell cycle
distribution to that seen in control melanocytes, and reduced cell death from
22.4% to 12.9%. Of interest, MITF overexpression promoted a similar degree of
cell death (11.4%) without altering cell cycle distribution, similar to
combined inhibition of ATF2 and MITF (Figure 6e, 6f). Together, these
observations suggest that simultaneous inhibition of ATF2 and MITF averts cell
cycle abrogation induced when expression of either of these factors is
perturbed individually, further substantiating regulation of MITF by ATF2.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Low nuclear MITF
expression in melanoma tumors that exhibit strong nuclear ATF2 expression is
associated with poor prognosis</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>The availability of a
melanoma TMA, consisting of over 500 melanoma samples and in which expression
of both ATF2 and MITF in the same tumors had been measured enabled us to assess
possible associations between ATF2 and MITF and their correlation with survival
and other clinical and pathological factors. Our earlier studies revealed that
ATF2 subcellular localization in tumors is significantly correlated with
prognosis: nuclear localization, reflecting constitutively active ATF2, was
associated with metastasizing tumors and poor outcome [7]. Here we quantitated
immunofluorescent staining of TMAs for MITF and ATF2 by employing our
automated, quantitative (AQUA) method. To normalize ATF2 and MITF levels, expression
of each of the two proteins in individual patients was divided by the median
expression level of the respective protein in all patients, and the nuclear
ATF2/MITF ratio was calculated and log-transformed. By ANOVA analysis, the
ratio was higher in metastatic than in primary specimens (t
value&#8202;=&#8202;2.823, P&#8202;=&#8202;0.0051), as shown in Figure 7a. No
association was found between nuclear ATF2/MITF ratio and disease-specific
survival among patients with metastatic melanoma (not shown). Significantly, a
high nuclear ATF2/MITF ratio in primary melanoma specimens was associated with
decreased 10-year disease-specific survival (P&#8202;=&#8202;0.0014; Figure
7b). On Cox multivariable analysis, this association with survival was
independent of patient age, Breslow thickness or the presence or absence of
ulceration (data not shown). Nuclear ATF2 alone in primary specimens was
associated with poor survival, but to a lesser degree than the ratio of nuclear
ATF2/MITF (P&#8202;=&#8202;0.0118 for ATF2 as a single discriminator versus
P&#8202;=&#8202;0.0014 for the ratio of nuclear ATF2/MITF). Nuclear MITF as a
single discriminator was not a significant predictor of survival
(P&#8202;=&#8202;0.185), as was reported previously using immunohistochemistry
[49]. These observations suggest that active (nuclear) ATF2 in melanoma can
suppress MITF expression, and that this phenomenon is associated with poor
prognosis.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 7</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure 7</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Analysis of nuclear ATF2
and MITF in melanoma specimens using quantitative immunofluorescence.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Go to:</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Discussion</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Identifying mechanisms
underlying early phases of melanocyte transformation and melanoma development
is central to understanding the etiology of this devastating tumor, as well as
for developing novel treatment approaches. Previous studies indicate the
presence of mutant BRAF in melanocytic lesions, as well as its effect on
pigment gene expression [6], [50], [51]. The present study enhances our
understanding of early events contributing to melanoma development. We
demonstrate that loss of a transcriptionally active form of ATF2 in melanocytes
inhibits melanoma development in an Nras/Ink4a model. Our quest to understand
mechanisms underlying ATF2 activity in this process led us to identify an
important role for ATF2 regulation of MITF, an important regulator of
melanocyte biogenesis and a factor implicated in melanoma progression [49].
Surprisingly, ATF2 negatively regulated MITF expression in mouse and human
melanocytes, suggesting that ATF2 transcriptional activities limit MITF
expression. We demonstrate that such negative regulation is elicited through
downregulation of SOX10 by ATF2, in cooperation with JunB. A putative AP1
response element has been identified in SOX10 promoter sequences and ChIP
analysis of this domain showed ATF2 and JunB binding. Overexpression of JunB
efficiently suppressed SOX10 expression in an ATF2-dependent manner and
inhibition of Jun transcriptional activities phenocopied the effect of shATF2,
suggesting that negative regulation of SOX10 by ATF2 is direct, and is mediated
in cooperation with JunB.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Importantly, ATF2-dependent
negative regulation of Sox10 and consequently of MITF seen in melanocytes, but
only in about 50% of the 18 melanoma cell lines studied here. Correspondingly,
JunB, which is required for ATF2-dependent inhibition of Sox10 transcription,
is no longer found on the promoter of SOX10 in melanoma cells (i.e. 501Mel)
that exhibit positive regulation by ATF2. Rather, CREB and ATF2 are found on
SOX10 and MITF promoters, pointing to a switch in ATF2 heterodimeric partners
to enable positive regulation of these genes. Notably, melanoma cell lines that
exhibit positive regulation of SOX10 and MITF by ATF2, also show high basal
levels of MITF expression [41], [42], suggesting that additional genetic or
epigenetic changes distinguish these lines from melanocytes and the other
melanoma lines in which ATF2 elicits negative regulation of MITF.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Notably, ATF2 control of
MITF expression affected the ability of BRAF600E-expressing melanocytes to
exhibit transformed phenotype in culture, monitored by their ability to grow on
soft agar. Inhibition of ATF2 abolished soft agar growth of BRAF600E-expressing
melanocytes, which was partially rescued upon KD of MITF. Interestingly, both
the over expression or the KD of MITF resulted in inhibition of melanocytes
ability to grow on soft agar, substantiating the notion that a fine balance of
MITF expression must be maintained in order to ensure its contribution to
cellular proliferation and transformation. We propose that excessively low or
high MITF levels block melanocyte transformation, whereas intermediate levels
allow transformation to occur. Overall, our observations demonstrate that ATF2
plays an important role in fine-tuning those levels and support the rheostat
model proposed for MITF's role in melanoma development and progression [48]. Of
importance, ATF2 and MITF affect the ability of BRAF600E-expressing melanocytes
to grow on soft agar via distinct mechanisms. Whereas specific inhibition of
ATF2 causes both accumulation of cells in G2 and induction of cell death, specific
alteration of MITF protein levels�particularly depletion�significantly affects
cell proliferation and inhibit growth on soft agar by non-lethally slowing cell
cycle progression at G2/M. These observations are consistent with a report from
Wellbrock and Marais [52], who showed that altered MITF expression inhibits
melanocyte proliferation.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Importantly, inhibiting
MITF expression in ATF2 KD melanocytes was sufficient to partially rescue
melanocyte growth on soft agar. While supportive of our finding in the
Nras::Ink4a mouse melanoma model, where expression of transcriptionally
inactive ATF2 inhibits melanoma formation, these observations provide the
foundation for a model in which ATF2 inhibition causes increased MITF levels
and concomitant inhibition of melanocyte growth, possible induction of cell
death and delayed development. The latter is suggested by IHC analysis of mouse
skin from ATF2md mice, which shows notably reduced S100 staining indicative of
delayed melanocyte development: ATF2 KO melanocytes appear to represent anagen
stage IV, whereas WT represent anagen stage VI. This delay was seen at the 4-
but not the 14-day time point, suggesting that an ATF2 effect might be limited
to a specific subpopulation or phase of melanocyte development. The early (4
day) time point is within the time frame that allows induction of melanoma
development by UV-irradiation of c-Met or H-Ras mutant mice [53]. It is
therefore plausible that timely control of MITF expression by ATF2 determines
melanocyte susceptibility to transformation.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Our analysis of genes
whose expression is altered by ATF2 KD in melanocytes identified a cluster of
pigmentation genes, many reportedly regulated by MITF [6], [54]. Therefore,
changes in TYRP1, DCT and SILVER expression could be attributed to altered MITF
expression. However, initial analysis points to a more complex mechanism since
(i) the degree of changes in expression of these genes was often greater than
that seen for MITF and (ii) expression of some pigmentation genes was found to
be independent of MITF in some melanoma and melanocyte cultures. Hence, further
studies are required to address mechanisms underlying ATF2 regulation of these
pigmentation genes and the significance of such regulation to melanocyte
transformation and melanoma development. While our present studies focused on
the ATF2-MITF axis, it is expected that additional ATF2-regulated genes
contribute to melanoma development [12]. In agreement, our earlier studies
using both human and mouse melanoma lines demonstrate that inhibition of ATF2
effectively inhibits tumorigenesis and blocks metastasis [22]�[26].</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Important for ATF2
function is its subcellular localization. While findings presented here
position ATF2 as an oncogene functioning in melanocyte transformation and
melanoma development, earlier studies from our laboratory and others suggest
that in keratinocytes and mammary glands, ATF2 elicits a tumor suppressor
function [55], [56]. Of interest, assessing the localization of ATF2 in the
melanoma cell lines studied here revealed that all express nuclear ATF2.
Interestingly, in most cases the nuclear staining revealed a punctate staining,
resembling the localization of ATF2 to DNA repair foci following DNA damage
(Figure S9). A possible link between the presence of ATF2 in repair foci in
most melanoma cells points to the possible presence of activated DNA damage
response which may be associated with genomic instability [19], [20]�aspects
that will be explored in future studies. Significantly, the appearance of nuclear
ATF2 is correlated with poor prognosis in melanoma, whereas melanomas that
exhibit cytosolic ATF2 exhibit a better survival. Notably, cytosolic ATF2 is
primarily seen in non-malignant skin tumors [55]. Here we demonstrate that high
nuclear ATF2/MITF ratios are associated with poor prognosis in primary
melanomas, but not with metastatic melanomas. The latter finding attests for
the important role ATF2 plays to control MITF expression in the early phase of
melanocyte transformation and melanoma development.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Overall, using the
mutant Nras/Ink4a melanoma model we provide genetic evidence for a central role
for ATF2 in melanoma development. We demonstrate that in the absence of
transcriptionally active ATF2, melanoma formation is largely inhibited.
Furthermore, our data point to an unexpected role of ATF2 in fine-tuning of
MITF transcription through regulation of its positive regulator SOX10. Mouse
melanoma models and in vitro transformation studies indicate that this newly
identified regulatory pathway is required for early phases of melanocyte
transformation. Given that ATF2 affects activity of the oncogenes N-Ras (mouse
model) and BRAF (melanocyte growth on soft agar); we expect that ATF2 play
significant roles in melanomas that carry either of these mutations.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Go to:</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Materials and Methods</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Ethics statement</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Research involving human
participants has been approved by the institutional review board at Yale
University (where the TMA was prepared and analyzed). All animal work has been
conducted according to relevant national and international guidelines in
accordance with recommendations of the Weatherall report and approved by the
IACUC committee at SBMRI.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Animal treatment and
tumor induction protocols</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Mice bearing a
conditional allele for mutant ATF2 in which the DNA binding domain and part of
the leucine zipper domain were deleted, were generated as previously described
[36], [55]. To study the function of ATF2 in melanocytes, we utilized the
Cre-loxP system for disruption of the ATF2 gene in melanocytes [37]. The
Tyr::CreER::Atf2md mice and their littermate controls (WT) were of
FVB/129P2/OlaHsd (TyrCreERT mice were FVB, ATF2fl/fl were 129P2/OlaHsd) and
N-Ras/Ink4a&#8722;/&#8722; mice were C57Bl/6/129SvJ. For melanoma studies we
have used Tyr::CreER::NrasQ61K::Ink4a&#8722;/&#8722; mice (developed at HMS by
LC) following their cross with the Tyr::CreER::Atf2md mice.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Immunohistochemistry</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Skin specimens were
fixed in neutral buffered formalin solution and processed for paraffin
embedding. Skin sections (5 �m in thickness) were prepared and deparaffinized
using xylene. For MITF, DCT and S100 immunostaining, tissue sections were
incubated in DAKO antigen retrieval solution, for 20 min in a boiling bath,
followed by treatment with 3% hydrogen peroxide for 20 min. Antibodies against
MITF (1</span><span lang=EN style='font-size:12.0pt;line-height:115%;
font-family:"Cambria","serif"'>&#8758;100 from Sigma), DCT (1&#8758;500, kind
gift from Dr. Vincent Hearing) and S100 (1&#8758;100, DAKOCytomation;
Carpinteria, CA) were allowed to react with tissue sections at 4�C overnight.
Biotinylated anti-rabbit IgG was allowed to react for 30 min at room
temperature and diaminobenzidineor Nova Red were used for the color reaction.
Hematoxylin was used for counterstaining. The control sections were treated
with normal mouse serum or normal rabbit serum instead of each antibody.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Cell culture</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Immortalized human melanocytes
Hermes 3A which has hTERT (puro) and CDK4 (neo) expression [57] were grown in
RPMI 1640 medium containing Fetal Bovine Serum (FBS, 10%),
12-O-tetradecanoyl-phorbol-13-acetate (TPA, 200 nM, Sigma, St. Louis, MO),
Cholera toxin (200 pM, Sigma), human stem cell factor (10 ng/ml, R&amp;D
systems, Minneapolis, MA), and endothelin 1 (10 nM, Bachem Bioscience Inc.,
Torrance, CA). Primary human melanocytes (NEM-LP; Invitrogen) were grown in
medium 254 and HMGS (Cascade Biologics). Mouse melanocytes (melan-Ink4a-Arf1)
were grown as for immortalized human melanocytes excluding human stem cell
factor and endothelin. Melanoma cell lines were grown in DMEM medium
supplemented with 10% FBS and penicillin/streptomycin (P/S; Cellgro). Melanoma
cell lines used in this study LU1205, WM793, 501MEL, WM35, WM1361, MeWO (kind
gift from Meenhard Herlyn), UACC903 were maintained in DMEM medium supplemented
with 10% FBS and Penicillin/Streptomycin. Melanoma cell lines SbCl2, WM9, WM4,
WM1650, A2068, WM1366, WM3629, WM1552, SKMEL2, SKMEL5, and SKMEL8 were
maintained in RPMI medium supplemented with 10% FBS and
Penicillin/Streptomycin. Primary melanocytes cultures were prepared from mice
carrying the Atf2 WT or mutant genotypes and N-Ras/Ink4a</span><span lang=EN
style='font-size:12.0pt;line-height:115%;font-family:"Cambria","serif"'>&#8722;</span><span
lang=EN style='font-size:12.0pt;line-height:115%;font-family:"Cambria","serif"'>/</span><span
lang=EN style='font-size:12.0pt;line-height:115%;font-family:"Cambria","serif"'>&#8722;</span><span
lang=EN style='font-size:12.0pt;line-height:115%;font-family:"Cambria","serif"'>
as follows. Dorsal-lateral skin was removed from one day-old pups, disinfected
with 70% ethanol for 1 min and then washed at least twice with sterile PBS. The
skin was submerged in 1� Trypsin/EDTA overnight at 4�C and next day, the skin
was placed in a Petri dish with mouse melanocyte culture medium (described
below). The epidermis and sheared tissue was removed and discarded with
forceps. The tissue was transferred to 15 ml centrifuge tubes and vortexed
vigorously until solution becomes cloudy (1�2 min). The cell suspension was
transferred to tissue culture flasks. After 3 days, melanocyte growth medium
containing 0.8 �g/ml geneticin (Sigma-Aldrich) was added to eliminate
contaminating fibroblasts (melanocytes are resistant to such treatment).
Geneticin-containing medium was removed and replaced with fresh media after 1
day. Media was changed twice a week. Primary mouse melanocytes were grown in
F-12 media (Invitrogen) containing 20% L-15 media (Invitrogen), 4% of FBS and
Horse serum (Invitrogen), Penicillin (100 units) and streptomycin (50 �g)
antibiotics, db-cAMP (40 �M, Sigma-Aldrich),
12-O-tetradecanoyl-phorbol-13-acetate (TPA, 50 ng/ml, Sigma-Aldrich),
alpha-Melanocyte stimulating hormone (</span><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&#945;-MSH, 80 nM,
Sigma-Aldrich), Fungizone (2.5 �g/ml, Sigma-Aldrich) and melanocyte growth
supplement (Invitrogen). Primary melanocytes were treated with 4-OHT (10 �M)
for 8h followed by addition of doxycycline (2 �g/ml) for 24h to inactivate ATF2
and induce expression of N-Ras.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Constructs</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>ATF2-specific shRNA
clones were obtained from Open Biosystems (catalog no. RHS4533). Five different
shRNA were obtained and tested for their efficiency of KD. Clone TRCN0000013714
was more efficient in inhibiting ATF2 in human cell lines while clone
TRCN0000013713 was more efficient for knocking down mouse ATF2. For subsequent
experiments we used the respective shATF2 clone depending on human or mouse
cell lines. We also tested 3 different clones for KD of ATF2 to rule out any
off target effect (Data not shown). siRNA control (cat # 4611) and three
SOX10-specific siRNA oligonucleotides were obtained from Ambion (cat #
4392420). Four FOXD3 specific siRNA were obtained from Dharmacon (Cat #
J-009152-06 -07, -08, -09). These siRNAs were pooled together in equimolar
ratio for transient transfection. An MITF specific shRNA, and MITF promoter
luciferase constructs (WT and mutant CRE-Luc constructs) were obtained from Dr.
David Fisher [58]. pGL3 vectors containing wild-type and BRN2-site-mutated MITF
promoters were obtained from Dr. Colin Goding [34]. pGL3 vectors containing wild-type
and SOX10-site-mutated MITF promoters were obtained from Dr. Michel Goossens
[59]. Retroviral vectors encoding a fusion protein consisting of full length
human BRAF and BRAFV600E linked to the T1 form of the human estrogen receptor
hormone-binding domain were generously provided by Dr. Martin McMahon [60].
SOX10 expression vector obtained from Dr. Alexey Terskikh, RSV-JunB, RSV-JunD
were obtained from Dr. Michael Karin and pBabe-Flag-TAM67 from Dr. Michael
Birrer.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Antibodies and
immunoblotting</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Antibodies against SOX10
and CREB (sc-1734 and sc-186 respectively) were from Santa Cruz
Biotechnologies; antibodies against ATF2, pERK and ERK (catalogue # 9226, 4337
and 4695 respectively) were obtained from Cell Signaling; antibodies against
MITF (C5) were purchased from Cell Lab vision. Protein extract (40�60 �g)
preparation and western blot analysis were done as described previously [8].
Specific bands were detected using fluorescent-labeled secondary antibodies
(Invitrogen, Carlsbad, CA) and analyzed using an Odyssey Infrared Scanner
(Li-COR Biosciences). &#946;-Actin antibody was used for monitoring loading.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Immunofluorescence</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Human melanoma and
melanocytes were grown in coverslips, fixed (4% paraformaldehyde and 2% sucrose
in 1�PBS), and then permeabilized and blocked (0.4% Triton X-100 and 2% BSA in
1�PBS) at room temperature. The cells were then washed (0.2% Triton X-100 and
0.2% BSA in 1�PBS) and incubated overnight at 4�C with monoclonal anti-rabbit
antibody against ATF2 (20F1, 1</span><span lang=EN style='font-size:12.0pt;
line-height:115%;font-family:"Cambria","serif"'>&#8758;100), followed by five
washes and then subsequent incubation at room temperature for 2 h with
anti-rabbit IgG (Invitrogen, 1&#8758;300) and Phalloidin (Molecular Probes,
1&#8758;1000). DNA was counterstained with 4,6-diamidino-2-phenylindole (DAPI;
Vector Laboratories) containing mounting medium.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Analysis of skin samples</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Skin samples were
collected from the backs of mice and immediately fixed with Z-fix, processed,
and embedded in paraffin. Paraffin sections were routinely stained by H&amp;E.
Dewaxed tissue sections (4.0�5.0 �m) were immunostained using rabbit polyclonal
antibodies to MITF (Sigma-Aldrich), S100 (S100B; DAKOCytomation; Carpinteria,
CA), and DCT (</span><span lang=EN style='font-size:12.0pt;line-height:115%;
font-family:"Cambria","serif"'>&#945;PEP8, kindly provided by Dr. Vincent
Hearing). Application of the primary antibody was followed by incubation with
goat anti-rabbit polymer-based EnVision-HRP-enzyme conjugate (DakoCytomation).
DAB (DakoCytomation) or SG-Vector (Vector Lab, Inc.; Burlingame, CA) chromogens
were applied, yielding brown (DAB) and black (SG) colors, respectively.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Quantitative analysis of
immunostaining</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Quantitative analysis
was performed as described previously [61]. Briefly, all slides were scanned at
an absolute magnification of 400� [resolution of 0.25 �m/pixel (100,000
pix/in.)] using the Aperio ScanScope CS system (Aperio Technologies; Vista,
CA). The acquired digital images representing whole tissue sections were
analyzed applying the Spectrum Analysis algorithm package and ImageScope
analysis software (version 9; Aperio Technologies, Inc.) to quantify IHC and
histochemical stainings. These algorithms make use of a color deconvolution
method [62] to separate stains. Algorithm parameters were set to achieve
concordance with manual scoring on a number of high-power fields, including
intensity thresholds for positivity and parameters that control cell segmentation
using the nuclear algorithm.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Microarray analysis</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Primary melanocytes were
treated with 4-OHT and Doxycycline before isolation of total RNA. 500 ng of
total RNA was used for synthesis of biotin-labeled cRNA using an RNA
amplification kit (Ambion). The biotinylated cRNA is labeled by incubation with
streptavidin-Cy3 to generate probe for hybridization with the Mouse-6
Expression BeadChip (Illumina MOUSE-6_V1_1_11234304_A) that represents 46.6K
mouse gene transcripts. We analyzed the BeadChips using the manufacturers
BeadArray Reader and collected primary data using the supplied Scanner
software. Data analysis was done as follows. First, expression intensities were
calculated for each gene probed on the array for all hybridizations using illumina's
BeadStudio 3.0 software. Second, intensity values were quality controlled and
normalized: quality control was carried out by using the BeadStudio detection
P-value set to &lt;0.01 as a cutoff. This removed genes which were never
detected in the arrays. All the arrays were then normalized using the cubic
spline routine from the BeadStudio 3.0 software. This procedure accounted for
any variation in hybridization intensity between the individual arrays.
Finally, these normalized data were analyzed for differentially expressed
genes. The groups of 2 biological and 2 technical replicates were described to
the BeadStudio 3.0 software and significantly differentially expressed genes
were determined on the basis of the difference changes in expression level (Illumina
DiffScor&gt;60 or DiffScore&lt;&#8722;60) and expression difference
p-value&lt;0.01. Microarray data are available under accession number GSE23860.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>ShRNA infection and RNA
interference</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Human embryonic kidney
293T cells were transfected with corresponding retro- or lentiviral shRNA
constructs (10 �g), Gag-pol (5 �g) and ENV expression vectors (10 �g) by
calcium phosphate transfection into 10 cm plates and supernatant was collected
after 48 hours to obtain viral particles. 2 million melanocytes and melanoma
cells in 10 cm plates were infected with 5 ml of viral supernatant along with 5
ml of medium in the presence of 8 �g/ml polybrene. The virus was replaced with
fresh media after 8 hours of infection. After two days, puromycin (1.5 �g/ml)
was used to select cells for 3 days. For human and mouse melanocytes the media
was changed to DMEM containing 10% FBS 24 h prior to harvesting cells. 50 nM
duplexes of scrambled and SOX10- or FOXD3- specific siRNA were transfected into
human melanocytes and WM1361 melanoma cells (2 million cells per transfection)
by Nucleofection using Amaxa reagents (NHEM-Neo Nucleofector and Solution R
respectively) for SOX10 or FOXD3 knock down. Over 90% of the cells transduced
were able to resist drug selection, indicating efficient infection of the
respective genes. GFP was also used to monitor efficiency of infection,
confirming &gt;90% GFP expression by fluorescence microscopy.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Real-time quantitative
reverse transcription�PCR (RT�PCR)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Quantitative PCR was
performed as described earlier [8]. Total RNA was isolated using an RNeasy mini
kit (Sigma, St. Louis, MO) and reverse transcribed using a high cDNA capacity
reverse transcription kit (Applied Biosystems, Foster City, CA) following the
manufacturer's instructions. Specific primers (Valuegene, San Diego, CA) used
for PCR were as follows: Human ATF2, forward: tgtggccagcgttttaccaa, reverse:
tgatgtgggctgtgcagttt., human MITF, forward: aaaccccaccaagtaccaca, reverse:
acatggcaagctcaggac., human SOX10, forward: caa gtaccagcccaggcggc, reverse:
gggtgccggtggtccaagtg., human FOXD3, forward: gcgacgggctggaagag, reverse:
gctgtccgtgatggggtgcc., human PAX3, forward: ggaactggagcgtgcttttg, reverse:
ggcggttgctaaaccagac., human BRN2, forward: gaaagagcgagcgaggaga, reverse:
caggctgtagtggttagacg., mouse MITF, forward: agatttgagatgctcatcccc, reverse:
gatgcgtgatgtcatactgga, mouse TYRP1, forward: ccctagcctatatctccctttt, reverse:
taccatcgtggggataatggc., mouse DCT, forward: gtcctccactcttttacagacg, reverse:
attcggttgtgaccaatgggt, mouse Silver, forward: tgacggtggaccctgcccat, reverse:
agctttgcgtggcccgtagc. The reaction mixture was denatured at 95�C for 10 min,
followed by 40 cycles of 95�C for 15s, annealing at 60�C for 30s and extension
at 72�C for 30s. Reactions were performed using the SYBR Green qPCR reagent
(Invitrogen) and run on an MX3000P qPCR machine (Stratagene, La Jolla, CA). The
specificity of the products was verified by melting curve analysis and agarose
gels. The amount of the target transcript was related to that of a reference
gene (Cyclophilin A for both human and mouse) by the Ct method. Each sample was
assayed at least in triplicate and was reproduced at least three times.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Chromatin
immunoprecipitation</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Chromatin
immunoprecipitation was performed using the Magna-Chip (Upstate) according to
the manufacturer's instructions. Control shRNA and ATF2 knocked down WM1361
cells (one 10 cm plate for each, 80% confluent) were fixed in 37% formaldehyde
and sheared chromatin was immunoprecipitated and subjected to PCR for 32
cycles. The following primers corresponding to the MITF promoter, spanning the
SOX10 binding site were used, forward: gcagtcggaagtggcag, reverse:
caactcactgtcagatcaa. Antibodies against Sox10 and CREB (sc-1734 and sc-186
respectively) were from Santa Cruz Biotechnologies. IgG control, and
glyceraldehyde-3-phosphate dehydrogenase oligonucleotides were provided by the
kit. Antibody against ATF2 (sc-6233), JunB (sc-8051), JunD (sc-74) were
obtained from Santa Cruz. Antibodies against ATFa were provided generated by
Nic Jones. For Sox10 promoter, the following primers spanning AP-1 binding site
were used; forward: cccagtgctggcctaatagc, reverse: cacccttgatatccccaagtga.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Luciferase assays</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>MeWo, WM35, WM1361,
Lu1205 cells in six-well plates were transiently transfected with 0.5 �g of
reporter plasmid containing WT or CRE mutant, BRN2 mutant or SOX10 mutant MITF
promoter and 0.1 �g of pSV-&#946;-Galactosidase (Promega, San Luis Obispo, CA)
using Lipofectamine 2000 reagent (Invitrogen). Human melanocytes (2 million)
were transfected with 2 �g of reporter plasmid containing WT or SOX10 mutant
MITF promoter and 0.3 �g of pSV-&#946;-Galactosidase using Amaxa reagent
(NHEM-Neo nucleofector kit, Lonza) according to the manufacturer's protocol.
Cell lysates were prepared from cells after 48 h. Luciferase activity was
measured using the Luciferase assay system (Promega) in a luminometer and
normalized to &#946;-galactosidase activity. The data were normalized to
&#946;-galactosidase and represent the mean and SD of assays performed in
triplicate. All experiments were performed a minimum of 3 times.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Colony formation assay</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Melan-Ink4a-Arf1 cells
were transduced with a retroviral vector expressing BRAFV600E:ERT1 and selected
with puromycin for 3 days. These cells were treated with 200 nM of estrogen
receptor antagonist ICI 182780 (ICI, Tocris Bioscience) to induce expression of
BRAFV600E. After one day, these cells were transduced with a lentiviral vector
expressing either shATF2 or shMITF separately, or in combination. Colony
formation was carried out as described by Franken et al. [63]. Briefly, 5,000
cells were plated into each well of a 6-well plate, and cells were grown in
mouse melanocyte media containing ICI and puromycin (1.5 �g/ml) for 3 weeks
until colonies became visible. The colonies were stained with
P-Iodonitrotetrazolium Violet (1 mg/ml Sigma, St. Louis, MO). This experiment
was performed in triplicate and reproduced 2 times.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Mouse genotyping</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Genomic DNA was isolated
from tail tissue was subjected to PCR resulting in amplification of a 549 bp
DNA fragment for Atf2 floxed and a 485 bp DNA fragment for wild type mice. PCR
conditions included one cycle at 95�C for 3 min; and 30 cycles of 94�C/30 sec,
55�C/30 sec and 72�C/1 min and one cycle at 72�C for 5 min. Primers used for
PCR reactions were forward: caatccactgccatggcctt, reverse: tcagataaagccaagtcgaatctgg.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Avidin-biotin
DNA�protein binding assay</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>MeWo cells were left
untreated or treated with 20 mJ/cm2 of UV-B for 1 h. The cells were lysed using
lysis buffer containing 1% Triton-100 and incubated with 4 �g of biotin-labeled
MITF promoter spanning the CRE site oligo
(5&#8242;-gaaaaaaaagcatgacgtcaagccaggggg-3&#8242;) in the presence of
poly-(dI-dC) (20 �g/ml) for 2h at 4�c. The oligo-bound proteins were captured
using streptavidin-agarose (Invitrogen) for 1 h incubation, followed by
extensive washes with washing buffer (20 mm HEPES, 150 mm NaCl, 20% glycerol,
0.5 mm EDTA, and 1% Triton-100) and analyzed using SDS-PAGE and western blots.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>BrdU, PI labeling,
Annexin V staining</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>To evaluate the cell
cycle index of Melan-Ink4a-Arf1 cells stably overexpressing BRAFV600E:ERT1
alone or in combination with shRNA to the genes indicated in Results, cells
were plated in media containing ICI and puromycin (1.5 �g/ml) at 2�106 cells
per 10 cm plate O/N. Cells were labeled with 10 �M of 5-bromo-2-deoxyuridine (BrdU;
Sigma Chemical Co.), for an hour. Cells were then washed, fixed, and stained
with anti-BrdU mAbs and propidium Iodide (BD Biosciences, San Jose, CA)
according to the manufacturer's protocol, and analyzed on a BD FACSCanto
machine. Cell cycle phase was analyzed using the Mod Fit LT v.2 program (Verity
Software, Topsham, ME). In a separate experiment the cells were stained with
Annexin V-APC and 7-AAD (BD Pharmingen, San Diego, CA) according to
manufacturer's protocol, to enable analysis of early apoptosis and cell death.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Hypoxia treatment</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Cells were treated under
hypoxia (1% O2) for indicated time points using a hypoxia chamber (In Vivo 400;
Ruskin Technologies Ltd, Bridgend, UK).</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>UV irradiation</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Mice were treated with
4-Hydroxytamoxifen (25 mg/ml in DMSO) by swabbing the entire body (excluding
the head) on days 1�3 after birth. On day 4 the pups were placed under UVB
light source (FL-15E; 320 nm) and exposed to 20 �W/cm2 for 22 seconds. Ninety
minutes after UVB treatment mice were sacrificed and entire skin was removed
and processed.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>TMA and AQUA staining</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Tissue microarrays were
constructed as previously described [21]. The arrays included a series of 192
sequentially collected primary melanomas and 299 metastatic melanomas. Slides
were stained for automated, quantitative analysis (AQUA) for ATF2 and MITF as
previously published [49], [64]. The AQUA scores for the two markers were
obtained from the AQUAmine database (</span><a
href="../Documents/www.tissuearray.org"><span lang=EN style='font-size:12.0pt;
line-height:115%;font-family:"Cambria","serif";color:blue'>www.tissuearray.org</span></a><span
lang=EN style='font-size:12.0pt;line-height:115%;font-family:"Cambria","serif"'>).</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Go to:</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Supporting Information</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure S1</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Representative staining
of non-melanoma tumor developed in the
TyrCre+::Atf2+/+::NrasQ61K::Ink4a&#8722;/&#8722; model.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>(7.25 MB TIF)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Click here for
additional data file.(6.9M, tif)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure S2</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>A. ATF2 negatively
regulates MITF in several melanoma lines. ATF2 was knocked down in WM793 cells
(left panel), in WM35 cells (middle panel) and in WM1361 cells (right panel)
(one 10 cm plate each, 50% confluent). Cells were lysed and Western blotting
was carried out (50 �g/lane) with the indicated antibodies. Lower panel, RNA
was extracted from the corresponding cell lines, and qPCR was carried out using
MITF primers. Cyclophilin A was served as an internal control. B. ATF2
positively regulates MITF in 501-MEL melanoma cells. Left panel, ATF2 was
knocked down in 501- Mel cells (one 10 cm plate each, 50% confluent). Cells
were lysed and Western blotting was carried out (40 �g/lane) with the indicated
antibodies. Right panel, RNA was extracted from the above samples and qPCR was
carried out using MITF primers; Cyclophilin A served as an internal control.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>(1.36 MB TIF)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Click here for
additional data file.(1.2M, tif)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure S3</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>A. ATF2 effect on Brn2
and Pax3 in human melanocytes and melanoma cells. Cells were infected with
shRNA control or shATF2 and RNA prepared was used for qPCR analysis of Pax3 and
Brn2 expression, relative t o cyclophilin used as control. B. Increased SOX10
mRNA expression in melanoma cells. ATF2 expression was inhibited in WM1361
cells and total mRNA was extracted. A qPCR was performed to quantify changes in
the expression of the indicated genes. Cyclophilin A served as an internal
control.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>(0.93 MB TIF)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Click here for
additional data file.(911K, tif)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure S4</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Effect of ATF2 and FoxD3
on Sox10 and MITF expression in human melanocytes. H3a human melanocytes cells
were infected with Scrambled Control siRNA (siSC), shATF2, siFOXD3 or their
combination. Protein or RNA were prepared 72h later and assessed in western and
qPCR.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>(1.16 MB TIF)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Click here for
additional data file.(1.1M, tif)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure S5</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Analysis of ATF2 effect
on SOX10 and MITF transcription in 12 melanoma cell lines. Melanoma cell lines
indicated were infected with shATF2 and subjected to selection to enrich for
ATF2 KD cells. RNA was prepared 4 days later and QPCR analysis was performed
for ATF2 (upper panel), MITF (middle panel) and SOX10 (lower panel)
transcripts. Data shown represent analysis of triplicate samples.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>(1.70 MB TIF)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Click here for
additional data file.(1.6M, tif)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure S6</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Altered MITF expression
by ATF2 +/&#8722; SOX10 in human melanocytes and melanomas. Indicated cell
lines were infected with shControl or shATF2, in the presence or absence of
SOX10 overexpression, and proteins were prepared to determine changes in expression
of SOX10. &#946;-actin was used as loading control. Analysis of MITF and SOX10
transcript levels was carried out in the indicated cell lines using qPCR on RNA
prepared from the cells that were infected with indicated vectors (C, C+SOX10,
shATF2, shATF2+SOX10).</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>(2.59 MB TIF)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Click here for
additional data file.(2.4M, tif)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure S7</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Effect of ATF2 on SOX10
and MITF in human melanoma cell lines exhibiting positive regulation by ATF2.
A. Melanoma cell lines MeWo and 501Mel were infected with shControl or shATF2
and level of SOX10 protein was assessed. B, C. The effect of SOX10 expression
on MITF (B) and on SOX10 (C) transcripts in MeWO (left graph) cells expressing
control or shATF2 was determined using qPCR. D. Effect of DN Jun and JunB on
SOX10 protein and MITF transcript levels. Indicated melanoma cells were
transfected with TAM67 (DN Jun) and JUNB either alone or in combination as
indicated. The cells were lysed and proteins and RNA were prepared. Western
blotting was carried out with the indicated antibodies. QPCR analysis for MITF
transcripts was performed (lower panel). E. Chromatin IP reveals loss of JunB
binding to SOX10 promoter in melanoma cells in which MITF is positively
regulated by ATF2. Melanoma cells were subjected to ChIP using the indicated
antibodies followed by PCR of SOX10 promoter sequences harboring the AP1
response element. F. Endogenous level of JUNB in various melanoma cells. Cell
lysates from indicated melanoma cells were subjected to Western blot analysis
using JUNB antibody. &#946;�actin was used as a loading control.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>(1.06 MB TIF)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Click here for
additional data file.(1.0M, tif)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure S8</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>A. Transcriptional
activity of MITF is dependent on ATF2 and a CRE element in specific melanoma
cells that express high MITF levels. WT and mutant (CRE) forms of MITF-Luc
constructs were used to assess the contribution of ATF2 to MITF transcription
in MeWo cells. Luciferase assays were performed as detailed in Materials and
Methods. B. ATF2 and CREB bind to a MITF CRE site. Human melanoma (MeWo) cells
were treated with 20mJ/cm2 of UVB, and proteins prepared after 1h were
incubated with dI/dT (20 �g/ml for 30 min at 4C) and then with a biotinylated
annealed oligo (4 �g, overnight at 4�C) containing the CRE site and flanking
sequences from the MITF promoter. Streptavidin beads were added to lysates and
bound material was analyzed on immunoblots with the indicated antibodies. C.
ChIP analysis shows ATF2 binding to the MITF CRE site in specific melanoma MeWo
cells. Chromatin was prepared and precipitated using an ATF2 antibody and bound
DNA was amplified using MITF specific primers that flank the CRE binding site
and analyzed on 1.5% agarose gel. Rabbit IgG served as a negative control and
CREB and ATF1 served as positive controls.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>(1.34 MB TIF)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Click here for
additional data file.(1.2M, tif)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Figure S9</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Subcellular localization
of ATF2 in melanoma cell lines. Immunostaining was carried out in indicated
melanoma and human melanocyte cells with ATF2 (green) antibody and Phalloidin
(red). The cells were counter stained with DAPI for nuclear staining.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>(6.98 MB TIF)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Click here for
additional data file.(6.6M, tif)</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Go to:</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Acknowledgments</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>We thank Drs. Vincent
Hearing for the &#945;PEP8 antibody, David Fisher and Colin Goding for MITF and
BRN2 constructs, Martin McMahon for the mutant B-Raf expression vector, Michel
Goossens for MITF wild-type and SOX10-mutated promoters, Michael Karin for the
JunB, JunD expression vectors, Michael Birrer for the TAM67 construct, Alexey
Terskikh for the Sox10 expression vector, Meenhard Herlyn for the melanoma cell
lines, Stephanie Papp for help with genotyping, and Kang Liu and Craig Hauser
for profiling array analysis.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Go to:</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Footnotes</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>The authors have
declared that no competing interests exist.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Support from NCI grant
CA099961 (ZAR) and Wellcome Trust grant 078327 (DCB) are gratefully
acknowledged. The funders had no role in study design, data collection and
analysis, decision to publish, or preparation of the manuscript.</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>&nbsp;</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>Go to:</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>References</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>1. Gray-Schopfer V,
Wellbrock C, Marais R. Melanoma biology and new targeted therapy. Nature.
2007;445:851�857. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>2. Fears TR, Tucker MA.
Re: Sun exposure and mortality from melanoma. J Natl Cancer Inst.
2005;97:1789�1790; author reply 1791. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>3. Chin L, Garraway LA,
Fisher DE. Malignant melanoma: genetics and therapeutics in the genomic era.
Genes Dev. 2006;20:2149�2182. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>4. Lopez-Bergami P,
Fitchman B, Ronai Z. Understanding signaling cascades in melanoma. Photochem
Photobiol. 2008;84:289�306. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>5. Van Raamsdonk CD,
Bezrookove V, Green G, Bauer J, Gaugler L, et al. Frequent somatic mutations of
GNAQ in uveal melanoma and blue naevi. Nature. 2009;457:599�602. [PMC free
article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>6. Levy C, Khaled M,
Fisher DE. MITF: master regulator of melanocyte development and melanoma
oncogene. Trends Mol Med. 2006;12:406�414. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>7. Berger AJ, Davis DW,
Tellez C, Prieto VG, Gershenwald JE, et al. Automated quantitative analysis of
activator protein-2alpha subcellular expression in melanoma tissue microarrays
correlates with survival prediction. Cancer Res. 2005;65:11185�11192. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>8. Lopez-Bergami P,
Huang C, Goydos JS, Yip D, Bar-Eli M, et al. Rewired ERK-JNK signaling pathways
in melanoma. Cancer Cell. 2007;11:447�460. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>9. Lopez-Bergami P, Lau
E, Ronai Z. Emerging roles of ATF2 and the dynamic AP1 network in cancer. Nat
Rev Cancer. 2010;10:65�76. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>10. Melnikova VO,
Bar-Eli M. Transcriptional control of the melanoma malignant phenotype. Cancer
Biol Ther. 2008;7:997�1003. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>11. Mitra D, Fisher DE.
Transcriptional regulation in melanoma. Hematol Oncol Clin North Am.
2009;23:447�465, viii. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>12. Lopez-Bergami P, Kim
H, Dewing A, Goydos J, Aaronson S, et al. c-Jun regulates phosphoinositide-dependent
kinase 1 transcription: implication for Akt and protein kinase C activities and
melanoma tumorigenesis. J Biol Chem. 2010;285:903�913. [PMC free article]
[PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>13. Raingeaud J,
Whitmarsh AJ, Barrett T, Derijard B, Davis RJ. MKK3- and MKK6-regulated gene
expression is mediated by the p38 mitogen-activated protein kinase signal
transduction pathway. Mol Cell Biol. 1996;16:1247�1255. [PMC free article]
[PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>14. Gupta S, Campbell D,
Derijard B, Davis RJ. Transcription factor ATF2 regulation by the JNK signal
transduction pathway. Science. 1995;267:389�393. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>15. van Dam H, Wilhelm
D, Herr I, Steffen A, Herrlich P, et al. ATF-2 is preferentially activated by
stress-activated protein kinases to mediate c-jun induction in response to
genotoxic agents. EMBO J. 1995;14:1798�1811. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>16. Nakamura T, Okuyama
S, Okamoto S, Nakajima T, Sekiya S, et al. Down-regulation of the cyclin A
promoter in differentiating human embryonal carcinoma cells is mediated by
depletion of ATF-1 and ATF-2 in the complex at the ATF/CRE site. Exp Cell Res.
1995;216:422�430. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>17. Benbrook DM, Jones
NC. Heterodimer formation between CREB and JUN proteins. Oncogene.
1990;5:295�302. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>18. van Dam H,
Castellazzi M. Distinct roles of Jun : Fos and Jun : ATF dimers in oncogenesis.
Oncogene. 2001;20:2453�2464. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>19. Bhoumik A, Takahashi
S, Breitweiser W, Shiloh Y, Jones N, et al. ATM-dependent phosphorylation of
ATF2 is required for the DNA damage response. Mol Cell. 2005;18:577�587. [PMC
free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>20. Li S, Ezhevsky S,
Dewing A, Cato MH, Scortegagna M, et al. Radiation Sensitivity and Tumor
Susceptibility in ATM phospho-mutant ATF2 mice. Genes &amp; Cancer. 2010 in
press. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>21. Berger AJ, Kluger
HM, Li N, Kielhorn E, Halaban R, et al. Subcellular localization of activating
transcription factor 2 in melanoma specimens predicts patient survival. Cancer
Res. 2003;63:8103�8107. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>22. Bhoumik A, Huang TG,
Ivanov V, Gangi L, Qiao RF, et al. An ATF2-derived peptide sensitizes melanomas
to apoptosis and inhibits their growth and metastasis. J Clin Invest.
2002;110:643�650. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>23. Bhoumik A, Gangi L,
Ronai Z. Inhibition of melanoma growth and metastasis by ATF2-derived peptides.
Cancer Res. 2004;64:8222�8230. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>24. Bhoumik A, Ivanov V,
Ronai Z. Activating transcription factor 2-derived peptides alter resistance of
human tumor cell lines to ultraviolet irradiation and chemical treatment. Clin
Cancer Res. 2001;7:331�342. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>25. Bhoumik A, Jones N,
Ronai Z. Transcriptional switch by activating transcription factor 2-derived
peptide sensitizes melanoma cells to apoptosis and inhibits their
tumorigenicity. Proc Natl Acad Sci U S A. 2004;101:4222�4227. [PMC free
article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>26. Ronai Z, Yang YM,
Fuchs SY, Adler V, Sardana M, et al. ATF2 confers radiation resistance to human
melanoma cells. Oncogene. 1998;16:523�531. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>27. Cheli Y, Ohanna M,
Ballotti R, Bertolotto C. Fifteen-year quest for microphthalmia-associated
transcription factor target genes. Pigment Cell Melanoma Res. 2010;23:27�40.
[PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>28. Hoek KS, Schlegel
NC, Eichhoff OM, Widmer DS, Praetorius C, et al. Novel MITF targets identified
using a two-step DNA microarray strategy. Pigment Cell Melanoma Res.
2008;21:665�676. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>29. Mascarenhas JB,
Littlejohn EL, Wolsky RJ, Young KP, Nelson M, et al. PAX3 and SOX10 activate
MET receptor expression in melanoma. Pigment Cell Melanoma Res 2010 [PMC free
article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>30. Yang G, Li Y, Nishimura
EK, Xin H, Zhou A, et al. Inhibition of PAX3 by TGF-beta modulates melanocyte
viability. Mol Cell. 2008;32:554�563. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>31. Jacquemin P, Lannoy
VJ, O'Sullivan J, Read A, Lemaigre FP, et al. The transcription factor onecut-2
controls the microphthalmia-associated transcription factor gene. Biochem
Biophys Res Commun. 2001;285:1200�1205. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>32. Yasumoto K, Takeda
K, Saito H, Watanabe K, Takahashi K, et al. Microphthalmia-associated
transcription factor interacts with LEF-1, a mediator of Wnt signaling. EMBO J.
2002;21:2703�2714. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>33. Saito H, Yasumoto K,
Takeda K, Takahashi K, Fukuzaki A, et al. Melanocyte-specific
microphthalmia-associated transcription factor isoform activates its own gene
promoter through physical interaction with lymphoid-enhancing factor 1. J Biol
Chem. 2002;277:28787�28794. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>34. Goodall J, Carreira
S, Denat L, Kobi D, Davidson I, et al. Brn-2 represses
microphthalmia-associated transcription factor expression and marks a distinct
subpopulation of microphthalmia-associated transcription factor-negative
melanoma cells. Cancer Res. 2008;68:7788�7794. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>35. Thomas AJ, Erickson
CA. FOXD3 regulates the lineage switch between neural crest-derived glial cells
and pigment cells by repressing MITF through a non-canonical mechanism.
Development. 2009;136:1849�1858. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>36. Breitwieser W, Lyons
S, Flenniken AM, Ashton G, Bruder G, et al. Feedback regulation of p38 activity
via ATF2 is essential for survival of embryonic liver cells. Genes Dev.
2007;21:2069�2082. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>37. Bosenberg M,
Muthusamy V, Curley DP, Wang Z, Hobbs C, et al. Characterization of
melanocyte-specific inducible Cre recombinase transgenic mice. Genesis.
2006;44:262�267. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>38. Serrano M, Lee H,
Chin L, Cordon-Cardo C, Beach D, et al. Role of the INK4a locus in tumor
suppression and cell mortality. Cell. 1996;85:27�37. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>39. Ackermann J,
Frutschi M, Kaloulis K, McKee T, Trumpp A, et al. Metastasizing melanoma
formation caused by expression of activated N-RasQ61K on an INK4a-deficient
background. Cancer Res. 2005;65:4005�4011. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>40. Sviderskaya EV, Hill
SP, Evans-Whipp TJ, Chin L, Orlow SJ, et al. p16(Ink4a) in melanocyte
senescence and differentiation. J Natl Cancer Inst. 2002;94:446�454. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>41. Kido K, Sumimoto H,
Asada S, Okada SM, Yaguchi T, et al. Simultaneous suppression of MITF and BRAF
V600E enhanced inhibition of melanoma cell proliferation. Cancer Sci.
2009;100:1863�1869. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>42. Vachtenheim J,
Ondrusova L, Borovansky J. SWI/SNF chromatin remodeling complex is critical for
the expression of microphthalmia-associated transcription factor in melanoma
cells. Biochem Biophys Res Commun. 2010;392:454�459. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>43. Goding C, Meyskens
FL., Jr Microphthalmic-associated transcription factor integrates melanocyte
biology and melanoma progression. Clin Cancer Res. 2006;12:1069�1073. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>44. Deal KK, Cantrell
VA, Chandler RL, Saunders TL, Mortlock DP, et al. Distant regulatory elements
in a Sox10-beta GEO BAC transgene are required for expression of Sox10 in the
enteric nervous system and other neural crest-derived tissues. Dev Dyn.
2006;235:1413�1432. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>45. Poser I, Bosserhoff
AK. Transcription factors involved in development and progression of malignant
melanoma. Histol Histopathol. 2004;19:173�188. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>46. Saha B, Singh SK,
Sarkar C, Bera R, Ratha J, et al. Activation of the Mitf promoter by
lipid-stimulated activation of p38-stress signalling to CREB. Pigment Cell Res.
2006;19:595�605. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>47. Wellbrock C, Rana S,
Paterson H, Pickersgill H, Brummelkamp T, et al. Oncogenic BRAF regulates
melanoma proliferation through the lineage specific factor MITF. PLoS ONE.
2008;3:e2734. doi: 10.1371/journal.pone.0002734. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>48. Vance KW, Goding CR.
The transcription network regulating melanocyte development and melanoma.
Pigment Cell Res. 2004;17:318�325. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>49. Garraway LA, Widlund
HR, Rubin MA, Getz G, Berger AJ, et al. Integrative genomic analyses identify
MITF as a lineage survival oncogene amplified in malignant melanoma. Nature.
2005;436:117�122. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>50. Pollock PM, Harper
UL, Hansen KS, Yudt LM, Stark M, et al. High frequency of BRAF mutations in
nevi. Nat Genet. 2003;33:19�20. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>51. Lin JY, Fisher DE.
Melanocyte biology and skin pigmentation. Nature. 2007;445:843�850. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>52. Wellbrock C, Marais
R. Elevated expression of MITF counteracts B-RAF-stimulated melanocyte and
melanoma cell proliferation. J Cell Biol. 2005;170:703�708. [PMC free article]
[PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>53. Noonan FP, Recio JA,
Takayama H, Duray P, Anver MR, et al. Neonatal sunburn and melanoma in mice.
Nature. 2001;413:271�272. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>54. Fernandez LP, Milne
RL, Pita G, Floristan U, Sendagorta E, et al. Pigmentation-related genes and
their implication in malignant melanoma susceptibility. Exp Dermatol.
2009;18:634�642. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>55. Bhoumik A, Fichtman
B, Derossi C, Breitwieser W, Kluger HM, et al. Suppressor role of activating
transcription factor 2 (ATF2) in skin cancer. Proc Natl Acad Sci U S A.
2008;105:1674�1679. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>56. Maekawa T, Shinagawa
T, Sano Y, Sakuma T, Nomura S, et al. Reduced levels of ATF-2 predispose mice
to mammary tumors. Mol Cell Biol. 2007;27:1730�1744. [PMC free article]
[PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>57. Gray-Schopfer VC,
Cheong SC, Chong H, Chow J, Moss T, et al. Cellular senescence in naevi and
immortalisation in melanoma: a role for p16? Br J Cancer. 2006;95:496�505. [PMC
free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>58. Huber WE, Price ER,
Widlund HR, Du J, Davis IJ, et al. A tissue-restricted cAMP transcriptional
response: SOX10 modulates alpha-melanocyte-stimulating hormone-triggered
expression of microphthalmia-associated transcription factor in melanocytes. J
Biol Chem. 2003;278:45224�45230. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>59. Bondurand N, Pingault
V, Goerich DE, Lemort N, Sock E, et al. Interaction among SOX10, PAX3 and MITF,
three genes altered in Waardenburg syndrome. Hum Mol Genet. 2000;9:1907�1917.
[PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>60. Cartlidge RA, Thomas
GR, Cagnol S, Jong KA, Molton SA, et al. Oncogenic BRAF(V600E) inhibits BIM
expression to promote melanoma cell survival. Pigment Cell Melanoma Res.
2008;21:534�544. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>61. Krajewska M, Smith
LH, Rong J, Huang X, Hyer ML, et al. Image analysis algorithms for
immunohistochemical assessment of cell death events and fibrosis in tissue
sections. J Histochem Cytochem. 2009;57:649�663. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>62. Ruifrok AC, Johnston
DA. Quantification of histochemical staining by color deconvolution. Anal Quant
Cytol Histol. 2001;23:291�299. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>63. Franken NA,
Rodermond HM, Stap J, Haveman J, van Bree C. Clonogenic assay of cells in
vitro. Nat Protoc. 2006;1:2315�2319. [PubMed]</span></p>

<p class=MsoNormal style='text-autospace:none'><span lang=EN style='font-size:
12.0pt;line-height:115%;font-family:"Cambria","serif"'>64. Gould Rothberg BE,
Berger AJ, Molinaro AM, Subtil A, Krauthammer MO, et al. Melanoma prognostic
model using tissue microarrays and genetic algorithms. J Clin Oncol.
2009;27:5772�5780. [PMC free article] [PubMed]</span></p>

<p class=MsoNormal>&nbsp;</p>

</div>

</body>

</html>
