<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=Generator content="Microsoft Word 14 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:Tahoma;
	panose-1:2 11 6 4 3 5 4 4 2 4;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";}
h1
	{mso-style-link:"Heading 1 Char";
	margin-right:0in;
	margin-left:0in;
	font-size:24.0pt;
	font-family:"Times New Roman","serif";
	font-weight:bold;}
h2
	{mso-style-link:"Heading 2 Char";
	margin-right:0in;
	margin-left:0in;
	font-size:18.0pt;
	font-family:"Times New Roman","serif";
	font-weight:bold;}
h3
	{mso-style-link:"Heading 3 Char";
	margin-right:0in;
	margin-left:0in;
	font-size:13.5pt;
	font-family:"Times New Roman","serif";
	font-weight:bold;}
a:link, span.MsoHyperlink
	{color:blue;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:purple;
	text-decoration:underline;}
p
	{margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-link:"Balloon Text Char";
	margin:0in;
	margin-bottom:.0001pt;
	font-size:8.0pt;
	font-family:"Tahoma","sans-serif";}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Times New Roman","serif";
	font-weight:bold;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-link:"Heading 2";
	font-family:"Times New Roman","serif";
	font-weight:bold;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-link:"Heading 3";
	font-family:"Times New Roman","serif";
	font-weight:bold;}
span.cit
	{mso-style-name:cit;}
span.fm-vol-iss-date
	{mso-style-name:fm-vol-iss-date;}
span.doi1
	{mso-style-name:doi1;}
span.fm-citation-ids-label
	{mso-style-name:fm-citation-ids-label;}
span.fm-role
	{mso-style-name:fm-role;}
p.p, li.p, div.p
	{mso-style-name:p;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";}
span.tag-json
	{mso-style-name:tag-json;}
span.mixed-citation
	{mso-style-name:mixed-citation;}
span.ref-journal
	{mso-style-name:ref-journal;}
span.ref-vol
	{mso-style-name:ref-vol;}
span.nowrap
	{mso-style-name:nowrap;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-link:"Balloon Text";
	font-family:"Tahoma","sans-serif";}
span.acknowledgment-journal-title
	{mso-style-name:acknowledgment-journal-title;}
.MsoChpDefault
	{font-family:"Calibri","sans-serif";}
.MsoPapDefault
	{margin-bottom:10.0pt;
	line-height:115%;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>

</head>

<body lang=EN-US link=blue vlink=purple>

<div class=WordSection1>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span style='position:absolute;z-index:251658240;margin-left:0px;
margin-top:0px;width:500px;height:74px'><img width=500 height=74
src="PMC3031508%20table%201_files/image001.png"></span><span style='font-size:
12.0pt;font-family:"Times New Roman","serif"'><img width=500 height=75
id="Picture 8" src="PMC3031508%20table%201_files/image002.gif"
alt="Logo of plosone"></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PLoS
One. 2011; 6(1): e14635. </span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Published
online 2011 Jan 31. doi:&nbsp; </span><a
href="http://dx.doi.org/10.1371%2Fjournal.pone.0014635" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>10.1371/journal.pone.0014635</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMCID:
PMC3031508</span></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:24.0pt;font-family:"Times New Roman","serif"'>Gene Expression
Meta-Analysis Identifies VDAC1 as a Predictor of Poor Outcome in Early Stage
Non-Small Cell Lung Cancer</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Grills%20C%5Bauth%5D"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Claire
Grills</span></a><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>,<sup>#1,*</sup>
</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Jithesh%20PV%5Bauth%5D"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Puthen
V. Jithesh</span></a><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>,<sup>#1</sup>
</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Blayney%20J%5Bauth%5D"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Jaine
Blayney</span></a><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>,<sup>1</sup>
</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Zhang%20SD%5Bauth%5D"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Shu-Dong
Zhang</span></a><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>,<sup>1</sup>
and </span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Fennell%20DA%5Bauth%5D"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Dean A.
Fennell</span></a><sup><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>1,2</span></sup></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Rory
Edward Morty, Editor</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><sup><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
display:none'>1</span></sup><span lang=EN style='font-size:12.0pt;font-family:
"Times New Roman","serif";display:none'>Centre for Biomedical Informatics,
Queen's University Belfast, Belfast, Northern Ireland, United Kingdom</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><sup><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
display:none'>2</span></sup><span lang=EN style='font-size:12.0pt;font-family:
"Times New Roman","serif";display:none'>Centre for Cancer Research and Cell
Biology, Queen's University Belfast, Belfast, Northern Ireland, United Kingdom</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
display:none'>University of Giessen Lung Center, Germany</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><sup><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
display:none'>#</span></sup><span lang=EN style='font-size:12.0pt;font-family:
"Times New Roman","serif";display:none'>Contributed equally.</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
display:none'>* E-mail: </span><a href="mailto:dev@null"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>ku.ca.buq@sllirg.c</span></a></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif";display:none'>Conceived and
designed the experiments: DAF. Analyzed the data: CG PJ JKB SDZ. Contributed
reagents/materials/analysis tools: CG PJ JKB. Wrote the paper: CG PJ DAF.</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Author
information &#9658;</span></a><span lang=EN style='font-size:12.0pt;font-family:
"Times New Roman","serif"'> </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Article notes
&#9658;</span></a><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>
</span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Copyright
and License information &#9658;</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
display:none'>Received 2010 Jul 5; Accepted 2010 Nov 23.</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a href="http://www.ncbi.nlm.nih.gov/pmc/about/copyright.html"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
display:none'>Copyright</span></a><span lang=EN style='font-size:12.0pt;
font-family:"Times New Roman","serif";display:none'> Grills et al. This is an
open-access article distributed under the terms of the Creative Commons
Attribution License, which permits unrestricted use, distribution, and
reproduction in any medium, provided the original author and source are credited.</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>This
article has been </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/citedby/"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>cited by</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> other
articles in PMC.</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/"
title="Go to other sections in this page"><span lang=EN style='font-size:12.0pt;
font-family:"Times New Roman","serif"'>Go to:</span></a></p>

<div style='border:none;border-bottom:solid #97B0C8 1.0pt;padding:0in 0in 0in 0in'>

<p class=MsoNormal style='line-height:normal;border:none;padding:0in'><b><span
lang=EN style='font-size:18.0pt;font-family:"Times New Roman","serif"'>Abstract</span></b></p>

</div>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>Background</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>The bioenergetic status of
non-small cell lung cancer correlates with tumour aggressiveness. <span
style='background:lime'>The voltage dependent anion channel type 1 (VDAC1) is a
component of the mitochondrial permeability transition pore, regulates
mitochondrial ATP/ADP exchange suggesting that its over-expression could be
associated with energy dependent processes including increased proliferation
and invasiveness.</span> <span style='background:yellow'>To test this
hypothesis, we conducted an in vivo gene-expression meta-analysis of surgically
resected non-small cell lung cancer (NSCLC) using 602 individual expression
profiles, to examine the impact of VDAC1 on survival.</span></span></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>Methodology/Principal
Findings</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif";background:yellow'>High VDAC1
expression was associated with shorter overall survival with hazard ratio
(HR)&#8202;=&#8202;0.6639 (95% confidence interval (CI) 0.4528 to 0.9721),
p&#8202;=&#8202;0.035352 corresponding to 52 versus 101 months.</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> <span
style='background:yellow'>VDAC1 predicted shorter time to recurrence and was
shown to be an independent prognostic factor compared with histology, gender,
age, nodal stage and tumour stage in a Cox multivariate analysis.</span>
Supervised analysis of all the datasets identified a 6-gene signature
comprising HNRNPC, HSPA4, HSPA9, UBE2D2, CSNK1A1 and G3BP1 with overlapping
functions involving regulation of protein turnover, RAS-RAF-MEK pathway and
transcription. VDAC1 predicted survival in breast cancer and myeloma and an
unsupervised analysis revealed enrichment of the VDAC1 signature in specific
subsets.</span></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>Conclusions</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>In summary, gene expression
analysis identifies VDAC1 gene expression as a predictor of poor outcome in
NSCLC and other cancers and is associated with dysregulation of a conserved set
of biological pathways, which may be causally associated with aggressive tumour
behaviour.</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/"
title="Go to other sections in this page"><span lang=EN style='font-size:12.0pt;
font-family:"Times New Roman","serif"'>Go to:</span></a></p>

<div style='border:none;border-bottom:solid #97B0C8 1.0pt;padding:0in 0in 0in 0in'>

<p class=MsoNormal style='line-height:normal;border:none;padding:0in'><b><span
lang=EN style='font-size:18.0pt;font-family:"Times New Roman","serif"'>Introduction</span></b></p>

</div>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif";background:lime'>Non-small cell
lung cancer (NSCLC) is a common, largely incurable cancer.</span><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'> In the early
disease setting, surgical resection is the standard of care and the benefit of
adjuvant chemotherapy is limited to 5�15% improvement in survival in stage II
but not stage I disease </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Vallieres1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[1]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.
However relapse is frequent in stage I cancer with 37% of patients dying within
five years, therefore there is a need to identify reliable biomarkers of poor
outcome in order to target individuals for whom adjuvant or novel targeted
therapy may improve survival. Alteration in tumour metabolism is a hallmark of
cancer and high levels of hexokinase-dependent glucose trapping in NSCLC
evidenced by standardized uptake values on positron emission tomography, is
associated with shorter survival </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Dakubo1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[2]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. This
is seen in both early and advanced NSCLC </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Hoang1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[3]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Dooms1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[4]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif";background:lime'>The Voltage
Dependent Anion Channel 1 (VDAC1) is an outer mitochondrial membrane protein
involved in the regulation of ATP/ADP exchange and respiratory control </span><span
style='background:lime'><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-VanderHeiden1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[5]</span></a></span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
background:lime'>.</span><span lang=EN style='font-size:12.0pt;font-family:
"Times New Roman","serif"'> VDAC1 interacts with proapoptotic BCL-2 family proteins
</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Hiller1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[6]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-VanderHeiden2"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[7]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.
However, its role in regulating mitochondrial outer membrane depolarization has
been controversial </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Madesh1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[8]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Shimizu1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[9]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Roy1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[10]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. Yet,
emerging data has identified a potential role for this protein in the
regulation of cell survival and for this reason we found it worthy of further
study. The permeability transition pore, which comprises VDAC1, is directly
regulated by the BH3-only proapoptotic BCL-2 family member BAD </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Roy1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[10]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> and
regulates cell survival. In contrast, the type II isoform (VDAC2) has been
identified as an antiapoptotic, negative regulator of BAK </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Shimizu1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[9]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Cheng1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[11]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Baines1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[12]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. The
adenine-nucleotide translocator (ANT) which is located in the inner
mitochondrial is also considered to be a major component of the permeability
transition pore </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Kumarswamy1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[13]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> and
along with VDAC it is believed to control mitochondrial remodeling processes
either directly or indirectly </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Park1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[14]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.
Recently, the bacterial protein FilA </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Sukumaran1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[15]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> has
been shown to regulate a complex of VDAC1 with hexokinase II </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Arzoine1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[16]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> which
inhibits apoptosis. A growing body of data suggests that hexokinase-VDAC
interaction is anti-apoptotic </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Pastorino1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[17]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, and
disruption of this interaction could be a strategy for inducing cell death </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Galluzzi1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[18]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Pastorino2"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[19]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. VDAC-hexokinase
complex has been reported to be regulated by glycogen synthase kinase 3 beta </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Pastorino2"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[19]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> and AKT
</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Majewski1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[20]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>Cancer cells with low VDAC1
induced by RNA interference exhibit a defect in cell growth in vivo </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Koren1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[21]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.
Conversely, high levels of VDAC1 expression levels may confer tumour cell
selection advantage by facilitating energy dependent processes such as
proliferation and invasiveness </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-AbuHamad1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[22]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.
Although hexokinase II expression has been linked to worse prognosis </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Rho1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[23]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, to
date there have been no studies exploring the potential prognostic impact of
VDAC1 in patients with cancer. It was therefore hypothesized that VDAC1
expression could impact survival following NSCLC resection. Here, we show using
gene expression meta-analysis that VDAC1 overexpression is a strong,
independent, poor prognostic factor in early stage non-small cell lung cancer
based on combined analysis of multiple gene expression datasets. VDAC1
overexpression is consistently associated with overexpression of 6 additional
genes involved in survival signalling, protein ubiquitination and ATP binding.
This association is maintained across multiple independent datasets and across
different cancers suggesting a potential role for these biological processes in
regulating tumour behaviour and therefore clinical outcome in NSCLC.</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/"
title="Go to other sections in this page"><span lang=EN style='font-size:12.0pt;
font-family:"Times New Roman","serif"'>Go to:</span></a></p>

<div style='border:none;border-bottom:solid #97B0C8 1.0pt;padding:0in 0in 0in 0in'>

<p class=MsoNormal style='line-height:normal;border:none;padding:0in'><b><span
lang=EN style='font-size:18.0pt;font-family:"Times New Roman","serif"'>Results</span></b></p>

</div>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>VDAC1
overexpression predicts shorter survival in NSCLC</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>In total, 8 independent NSCLC
datasets were curated from Gene Expression Omnibus (GEO), with a total of 602
microarrays. In all datasets high expression of VDAC1 (upper tertile) was
associated with significantly shorter survival compared to that of the lower
tertile for VDAC1 expression, p&#8202;=&#8202;0.0353, HR&#8202;=&#8202;0.6639
(0.4528 to 0.9721) (</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/figure/pone-0014635-g001/"
target=figure><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Figure
1A</span></a><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>).
The average overall survival was 101.6 months in patients with the lowest VDAC1
tertile and 52 months in patients with highest VDAC1 tertile.</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/figure/pone-0014635-g001/"
target=figure><span style='font-size:12.0pt;font-family:"Times New Roman","serif";
text-decoration:none'><img border=0 width=100 height=95 id="Picture 7"
src="PMC3031508%20table%201_files/image003.gif" alt="Figure 1"></span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/figure/pone-0014635-g001/"
target=figure><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Figure
1</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><b><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Survival
curves for all data and for Stage I patients based on VDAC1 expression.</span></b></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>VDAC1 expression
is an independent prognostic factor</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif";background:yellow'>To further
explore the impact of VDAC1 on survival, we carried out correlation and Cox
regression analysis as well as multivariate tests using PASW Statistics v.18
(SPSS: An IBM company).</span><span lang=EN style='font-size:12.0pt;font-family:
"Times New Roman","serif"'> High VDAC1 expression was shown to be an
independent prognostic variable for shorter survival p&lt;0.001 level. Stage
(p&#8202;=&#8202;0.013) and pT, (primary tumour), (p&#8202;=&#8202;0.022) also
proved to be independent variables correlated with survival, while multivariate
analysis showed that the combination of stage and histology significantly influences
patient survival (p&#8202;=&#8202;0.027). <span style='background:yellow'>The
results from the Cox regression analysis can be seen in Table
1, where VDAC1 has the most significant influence on survival
followed by pT and stage.</span></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/table/pone-0014635-t001/"
target=table><span style='font-size:12.0pt;font-family:"Times New Roman","serif";
text-decoration:none'><img border=0 width=204 height=142 id="Picture 6"
src="PMC3031508%20table%201_files/image004.png" alt="Table 1"></span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/table/pone-0014635-t001/"
target=table><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Table
1</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><b><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Cox
Regression analysis based on all patient data and VDAC1 expression.</span></b></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>VDAC1
overexpression predicts shorter time to recurrence following NSCLC resection</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>The influence of VDAC1 expression
on time to relapse following surgical resection was of interest as this
clinical variable is not confounded by therapy after relapse. It is therefore a
more accurate measure of prognostic significance. So we analysed only the
recurrence data and univariate analysis showed VDAC1 to significantly predict
time to recurrence (p&lt;0.001). Information on histological subtype, gender
and age was available for all samples and they were found to be not significant
with p values of 0.35, 0.82 and 0.73 respectively. A summary of this analysis
can be found in </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/table/pone-0014635-t002/"
target=true><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Table
2</span></a><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.
Similarly multivariate analysis ranked these variables in the same order, with
VDAC1 being the only independent factor (p&lt;0.001).</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/table/pone-0014635-t002/"
target=table><span style='font-size:12.0pt;font-family:"Times New Roman","serif";
text-decoration:none'><img border=0 width=204 height=87 id="Picture 5"
src="PMC3031508%20table%201_files/image005.png" alt="Table 2"></span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/table/pone-0014635-t002/"
target=table><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Table
2</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><b><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Univariate
Analysis of Time to Recurrence Data.</span></b></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>VDAC1
overexpression in Stage I samples leads to poor survival</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>Information on stage was not
available with all the datasets but we analysed what we had access to. We had
89 Stage I samples and the Kaplan-Meier plot for these patients is shown in </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/figure/pone-0014635-g001/"
target=figure><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Figure
1B</span></a><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>;
p&#8202;=&#8202;0.0455, with an associated hazard ratio of 0.52
(0.1492�0.7653). We found the correlation between VDAC1 expression and survival
to be very strong with p&lt;0.001. Cox regression analysis in stage 1 NSCLC,
where adjuvant chemotherapy has not been shown to improve patient outcome </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Winton1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[24]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Arriagada1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[25]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Douillard1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[26]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>,
revealed no variables to be statistically significant but, similarly to the
analysis of all the data, VDAC1 was the most influential predictor of survival
(p&#8202;=&#8202;0.241), and was ranked above histology
(p&#8202;=&#8202;0.344), age (p&#8202;=&#8202;0.465), pT
(p&#8202;=&#8202;0.970), pN, (nodal stage), (p&#8202;=&#8202;0.975) and gender
(p&#8202;=&#8202;0.999). The regression coefficient for VDAC1 was 29.18
indicating a very poor prognosis for patients with high levels of VDAC1.The
sample numbers were low for stages II and III with 68 samples being Stage II
and only 16 samples being Stage III, this made it difficult to obtain
statistically significant results. The correlation between VDAC1 expression and
survival in stage II samples and Stage III samples was insignificant with
p&#8202;=&#8202;0.853 and p&#8202;=&#8202;0.701 respectively. We had an
inadequate number of events in both stages to produce an accurate Kaplan Meier
plot with less than 10 incidences of death per category. The lack of higher
stage samples led us to focus on the output from Stage I samples and from the
combined data.</span></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>VDAC1
overexpression correlates with a conserved 6-gene signature</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>In order to gain an insight into
the biology underlying the poor prognosis of VDAC1 overexpressing NSCLC in
vivo, an analysis was conducted to identify which genes and gene networks are
consistently associated with high VDAC1 expression. A subset of 6 genes was
identified in VDAC1 overexpressing NSCLC datasets associated with significant
differential expression in 50% or more of the datasets. These genes were
CSNK1A1, G3BP1, HNRNPC, HSPA4, HSPA9 and UBE2D2; see </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/figure/pone-0014635-g002/"
target=figure><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Figure
2</span></a><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>
where the difference in the average expression of these genes when VDAC1 is low
and high is shown. As defined by the Gene Ontology Consortium </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Ashburner1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[27]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> HNRNPC,
HSPA4, HSPA9 and UBE2D2 are involved in protein ubiquitination pathways, and
CSNK1A1 regulates EIF2 function. HSPA9 is also involved in anti-apoptosis
pathways while G3BP1 is linked to Ras protein signal transduction and ATP
dependent DNA and RNA helicase activity. CSNK1A1, HSPA4 and HSPA9 are linked to
ATP binding while HNRNPC is involved in RNA and nucleotide binding. A summary
of the GO terms for these genes can be found in </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/table/pone-0014635-t003/"
target=true><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Table
3</span></a><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.
HNRNPC, HSPA9, G3BP1 and UBE2D2 were determined as being prognostic
individually when a univariate analysis was carried out on their expression
levels and survival time, the associated p values were 0.027, 0.002, 0.02 and
0.003 respectively.</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/figure/pone-0014635-g002/"
target=figure><span style='font-size:12.0pt;font-family:"Times New Roman","serif";
text-decoration:none'><img border=0 width=100 height=92 id="Picture 4"
src="PMC3031508%20table%201_files/image006.gif" alt="Figure 2"></span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/figure/pone-0014635-g002/"
target=figure><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Figure
2</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><b><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Change
in gene expression of our six significantly differentially expressed genes
linked to VDAC1 expression.</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/table/pone-0014635-t003/"
target=table><span style='font-size:12.0pt;font-family:"Times New Roman","serif";
text-decoration:none'><img border=0 width=204 height=113 id="Picture 3"
src="PMC3031508%20table%201_files/image007.png" alt="Table 3"></span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/table/pone-0014635-t003/"
target=table><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Table
3</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><b><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>A
table representing the gene ontology processes that define the genes in our
meta-signature.</span></b></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>Unsupervised
analysis identifies VDAC1 as prognostic and associated with a 6-gene signature</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>We also analysed the data without
filtering for VDAC1 expression to reaffirm VDAC1 would be significantly
differentially expressed when expression levels of living and dead patients are
compared. Using this unsupervised method and ranking gene expression according
to overall survival time, VDAC1 was found to be downregulated in patients with
longer survival. Similarly the 6 genes that are regulated alongside VDAC1
behaved in the same way, each of them were downregulated in samples from
patients that survived for a long period of time.</span></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>6-gene Signature
association with VDAC1 overexpression is independent of cancer type</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>To confirm the validity and
potential biological relevance of the 6-gene signature covariant with VDAC1 we
tested its discriminative power on an independent breast cancer dataset. A
t-test was performed comparing high and low VDAC1 expressing samples. We found
all 6 genes were upregulated when VDAC1 was highly expressed. Similarly,
expression was low when VDAC1 expression was low and so our signature of genes
covariant with VDAC1 in NSCLC data exhibited the same expression patterns in breast
cancer data. We also carried out the same test using an independent myeloma
cancer dataset, performing a t-test on the data which was split based on VDAC1
high and low expression. We again found all of the 6 genes in our
meta-signature returned as being statistically significant at a p&lt;0.001
level. Additional validation was found through the analysis of the NCI60 cell
lines found in </span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE2003"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE2003&quot;,&quot;term_id&quot;:&quot;2003&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE2003</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. We
again compared the samples based on VDAC1 expression and found the 6-gene
signature was conserved in 8 of these cell lines, namely UACC62 and SK_MEL2
(melanoma), SF295 and SF539 (CNS), A498 and SNC12C (renal), H23 and HOP62
(NSCLC). Analysing the NCI60 data for low VDAC1 with low expression of the
6-gene signature identified 5 cell lines, CCRF-CEM, RPMI-8226 and K562B
(leukaemia), IGROV1 (ovarian) and H522 (NSCLC).</span></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>Validation of
6-gene signature using Principal Components Analysis (PCA)</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>PCA helps to explain the variance
in data and is a common technique for dimensionality reduction in high
dimensional data. We performed the PCA on the NSCLC dataset </span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE3141"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE3141&quot;,&quot;term_id&quot;:&quot;3141&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE3141</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> and
found that although there were some outliers using the first principal
component, which accounted for as much of the variability as possible, all 6 of
the genes in our meta-signature were in the top 1% based on PC1 component
loadings. HNRNPC and G3BP1 performed particularly well being placed fourth and
fifth in the list respectively.</span></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>Protein-Protein
Interaction (PPI) Networks</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>Within the VDAC1 PPI network,
using Yu et al's </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Yu1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[28]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>
categorisation, encoded proteins were classed as either hubs (nodes with high
degree values constituting vulnerable areas of the network) and/or bottlenecks
(those with high betweenness centrality scores corresponding to key
intersecting nodes). VDAC1 and four of the genes, CSNK1A1, HSPA4, HNRNPC and
UBE2D2 were considered as both hub-bottlenecks.</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>Clustering identified potential
groups of genes operating in the same pathways, processes or molecular complexes.
Five of the genes, G3BP1, UBE2D2, VDAC1, CSNK1A1 and HSPA4 belonged to four
high-scoring clusters, the highest-ranked of these containing both UBE2D2 and
G3BP1. Functional analysis revealed this cluster to be most significantly
associated with the SH2 domain, ATP binding sites, protein and protein tyrosine
kinase activity, autophosphorylation, DNA binding and transcription factor
activity. The next most significant VDAC1-related cluster included HSPA4 which
was enriched in nuclear lumen and nucleoplasm components and RNA-binding and
mRNA splicing processes. VDAC1 itself was associated with a top-scoring cluster
functionally enriched in regulation of programmed cell death, apoptosis,
anti-apoptosis, nucleoplasm, intracellular organelle lumen, transcription
factor binding and repressor activities. The cluster of which CSNK1A1 was a
member was associated with significant functionality in coated pit,
endomembrane system, endocytosis, endosome, binding and transport activities.</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>Shortest path analysis revealed a
sub-network of 14 genes, comprising the VDAC1 signature genes and key
inter-connecting partners including TP53, RASA1, GRB2, CBL, CSK, RAF1 and
KPNA2, see </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/figure/pone-0014635-g003/"
target=figure><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Figure
3</span></a><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.
Only two of the genes HNRNPC and CSNK1A1 were direct interactors; HNRNPC's
binding ability being modulated by CSNK1A1-mediated phosphorylation.</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/figure/pone-0014635-g003/"
target=figure><span style='font-size:12.0pt;font-family:"Times New Roman","serif";
text-decoration:none'><img border=0 width=100 height=55 id="Picture 2"
src="PMC3031508%20table%201_files/image008.gif" alt="Figure 3"></span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/figure/pone-0014635-g003/"
target=figure><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Figure
3</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><b><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Shortest-path
PPI analysis of VDAC1 target genes.</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>Functional analysis revealed
enriched processes primarily associated with the VDAC1 target genes including
ATP-binding and nucleotide binding. Both the VDAC1-target and internal
shortest-path genes jointly participated in a number of significantly enriched
processes and pathways. These included signalling processes (intracellular
signalling cascade, RAS protein signal transduction, small GTPase mediated
signal transduction), transport (nucleocytoplasmic transport, nuclear
transport, intracellular protein transport) and anti-apoptosis.</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>The genes providing internal
links within the shortest-path network were significantly enriched in
signalling pathways, e.g. Neurotrophin, IGF-1, Insulin, TPO, PDGF, EGF,
Integrin, T Cell Receptor, MAPK and ERKB. Further pathways identified included:
signalling of hepatocyte growth factor receptor, transmembrane receptor protein
tyrosine kinase signalling pathway, enzyme linked receptor protein signalling
pathway, sprouty regulation of tyrosine kinase signals and IL-2 Receptor Beta
Chain in T cell Activation.</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>The main combined activities of
the VDAC1 genes (CSNK1A1, G3BP1, HSPA4, HSPA9 and UBE2D2) were focused on ATP
and nucleotide binding. Specific individual enriched functionality also
occurred, e.g. protein kinase activity and host-virus interaction (associated
with genes CSNK1A1 and VDAC1, respectively).</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>Interestingly, although less
prominent within the PPI network, the two non-hub-non-bottleneck genes, HSPA9
and G3BP1, were enriched in distinct sets of functions. In the case of G3BP1,
signal transduction was highlighted (both small GTPase and RAS protein) in
conjunction with the connecting genes, RAF1 and GRB2. HSPA9, in particular, was
associated with two groups of functions: protein targeting and transport
(intracellular protein, nuclear and nucleocytoplasmic) in conjunction with TP53
and KPNA2; and negative regulation of cell death, apoptosis and programmed cell
death (together with TP53 and RASA1).</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/"
title="Go to other sections in this page"><span lang=EN style='font-size:12.0pt;
font-family:"Times New Roman","serif"'>Go to:</span></a></p>

<div style='border:none;border-bottom:solid #97B0C8 1.0pt;padding:0in 0in 0in 0in'>

<p class=MsoNormal style='line-height:normal;border:none;padding:0in'><b><span
lang=EN style='font-size:18.0pt;font-family:"Times New Roman","serif"'>Discussion</span></b></p>

</div>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>VDAC1 overexpression predicts
shorter time to recurrence and overall survival for NSCLC as evidenced from
pooled gene expression analysis. It is an independent prognostic factor as
evidenced from Cox regression analysis and predicts survival in stage 1
disease, where subgroup analysis of randomized controlled clinical trials of
adjuvant chemotherapy have failed to show clinical benefit </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Winton1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[24]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Arriagada1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[25]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Douillard1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[26]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. A
combined gene expression data analysis approach has been employed to
investigate the impact of VDAC1 expression on survival, which was statistically
significant in all individual datasets examined.</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>Several recent studies analysing
microarray data for prognostic markers in NSCLC have produced inconsistent
results </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Boutros1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[29]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. The
distinct lack of overlap associated between these signatures reflects
instability and is attributed to small sample sizes with less than 200 samples
used per study. Analysis of multiple datasets, for example combined analysis of
microarray gene expression datasets addressing similar biological questions
conducted at an interpretational level by meta-analysis, can enable more
accurate results. Many studies propose methods for meta-analysis of microarray
data with the aim of identifying significantly differentially expressed genes
across studies using statistical techniques that avoid the direct comparison of
gene expression values </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Jiang1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[30]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. The
evaluation of multiple datasets as employed in this study has been shown to
yield more reliable and valid results, because they are based on large sample
numbers and the individual bias caused by each study is weakened </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Jiang1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[30]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>It is not known why VDAC1
correlates with poor survival outcomes. To understand the most relevant genetic
features of VDAC1 overexpressing NSCLC, we conducted a gene expression
meta-analysis </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-vanVliet1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[31]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Rhodes1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[32]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> to
identify a subset of genes (signature) that were significantly enriched. We
employed stringent statistical criteria, combined with a large sample size to
support the identification of these VDAC1-covariant genes. VDAC1 and 6 gene
signature was then validated across breast, myeloma and NCI-60 datasets </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Subramanian1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[33]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>,
suggesting enrichment of genes which were independent of the type of cancer.</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>Interestingly, of the 6 genes
identified as being conserved and significantly differentially regulated in the
high VDAC1 expressing group, most were functionally linked to the regulation of
protein turnover. These genes included heat shock 70kDa protein 4 (HSPA4),
ubiquitin-conjugating enzyme E2D 2 (UBE2D2), and heat shock 70kDa protein 9
(mortalin/HSPA9), encoding a glucose regulated 75 kilodalton protein previously
reported as correlating with poor survival in colorectal cancer </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Dundas1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[34]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. HSPA9
also binds and inactivates wild type p53 </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Kaul1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[35]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, and
regulates the RAS RAF MEK pathway<a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Wadhwa1">[36]</a>.
Similarly, p53 and RAS are targets of GTPase activating protein (SH3 domain)
binding protein 1 (G3BP1) </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Kim1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[37]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, and
shown to predict shorter survival in oesophageal cancer </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Zhang1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[38]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. Casein
kinase 1, alpha 1 (CSNK1A1) regulates protein turnover via initiation of
translation via EIF2 and participates in wnt signalling; a homologue CSKNK2A1
has been previously identified as an independent predictor of survival in
squamous cell lung cancer </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Oc1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[39]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. Like
G3BP1, heterogeneous nuclear ribonucleoprotein C (C1/C2) or HNRNPC is involved
in RNA binding and transcription </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Zinszner1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[40]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>The PPI networks identified that
the main combined activities of the VDAC1 genes (CSNK1A1, G3BP1, HSPA4, HSPA9
and UBE2D2) were focussed on ATP and nucleotide binding. Specific individual
enriched functionality also occurred, e.g. protein kinase activity and
host-virus interaction (associated with genes CSNK1A1 and VDAC1, respectively).
Interestingly, although less prominent within the PPI network, the two
non-hub-non-bottleneck genes, HSPA9 and G3BP1, were enriched in distinct sets
of functions. In the case of G3BP1, signal transduction was highlighted (both
small GTPase and RAS protein) in conjunction with the connecting genes, RAF1
and GRB2. HSPA9, in particular, was associated with two groups of functions:
protein targeting and transport (intracellular protein, nuclear and
nucleocytoplasmic) in conjunction with TP53 and KPNA2; and negative regulation
apoptosis (together with TP53 and RASA1).</span></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>Conclusions</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>In summary, based on the poor
prognosis associated with overexpression of VDAC1 and the associated 6-gene
signature, methods to target this molecular subclass of NSCLC may be effective
in improve survival outcomes after surgery. We have identified cell lines which
have upregulation of the 6-gene signature found in VDAC1 overexpressing NSCLC.
Ongoing work aims to target this signature to determine the effects on cell
viability utilizing the connectivity map. We propose that regulating the
expression of VDAC1 and/or genes with the 6-gene signature, could provide a
novel therapeutic strategy for targeting poor risk patients with NSCLC.</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/"
title="Go to other sections in this page"><span lang=EN style='font-size:12.0pt;
font-family:"Times New Roman","serif"'>Go to:</span></a></p>

<div style='border:none;border-bottom:solid #97B0C8 1.0pt;padding:0in 0in 0in 0in'>

<p class=MsoNormal style='line-height:normal;border:none;padding:0in'><b><span
lang=EN style='font-size:18.0pt;font-family:"Times New Roman","serif"'>Materials
and Methods</span></b></p>

</div>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>Gene expression
datasets</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>To conduct gene expression
meta-analysis, a library of 8 NSCLC gene expression datasets was curated from
NCBI Gene Expression Omnibus (GEO) </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Edgar1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[41]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>: </span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE8894"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE8894&quot;,&quot;term_id&quot;:&quot;8894&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE8894</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE3141"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE3141&quot;,&quot;term_id&quot;:&quot;3141&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE3141</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> (array
type: HG-U133 Plus2.0), </span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE6253"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE6253&quot;,&quot;term_id&quot;:&quot;6253&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE6253</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE4573"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE4573&quot;,&quot;term_id&quot;:&quot;4573&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE4573</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> (array
type: HG-U133A), </span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE6253"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE6253&quot;,&quot;term_id&quot;:&quot;6253&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE6253</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> (array
type: HG-U133B), </span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE6253"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE6253&quot;,&quot;term_id&quot;:&quot;6253&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE6253</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> (array
type: HG-U95Av2), </span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE4716"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE4716&quot;,&quot;term_id&quot;:&quot;4716&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE4716</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> (array
type: GeneFilter Human Microarray Release II) and </span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE5123"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE5123&quot;,&quot;term_id&quot;:&quot;5123&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE5123</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> (array
type: PC Human Operon v 2 21k). They had 602 samples in total and contained
gene expression information on primarily resected tumour samples that had
received no previous treatment. They were pre-processed using R </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-RDevelopmentCoreTeam1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[42]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> and
Bioconductor </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Gentleman1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[43]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.
Normalization was carried out using the RMA algorithm </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Irizarry1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[44]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> as
implemented in Bioconductor. Data was base-two log-transformed where
applicable. Along with gene expression measurements, there were also recordings
of patient information, including age, sex, histology and overall survival
time, and in some cases stage, recurrence free survival time, tumour stage (pT)
and nodal stage (pN) were also recorded. The genes that were found to be
significantly differentially expressed in the NSCLC datasets were then
validated in independent breast cancer datasets obtained from GEO: </span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE6434"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE6434&quot;,&quot;term_id&quot;:&quot;6434&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE6434</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>
containing HG-U95Av2 array data from 24 patients who had received chemotherapy
and </span><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE2034"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE2034&quot;,&quot;term_id&quot;:&quot;2034&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE2034</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> with
HG-U133A array data from 286 samples. We also tested the meta-signature in a
myeloma dataset (</span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE2658"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE2658&quot;,&quot;term_id&quot;:&quot;2658&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE2658</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>) with
HG-U133 Plus2.0 array data from 559 patients who had received chemotherapy.
Further validation was found through the analysis of the NCI60 cell line panel
(</span><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE2003"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE2003&quot;,&quot;term_id&quot;:&quot;2003&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE2003</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>) were
we checked gene expression matched what we found in our subset of significant
genes. Additionally we checked our results by carrying out principal components
analysis on the data in </span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE3141"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE3141&quot;,&quot;term_id&quot;:&quot;3141&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE3141</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.</span></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>Univariate and
multivariate survival analysis</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>The distribution of VDAC1
expression was subdivided into tertiles for each data set, corresponding to
low, medium and high. Kaplan-Meier curves were plotted using Prism (GraphPad
Software, San Diego California USA, </span><a href="http://www.graphpad.com/"
target="pmc_ext"><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>www.graphpad.com</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>) based
on survival data, comparing the highest versus lowest tertile for VDAC1,
initially these plots were produced for all data and then only for stage I.
Hazard ratios comparing low and high VDAC1 expressing samples were calculated.
We carried out correlation analysis between VDAC1 expression and survival time
using Spearman's Rank correlation coefficient. We also performed univariate and
multivariate analysis using PASW (SPSS: An IBM company), recurrence-free and
overall survival were used as clinical endpoints to assess prognostic significance
of VDAC1; the logrank ratio was considered to be significant if p&lt;0.05. <span
style='background:yellow'>Cox multivariate analysis was used to determine
whether VDAC1 is an independent prognostic variable in relation to 7 clinical
variables: age, gender, stage, histology, tumour size (pT), nodal stage (pN)
and survival.</span></span></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>Supervised
microarray meta-analysis</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>We adopted the method termed
meta-analysis of microarrays to determine what genes were regulated alongside
VDAC1 </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Ramasamy1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[45]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. Each
dataset was subdivided according to VDAC1 expression into tertiles; we compared
the top and bottom VDAC1 tertiles samples, using TMeV </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Saeed1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[46]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. Each
gene was assessed for differential expression using a two class t-test for each
dataset with class 1 being low VDAC1 expression and class 2, high VDAC1
expression. Bonferroni's correction was also used and this analysis was carried
out on all the 8 datasets. Bonferroni's correction helps minimise the
likelihood of generating false positives. The number of probes is vast with
some datasets containing more than 54,000, so to reduce our significant genes
to a manageable list per dataset we used a stringent cut off criteria where a
gene was labelled as significantly differentially expressed if the t-test
returned a p value of less than 0.001. The same hypotheses were tested in each
NSCLC dataset independently e.g. genes differentially expressed based on VDAC1
expression. To enable multiple hypothesis testing, we used a method that
compares statistical measures and Q values (estimated false discovery rates)
calculated independently from each dataset </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Rhodes1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[32]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Storey1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[47]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, where</span></p>

<p class=MsoNormal align=center style='text-align:center;line-height:normal;
vertical-align:middle'><span style='font-size:12.0pt;font-family:"Times New Roman","serif"'><img
border=0 width=364 height=87 id="Picture 1"
src="PMC3031508%20table%201_files/image009.jpg" alt="equation image"></span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>Where p is the probability, n is
the total number of genes and i is the sorted rank of the p value. The strict p
value cut-off we applied was carried forward when Q values were calculated.
After ranking the significantly differentially expressed genes based on their Q
value a direction and significance threshold of Q&lt;0.10 was set to define
differential expression signatures from the already computed differential
expression analysis. Having obtained an ordered list of differentially
expressed genes for each dataset genes were then filtered based on the number
of datasets in which they were differentially expressed.. The genes that were
defined as being significantly differentially expressed were then validated in
independent breast cancer datasets (</span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE6434"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE6434&quot;,&quot;term_id&quot;:&quot;6434&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE6434</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> and </span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE2034"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE2034&quot;,&quot;term_id&quot;:&quot;2034&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE2034</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>)
obtained from GEO with 24 and 286 samples respectively. We also tested the
meta-signature in a myeloma dataset (</span><a
href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE2658"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif";display:none'>{&quot;type&quot;:&quot;entrez-geo&quot;,&quot;attrs&quot;:{&quot;text&quot;:&quot;GSE2658&quot;,&quot;term_id&quot;:&quot;2658&quot;}}</span><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>GSE2658</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>) with
559 samples. T-tests were performed on this additional data; we compared high
and low VDAC1 expressing samples to determine if the genes from our signature
were identified once again. The same procedure was used on the NCI60 cell lines
to identify cell lines in which expression levels matched our signature.</span></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:13.5pt;font-family:"Times New Roman","serif"'>Protein protein
interaction network analysis</span></b></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN style='font-size:
12.0pt;font-family:"Times New Roman","serif"'>Using VDAC1 and the six
associated SDE genes as seeds together with their interactions found in the
Human Protein Reference Database (HPRD) </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Peri1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[48]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>, a
parent PPI network (including recursive interactions) was created using an
in-house Java-based program. Using these seed proteins as the initial depth
(level 0), subsequent depths were obtained using snowball sampling. The
resulting parent PPI network (9,266 gene nodes, 38,800) was then viewed,
manipulated and analyzed in Cytoscape </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Shannon1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[49]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>. Using
the Network Analyzer plug-in </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Assenov1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[50]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'> gene
nodes were then identified as either hubs or bottlenecks using the
classification of Yu et al </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Yu1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[28]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>; the
top 20% of nodes by degree value were identified as hubs, the remainder as
non-hubs. The same ranking method was applied to the betweenness centrality
metric to identify bottleneck nodes. Each node in the PPI network was thus
classified in one of four categories: non-hub-non-bottlenecks (NH/NB),
hub-non-bottlenecks (H/NB), non-hub-bottlenecks (NH/B), and hub-bottlenecks
(H/B). Multiple shortest paths between the seven genes were identified using
the PeSca tool. Key shortest paths were identified by considering direct
connections between the target genes, paths in which the target genes
themselves acted as inter-connectors, and the most frequently occurring genes
providing internal linkages. Modular functional enrichment analysis of the gene
clusters and shortest path network was carried out using the online tool DAVID </span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#pone.0014635-Huangda1"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>[51]</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>.</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/"
title="Go to other sections in this page"><span lang=EN style='font-size:12.0pt;
font-family:"Times New Roman","serif"'>Go to:</span></a></p>

<div style='border:none;border-bottom:solid #97B0C8 1.0pt;padding:0in 0in 0in 0in'>

<p class=MsoNormal style='line-height:normal;border:none;padding:0in'><b><span
lang=EN style='font-size:18.0pt;font-family:"Times New Roman","serif"'>Footnotes</span></b></p>

</div>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Competing
Interests: </span></b><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>The
authors have declared that no competing interests exist.</span></p>

<p class=MsoNormal style='line-height:normal'><b><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Funding: </span></b><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>CG is
funded by the Department of Employment and Learning. DAF is funded by Cancer
Research United Kingdom on a Clinician Scientist Fellowship. The funders had no
role in study design, data collection and analysis, decision to publish, or
preparation of the manuscript.</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/"
title="Go to other sections in this page"><span lang=EN style='font-size:12.0pt;
font-family:"Times New Roman","serif"'>Go to:</span></a></p>

<div style='border:none;border-bottom:solid #97B0C8 1.0pt;padding:0in 0in 0in 0in'>

<p class=MsoNormal style='line-height:normal;border:none;padding:0in'><b><span
lang=EN style='font-size:18.0pt;font-family:"Times New Roman","serif"'>References</span></b></p>

</div>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>1.
Vallieres E. Role of Adjuvant Systemic Therapy for Stage I NSCLC. Thoracic
Surgery Clinics. 2007;17:279�285. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/17626406" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>2.
Dakubo GD, Parr RL, Costello LC, Franklin RB, Thayer RE. Altered metabolism and
mitochondrial genome in prostate cancer. Journal of clinical pathology.
2006;59:10�16. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1860255/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/16394275" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>3.
Hoang JK, Hoagland LF, Coleman RE, Coan AD, Herndon JE, 2nd, et al. Prognostic
value of fluorine-18 fluorodeoxyglucose positron emission tomography imaging in
patients with advanced-stage non-small-cell lung carcinoma. Journal of clinical
oncology : official journal of the American Society of Clinical Oncology.
2008;26:1459�1464. [</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349396"
target="pmc_ext"><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>4.
Dooms C, van Baardwijk A, Verbeken E, van Suylen RJ, Stroobants S, et al.
Association between 18F-fluoro-2-deoxy-D-glucose uptake values and tumor
vitality: prognostic value of positron emission tomography in early-stage
non-small cell lung cancer. Journal of thoracic oncology : official publication
of the International Association for the Study of Lung Cancer. 2009;4:822�828.
[</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/19487964" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal><span lang=EN style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>5. Vander HeidenMG, Chandel NS,
Schumacker PT, Thompson CB. Bcl-xL Prevents Cell Death following Growth Factor
Withdrawal by Facilitating Mitochondrial ATP/ADP. Exchange Molecular Cell.
1999;3:159�167. [</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/10078199"
target="pmc_ext"><span lang=EN style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif"'>PubMed</span></a><span lang=EN
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>6.
Hiller S, Garces RG, Malia TJ, Orekhov VY, Colombini M, et al. Solution
structure of the integral human membrane protein VDAC-1 in detergent micelles.
Science (New York, NY) 2008;321:1206�1210. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2579273/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/18755977" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>7.
Vander Heiden MG, Li XX, Gottleib E, Hill RB, Thompson CB, et al. Bcl-xL
promotes the open configuration of the voltage-dependent anion channel and
metabolite passage through the outer mitochondrial membrane. The Journal of
biological chemistry. 2001;276:19414�19419. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/11259441" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>8.
Madesh M, Hajnoczky G. VDAC-dependent permeabilization of the outer
mitochondrial membrane by superoxide induces rapid and massive cytochrome c
release. The Journal of cell biology. 2001;155:1003�1015. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2150912/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/11739410" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>9.
Shimizu S, Narita M, Tsujimoto Y. Bcl-2 family proteins regulate the release of
apoptogenic cytochrome c by the mitochondrial channel VDAC. Nature.
1999;399:483�487. [</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/10365962"
target="pmc_ext"><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>10.
Roy SS, Madesh M, Davies E, Antonsson B, Danial N, et al. Bad targets the
permeability transition pore independent of Bax or Bak to switch between
Ca2+-dependent cell survival and death. Molecular cell. 2009;33:377�388. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2662194/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/19217411" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>11.
Cheng EH, Sheiko TV, Fisher JK, Craigen WJ, Korsmeyer SJ. VDAC2 inhibits BAK
activation and mitochondrial apoptosis. New York: Science. 2003;301:513�517. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/12881569" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>12.
Baines CP, Kaiser RA, Sheiko T, Craigen WJ, Molkentin JD. Voltage-dependent
anion channels are dispensable for mitochondrial-dependent cell death. Nature
cell biology. 2007;9:550�555. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2680246/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/17417626" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>13.
Kumarswamy R, Chandna S. Putative partners in Bax mediated cytochrome-c
release: ANT, CypD, VDAC or none of them? Mitochondrion. 2009;9:1�8. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/18992370" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>14.
Park J, Kim Y, Choi S, Koh H, Lee HE, et al. Drosophila Porin/VDAC Affects
Mitochondrial Morphology. Plos One. 2010 [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2951900/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/20949033" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>15.
Sukumaran SK, Fu NY, Tin CB, Wan KF, Lee SS, et al. A soluble form of the pilus
protein FimA targets the VDAC-hexokinase complex at mitochondria to suppress
host cell apoptosis. Molecular cell. 37:768�783. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/20347420" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>16.
Arzoine L, Zilberberg N, Ben-Romano R, Shoshan-Barmatz V. Voltage-dependent
anion channel 1-based peptides interact with hexokinase to prevent its
anti-apoptotic activity. The Journal of biological chemistry.
2009;284:3946�3955. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/19049977" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>17.
Pastorino JG, Shulga N, Hoek JB. Mitochondrial binding of hexokinase II
inhibits Bax-induced cytochrome c release and apoptosis. The Journal of
biological chemistry. 2002;277:7610�7618. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/11751859" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>18.
Galluzzi L, Kepp O, Tajeddine N, Kroemer G. Disruption of the hexokinase-VDAC
complex for tumor therapy. Oncogene. 2008;27:4633�4635. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/18469866" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>19.
Pastorino JG, Hoek JB, Shulga N. Activation of glycogen synthase kinase 3beta
disrupts the binding of hexokinase II to mitochondria by phosphorylating
voltage-dependent anion channel and potentiates chemotherapy-induced
cytotoxicity. Cancer research. 2005;65:10545�10554. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/16288047" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>20.
Majewski N, Nogueira V, Bhaskar P, Coy PE, Skeen JE, et al.
Hexokinase-mitochondria interaction mediated by Akt is required to inhibit
apoptosis in the presence or absence of Bax and Bak. Mol Cell. 2004;16:819�830.
[</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/15574336" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>21.
Koren I, Raviv Z, Shoshan-Barmatz V. Downregulation of voltage-dependent anion
channel-1 expression by RNA interference prevents cancer cell growth in vivo.
Cancer Biology Therapy. 2010;9 [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/20404552" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>22.
Abu-Hamad S, Sivan S, Shoshan-Barmatz V. The expression level of the
voltage-dependent anion channel controls life and death of the cell.
Proceedings of the National Academy of Sciences of the United States of
America. 2006;103:5787�5792. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1458651/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/16585511" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>23.
Rho M, Kim J, Jee CD, Lee YM, Lee HE, et al. Expression of type 2 hexokinase
and mitochondria-related genes in gastric carcinoma tissues and cell lines.
Anticancer Res. 2007;27:251�258. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/17352240" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>24.
Winton T, Livingston R, Johnson D, Rigas J, Johnston M, et al. Vinorelbine plus
cisplatin vs. observation in resected non-small-cell lung cancer. N Engl J Med.
2005;352:2589�2597. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/15972865" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>25.
Arriagada R, Bergman B, Dunant A, Le Chevalier T, Pignon JP, et al.
Cisplatin-based adjuvant chemotherapy in patients with completely resected
non-small-cell lung cancer. N Engl J Med. 2004;350:351�360. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/14736927" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>26.
Douillard JY, Rosell R, De Lena M, Carpagnano F, Ramlau R, et al. Adjuvant
vinorelbine plus cisplatin versus observation in patients with completely
resected stage IB-IIIA non-small-cell lung cancer (Adjuvant Navelbine
International Trialist Association [ANITA]): a randomised controlled trial.
Lancet Oncol. 2006;7:719�727. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/16945766" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>27.
Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, et al. Gene ontology:
tool for the unification of biology. The Gene Ontology Consortium. Nat Genet.
2000;25:25�29. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3037419/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>28.
Yu H, Kim PM, Sprecher E, Trifonov V, Gerstein M. The importance of bottlenecks
in protein networks: correlation with gene essentiality and expression
dynamics. PLoS computational biology. 2007;3:e59. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1853125/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/17447836" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>29.
Boutros PC, Lau SK, Pintilie M, Liu N, Shepherd FA, et al. Prognostic gene
signatures for non-small-cell lung cancer. Proc Natl Acad Sci U S A.
2009;106:2824�2828. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2636731/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/19196983" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>30.
Jiang H, Deng Y, Chen HS, Tao L, Sha Q, et al. Joint analysis of two microarray
gene-expression data sets to select lung adenocarcinoma marker genes. BMC
Bioinformatics. 2004;5:81. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC476733/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/15217521" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>31.
van Vliet MH, Reyal F, Horlings HM, van de Vijver MJ, Reinders MJ, et al.
Pooling breast cancer datasets has a synergetic effect on classification
performance and improves signature stability. BMC Genomics. 2008;9:375. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2527336/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/18684329" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>32.
Rhodes DR, Yu J, Shanker K, Deshpande N, Varambally R, et al. Large-scale
meta-analysis of cancer microarray data identifies common transcriptional
profiles of neoplastic transformation and progression. Proc Natl Acad Sci U S
A. 2004;101:9309�9314. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC438973/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/15184677" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>33.
Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, et al. Gene set
enrichment analysis: a knowledge-based approach for interpreting genome-wide
expression profiles. Proc Natl Acad Sci U S A. 2005;102:15545�15550. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1239896/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/16199517" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>34.
Dundas SR, Lawrie LC, Rooney PH, Murray GI. Mortalin is over-expressed by
colorectal adenocarcinomas and correlates with poor survival. J Pathol.
2005;205:74�81. [</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/15532096"
target="pmc_ext"><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>35.
Kaul SC, Aida S, Yaguchi T, Kaur K, Wadhwa R. Activation of wild type p53
function by its mortalin-binding, cytoplasmically localizing carboxyl terminus
peptides. J Biol Chem. 2005;280:39373�39379. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/16176931" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>36.
Wadhwa R, Yaguchi T, Hasan MK, Taira K, Kaul SC. Mortalin-MPD (mevalonate
pyrophosphate decarboxylase) interactions and their role in control of cellular
proliferation. Biochem Biophys Res Commun. 2003;302:735�742. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/12646231" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>37.
Kim MM, Wiederschain D, Kennedy D, Hansen E, Yuan ZM. Modulation of p53 and
MDM2 activity by novel interaction with Ras-GAP binding proteins (G3BP).
Oncogene. 2007;26:4209�4215. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/17297477" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>38.
Zhang HZ, Liu JG, Wei YP, Wu C, Cao YK, et al. Expression of G3BP and RhoC in
esophageal squamous carcinoma and their effect on prognosis. World J
Gastroenterol. 2007;13:4126�4130. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4205318/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/17696235" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>39.
Oc P, Rusch V, Talbot SG, Sarkaria I, Viale A, et al. Casein kinase II alpha
subunit and C1-inhibitor are independent predictors of outcome in patients with
squamous cell carcinoma of the lung. Clin Cancer Res. 2004;10:5792�5803. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/15355908" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>40.
Zinszner H, Albalat R, Ron D. A novel effector domain from the RNA-binding
protein TLS or EWS is required for oncogenic transformation by CHOP. Genes Dev.
1994;8:2513�2526. [</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/7958914"
target="pmc_ext"><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>41.
Edgar R, Domrachev M, Lash AE. Gene Expression Omnibus: NCBI gene expression
and hybridization array data repository. Nucleic Acids Res. 2002;30:207�210. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC99122/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/11752295" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>42.
R_Development_Core_Team. R: A Language and Environement for Statistical
Computing. Vienna, Austria: 2010. </span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>43.
Gentleman RC, Carey VJ, Bates DM, Bolstad B, Dettling M, et al. Bioconductor:
open software development for computational biology and bioinformatics. Genome
Biol. 2004;5:R80. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC545600/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/15461798" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>44.
Irizarry RA, Hobbs B, Collin F, Beazer-Barclay YD, Antonellis KJ, et al.
Exploration, normalization, and summaries of high density oligonucleotide array
probe level data. Biostatistics. 2003;4:249�264. [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/12925520" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>45.
Ramasamy A, Mondry A, Holmes CC, Altman DG. Key Issues in Conducting a
Meta-Analysis of Gene Expression Microarray Datasets. Plos Medicine. 2008;5 [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2528050/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/18767902" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>46.
Saeed AI, Sharov V, White J, Li J, Liang W, et al. TM4: a free, open-source
system for microarray data management and analysis. Biotechniques.
2003;34:374�378. [</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/12613259"
target="pmc_ext"><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>47.
Storey JD, Tibshirani R. Statistical significance for genomewide studies.
Proceedings of the National Academy of Sciences of the United States of
America. 2003;100:9440�9445. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC170937/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/12883005" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>48.
Peri S, Navarro JD, Amanchy R, Kristiansen TZ, Jonnalagadda CK, et al.
Development of human protein reference database as an initial platform for
approaching systems biology in humans. Genome Res. 2003;13:2363�2371. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC403728/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/14525934" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>49.
Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, et al. Cytoscape: a software
environment for integrated models of biomolecular interaction networks. Genome
research. 2003;13:2498�2504. [</span><a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC403769/"><span lang=EN
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PMC free article</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>] [</span><a
href="http://www.ncbi.nlm.nih.gov/pubmed/14597658" target="pmc_ext"><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>50.
Assenov Y, Ramirez F, Schelhorn SE, Lengauer T, Albrecht M. Computing
topological parameters of biological networks. Bioinformatics (Oxford, England)
2008;24:282�284. [</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/18006545"
target="pmc_ext"><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>51.
Huang da W, Sherman BT, Lempicki RA. Systematic and integrative analysis of
large gene lists using DAVID bioinformatics resources. Nat Protoc.
2009;4:44�57. [</span><a href="http://www.ncbi.nlm.nih.gov/pubmed/19131956"
target="pmc_ext"><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>PubMed</span></a><span
lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>]</span></p>

<ul type=disc>
 <li class=MsoNormal style='line-height:normal'><a
     href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#__abstractid593971title"><span
     lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
     display:none'>Abstract</span></a></li>
 <li class=MsoNormal style='line-height:normal'><a
     href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#s1title"><span
     lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
     display:none'>Introduction</span></a></li>
 <li class=MsoNormal style='line-height:normal'><a
     href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#s2title"><span
     lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
     display:none'>Results</span></a></li>
 <li class=MsoNormal style='line-height:normal'><a
     href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#s3title"><span
     lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
     display:none'>Discussion</span></a></li>
 <li class=MsoNormal style='line-height:normal'><a
     href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#s4title"><span
     lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
     display:none'>Materials and Methods</span></a></li>
 <li class=MsoNormal style='line-height:normal'><a
     href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#__fn-groupid876068title"><span
     lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
     display:none'>Footnotes</span></a></li>
 <li class=MsoNormal style='line-height:normal'><a
     href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031508/#__ref-listid930833title"><span
     lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif";
     display:none'>References</span></a></li>
</ul>

<div class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
text-align:center;line-height:normal'><span lang=EN style='font-size:12.0pt;
font-family:"Times New Roman","serif"'>

<hr size=2 width="100%" align=center>

</span></div>

<p class=MsoNormal align=center style='margin-bottom:0in;margin-bottom:.0001pt;
text-align:center;line-height:normal'><span lang=EN style='font-size:12.0pt;
font-family:"Arial","sans-serif"'>Articles from PLoS ONE are provided here
courtesy of <b>Public Library of Science</b></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;line-height:
normal'><span lang=EN style='font-size:12.0pt;font-family:"Times New Roman","serif"'>&nbsp;</span></p>

<p class=MsoNormal>&nbsp;</p>

</div>

</body>

</html>
