<article xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"> 
 <!--?properties open_acces=""?--> 
 <front> 
  <journal-meta> 
   <journal-id journal-id-type="nlm-ta">
     BMC Cancer 
   </journal-id> 
   <journal-h3>
     BMC Cancer 
   </journal-h3> 
   <issn pub-type="epub">
     1471-2407 
   </issn> 
   <publisher> 
    <publisher-name>
      BioMed Central 
    </publisher-name> 
   </publisher> 
  </journal-meta> 
  <article-meta> 
   <article-id pub-id-type="pmid">
     18597688 
   </article-id> 
   <article-id pub-id-type="pmc">
     2474852 
   </article-id> 
   <article-id pub-id-type="publisher-id">
     1471-2407-8-189 
   </article-id> 
   <article-id pub-id-type="doi">
     10.1186/1471-2407-8-189 
   </article-id> 
   <article-categories> 
    <subj-group subj-group-type="heading"> 
     <subject>
       Research Article 
     </subject> 
    </subj-group> 
   </article-categories> 
   <h3-group> 
    <article-h3>
      TDAG51 is an ERK signaling target that opposes ERK-mediated HME16C mammary epithelial cell transformation 
    </article-h3> 
   </h3-group> 
   <contrib-group> 
    <contrib contrib-type="author" id="A1"> 
     <name> 
      <surname>
        Oberst 
      </surname> 
      <given-names>
        Michael D 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       michaeloberst@gmail.com 
     </email> 
    </contrib> 
    <contrib contrib-type="author" id="A2"> 
     <name> 
      <surname>
        Beberman 
      </surname> 
      <given-names>
        Stacey J 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       sjb115@gmail.com 
     </email> 
    </contrib> 
    <contrib contrib-type="author" id="A3"> 
     <name> 
      <surname>
        Zhao 
      </surname> 
      <given-names>
        Liu 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       caljimmy@gmail.com 
     </email> 
    </contrib> 
    <contrib contrib-type="author" id="A4"> 
     <name> 
      <surname>
        Yin 
      </surname> 
      <given-names>
        Juan Juan 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       yinjuan@mail.nih.gov 
     </email> 
    </contrib> 
    <contrib contrib-type="author" id="A5"> 
     <name> 
      <surname>
        Ward 
      </surname> 
      <given-names>
        Yvona 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       yward@helix.nih.gov 
     </email> 
    </contrib> 
    <contrib contrib-type="author" corresp="yes" id="A6"> 
     <name> 
      <surname>
        Kelly 
      </surname> 
      <given-names>
        Kathleen 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       kellyka@mail.nih.gov 
     </email> 
    </contrib> 
   </contrib-group> 
   <aff id="I1"> 
    <label>1</label>Cell and Cancer Biology Branch, Center for Cancer Research, National Cancer Institute, National Institutes of Health, 37 Convent Drive, Room 1066, Bethesda, MD 20892, USA 
   </aff> 
   <pub-date pub-type="collection"> 
    <year>
      2008 
    </year> 
   </pub-date> 
   <pub-date pub-type="epub"> 
    <day>
      2 
    </day> 
    <month>
      7 
    </month> 
    <year>
      2008 
    </year> 
   </pub-date> 
   <volume>
     8 
   </volume> 
   <fpage>
     189 
   </fpage> 
   <lpage>
     189 
   </lpage> 
   <ext-link ext-link-type="uri" xlink:href="http://www.biomedcentral.com/1471-2407/8/189"></ext-link> 
   <history> 
    <date date-type="received"> 
     <day>
       24 
     </day> 
     <month>
       1 
     </month> 
     <year>
       2008 
     </year> 
    </date> 
    <date date-type="accepted"> 
     <day>
       2 
     </day> 
     <month>
       7 
     </month> 
     <year>
       2008 
     </year> 
    </date> 
   </history> 
   <permissions> 
    <copyright-statement>
      Copyright ? 2008 Oberst et al; licensee BioMed Central Ltd. 
    </copyright-statement> 
    <copyright-year>
      2008 
    </copyright-year> 
    <copyright-holder>
      Oberst et al; licensee BioMed Central Ltd. 
    </copyright-holder> 
    <license license-type="open-access" xlink:href="http://creativecommons.org/licenses/by/2.0"> 
     <p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License ( 
      <ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/2.0"></ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p> 
     <!--<rdf xmlns="http://web.resource.org/cc/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:dc="http://purl.org/dc/elements/1.1" xmlns:dcterms="http://purl.org/dc/terms"><Work xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dcterms="http://purl.org/dc/terms/" rdf:about=""><license rdf:resource="http://creativecommons.org/licenses/by/2.0"/><dc:type rdf:resource="http://purl.org/dc/dcmitype/Text"/><dc:author>
               Oberst
               D
               Michael
               
               michaeloberst@gmail.com
            </dc:author><dc:h3>
            TDAG51 is an ERK signaling target that opposes ERK-mediated HME16C mammary epithelial cell transformation
         </dc:h3><dc:date>2008</dc:date><dcterms:bibliographicCitation>BMC Cancer 8(1): 189-. (2008)</dcterms:bibliographicCitation><dc:identifier type="sici">1471-2407(2008)8:1&#x0003c;189&#x0003e;</dc:identifier><dcterms:isPartOf>urn:ISSN:1471-2407</dcterms:isPartOf><License rdf:about="http://creativecommons.org/licenses/by/2.0"><permits rdf:resource="http://web.resource.org/cc/Reproduction" xmlns=""/><permits rdf:resource="http://web.resource.org/cc/Distribution" xmlns=""/><requires rdf:resource="http://web.resource.org/cc/Notice" xmlns=""/><requires rdf:resource="http://web.resource.org/cc/Attribution" xmlns=""/><permits rdf:resource="http://web.resource.org/cc/DerivativeWorks" xmlns=""/></License></Work></rdf>--> 
    </license> 
   </permissions> 
   <abstract> 
    <h3>Abstract</h3> 
    <sec> 
     <h3>Introduction</h3> 
     <p><span style="background-color:#ffc8c8">Signaling downstream of Ras is mediated by three major pathways, Raf/ERK, phosphatidylinositol 3 kinase (PI3K), and Ral guanine nucleotide exchange factor (RalGEF)</span>. <span style="background-color:#ffa6a6">Ras signal transduction pathways play an important role in breast cancer progression, as evidenced by the frequent over-expression of the Ras-activating epidermal growth factor receptors EGFR and ErbB2</span>. <span style="background-color:#fff5f5">Here we investigated which signal transduction pathways downstream of Ras contribute to EGFR-dependent transformation of telomerase-immortalized mammary epithelial cells HME16C</span>. <span style="background-color:#fffbfb">Furthermore, we examined whether a highly transcriptionally regulated ERK pathway target, PHLDA1 (TDAG51), suggested to be a tumor suppressor in breast cancer and melanoma, might modulate the transformation process</span>.</p> 
    </sec> 
    <sec sec-type="methods"> 
     <h3>Methods</h3> 
     <p><span style="background-color:#ffd7d7">Cellular transformation of human mammary epithelial cells by downstream Ras signal transduction pathways was examined using anchorage-independent growth assays in the presence and absence of EGFR inhibition</span>. <span style="background-color:#ffecec">TDAG51 protein expression was down-regulated by interfering small hairpin RNA (shRNA), and the effects on cell proliferation and death were examined in Ras pathway-transformed breast epithelial cells</span>.</p> 
    </sec> 
    <sec> 
     <h3>Results</h3> 
     <p><span style="background-color:#ffa5a5">Activation of both the ERK and PI3K signaling pathways was sufficient to induce cellular transformation, which was accompanied by up-regulation of EGFR ligands, suggesting autocrine EGFR stimulation during the transformation process</span>. <span style="background-color:#fff0f0">Only activation of the ERK pathway was sufficient to transform cells in the presence of EGFR inhibition and was sufficient for tumorigenesis in xenografts</span>. <span style="background-color:#ffbebe">Up-regulation of the PHLDA1 gene product, TDAG51, was found to correlate with persistent ERK activation and anchorage-independent growth in the absence or presence of EGFR inhibition</span>. <span style="background-color:#fff0f0">Knockdown of this putative breast cancer tumor-suppressor gene resulted in increased ERK pathway activation and enhanced matrix-detached cellular proliferation of Ras/Raf transformed cells</span>.</p> 
    </sec> 
    <sec> 
     <h3>Conclusion</h3> 
     <p><span style="background-color:#ffe9e9">Our results suggest that multiple Ras signal transduction pathways contribute to mammary epithelial cell transformation, but that the ERK signaling pathway may be a crucial component downstream of EGFR activation during tumorigenesis</span>. <span style="background-color:#ffffff">Furthermore, persistent activation of ERK signaling up-regulates TDAG51</span>. <span style="background-color:#fff0f0">This event serves as a negative regulator of both Erk activation as well as matrix-detached cellular proliferation and suggests that TDAG51 opposes ERK-mediated transformation in breast epithelial cells</span>.</p> 
    </sec> 
   </abstract> 
  </article-meta> 
 </front> 
 <sec> 
  <h3>Background</h3> 
  <p><span style="background-color:#fffbfb">The development and progression of breast cancer is the result of multiple genetic changes, which lead to complex alterations in signal transduction networks in breast cancer cells relative to their normal epithelial counterparts</span>. <span style="background-color:#fff1f1">Signaling differences between tumor and normal cells are reflected in altered gene expression patterns, a finding that has been widely investigated using various molecular techniques</span>. <span style="background-color:#fff2f2">Patterns of differential gene expression have been used for classification and prognostication of certain cancers and may be valuable for prospectively predicting responsiveness to therapeutic treatments [ 1 ]</span>. <span style="background-color:#ffe4e4">In addition to serving as biomarkers, some differentially expressed gene products have been informative in defining the physiological differences between normal and cancer cells</span>. <span style="background-color:#ffefef">In order to understand the impact of dysregulated signaling pathways upon gene expression and function, it is necessary to connect differential gene expression to upstream signaling pathways</span>.</p> 
  <p><span style="background-color:#ffecec">Ras activation is a common intermediary in signaling pathways initiated by a variety of cell surface receptors, and signaling pathways downstream of Ras have been implicated repeatedly in oncogenesis</span>. <span style="background-color:#ffd2d2">Ras proteins are frequently mutated to an activated form in human cancers, particularly in tumors of the pancreas, colon, thyroid, and lung [ 2 ]</span>. <span style="background-color:#fff3f3">Ras mutations are rare in breast cancer, where the mutation rate is less than 5% [ 3 ]</span>. <span style="background-color:#fff4f4">However, Ras signaling is hyperactive in many primary breast tumors [ 4 ]</span>. <span style="background-color:#ffaeae">This is most likely due to the activation of growth factor receptors that activate Ras exchange factors, such as the epidermal growth factor receptors EGFR (ErbB1) and ErbB2 (HER2/neu) and the colony stimulating factor-1 (CSF-1) receptor c-fms [ 5 , 6 ]</span>. <span style="background-color:#fff1f1">EGFR and ErbB2 receptor over-expression occurs in one quarter to one-half of all breast tumors, and this correlates with a significantly decreased disease-free and overall survival rates [ 5 , 7 ]</span>. <span style="background-color:#ff2424">In addition, ligands for the EGFR such as transforming growth factor-a, amphiregulin, epiregulin, betacellulin, and heparin-binding EGF stimulate the receptor in primary breast tumors to enhance local growth and progression of the disease [ 8 ]</span>. <span style="background-color:#fff2f2">Finally, activating mutations in the PI3K pathway, an immediate downstream effector of Ras, occur in a subset of breast tumors [ 9 , 10 ]</span>. <span style="background-color:#fff2f2">Taken together, these findings suggest that breast cancer is a relevant model in which to study the biology of downstream Ras signal transduction</span>.</p> 
  <p><span style="background-color:#fff2f2">The Ras oncogene binds numerous effectors that in turn activate a variety of signaling pathways</span>. <span style="background-color:#ffc2c2">The most highly characterized of these are the Raf proteins, PI3K, and the Ral guanidine nucleotide exchange factors (RalGEFs), although Ras does recruit and activate other potentially transforming effectors [ 11 ]</span>. <span style="background-color:#fff1f1">The dissection of individual Ras signaling pathways is possible using effector domain mutants (EDMs) of Ras, for which single amino acid mutations in the effector binding domain allow the binding and activation of specific effectors, but not others [ 12 ]</span>. <span style="background-color:#fff4f4">These Ras EDMs, together with activated or dominant-negative versions of downstream effectors, have been used to characterize the pathways that contribute to the transformation of immortalized cells</span>.</p> 
  <p><span style="background-color:#ffdfdf">In mammary epithelial cells, both species- as well as cell line-dependent differences in Ras downstream signaling pathways have been found to induce tumorigenesis and/or anchorage-independent growth, a measure of in vitro transformation</span>. <span style="background-color:#fff0f0">Studies using immortalized mouse EpH4 mammary epithelial cells have implicated Raf as well as PI3K pathways in supporting transformation and tumorigenesis [ 13 ]</span>. <span style="background-color:#fffdfd">For human immortalized mammary epithelial cells, Raf and PI3K clearly contribute to transformation, although each is usually not sufficient for tumor formation in animal models [ 14 , 15 ]</span>. <span style="background-color:#fff2f2">In fact, the immortalized human breast epithelial cell line HMLE required simultaneous activation of Raf, PI3K, and the RalGEF pathways for maximal anchorage-independent growth and tumorigenic transformation [ 14 ]</span></p> 
  <p><span style="background-color:#fffefe">Dissecting the physiological consequences of individual Ras-mediated signaling pathways with respect to mammary epithelial transformation is of obvious interest</span>. <span style="background-color:#ffe9e9">The ability of activated Ras and Raf to induce autocrine expression of epidermal-like growth factors has been implicated in the protection of MCF10A mammary epithelial cells from anoikis [ 16 ]</span>. <span style="background-color:#ffe3e3">Using HMEC16C cells, a telomerase-immortalized human mammary epithelial cell line, we have investigated the contribution of EGFR signaling to anchorage-independent growth initiated by Raf and additional signaling pathways downstream of Ras</span>. <span style="background-color:#ffdada">We determined that ERK but not PI3K or RalGEF activation of HMEC16C cells supports anchorage independent proliferation independent of EGFR activation</span>.</p> 
  <p><span style="background-color:#ffcfcf">We performed a functional analysis of one gene in particular, TDAG51, whose expression is regulated by ERK through EGFR dependent and independent mechanisms</span>. <span style="background-color:#fffbfb">The loss of TDAG51 mRNA and protein has been correlated with breast adenocarcinoma and melanoma progression in clinical samples [ 17 , 18 ]</span>. <span style="background-color:#fffefe">The importance of TDAG51 regulation on the transformed phenotype of Ras-infected cells was addressed using TDAG51-specific interfering small hairpin RNA (shRNA) to reduce TDAG51 levels</span>. <span style="background-color:#ffffff">Consistent with a tumor suppressor role, loss of TDAG51 increased ERK-mediated cellular proliferation</span>.</p> 
 </sec> 
 <sec sec-type="methods"> 
  <h3>Methods</h3> 
  <sec> 
   <h3>Culture of human epithelial cell lines</h3> 
   <p><span style="background-color:#fff3f3">HME16C human mammary cells [ 19 ] were cultured in Clonetics Mammary Epithelial Basal Media (MEBM) with MEGM SingleQuot supplements (Cambrex), and HEK-HT human embryonic kidney epithelial cells [ 20 ] in DMEM (Invitrogen) plus 10% fetal bovine serum (Invitrogen)</span>. <span style="background-color:#fffefe">All cells were maintained at 37?C and 5% CO2</span>. <span style="background-color:#fffefe">For induction of proteins from the tetracycline-inducible retroviral expression vector pLRT, 250 ng/mL of doxycycline (Sigma-Aldrich) was added to culture media</span>.</p> 
  </sec> 
  <sec> 
   <h3>Retroviral and lentiviral constructs and infections</h3> 
   <p><span style="background-color:#fff6f6">Constructs for the inducible expression of H-Ras, H-Ras effector domain mutants, and Rlf-CAAX were created by PCR subcloning the sequences of HA-tagged H-RasG12V (RasV12), H-RasG12V, E37G (RasV12G37), H-RasG12V, T35S (RasV12S35), H-RasG12V, Y40C (RasV12C40), and Rlf-CAAX [ 21 ] into the tetracycline-inducible retroviral expression vector pLRT [ 22 ]</span>. <span style="background-color:#fff1f1">The generation of retroviruses and lentiviruses was as described [ 23 ]</span>. <span style="background-color:#ffffff">Polyclonal populations of pLRT infected cells were selected with 5 ?g/mL blasticidin</span>. <span style="background-color:#fff7f7">The expression of the anti-TDAG51 shRNA 36-1 was accomplished by sub-cloning the anti-TDAG51 shRNA sequence 5'-GGAACTGCACTTCTCCAACTTCAAGAGAGTTGGAGAAGTGCAGTTCCTTTTT-3' into the pLVTHM lentivirus (kindly provided by Dr</span>. D. <span style="background-color:#ffffff">Trono, Lausanne, Switzerland; [ 24 ])</span>. <span style="background-color:#fff9f9">After infection, cells with integrated lentivirus were selected by sterile sorting for GFP</span>. <span style="background-color:#fff9f9">Three sterile sorts were performed to acquire a polyclonal population of cells exclusively expressing GFP that was subsequently examined for TDAG51 protein reduction by western blotting</span>.</p> 
  </sec> 
  <sec> 
   <h3>Western blotting and Ral activation assays</h3> 
   <p><span style="background-color:#ffffff">Western blots were performed using Chemilluminescence (Pierce)</span>. <span style="background-color:#fff8f8">Antibodies used for western blots included anti-Ras mAb (BD Biosciences, cat#R02120), rat anti-HA mAb (Roche, cat#1988506), anti-phospho Erk mAb (Santa Cruz, cat#sc-7383), goat anti- Erk2 (Santa Cruz, C14-G, cat#154-G), rabbit anti-phospho-Akt Ser473 (Cell Signaling Technology), goat anti-Akt (Santa Cruz, C-20, cat#sc-1618), or an anti-alpha tubulin mAb (clone DM1A, NeoMarkers), followed by detection using appropriate anti-mouse, rabbit, or goat HRP-conjugated secondary antibodies (Jackson Labs) and ECL detection</span>. <span style="background-color:#fffefe">For Ral activation assays, the Ral Activation Assay Kit (Upstate Cell Signaling Solutions) was used according to the manufacturer's protocol</span>.</p> 
  </sec> 
  <sec> 
   <h3>Soft agar assays</h3> 
   <p><span style="background-color:#ffb2b2">Cells were trypsinized, neutralized, and 1.2 ? 104 cells were plated per well in 0.36% bacto-agar (BD Biosciences), w/v in growth media, on a 0.6% bacto-agar support in 6-well culture plates</span>. <span style="background-color:#fff2f2">Prior to cell plating, 250 ng/mL doxycycline, DMSO vehicle, or the anti-EGFR compound PD153035 (Calbiochem), was added to the agar, as appropriate</span>. <span style="background-color:#ffffff">Appropriately-supplememented fresh media was added to wells every 4 days</span>. <span style="background-color:#fffefe">Colonies greater than 100 ?m in diameter were counted after 18 days using a 2? lens equipped with a graded grid to determine colony size</span>.</p> 
  </sec> 
  <sec> 
   <h3>Alamar Blue growth assays in ultra-low attachment tissue culture plates</h3> 
   <p><span style="background-color:#fff4f4">Cells were trypsinized and plated in 500 ?L appropriate growth media into 24-well ultra-low attachment plates (Corning) at 25,000 cells per well</span>. <span style="background-color:#fffdfd">At the times indicated, 50 ?L of alamar blue (BioSource International) was added to each well and incubated for 6 hours under standard culture conditions</span>. <span style="background-color:#fff6f6">The amount of alamar blue reduction was quantified by measuring the fluorescence of each sample at 530 nm excitation/590 nm emission, corrected for the background fluorescence in wells containing media but no cells</span>. <span style="background-color:#ffdddd">The number of cells for initial plating and the alamar blue incubation times were determined by generating a standard curve of fluorescence versus cell number incubated with alamar blue for various times</span>. <span style="background-color:#fffefe">Plating cells initially at 25,000/well insures that fluorescence measurements will occur on a linear part of the curve with a 6-hour incubation time over the course of an experiment</span>. <span style="background-color:#ffdbdb">It was determined that TDAG51-specific shRNA did not affect the redox potential of HME16C at various cell numbers</span>. <span style="background-color:#ffeeee">Statistical analysis was performed using 2-way analysis of variance (ANOVA) with Bonferroni post-tests using the GraphPad Prism software program (GraphPad Software, Inc.)</span>.</p> 
  </sec> 
  <sec> 
   <h3>Tumorigenicity assays</h3> 
   <p><span style="background-color:#fff3f3">Tumorigenicity assays were done essentially as described [ 23 ]</span>. <span style="background-color:#fff1f1">To induce gene expression, cells were treated with 250 ng/mL doxycycline three days prior to injection, and mice were fed with doxycycline-containing diet (200 mg/kg, BioServ, Inc.) beginning four days prior to cell inoculation and continuing for the duration of the experiment</span>.</p> 
  </sec> 
  <sec> 
   <h3>Microarray analysis</h3> 
   <p><span style="background-color:#ffdbdb">Total RNA was extracted using Trizol reagent (Invitrogen) after treatment of cells with 250 ng/mL doxycycline for 72 hours to induce gene expression, or with doxycycline and 0.25 ?M EGFR receptor kinase inhibitor PD153035 as indicated</span>. <span style="background-color:#ffcece">Twenty micrograms of RNA were used for cDNA generation, and cDNA labeled with Cy3 or Cy5 monofunctional reactive dye to amino allyl-modified dUTP incorporated into cDNA using the FairPlay Microarray labeling kit (Stratagene)</span>. <span style="background-color:#ffd0d0">Labeled cDNA was hybridized to long-oligo (70-mer) cDNA microarrays from the NCI/CCR Microarray Center, NCI, Frederick, MD, according to standard protocols [ 25 ]</span>. <span style="background-color:#fffefe">Hybridized arrays were analyzed using a GenePix 4000B array scanner and GenePix Pro 4.0 software (Axon Instruments, Molecular Devices Corporation)</span>. <span style="background-color:#ffeded">Data from GenePix Pro 4.0 was uploaded to the microarray database at the NCI/CCR Microarray Center website [ 25 ] for further analysis</span>. <span style="background-color:#ffe6e6">Signal intensities of microarray features were calculated by subtracting the median local background from the median signal intensity</span>. <span style="background-color:#ffefef">Features were considered for analysis if the signal intensity was greater than one standard deviation above background with at least a 2:1 signal-to-background ratio</span>. <span style="background-color:#ffeded">Signal intensities for an entire microarray were normalized to the 50% percentile median value</span>. <span style="background-color:#ffd4d4">After filtering and normalization, the Cy3 and Cy5 values were expressed as a ratio to indicate the fold up- or down-regulation</span>. <span style="background-color:#ffffff">Two independent experiments for each comparison were performed, with a dye switch for each experiment, therefore yielding four separate data sets</span>. <span style="background-color:#fff3f3">For determining gene expression changes greater than or less than 2-fold, data sets were filtered for genes containing at least two significant values out of four array sets</span>. <span style="background-color:#ffd0d0">Prior to filtering, all data points were analyzed using statistical analysis of microarray (SAM) data and a resultant gene set was chosen at a delta value of 0.4 that limited the false discovery rate (number of predicted false positive values/number of significant genes) for each analysis to less than 1%</span>. <span style="background-color:#ffe1e1">Minimal information about a microarray experiment (MIAME)-compliant microarray data has been deposited with the National Center for Biotechnology Information (NCBI) Gene Expression Omnibus, accession number GSE8916, available at [ 26 ]</span></p> 
  </sec> 
  <sec> 
   <h3>Real-time RT-PCR analysis</h3> 
   <p><span style="background-color:#ff9f9f">cDNA was synthesized from RNA obtained for microarray analysis using the SuperScript? III First-Strand Synthesis System for RT-PCR (Invitrogen)</span>. <span style="background-color:#ff5858">Quantification of relative cDNA levels for each gene was accomplished using the Platinum? SYBR? Green qPCR Supermix UDG real-time RT-PCR kit (Invitrogen) and a Rotor-Gene3000? thermocycler with Rotor-Gene 5.0.37 software (Corbett Research) that calculates relative PCR synthesis rates by comparative quantification</span>. <span style="background-color:#ffc3c3">The specificity of product synthesis was verified by melting curve analysis by the Rotor-Gene 5.0.37 software, and by running of real-time PCR products on 2% agarose gels to verify product size and rule out primer-dimer contribution to calculated values</span>. <span style="background-color:#ff8181">The sequence-specific primers used in real-time RT-PCR can be found in Additional file 2 </span></p> 
  </sec> 
  <sec> 
   <h3>Cell cycle analysis</h3> 
   <p><span style="background-color:#fffcfc">Cells grown under anchorage-independent conditions were spun down, washed once in sterile PBS, and suspended in a 50% mixture of PBS and ACCUMAX cell detachment solution (Chemicon International) for 10 minutes at 25?C to dissociate cell clumps</span>. <span style="background-color:#fffdfd">After dissociation, cells were washed once in cold PBS and then fixed with 70% ethanol</span>. <span style="background-color:#fffefe">Fixed cells were treated with RNAse A for 20 minutes at 37?C, and nuclear DNA was stained with 50 ?g/mL propidium iodide at 4?C</span>. <span style="background-color:#fffdfd">Cell cycle profiles were generated using a FACSCalibur flow cytometer (BD Biosciences) and modeling cell cycle phases with the cell cycle option of FlowJo flow cytometry software (Tree Star, Inc.) after gating for viable, single cells</span>. <span style="background-color:#ffefef">Statistical analysis was performed using 2-way ANOVA with Bonferroni post tests using the GraphPad Prism software program (GraphPad Software, Inc.)</span>.</p> 
  </sec> 
  <sec> 
   <h3>EdU cell proliferation assay</h3> 
   <p><span style="background-color:#fff7f7">Measurement of cell proliferation by 5-ethynyl-2'-deoxyuridine (EdU) incorporation was measured using the Click-iT EdU cell proliferation Assay Kit for Flow Cytometry (Invitrogen) according to the manufacturer's protocol</span>. <span style="background-color:#fffefe">Briefly, cells were plated at 1 ? 106 cells per well in 6-well ultra-low attachment plates or 5 ? 105 cells per dish in 60 mm tissue culture dishes for attached control cells</span>. <span style="background-color:#fffdfd">Attached cells were treated with either DMSO or 10 ?M EdU for 4 hours, and cells grown in low attachment plates were treated with DMSO or 10 ?M EdU for 24 hours</span>. <span style="background-color:#ffe3e3">Using FlowJo flow cytometry software (Tree Star, Inc.), DMSO-treated control cells were used to determine the threshold above which cells could be considered EdU positive, and the fraction of cells incorporating EdU was then determined for experimental samples</span>.</p> 
  </sec> 
  <sec> 
   <h3>Cytotoxicity assay</h3> 
   <p><span style="background-color:#fffefe">The release of lactose dehydrogenase (LDH) into cell culture supernatant was measured using the cytotoxicity detection kit LDH (Roche Molecular Biochemicals) according to the manufacturer's instructions</span>.</p> 
  </sec> 
 </sec> 
 <sec> 
  <h3>Results</h3> 
  <sec> 
   <h3>Generation of H-Ras- and Rlf-CAAX-expressing HME16C cell lines</h3> 
   <p><span style="background-color:#fff3f3">Retroviral vectors coding for amino terminal HA-tagged activated H-RasV12, the H-RasV12 effector domain mutants (EDM) H-RasV12G37, H-RasV12S35, and H-RasV12C40, and the constitutively activated version of a RalGEF, Rlf-CAAX, were used to infect telomerase immortalized HME16C human mammary epithelial cells</span>. <span style="background-color:#ffe9e9">Anti-Ras and anti-HA western blotting demonstrated approximately equal levels of ectopic Ras expression among Ras-infected cells, with slightly lower levels in HME16C RasV12-infected cells relative to EDM-infected cells (Figure 1A )</span>. <span style="background-color:#ffecec">Analysis of activated, GTP-bound Ral A demonstrated highly elevated levels of activated Ral A only in Rlf-CAAX-expressing cells and not in Ras-infected cells grown under standard culture conditions (Figure 1B )</span>. <span style="background-color:#ffe0e0">To assess activation of the ERK pathway, anti-phospho Erk western blotting was performed and showed significantly elevated Erk phosphoprotein in RasV12- and RasV12S35-infected HME16C cells relative to control pLRT-infected cells (Figure 1A )</span>. <span style="background-color:#ffeaea">Elevated levels of phosphorylated Erk were also observed in RasV12G37- and RasV12C40-infected cells, although at a much lower level than that found in RasV12- and RasV12S35-infected cells</span>. <span style="background-color:#fffefe">To assess activation of the PI3K signaling pathway, anti-phosphoAkt (Ser473) western blotting was performed to detect activated, phosphorylated Akt</span>. <span style="background-color:#ffe1e1">In cells that were serum starved for 24 hours and matrix detached for 6 hours, elevated levels of phosphorylated Akt were observed in RasV12-, RasV12C40-, and RasV12G37-infected HME16C, with highest levels present in RasV12-infected cells (Figure 1C )</span></p> 
   <fig id="F1" position="float"> 
    <label>Figure 1</label> 
    <p>
     <bold> 
      <span style="background-color:#ffdada">Protein expression and pathway activation in HME16C cell lines </span> 
     </bold>. <span style="background-color:#fff3f3">(A) Expression of HA-tagged H-RasV12, H-RasV12 effector domain mutants, and Rlf-CAAX in HME16C mammary epithelial cells</span>. <span style="background-color:#ffefef">In this and subsequent figures, cells infected with the following vectors are indicated as follows: pLRT control (pLRT), H-RasV12 (V12), H-RasV12G37 (G37), H-RasV12S35 (S35), H-RasV12C40 (C40), and Rlf-CAAX (Rlf-CAAX)</span>. <span style="background-color:#fff3f3">Cells were treated with 250 ng/mL doxycycline for 72 hours to induce gene expression, and anti-HA (HA tag) and anti-Ras (Ras) western blotting were performed</span>. <span style="background-color:#ffe0e0">Erk activation was determined by western blotting for phosphorylated Erk (Phospho-Erk)</span>. <span style="background-color:#fffdfd">The blot was subsequently stripped and probed with an anti-Erk2 antibody (Erk2)</span>. <span style="background-color:#ffe0e0">(B) Ral A activation was determined by a pull-down assay for GTP-bound, activated Ral A (GTP-Ral A)</span>. <span style="background-color:#ffdddd">Total Ral A protein from lysates used in the Ral activation assay pull-down was determined by anti-Ral A western blotting (Ral A)</span>. <span style="background-color:#fff7f7">(C) Akt activation in cell lysates was assessed by western blotting for phosphorylated Akt (Phospho-Akt)</span>. <span style="background-color:#fffdfd">The membrane was stripped and probed with an anti-Akt antibody (Total Akt)</span>.</p> 
    <graphic xlink:href="1471-2407-8-189-1"></graphic> 
   </fig> 
  </sec> 
  <sec> 
   <h3>Anchorage-independent growth of mammary epithelial cell lines</h3> 
   <p><span style="background-color:#ffe4e4">To assess transformation by different Ras signaling pathways, anchorage-independent growth assays were performed in soft agar and in ultra-low attachment tissue culture plates</span>. <span style="background-color:#ffeaea">Ras- and Ras EDM-infected HME16C cells formed significantly more soft agar colonies &gt;100 ?m in diameter than pLRT vector-infected cells (Figure 2A )</span>. <span style="background-color:#fff5f5">The RasV12-infected cells formed large colonies, many exceeding 1000 ?m, although the total number exceeding 100 ?m was typically less than that for the Ras EDM (Figure 2B )</span>. <span style="background-color:#fff5f5">Among the Ras EDM-infected cells, the RasV12S35-infected cells formed the largest colonies</span>. <span style="background-color:#fffefe">These were similar to, but smaller than, the RasV12-infected cells</span>. <span style="background-color:#fffcfc">For colonies above 100 ?m in diameter, RasV12C40-infected cells were the most efficient at colony formation, despite the smaller mean size of colonies</span>. <span style="background-color:#ffeeee">Rlf-CAAX-infected cells formed slightly more colonies above 100 ?m than vector-transfected control cells, but these were significantly smaller than those formed by Ras- and Ras EDM-infected cells</span>.</p> 
   <fig id="F2" position="float"> 
    <label>Figure 2</label> 
    <p>
     <bold>
       Anchorage-independent growth of HME16C cell lines 
     </bold>. <span style="background-color:#ffecec">(A) Number of soft agar colonies exceeding 100 ?M after 21 days of growth for HME16C cell lines infected with pLRT control, H-RasV12, H-RasV12G37, H-RasV12S35, H-RasV12C40, and Rlf-CAAX-expressing constructs</span>. <span style="background-color:#fffdfd">(B) Representative pictures of colony sizes in soft agar assays</span>. <span style="background-color:#ffffff">Bar represents 200 ?m</span>. <span style="background-color:#fff5f5">(C) Anchorage-independent growth of cell lines grown on ultra-low attachment plates treated with DMSO vehicle (left) or 0.25 ?M PD153035 (right) measured using the alamar blue assay</span>. <span style="background-color:#ffffff">RFU, relative fluorescence units of alamar blue product formed</span>.</p> 
    <graphic xlink:href="1471-2407-8-189-2"></graphic> 
   </fig> 
   <p><span style="background-color:#fff4f4">When grown under anchorage-independent conditions in ultra-low attachment plates, the accumulation of cells for the various infected cell lines roughly paralleled the total cell masses seen in soft agar growth assays (Figure 2C )</span>. <span style="background-color:#ffaaaa">The RasV12- and RasV12 EDM-expressing cells grew well, while the growth of the Rlf-CAAX-expressing cells was significantly less</span>. <span style="background-color:#fff3f3">The HME16C cells maintained viability but did not increase in number</span>.</p> 
   <p><span style="background-color:#fff1f1">To compare our results to others, we verified the function of pLRT vector driven Ras EDM mutants and Rlf-CAAX in HEK-HT cells, which previously have been reported to form colonies in soft agar upon expression of H-RasV12G37 and Rlf-CAAX [ 27 ]</span>. <span style="background-color:#fff2f2">In our hands, expression of H-RasV12, H-RasV12G37, and Rlf-CAAX from the pLRT vector induced efficient soft agar colony growth, and H-RasV12S35 and H-RasV12C40 did not (not shown), identical to previously reported results [ 27 ]</span>. <span style="background-color:#fff3f3">Expression of exogenous H-Ras and Rlf-CAAX, and activation of endogenous RalA, in this cell line was similar to that observed in HME16C cells (not shown)</span>. <span style="background-color:#ffe9e9">Therefore, expression of Rlf-CAAX and subsequent RalA activation did not appear to be sufficient to induce anchorage-independent growth in the HME16C mammary epithelial cell line in contrast to HEK-HT and various other immortalized human cell types [ 27 ]</span></p> 
  </sec> 
  <sec> 
   <h3>Tumorigenesis of HME16C cell lines in nude mice</h3> 
   <p><span style="background-color:#ffe8e8">Anchorage-independent growth often predicts the ability of cells to grow as xenografted tumors in immuno-compromised mice</span>. <span style="background-color:#ffffff">Tumor formation was assessed following subcutaneous inoculation</span>. <span style="background-color:#fff3f3">The RasV12-infected HME16C cells formed rapidly growing, fluid-filled tumors with an average latency of 4 weeks and a mean tumor volume of 808.8 mm3 at 6 weeks (Table 1 )</span>. <span style="background-color:#fffdfd">Approximately one-half of the tumors were aspirated prior to sacrifice, and a sero-sanguinous fluid was observed, on average accounting for roughly one-third of the measured tumor volume</span>. <span style="background-color:#ffeeee">Histological analysis of H&amp;E stained tumor sections revealed poorly differentiated spindle-shaped tumor cells with prominent squamous cell differentiation and extracellular keratin deposition (not shown)</span>. <span style="background-color:#ffffff">Tumors also contained a strong inflammatory component</span>. <span style="background-color:#fff1f1">Of the other cell lines tested, only the RasV12S35-infected HME16C cells formed palpable tumors in 50% of injected animals with an average latency of approximately 12 weeks and a mean tumor volume of 109.0 mm3 at 16 weeks, considerably smaller than RasV12-expressing tumors (Table 1 )</span>. <span style="background-color:#fffdfd">Cells within RasV12S35-infected tumors resembled the histology of RasV12 tumor cells but with less keratin deposition and without the formation of fluid-filled spaces (not shown)</span>. <span style="background-color:#fffefe">Empty vector-, RasV12G37-, RasV12C40-, and Rlf-CAAX-infected cells failed to form palpable tumors four months after injection</span>. <span style="background-color:#ffd3d3">The metastatic potential of RasV12- and RasV12G37-expressing cell lines was tested by tail-vein injection in nude mice, but no metastatic lesions were observed by histological analysis in lungs, liver, spleen, or kidneys at 16 weeks post injection (not shown)</span></p> 
   <table-wrap id="T1" position="float"> 
    <label>Table 1</label> 
    <p>Growth of HME16C tumors in nude mice</p> 
    <table frame="hsides" rules="groups"> 
     <thead> 
      <tr> 
       <td align="left">Cell Line Innoculated</td> 
       <td align="left">Number of Mice with Tumors</td> 
       <td align="left">Mean Tumor Latency</td> 
       <td align="left">Mean Tumor Volume at Sacrifice (mm<sup>3</sup>)*</td> 
      </tr> 
     </thead> 
     <tbody> 
      <tr> 
       <td align="left">pLRT</td> 
       <td align="left">0/10</td> 
       <td align="left">-</td> 
       <td align="left">-</td> 
      </tr> 
      <tr> 
       <td align="left">H-Ras<sup>V12</sup></td> 
       <td align="left">10/10</td> 
       <td align="left">4 weeks</td> 
       <td align="left">808.8 +/- 635.2</td> 
      </tr> 
      <tr> 
       <td align="left">H-Ras<sup>V12G37</sup></td> 
       <td align="left">0/10</td> 
       <td align="left">-</td> 
       <td align="left">-</td> 
      </tr> 
      <tr> 
       <td align="left">H-Ras<sup>V12S35</sup></td> 
       <td align="left">5/10</td> 
       <td align="left">12 weeks</td> 
       <td align="left">109.0 +/- 171.0</td> 
      </tr> 
      <tr> 
       <td align="left">H-Ras<sup>V12C40</sup></td> 
       <td align="left">0/10</td> 
       <td align="left">-</td> 
       <td align="left">-</td> 
      </tr> 
      <tr> 
       <td align="left">Rlf-CAAX</td> 
       <td align="left">0/10</td> 
       <td align="left">-</td> 
       <td align="left">-</td> 
      </tr> 
     </tbody> 
    </table> 
    <table-wrap-foot> 
     <p>Cells infected with control, Ras-expressing, or Rlf-CAAX-expressing retrovirus were injected subcutaneously into nude mice and examined for tumor formation and growth. <span style="background-color:#ffffff">*For mice injected with H-RasV12S35 cells, includes only sites that formed tumors</span></p> 
    </table-wrap-foot> 
   </table-wrap> 
  </sec> 
  <sec> 
   <h3>Autocrine EGFR signaling is required for Ras<sup>V12G37</sup>- and Ras<sup>V12C40</sup>-mediated, but not Ras<sup>V12S35</sup>-mediated, HME16C cell anchorage- independent growth</h3> 
   <p><span style="background-color:#fff1f1">EGFR signaling is frequently altered in breast cancer, where EGFR and ErbB2 over-expression are common events</span>. <span style="background-color:#ff0000">cDNA microarray and real-time RT-PCR analysis of HME16C RasV12- and Ras EDM-infected cells (Additional file 1 ) revealed increased levels of mRNAs for EGFR ligands, including epiregulin, amphiregulin, and TGFa (Table 2 )</span>. <span style="background-color:#ffe9e9">In addition, increased levels of phospho-Erk were unexpectedly seen in RasV12G37- and RasV12C40-infected cells, possibly the result of autocrine activation of endogenous EGFR by secreted EGFR ligands</span>. <span style="background-color:#fff3f3">The presence of EGFR was established using Western blots (not shown)</span>. <span style="background-color:#ffdada">Therefore, we sought to determine if autocrine signaling by the EGFR was required for the anchorage-independent growth of Ras- or Ras EDM-infected cells</span>. <span style="background-color:#ffd6d6">RasV12- and Ras EDM-infected HME16C cells were grown in soft agar in the presence or absence of the EGFR-specific inhibitors, PD153035 and PD168393</span>. <span style="background-color:#ffe0e0">Complete inhibition of colony formation for RasV12G37-infected and RasV12C40-infected cells was observed at 0.25 ?M PD153035, a concentration that specifically inhibits EGFR [ 28 ], whereas both the RasV12- and RasV12S35-infected cells formed colonies efficiently at this same concentration of inhibitor (Figure 3A and 3B )</span>. <span style="background-color:#fff2f2">Identical results were found for the EGFR-specific inhibitor PD168393 used at 0.1 ?M, a concentration that specifically inhibits EGFR and Her-2 receptors [ 29 ](not shown)</span>. <span style="background-color:#ffcdcd">Similarly, treatment of cells grown in ultra-low-attachment plates also demonstrated that EGFR inhibition substantially inhibited growth of RasV12G37-and RasV12C40-expressing cells relative to that of RasV12- and RasV12S35-expressing HME16C (Figure 2C )</span>. <span style="background-color:#ffe9e9">Western blotting of cellular lysates from cells treated with 0.25 ?M PD153035 showed that high levels of phosphorylated Erk were maintained only in RasV12- and RasV12S35-infected cells, but were significantly reduced in RasV12G37- or RasV12C40-infected cells treated with the inhibitor (Figure 3C ), while phoshorylated Akt was minimally affected (not shown)</span>. <span style="background-color:#ffebeb">Anchorage-independent growth therefore correlated with maintenance of high Erk activity in HME16C cells</span>. <span style="background-color:#fff6f6">Consistent with this observation, inhibition of MEK, and therefore ERK signaling, using the MEK-specific inhibitor PD98059 at 10 ?M, significantly inhibited soft agar colony formation by all cell lines (not shown)</span>.</p> 
   <table-wrap id="T2" position="float"> 
    <label>Table 2</label> 
    <p>Up-regulation of EGFR ligands in Ras<sup>V12</sup>- and Ras EDM-expressing HME16C</p> 
    <table frame="hsides" rules="groups"> 
     <thead> 
      <tr> 
       <td align="left">Gene</td> 
       <td align="left">V12</td> 
       <td align="left">S35</td> 
       <td align="left">G37</td> 
       <td align="left">C40</td> 
      </tr> 
     </thead> 
     <tbody> 
      <tr> 
       <td align="left">epiregulin</td> 
       <td align="left"> 
        <bold>
          2.6 
        </bold>(7.0)</td> 
       <td align="left"> 
        <bold>
          1.9 
        </bold>(1.0)</td> 
       <td align="left"> 
        <bold>
          1.8 
        </bold>(2.4)</td> 
       <td align="left"> 
        <bold>
          1.2 
        </bold>(1.3)</td> 
      </tr> 
      <tr> 
       <td align="left">TGFa</td> 
       <td align="left"> 
        <bold>
          3.5 
        </bold>(3.4)</td> 
       <td align="left"> 
        <bold>
          2.9 
        </bold>(1.3)</td> 
       <td align="left"> 
        <bold>
          -1.1 
        </bold>(-1.4)</td> 
       <td align="left"> 
        <bold>
          1.9 
        </bold>(-1.1)</td> 
      </tr> 
      <tr> 
       <td align="left">amphiregulin</td> 
       <td align="left"> 
        <bold>
          1.4 
        </bold>(2.4)</td> 
       <td align="left"> 
        <bold>
          2.9 
        </bold>(1.0)</td> 
       <td align="left"> 
        <bold>
          2.3 
        </bold>(1.7)</td> 
       <td align="left"> 
        <bold>
          2.2 
        </bold>(1.0)</td> 
      </tr> 
     </tbody> 
    </table> 
    <table-wrap-foot> 
     <p>Fold up-regulation of the EGFR ligands epiregulin, transforming growth factor alpha (TGFa), and epiregulin in Ras<sup>V12</sup>- and Ras EDM-expressing HME16C as determined by real-time RT-PCR quantification (boldface) and cDNA microarray analysis (in parentheses).</p> 
    </table-wrap-foot> 
   </table-wrap> 
   <fig id="F3" position="float"> 
    <label>Figure 3</label> 
    <p><span style="background-color:#ffc9c9">Anchorage-independent growth of HME16C cell lines after EGFR inhibition </span>. <span style="background-color:#ffe5e5">(A) Cell lines, as indicated, were grown in the soft agar assay for 21 days either in the presence of DMSO vehicle or 0.25 ?M PD153035 EGFR inhibitor</span>. <span style="background-color:#ffffff">Scale bar, 200 ?m</span>. <span style="background-color:#fffcfc">(B) Number of colonies &gt;100 ?m in size for each cell line treated with either DMSO vehicle or 0.25 ?M of the EGFR-specific inhibitor PD153035</span>. <span style="background-color:#ffefef">Statistical analysis was performed using one-way analysis of variance with Bonferroni post-test analysis</span>. <span style="background-color:#ffffff">NS, not significant, p &gt; 0.05; *, p &lt; 0.0001</span>. <span style="background-color:#fff4f4">(C) Activation of Erk after EGFR inhibition</span>. <span style="background-color:#fff6f6">Cell lines were grown in the presence of DMSO vehicle or 0.25 ?M PD153035 for 48 hours, followed by western blotting for phosphorylated Erk (Phospho-Erk)</span>. <span style="background-color:#ffdede">The blot was stripped and probed with an anti-Erk 2 antibody (Erk 2) to verify equal protein loading</span>. <span style="background-color:#fffcfc">The phospho-ERK blot was overexposed in order to reveal minimal levels of protein</span>.</p> 
    <graphic xlink:href="1471-2407-8-189-3"></graphic> 
   </fig> 
  </sec> 
  <sec> 
   <h3>Microarray analysis of gene expression changes in Ras<sup>V12</sup>-, Ras<sup>V12G37</sup>-, Ras<sup>V12S35</sup>-, and Ras<sup>V12C40</sup>-infected HME16C cells</h3> 
   <p><span style="background-color:#ffd3d3">Activation of the Ras oncogene is accompanied by the stimulation of multiple signal transduction pathways leading to the activation or repression of numerous transcription factors as well as changes in mRNA translation and stability, and thus, the modulation of gene expression</span>. <span style="background-color:#ffaaaa">To determine which gene expression changes accompany the transformation of HME16C human epithelial cells by activated Ras, we examined our transformed HME16C cells by cDNA microarray analysis</span>. <span style="background-color:#fff3f3">To do this, RNA was isolated from H-RasV12 and H-RasV12 EDM expressing cells after treatment with doxycycline to fully induce gene expression and compared to RNA from identically treated pLRT vector-infected control cells</span>. <span style="background-color:#ffdcdc">Statistical analysis of microarray (SAM) data analysis was performed for the datasets, and a delta value of 0.4 was chosen for each dataset, which maintains the estimated false discovery rate (FDR) below 1% for each</span>. <span style="background-color:#ffdcdc">A summary of the genes up- or down-regulated greater than 2-fold in the H-RasV12- and Ras effector domain mutant-infected HME16C cell lines is presented in Additional file 1 , organized according to broad categories of gene function</span>. <span style="background-color:#ff5252">To validate gene expression changes identified by cDNA microarray analysis, quantitative RT-PCR was performed using RNA from the same samples used in microarray analysis, and is presented in Additional file 2 </span>. <span style="background-color:#ffafaf">Results for 22 of 26 genes chosen to reflect genes up- or down-regulated both strongly or weakly showed strong agreement with microarray data, demonstrating that the microarray dataset represents a reliable quantification of gene expression changes</span>.</p> 
   <p><span style="background-color:#ffb1b1">To evaluate the effect of EGFR inhibition on gene expression, RasV12-, RasV12S35-, and RasV12G37-infected cells were induced with doxycycline and subsequently incubated either in the presence or absence of 0.25 ?M PD153035, and microarray analysis comparisons were made to vehicle-treated pLRT-infected cells</span>. <span style="background-color:#ffe3e3">Almost all Ras and Ras EDM-induced upregulated transcriptional responses were blocked by pharmacological inhibition of EGFR, consistent with previous reports for inhibition of Raf-regulated transcription [ 30 ]</span>. <span style="background-color:#ffabab">Our analysis identified PHLDA1 as an up-regulated gene in both vehicle-treated and PD153035-treated RasV12 and RasV12S35 cells, although the relative fold increase was reduced following EGFR inhibition</span>. <span style="background-color:#fff7f7">By comparison, PHLDA1 was down-regulated in PD153035-treated RasV12G37 relative to vehicle-treated cells (not shown)</span>. <span style="background-color:#ffeaea">Therefore, PHLDA1 represents a Raf/ERK-responsive gene whose expression parallels EGFR-independent HME16C mammary epithelial cell transformation</span>.</p> 
  </sec> 
  <sec> 
   <h3>TDAG51 expression is up-regulated by Ras signaling in a ERK-dependent manner, and is associated with EGFR-independent transformation</h3> 
   <p><span style="background-color:#ffe3e3">The PHLDA1 gene is of interest as it has been suggested to be a tumor suppressor in breast adenocarcinoma and melanoma [ 17 , 18 ]</span>. <span style="background-color:#fff1f1">We further analyzed the signal dependent expression of the PHLDA1 gene and its protein product, TDAG51</span>. <span style="background-color:#ff9e9e">Microarray analysis identified the PHLDA1 gene as being dramatically up-regulated in RasV12- and Ras EDM-infected cells to levels that correlated with the level of ERK activation and the extent of anchorage-independent growth (Additional file 1 )</span>. <span style="background-color:#fffefe">Western blotting confirmed that TDAG51 was also upregulated in a similar manner (Figure 4A )</span>. <span style="background-color:#ffd2d2">The PHLDA1 gene was elevated in PD153035-treated RasV12- and RasV12S35-infected cells but was significantly dependent upon EGFR tyrosine kinase activity for upregulation in RasV12G37- and RasV12C40-infected cells (not shown), and the expression of the encoded TDAG51 protein approximately paralleled PHLDA1 RNA expression (Figure 4B )</span>. <span style="background-color:#ffd3d3">As shown in Figure 3C , EGFR inhibition significantly reduced ERK signaling in RasV12G37- and RasV12C40-infected cells without affecting RasV12- and RasV12S35-infected cells</span>. <span style="background-color:#ffc8c8">To confirm that TDAG51 up-regulation was induced specifically by ERK activation, we treated pLRT-, RasV12-, and RasV12S35-infected cells with the MEK-specific inhibitor PD98059</span>. <span style="background-color:#fff3f3">PD98059 used at 20 ?M appears to be specific for MEK1 as it does not nonspecifically inhibit a variety of other protein kinases that have been assayed</span>. <span style="background-color:#ffffff">[ 31 ]</span>. <span style="background-color:#ffdcdc">TDAG51 up-regulation was attenuated by MEK inhibition (Figure 5 )</span>. <span style="background-color:#ffcbcb">TDAG51 therefore represents an ERK-inducible gene whose up-regulation in HME16C is correlated with an EGFR-independent, ERK-mediated transformation</span>.</p> 
   <fig id="F4" position="float"> 
    <label>Figure 4</label> 
    <p>
     <bold> 
      <span style="background-color:#ffcece">Expression of TDAG51 in HME16C cell lines in the absence or presence of the EGFR inibition </span> 
     </bold>. <span style="background-color:#fff4f4">Western blotting of TDAG51 in HME16C cell lines</span>. <span style="background-color:#ffeeee">(A) Twenty-five micrograms of protein from HME16C cellular lysates were resolved by SDS-PAGE and probed with an anti-TDAG51 monoclonal antibody (TDAG51)</span>. <span style="background-color:#ffd8d8">Equal protein loading was verified by stripping the blot and probing with an alpha tubulin monoclonal antibody (Alpha tubulin)</span>. Band intensities were determined using Image J software (NIH, Bethesda, MD) and TDAG51 values were normalized using alpha tubulin values. <span style="background-color:#ff0000">Fold expression of each band relative to HME16C parental: HME16C parental, 1.0; pLRT, 1.0; V12, 3.0; G37, 2.3; S35, 2.4; C40, 1.3; Rlf-CAAX, 0.9</span>. Similar experiments to that shown were performed 2 additional times with similar results. <span style="background-color:#fff5f5">(B) HME16C cell lines were treated with DMSO vehicle or 0.25 ?M PD153035 EGFR-specific inhibitor for 48 hours, and cell lysates were prepared</span>. <span style="background-color:#fff7f7">Twenty five micrograms of protein were resolved by SDS-PAGE and western blotted with anti-TDAG51 mAb (TDAG51)</span>. <span style="background-color:#ffdede">The blot was subsequently stripped and probed with an alpha tubulin mAb (Alpha tubulin)</span>. Band intensities were determined using Image J software (NIH, Bethesda, MD) and TDAG51 values were normalized using alpha tubulin values. <span style="background-color:#fff8f8">DMSO-treated cells are indicated by a -, and PD1535035-treated cells are indicated by a +</span>. <span style="background-color:#ff2020">Fold expression relative to DMSO-treated pLRT: pLRT -(1.0), +(0.2); V12 -(2.3), +(2.6); G37 -(0.8), +(0.3); S35 -(1.4), +(0.9); C40 -(1.1), +(0.6); Rlf-CAAX -(1.6), +(1.0)</span>. Similar experiments to that shown were performed 2 additional times with similar results.</p> 
    <graphic xlink:href="1471-2407-8-189-4"></graphic> 
   </fig> 
   <fig id="F5" position="float"> 
    <label>Figure 5</label> 
    <p>
     <bold> 
      <span style="background-color:#ffe0e0">TDAG51 expression is induced by ERK pathway activation </span> 
     </bold>. <span style="background-color:#fff0f0">HME16C cell lines infected with pLRT empty vector (pLRT), RasV12 (V12), or RasV12S35 (S35) were treated with 0.2% DMSO vehicle or 20 ?M PD98059 for 24 hours, followed by western blotting for TDAG51</span>. <span style="background-color:#ffdede">The blot was stripped and probed with an alpha tubulin monoclonal antibody (Alpha tubulin) to demonstrate equal protein loading</span>. Band intensities were determined using Image J software (NIH, Bethesda, MD) and TDAG51 values were normalized using alpha tubulin values. <span style="background-color:#ff9090">Fold expression of PD98059-treated relative to DMSO-treated contols: pLRT (0.6); V12 (0.6); S35 (0.3)</span>. <span style="background-color:#ffffff">Similar experiments to that shown was performed 1 additional time with similar results</span>.</p> 
    <graphic xlink:href="1471-2407-8-189-5"></graphic> 
   </fig> 
  </sec> 
  <sec> 
   <h3>TDAG51 up-regulation opposes ERK-mediated HME16C transformation</h3> 
   <p><span style="background-color:#ffe1e1">To analyze the role of TDAG51 in ERK-dependent growth, we reduced TDAG51 expression in RasV12- and RasV12S35-infected cells to a level comparable to that in non-transformed vector-infected control cells using stably expressed TDAG51-specific shRNA (Figure 6A )</span>. <span style="background-color:#fff9f9">Cell proliferation of attached cells grown on tissue culture plastic was unaffected by lowered TDAG51 protein levels (not shown)</span>. <span style="background-color:#ffefef">However, cell growth under anchorage-independent conditions in ultra-low attachment plates was significantly enhanced by TDAG51 knock-down in RasV12S35-infected cells (Figure 6B )</span>. <span style="background-color:#fff7f7">Likewise, results with RasV12-infected cells stably infected with TDAG51-targeting shRNA also showed enhanced growth relative to vector-infected control cells, although to a lesser extent than that seen with RasV12S35-infected cells (Figure 6C )</span>. <span style="background-color:#ffecec">This suggests that Ras signaling pathways other than ERK compensate partially for the negative growth effects of TDAG51, or that RasV12-infected cells are already close to maximally transformed under anchorage-independent growth conditions</span></p> 
   <fig id="F6" position="float"> 
    <label>Figure 6</label> 
    <p><span style="background-color:#ffabab">Reduction of TDAG51 expression in Ras V12 and Ras V12S35-infected cells and their anchorage-independent growth </span>. <span style="background-color:#fff2f2">(A) Western blotting of TDAG51 protein (TDAG51) from 5 ?g of cellular lysates from non-silencing vector control (pLVTHM) and anti-TDAG51 shRNA (36-1) -expressing cell lines as indicated</span>. <span style="background-color:#fffdfd">The blot was subsequently stripped and probed with an anti-alpha tubulin mAb (Alpha tubulin)</span>. <span style="background-color:#ffc6c6">Band intensities were determined using Image J software (NIH, Bethesda, MD) and TDAG51 values were normalized using alpha tubulin values</span>. <span style="background-color:#ff4444">Fold expression of each band relative to HME16C pLRT pLVTHM: pLRT pLVTHM (1.0), 36-1 (0.4); V12 pLVTHM (2.3), 36-1 (1.0); S35 pLVTHM (1.5), 36-1 (0.7)</span>. <span style="background-color:#ffcece">Similar experiments to that shown were performed 2 additional times with similar results (B) Growth of pLVTHM vector control or TDAG51 shRNA-expressing RasV12S35-transduced cells either in the presence of 0.2% DMSO control (left) or 0.25 ?M PD153035 (right) in low-attachment plates, assessed by the alamar blue growth assay</span>. <span style="background-color:#ffffff">RFU, relative fluorescence units</span>. <span style="background-color:#ffeded">(C) Growth of pLVTHM vector control or TDAG51 shRNA-expressing RasV12-transduced cells either in the presence of 0.2% DMSO control (left) or 0.25 ?M PD153035 (right) in ultra-low-attachment plates, assessed by the alamar blue growth assay</span>. RFU, relative fluorescence units.</p> 
    <graphic xlink:href="1471-2407-8-189-6"></graphic> 
   </fig> 
  </sec> 
  <sec> 
   <h3>Loss of TDAG51 expression in transformed cells stimulates both cell cycle progression and apoptosis</h3> 
   <p><span style="background-color:#ffeded">To characterize the effect of TDAG51 on cell proliferation under anchorage-independent conditions, cell cycle analysis and cell proliferation assays for RasV12S35- and RasV12-infected cells were performed</span>. <span style="background-color:#ffe4e4">Both RasV12S35- and RasV12- TDAG51 shRNA-expressing cells demonstrated an increased S-phase fraction versus pLVTHM vector control cells at various time points during anchorage-independent growth (Figure 7A )</span>. <span style="background-color:#fffbfb">In concordance with these results, RasV12S35 and RasV12 cells expressing the TDAG51-specific shRNA showed enhanced incorporation of 5-ethynyl-2'-deoxyuridine (EdU) in cell proliferation assays, indicating a higher rate of DNA synthesis in cells with reduced TDAG51 protein (Figure 7C )</span></p> 
   <fig id="F7" position="float"> 
    <label>Figure 7</label> 
    <p>
     <bold> 
      <span style="background-color:#ffe4e4">Cell proliferation is increased in Ras-transformed cell lines with reduced TDAG51 expression </span> 
     </bold>. <span style="background-color:#ffb0b0">(A) S phase fraction of RasV12S35-expressing (left panel) or RasV12-expressing (right panel) HME16C infected with pLVTHM non-silencing vector control or anti-TDAG51 shRNA 36-1 after 48, 72, 96 or 120 hours growth in ultra-low attachment plates as determined by cell cycle analysis of propidium iodide-stained nuclei</span>. * p &lt; 0.05; **p &lt; 0.01; ***p &lt; 0.001. <span style="background-color:#ffeaea">(B) Incorporation of the nucleoside analogue EdU into cellular DNA measured in RasV12S35-expressing (left panel) or RasV12-expressing (right panel) HME16C infected with pLVTHM non-silencing vector control or anti-TDAG51 shRNA 36-1 after 48, 72, 96 or 120 hours growth in ultra-low attachment plates</span>. <span style="background-color:#fff7f7">Incorporated EdU was quantified by detection of Alexa Fluor 488-labeled EdU in viable single cells by flow cytometry analysis</span>.</p> 
    <graphic xlink:href="1471-2407-8-189-7"></graphic> 
   </fig> 
   <p><span style="background-color:#fff4f4">Since cell growth under anchorage-independent conditions is a balance between cell proliferation and cell death, we sought to evaluate the effect upon cellular death of TDAG51 knock-down</span>. <span style="background-color:#fffefe">We used an assay of cellular cytotoxicity that measures the release of the lactose deydrogenase enzyme, LDH</span>. <span style="background-color:#ffefef">LDH release was increased by TDAG51 shRNA-expressing RasV12S35 cells relative to pLVTHM-infected cells at various time points after the initiation of matrix-detached growth (Figure 8 )</span>. <span style="background-color:#fffbfb">The difference in LDH release for TDAG51 shRNA-expressing RasV12 cells was minimal and rarely approached statistical significance</span>. <span style="background-color:#ffefef">Sub-G1 peaks indicative of dead cells were sometimes seen with cell cycle analysis at late time points, but varied from experiment to experiment</span>. <span style="background-color:#fff4f4">Therefore, for RasV12S35-infected cells, the differences in cell growth after TDAG51 reduction under anchorage-independent conditions resulted from an enhanced rate of cellular proliferation that exceeded a concomitant increase in cell death</span></p> 
   <fig id="F8" position="float"> 
    <label>Figure 8</label> 
    <p><span style="background-color:#ffcdcd">LDH release by Ras V12S35- or Ras V12-expressing HME16C infected with pLVTHM vector or anti-TDAG51 shRNA 36-1 </span>. <span style="background-color:#fff6f6">(A) Relative LDH activity released by pLVTHM- or anti-TDAG51 shRNA 36-1-infected RasV12S35 cells was measured in the culture media of cells grown under anchorage-independent conditions in ultra-low attachment plates for the times indicated</span>. <span style="background-color:#ffffff">* p &lt; 0.05; **p &lt; 0.01; ***p &lt; 0.001</span>. <span style="background-color:#fff9f9">(B) Relative LDH activity released by pLVTHM- or anti-TDAG51 shRNA 36-1-infected RasV12 cells</span>. <span style="background-color:#ffffff">* p &lt; 0.05</span></p> 
    <graphic xlink:href="1471-2407-8-189-8"></graphic> 
   </fig> 
  </sec> 
  <sec> 
   <h3>Reduction of TDAG51 in transformed cells enhances proximal ERK signaling</h3> 
   <p><span style="background-color:#fff5f5">Reducing TDAG51 protein levels in ERK-driven cellular transformation enhanced cell growth under anchorage-independent, but not attached, conditions</span>. <span style="background-color:#fffcfc">To test whether TDAG51 might affect proximal ERK signaling, we examined the activation status of Erk in cells expressing TDAG51-specific shRNA</span>. <span style="background-color:#fffbfb">Interestingly, the levels of phosphorylated Erk were enhanced when TDAG51 protein levels were reduced in RasV12S35 and RasV12 cells grown under anchorage-independent, but not attached, conditions (Figure 9 )</span>. <span style="background-color:#fffcfc">The fact that the activation status of Erk was unchanged in cells grown under attached conditions suggests that reducing TDAG51 expression had no selective effect with regard to ERK activation in these cells</span>. <span style="background-color:#fff5f5">Rather, the enhanced activation of Erk was specific to anchorage-independent growth conditions</span>.</p> 
   <fig id="F9" position="float"> 
    <label>Figure 9</label> 
    <p><span style="background-color:#ffb4b4">ERK pathway activation in Ras V12 and Ras V12S35 cells infected with pLVTHM or anti-TDAG51 shRNA 36-1 </span>. <span style="background-color:#fffcfc">RasV12 or RasV12S35 cells infected with either pLVTHM vector control or anti-TDAG51 shRNA 36-1 were grown for 72 hours attached to tissue culture plastic (attached) or in ultra-low attachment plates (anchorage-independent) in the presence of 250 ng/mL doxycycline</span>. <span style="background-color:#fff9f9">ERK pathway activation was measured by determination of phosphorylated Erk1/2 levels by western blotting using an anti-phosphorylated Erk mAb (phospho-Erk)</span>. <span style="background-color:#fffdfd">The blot was stripped and subsequently probed with an anti-Erk2 mAb (Erk2)</span>.</p> 
    <graphic xlink:href="1471-2407-8-189-9"></graphic> 
   </fig> 
  </sec> 
 </sec> 
 <sec> 
  <h3>Discussion</h3> 
  <p><span style="background-color:#fff6f6">Ras is a common signaling node for various cell surface receptors that contribute to epithelial cell transformation [ 14 ]</span>. <span style="background-color:#ffe8e8">In this study, we used the hTERT-immortalized human mammary epithelial cell line HME16C to examine which Ras signaling pathways are sufficient for transformation and to identify transcriptional targets downstream of those pathways that might modulate this phenotype</span>. <span style="background-color:#ffe2e2">Transduction of HME16C with pathway-discriminating Ras effector domain mutants demonstrated that multiple downstream Ras signal transduction pathways contribute to anchorage-independent growth including Raf-, RalGEF-, and PI3K-mediated signaling</span>. <span style="background-color:#ffeeee">Transformation of HME16C by the RasV12G37 effector domain mutant but not activated Rlf-CAAX suggest that RasV12G37-binding effectors other than RalGEF contribute to mammary epithelial transformation</span></p> 
  <p><span style="background-color:#ffafaf">Microarray analyses of RasV12 and Ras effector domain mutant transduced cells demonstrated a common upregulation of EGFR ligands among transformed cell lines</span>. <span style="background-color:#fff4f4">This suggested that autocrine EGFR ligand secretion was an important component of Ras-mediated cellular transformation</span>. <span style="background-color:#ffdfdf">Following blockade of EGFR signaling with the EGFR-specific inhibitor PD153035, the only pathway downstream of Ras that promoted anchorage-independent growth was Raf/ERK, suggesting that Raf activation was able to substitute for EGFR activity in this cell line</span>. <span style="background-color:#fff2f2">In contrast, previous studies with MCF10A cells demonstrated that EGFR tyrosine kinase activity was necessary to inhibit anoikis upon matrix detachment, even in cells expressing activated Raf [ 16 ]</span>. <span style="background-color:#ffefef">By contrast, under matrix detached conditions, the parental HME16C cells are non-proliferative, but do not actively undergo anoikis</span>. <span style="background-color:#fffcfc">The studies here measure a proliferative response of Raf/ERK activation in the absence of a strong apoptotic component</span>. <span style="background-color:#ffe9e9">In addition, blockade of MEK activity with the MEK-specific inhibitor PD98059 prevented the transformation of RasV12- and RasV12S35-expressing cells</span>. <span style="background-color:#ffe0e0">This, taken together with the finding that RasV12- and RasV12S35-transduced HME16C cells were able to form tumors in nude mice, indicates that the Raf-MEK-Erk axis plays a crucial role in mediating transformation and tumorigenesis in this model</span>. <span style="background-color:#ffe8e8">However, it should be noted that Ras-transformed HME16C, as well as the similar HMLER [ 32 ], give rise to squamous metaplasias and not adenocarcinomas</span>. <span style="background-color:#fffefe">It has been hypothesized that HMLE human mammary epithelial cells represent a distinct precursor population from those mammary epithelial cells that give rise to glandular adenocarcinomas, the predominant form of breast cancer [ 32 ]</span></p> 
  <p><span style="background-color:#ffd6d6">The significantly reduced tumorigenicity of RasV12S35- as compared to RasV12- expressing cells emphasizes the importance of cooperation between Ras signal transduction pathways for a fully transformed phenotype</span>. <span style="background-color:#ffebeb">Following EGFR inhibition, RasV12C40- and RasV12G37- transduced lines lost anchorage-independent growth, suggesting that non-ERK signaling pathways contribute to cellular transformation through an EGFR dependent mechanism</span>. <span style="background-color:#fff0f0">Indeed, one consequence of EGFR activation is ERK pathway activation and may explain the role of EGFR in cooperating with RasV12G37- and RasV12C40-mediated cell signaling to promote anchorage-independent growth</span></p> 
  <p><span style="background-color:#ffb7b7">Microarray analyses identified up-regulation of the PHLDA1 gene product as being correlated with ERK-mediated cellular transformation</span>. <span style="background-color:#fff2f2">Likewise, the protein product of the PHLDA1 gene, TDAG51, displayed an identical expression pattern</span>. <span style="background-color:#fffcfc">Expression of PHLDA1 mRNA and TDAG51 protein in breast cancer has recently been described [ 17 ]</span>. <span style="background-color:#fffcfc">Using TDAG51 immunohistochemistry of tissue microarrays, 699 individual primary invasive breast tumor specimens were examined; loss of TDAG51 was found to correlate with a poorer disease-free and overall survival rate in multivariate analysis</span>. <span style="background-color:#fffbfb">Similar results have been found in a small series of melanomas, where TDAG51 immunoreactivity was found to decrease during the progression of melanocytic nevi to primary melanomas and finally to metastatic melanoma [ 18 ]</span>. <span style="background-color:#fff1f1">These studies suggest that TDAG51 has a suppressive effect on tumor progression and prompted us to evaluate the cell biological function of TDAG51 in the HME16C transformation model</span>.</p> 
  <p><span style="background-color:#fff1f1">The PHLDA1 gene is a member of the pleckstrin homology-related domain family that includes Ipl/Tssc and Tih [ 33 ]</span>. <span style="background-color:#ffe5e5">PHDLA1 has been described as an immediate early gene with transcriptional activation resulting from engagement of receptors such as the FGF and IGF tyrosine kinase receptors and the T-cell receptor [ 34 - 36 ]</span>. <span style="background-color:#fff7f7">Strong TDAG51 induction by RasV12S35 transformation of mammary epithelial cells suggests that ERK activation may at least partially explain TDAG induction in these previous reports</span>. <span style="background-color:#ffefef">In vitro studies have suggested a role for TDAG51 in the control of cellular proliferation and in the induction of apoptosis in response to a variety of stresses including proteotoxic cellular stresses such as lung cancer cell responses to chemotherapy[ 35 , 37 - 39 ]</span></p> 
  <p><span style="background-color:#fffcfc">Overexpression of TDAG51 has been shown to reduce cell proliferation and induce cell death in a variety of cell types including T cells, neuronal, endothelial, melanoma, and cervical carcinoma cell types [ 34 , 35 , 38 ]</span>. <span style="background-color:#ffbfbf">By contrast, TDAG51 functioned as an anti-apoptotic factor downstream of insulin-like growth factor receptor (IGFR) signaling as TDAG51 was required to protect NIH3T3 cells from apoptosis upon IGF-1 withdrawal [ 36 ]</span></p> 
  <p><span style="background-color:#fffcfc">Reduction of TDAG51 levels in RasV12 and RasV12S35 cells enhanced cell proliferation under anchorage-independent conditions</span>. <span style="background-color:#ffffff">This suggests that TDAG51 expression limits cellular proliferation</span>. <span style="background-color:#fffcfc">In addition to enhancing cellular proliferation, reduction of TDAG51 protein levels also increased the absolute amount of cell death in transformed RasV12S35 cells under these same conditions</span>. <span style="background-color:#fff0f0">Therefore, TDAG51 loss, in the context of oncogene activation, may indirectly promote cell death as a consequence of enhanced cell cycling</span>. <span style="background-color:#fffdfd">However, the overall increased cell numbers in anchorage-independent conditions showed that enhanced cellular proliferation exceeded enhanced cell death</span>.</p> 
  <p><span style="background-color:#fff4f4">Interestingly, a decrease in TDAG51 expression during Ras-mediated cellular transformation promoted the growth of cells under anchorage-independent conditions but did not affect the growth of attached cells</span>. <span style="background-color:#fffdfd">This suggests that TDAG51 might act in conjunction with both cellular stress, in this case matrix detachment, and a proliferative signal, in this case oncogenic activation</span>. <span style="background-color:#fffbfb">Other studies also have implicated TDAG51 in functioning during cellular stress and survival, particularly endoplasmic reticulum stress associated with the unfolded protein response [ 37 , 38 ]</span>. <span style="background-color:#ffffff">A mechanism of action for TDAG51 is not known</span>. <span style="background-color:#fffdfd">The preliminary finding that TDAG51 binds to proteins involved in protein translation has been used to suggest that TDAG51 may play a role in regulating protein synthesis in response to proteotoxic stress [ 40 ]</span></p> 
  <p><span style="background-color:#fffdfd">Reduction of TDAG51 expression resulted in an increase in Erk activation in cells grown under anchorage-independent conditions</span>. <span style="background-color:#ffffff">How TDAG51 expression might suppress ERK signaling is not known, but appears to represent a negative feedback pathway that directly or indirectly limits ERK activation</span>. <span style="background-color:#fff9f9">This is not likely to be due to an inhibition of ERK protein synthesis by TDAG51, since Erk2 levels were unaffected by reduced TDAG51</span>. <span style="background-color:#fff7f7">However, another important component of ERK activation is the dual specific ERK phosphatases, a highly regulated class of proteins, whose relative level of activity may be affected by TDAG51 protein levels</span>. <span style="background-color:#fff6f6">Understanding the mechanisms by which TDAG51 regulates ERK pathway activation and the balance between cellular proliferation and apoptosis of transformed cells represents a future challenge</span>. <span style="background-color:#ffebeb">Finally, TDAG51 acts in a suppressive manner during matrix-detached growth of HME16C cells</span>. <span style="background-color:#ffdbdb">Taken together with the identification of TDAG51 as a stress-induced gene in a variety of cell lines and a growth inhibitor in melanoma cell lines, it is reasonable to suggest that loss of TDAG51 may act to promote progression of breast cancer through an intrinsic growth regulatory mechanism</span>.</p> 
 </sec> 
 <sec> 
  <h3>Conclusion</h3> 
  <p><span style="background-color:#ffb9b9">Expression of activated Ras effector domain mutants that bind Raf, PI3K, or RalGEF are sufficient to induce the anchorage-independent growth of the human mammary epithelial cell line HME16C and are associated with up-regulation of EGFR ligands</span>. <span style="background-color:#fff0f0">However, only the ERK pathway is capable of supporting transformation in the absence of EGFR signaling and of supporting tumorigenesis in nude mice</span>. <span style="background-color:#ffd8d8">Up-regulation of TDAG51 occurs during Ras-mediated transformation in an ERK-dependent fashion, but opposes ERK-mediated transformation by suppressing ERK signaling and decreasing cellular proliferation under matrix-detached conditions</span>. <span style="background-color:#ffe7e7">Therefore, in this model of mammary epithelial cell transformation, TDAG51 acts as a growth inhibitor of ERK-driven proliferation and may help explain why loss of TDAG51 expression has been found to correlate with progression in human breast cancer and melanoma</span>.</p> 
 </sec> 
 <sec> 
  <h3>Abbreviations</h3> 
  <p><span style="background-color:#ffa5a5">EGFR: epidermal growth factor receptor; ErbB2: epidermal growth factor receptor family member B2; Erk: extracellular regulated kinase; PHDLA1: pleckstrin homology domain-like, family A, member 1; PI3K: phsphatidyl inositol 3-kinase; PIK3CA: phosphatidyl inositol 3-kinase, catalytic, alpha polypeptide; PTEN: phosphatase and tensin homologue deleted from chromosome 10; Ral: ras-like gene; RalGEF: Ral guanidine nucleotide exchange factor; Rlf: RalGEF-like factor; shRNA: small hairpin ribonucleic acid; TDAG51: T-cell death associated gene of 51 kDa</span>.</p> 
 </sec> 
 <sec> 
  <h3>Competing interests</h3> 
  <p><span style="background-color:#fffefe">The authors declare that they have no competing interests</span>. <span style="background-color:#fff7f7">The final text was reviewed and approved by all the listed co-authors</span>. <span style="background-color:#fffbfb">They all contributed to the intellectual content and to the experimental work in an honest manner</span>. <span style="background-color:#fff8f8">The contents have not been published by or submitted to any other journal</span>. <span style="background-color:#fffdfd">No part of this text has been included in any other paper</span>.</p> 
 </sec> 
 <sec> 
  <h3>Authors' contributions</h3> 
  <p><span style="background-color:#fffefe">Conception and design: MDO and KK</span>. <span style="background-color:#fffefe">Provision of study materials: MDO, JJY, YW, and KK</span>. <span style="background-color:#fffefe">Collection and assembly of data: MDO, SJB, and LZ</span>. <span style="background-color:#ffefef">Data analysis and interpretation: MDO and KK</span>. <span style="background-color:#fffefe">Manuscript writing: MDO and KK</span>. <span style="background-color:#fffefe">Final approval of manuscript: MDO, SJB, LZ, JJY, YW, and KK</span>.</p> 
 </sec> 
 <sec> 
  <h3>Pre-publication history</h3> 
  <p>The pre-publication history for this paper can be accessed here:</p> 
  <p> 
   <ext-link ext-link-type="uri" xlink:href="http://www.biomedcentral.com/1471-2407/8/189/prepub"></ext-link> </p> 
 </sec> 
 <sec sec-type="supplementary-material"> 
  <h3>Supplementary Material</h3> 
  <supplementary-material content-type="local-data" id="S1"> 
   <h3>Additional file 1</h3> 
   <p>
    <bold>
      Selected microarray data for HME16C cell lines 
    </bold>. <span style="background-color:#ff9595">The fold up-regulation or down-regulation of genes identified in microarray analysis is indicated for genes where this value exceeded two-fold, organized into categories according to broadly defined gene functions</span>. PHLDA1 is categorized as an up-regulated gene under "Miscellaneous."</p> 
   <media mime-subtype="msword" mimetype="application" xlink:href="1471-2407-8-189-S1.doc"> 
    <p>Click here for file</p> 
   </media> 
  </supplementary-material> 
  <supplementary-material content-type="local-data" id="S2"> 
   <h3>Additional file 2</h3> 
   <p>
    <bold>
      Real-time RT-PCR confirmation of selected microarray data 
    </bold>. <span style="background-color:#ffa0a0">Results are for transcripts identified as up- or down-regulated in microarray analysis of cDNA from RasV12-, RasV12G37-, RasV12S35-, RasV12C40-, and Rlf-CAAX-infected HME16C</span>. <span style="background-color:#ffe2e2">Numbers represent fold expression changes for transcript levels relative to vector-infected pLRT control cells</span>. <span style="background-color:#ffa4a4">Parenthetical numbers indicate mean fold expression changes from cDNA microarray analysis (see Additional file above)</span>. <span style="background-color:#ff1717">Sequences of primers used for real-time RT-PCR analysis are as follows: amphiregulin Forward 5'-CGCCGGTGGTGCTGTCGCTCTT-3' Reverse 5'-TCACTCACAGGGGAAATCTCACTC-3', anterior gradient 2 Forward 5'-GGGGTGACCAACTCATCTGGACTCAG-3' Reverse 5'-GACATACTGGCCATCAGGAGAAAGGTGT-3', CD24 Forward 5'-GCTCCTACCCACGCAGATTTATT-3' Reverse 5'-CACGAAGAGACTGGCTGTTGACT-3', c-met Forward 5'-AAATGGCCACGGGACAACACAA-3' Reverse 5'-TGGGCTGGGGTATAACATTCAAGA-3', CTGF Forward 5'-CTGCCCGGGAAATGCTGCGAGGAGT-3' Reverse 5'-CTGCAGGAGGCGTTGTCATTGGTAA-3', CYR61 Forward 5'-CGGCCCAAGTACTGCGGTTCCT-3' Reverse 5'-ATTGGCATGCGGGCAGTTGTAGTT-3', Decorin Forward 5'-ACTTCTGCCCACCTGGACACAACA-3' Reverse 5'-ATGGCAGAGCGCACGTAGACACA-3', DICER-1 Forward 5'-CAGGAAATACCCGTGCAACCAACTA-3' Reverse 5'-GCATTACGGCCATCACAGGACTTC-3', E-cadherin Forward 5'-GGTATCTTCCCCGCCCTGCCAATCC-3' Reverse 5'-AACCGCTTCCTTCATAGTCAAACACGAG-3', EphA2 Forward 5'-CCCCTTCCGCCCCACACTACCTCACAGC-3' Reverse 5'-ACACGGCCCGCATTCCCCAGACTCG-3', Epiregulin Forward 5'-TTGTATTTTTAGTAGAGGCGGGGTTTCA-3' Reverse 5'-TCGGGCACAGATGTTCAAGTCAC-3', ETS-1 Forward 5'-ACTCGGGGGCCAGGACTCTTTTGAA-3' Reverse 5'-CACGGTCCCGCACATAGTCCTTGAA-3', ETS variant gene 5 Forward 5'-TCGGGGACGTCTACGGTTTCTACT-3' Reverse 5'-AAGACTGTAAACGGCTACCATTGA-3', FAT-2 Forward 5'-GCTGGACATCAAACGGGCTAACAT-3' Reverse 5'-ACCGCATCTGAACCCCCACTGAAT-3', HMGA-1 Forward 5'-GCTCACCCTGCCCGCTCCCAACC-3' Reverse 5'-GCCCCAGCCCCTCTTCCCCACAAA-3', HMGA-2 Forward 5'-CTGATAAGCAAGAGTGGGCGGGTGAGAA-3' Reverse 5'-ACAGGGAGTGGGTTGGGGTGGTATTTGA-3', Secreted frizzled-related protein 1 Forward 5'-CCCGCTCCCTTTCCCTCCATA-3' Reverse 5'-GCTCTCACTTTCCGCCCAATCC-3', Tenascin C Forward 5'-GCGGCCCAGAGCGAGGAA-3' Reverse 5'-TATTGCGATGTTGTCACTGGGAGATTTT-3', TGF-a Forward 5'-CCGCCTTGGTGGTGGTCTCC-3' Reverse 5'-AGGGCGCTGGGCTTCTCGTG-3', TGF-?2 Forward 5'-CCCGCCCACTTTCTACAGACCCTACT-3' Reverse 5'-CATTCGCCTTCTGCTCTTGTTTTCAC-3', TGF-? receptor 2 Forward 5'-ACAGTGGCAGTCAAGATCTTTCCCTATG-3' Reverse 5'-CAACTCCGTCTTCCGCTCCTCAG-3', TRIM29 Forward 5'-ATGCATGCGCCACGTTGAGAAGAT-3' Reverse 5'-GGTGAAGCGGCCAGGAGACGAG-3'</span>.</p> 
   <media mime-subtype="msword" mimetype="application" xlink:href="1471-2407-8-189-S2.doc"> 
    <p>Click here for file</p> 
   </media> 
  </supplementary-material> 
 </sec> 
 <back> 
  <ack> 
   <sec> 
    <h3>Acknowledgements</h3> 
    <p>This research was supported by the Intramural Research Program of the NIH, National Cancer Institute, Center for Cancer Research.</p> 
   </sec> 
  </ack> 
  <ref-list> 
   <ref id="B1"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         van 't Veer 
       </surname> 
       <given-names>
         LJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dai 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vijver 
       </surname> 
       <given-names>
         MJ van de 
       </given-names> 
      </name> 
      <name> 
       <surname>
         He 
       </surname> 
       <given-names>
         YD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hart 
       </surname> 
       <given-names>
         AA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mao 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Peterse 
       </surname> 
       <given-names>
         HL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kooy 
       </surname> 
       <given-names>
         K van der 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Marton 
       </surname> 
       <given-names>
         MJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Witteveen 
       </surname> 
       <given-names>
         AT 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       Gene expression profiling predicts clinical outcome of breast cancer 
     </article-h3> 
     <source />Nature 
     <year>
       2002 
     </year> 
     <volume>
       415 
     </volume> 
     <fpage>
       530 
     </fpage> 
     <lpage>
       536 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11823860 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/415530a 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B2"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Bos 
       </surname> 
       <given-names>
         JL 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       ras oncogenes in human cancer: a review 
     </article-h3> 
     <source />Cancer Res 
     <year>
       1989 
     </year> 
     <volume>
       49 
     </volume> 
     <fpage>
       4682 
     </fpage> 
     <lpage>
       4689 
     </lpage> 
     <pub-id pub-id-type="pmid">
       2547513 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B3"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Malaney 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Daly 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The ras signaling pathway in mammary tumorigenesis and metastasis 
     </article-h3> 
     <source />J Mammary Gland Biol Neoplasia 
     <year>
       2001 
     </year> 
     <volume>
       6 
     </volume> 
     <fpage>
       101 
     </fpage> 
     <lpage>
       113 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11467446 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1023/A:1009572700317 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B4"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Mueller 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Flury 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Eppenberger-Castori 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kueng 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         David 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Eppenberger 
       </surname> 
       <given-names>
         U 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Potential prognostic value of mitogen-activated protein kinase activity for disease-free survival of primary breast cancer patients 
     </article-h3> 
     <source />Int J Cancer 
     <year>
       2000 
     </year> 
     <volume>
       89 
     </volume> 
     <fpage>
       384 
     </fpage> 
     <lpage>
       388 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10956414 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1002/1097-0215(20000720)89:4&lt;384::AID-IJC11&gt;3.0.CO;2-R 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B5"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Abd El-Rehim 
       </surname> 
       <given-names>
         DM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pinder 
       </surname> 
       <given-names>
         SE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Paish 
       </surname> 
       <given-names>
         CE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bell 
       </surname> 
       <given-names>
         JA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rampaul 
       </surname> 
       <given-names>
         RS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Blamey 
       </surname> 
       <given-names>
         RW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Robertson 
       </surname> 
       <given-names>
         JF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nicholson 
       </surname> 
       <given-names>
         RI 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ellis 
       </surname> 
       <given-names>
         IO 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Expression and co-expression of the members of the epidermal growth factor receptor (EGFR) family in invasive breast carcinoma 
     </article-h3> 
     <source />Br J Cancer 
     <year>
       2004 
     </year> 
     <volume>
       91 
     </volume> 
     <fpage>
       1532 
     </fpage> 
     <lpage>
       1542 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15480434 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/sj.bjc.6602184 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B6"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Sapi 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The role of CSF-1 in normal physiology of mammary gland and breast cancer: an update 
     </article-h3> 
     <source />Exp Biol Med (Maywood) 
     <year>
       2004 
     </year> 
     <volume>
       229 
     </volume> 
     <fpage>
       1 
     </fpage> 
     <lpage>
       11 
     </lpage> 
     <pub-id pub-id-type="pmid">
       14709771 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B7"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Slamon 
       </surname> 
       <given-names>
         DJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Clark 
       </surname> 
       <given-names>
         GM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wong 
       </surname> 
       <given-names>
         SG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Levin 
       </surname> 
       <given-names>
         WJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ullrich 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McGuire 
       </surname> 
       <given-names>
         WL 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Human breast cancer: correlation of relapse and survival with amplification of the HER-2/neu oncogene 
     </article-h3> 
     <source />Science 
     <year>
       1987 
     </year> 
     <volume>
       235 
     </volume> 
     <fpage>
       177 
     </fpage> 
     <lpage>
       182 
     </lpage> 
     <pub-id pub-id-type="pmid">
       3798106 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1126/science.3798106 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B8"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Normanno 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bianco 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         De Luca 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Salomon 
       </surname> 
       <given-names>
         DS 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The role of EGF-related peptides in tumor growth 
     </article-h3> 
     <source />Front Biosci 
     <year>
       2001 
     </year> 
     <volume>
       6 
     </volume> 
     <fpage>
       D685 
     </fpage> 
     <lpage>
       707 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11333208 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.2741/Normano 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B9"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Bachman 
       </surname> 
       <given-names>
         KE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Argani 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Samuels 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Silliman 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ptak 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Szabo 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Konishi 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Karakas 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Blair 
       </surname> 
       <given-names>
         BG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lin 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       The PIK3CA gene is mutated with high frequency in human breast cancers 
     </article-h3> 
     <source />Cancer Biol Ther 
     <year>
       2004 
     </year> 
     <volume>
       3 
     </volume> 
     <fpage>
       772 
     </fpage> 
     <lpage>
       775 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15254419 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B10"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Perez-Tenorio 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Alkhori 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Olsson 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Waltersson 
       </surname> 
       <given-names>
         MA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nordenskjold 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rutqvist 
       </surname> 
       <given-names>
         LE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Skoog 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stal 
       </surname> 
       <given-names>
         O 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       PIK3CA mutations and PTEN loss correlate with similar prognostic factors and are not mutually exclusive in breast cancer 
     </article-h3> 
     <source />Clin Cancer Res 
     <year>
       2007 
     </year> 
     <volume>
       13 
     </volume> 
     <fpage>
       3577 
     </fpage> 
     <lpage>
       3584 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17575221 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1158/1078-0432.CCR-06-1609 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B11"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Rodriguez-Viciana 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sabatier 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McCormick 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Signaling specificity by Ras family GTPases is determined by the full spectrum of effectors they regulate 
     </article-h3> 
     <source />Mol Cell Biol 
     <year>
       2004 
     </year> 
     <volume>
       24 
     </volume> 
     <fpage>
       4943 
     </fpage> 
     <lpage>
       4954 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15143186 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1128/MCB.24.11.4943-4954.2004 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B12"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         White 
       </surname> 
       <given-names>
         MA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nicolette 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Minden 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Polverino 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Van Aelst 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Karin 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wigler 
       </surname> 
       <given-names>
         MH 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Multiple Ras functions can contribute to mammalian cell transformation 
     </article-h3> 
     <source />Cell 
     <year>
       1995 
     </year> 
     <volume>
       80 
     </volume> 
     <fpage>
       533 
     </fpage> 
     <lpage>
       541 
     </lpage> 
     <pub-id pub-id-type="pmid">
       7867061 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/0092-8674(95)90507-3 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B13"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Janda 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lehmann 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Killisch 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jechlinger 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Herzig 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Downward 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Beug 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Grunert 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Ras and TGF[beta] cooperatively regulate epithelial cell plasticity and metastasis: dissection of Ras signaling pathways 
     </article-h3> 
     <source />J Cell Biol 
     <year>
       2002 
     </year> 
     <volume>
       156 
     </volume> 
     <fpage>
       299 
     </fpage> 
     <lpage>
       313 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11790801 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1083/jcb.200109037 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B14"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Rangarajan 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hong 
       </surname> 
       <given-names>
         SJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gifford 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Weinberg 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Species- and cell type-specific requirements for cellular transformation 
     </article-h3> 
     <source />Cancer Cell 
     <year>
       2004 
     </year> 
     <volume>
       6 
     </volume> 
     <fpage>
       171 
     </fpage> 
     <lpage>
       183 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15324700 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/j.ccr.2004.07.009 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B15"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Zhao 
       </surname> 
       <given-names>
         JJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gjoerup 
       </surname> 
       <given-names>
         OV 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Subramanian 
       </surname> 
       <given-names>
         RR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cheng 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chen 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Roberts 
       </surname> 
       <given-names>
         TM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hahn 
       </surname> 
       <given-names>
         WC 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Human mammary epithelial cell transformation through the activation of phosphatidylinositol 3-kinase 
     </article-h3> 
     <source />Cancer Cell 
     <year>
       2003 
     </year> 
     <volume>
       3 
     </volume> 
     <fpage>
       483 
     </fpage> 
     <lpage>
       495 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12781366 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/S1535-6108(03)00088-6 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B16"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Schulze 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lehmann 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jefferies 
       </surname> 
       <given-names>
         HB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McMahon 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Downward 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Analysis of the transcriptional program induced by Raf in epithelial cells 
     </article-h3> 
     <source />Genes Dev 
     <year>
       2001 
     </year> 
     <volume>
       15 
     </volume> 
     <fpage>
       981 
     </fpage> 
     <lpage>
       994 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11316792 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1101/gad.191101 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B17"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Nagai 
       </surname> 
       <given-names>
         MA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fregnani 
       </surname> 
       <given-names>
         JH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Netto 
       </surname> 
       <given-names>
         MM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Brentani 
       </surname> 
       <given-names>
         MM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Soares 
       </surname> 
       <given-names>
         FA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Down-regulation of PHLDA1 gene expression is associated with breast cancer progression 
     </article-h3> 
     <source />Breast Cancer Res Treat 
     <year>
       2007 
     </year> 
     <volume>
       106 
     </volume> 
     <fpage>
       49 
     </fpage> 
     <lpage>
       56 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17211533 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1007/s10549-006-9475-6 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B18"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Neef 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kuske 
       </surname> 
       <given-names>
         MA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Prols 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Johnson 
       </surname> 
       <given-names>
         JP 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Identification of the human PHLDA1/TDAG51 gene: down-regulation in metastatic melanoma contributes to apoptosis resistance and growth deregulation 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2002 
     </year> 
     <volume>
       62 
     </volume> 
     <fpage>
       5920 
     </fpage> 
     <lpage>
       5929 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12384558 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B19"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Haegen 
       </surname> 
       <given-names>
         BA Van der 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shay 
       </surname> 
       <given-names>
         JW 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Immortalization of human mammary epithelial cells by SV40 large T-antigen involves a two step mechanism 
     </article-h3> 
     <source />In Vitro Cell Dev Biol 
     <year>
       1993 
     </year> 
     <volume>
       29A 
     </volume> 
     <fpage>
       180 
     </fpage> 
     <lpage>
       182 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8463179 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1007/BF02634177 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B20"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Hahn 
       </surname> 
       <given-names>
         WC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Counter 
       </surname> 
       <given-names>
         CM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lundberg 
       </surname> 
       <given-names>
         AS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Beijersbergen 
       </surname> 
       <given-names>
         RL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Brooks 
       </surname> 
       <given-names>
         MW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Weinberg 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Creation of human tumour cells with defined genetic elements 
     </article-h3> 
     <source />Nature 
     <year>
       1999 
     </year> 
     <volume>
       400 
     </volume> 
     <fpage>
       464 
     </fpage> 
     <lpage>
       468 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10440377 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/22780 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B21"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Wolthuis 
       </surname> 
       <given-names>
         RM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         de Ruiter 
       </surname> 
       <given-names>
         ND 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cool 
       </surname> 
       <given-names>
         RH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bos 
       </surname> 
       <given-names>
         JL 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Stimulation of gene induction and cell growth by the Ras effector Rlf 
     </article-h3> 
     <source />Embo J 
     <year>
       1997 
     </year> 
     <volume>
       16 
     </volume> 
     <fpage>
       6748 
     </fpage> 
     <lpage>
       6761 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9362489 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1093/emboj/16.22.6748 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B22"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Leaner 
       </surname> 
       <given-names>
         VD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kinoshita 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Birrer 
       </surname> 
       <given-names>
         MJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       AP-1 complexes containing cJun and JunB cause cellular transformation of Rat1a fibroblasts and share transcriptional targets 
     </article-h3> 
     <source />Oncogene 
     <year>
       2003 
     </year> 
     <volume>
       22 
     </volume> 
     <fpage>
       5619 
     </fpage> 
     <lpage>
       5629 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12944910 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/sj.onc.1206644 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B23"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Yin 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pollock 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tracy 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chock 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Martin 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Oberst 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kelly 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Activation of the RalGEF/Ral pathway promotes prostate cancer metastasis to bone 
     </article-h3> 
     <source />Mol Cell Biol 
     <year>
       2007 
     </year> 
     <volume>
       27 
     </volume> 
     <fpage>
       7538 
     </fpage> 
     <lpage>
       7550 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17709381 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1128/MCB.00955-07 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B24"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Zufferey 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dull 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mandel 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bukovsky 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Quiroz 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Naldini 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Trono 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Self-inactivating lentivirus vector for safe and efficient in vivo gene delivery 
     </article-h3> 
     <source />J Virol 
     <year>
       1998 
     </year> 
     <volume>
       72 
     </volume> 
     <fpage>
       9873 
     </fpage> 
     <lpage>
       9880 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9811723 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B25"> 
    <citation citation-type="other"> 
     <ext-link ext-link-type="uri" xlink:href="http://nciarray.nci.nih.gov/"></ext-link> 
    </citation> 
   </ref> 
   <ref id="B26"> 
    <citation citation-type="other"> 
     <ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/geo"></ext-link> 
    </citation> 
   </ref> 
   <ref id="B27"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Hamad 
       </surname> 
       <given-names>
         NM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Elconin 
       </surname> 
       <given-names>
         JH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Karnoub 
       </surname> 
       <given-names>
         AE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bai 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rich 
       </surname> 
       <given-names>
         JN 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Abraham 
       </surname> 
       <given-names>
         RT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Der 
       </surname> 
       <given-names>
         CJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Counter 
       </surname> 
       <given-names>
         CM 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Distinct requirements for Ras oncogenesis in human versus mouse cells 
     </article-h3> 
     <source />Genes Dev 
     <year>
       2002 
     </year> 
     <volume>
       16 
     </volume> 
     <fpage>
       2045 
     </fpage> 
     <lpage>
       2057 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12183360 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1101/gad.993902 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B28"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Fry 
       </surname> 
       <given-names>
         DW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kraker 
       </surname> 
       <given-names>
         AJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McMichael 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ambroso 
       </surname> 
       <given-names>
         LA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nelson 
       </surname> 
       <given-names>
         JM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Leopold 
       </surname> 
       <given-names>
         WR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Connors 
       </surname> 
       <given-names>
         RW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bridges 
       </surname> 
       <given-names>
         AJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       A specific inhibitor of the epidermal growth factor receptor tyrosine kinase 
     </article-h3> 
     <source />Science 
     <year>
       1994 
     </year> 
     <volume>
       265 
     </volume> 
     <fpage>
       1093 
     </fpage> 
     <lpage>
       1095 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8066447 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1126/science.8066447 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B29"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Fry 
       </surname> 
       <given-names>
         DW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bridges 
       </surname> 
       <given-names>
         AJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Denny 
       </surname> 
       <given-names>
         WA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Doherty 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Greis 
       </surname> 
       <given-names>
         KD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hicks 
       </surname> 
       <given-names>
         JL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hook 
       </surname> 
       <given-names>
         KE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Keller 
       </surname> 
       <given-names>
         PR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Leopold 
       </surname> 
       <given-names>
         WR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Loo 
       </surname> 
       <given-names>
         JA 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       Specific, irreversible inactivation of the epidermal growth factor receptor and erbB2, by a new class of tyrosine kinase inhibitor 
     </article-h3> 
     <source />Proc Natl Acad Sci USA 
     <year>
       1998 
     </year> 
     <volume>
       95 
     </volume> 
     <fpage>
       12022 
     </fpage> 
     <lpage>
       12027 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9751783 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1073/pnas.95.20.12022 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B30"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Schulze 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nicke 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Warne 
       </surname> 
       <given-names>
         PH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tomlinson 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Downward 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The transcriptional response to Raf activation is almost completely dependent on Mitogen-activated Protein Kinase Kinase activity and shows a major autocrine component 
     </article-h3> 
     <source />Mol Biol Cell 
     <year>
       2004 
     </year> 
     <volume>
       15 
     </volume> 
     <fpage>
       3450 
     </fpage> 
     <lpage>
       3463 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15090615 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1091/mbc.E03-11-0807 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B31"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Davies 
       </surname> 
       <given-names>
         SP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Reddy 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Caivano 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cohen 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Specificity and mechanism of action of some commonly used protein kinase inhibitors 
     </article-h3> 
     <source />Biochem J 
     <year>
       2000 
     </year> 
     <volume>
       351 
     </volume> 
     <fpage>
       95 
     </fpage> 
     <lpage>
       105 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10998351 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1042/0264-6021:3510095 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B32"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ince 
       </surname> 
       <given-names>
         TA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Richardson 
       </surname> 
       <given-names>
         AL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bell 
       </surname> 
       <given-names>
         GW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Saitoh 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Godar 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Karnoub 
       </surname> 
       <given-names>
         AE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Iglehart 
       </surname> 
       <given-names>
         JD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Weinberg 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Transformation of different human breast epithelial cell types leads to distinct tumor phenotypes 
     </article-h3> 
     <source />Cancer Cell 
     <year>
       2007 
     </year> 
     <volume>
       12 
     </volume> 
     <fpage>
       160 
     </fpage> 
     <lpage>
       170 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17692807 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/j.ccr.2007.06.013 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B33"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Frank 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mendelsohn 
       </surname> 
       <given-names>
         CL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ciccone 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Svensson 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ohlsson 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tycko 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       A novel pleckstrin homology-related gene family defined by Ipl/Tssc3, TDAG51, and Tih1: tissue-specific expression, chromosomal location, and parental imprinting 
     </article-h3> 
     <source />Mamm Genome 
     <year>
       1999 
     </year> 
     <volume>
       10 
     </volume> 
     <fpage>
       1150 
     </fpage> 
     <lpage>
       1159 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10594239 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1007/s003359901182 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B34"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Gomes 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Xiong 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Miki 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rosner 
       </surname> 
       <given-names>
         MR 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       A proline- and glutamine-rich protein promotes apoptosis in neuronal cells 
     </article-h3> 
     <source />J Neurochem 
     <year>
       1999 
     </year> 
     <volume>
       73 
     </volume> 
     <fpage>
       612 
     </fpage> 
     <lpage>
       622 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10428057 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1046/j.1471-4159.1999.0730612.x 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B35"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Park 
       </surname> 
       <given-names>
         CG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         SY 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kandala 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         SY 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Choi 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       A novel gene product that couples TCR signaling to Fas(CD95) expression in activation-induced cell death 
     </article-h3> 
     <source />Immunity 
     <year>
       1996 
     </year> 
     <volume>
       4 
     </volume> 
     <fpage>
       583 
     </fpage> 
     <lpage>
       591 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8673705 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/S1074-7613(00)80484-7 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B36"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Toyoshima 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Karas 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yakar 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dupont 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         LeRoith 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       TDAG51 mediates the effects of insulin-like growth factor I (IGF-I) on cell survival 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       2004 
     </year> 
     <volume>
       279 
     </volume> 
     <fpage>
       25898 
     </fpage> 
     <lpage>
       25904 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15037619 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.M400661200 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B37"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Hayashida 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Inouye 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fujimoto 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tanaka 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Izu 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Takaki 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ichikawa 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rho 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nakai 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       A novel HSF1-mediated death pathway that is suppressed by heat shock proteins 
     </article-h3> 
     <source />Embo J 
     <year>
       2006 
     </year> 
     <volume>
       25 
     </volume> 
     <fpage>
       4773 
     </fpage> 
     <lpage>
       4783 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17024176 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/sj.emboj.7601370 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B38"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Hossain 
       </surname> 
       <given-names>
         GS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         van Thienen 
       </surname> 
       <given-names>
         JV 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Werstuck 
       </surname> 
       <given-names>
         GH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zhou 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sood 
       </surname> 
       <given-names>
         SK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dickhout 
       </surname> 
       <given-names>
         JG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         de Koning 
       </surname> 
       <given-names>
         AB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tang 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wu 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Falk 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       TDAG51 is induced by homocysteine, promotes detachment-mediated programmed cell death, and contributes to the cevelopment of atherosclerosis in hyperhomocysteinemia 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       2003 
     </year> 
     <volume>
       278 
     </volume> 
     <fpage>
       30317 
     </fpage> 
     <lpage>
       30327 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12738777 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.M212897200 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B39"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Joo 
       </surname> 
       <given-names>
         JH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liao 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Collins 
       </surname> 
       <given-names>
         JB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Grissom 
       </surname> 
       <given-names>
         SF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jetten 
       </surname> 
       <given-names>
         AM 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Farnesol-induced apoptosis in human lung carcinoma cells is coupled to the endoplasmic reticulum stress response 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2007 
     </year> 
     <volume>
       67 
     </volume> 
     <fpage>
       7929 
     </fpage> 
     <lpage>
       7936 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17699800 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1158/0008-5472.CAN-07-0931 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B40"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Hinz 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Flindt 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Marx 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Janssen 
       </surname> 
       <given-names>
         O 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kabelitz 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
      Inhibition of protein synthesis by the T cell receptor-inducible human T
      <span style="background-color:#ffffff">D</span>AG51 gene product
     </article-h3> 
     <source />Cell Signal 
     <year>
       2001 
     </year> 
     <volume>
       13 
     </volume> 
     <fpage>
       345 
     </fpage> 
     <lpage>
       352 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11369516 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/S0898-6568(01)00141-3 
     </pub-id> 
    </citation> 
   </ref> 
  </ref-list> 
 </back> 
</article>