<article article-type="other"> 
 <!--?properties open_acces=""?--> 
 <!-- Original-type: cy--> 
 <front> 
  <journal-meta> 
   <journal-id journal-id-type="nlm-ta">
     Br J Cancer 
   </journal-id> 
   <journal-h3>
     British Journal of Cancer 
   </journal-h3> 
   <issn pub-type="ppub">
     0007-0920 
   </issn> 
   <issn pub-type="epub">
     1532-1827 
   </issn> 
   <publisher> 
    <publisher-name>
      Nature Publishing Group 
    </publisher-name> 
   </publisher> 
  </journal-meta> 
  <article-meta> 
   <article-id pub-id-type="pmid">
     18594528 
   </article-id> 
   <article-id pub-id-type="pmc">
     2480968 
   </article-id> 
   <article-id pub-id-type="pii">
     6604469 
   </article-id> 
   <article-id pub-id-type="doi">
     10.1038/sj.bjc.6604469 
   </article-id> 
   <article-categories> 
    <subj-group subj-group-type="heading"> 
     <subject>
       Clinical Studies 
     </subject> 
    </subj-group> 
   </article-categories> 
   <h3-group> 
    <article-h3>
      Mutations in the LKB1 tumour suppressor are frequently detected in tumours from Caucasian but not Asian lung cancer patients 
    </article-h3> 
   </h3-group> 
   <contrib-group> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Koivunen 
      </surname> 
      <given-names>
        J P 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1">
       1 
     </xref> 
     <xref ref-type="aff" rid="aff2">
       2 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Kim 
      </surname> 
      <given-names>
        J 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff3">
       3 
     </xref> 
     <xref ref-type="aff" rid="aff4">
       4 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Lee 
      </surname> 
      <given-names>
        J 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff3">
       3 
     </xref> 
     <xref ref-type="aff" rid="aff4">
       4 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Rogers 
      </surname> 
      <given-names>
        A M 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff2">
       2 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Park 
      </surname> 
      <given-names>
        J O 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1">
       1 
     </xref> 
     <xref ref-type="aff" rid="aff2">
       2 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Zhao 
      </surname> 
      <given-names>
        X 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff2">
       2 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Naoki 
      </surname> 
      <given-names>
        K 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff2">
       2 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Okamoto 
      </surname> 
      <given-names>
        I 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff5">
       5 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Nakagawa 
      </surname> 
      <given-names>
        K 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff5">
       5 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Yeap 
      </surname> 
      <given-names>
        B Y 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff6">
       6 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Meyerson 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1">
       1 
     </xref> 
     <xref ref-type="aff" rid="aff2">
       2 
     </xref> 
     <xref ref-type="aff" rid="aff7">
       7 
     </xref> 
     <xref ref-type="aff" rid="aff8">
       8 
     </xref> 
     <xref ref-type="aff" rid="aff9">
       9 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Wong 
      </surname> 
      <given-names>
        K-K 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1">
       1 
     </xref> 
     <xref ref-type="aff" rid="aff2">
       2 
     </xref> 
     <xref ref-type="aff" rid="aff9">
       9 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Richards 
      </surname> 
      <given-names>
        W G 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff10">
       10 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Sugarbaker 
      </surname> 
      <given-names>
        D J 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff10">
       10 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Johnson 
      </surname> 
      <given-names>
        B E 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1">
       1 
     </xref> 
     <xref ref-type="aff" rid="aff2">
       2 
     </xref> 
     <xref ref-type="aff" rid="aff9">
       9 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        J?nne 
      </surname> 
      <given-names>
        P A 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1">
       1 
     </xref> 
     <xref ref-type="aff" rid="aff2">
       2 
     </xref> 
     <xref ref-type="aff" rid="aff9">
       9 
     </xref> 
     <xref ref-type="corresp" rid="caf1">
       * 
     </xref> 
    </contrib> 
   </contrib-group> 
   <aff id="aff1"> 
    <label>1</label> 
    <institution>
      Lowe Center for Thoracic Oncology, Dana-Farber Cancer Institute 
    </institution> Boston, MA, 
    <country>
      USA 
    </country> 
   </aff> 
   <aff id="aff2"> 
    <label>2</label> 
    <institution>
      Department of Medical Oncology, Dana-Farber Cancer Institute, Harvard Medical School 
    </institution> Boston, MA, 
    <country>
      USA 
    </country> 
   </aff> 
   <aff id="aff3"> 
    <label>3</label> 
    <institution>
      Department of Thoracic Surgery, Samsung Medical Center 
    </institution> Seoul, 
    <country>
      Korea 
    </country> 
   </aff> 
   <aff id="aff4"> 
    <label>4</label> 
    <institution>
      School of Medicine, Sungkyunkwan University 
    </institution> Seoul, 
    <country>
      Korea 
    </country> 
   </aff> 
   <aff id="aff5"> 
    <label>5</label> 
    <institution>
      Department of Medical Oncology, School of Medicine, Kinki University 
    </institution> Osaka, 
    <country>
      Japan 
    </country> 
   </aff> 
   <aff id="aff6"> 
    <label>6</label> 
    <institution>
      Department of Medicine, Massachusetts General Hospital 
    </institution> Boston, MA, 
    <country>
      USA 
    </country> 
   </aff> 
   <aff id="aff7"> 
    <label>7</label> 
    <institution>
      Department of Pathology, Harvard Medical School 
    </institution> Boston, MA, 
    <country>
      USA 
    </country> 
   </aff> 
   <aff id="aff8"> 
    <label>8</label> 
    <institution>
      The Broad Institute of MIT and Harvard Universities 
    </institution> Cambridge, MA, 
    <country>
      USA 
    </country> 
   </aff> 
   <aff id="aff9"> 
    <label>9</label> 
    <institution>
      Department of Medicine, Brigham and Women's Hospital and Harvard Medical School 
    </institution> Boston, MA, 
    <country>
      USA 
    </country> 
   </aff> 
   <aff id="aff10"> 
    <label>10</label> 
    <institution>
      Department of Surgery, Brigham and Women's Hospital 
    </institution> Boston, MA, 
    <country>
      USA 
    </country> 
   </aff> 
   <author-notes> 
    <corresp id="caf1"> 
     <label>*</label>Author for correspondence: 
     <email xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="mailto:pjanne@partners.org">
       pjanne@partners.org 
     </email> 
    </corresp> 
   </author-notes> 
   <pub-date pub-type="epub"> 
    <day>
      01 
    </day> 
    <month>
      07 
    </month> 
    <year>
      2008 
    </year> 
   </pub-date> 
   <pub-date pub-type="collection"> 
    <day>
      15 
    </day> 
    <month>
      07 
    </month> 
    <year>
      2008 
    </year> 
   </pub-date> 
   <pub-date pub-type="ppub"> 
    <day>
      22 
    </day> 
    <month>
      7 
    </month> 
    <year>
      2008 
    </year> 
   </pub-date> 
   <volume>
     99 
   </volume> 
   <issue>
     2 
   </issue> 
   <fpage>
     245 
   </fpage> 
   <lpage>
     252 
   </lpage> 
   <history> 
    <date date-type="rev-recd"> 
     <day>
       09 
     </day> 
     <month>
       05 
     </month> 
     <year>
       2008 
     </year> 
    </date> 
    <date date-type="accepted"> 
     <day>
       28 
     </day> 
     <month>
       05 
     </month> 
     <year>
       2008 
     </year> 
    </date> 
   </history> 
   <copyright-statement>
     Copyright 2008, Cancer Research UK 
   </copyright-statement> 
   <copyright-year>
     2008 
   </copyright-year> 
   <permissions> 
    <copyright-holder>
      Cancer Research UK 
    </copyright-holder> 
   </permissions> 
   <abstract> 
    <h3>Abstract</h3> 
    <p>Somatic mutations of 
     <italic>
       LKB1 
     </italic> tumour suppressor gene have been detected in human cancers including non-small cell lung cancer (NSCLC). The relationship between 
     <italic>
       LKB1 
     </italic> mutations and clinicopathological characteristics and other common oncogene mutations in NSCLC is inadequately described. In this study we evaluated tumour specimens from 310 patients with NSCLC including those with adenocarcinoma, adenosquamous carcinoma, and squamous cell carcinoma histologies. Tumours were obtained from patients of US ( 
     <italic>
       n 
     </italic>=143) and Korean ( 
     <italic>
       n 
     </italic>=167) origin and screened for 
     <italic>
       LKB1 
     </italic>, 
     <italic>
       KRAS 
     </italic>, 
     <italic>
       BRAF 
     </italic>, and 
     <italic>
       EGFR 
     </italic> mutations using RT?PCR-based SURVEYOR-WAVE method followed by Sanger sequencing. We detected mutations in the 
     <italic>
       LKB1 
     </italic> gene in 34 tumours (11%). 
     <italic>
       LKB1 
     </italic> mutation frequency was higher in NSCLC tumours of US origin (17%) compared with 5% in NSCLCs of Korean origin ( 
     <italic>
       P 
     </italic>=0.001). They tended to occur more commonly in adenocarcinomas (13%) than in squamous cell carcinomas (5%) ( 
     <italic>
       P 
     </italic>=0.066). 
     <italic>
       LKB1 
     </italic> mutations associated with smoking history ( 
     <italic>
       P 
     </italic>=0.007) and 
     <italic>
       KRAS 
     </italic> mutations ( 
     <italic>
       P 
     </italic>=0.042) were almost mutually exclusive with 
     <italic>
       EGFR 
     </italic> mutations ( 
     <italic>
       P 
     </italic>=0.002). The outcome of stages I and II NSCLC patients treated with surgery alone did not significantly differ based on 
     <italic>
       LKB1 
     </italic> mutation status. Our study provides clinical and molecular characteristics of NSCLC, which harbour 
     <italic>
       LKB1 
     </italic> mutations.</p> 
   </abstract> 
   <kwd-group> 
    <kwd>
      carcinoma 
    </kwd> 
    <kwd>
      non-small cell lung 
    </kwd> 
    <kwd>
      mutation 
    </kwd> 
    <kwd>
      LKB1 
    </kwd> 
    <kwd>
      EGFR 
    </kwd> 
    <kwd>
      KRAS 
    </kwd> 
   </kwd-group> 
  </article-meta> 
 </front> 
 <p>Peutz?Jeghers syndrome (PJS) is caused by mutations in the 
  <italic>
    LKB1 
  </italic> tumour suppressor gene ( 
  <xref ref-type="other" rid="bib14">
    Hemminki 
   <italic>
     et al 
   </italic>, 1998 
  </xref>). LKB1 is serine?threonine kinase, which has been shown to regulate cell cycle progression, apoptosis, and cell polarity ( 
  <xref ref-type="other" rid="bib29">
    Tiainen 
   <italic>
     et al 
   </italic>, 1999 
  </xref>). The major target of LKB1 kinase activity is thought to be AMP-activated protein kinase (AMPK). AMPK is activated under low cellular energy conditions by raising AMP levels and it phosphorylates multiple downstream targets including tuberosclerosis complex 2 gene, which represses mTOR signalling. Phosphorylation of AMPK by LKB1 is needed for full activity of AMPK and suppression of mTOR activity under low energy conditions ( 
  <xref ref-type="other" rid="bib28">
    Shaw 
   <italic>
     et al 
   </italic>, 2004 
  </xref>). The hallmarks of PJS include mucocutaneous pigmentation and hamartomatous polyps of the gastrointestinal tract. Patients with PJS have an increased risk of developing gastrointestinal, pancreatic, breast, gynecological, and non-small cell lung cancers (NSCLC). The overall risk for cancers is increased 5- to 12-fold in different age groups compared with the general population ( 
  <xref ref-type="other" rid="bib13">
    Hearle 
   <italic>
     et al 
   </italic>, 2006 
  </xref>). Somatic mutations of the 
  <italic>
    LKB1 
  </italic> tumour suppressor have rarely been found in cancers from patients who do not have PJS except for NSCLC ( 
  <xref ref-type="other" rid="bib3">
    Avizienyte 
   <italic>
     et al 
   </italic>, 1999 
  </xref>). Previous reports have suggested the 
  <italic>
    LKB1 
  </italic> mutation rate to be as high as 30% in NSCLC tumours and cell lines derived from patients of Caucasian origin ( 
  <xref ref-type="other" rid="bib9">
    Carretero 
   <italic>
     et al 
   </italic>, 2004 
  </xref>; 
  <xref ref-type="other" rid="bib22">
    Matsumoto 
   <italic>
     et al 
   </italic>, 2007 
  </xref>) and to be infrequent in NSCLC patients of Asian origin (3%) ( 
  <xref ref-type="other" rid="bib25">
    Onozato 
   <italic>
     et al 
   </italic>, 2007 
  </xref>). Furthermore, 
  <italic>
    LKB1 
  </italic> mutations have been shown to be associated with adenocarcinoma histology, male gender, and smoking history ( 
  <xref ref-type="other" rid="bib22">
    Matsumoto 
   <italic>
     et al 
   </italic>, 2007 
  </xref>). A recent report of using a mouse model for 
  <italic>
    lkb1 
  </italic> inactivation in NSCLC has provided insights into the role of the gene in this cancer. This study showed that 
  <italic>
    lkb1 
  </italic> inactivation in combination with activating mutations of 
  <italic>
    kras 
  </italic> using inducible promoters in the lung was associated with decreased survival compared with 
  <italic>
    kras 
  </italic> mutation alone ( 
  <xref ref-type="other" rid="bib17">
    Ji 
   <italic>
     et al 
   </italic>, 2007 
  </xref>).</p> 
 <p>Current screening techniques for 
  <italic>
    LKB1 
  </italic> tumour suppressor mutations rely on conventional exonic sequencing of the DNA, which can identify single base pair changes and small deletions/insertions ( 
  <xref ref-type="other" rid="bib5">
    Ballhausen and Gunther, 2003 
  </xref>). The addition of multiple ligation-dependent probe amplification (MLPA), which enables detection of exonic and whole gene deletions, with exonic sequencing has increased the mutation detection rates to 80% in patients with PJS phenotype ( 
  <xref ref-type="other" rid="bib31">
    Volikos 
   <italic>
     et al 
   </italic>, 2006 
  </xref>). Conventional sequencing has also been used to detect mutations of 
  <italic>
    LKB1 
  </italic> at mRNA level and some mutations missed by sequencing at the DNA level have been discovered by mRNA-based approaches ( 
  <xref ref-type="other" rid="bib1">
    Abed 
   <italic>
     et al 
   </italic>, 2001 
  </xref>). However, mutant forms of 
  <italic>
    LKB1 
  </italic> mRNAs can have a shortened half-life because of nonsense-mediated decay, which can potentially interfere with mutation detection ( 
  <xref ref-type="other" rid="bib1">
    Abed 
   <italic>
     et al 
   </italic>, 2001 
  </xref>).</p> 
 <p>We have recently described a rapid and sensitive enzymatic method to detect mutations in epidermal growth factor ( 
  <italic>
    EGFR 
  </italic>) of DNA from fresh tissue and paraffin-embedded tissues ( 
  <xref ref-type="other" rid="bib15">
    Janne 
   <italic>
     et al 
   </italic>, 2006 
  </xref>). This method includes amplification of region of interest with PCR, SURVEYOR endonuclease digestion of the products, which cleaves mismatched heteroduplex DNAs, and detection of DNA fragments by sensitive high-performance liquid chromatography (HPLC) WAVE HS system. Subsequently, SURVEYOR-positive specimens are fractionated in partially denaturing conditions and are Sanger-sequenced. The major advantages of SURVEYOR-WAVE method are the fast exclusion of wild-type specimens without laborious conventional sequencing and high sensitivity. The SURVEYOR-WAVE method is more sensitive than conventional sequencing as it can detect mutant DNA sequences when they are present in 1% or more of total DNA ( 
  <xref ref-type="other" rid="bib15">
    Janne 
   <italic>
     et al 
   </italic>, 2006 
  </xref>).</p> 
 <p>The current study was designed to analyse the incidence of 
  <italic>
    LKB1 
  </italic> mutations in NSCLC. Furthermore, we wanted to investigate the 
  <italic>
    LKB1 
  </italic> mutational frequency in different histologies and ethnic backgrounds, and assess their correlation to smoking history, gender, stage, survival, and other oncogenic mutations in NSCLC.</p> 
 <sec sec-type="materials|methods"> 
  <h3>Materials and Methods</h3> 
  <sec> 
   <h3>Cell lines and tumour specimens</h3> 
   <p><span style="background-color:#fff7f7">The NSCLC cell lines A549, NCI-H1395, NCI-H1650, NCI-H1666, NCI-H1781, NCI-1975, NCI-H23, NCI-H2126, NCI-H441, NCI-H820, HCC2935, HCC4006, and HCC827 were purchased from ATCC (Manassas, VA, USA)</span>. <span style="background-color:#fffefe">H3255, H3255GR, HCC2279, and PC-9 have been previously described ( Ono et al , 2004 ; Tracy et al , 2004 ; Engelman et al , 2006 )</span>. <span style="background-color:#ffe9e9">Ma1, and Ma70 are NSCLC cell lines harbouring EGFR mutations that were established at the Kinki University, Osaka, Japan</span>. <span style="background-color:#fffafa">A549, NCI-H1395, NCI-H1666, NCI-H23, NCI-H2122, NCI-H2126, and NCI-H460 have previously been reported to contain LKB1 mutations ( Sanchez-Cespedes et al , 2002 ; Bamford et al , 2004 ; Carretero et al , 2004 )</span></p> 
   <p><span style="background-color:#ffebeb">NSCLC tumours ( n =310) were collected from surgical resections from patients with stages I?IV NSCLC when sufficient material for RNA extraction was available</span>. <span style="background-color:#ffffff">The majority of the specimens ( n =167) was collected at the Samsung Medical Center, Seoul, Korea</span>. <span style="background-color:#fff8f8">Frozen tumour tissues were collected from 809 out of 2442 patients who underwent curative resection for NSCLC from November 1995 to February 2007 at Samsung Medical Center</span>. <span style="background-color:#ffffff">One or two pieces from the periphery of the tumour masses ? avoiding necrotic regions ? were immediately frozen at ?80?C until retrieved</span>. <span style="background-color:#fffefe">Medical records and haematoxylin and eosin-stained slides of the specimen were reviewed by a single pathologist</span>. <span style="background-color:#ffffff">Only frozen tumour tissues from adenocarcinoma or squamous cell carcinoma (according to the 2004 World Health Organization histopathological criteria) were included</span>. <span style="background-color:#fffcfc">Only frozen tumour tissues with a tumour cell content of more than 70% were used for further analysis</span>. <span style="background-color:#fffafa">In addition, frozen tumour tissues of the following patients were excluded from the study: patients who had received preoperative neoadjuvant treatments, patients with double primary lung cancer, and patients who had undergone incomplete resections or who had not been subjected to mediastinal lymph node dissections</span>. <span style="background-color:#ffffff">Selected frozen tumour tissues were used for the microdissection</span>. <span style="background-color:#fffcfc">Briefly, frozen tissues were lightly stained with haematoxylin?eosin to improve visualisation, and necrotic tumour tissues and intervening normal tissues were removed</span>. <span style="background-color:#fffafa">Each of the microdissected tumour tissues with a tumour cell content of more than 90% was placed in 1?ml Easy Blue reagent of a commercially available RNA isolation kit (easy-spin? Total RNA Extraction Kit, iNtRON Biotechnology, Gyeonggi-do, Korea), immediately homogenised by vortexing, and the total RNA was extracted</span>. <span style="background-color:#fffefe">The quantity and quality of RNA were analysed using a spectrometer (Nanodrop Technologies, Rockland, DE, USA) and Agilent 2100 Bioanalyzer (Agilent RNA 6000 Nano Kit, Agilent Technologies Inc., B?blingen, Germany), respectively</span>. <span style="background-color:#fffcfc">Finally, 167 frozen tissues with acceptable quality of RNA (RNA Integrity Number (RIN) value over 7.0) were used for the current studies</span>. All patients provided written informed consent.</p> 
   <p><span style="background-color:#fff3f3">The tumours from Caucasian patients ( n =143) were collected at the Brigham and Women's Hospital, Boston, MA, USA between 1991 and 1997 and have been previously published for patient characteristics and histology, and for expression profile-based clustering of the tumours ( Bhattacharjee et al , 2001 ; Hayes et al , 2006 )</span>. <span style="background-color:#fff3f3">Frozen samples of resected lung tumours were obtained within 30?min of resection and subdivided into 100?mg samples and snap frozen at ?80?C</span>. <span style="background-color:#fffafa">Each specimen was associated with an immediately adjacent sample embedded for histology in an optimal cutting temperature medium and stored at ?80?C</span>. <span style="background-color:#fffbfb">Six micrometres of frozen sections of embedded samples stained with haematoxylin and eosin were used to confirm the postoperative pathological diagnosis and to estimate the cellular composition of adjacent samples</span>. <span style="background-color:#ffffff">All specimens underwent pathological review by two pathologists</span>. <span style="background-color:#fff3f3">In all 109 tumours obtained during the same time period were excluded because they did not meet one or more of the eligibility criteria</span>. <span style="background-color:#fffdfd">Tissue samples were homogenised in Trizol (Life Technologies, Gaithersburg, MD, USA) and RNA was extracted and purified by using the RNeasy column purification kit (Qiagen, Chatsworth, CA, USA)</span>. <span style="background-color:#ffffff">Denaturing formaldehyde gel electrophoresis followed by northern blotting using a ? -actin probe assessed RNA integrity</span>. <span style="background-color:#ffffff">Samples were excluded if ? -actin was not full length</span>. <span style="background-color:#ffffff">All patients provided written informed consent</span>. <span style="background-color:#ffd7d7">The US cohort included specimens that have previously undergone analyses and the results have been published for EGFR, KRAS , and BRAF mutations ( Bhattacharjee et al , 2001 ; Naoki et al , 2002 ; Hayes et al , 2006 )</span>. <span style="background-color:#fffafa">We reconfirmed the mutations in 30 of these specimens using the SURVEYOR-based analysis (see section SURVEYOR digestion and HPLC analysis) and found 100% concordance between the two methods (data not shown)</span>.</p> 
   <p><span style="background-color:#fffefe">Cell line specimens were snap frozen and stored at ?80?C</span>. <span style="background-color:#fff1f1">RNA was extracted from tumours and cell lines using Trizol (Invitrogen, Carlsbad, CA, USA), purified with Rneasy Mini Kit (Qiagen, Valencia, CA, USA) and was used for cDNA synthesis using the QuantiTect reverse transcription kit (Qiagen, Valencia, CA, USA)</span>.</p> 
  </sec> 
  <sec> 
   <h3>PCR primers and cycling conditions</h3> 
   <p><span style="background-color:#fffdfd">For LKB1 gene analysis, PCR primers were designed to amplify the cDNA in two amplicons</span>. <span style="background-color:#ffeaea">PCR primers of the first amplicon were designed to hybridise to the noncoding area of the mRNA upstream of exon 1 (5?-agggaagtcggaacacaagg-3?) and to exon 5 (5?-ccagatgtccaccttgaagc-3?) generating a PCR product of 797?bp</span>. <span style="background-color:#ffd4d4">The primers for the second amplicon located at exon 5 (5?-aacggcctggacaccttct-3?) and to noncoding exon 10 (5?-gaaccggcaggaagactgag-3?) generating a product of 702?bp, which has an overlapping part with first amplicon</span>. <span style="background-color:#fff2f2">For SURVEYOR-WAVE analysis of KRAS , PCR primers (5?-ggcctgctgaaaatgactga-3?, 5?-tcctgagcctgttttgtgtct-3?) were designed to generate an amplicon of 407?bp covering codons 12, 13 and 61, which are the codons commonly mutated in lung cancers</span>. <span style="background-color:#ffe7e7">For SURVEYOR-WAVE mutation analysis of BRAF , cDNA was amplified in two overlapping amplicons (5?-aggatttcgtggtgatggag-3?, 5?-gatgacttctggtgccatcc-3?, and 5?-gacgggactcgagtgatgat-3?, 5?-ggtatcctcgtcccaccata-3?) covering codons 387?673</span>. <span style="background-color:#fff3f3">For SURVEYOR-WAVE analysis of EGFR , PCR amplification was done in a single amplicon (5?-ggagcctcttacacccagtg-3?, 5?-aggtcatcaactcccaaacg-3?), which covered exons 18?21 of the gene</span>. <span style="background-color:#ffffff">PCR amplification was done using JumpStart Taq (Sigma, St Louis, MO, USA) under the manufacturer's guidelines</span>. <span style="background-color:#ffd6d6">A part of the specimens ( n =103) was previously characterised for KRAS, BRAF, EGFR mutations using reverse transcritase (RT)?PCR and direct sequencing of the PCR products ( Naoki et al , 2002 ; Hayes et al , 2006 )</span></p> 
  </sec> 
  <sec> 
   <h3>SURVEYOR digestion and HPLC analysis</h3> 
   <p><span style="background-color:#fffefe">SURVEYOR digestion and HPLC analysis were carried out as described previously ( Janne et al , 2006 )</span>. <span style="background-color:#fffcfc">In brief, PCR products were digested in reaction mixture containing equal volumes of SURVEYOR enzyme (Transgenomics, Omaha, NE, USA) and Enhancer (Transgenomics, Omaha, NE, USA) at 42?C for 20?min followed by termination of the reaction by Stop Solution (Transgenomics, Omaha, NE, USA)</span>. <span style="background-color:#fffcfc">Specimens were then loaded to the WAVE HS HPLC (Transgenomics, Omaha, NE, USA) at 50?C, eluted with an increasing acetonitrile gradient, and detected by UV detector using DNA intercalating fluorescence dye (Transgenomics, Omaha, NE, USA)</span>. <span style="background-color:#fff5f5">When cell lines known to be homozygous for specific mutation were analysed, PCR products were mixed 1?:?1 with PCR products of a wild-type cell line, denatured by heating, and slowly renatured to generate heteroduplexes</span>.</p> 
  </sec> 
  <sec> 
   <h3>Sequencing and fractionation</h3> 
   <p><span style="background-color:#fffefe">Specimens that showed an altered pattern on the SURVEYOR tracings were purified using QIAquick kit (Qiagen, Valencia, CA, USA) and sequenced bi-directionally by molecular biology core facility of Dana?Farber Cancer Institute</span>. <span style="background-color:#fffefe">If a specimen showed an altered pattern on the SURVEYOR tracing but had a wild-type sequence by direct DNA sequencing, it underwent fractionation by WAVE HS HPLC in partially denaturing conditions</span>. <span style="background-color:#ffffff">Running temperatures for specific amplicons were calculated by the Navigator Software (Transgenomics, Omaha, NE, USA)</span>. <span style="background-color:#fffafa">Collected fragments were amplified with PCR using the same primers as in the original amplification, purified and sequenced as previously described above</span>.</p> 
  </sec> 
  <sec> 
   <h3>Statistical analysis</h3> 
   <p><span style="background-color:#ffacac">Fisher's exact test was used to assess the association of LKB1 mutation status with other clinical, pathological, and genetic characteristics</span>. <span style="background-color:#ffdfdf">To adjust for any difference between ethnic groups, the association between LKB1 mutation rate and each characteristic was also evaluated as stratified contingency tables</span>. <span style="background-color:#fffefe">If we did not reject that the odds ratios were the same across ethnic groups, we then tested whether the common odds ratios were unity based on the stratified Mantel?Haenszel estimate ( Breslow and Day, 1980 )</span>. <span style="background-color:#fffdfd">Overall survival was estimated using the Kaplan?Meier method, with differences between the groups compared using the log-rank test</span>. <span style="background-color:#ffeeee">All P -values were based on a two-sided hypothesis, with P &lt;0.05 considered to be statistically significant and 0.05&lt; P &lt;0.10 considered to be borderline significant</span></p> 
  </sec> 
 </sec> 
 <sec> 
  <h3>Results</h3> 
  <sec> 
   <h3>SURVEYOR-WAVE mutation detection of 
    <italic>
      LKB1 
    </italic> tumour suppressor in NSCLC cell lines</h3> 
   <p><span style="background-color:#fff1f1">The impact of the stability of LKB1 mRNA on detecting LKB1 mutations was tested using RT?PCR with mRNA extracted from NSCLC cell lines that had previously been characterised for LKB1 mutations</span>. <span style="background-color:#fffdfd">These included NCI-H441 (wild type) and A549, NCI-H1395, NCI-H23, and NCI-H2126 (all containing LKB1 mutations)</span>. <span style="background-color:#fff8f8">Reverse transcriptase?PCR amplification of the whole coding region of the LKB1 mRNA showed that cell lines with nonsense (A549, NCI-H23) mutations or 1?bp deletion (H1395) expressed mRNA with comparable size to the wild-type H441 cell line (1460?bp)</span>. <span style="background-color:#fffcfc">H2126 cell line, which is known to have homozygous deletion of exons 4?6, expressed mRNA with substantially smaller size (?1000?bp) corresponding to deletion of 398?bp</span>. <span style="background-color:#fffdfd">RT?PCR revealed no major difference in LKB1 mRNA expression levels between LKB1 mutant or wild-type cell lines ( Figure 1A )</span></p> 
   <p><span style="background-color:#fff7f7">As LKB1 mutant and wild-type cell lines expressed comparable amounts of LKB1 mRNA with RT?PCR, we studied the cDNA for mutations using the SURVEYOR-WAVE method</span>. <span style="background-color:#fffcfc">The WAVE HPLC provides a system to analyse DNA fragments smaller than 900?bp and therefore we designed two overlapping amplicons covering exons 1?5 (797?bp) and 5?9 (702?bp) to amplify the whole coding region of LKB1 mRNA</span>. <span style="background-color:#fffbfb">PCR products of LKB1 mutant cell lines were mixed 1?:?1 with the products from wild-type cell lines (H441) to generate heteroduplexes as LKB1 mutant cell lines were previously reported to be homozygous for the inactivation of the gene</span>. <span style="background-color:#fffbfb">SURVEYOR-WAVE analysis of the amplicon covering exons 1?5 revealed novel peaks with the cDNA for A549, and NCI-H1395 cell lines compared with the wild type from NCI-H441 ( Figure 1B )</span>. <span style="background-color:#ffffff">SURVEYOR-WAVE analysis of exons 5?9 showed novel peaks for the NCI-H23 cell line as well</span>. <span style="background-color:#fff9f9">The mutations detected with SURVEOR-WAVE were confirmed by conventional DNA sequencing and they corresponded to previous reports ( Sanchez-Cespedes et al , 2002 ; Carretero et al , 2004 )</span>. <span style="background-color:#fff4f4">We could not detect the LKB1 mutation of H2126 cell line with SURVEYOR-WAVE method using a two-amplicon approach because this cell line has a homozygous deletion of exons 4?6 and the reverse primer of the first amplicon and the forward primer of the second amplicon, which lie on the deleted part of the gene (data not shown)</span></p> 
  </sec> 
  <sec> 
   <h3> 
    <italic>
      LKB1 
    </italic> tumour suppressor gene mutations in NSCLC tumours</h3> 
   <p><span style="background-color:#fff7f7">We next used the SURVEYOR-WAVE method to screen NSCLC tumour specimens ( n =310) for LKB1 mutations</span>. <span style="background-color:#ffd1d1">We detected 34 LKB1 mutations (11%) in the NSCLC tumour specimens ( Table 1 )</span>. <span style="background-color:#fffbfb">The majority of the LKB1 mutations detected was deletions or insertions ( n =25, 74%)</span>. <span style="background-color:#ffeeee">The remainder was missense ( n =7, 21%) and nonsense ( n =2, 6%) mutations ( Table 2 , Figure 1C )</span>. <span style="background-color:#fffcfc">About one-half of the deletions and insertions were small, covering &lt;15?bp ( n =14, 56%), whereas larger deletions ( n =11, 44%) covering hundreds of base pairs were detected in the remaining specimens</span>. <span style="background-color:#ffffff">Some mutational hotspots were discovered</span>. <span style="background-color:#ffdfdf">The areas that had the same mutation in more than one tumour specimen included deletion of exon 4 ( n =4), deletion of exons 2 and 3 ( n =3), D194Y ( n =2), and P281L ( n =2)</span>. <span style="background-color:#ffdada">Interestingly, a significant portion of the mutations was located in exon 1 ( n =11, 32%) but there was no area of recurrent mutations in this exon ( Table 2 )</span>. <span style="background-color:#fffbfb">Of the missense mutations detected in the current study, all except R426W are in the kinase domain of the protein</span>. <span style="background-color:#fffafa">Missense mutations in codons 176 and 194 have been previously characterised in PJS ( Launonen, 2005 )</span>. <span style="background-color:#fffcfc">We also found four F354L alterations (data not shown) but we did not consider these as missense mutations as this alteration has previously been reported to be a rare polymorphism of the gene ( Launonen et al , 2000 )</span>. <span style="background-color:#fffbfb">We did not have access to the corresponding normal tissues and therefore, we could not verify if some of the missense mutations were somatic or germline</span>.</p> 
  </sec> 
  <sec> 
   <h3>Association of 
    <italic>
      LKB1 
    </italic> tumour suppressor mutations in NSCLC with clinicopathological characteristics</h3> 
   <p><span style="background-color:#ffd5d5">The mutation frequency of LKB1 gene was significantly higher in NSCLCs in the Caucasian cohort ( Table 1 )</span>. <span style="background-color:#ffcbcb">Twenty-five (17% of specimens) of the LKB1 mutations were detected in NSCLCs collected from patients in the United States, whereas only nine mutations (5% of specimens) were detected in the Korean cohort ( P =0.001) ( Table 1 )</span>. <span style="background-color:#ffe7e7">The LKB1 mutation rate tended to be higher in adenocarcinomas (13%) compared with squamous cell carcinomas (5%) ( P =0.067)</span>. <span style="background-color:#ffebeb">Differences in histological subgroups were relatively modest in the US cohort with mutations in 18 out of 94 (19%) adenocarcinomas vs 5 out of 38 (13%) in squamous cell cancers ( P =0.461)</span>. <span style="background-color:#ffebeb">This is in contrast to the findings in the Asian patients where all of the LKB1 mutations were detected in adenocarcinomas (9 out of 113 (8%)) and none were detected in squamous cell cancers (0 out of 54 (0%); P =0.032)</span>. <span style="background-color:#ffe6e6">Nevertheless, the higher rate of LKB1 mutation in adenocarcinomas compared with squamous cell carcinomas retains the same level of statistical significance (stratified P =0.064) after adjusting for fluctuation between ethnic groups</span>. <span style="background-color:#ffd7d7">The US cohort also included nine specimens from adenosquamous carcinomas and two out of nine (22%) had LKB1 mutations, which is similar to the frequency in adenocarcinomas in this population ( Table 1 )</span>. <span style="background-color:#ffdada">There was no association between LKB1 mutations and the clinical stage of the NSCLC patients</span>. <span style="background-color:#ffd9d9">Kaplan?Meier survival curves of stages I and II NSCLC patients showed a tendency for shorter survival in patients with LKB1 mutant tumours but this, however, did not reach statistical significance ( P =0.17) ( Figure 2 )</span>. <span style="background-color:#ffe4e4">No differences in survival were observed in patients who harboured both LKB1 and KRAS mutations compared with those with KRAS or LKB1 alone but the total number of patients with both mutations who had stages I or II NSCLC was small ( n =9; data not shown)</span>. <span style="background-color:#ff9d9d">We detected an association of LKB1 mutations with a smoking history ( P =0.007) and only two mutations were detected in tumours from 72 NSCLC patients who were either never or light (?10 pack years) former smokers ( Table 1 )</span>. <span style="background-color:#ffe8e8">After adjusting for ethnic group, the higher rate of LKB1 mutation among patients with a smoking history is borderline significant (stratified P =0.067)</span>. <span style="background-color:#fff7f7">The reduction in statistical significance is likely owing to the loss of power associated with the overall rarity of LKB1 mutations among never or light former smokers</span>. <span style="background-color:#ffefef">For these analyses we combined both never smokers and light (?10 pack years) smokers as the frequency of mutations in other oncogenes such as EGFR is similar in these two patient groups ( Pham et al , 2006 )</span>. <span style="background-color:#fffafa">There were no correlations between LKB1 mutations and gender or age of a patient</span></p> 
  </sec> 
  <sec> 
   <h3>Association of 
    <italic>
      LKB1 
    </italic> mutations with 
    <italic>
      K-Ras 
    </italic>, 
    <italic>
      B-Raf 
    </italic>, and 
    <italic>
      EGFR 
    </italic> mutations in NSCLC</h3> 
   <p><span style="background-color:#ffd6d6">Previous reports have suggested that in NSCLC cell lines, LKB1 mutations often occur concurrently with KRAS or BRAF mutations ( Sanchez-Cespedes et al , 2002 ; Carretero et al , 2004 )</span>. <span style="background-color:#ffdcdc">Furthermore, EGFR mutations are often mutually exclusive with KRAS mutations in NSCLC ( Kosaka et al , 2004 ; Marchetti et al , 2005 )</span>. <span style="background-color:#ff8b8b">We used combined data from previous papers ( Sanchez-Cespedes et al , 2002 ; Carretero et al , 2004 ) and from Sanger institute's databases ( Bamford et al , 2004 ) to analyse association of LKB1 mutations with mutations of KRAS , BRAF , and EGFR </span>. <span style="background-color:#ffd2d2">Analysis of LKB1 mutation harbouring NSCLC cell lines (A-427, A549, NCI-H1395, NCI-H1666, NCI-H2122, NCI-H2126, NCI-H23, and NCI-H460) showed that five of the cell lines (63%) had concurrent LKB1 and KRAS mutations, two (25%) had concurrent LKB1 and BRAF mutations, and only one (13%) had neither KRAS nor BRAF mutations</span>. <span style="background-color:#fff4f4">None of these cell lines had EGFR mutations</span>.</p> 
   <p><span style="background-color:#ffcaca">As our findings in NSCLC cell lines suggested concurrency of KRAS or BRAF and mutual exclusiveness of EGFR mutations with LKB1 mutations, we analysed the mutational status of these genes in our primary NSCLC tumour specimens</span>. <span style="background-color:#ff8b8b">KRAS mutations were detected in 49 (16% in the whole tumour set, 25% in Caucasian and 8% in Asian specimens) tumour specimens with 10 (20% of KRAS mutants) of these occurring concurrently with an LKB1 mutation ( P =0.042) ( Table 3 )</span>. <span style="background-color:#ffd3d3">Four BRAF mutations (1%) were found in the tumour set (G465V, N581S, L596R, and T599I) and one of these (N581S) occurred concurrently with LKB1 mutation ( P =0.373)</span>. <span style="background-color:#ffcece">Seventy tumours (23% in the whole tumour set, 9% in Caucasian, 34% in Asian specimens) had EGFR kinase domain mutations with only one of them occurring concurrently with an LKB1 mutation ( P =0.002)</span>. <span style="background-color:#ffe9e9">The tumour with a concurrent EGFR and LKB1 mutation had a missense mutation of LKB1 outside the kinase domain (R426W)</span>. <span style="background-color:#ffffff">No germ line DNA was available from this patient</span>. <span style="background-color:#fffefe">However, a recent report has suggested that R426W is in fact a rare polymorphism of the gene ( Onozato et al , 2007 )</span>. <span style="background-color:#ffe3e3">Taken together our findings suggest that unlike KRAS , mutations in EGFR and LKB1 are mutually exclusive in NSCLC</span></p> 
   <p><span style="background-color:#ffcaca">Previous reports ( Sanchez-Cespedes et al , 2002 ; Carretero et al , 2004 ) and Cancer Genome Project by Sanger Institute ( Bamford et al , 2004 ) have extensively characterised LKB1 mutations in NSCLC cell lines with KRAS and BRAF mutations, but LKB1 status of EGFR mutant NSCLC cell lines has not been extensively analysed</span>. <span style="background-color:#ffe9e9">Therefore, we analysed the LKB1 genotype of NSCLC cell lines with known EGFR or ERBB2 mutations</span>. <span style="background-color:#fff2f2">Twelve EGFR mutant and one ERBB2 mutant cell lines were analysed for LKB1 genotype</span>. <span style="background-color:#ffeded">No LKB1 mutations were detected in these cell lines ( Table 4 )</span></p> 
  </sec> 
 </sec> 
 <sec> 
  <h3>Discussion</h3> 
  <p><span style="background-color:#ffefef">The present study characterised LKB1 mutation frequency in NSCLC using one of the largest tumour sets to date ( n =310)</span>. <span style="background-color:#fff0f0">Our study analysed tumours from different histologies and of both a US and Korean origin to determine potential histological and ethnic variation in LKB1 mutational frequency</span>. <span style="background-color:#fff7f7">The large size of our study enabled us to study associations of LKB1 mutations with clinocopathological factors, which have been incompletely characterised in previous studies ( Sanchez-Cespedes et al , 2002 ; Carretero et al , 2004 ; Fernandez et al , 2004 ; Matsumoto et al , 2007 ; Onozato et al , 2007 )</span>. <span style="background-color:#fff4f4">In addition, we used a modification of a sensitive mutation screening technique that we have previously developed to facilitate the rapid detection of LKB1 mutations ( Janne et al , 2006 )</span></p> 
  <p><span style="background-color:#fff2f2">Findings from our study confirm the high frequency of LKB1 mutations in NSCLC (11%), which in contrast, are rare (0?4%) in other common solid malignancies ( Avizienyte et al , 1998 , 1999 )</span>. <span style="background-color:#fffcfc">The reason behind these observations is presently unknown but might reflect the differences in carcinogen exposure in the lungs compared with other tissues</span>. <span style="background-color:#ffcbcb">In support of this hypothesis, we find that LKB1 mutations are significantly ( P =0.007) more common in smokers than in never or light (?10 pack years) cigarette smokers ( Table 1 )</span>. <span style="background-color:#fffafa">Male PJS patients (age ?50 years) have an increased risk of developing lung cancer compared with the general population but the relationship of smoking and the increased risk of lung cancer in PJS is unknown ( Hearle et al , 2006 )</span>. <span style="background-color:#fff3f3">Interestingly, the LKB1 mutation spectrum found in the current study is very similar to those previously published for PJS (deletions 34%, insertions 15%, splice site mutations 14%, missense mutations 21%, and nonsense mutations 12%) ( Launonen, 2005 ) and, as in PJS, no clear mutational hotspots were detected</span></p> 
  <p><span style="background-color:#ffe7e7">Our study also demonstrated that the LKB1 mutation frequency was significantly higher in cancers derived from a US population compared with those found in Korean patients (17 vs 5%; P =0.001)</span>. <span style="background-color:#fffbfb">These differences also track with cigarette smoking, as the number of never/light former smokers was much higher in the Korean cohort compared with the US cohort of patients (38 vs 13%)</span>. <span style="background-color:#fff7f7">Similarly, a recent study of 100 Japanese NSCLCs found that only 3% contained an LKB1 mutation ( Onozato et al , 2007 )</span>. <span style="background-color:#ffe5e5">These findings are in contrast to EGFR mutations, which are more frequently detected in tumours from never/light cigarette smokers and from Asian patients ( Janne and Johnson, 2006 )</span>. <span style="background-color:#fffcfc">Our studies further highlight ethnic and environmental differences in the origins of NSCLC</span>.</p> 
  <p><span style="background-color:#fff0f0">Given the differences in LKB1 mutation frequencies in smokers vs never/light smokers and in the US compared with Korean patients, we further determined whether these were also associated with other oncogene mutations known to vary in these subgroups of patients</span>. <span style="background-color:#ffa8a8">Consistent with prior studies we found a significant association with concurrent KRAS mutations, which are common in smokers ( Ahrendt et al , 2001 ), in one out of three of NSCLC with LKB1 mutations ( Table 3 )</span>. <span style="background-color:#ffc1c1">In contrast, there was a significant inverse relationship of LKB1 mutations with EGFR mutations in both NSCLC tumours and cell lines, which has not previously been described ( Tables 3 and 4 )</span>. <span style="background-color:#fffcfc">These differences may relate to the biological role of LKB1 in lung cancer</span>. <span style="background-color:#ffeaea">It is possible that in EGFR mutant lung cancers there is already maximal activation of the PI3K/Akt/mTOR signalling pathway and thus an LKB1 mutation may not be required to further potentiate this signalling pathway</span>. <span style="background-color:#ffecec">In contrast in KRAS mutant cancers, a concurrent LKB1 mutation may be required to enhance mTOR activation</span>. <span style="background-color:#ffe2e2">Mice with concurrent KRAS mutations and LKB1 inactivation have more aggressive tumours and a shorter survival than those with only KRAS mutant cancers ( Ji et al , 2007 )</span>. <span style="background-color:#ffe2e2">In our study, we were not able to detect a significant survival difference for patients whose tumours contained LKB1 mutations alone or concurrently with KRAS mutations (data not shown) likely because of the limited number of tumour specimens</span>. <span style="background-color:#fff7f7">Additional studies are needed to clarify the prognostic impact of LKB1 mutations in humans with NSCLC</span>. <span style="background-color:#ffc1c1">In the present study ?2 out of 3 of LKB1 mutant tumours were KRAS wild type ( Table 3 )</span>. <span style="background-color:#ffeded">One possibility is that such tumours contain a concurrent mutation in another oncogene that activates the same signalling pathway as KRAS</span>. <span style="background-color:#ffdada">For this reason, we examined our tumours for BRAF mutations, which are found in 1?2% of NSCLC ( Naoki et al , 2002 )</span>. <span style="background-color:#ffb7b7">We detected a concurrent LKB1 mutation in one of the four BRAF mutant tumours ( Table 3 )</span>. <span style="background-color:#fff5f5">This tumour was wild type for KRAS (data not shown)</span>. <span style="background-color:#ffe0e0">In addition, some of the BRAF mutant NSCLC cell lines (NCI-H1395, G469A; NCI-H1666, G466V) also contain a concurrent LKB1 mutation ( Sanchez-Cespedes et al , 2002 ; Bamford et al , 2004 ; Carretero et al , 2004 )</span>. <span style="background-color:#ffeeee">Future studies will help further clarify whether LKB1 mutations occur concurrently with other genomic alterations in NSCLC and the impact of this on patient outcome</span></p> 
  <p><span style="background-color:#fff5f5">Our study employed a mutation scanning technology to screen for LKB1 mutations at the cDNA level ( Janne et al , 2006 )</span>. <span style="background-color:#fff7f7">This was advantageous as the entire coding region of LKB1 could be rapidly screened for a mutation using just two overlapping cDNA fragments</span>. <span style="background-color:#fffefe">LKB1 is a challenging gene to analyse at the genomic DNA level because of its high guanine?cytosine content</span>. <span style="background-color:#ffecec">In addition, as many of the LKB1 mutations are small deletions ( Table 2 ) or involve deletions of entire exons, these would be missed using exon-specific genome sequencing methods</span>. <span style="background-color:#fffefe">Our method, however, does have limitations as it would miss deletions at the site of PCR primers, whole gene deletions, or deletions within the promoter region all of which have been infrequently detected in PJS ( Volikos et al , 2006 )</span>. <span style="background-color:#fff7f7">Thus our studies may underestimate the true LKB1 mutation frequency in NSCLC</span>. <span style="background-color:#fff6f6">In addition, our method is limited to the analysis of fresh tumour specimens, which are available only from the minority of NSCLC patients</span>. <span style="background-color:#ffe7e7">Furthermore, as techniques isolating mRNA from formalin-fixed paraffin-embedded tumour specimens continue to improve, this rapid mutation scanning technique can be used to analyse broader populations of tumours from NSCLC patients</span>. <span style="background-color:#fff6f6">Future studies may need to employ a combination of LKB1 mutation detection methodologies including the current method, MLPA and direct sequencing.</span></p> 
 </sec> 
 <back> 
  <ack> 
   <p>This study is supported by grants from the National Institutes of Health 1RO1CA114465-01 (BYY, BEJ, and PAJ), the National Cancer Institute Lung SPORE P20CA90578-02 (BYY and BEJ), American Cancer Society RSG-06-102-01-CCE (BYY and PAJ), Finnish Medical Foundation (JPK), Finnish Cultural Foundation (JPK), and Academy of Finland (JPK). PAJ, MM, and BEJ are part of a pending patent application on 
    <italic>
      EGFR 
    </italic> mutations.</p> 
  </ack> 
  <ref-list> 
   <ref id="bib1"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Abed 
       </surname> 
       <given-names>
         AA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gunther 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kraus 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hohenberger 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ballhausen 
       </surname> 
       <given-names>
         WG 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Mutation screening at the RNA level of the STK11/LKB1 gene in Peutz-Jeghers syndrome reveals complex splicing abnormalities and a novel mRNA isoform (STK11 c.597(insertion mark)598insIVS4) 
     </article-h3> 
     <source />Hum Mutat 
     <year>
       2001 
     </year> 
     <volume>
       18 
     </volume> 
     <fpage>
       397 
     </fpage> 
     <lpage>
       410 
     </lpage> 
     <!--PubMed citation query: 'Hum Mutat||18|397||bib1|'--> 
     <pub-id pub-id-type="pmid">
       11668633 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib2"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ahrendt 
       </surname> 
       <given-names>
         SA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Decker 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Alawi 
       </surname> 
       <given-names>
         EA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zhu Yr 
       </surname> 
       <given-names>
         YR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sanchez-Cespedes 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yang 
       </surname> 
       <given-names>
         SC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Haasler 
       </surname> 
       <given-names>
         GB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kajdacsy-Balla 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Demeure 
       </surname> 
       <given-names>
         MJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sidransky 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Cigarette smoking is strongly associated with mutation of the K-ras gene in patients with primary adenocarcinoma of the lung 
     </article-h3> 
     <source />Cancer 
     <year>
       2001 
     </year> 
     <volume>
       92 
     </volume> 
     <fpage>
       1525 
     </fpage> 
     <lpage>
       1530 
     </lpage> 
     <!--PubMed citation query: 'Cancer||92|1525||bib2|'--> 
     <pub-id pub-id-type="pmid">
       11745231 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib3"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Avizienyte 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Loukola 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Roth 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hemminki 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tarkkanen 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Salovaara 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Arola 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Butzow 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Husgafvel-Pursiainen 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kokkola 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jarvinen 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Aaltonen 
       </surname> 
       <given-names>
         LA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       LKB1 somatic mutations in sporadic tumors 
     </article-h3> 
     <source />Am J Pathol 
     <year>
       1999 
     </year> 
     <volume>
       154 
     </volume> 
     <fpage>
       677 
     </fpage> 
     <lpage>
       681 
     </lpage> 
     <!--PubMed citation query: 'Am J Pathol||154|677||bib3|'--> 
     <pub-id pub-id-type="pmid">
       10079245 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib4"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Avizienyte 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Roth 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Loukola 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hemminki 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lothe 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stenwig 
       </surname> 
       <given-names>
         AE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fossa 
       </surname> 
       <given-names>
         SD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Salovaara 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Aaltonen 
       </surname> 
       <given-names>
         LA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Somatic mutations in LKB1 are rare in sporadic colorectal and testicular tumors 
     </article-h3> 
     <source />Cancer Res 
     <year>
       1998 
     </year> 
     <volume>
       58 
     </volume> 
     <fpage>
       2087 
     </fpage> 
     <lpage>
       2090 
     </lpage> 
     <!--PubMed citation query: 'Cancer Res||58|2087||bib4|'--> 
     <pub-id pub-id-type="pmid">
       9605748 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib5"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ballhausen 
       </surname> 
       <given-names>
         WG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gunther 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Genetic screening for Peutz-Jeghers syndrome 
     </article-h3> 
     <source />Expert Rev Mol Diagn 
     <year>
       2003 
     </year> 
     <volume>
       3 
     </volume> 
     <fpage>
       471 
     </fpage> 
     <lpage>
       479 
     </lpage> 
     <!--PubMed citation query: 'Expert Rev Mol Diagn||3|471||bib5|'--> 
     <pub-id pub-id-type="pmid">
       12877386 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib6"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Bamford 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dawson 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Forbes 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Clements 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pettett 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dogan 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Flanagan 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Teague 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Futreal 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stratton 
       </surname> 
       <given-names>
         MR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wooster 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The COSMIC (Catalogue of Somatic Mutations in Cancer) database and website 
     </article-h3> 
     <source />Br J Cancer 
     <year>
       2004 
     </year> 
     <volume>
       91 
     </volume> 
     <fpage>
       355 
     </fpage> 
     <lpage>
       358 
     </lpage> 
     <!--PubMed citation query: 'Br J Cancer||91|355||bib6|'--> 
     <pub-id pub-id-type="pmid">
       15188009 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib7"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Bhattacharjee 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Richards 
       </surname> 
       <given-names>
         WG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Staunton 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Li 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Monti 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vasa 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ladd 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Beheshti 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bueno 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gillette 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Loda 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Weber 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mark 
       </surname> 
       <given-names>
         EJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lander 
       </surname> 
       <given-names>
         ES 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wong 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Johnson 
       </surname> 
       <given-names>
         BE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Golub 
       </surname> 
       <given-names>
         TR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sugarbaker 
       </surname> 
       <given-names>
         DJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Meyerson 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Classification of human lung carcinomas by mRNA expression profiling reveals distinct adenocarcinoma subclasses 
     </article-h3> 
     <source />Proc Natl Acad Sci USA 
     <year>
       2001 
     </year> 
     <volume>
       98 
     </volume> 
     <fpage>
       13790 
     </fpage> 
     <lpage>
       13795 
     </lpage> 
     <!--PubMed citation query: 'Proc Natl Acad Sci USA||98|13790||bib7|'--> 
    </citation> 
   </ref> 
   <ref id="bib8"> 
    <citation citation-type="book"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Breslow 
       </surname> 
       <given-names>
         NE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Day 
       </surname> 
       <given-names>
         NE 
       </given-names> 
      </name> 
      <collab>
        International Agency for Research on Cancer 
      </collab> 
     </person-group> 
     <article-h3>
       Statistical methods in cancer research 
     </article-h3> 
     <year>
       1980 
     </year> 
     <publisher-name>
       International Agency for Research on Cancer: Lyon 
     </publisher-name>Vol. 1. Analysis of case?control studies. IARC scientific publications; no. 32. 
    </citation> 
   </ref> 
   <ref id="bib9"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Carretero 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Medina 
       </surname> 
       <given-names>
         PP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pio 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Montuenga 
       </surname> 
       <given-names>
         LM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sanchez-Cespedes 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Novel and natural knockout lung cancer cell lines for the LKB1/STK11 tumor suppressor gene 
     </article-h3> 
     <source />Oncogene 
     <year>
       2004 
     </year> 
     <volume>
       23 
     </volume> 
     <fpage>
       4037 
     </fpage> 
     <lpage>
       4040 
     </lpage> 
     <!--PubMed citation query: 'Oncogene||23|4037||bib9|'--> 
     <pub-id pub-id-type="pmid">
       15021901 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib10"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Engelman 
       </surname> 
       <given-names>
         JA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mukohara 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zejnullahu 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lifshits 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Borras 
       </surname> 
       <given-names>
         AM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gale 
       </surname> 
       <given-names>
         CM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Naumov 
       </surname> 
       <given-names>
         GN 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yeap 
       </surname> 
       <given-names>
         BY 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jarrell 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sun 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tracy 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zhao 
       </surname> 
       <given-names>
         X 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Heymach 
       </surname> 
       <given-names>
         JV 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Johnson 
       </surname> 
       <given-names>
         BE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cantley 
       </surname> 
       <given-names>
         LC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Janne 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Allelic dilution obscures detection of a biologically significant resistance mutation in EGFR-amplified lung cancer 
     </article-h3> 
     <source />J Clin Invest 
     <year>
       2006 
     </year> 
     <volume>
       116 
     </volume> 
     <fpage>
       2695 
     </fpage> 
     <lpage>
       2706 
     </lpage> 
     <!--PubMed citation query: 'J Clin Invest||116|2695||bib10|'--> 
     <pub-id pub-id-type="pmid">
       16906227 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib11"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Fernandez 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Carretero 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Medina 
       </surname> 
       <given-names>
         PP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jimenez 
       </surname> 
       <given-names>
         AI 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rodriguez-Perales 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Paz 
       </surname> 
       <given-names>
         MF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cigudosa 
       </surname> 
       <given-names>
         JC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Esteller 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lombardia 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Morente 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sanchez-Verde 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sotelo 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sanchez-Cespedes 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Distinctive gene expression of human lung adenocarcinomas carrying LKB1 mutations 
     </article-h3> 
     <source />Oncogene 
     <year>
       2004 
     </year> 
     <volume>
       23 
     </volume> 
     <fpage>
       5084 
     </fpage> 
     <lpage>
       5091 
     </lpage> 
     <!--PubMed citation query: 'Oncogene||23|5084||bib11|'--> 
     <pub-id pub-id-type="pmid">
       15077168 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib12"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Hayes 
       </surname> 
       <given-names>
         DN 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Monti 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Parmigiani 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gilks 
       </surname> 
       <given-names>
         CB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Naoki 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bhattacharjee 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Socinski 
       </surname> 
       <given-names>
         MA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Perou 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Meyerson 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Gene expression profiling reveals reproducible human lung adenocarcinoma subtypes in multiple independent patient cohorts 
     </article-h3> 
     <source />J Clin Oncol 
     <year>
       2006 
     </year> 
     <volume>
       24 
     </volume> 
     <fpage>
       5079 
     </fpage> 
     <lpage>
       5090 
     </lpage> 
     <!--PubMed citation query: 'J Clin Oncol||24|5079||bib12|'--> 
     <pub-id pub-id-type="pmid">
       17075127 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib13"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Hearle 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Schumacher 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Menko 
       </surname> 
       <given-names>
         FH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Olschwang 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Boardman 
       </surname> 
       <given-names>
         LA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gille 
       </surname> 
       <given-names>
         JJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Keller 
       </surname> 
       <given-names>
         JJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Westerman 
       </surname> 
       <given-names>
         AM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Scott 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lim 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Trimbath 
       </surname> 
       <given-names>
         JD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Giardiello 
       </surname> 
       <given-names>
         FM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gruber 
       </surname> 
       <given-names>
         SB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Offerhaus 
       </surname> 
       <given-names>
         GJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         de Rooij 
       </surname> 
       <given-names>
         FW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wilson 
       </surname> 
       <given-names>
         JH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hansmann 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Moslein 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Royer-Pokora 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vogel 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Phillips 
       </surname> 
       <given-names>
         RK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Spigelman 
       </surname> 
       <given-names>
         AD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Houlston 
       </surname> 
       <given-names>
         RS 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Frequency and spectrum of cancers in the Peutz-Jeghers syndrome 
     </article-h3> 
     <source />Clin Cancer Res 
     <year>
       2006 
     </year> 
     <volume>
       12 
     </volume> 
     <fpage>
       3209 
     </fpage> 
     <lpage>
       3215 
     </lpage> 
     <!--PubMed citation query: 'Clin Cancer Res||12|3209||bib13|'--> 
     <pub-id pub-id-type="pmid">
       16707622 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib14"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Hemminki 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Markie 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tomlinson 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Avizienyte 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Roth 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Loukola 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bignell 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Warren 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Aminoff 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hoglund 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jarvinen 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kristo 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pelin 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ridanpaa 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Salovaara 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Toro 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bodmer 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Olschwang 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Olsen 
       </surname> 
       <given-names>
         AS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stratton 
       </surname> 
       <given-names>
         MR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         de la Chapelle 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Aaltonen 
       </surname> 
       <given-names>
         LA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       A serine/threonine kinase gene defective in Peutz-Jeghers syndrome 
     </article-h3> 
     <source />Nature 
     <year>
       1998 
     </year> 
     <volume>
       391 
     </volume> 
     <fpage>
       184 
     </fpage> 
     <lpage>
       187 
     </lpage> 
     <!--PubMed citation query: 'Nature||391|184||bib14|'--> 
     <pub-id pub-id-type="pmid">
       9428765 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib15"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Janne 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Borras 
       </surname> 
       <given-names>
         AM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kuang 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rogers 
       </surname> 
       <given-names>
         AM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Joshi 
       </surname> 
       <given-names>
         VA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liyanage 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lindeman 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         JC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Halmos 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Maher 
       </surname> 
       <given-names>
         EA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Distel 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Meyerson 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Johnson 
       </surname> 
       <given-names>
         BE 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       A rapid and sensitive enzymatic method for epidermal growth factor receptor mutation screening 
     </article-h3> 
     <source />Clin Cancer Res 
     <year>
       2006 
     </year> 
     <volume>
       12 
     </volume> 
     <fpage>
       751 
     </fpage> 
     <lpage>
       758 
     </lpage> 
     <!--PubMed citation query: 'Clin Cancer Res||12|751||bib15|'--> 
     <pub-id pub-id-type="pmid">
       16467085 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib16"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Janne 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Johnson 
       </surname> 
       <given-names>
         BE 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Effect of epidermal growth factor receptor tyrosine kinase domain mutations on the outcome of patients with non-small cell lung cancer treated with epidermal growth factor receptor tyrosine kinase inhibitors 
     </article-h3> 
     <source />Clin Cancer Res 
     <year>
       2006 
     </year> 
     <volume>
       12 
     </volume> 
     <fpage>
       4416s 
     </fpage> 
     <lpage>
       4420s 
     </lpage> 
     <!--PubMed citation query: 'Clin Cancer Res||12|4416s||bib16|'--> 
     <pub-id pub-id-type="pmid">
       16857820 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib17"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ji 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ramsey 
       </surname> 
       <given-names>
         MR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hayes 
       </surname> 
       <given-names>
         DN 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fan 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McNamara 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kozlowski 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Torrice 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wu 
       </surname> 
       <given-names>
         MC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shimamura 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Perera 
       </surname> 
       <given-names>
         SA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liang 
       </surname> 
       <given-names>
         MC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cai 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Naumov 
       </surname> 
       <given-names>
         GN 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bao 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Contreras 
       </surname> 
       <given-names>
         CM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Li 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chen 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Krishnamurthy 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Koivunen 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chirieac 
       </surname> 
       <given-names>
         LR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Padera 
       </surname> 
       <given-names>
         RF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bronson 
       </surname> 
       <given-names>
         RT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lindeman 
       </surname> 
       <given-names>
         NI 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Christiani 
       </surname> 
       <given-names>
         DC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lin 
       </surname> 
       <given-names>
         X 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shapiro 
       </surname> 
       <given-names>
         GI 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Janne 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Johnson 
       </surname> 
       <given-names>
         BE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Meyerson 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kwiatkowski 
       </surname> 
       <given-names>
         DJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Castrillon 
       </surname> 
       <given-names>
         DH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bardeesy 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sharpless 
       </surname> 
       <given-names>
         NE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wong 
       </surname> 
       <given-names>
         KK 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       LKB1 modulates lung cancer differentiation and metastasis 
     </article-h3> 
     <source />Nature 
     <year>
       2007 
     </year> 
     <volume>
       448 
     </volume> 
     <fpage>
       807 
     </fpage> 
     <lpage>
       810 
     </lpage> 
     <!--PubMed citation query: 'Nature||448|807||bib17|'--> 
     <pub-id pub-id-type="pmid">
       17676035 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib18"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Kosaka 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yatabe 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Endoh 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kuwano 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Takahashi 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mitsudomi 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Mutations of the epidermal growth factor receptor gene in lung cancer: biological and clinical implications 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2004 
     </year> 
     <volume>
       64 
     </volume> 
     <fpage>
       8919 
     </fpage> 
     <lpage>
       8923 
     </lpage> 
     <!--PubMed citation query: 'Cancer Res||64|8919||bib18|'--> 
     <pub-id pub-id-type="pmid">
       15604253 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib19"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Launonen 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Mutations in the human LKB1/STK11 gene 
     </article-h3> 
     <source />Hum Mutat 
     <year>
       2005 
     </year> 
     <volume>
       26 
     </volume> 
     <fpage>
       291 
     </fpage> 
     <lpage>
       297 
     </lpage> 
     <!--PubMed citation query: 'Hum Mutat||26|291||bib19|'--> 
     <pub-id pub-id-type="pmid">
       16110486 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib20"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Launonen 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Avizienyte 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Loukola 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Laiho 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Salovaara 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jarvinen 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mecklin 
       </surname> 
       <given-names>
         JP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Oku 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shimane 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kim 
       </surname> 
       <given-names>
         HC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kim 
       </surname> 
       <given-names>
         JC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nezu 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Aaltonen 
       </surname> 
       <given-names>
         LA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       No evidence of Peutz-Jeghers syndrome gene LKB1 involvement in left-sided colorectal carcinomas 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2000 
     </year> 
     <volume>
       60 
     </volume> 
     <fpage>
       546 
     </fpage> 
     <lpage>
       548 
     </lpage> 
     <!--PubMed citation query: 'Cancer Res||60|546||bib20|'--> 
     <pub-id pub-id-type="pmid">
       10676634 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib21"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Marchetti 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Martella 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Felicioni 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Barassi 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Salvatore 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chella 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Camplese 
       </surname> 
       <given-names>
         PP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Iarussi 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mucilli 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mezzetti 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cuccurullo 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sacco 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Buttitta 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       EGFR mutations in non-small-cell lung cancer: analysis of a large series of cases and development of a rapid and sensitive method for diagnostic screening with potential implications on pharmacologic treatment 
     </article-h3> 
     <source />J Clin Oncol 
     <year>
       2005 
     </year> 
     <volume>
       23 
     </volume> 
     <fpage>
       857 
     </fpage> 
     <lpage>
       865 
     </lpage> 
     <!--PubMed citation query: 'J Clin Oncol||23|857||bib21|'--> 
     <pub-id pub-id-type="pmid">
       15681531 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib22"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Matsumoto 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Iwakawa 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Takahashi 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kohno 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nakanishi 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Matsuno 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Suzuki 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nakamoto 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shimizu 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Minna 
       </surname> 
       <given-names>
         JD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yokota 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Prevalence and specificity of LKB1 genetic alterations in lung cancers 
     </article-h3> 
     <source />Oncogene 
     <year>
       2007 
     </year> 
     <volume>
       26 
     </volume> 
     <fpage>
       5911 
     </fpage> 
     <lpage>
       5918 
     </lpage> 
     <!--PubMed citation query: 'Oncogene||26|5911||bib22|'--> 
     <pub-id pub-id-type="pmid">
       17384680 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib23"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Naoki 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chen 
       </surname> 
       <given-names>
         TH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Richards 
       </surname> 
       <given-names>
         WG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sugarbaker 
       </surname> 
       <given-names>
         DJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Meyerson 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Missense mutations of the BRAF gene in human lung adenocarcinoma 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2002 
     </year> 
     <volume>
       62 
     </volume> 
     <fpage>
       7001 
     </fpage> 
     <lpage>
       7003 
     </lpage> 
     <!--PubMed citation query: 'Cancer Res||62|7001||bib23|'--> 
     <pub-id pub-id-type="pmid">
       12460919 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib24"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ono 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hirata 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kometani 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Miyagawa 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ueda 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kinoshita 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fujii 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kuwano 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Sensitivity to gefitinib (Iressa, ZD1839) in non-small cell lung cancer cell lines correlates with dependence on the epidermal growth factor (EGF) receptor/extracellular signal-regulated kinase 1/2 and EGF receptor/Akt pathway for proliferation 
     </article-h3> 
     <source />Mol Cancer Ther 
     <year>
       2004 
     </year> 
     <volume>
       3 
     </volume> 
     <fpage>
       465 
     </fpage> 
     <lpage>
       472 
     </lpage> 
     <!--PubMed citation query: 'Mol Cancer Ther||3|465||bib24|'--> 
     <pub-id pub-id-type="pmid">
       15078990 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib25"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Onozato 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kosaka 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Achiwa 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kuwano 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Takahashi 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yatabe 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mitsudomi 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       LKB1 gene mutations in Japanese lung cancer patients 
     </article-h3> 
     <source />Cancer Sci 
     <year>
       2007 
     </year> 
     <volume>
       98 
     </volume> 
     <fpage>
       174 
     </fpage> 
     <lpage>
       175 
     </lpage> 
     <!--PubMed citation query: 'Cancer Sci||98|174||bib25|'--> 
     <pub-id pub-id-type="pmid">
       17233835 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib26"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Pham 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kris 
       </surname> 
       <given-names>
         MG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Riely 
       </surname> 
       <given-names>
         GJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sarkaria 
       </surname> 
       <given-names>
         IS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McDonough 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chuai 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Venkatraman 
       </surname> 
       <given-names>
         ES 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Miller 
       </surname> 
       <given-names>
         VA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ladanyi 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pao 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wilson 
       </surname> 
       <given-names>
         RK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Singh 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rusch 
       </surname> 
       <given-names>
         VW 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Use of cigarette-smoking history to estimate the likelihood of mutations in epidermal growth factor receptor gene exons 19 and 21 in lung adenocarcinomas 
     </article-h3> 
     <source />J Clin Oncol 
     <year>
       2006 
     </year> 
     <volume>
       24 
     </volume> 
     <fpage>
       1700 
     </fpage> 
     <lpage>
       1704 
     </lpage> 
     <!--PubMed citation query: 'J Clin Oncol||24|1700||bib26|'--> 
     <pub-id pub-id-type="pmid">
       16505411 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib27"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Sanchez-Cespedes 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Parrella 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Esteller 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nomoto 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Trink 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Engles 
       </surname> 
       <given-names>
         JM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Westra 
       </surname> 
       <given-names>
         WH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Herman 
       </surname> 
       <given-names>
         JG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sidransky 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Inactivation of LKB1/STK11 is a common event in adenocarcinomas of the lung 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2002 
     </year> 
     <volume>
       62 
     </volume> 
     <fpage>
       3659 
     </fpage> 
     <lpage>
       3662 
     </lpage> 
     <!--PubMed citation query: 'Cancer Res||62|3659||bib27|'--> 
     <pub-id pub-id-type="pmid">
       12097271 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib28"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Shaw 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bardeesy 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Manning 
       </surname> 
       <given-names>
         BD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lopez 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kosmatka 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         DePinho 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cantley 
       </surname> 
       <given-names>
         LC 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The LKB1 tumor suppressor negatively regulates mTOR signaling 
     </article-h3> 
     <source />Cancer Cell 
     <year>
       2004 
     </year> 
     <volume>
       6 
     </volume> 
     <fpage>
       91 
     </fpage> 
     <lpage>
       99 
     </lpage> 
     <!--PubMed citation query: 'Cancer Cell||6|91||bib28|'--> 
     <pub-id pub-id-type="pmid">
       15261145 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib29"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Tiainen 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ylikorkala 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Makela 
       </surname> 
       <given-names>
         TP 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Growth suppression by Lkb1 is mediated by a G(1) cell cycle arrest 
     </article-h3> 
     <source />Proc Natl Acad Sci USA 
     <year>
       1999 
     </year> 
     <volume>
       96 
     </volume> 
     <fpage>
       9248 
     </fpage> 
     <lpage>
       9251 
     </lpage> 
     <!--PubMed citation query: 'Proc Natl Acad Sci USA||96|9248||bib29|'--> 
    </citation> 
   </ref> 
   <ref id="bib30"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Tracy 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mukohara 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hansen 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Meyerson 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Johnson 
       </surname> 
       <given-names>
         BE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Janne 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Gefitinib induces apoptosis in the EGFRL858R non-small-cell lung cancer cell line H3255 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2004 
     </year> 
     <volume>
       64 
     </volume> 
     <fpage>
       7241 
     </fpage> 
     <lpage>
       7244 
     </lpage> 
     <!--PubMed citation query: 'Cancer Res||64|7241||bib30|'--> 
     <pub-id pub-id-type="pmid">
       15492241 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib31"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Volikos 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Robinson 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Aittomaki 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mecklin 
       </surname> 
       <given-names>
         JP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jarvinen 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Westerman 
       </surname> 
       <given-names>
         AM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         de Rooji 
       </surname> 
       <given-names>
         FW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vogel 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Moeslein 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Launonen 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tomlinson 
       </surname> 
       <given-names>
         IP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Silver 
       </surname> 
       <given-names>
         AR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Aaltonen 
       </surname> 
       <given-names>
         LA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       LKB1 exonic and whole gene deletions are a common cause of Peutz-Jeghers syndrome 
     </article-h3> 
     <source />J Med Genet 
     <year>
       2006 
     </year> 
     <volume>
       43 
     </volume> 
     <fpage>
       e18 
     </fpage> 
     <!--PubMed citation query: 'J Med Genet||43|e18||bib31|'--> 
     <pub-id pub-id-type="pmid">
       16648371 
     </pub-id> 
    </citation> 
   </ref> 
  </ref-list> 
 </back> 
 <floats-wrap> 
  <fig id="fig1"> 
   <label>Figure 1</label> 
   <p>Mutation analysis of 
    <italic>
      LKB1 
    </italic> gene in NSCLC cell lines and tumours. RT?PCR amplification of cDNA from 
    <italic>
      LKB1 
    </italic> wt (H441) and 
    <italic>
      LKB1 
    </italic> mutant (A549, H1395, and H23) cell lines display the full length LKB1 mRNA (1.4?kbp) while the 
    <italic>
      LKB1 
    </italic> mutant cell line, H2126 with a deletion of exons 4?6 expresses a shorter mRNA (1.0?kbp) ( 
    <bold>
      A 
    </bold>). HPLC tracings of SURVEYOR-WAVE mutation analysis of NSCLC cell lines A549, H1395, or H23 (continuous line), and H441 (dashed line). Time in minutes is shown on the 
    <italic>
      X 
    </italic>-axis, voltage in mV on the 
    <italic>
      Y 
    </italic>-axis ( 
    <bold>
      B 
    </bold>). A549 and H1395 show novel peaks (<sup>*</sup>) in the amplicon covering exons 1?5 (ex1?5) corresponding to 109C&gt;T, Q37X and 165_169delG, frameshift and truncation (FS truncation) mutations. The analysis from H23 demonstrates novel peaks in the amplicon covering exons 5?9 (ex5?9) corresponding to 996G&gt;T, W332X mutation. 
    <italic>
      LKB1 
    </italic> wild-type cDNA (H441) was added to PCR products 1?:?1, denatured by heating and slowly renaturated to generate heteroduplexes since A549, H1395, and H23 have previously reported to be homozygous for the 
    <italic>
      LKB1 
    </italic> mutations. SURVEYOR-WAVE mutation analyses of NSCLC tumours ( 
    <bold>
      C 
    </bold>). AD367 and AD368 tumours showed novel peaks in the ex1?5 amplicon corresponding to 475C&gt;T, Q159X, and 142_143delA, FS, truncation mutations. AD362 tumour had novel peaks in ex5?9 amplicon corresponding to deletion of exon 6. Mutant sequences for AD367 and AD368 are displayed from sequences using the forward primer while mutation of the AD362 is showed with reverse primer.</p> 
   <graphic mime-subtype="eps" xlink:href="6604469f1"></graphic> 
  </fig> 
  <fig id="fig2"> 
   <label>Figure 2</label> 
   <p>Kaplan?Meyer survival curves of stage I and II NSCLC patients with LKB1 wildtype (red line, 
    <italic>
      n 
    </italic>=198) 
    <italic>
      vs 
    </italic> LKB1 mutant (blue line, 
    <italic>
      n 
    </italic>=23) tumours.</p> 
   <graphic mime-subtype="eps" xlink:href="6604469f2"></graphic> 
  </fig> 
  <table-wrap id="tbl1" position="float"> 
   <label>Table 1</label> 
   <p content-type="table-h3">Frequency of 
    <italic>
      LKB1 
    </italic> mutations in NSCLC tumours and their association with clinicopathological characteristics</p> 
   <table border="1" frame="hsides" rules="groups" width="85%"> 
    <colgroup> 
     <col align="left" /> 
     <col align="char" char="(" /> 
     <col align="char" char="(" /> 
     <col align="left" /> 
    </colgroup> 
    <thead valign="bottom"> 
     <tr> 
      <th align="left" charoff="50" valign="top">&nbsp;</th> 
      <th align="center" char="(" charoff="50" colspan="2" valign="top"> 
       <bold>
         LKB1 mutation 
       </bold> 
       <hr /> </th> 
      <th align="left" charoff="50" valign="top">&nbsp;</th> 
     </tr> 
     <tr> 
      <th align="left" charoff="50" valign="top">&nbsp;</th> 
      <th align="char" char="(" charoff="50" valign="top">+</th> 
      <th align="char" char="(" charoff="50" valign="top">?</th> 
      <th align="left" charoff="50" valign="top"> 
       <bold> 
        <italic>
          P 
        </italic>-value* 
       </bold> </th> 
     </tr> 
    </thead> 
    <tbody valign="top"> 
     <tr> 
      <td align="left" charoff="50" valign="top">All tumours</td> 
      <td align="char" char="(" charoff="50" valign="top">34 (11%)</td> 
      <td align="char" char="(" charoff="50" valign="top">276 (89%)</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">Age, median</td> 
      <td align="char" char="(" charoff="50" valign="top">61.2</td> 
      <td align="char" char="(" charoff="50" valign="top">62.2</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="4" valign="top"> 
       <italic>
         Ethnicity 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">?Caucasian cohort</td> 
      <td align="char" char="(" charoff="50" valign="top">25 (17%)</td> 
      <td align="char" char="(" charoff="50" valign="top">118 (83%)</td> 
      <td align="left" charoff="50" valign="top">0.001</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">?Asian cohort</td> 
      <td align="char" char="(" charoff="50" valign="top">9 (5%)</td> 
      <td align="char" char="(" charoff="50" valign="top">158 (95%)</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">Gender</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">?Male</td> 
      <td align="char" char="(" charoff="50" valign="top">20 (11%)</td> 
      <td align="char" char="(" charoff="50" valign="top">167 (89%)</td> 
      <td align="left" charoff="50" valign="top">NS</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">?Female</td> 
      <td align="char" char="(" charoff="50" valign="top">14 (12%)</td> 
      <td align="char" char="(" charoff="50" valign="top">107 (88%)</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="4" valign="top"> 
       <italic>
         Smoking 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">?Never (&lt;10?py)</td> 
      <td align="char" char="(" charoff="50" valign="top">2 (3%)</td> 
      <td align="char" char="(" charoff="50" valign="top">70 (97%)</td> 
      <td align="left" charoff="50" valign="top">0.007</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">?Smoker (&gt;10?py)</td> 
      <td align="char" char="(" charoff="50" valign="top">26 (14%)</td> 
      <td align="char" char="(" charoff="50" valign="top">161 (86%)</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="4" valign="top"> 
       <italic>
         Tumour stage 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">?I</td> 
      <td align="char" char="(" charoff="50" valign="top">19 (10%)</td> 
      <td align="char" char="(" charoff="50" valign="top">169 (90%)</td> 
      <td align="left" charoff="50" valign="top">NS</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">?II</td> 
      <td align="char" char="(" charoff="50" valign="top">8 (14%)</td> 
      <td align="char" char="(" charoff="50" valign="top">51 (86%)</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">?III</td> 
      <td align="char" char="(" charoff="50" valign="top">5 (11%)</td> 
      <td align="char" char="(" charoff="50" valign="top">42 (89%)</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">?IV</td> 
      <td align="char" char="(" charoff="50" valign="top">1 (12%)</td> 
      <td align="char" char="(" charoff="50" valign="top">7 (88%)</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="4" valign="top"> 
       <italic>
         Histology 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">?Adenocarcinoma</td> 
      <td align="char" char="(" charoff="50" valign="top">27 (13%)</td> 
      <td align="char" char="(" charoff="50" valign="top">180 (87%)</td> 
      <td align="left" charoff="50" valign="top">0.047</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">?Squamous carcinoma</td> 
      <td align="char" char="(" charoff="50" valign="top">5 (5%)</td> 
      <td align="char" char="(" charoff="50" valign="top">87 (95%)</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">?Adenosquamous</td> 
      <td align="char" char="(" charoff="50" valign="top">2 (22%)</td> 
      <td align="char" char="(" charoff="50" valign="top">7 (78%)</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
    </tbody> 
   </table> 
   <table-wrap-foot> 
    <fn id="t1-fn1"> 
     <label></label> 
     <p> <sup>*</sup>Fisher's exact test, NS=not statistically significant ( 
      <italic>
        P 
      </italic>&gt;0.05).</p> 
    </fn> 
   </table-wrap-foot> 
  </table-wrap> 
  <table-wrap id="tbl2" position="float"> 
   <label>Table 2</label> 
   <p content-type="table-h3">The specific 
    <italic>
      LKB1 
    </italic> mutations in NSCLC tumours</p> 
   <table border="1" frame="hsides" rules="groups" width="85%"> 
    <colgroup> 
     <col align="left" /> 
     <col align="char" char="(" /> 
     <col align="center" /> 
     <col align="center" /> 
     <col align="center" /> 
     <col align="center" /> 
    </colgroup> 
    <thead valign="bottom"> 
     <tr> 
      <th align="left" charoff="50" valign="top"> 
       <bold>
         Mutation type 
       </bold> </th> 
      <th align="char" char="(" charoff="50" valign="top"> 
       <bold>
         No. (%) 
       </bold> </th> 
      <th align="center" charoff="50" valign="top"> 
       <bold>
         Mutation 
       </bold> </th> 
      <th align="center" charoff="50" valign="top"> 
       <bold>
         Amino acid change 
       </bold> </th> 
      <th align="center" charoff="50" valign="top"> 
       <bold>
         Exon 
       </bold> </th> 
      <th align="center" charoff="50" valign="top"> 
       <bold>
         Histology 
       </bold> </th> 
     </tr> 
    </thead> 
    <tbody valign="top"> 
     <tr> 
      <td align="left" charoff="50" valign="top">Missense</td> 
      <td align="char" char="(" charoff="50" valign="top">7 (21)</td> 
      <td align="center" charoff="50" valign="top"><sup>a</sup>526G&gt;T</td> 
      <td align="center" charoff="50" valign="top">D176Y</td> 
      <td align="center" charoff="50" valign="top">4</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top"><sup>a</sup>580G&gt;T</td> 
      <td align="center" charoff="50" valign="top">D194Y</td> 
      <td align="center" charoff="50" valign="top">4</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">580G&gt;T</td> 
      <td align="center" charoff="50" valign="top">D194Y</td> 
      <td align="center" charoff="50" valign="top">4</td> 
      <td align="center" charoff="50" valign="top">Sq</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">829G&gt;T</td> 
      <td align="center" charoff="50" valign="top">D277Y</td> 
      <td align="center" charoff="50" valign="top">6</td> 
      <td align="center" charoff="50" valign="top">AdSq</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top"><sup>a</sup>842C&gt;T</td> 
      <td align="center" charoff="50" valign="top">P281L</td> 
      <td align="center" charoff="50" valign="top">6</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top"><sup>a</sup>842C&gt;T</td> 
      <td align="center" charoff="50" valign="top">P281L</td> 
      <td align="center" charoff="50" valign="top">6</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">1276C&gt;T</td> 
      <td align="center" charoff="50" valign="top">R426W</td> 
      <td align="center" charoff="50" valign="top">9</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">Nonsense</td> 
      <td align="char" char="(" charoff="50" valign="top">2 (6)</td> 
      <td align="center" charoff="50" valign="top">206C&gt;A</td> 
      <td align="center" charoff="50" valign="top">S69X</td> 
      <td align="center" charoff="50" valign="top">1</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">475C&gt;T</td> 
      <td align="center" charoff="50" valign="top">Q159X</td> 
      <td align="center" charoff="50" valign="top">4</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">Deletion/ insertion</td> 
      <td align="char" char="(" charoff="50" valign="top">25 (74)</td> 
      <td align="center" charoff="50" valign="top"><sup>a</sup>75_76del2&amp;insT</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">1</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">120_130del11</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">1</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">125_127insGG</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">1</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">128_129delC</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">1</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">142_143delA</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">1</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">180delC</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">1</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">209delA</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">1</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">227_228delC</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">1</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">47_651del604</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">1-5</td> 
      <td align="center" charoff="50" valign="top">Sq</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">153_536del384</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">1-4</td> 
      <td align="center" charoff="50" valign="top">AdSq</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top"><sup>a</sup>exon 2-3del</td> 
      <td align="center" charoff="50" valign="top">Truncates</td> 
      <td align="center" charoff="50" valign="top">2-3</td> 
      <td align="center" charoff="50" valign="top">Sq</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top"><sup>a</sup>exon 2-3del</td> 
      <td align="center" charoff="50" valign="top">Truncates</td> 
      <td align="center" charoff="50" valign="top">2-3</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">exon 2-3del</td> 
      <td align="center" charoff="50" valign="top">Truncates</td> 
      <td align="center" charoff="50" valign="top">2-3</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">exon 2-4del</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">2-4</td> 
      <td align="center" charoff="50" valign="top">Sq</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">464_465del2insTTTGCT</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">3-4</td> 
      <td align="center" charoff="50" valign="top">Sq</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">562_563delG</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">4</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top"><sup>a</sup>exon 4del</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">4</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">exon 4del</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">4</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">exon 4del</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">4</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">exon 4del</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">4</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">610_623del14</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">5</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top"><sup>a</sup>837_844delC</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">6</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">837_844insC</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">6</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">exon 6del</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">6</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">1038_1040insG</td> 
      <td align="center" charoff="50" valign="top">FS, truncates</td> 
      <td align="center" charoff="50" valign="top">8</td> 
      <td align="center" charoff="50" valign="top">Ad</td> 
     </tr> 
    </tbody> 
   </table> 
   <table-wrap-foot> 
    <fn id="t2-fn1"> 
     <label></label> 
     <p>Ad=Adenocarcinoma; AdSq=Adenosquamous carcinoma; Sq=Squamous cell carcinoma; .</p> 
    </fn> 
    <fn id="t2-fn2"> 
     <label>a</label> 
     <p>These mutations were detected in Korean NSCLC patients.</p> 
    </fn> 
   </table-wrap-foot> 
  </table-wrap> 
  <table-wrap id="tbl3" position="float"> 
   <label>Table 3</label> 
   <p content-type="table-h3">Association of 
    <italic>
      LKB1 
    </italic> mutations with 
    <italic>
      KRAS 
    </italic>, 
    <italic>
      BRAF, 
    </italic> and 
    <italic>
      EGFR 
    </italic> mutations in NSCLC tumours</p> 
   <table border="1" frame="hsides" rules="groups" width="85%"> 
    <colgroup> 
     <col align="left" /> 
     <col align="center" /> 
     <col align="char" char="." /> 
     <col align="char" char="." /> 
     <col align="char" char="." /> 
    </colgroup> 
    <thead valign="bottom"> 
     <tr> 
      <th align="left" charoff="50" valign="top">&nbsp;</th> 
      <th align="center" charoff="50" valign="top">&nbsp;</th> 
      <th align="center" char="." charoff="50" colspan="2" valign="top"> 
       <bold> 
        <italic>
          LKB1 
        </italic> mutation 
       </bold> 
       <hr /> </th> 
      <th align="char" char="." charoff="50" valign="top">&nbsp;</th> 
     </tr> 
     <tr> 
      <th align="left" charoff="50" valign="top">&nbsp;</th> 
      <th align="center" charoff="50" valign="top">&nbsp;</th> 
      <th align="char" char="." charoff="50" valign="top">+</th> 
      <th align="char" char="." charoff="50" valign="top">?</th> 
      <th align="char" char="." charoff="50" valign="top"> 
       <bold> 
        <italic>
          P 
        </italic>-value* 
       </bold> </th> 
     </tr> 
    </thead> 
    <tbody valign="top"> 
     <tr> 
      <td align="left" charoff="50" valign="top"> 
       <italic>
         EGFR 
       </italic> mutation</td> 
      <td align="center" charoff="50" valign="top">+</td> 
      <td align="char" char="." charoff="50" valign="top">1</td> 
      <td align="char" char="." charoff="50" valign="top">69</td> 
      <td align="char" char="." charoff="50" valign="top">0.002</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">?</td> 
      <td align="char" char="." charoff="50" valign="top">33</td> 
      <td align="char" char="." charoff="50" valign="top">207</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top"> 
       <italic>
         K-Ras 
       </italic> mutation</td> 
      <td align="center" charoff="50" valign="top">+</td> 
      <td align="char" char="." charoff="50" valign="top">10</td> 
      <td align="char" char="." charoff="50" valign="top">39</td> 
      <td align="char" char="." charoff="50" valign="top">0.042</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">?</td> 
      <td align="char" char="." charoff="50" valign="top">24</td> 
      <td align="char" char="." charoff="50" valign="top">237</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">B-Raf 
       <italic>
         mutation 
       </italic> </td> 
      <td align="center" charoff="50" valign="top">+</td> 
      <td align="char" char="." charoff="50" valign="top">1</td> 
      <td align="char" char="." charoff="50" valign="top">3</td> 
      <td align="char" char="." charoff="50" valign="top">0.373</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="center" charoff="50" valign="top">?</td> 
      <td align="char" char="." charoff="50" valign="top">33</td> 
      <td align="char" char="." charoff="50" valign="top">273</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
     </tr> 
    </tbody> 
   </table> 
   <table-wrap-foot> 
    <fn id="t3-fn1"> 
     <label></label> 
     <p> <sup>*</sup>Fisher's exact test.</p> 
    </fn> 
   </table-wrap-foot> 
  </table-wrap> 
  <table-wrap id="tbl4" position="float"> 
   <label>Table 4</label> 
   <p content-type="table-h3">LKB1 genotypes of NSCLC cell lines with EGFR or ERBB2 mutations</p> 
   <table border="1" frame="hsides" rules="groups" width="85%"> 
    <colgroup> 
     <col align="left" /> 
     <col align="center" /> 
     <col align="center" /> 
     <col align="center" /> 
    </colgroup> 
    <thead valign="bottom"> 
     <tr> 
      <th align="left" charoff="50" valign="top"> 
       <bold>
         Cell line 
       </bold> </th> 
      <th align="center" charoff="50" valign="top"> 
       <bold> 
        <italic>
          EGFR 
        </italic> genotype 
       </bold> </th> 
      <th align="center" charoff="50" valign="top"> 
       <bold> 
        <italic>
          HER2 
        </italic> genotype 
       </bold> </th> 
      <th align="center" charoff="50" valign="top"> 
       <bold> 
        <italic>
          LKB1 
        </italic> genotype 
       </bold> </th> 
     </tr> 
    </thead> 
    <tbody valign="top"> 
     <tr> 
      <td align="left" charoff="50" valign="top">H1650</td> 
      <td align="center" charoff="50" valign="top">E746_A750del</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">H1781</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
      <td align="center" charoff="50" valign="top">G776V, Cins</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">H1975</td> 
      <td align="center" charoff="50" valign="top">L858R, T790M</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">H3255</td> 
      <td align="center" charoff="50" valign="top">L858R</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">H3255GR</td> 
      <td align="center" charoff="50" valign="top">L858R, T790M</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">H820</td> 
      <td align="center" charoff="50" valign="top">L747_L751del, T790M</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">HCC2279</td> 
      <td align="center" charoff="50" valign="top">E746_A750del</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">HCC2935</td> 
      <td align="center" charoff="50" valign="top">E746_T751del, S752I</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">HCC4006</td> 
      <td align="center" charoff="50" valign="top">L747_E749del, A750P</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">HCC827</td> 
      <td align="center" charoff="50" valign="top">E746_A750del</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">Ma-1</td> 
      <td align="center" charoff="50" valign="top">E746_A750del</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">Ma-70</td> 
      <td align="center" charoff="50" valign="top">L858R</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">PC-9</td> 
      <td align="center" charoff="50" valign="top">E746_A750del</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
      <td align="center" charoff="50" valign="top">Wt</td> 
     </tr> 
    </tbody> 
   </table> 
   <table-wrap-foot> 
    <fn id="t4-fn1"> 
     <label></label> 
     <p>Wt=wild type.</p> 
    </fn> 
   </table-wrap-foot> 
  </table-wrap> 
 </floats-wrap> 
</article>