<article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"> 
 <!--?properties open_acces=""?--> 
 <front> 
  <journal-meta> 
   <journal-id journal-id-type="nlm-ta">
     PLoS One 
   </journal-id> 
   <journal-id journal-id-type="publisher-id">
     plos 
   </journal-id> 
   <journal-id journal-id-type="pmc">
     plosone 
   </journal-id> 
   <journal-h3-group> 
    <journal-h3>
      PLoS ONE 
    </journal-h3> 
   </journal-h3-group> 
   <issn pub-type="epub">
     1932-6203 
   </issn> 
   <publisher> 
    <publisher-name>
      Public Library of Science 
    </publisher-name> 
    <publisher-loc>
      San Francisco, USA 
    </publisher-loc> 
   </publisher> 
  </journal-meta> 
  <article-meta> 
   <article-id pub-id-type="pmid">
     21297950 
   </article-id> 
   <article-id pub-id-type="pmc">
     3031508 
   </article-id> 
   <article-id pub-id-type="publisher-id">
     10-PONE-RA-20649R1 
   </article-id> 
   <article-id pub-id-type="doi">
     10.1371/journal.pone.0014635 
   </article-id> 
   <article-categories> 
    <subj-group subj-group-type="heading"> 
     <subject>
       Research Article 
     </subject> 
    </subj-group> 
    <subj-group subj-group-type="Discipline"> 
     <subject>
       Cell Biology 
     </subject> 
     <subject>
       Cell Biology/Gene Expression 
     </subject> 
     <subject>
       Oncology/Lung Cancer 
     </subject> 
    </subj-group> 
   </article-categories> 
   <h3-group> 
    <article-h3>
      Gene Expression Meta-Analysis Identifies VDAC1 as a Predictor of Poor Outcome in Early Stage Non-Small Cell Lung Cancer 
    </article-h3> 
    <alt-h3 alt-h3-type="running-head">
      VDAC1 Predicts NSCLC Survival 
    </alt-h3> 
   </h3-group> 
   <contrib-group> 
    <contrib contrib-type="author" equal-contrib="yes"> 
     <name> 
      <surname>
        Grills 
      </surname> 
      <given-names>
        Claire 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1"> 
      <sup>1</sup> 
     </xref> 
     <xref ref-type="corresp" rid="cor1"> 
      <sup>*</sup> 
     </xref> 
    </contrib> 
    <contrib contrib-type="author" equal-contrib="yes"> 
     <name> 
      <surname>
        Jithesh 
      </surname> 
      <given-names>
        Puthen V. 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1"> 
      <sup>1</sup> 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Blayney 
      </surname> 
      <given-names>
        Jaine 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1"> 
      <sup>1</sup> 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Zhang 
      </surname> 
      <given-names>
        Shu-Dong 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1"> 
      <sup>1</sup> 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Fennell 
      </surname> 
      <given-names>
        Dean A. 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1"> 
      <sup>1</sup> 
     </xref> 
     <xref ref-type="aff" rid="aff2"> 
      <sup>2</sup> 
     </xref> 
    </contrib> 
   </contrib-group> 
   <aff id="aff1"> 
    <label>1</label> 
    <addr-line>
      Centre for Biomedical Informatics, Queen's University Belfast, Belfast, Northern Ireland, United Kingdom 
    </addr-line> 
   </aff> 
   <aff id="aff2"> 
    <label>2</label> 
    <addr-line>
      Centre for Cancer Research and Cell Biology, Queen's University Belfast, Belfast, Northern Ireland, United Kingdom 
    </addr-line> 
   </aff> 
   <contrib-group> 
    <contrib contrib-type="editor"> 
     <name> 
      <surname>
        Morty 
      </surname> 
      <given-names>
        Rory Edward 
      </given-names> 
     </name> 
     <role>
       Editor 
     </role> 
     <xref ref-type="aff" rid="edit1"></xref> 
    </contrib> 
   </contrib-group> 
   <aff id="edit1">
     University of Giessen Lung Center, Germany 
   </aff> 
   <author-notes> 
    <corresp id="cor1">
      * E-mail: 
     <email>
       c.grills@qub.ac.uk 
     </email> 
    </corresp> 
    <fn fn-type="con"> 
     <p>Conceived and designed the experiments: DAF. Analyzed the data: CG PJ JKB SDZ. Contributed reagents/materials/analysis tools: CG PJ JKB. Wrote the paper: CG PJ DAF.</p> 
    </fn> 
   </author-notes> 
   <pub-date pub-type="collection"> 
    <year>
      2011 
    </year> 
   </pub-date> 
   <pub-date pub-type="epub"> 
    <day>
      31 
    </day> 
    <month>
      1 
    </month> 
    <year>
      2011 
    </year> 
   </pub-date> 
   <volume>
     6 
   </volume> 
   <issue>
     1 
   </issue> 
   <elocation-id>
     e14635 
   </elocation-id> 
   <history> 
    <date date-type="received"> 
     <day>
       5 
     </day> 
     <month>
       7 
     </month> 
     <year>
       2010 
     </year> 
    </date> 
    <date date-type="accepted"> 
     <day>
       23 
     </day> 
     <month>
       11 
     </month> 
     <year>
       2010 
     </year> 
    </date> 
   </history> 
   <permissions> 
    <copyright-statement>
      Grills et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited. 
    </copyright-statement> 
   </permissions> 
   <abstract> 
    <h3>Abstract</h3> 
    <sec> 
     <h3>Background</h3> 
     <p><span style="background-color:#fffefe">The bioenergetic status of non-small cell lung cancer correlates with tumour aggressiveness</span>. <span style="background-color:#ffcfcf">The voltage dependent anion channel type 1 (VDAC1) is a component of the mitochondrial permeability transition pore, regulates mitochondrial ATP/ADP exchange suggesting that its over-expression could be associated with energy dependent processes including increased proliferation and invasiveness</span>. <span style="background-color:#fff9f9">To test this hypothesis, we conducted an in vivo gene-expression meta-analysis of surgically resected non-small cell lung cancer (NSCLC) using 602 individual expression profiles, to examine the impact of VDAC1 on survival</span>.</p> 
    </sec> 
    <sec> 
     <h3>Methodology/Principal Findings</h3> 
     <p><span style="background-color:#fffcfc">High VDAC1 expression was associated with shorter overall survival with hazard ratio (HR)?=?0.6639 (95% confidence interval (CI) 0.4528 to 0.9721), p?=?0.035352 corresponding to 52 versus 101 months</span>. <span style="background-color:#fff5f5">VDAC1 predicted shorter time to recurrence and was shown to be an independent prognostic factor compared with histology, gender, age, nodal stage and tumour stage in a Cox multivariate analysis</span>. <span style="background-color:#ff6868">Supervised analysis of all the datasets identified a 6-gene signature comprising HNRNPC, HSPA4, HSPA9, UBE2D2, CSNK1A1 and G3BP1 with overlapping functions involving regulation of protein turnover, RAS-RAF-MEK pathway and transcription</span>. <span style="background-color:#fffcfc">VDAC1 predicted survival in breast cancer and myeloma and an unsupervised analysis revealed enrichment of the VDAC1 signature in specific subsets</span>.</p> 
    </sec> 
    <sec> 
     <h3>Conclusions</h3> 
     <p><span style="background-color:#ffe6e6">In summary, gene expression analysis identifies VDAC1 gene expression as a predictor of poor outcome in NSCLC and other cancers and is associated with dysregulation of a conserved set of biological pathways, which may be causally associated with aggressive tumour behaviour</span>.</p> 
    </sec> 
   </abstract> 
   <counts> 
    <page-count count="8"></page-count> 
   </counts> 
  </article-meta> 
 </front> 
 <sec id="s1"> 
  <h3>Introduction</h3> 
  <p><span style="background-color:#ffcece">Non-small cell lung cancer (NSCLC) is a common, largely incurable cancer</span>. <span style="background-color:#ffe9e9">In the early disease setting, surgical resection is the standard of care and the benefit of adjuvant chemotherapy is limited to 5?15% improvement in survival in stage II but not stage I disease [1] </span>. <span style="background-color:#ffc9c9">However relapse is frequent in stage I cancer with 37% of patients dying within five years, therefore there is a need to identify reliable biomarkers of poor outcome in order to target individuals for whom adjuvant or novel targeted therapy may improve survival</span>. <span style="background-color:#ffdada">Alteration in tumour metabolism is a hallmark of cancer and high levels of hexokinase-dependent glucose trapping in NSCLC evidenced by standardized uptake values on positron emission tomography, is associated with shorter survival [2] </span>. <span style="background-color:#ffeeee">This is seen in both early and advanced NSCLC [3] , [4] </span></p> 
  <p><span style="background-color:#ffbebe">The Voltage Dependent Anion Channel 1 (VDAC1) is an outer mitochondrial membrane protein involved in the regulation of ATP/ADP exchange and respiratory control [5] </span>. <span style="background-color:#fff7f7">VDAC1 interacts with proapoptotic BCL-2 family proteins [6] , [7] </span>. <span style="background-color:#ffd0d0">However, its role in regulating mitochondrial outer membrane depolarization has been controversial [8] , [9] , [10] </span>. <span style="background-color:#ffe2e2">Yet, emerging data has identified a potential role for this protein in the regulation of cell survival and for this reason we found it worthy of further study</span>. <span style="background-color:#ffe9e9">The permeability transition pore, which comprises VDAC1, is directly regulated by the BH3-only proapoptotic BCL-2 family member BAD [10] and regulates cell survival</span>. <span style="background-color:#fff4f4">In contrast, the type II isoform (VDAC2) has been identified as an antiapoptotic, negative regulator of BAK [9] , [11] , [12] </span>. <span style="background-color:#ffbfbf">The adenine-nucleotide translocator (ANT) which is located in the inner mitochondrial is also considered to be a major component of the permeability transition pore [13] and along with VDAC it is believed to control mitochondrial remodeling processes either directly or indirectly [14] </span>. <span style="background-color:#fff6f6">Recently, the bacterial protein FilA [15] has been shown to regulate a complex of VDAC1 with hexokinase II [16] which inhibits apoptosis</span>. <span style="background-color:#ffdcdc">A growing body of data suggests that hexokinase-VDAC interaction is anti-apoptotic [17] , and disruption of this interaction could be a strategy for inducing cell death [18] , [19] </span>. <span style="background-color:#ffbebe">VDAC-hexokinase complex has been reported to be regulated by glycogen synthase kinase 3 beta [19] and AKT [20] </span></p> 
  <p><span style="background-color:#ffc1c1">Cancer cells with low VDAC1 induced by RNA interference exhibit a defect in cell growth in vivo [21] </span>. <span style="background-color:#ffd1d1">Conversely, high levels of VDAC1 expression levels may confer tumour cell selection advantage by facilitating energy dependent processes such as proliferation and invasiveness [22] </span>. <span style="background-color:#ffdede">Although hexokinase II expression has been linked to worse prognosis [23] , to date there have been no studies exploring the potential prognostic impact of VDAC1 in patients with cancer</span>. <span style="background-color:#fff2f2">It was therefore hypothesized that VDAC1 expression could impact survival following NSCLC resection</span>. <span style="background-color:#ffd9d9">Here, we show using gene expression meta-analysis that VDAC1 overexpression is a strong, independent, poor prognostic factor in early stage non-small cell lung cancer based on combined analysis of multiple gene expression datasets</span>. <span style="background-color:#ff9797">VDAC1 overexpression is consistently associated with overexpression of 6 additional genes involved in survival signalling, protein ubiquitination and ATP binding</span>. <span style="background-color:#ffdcdc">This association is maintained across multiple independent datasets and across different cancers suggesting a potential role for these biological processes in regulating tumour behaviour and therefore clinical outcome in NSCLC</span>.</p> 
 </sec> 
 <sec id="s2"> 
  <h3>Results</h3> 
  <sec id="s2a"> 
   <h3>VDAC1 overexpression predicts shorter survival in NSCLC</h3> 
   <p><span style="background-color:#ffdfdf">In total, 8 independent NSCLC datasets were curated from Gene Expression Omnibus (GEO), with a total of 602 microarrays</span>. <span style="background-color:#ffecec">In all datasets high expression of VDAC1 (upper tertile) was associated with significantly shorter survival compared to that of the lower tertile for VDAC1 expression, p?=?0.0353, HR?=?0.6639 (0.4528 to 0.9721) ( Figure 1A )</span>. <span style="background-color:#fffdfd">The average overall survival was 101.6 months in patients with the lowest VDAC1 tertile and 52 months in patients with highest VDAC1 tertile</span>.</p> 
   <fig id="pone-0014635-g001" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0014635.g001 
    </object-id> 
    <label>Figure 1</label> 
    <h3>Survival curves for all data and for Stage I patients based on VDAC1 expression.</h3> 
    <p><span style="background-color:#ffffff">Figure 1A: Displays the Kaplan Meier plot depicting patient survival dependent on high and low VDAC1 expression</span>. <span style="background-color:#ffbdbd">The distinct difference in survival is visibly evident with the median survival of high VDAC1 expressing patients being only half that of low expressing patients</span>. <span style="background-color:#ffdfdf">The curves are significantly different with p?=?0.0353 by the logrank test</span>. <span style="background-color:#ffffff">Figure 1B: The survival plot for stage I patients</span>. <span style="background-color:#fff7f7">Again the difference in high and low expressing samples is clear with low VDAC1 patients having better survival rates</span>. <span style="background-color:#ffeeee">The logrank test proves the curves are significantly different with p?=?0.0455</span>. <span style="background-color:#ffdcdc">Stage I samples follow the same pattern as all the combined data from all stages</span>.</p> 
    <graphic xlink:href="pone.0014635.g001"></graphic> 
   </fig> 
  </sec> 
  <sec id="s2b"> 
   <h3>VDAC1 expression is an independent prognostic factor</h3> 
   <p><span style="background-color:#fffafa">To further explore the impact of VDAC1 on survival, we carried out correlation and Cox regression analysis as well as multivariate tests using PASW Statistics v.18 (SPSS: An IBM company)</span>. <span style="background-color:#fffcfc">High VDAC1 expression was shown to be an independent prognostic variable for shorter survival p&lt;0.001 level</span>. <span style="background-color:#ffdddd">Stage (p?=?0.013) and pT, (primary tumour), (p?=?0.022) also proved to be independent variables correlated with survival, while multivariate analysis showed that the combination of stage and histology significantly influences patient survival (p?=?0.027)</span>. <span style="background-color:#ff9797">The results from the Cox regression analysis can be seen in Table 1 , where VDAC1 has the most significant influence on survival followed by pT and stage</span></p> 
   <table-wrap id="pone-0014635-t001" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0014635.t001 
    </object-id> 
    <label>Table 1</label> 
    <h3>Cox Regression analysis based on all patient data and VDAC1 expression.</h3> 
    <alternatives> 
     <graphic id="pone-0014635-t001-1" xlink:href="pone.0014635.t001"></graphic> 
     <table frame="hsides" rules="groups"> 
      <colgroup span="1"> 
       <col align="left" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
      </colgroup> 
      <thead> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1">Regression Coefficient</td> 
        <td align="left" colspan="1" rowspan="1">Standard Error (SE)</td> 
        <td align="left" colspan="1" rowspan="1">p-value</td> 
        <td align="left" colspan="1" rowspan="1">Hazard Ratio</td> 
       </tr> 
      </thead> 
      <tbody> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           VDAC1 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.239</td> 
        <td align="left" colspan="1" rowspan="1">0.78</td> 
        <td align="left" colspan="1" rowspan="1">0.002</td> 
        <td align="left" colspan="1" rowspan="1">1.27 (1.09?1.48)</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           Tumour Type 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.044</td> 
        <td align="left" colspan="1" rowspan="1">0.137</td> 
        <td align="left" colspan="1" rowspan="1">0.75</td> 
        <td align="left" colspan="1" rowspan="1">1.045 (0.80?1.37)</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           Sex 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">?0.078</td> 
        <td align="left" colspan="1" rowspan="1">0.238</td> 
        <td align="left" colspan="1" rowspan="1">0.745</td> 
        <td align="left" colspan="1" rowspan="1">0.925 (0.58?1.48)</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           Age 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">?0.02</td> 
        <td align="left" colspan="1" rowspan="1">0.011</td> 
        <td align="left" colspan="1" rowspan="1">0.834</td> 
        <td align="left" colspan="1" rowspan="1">0.998 (0.98?1.02)</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           Stage 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.515</td> 
        <td align="left" colspan="1" rowspan="1">0.249</td> 
        <td align="left" colspan="1" rowspan="1">0.038</td> 
        <td align="left" colspan="1" rowspan="1">1.67 (1.03?2.73)</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           pT 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">1.623</td> 
        <td align="left" colspan="1" rowspan="1">0.695</td> 
        <td align="left" colspan="1" rowspan="1">0.019</td> 
        <td align="left" colspan="1" rowspan="1">5.07 (1.30?19.78)</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           pN 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">1.977</td> 
        <td align="left" colspan="1" rowspan="1">1.058</td> 
        <td align="left" colspan="1" rowspan="1">0.062</td> 
        <td align="left" colspan="1" rowspan="1">7.22 (0.91?57.45)</td> 
       </tr> 
      </tbody> 
     </table> 
    </alternatives> 
    <table-wrap-foot> 
     <fn id="nt101"> 
      <label></label> 
      <p><span style="background-color:#ffbebe">The influence VDAC1 and stage have on survival is evident from this figure with high VDAC1 expression inducing poor survival</span>. <span style="background-color:#ffc5c5">VDAC1 is the most significant predictor of survival time with stage and pT also displaying predictive power</span>. <span style="background-color:#fff9f9">The age of the patient, their gender or whether the tumour was squamous or adenocarcinoma didn't have a large effect on overall survival time</span>.</p> 
     </fn> 
    </table-wrap-foot> 
   </table-wrap> 
  </sec> 
  <sec id="s2c"> 
   <h3>VDAC1 overexpression predicts shorter time to recurrence following NSCLC resection</h3> 
   <p><span style="background-color:#ffe5e5">The influence of VDAC1 expression on time to relapse following surgical resection was of interest as this clinical variable is not confounded by therapy after relapse</span>. <span style="background-color:#fff3f3">It is therefore a more accurate measure of prognostic significance</span>. <span style="background-color:#fffbfb">So we analysed only the recurrence data and univariate analysis showed VDAC1 to significantly predict time to recurrence (p&lt;0.001)</span>. <span style="background-color:#ffecec">Information on histological subtype, gender and age was available for all samples and they were found to be not significant with p values of 0.35, 0.82 and 0.73 respectively</span>. <span style="background-color:#ffaeae">A summary of this analysis can be found in Table 2 </span>. <span style="background-color:#ffe9e9">Similarly multivariate analysis ranked these variables in the same order, with VDAC1 being the only independent factor (p&lt;0.001)</span></p> 
   <table-wrap id="pone-0014635-t002" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0014635.t002 
    </object-id> 
    <label>Table 2</label> 
    <h3>Univariate Analysis of Time to Recurrence Data.</h3> 
    <alternatives> 
     <graphic id="pone-0014635-t002-2" xlink:href="pone.0014635.t002"></graphic> 
     <table frame="hsides" rules="groups"> 
      <colgroup span="1"> 
       <col align="left" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
      </colgroup> 
      <thead> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1">Type III Sum of Squares</td> 
        <td align="left" colspan="1" rowspan="1">F</td> 
        <td align="left" colspan="1" rowspan="1">Sig.</td> 
       </tr> 
      </thead> 
      <tbody> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">VDAC1</td> 
        <td align="left" colspan="1" rowspan="1">5103.97</td> 
        <td align="left" colspan="1" rowspan="1">14.55</td> 
        <td align="left" colspan="1" rowspan="1">0.000</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Tumour Type</td> 
        <td align="left" colspan="1" rowspan="1">308.37</td> 
        <td align="left" colspan="1" rowspan="1">0.88</td> 
        <td align="left" colspan="1" rowspan="1">0.35</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Sex</td> 
        <td align="left" colspan="1" rowspan="1">18.17</td> 
        <td align="left" colspan="1" rowspan="1">0.52</td> 
        <td align="left" colspan="1" rowspan="1">0.82</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Age</td> 
        <td align="left" colspan="1" rowspan="1">41.47</td> 
        <td align="left" colspan="1" rowspan="1">0.118</td> 
        <td align="left" colspan="1" rowspan="1">0.73</td> 
       </tr> 
      </tbody> 
     </table> 
    </alternatives> 
    <table-wrap-foot> 
     <fn id="nt102"> 
      <label></label> 
      <p><span style="background-color:#ffd9d9">When dealing with time to recurrence VDAC1 expression was again shown to be the most significant variable at predicting time to recurrence</span>. <span style="background-color:#fff0f0">We found no significant relationship between recurrence and tumour type, gender or age</span>.</p> 
     </fn> 
    </table-wrap-foot> 
   </table-wrap> 
  </sec> 
  <sec id="s2d"> 
   <h3>VDAC1 overexpression in Stage I samples leads to poor survival</h3> 
   <p><span style="background-color:#ffffff">Information on stage was not available with all the datasets but we analysed what we had access to</span>. <span style="background-color:#fff6f6">We had 89 Stage I samples and the Kaplan-Meier plot for these patients is shown in Figure 1B ; p?=?0.0455, with an associated hazard ratio of 0.52 (0.1492?0.7653)</span>. <span style="background-color:#fffbfb">We found the correlation between VDAC1 expression and survival to be very strong with p&lt;0.001</span>. <span style="background-color:#ffd6d6">Cox regression analysis in stage 1 NSCLC, where adjuvant chemotherapy has not been shown to improve patient outcome [24] , [25] , [26] , revealed no variables to be statistically significant but, similarly to the analysis of all the data, VDAC1 was the most influential predictor of survival (p?=?0.241), and was ranked above histology (p?=?0.344), age (p?=?0.465), pT (p?=?0.970), pN, (nodal stage), (p?=?0.975) and gender (p?=?0.999)</span>. <span style="background-color:#ffe7e7">The regression coefficient for VDAC1 was 29.18 indicating a very poor prognosis for patients with high levels of VDAC1.The sample numbers were low for stages II and III with 68 samples being Stage II and only 16 samples being Stage III, this made it difficult to obtain statistically significant results</span>. <span style="background-color:#fffdfd">The correlation between VDAC1 expression and survival in stage II samples and Stage III samples was insignificant with p?=?0.853 and p?=?0.701 respectively</span>. <span style="background-color:#fffafa">We had an inadequate number of events in both stages to produce an accurate Kaplan Meier plot with less than 10 incidences of death per category</span>. <span style="background-color:#ffebeb">The lack of higher stage samples led us to focus on the output from Stage I samples and from the combined data</span>.</p> 
  </sec> 
  <sec id="s2e"> 
   <h3>VDAC1 overexpression correlates with a conserved 6-gene signature</h3> 
   <p><span style="background-color:#ffd8d8">In order to gain an insight into the biology underlying the poor prognosis of VDAC1 overexpressing NSCLC in vivo, an analysis was conducted to identify which genes and gene networks are consistently associated with high VDAC1 expression</span>. <span style="background-color:#ffe5e5">A subset of 6 genes was identified in VDAC1 overexpressing NSCLC datasets associated with significant differential expression in 50% or more of the datasets</span>. <span style="background-color:#ff5b5b">These genes were CSNK1A1, G3BP1, HNRNPC, HSPA4, HSPA9 and UBE2D2; see Figure 2 where the difference in the average expression of these genes when VDAC1 is low and high is shown</span>. <span style="background-color:#ff0000">As defined by the Gene Ontology Consortium [27] HNRNPC, HSPA4, HSPA9 and UBE2D2 are involved in protein ubiquitination pathways, and CSNK1A1 regulates EIF2 function</span>. <span style="background-color:#ff5353">HSPA9 is also involved in anti-apoptosis pathways while G3BP1 is linked to Ras protein signal transduction and ATP dependent DNA and RNA helicase activity</span>. <span style="background-color:#ff2525">CSNK1A1, HSPA4 and HSPA9 are linked to ATP binding while HNRNPC is involved in RNA and nucleotide binding</span>. <span style="background-color:#ffa0a0">A summary of the GO terms for these genes can be found in Table 3 </span>. <span style="background-color:#ffa8a8">HNRNPC, HSPA9, G3BP1 and UBE2D2 were determined as being prognostic individually when a univariate analysis was carried out on their expression levels and survival time, the associated p values were 0.027, 0.002, 0.02 and 0.003 respectively</span>.</p> 
   <fig id="pone-0014635-g002" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0014635.g002 
    </object-id> 
    <label>Figure 2</label> 
    <h3>Change in gene expression of our six significantly differentially expressed genes linked to VDAC1 expression.</h3> 
    <p><span style="background-color:#ffd2d2">This chart shows the variation in gene expression of our differentially expressed genes in the samples where VDAC1 expression is low and when it is high</span>. <span style="background-color:#ffc7c7">All the genes that collectively made up our signature are upregulated and so increase when VDAC1 increases</span>.</p> 
    <graphic xlink:href="pone.0014635.g002"></graphic> 
   </fig> 
   <table-wrap id="pone-0014635-t003" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0014635.t003 
    </object-id> 
    <label>Table 3</label> 
    <h3>A table representing the gene ontology processes that define the genes in our meta-signature.</h3> 
    <alternatives> 
     <graphic id="pone-0014635-t003-3" xlink:href="pone.0014635.t003"></graphic> 
     <table frame="hsides" rules="groups"> 
      <colgroup span="1"> 
       <col align="left" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
      </colgroup> 
      <thead> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1">Ubquitination/chaparone</td> 
        <td align="left" colspan="1" rowspan="1">RAS Signalling</td> 
        <td align="left" colspan="1" rowspan="1">P53 Regulation</td> 
        <td align="left" colspan="1" rowspan="1">Translation</td> 
        <td align="left" colspan="1" rowspan="1">RNA binding</td> 
       </tr> 
      </thead> 
      <tbody> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           HNRNPC 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1">X</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           UBE2D2 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">X</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           HSPA4 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">X</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           HSPA9 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">X</td> 
        <td align="left" colspan="1" rowspan="1">X</td> 
        <td align="left" colspan="1" rowspan="1">X</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           CSNK1A1 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1">X</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           G3BP1 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1">X</td> 
        <td align="left" colspan="1" rowspan="1">X</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
      </tbody> 
     </table> 
    </alternatives> 
    <table-wrap-foot> 
     <fn id="nt103"> 
      <label></label> 
      <p><span style="background-color:#ff0000">Any genes involved in the same processes are easily identifiable and overlap is evident</span>. <span style="background-color:#ff0000">Ubiquitination is the most common known process shared by the significant genes; however RAS signalling and P53 regulation are also linked to 2 of the genes from the signature</span>.</p> 
     </fn> 
    </table-wrap-foot> 
   </table-wrap> 
  </sec> 
  <sec id="s2f"> 
   <h3>Unsupervised analysis identifies VDAC1 as prognostic and associated with a 6-gene signature</h3> 
   <p><span style="background-color:#ffdada">We also analysed the data without filtering for VDAC1 expression to reaffirm VDAC1 would be significantly differentially expressed when expression levels of living and dead patients are compared</span>. <span style="background-color:#ffefef">Using this unsupervised method and ranking gene expression according to overall survival time, VDAC1 was found to be downregulated in patients with longer survival</span>. <span style="background-color:#ffb2b2">Similarly the 6 genes that are regulated alongside VDAC1 behaved in the same way, each of them were downregulated in samples from patients that survived for a long period of time</span>.</p> 
  </sec> 
  <sec id="s2g"> 
   <h3>6-gene Signature association with VDAC1 overexpression is independent of cancer type</h3> 
   <p><span style="background-color:#fffafa">To confirm the validity and potential biological relevance of the 6-gene signature covariant with VDAC1 we tested its discriminative power on an independent breast cancer dataset</span>. <span style="background-color:#fffafa">A t-test was performed comparing high and low VDAC1 expressing samples</span>. <span style="background-color:#fffbfb">We found all 6 genes were upregulated when VDAC1 was highly expressed</span>. <span style="background-color:#ffcfcf">Similarly, expression was low when VDAC1 expression was low and so our signature of genes covariant with VDAC1 in NSCLC data exhibited the same expression patterns in breast cancer data</span>. <span style="background-color:#ffd6d6">We also carried out the same test using an independent myeloma cancer dataset, performing a t-test on the data which was split based on VDAC1 high and low expression</span>. <span style="background-color:#ffafaf">We again found all of the 6 genes in our meta-signature returned as being statistically significant at a p&lt;0.001 level</span>. <span style="background-color:#fffdfd">Additional validation was found through the analysis of the NCI60 cell lines found in GSE2003</span>. <span style="background-color:#fffcfc">We again compared the samples based on VDAC1 expression and found the 6-gene signature was conserved in 8 of these cell lines, namely UACC62 and SK_MEL2 (melanoma), SF295 and SF539 (CNS), A498 and SNC12C (renal), H23 and HOP62 (NSCLC)</span>. <span style="background-color:#fffefe">Analysing the NCI60 data for low VDAC1 with low expression of the 6-gene signature identified 5 cell lines, CCRF-CEM, RPMI-8226 and K562B (leukaemia), IGROV1 (ovarian) and H522 (NSCLC)</span>.</p> 
  </sec> 
  <sec id="s2h"> 
   <h3>Validation of 6-gene signature using Principal Components Analysis (PCA)</h3> 
   <p><span style="background-color:#ffc9c9">PCA helps to explain the variance in data and is a common technique for dimensionality reduction in high dimensional data</span>. <span style="background-color:#ffb5b5">We performed the PCA on the NSCLC dataset GSE3141 and found that although there were some outliers using the first principal component, which accounted for as much of the variability as possible, all 6 of the genes in our meta-signature were in the top 1% based on PC1 component loadings</span>. <span style="background-color:#ffd8d8">HNRNPC and G3BP1 performed particularly well being placed fourth and fifth in the list respectively</span>.</p> 
  </sec> 
  <sec id="s2i"> 
   <h3>Protein-Protein Interaction (PPI) Networks</h3> 
   <p><span style="background-color:#fffbfb">Within the VDAC1 PPI network, using Yu et al's [28] categorisation, encoded proteins were classed as either hubs (nodes with high degree values constituting vulnerable areas of the network) and/or bottlenecks (those with high betweenness centrality scores corresponding to key intersecting nodes)</span>. <span style="background-color:#ffa0a0">VDAC1 and four of the genes, CSNK1A1, HSPA4, HNRNPC and UBE2D2 were considered as both hub-bottlenecks</span>.</p> 
   <p><span style="background-color:#ffcece">Clustering identified potential groups of genes operating in the same pathways, processes or molecular complexes</span>. <span style="background-color:#ffa0a0">Five of the genes, G3BP1, UBE2D2, VDAC1, CSNK1A1 and HSPA4 belonged to four high-scoring clusters, the highest-ranked of these containing both UBE2D2 and G3BP1</span>. <span style="background-color:#ffcdcd">Functional analysis revealed this cluster to be most significantly associated with the SH2 domain, ATP binding sites, protein and protein tyrosine kinase activity, autophosphorylation, DNA binding and transcription factor activity</span>. <span style="background-color:#ffc4c4">The next most significant VDAC1-related cluster included HSPA4 which was enriched in nuclear lumen and nucleoplasm components and RNA-binding and mRNA splicing processes</span>. <span style="background-color:#ffc9c9">VDAC1 itself was associated with a top-scoring cluster functionally enriched in regulation of programmed cell death, apoptosis, anti-apoptosis, nucleoplasm, intracellular organelle lumen, transcription factor binding and repressor activities</span>. <span style="background-color:#ffbcbc">The cluster of which CSNK1A1 was a member was associated with significant functionality in coated pit, endomembrane system, endocytosis, endosome, binding and transport activities</span>.</p> 
   <p><span style="background-color:#ffc6c6">Shortest path analysis revealed a sub-network of 14 genes, comprising the VDAC1 signature genes and key inter-connecting partners including TP53, RASA1, GRB2, CBL, CSK, RAF1 and KPNA2, see Figure 3 </span>. <span style="background-color:#ffaaaa">Only two of the genes HNRNPC and CSNK1A1 were direct interactors; HNRNPC's binding ability being modulated by CSNK1A1-mediated phosphorylation</span>.</p> 
   <fig id="pone-0014635-g003" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0014635.g003 
    </object-id> 
    <label>Figure 3</label> 
    <h3>Shortest-path PPI analysis of VDAC1 target genes.</h3> 
    <p><span style="background-color:#ffeeee">These are shown with most-frequently occurring inter-connecting genes</span>. <span style="background-color:#fff4f4">Shortest path analysis revealed a sub-network of 14 genes, comprising VDAC1 and the 6 differentially expressed genes associated with VDAC1 expression</span>. <span style="background-color:#ffffff">The key inter-connecting partners included TP53, RASA1, GRB2, CBL, CSK, RAF1 and KPNA2</span>. <span style="background-color:#ffd7d7">Only HNRNPC and CSNK1A1 interacted directly</span>.</p> 
    <graphic xlink:href="pone.0014635.g003"></graphic> 
   </fig> 
   <p><span style="background-color:#ffe4e4">Functional analysis revealed enriched processes primarily associated with the VDAC1 target genes including ATP-binding and nucleotide binding</span>. <span style="background-color:#ffdddd">Both the VDAC1-target and internal shortest-path genes jointly participated in a number of significantly enriched processes and pathways</span>. <span style="background-color:#ffb4b4">These included signalling processes (intracellular signalling cascade, RAS protein signal transduction, small GTPase mediated signal transduction), transport (nucleocytoplasmic transport, nuclear transport, intracellular protein transport) and anti-apoptosis</span>.</p> 
   <p><span style="background-color:#ffdede">The genes providing internal links within the shortest-path network were significantly enriched in signalling pathways, e.g</span>. <span style="background-color:#ffffff">Neurotrophin, IGF-1, Insulin, TPO, PDGF, EGF, Integrin, T Cell Receptor, MAPK and ERKB</span>. <span style="background-color:#ffb0b0">Further pathways identified included: signalling of hepatocyte growth factor receptor, transmembrane receptor protein tyrosine kinase signalling pathway, enzyme linked receptor protein signalling pathway, sprouty regulation of tyrosine kinase signals and IL-2 Receptor Beta Chain in T cell Activation</span>.</p> 
   <p><span style="background-color:#ffc1c1">The main combined activities of the VDAC1 genes (CSNK1A1, G3BP1, HSPA4, HSPA9 and UBE2D2) were focused on ATP and nucleotide binding</span>. Specific individual enriched functionality also occurred, e.g. protein kinase activity and host-virus interaction (associated with genes CSNK1A1 and VDAC1, respectively).</p> 
   <p>Interestingly, although less prominent within the PPI network, the two non-hub-non-bottleneck genes, HSPA9 and G3BP1, were enriched in distinct sets of functions. In the case of G3BP1, signal transduction was highlighted (both small GTPase and RAS protein) in conjunction with the connecting genes, RAF1 and GRB2. <span style="background-color:#ffd6d6">HSPA9, in particular, was associated with two groups of functions: protein targeting and transport (intracellular protein, nuclear and nucleocytoplasmic) in conjunction with TP53 and KPNA2; and negative regulation of cell death, apoptosis and programmed cell death (together with TP53 and RASA1)</span>.</p> 
  </sec> 
 </sec> 
 <sec id="s3"> 
  <h3>Discussion</h3> 
  <p><span style="background-color:#ffe2e2">VDAC1 overexpression predicts shorter time to recurrence and overall survival for NSCLC as evidenced from pooled gene expression analysis</span>. <span style="background-color:#ffdbdb">It is an independent prognostic factor as evidenced from Cox regression analysis and predicts survival in stage 1 disease, where subgroup analysis of randomized controlled clinical trials of adjuvant chemotherapy have failed to show clinical benefit [24] , [25] , [26] </span>. <span style="background-color:#ffdbdb">A combined gene expression data analysis approach has been employed to investigate the impact of VDAC1 expression on survival, which was statistically significant in all individual datasets examined</span>.</p> 
  <p><span style="background-color:#fff5f5">Several recent studies analysing microarray data for prognostic markers in NSCLC have produced inconsistent results [29] </span>. <span style="background-color:#ffc8c8">The distinct lack of overlap associated between these signatures reflects instability and is attributed to small sample sizes with less than 200 samples used per study</span>. <span style="background-color:#ffe5e5">Analysis of multiple datasets, for example combined analysis of microarray gene expression datasets addressing similar biological questions conducted at an interpretational level by meta-analysis, can enable more accurate results</span>. <span style="background-color:#ffd8d8">Many studies propose methods for meta-analysis of microarray data with the aim of identifying significantly differentially expressed genes across studies using statistical techniques that avoid the direct comparison of gene expression values [30] </span>. <span style="background-color:#ffcaca">The evaluation of multiple datasets as employed in this study has been shown to yield more reliable and valid results, because they are based on large sample numbers and the individual bias caused by each study is weakened [30] </span></p> 
  <p><span style="background-color:#ffcdcd">It is not known why VDAC1 correlates with poor survival outcomes</span>. <span style="background-color:#ffc5c5">To understand the most relevant genetic features of VDAC1 overexpressing NSCLC, we conducted a gene expression meta-analysis [31] , [32] to identify a subset of genes (signature) that were significantly enriched</span>. <span style="background-color:#fff6f6">We employed stringent statistical criteria, combined with a large sample size to support the identification of these VDAC1-covariant genes</span>. <span style="background-color:#ffeeee">VDAC1 and 6 gene signature was then validated across breast, myeloma and NCI-60 datasets [33] , suggesting enrichment of genes which were independent of the type of cancer</span></p> 
  <p><span style="background-color:#ffafaf">Interestingly, of the 6 genes identified as being conserved and significantly differentially regulated in the high VDAC1 expressing group, most were functionally linked to the regulation of protein turnover</span>. <span style="background-color:#ffd0d0">These genes included heat shock 70kDa protein 4 (HSPA4), ubiquitin-conjugating enzyme E2D 2 (UBE2D2), and heat shock 70kDa protein 9 (mortalin/HSPA9), encoding a glucose regulated 75 kilodalton protein previously reported as correlating with poor survival in colorectal cancer [34] </span>. <span style="background-color:#ff9797">HSPA9 also binds and inactivates wild type p53 [35] , and regulates the RAS RAF MEK pathway [36] </span>. <span style="background-color:#ff8787">Similarly, p53 and RAS are targets of GTPase activating protein (SH3 domain) binding protein 1 (G3BP1) [37] , and shown to predict shorter survival in oesophageal cancer [38] </span>. <span style="background-color:#ff9f9f">Casein kinase 1, alpha 1 (CSNK1A1) regulates protein turnover via initiation of translation via EIF2 and participates in wnt signalling; a homologue CSKNK2A1 has been previously identified as an independent predictor of survival in squamous cell lung cancer [39] </span>. <span style="background-color:#ff7575">Like G3BP1, heterogeneous nuclear ribonucleoprotein C (C1/C2) or HNRNPC is involved in RNA binding and transcription [40] </span></p> 
  <p><span style="background-color:#ffb4b4">The PPI networks identified that the main combined activities of the VDAC1 genes (CSNK1A1, G3BP1, HSPA4, HSPA9 and UBE2D2) were focussed on ATP and nucleotide binding</span>. <span style="background-color:#fff0f0">Specific individual enriched functionality also occurred, e.g</span>. <span style="background-color:#ffe7e7">protein kinase activity and host-virus interaction (associated with genes CSNK1A1 and VDAC1, respectively)</span>. <span style="background-color:#ffd6d6">Interestingly, although less prominent within the PPI network, the two non-hub-non-bottleneck genes, HSPA9 and G3BP1, were enriched in distinct sets of functions</span>. <span style="background-color:#ffcece">In the case of G3BP1, signal transduction was highlighted (both small GTPase and RAS protein) in conjunction with the connecting genes, RAF1 and GRB2</span>. <span style="background-color:#ffd6d6">HSPA9, in particular, was associated with two groups of functions: protein targeting and transport (intracellular protein, nuclear and nucleocytoplasmic) in conjunction with TP53 and KPNA2; and negative regulation apoptosis (together with TP53 and RASA1)</span>.</p> 
  <sec id="s3a"> 
   <h3>Conclusions</h3> 
   <p><span style="background-color:#fff9f9">In summary, based on the poor prognosis associated with overexpression of VDAC1 and the associated 6-gene signature, methods to target this molecular subclass of NSCLC may be effective in improve survival outcomes after surgery</span>. <span style="background-color:#fffdfd">We have identified cell lines which have upregulation of the 6-gene signature found in VDAC1 overexpressing NSCLC</span>. <span style="background-color:#fffcfc">Ongoing work aims to target this signature to determine the effects on cell viability utilizing the connectivity map</span>. <span style="background-color:#ffe8e8">We propose that regulating the expression of VDAC1 and/or genes with the 6-gene signature, could provide a novel therapeutic strategy for targeting poor risk patients with NSCLC</span>.</p> 
  </sec> 
 </sec> 
 <sec id="s4" sec-type="materials|methods"> 
  <h3>Materials and Methods</h3> 
  <sec id="s4a"> 
   <h3>Gene expression datasets</h3> 
   <p><span style="background-color:#ffc0c0">To conduct gene expression meta-analysis, a library of 8 NSCLC gene expression datasets was curated from NCBI Gene Expression Omnibus (GEO) [41] : GSE8894, GSE3141 (array type: HG-U133 Plus2.0), GSE6253, GSE4573 (array type: HG-U133A), GSE6253 (array type: HG-U133B), GSE6253 (array type: HG-U95Av2), GSE4716 (array type: GeneFilter Human Microarray Release II) and GSE5123 (array type: PC Human Operon v 2 21k)</span>. <span style="background-color:#ffe5e5">They had 602 samples in total and contained gene expression information on primarily resected tumour samples that had received no previous treatment</span>. <span style="background-color:#fff6f6">They were pre-processed using R [42] and Bioconductor [43] </span>. <span style="background-color:#fffefe">Normalization was carried out using the RMA algorithm [44] as implemented in Bioconductor</span>. <span style="background-color:#ffffff">Data was base-two log-transformed where applicable</span>. <span style="background-color:#ffe2e2">Along with gene expression measurements, there were also recordings of patient information, including age, sex, histology and overall survival time, and in some cases stage, recurrence free survival time, tumour stage (pT) and nodal stage (pN) were also recorded</span>. <span style="background-color:#ffd9d9">The genes that were found to be significantly differentially expressed in the NSCLC datasets were then validated in independent breast cancer datasets obtained from GEO: GSE6434 containing HG-U95Av2 array data from 24 patients who had received chemotherapy and GSE2034 with HG-U133A array data from 286 samples</span>. <span style="background-color:#ffbaba">We also tested the meta-signature in a myeloma dataset (GSE2658) with HG-U133 Plus2.0 array data from 559 patients who had received chemotherapy</span>. <span style="background-color:#ffcfcf">Further validation was found through the analysis of the NCI60 cell line panel (GSE2003) were we checked gene expression matched what we found in our subset of significant genes</span>. <span style="background-color:#ffdbdb">Additionally we checked our results by carrying out principal components analysis on the data in GSE3141</span>.</p> 
  </sec> 
  <sec id="s4b"> 
   <h3>Univariate and multivariate survival analysis</h3> 
   <p><span style="background-color:#fffafa">The distribution of VDAC1 expression was subdivided into tertiles for each data set, corresponding to low, medium and high</span>. <span style="background-color:#ffffff">Kaplan-Meier curves were plotted using Prism (GraphPad Software, San Diego California USA, www.graphpad.com ) based on survival data, comparing the highest versus lowest tertile for VDAC1, initially these plots were produced for all data and then only for stage I</span>. <span style="background-color:#ffffff">Hazard ratios comparing low and high VDAC1 expressing samples were calculated</span>. <span style="background-color:#ffffff">We carried out correlation analysis between VDAC1 expression and survival time using Spearman's Rank correlation coefficient</span>. <span style="background-color:#ffdcdc">We also performed univariate and multivariate analysis using PASW (SPSS: An IBM company), recurrence-free and overall survival were used as clinical endpoints to assess prognostic significance of VDAC1; the logrank ratio was considered to be significant if p&lt;0.05</span>. <span style="background-color:#fff3f3">Cox multivariate analysis was used to determine whether VDAC1 is an independent prognostic variable in relation to 7 clinical variables: age, gender, stage, histology, tumour size (pT), nodal stage (pN) and survival</span>.</p> 
  </sec> 
  <sec id="s4c">
   <h3>Supervised microarray meta-analysis</h3> 
   <p><span style="background-color:#ffeded">We adopted the method termed meta-analysis of microarrays to determine what genes were regulated alongside VDAC1 [45] </span>. <span style="background-color:#fff2f2">Each dataset was subdivided according to VDAC1 expression into tertiles; we compared the top and bottom VDAC1 tertiles samples, using TMeV [46] </span>. <span style="background-color:#ffd4d4">Each gene was assessed for differential expression using a two class t-test for each dataset with class 1 being low VDAC1 expression and class 2, high VDAC1 expression</span>. <span style="background-color:#ffefef">Bonferroni's correction was also used and this analysis was carried out on all the 8 datasets</span>. <span style="background-color:#fffefe">Bonferroni's correction helps minimise the likelihood of generating false positives</span>. <span style="background-color:#ffbdbd">The number of probes is vast with some datasets containing more than 54,000, so to reduce our significant genes to a manageable list per dataset we used a stringent cut off criteria where a gene was labelled as significantly differentially expressed if the t-test returned a p value of less than 0.001</span>. <span style="background-color:#ffe9e9">The same hypotheses were tested in each NSCLC dataset independently e.g</span>. <span style="background-color:#fffbfb">genes differentially expressed based on VDAC1 expression</span>. <span style="background-color:#ffcece">To enable multiple hypothesis testing, we used a method that compares statistical measures and Q values (estimated false discovery rates) calculated independently from each dataset [32] , [47] , where Where p is the probability, n is the total number of genes and i is the sorted rank of the p value</span>. <span style="background-color:#ffffff">The strict p value cut-off we applied was carried forward when Q values were calculated</span>. <span style="background-color:#ffaeae">After ranking the significantly differentially expressed genes based on their Q value a direction and significance threshold of Q&lt;0.10 was set to define differential expression signatures from the already computed differential expression analysis</span>. <span style="background-color:#fff8f8">Having obtained an ordered list of differentially expressed genes for each dataset genes were then filtered based on the number of datasets in which they were differentially expressed.</span>. <span style="background-color:#ffdddd">The genes that were defined as being significantly differentially expressed were then validated in independent breast cancer datasets (GSE6434 and GSE2034) obtained from GEO with 24 and 286 samples respectively</span>. <span style="background-color:#ffc8c8">We also tested the meta-signature in a myeloma dataset (GSE2658) with 559 samples</span>. <span style="background-color:#ffd4d4">T-tests were performed on this additional data; we compared high and low VDAC1 expressing samples to determine if the genes from our signature were identified once again</span>. <span style="background-color:#ffd2d2">The same procedure was used on the NCI60 cell lines to identify cell lines in which expression levels matched our signature</span>.</p>
  </sec> 
  <sec id="s4d"> 
   <h3>Protein protein interaction network analysis</h3> 
   <p><span style="background-color:#fff4f4">Using VDAC1 and the six associated SDE genes as seeds together with their interactions found in the Human Protein Reference Database (HPRD) [48] , a parent PPI network (including recursive interactions) was created using an in-house Java-based program</span>. <span style="background-color:#ffffff">Using these seed proteins as the initial depth (level 0), subsequent depths were obtained using snowball sampling</span>. <span style="background-color:#ffe9e9">The resulting parent PPI network (9,266 gene nodes, 38,800) was then viewed, manipulated and analyzed in Cytoscape [49] </span>. <span style="background-color:#ffe4e4">Using the Network Analyzer plug-in [50] gene nodes were then identified as either hubs or bottlenecks using the classification of Yu et al [28] ; the top 20% of nodes by degree value were identified as hubs, the remainder as non-hubs</span>. <span style="background-color:#ffe7e7">The same ranking method was applied to the betweenness centrality metric to identify bottleneck nodes</span>. <span style="background-color:#fffcfc">Each node in the PPI network was thus classified in one of four categories: non-hub-non-bottlenecks (NH/NB), hub-non-bottlenecks (H/NB), non-hub-bottlenecks (NH/B), and hub-bottlenecks (H/B)</span>. <span style="background-color:#fffbfb">Multiple shortest paths between the seven genes were identified using the PeSca tool</span>. <span style="background-color:#ffd6d6">Key shortest paths were identified by considering direct connections between the target genes, paths in which the target genes themselves acted as inter-connectors, and the most frequently occurring genes providing internal linkages</span>. <span style="background-color:#fff2f2">Modular functional enrichment analysis of the gene clusters and shortest path network was carried out using the online tool DAVID [51] .</span></p> 
  </sec> 
 </sec> 
 <back> 
  <fn-group> 
   <fn fn-type="conflict"> 
    <p> 
     <bold>
       Competing Interests: 
     </bold>The authors have declared that no competing interests exist.</p> 
   </fn> 
   <fn fn-type="financial-disclosure"> 
    <p> 
     <bold>
       Funding: 
     </bold>CG is funded by the Department of Employment and Learning. DAF is funded by Cancer Research United Kingdom on a Clinician Scientist Fellowship. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p> 
   </fn> 
  </fn-group> 
  <ref-list> 
   <h3>References</h3> 
   <ref id="pone.0014635-Vallieres1"> 
    <label>1</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Vallieres 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2007 
     </year> 
     <article-h3>
       Role of Adjuvant Systemic Therapy for Stage I NSCLC. 
     </article-h3> 
     <source />Thoracic Surgery Clinics 
     <volume>
       17 
     </volume> 
     <fpage>
       279 
     </fpage> 
     <lpage>
       285 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17626406 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Dakubo1"> 
    <label>2</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Dakubo 
       </surname> 
       <given-names>
         GD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Parr 
       </surname> 
       <given-names>
         RL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Costello 
       </surname> 
       <given-names>
         LC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Franklin 
       </surname> 
       <given-names>
         RB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Thayer 
       </surname> 
       <given-names>
         RE 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2006 
     </year> 
     <article-h3>
       Altered metabolism and mitochondrial genome in prostate cancer. 
     </article-h3> 
     <source />Journal of clinical pathology 
     <volume>
       59 
     </volume> 
     <fpage>
       10 
     </fpage> 
     <lpage>
       16 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16394275 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Hoang1"> 
    <label>3</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Hoang 
       </surname> 
       <given-names>
         JK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hoagland 
       </surname> 
       <given-names>
         LF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Coleman 
       </surname> 
       <given-names>
         RE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Coan 
       </surname> 
       <given-names>
         AD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Herndon 
       </surname> 
       <given-names>
         JE 
       </given-names> 
       <suffix>
         2nd 
       </suffix> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       Prognostic value of fluorine-18 fluorodeoxyglucose positron emission tomography imaging in patients with advanced-stage non-small-cell lung carcinoma. 
     </article-h3> 
     <source />Journal of clinical oncology : official journal of the American Society of Clinical Oncology 
     <volume>
       26 
     </volume> 
     <fpage>
       1459 
     </fpage> 
     <lpage>
       1464 
     </lpage> 
     <pub-id pub-id-type="pmid">
       18349396 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Dooms1"> 
    <label>4</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Dooms 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         van Baardwijk 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Verbeken 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         van Suylen 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stroobants 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Association between 18F-fluoro-2-deoxy-D-glucose uptake values and tumor vitality: prognostic value of positron emission tomography in early-stage non-small cell lung cancer. 
     </article-h3> 
     <source />Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer 
     <volume>
       4 
     </volume> 
     <fpage>
       822 
     </fpage> 
     <lpage>
       828 
     </lpage> 
     <pub-id pub-id-type="pmid">
       19487964 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-VanderHeiden1"> 
    <label>5</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Vander Heiden 
       </surname> 
       <given-names>
         MG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chandel 
       </surname> 
       <given-names>
         NS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Schumacker 
       </surname> 
       <given-names>
         PT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Thompson 
       </surname> 
       <given-names>
         CB 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       1999 
     </year> 
     <article-h3>
       Bcl-xL Prevents Cell Death following Growth Factor Withdrawal by Facilitating Mitochondrial ATP/ADP. 
     </article-h3> 
     <source />Exchange Molecular Cell 
     <volume>
       3 
     </volume> 
     <fpage>
       159 
     </fpage> 
     <lpage>
       167 
     </lpage> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Hiller1"> 
    <label>6</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Hiller 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Garces 
       </surname> 
       <given-names>
         RG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Malia 
       </surname> 
       <given-names>
         TJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Orekhov 
       </surname> 
       <given-names>
         VY 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Colombini 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       Solution structure of the integral human membrane protein VDAC-1 in detergent micelles. 
     </article-h3> 
     <source />Science (New York, NY) 
     <volume>
       321 
     </volume> 
     <fpage>
       1206 
     </fpage> 
     <lpage>
       1210 
     </lpage> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-VanderHeiden2"> 
    <label>7</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Vander Heiden 
       </surname> 
       <given-names>
         MG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Li 
       </surname> 
       <given-names>
         XX 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gottleib 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hill 
       </surname> 
       <given-names>
         RB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Thompson 
       </surname> 
       <given-names>
         CB 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2001 
     </year> 
     <article-h3>
       Bcl-xL promotes the open configuration of the voltage-dependent anion channel and metabolite passage through the outer mitochondrial membrane. 
     </article-h3> 
     <source />The Journal of biological chemistry 
     <volume>
       276 
     </volume> 
     <fpage>
       19414 
     </fpage> 
     <lpage>
       19419 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11259441 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Madesh1"> 
    <label>8</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Madesh 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hajnoczky 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2001 
     </year> 
     <article-h3>
       VDAC-dependent permeabilization of the outer mitochondrial membrane by superoxide induces rapid and massive cytochrome c release. 
     </article-h3> 
     <source />The Journal of cell biology 
     <volume>
       155 
     </volume> 
     <fpage>
       1003 
     </fpage> 
     <lpage>
       1015 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11739410 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Shimizu1"> 
    <label>9</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Shimizu 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Narita 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tsujimoto 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       1999 
     </year> 
     <article-h3>
       Bcl-2 family proteins regulate the release of apoptogenic cytochrome c by the mitochondrial channel VDAC. 
     </article-h3> 
     <source />Nature 
     <volume>
       399 
     </volume> 
     <fpage>
       483 
     </fpage> 
     <lpage>
       487 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10365962 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Roy1"> 
    <label>10</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Roy 
       </surname> 
       <given-names>
         SS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Madesh 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Davies 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Antonsson 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Danial 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Bad targets the permeability transition pore independent of Bax or Bak to switch between Ca2+-dependent cell survival and death. 
     </article-h3> 
     <source />Molecular cell 
     <volume>
       33 
     </volume> 
     <fpage>
       377 
     </fpage> 
     <lpage>
       388 
     </lpage> 
     <pub-id pub-id-type="pmid">
       19217411 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Cheng1"> 
    <label>11</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Cheng 
       </surname> 
       <given-names>
         EH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sheiko 
       </surname> 
       <given-names>
         TV 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fisher 
       </surname> 
       <given-names>
         JK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Craigen 
       </surname> 
       <given-names>
         WJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Korsmeyer 
       </surname> 
       <given-names>
         SJ 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2003 
     </year> 
     <article-h3>
       VDAC2 inhibits BAK activation and mitochondrial apoptosis. 
     </article-h3> 
     <source />New York: Science 
     <volume>
       301 
     </volume> 
     <fpage>
       513 
     </fpage> 
     <lpage>
       517 
     </lpage> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Baines1"> 
    <label>12</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Baines 
       </surname> 
       <given-names>
         CP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kaiser 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sheiko 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Craigen 
       </surname> 
       <given-names>
         WJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Molkentin 
       </surname> 
       <given-names>
         JD 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2007 
     </year> 
     <article-h3>
       Voltage-dependent anion channels are dispensable for mitochondrial-dependent cell death. 
     </article-h3> 
     <source />Nature cell biology 
     <volume>
       9 
     </volume> 
     <fpage>
       550 
     </fpage> 
     <lpage>
       555 
     </lpage> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Kumarswamy1"> 
    <label>13</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Kumarswamy 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chandna 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Putative partners in Bax mediated cytochrome-c release: ANT, CypD, VDAC or none of them? 
     </article-h3> 
     <source />Mitochondrion 
     <volume>
       9 
     </volume> 
     <fpage>
       1 
     </fpage> 
     <lpage>
       8 
     </lpage> 
     <pub-id pub-id-type="pmid">
       18992370 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Park1"> 
    <label>14</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Park 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kim 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Choi 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Koh 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         HE 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2010 
     </year> 
     <article-h3>
       Drosophila Porin/VDAC Affects Mitochondrial Morphology. 
     </article-h3> 
     <source />Plos One 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Sukumaran1"> 
    <label>15</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Sukumaran 
       </surname> 
       <given-names>
         SK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fu 
       </surname> 
       <given-names>
         NY 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tin 
       </surname> 
       <given-names>
         CB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wan 
       </surname> 
       <given-names>
         KF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         SS 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       A soluble form of the pilus protein FimA targets the VDAC-hexokinase complex at mitochondria to suppress host cell apoptosis. 
     </article-h3> 
     <source />Molecular cell 
     <volume>
       37 
     </volume> 
     <fpage>
       768 
     </fpage> 
     <lpage>
       783 
     </lpage> 
     <pub-id pub-id-type="pmid">
       20347420 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Arzoine1"> 
    <label>16</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Arzoine 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zilberberg 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ben-Romano 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shoshan-Barmatz 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Voltage-dependent anion channel 1-based peptides interact with hexokinase to prevent its anti-apoptotic activity. 
     </article-h3> 
     <source />The Journal of biological chemistry 
     <volume>
       284 
     </volume> 
     <fpage>
       3946 
     </fpage> 
     <lpage>
       3955 
     </lpage> 
     <pub-id pub-id-type="pmid">
       19049977 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Pastorino1"> 
    <label>17</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Pastorino 
       </surname> 
       <given-names>
         JG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shulga 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hoek 
       </surname> 
       <given-names>
         JB 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2002 
     </year> 
     <article-h3>
       Mitochondrial binding of hexokinase II inhibits Bax-induced cytochrome c release and apoptosis. 
     </article-h3> 
     <source />The Journal of biological chemistry 
     <volume>
       277 
     </volume> 
     <fpage>
       7610 
     </fpage> 
     <lpage>
       7618 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11751859 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Galluzzi1"> 
    <label>18</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Galluzzi 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kepp 
       </surname> 
       <given-names>
         O 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tajeddine 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kroemer 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       Disruption of the hexokinase-VDAC complex for tumor therapy. 
     </article-h3> 
     <source />Oncogene 
     <volume>
       27 
     </volume> 
     <fpage>
       4633 
     </fpage> 
     <lpage>
       4635 
     </lpage> 
     <pub-id pub-id-type="pmid">
       18469866 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Pastorino2"> 
    <label>19</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Pastorino 
       </surname> 
       <given-names>
         JG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hoek 
       </surname> 
       <given-names>
         JB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shulga 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2005 
     </year> 
     <article-h3>
       Activation of glycogen synthase kinase 3beta disrupts the binding of hexokinase II to mitochondria by phosphorylating voltage-dependent anion channel and potentiates chemotherapy-induced cytotoxicity. 
     </article-h3> 
     <source />Cancer research 
     <volume>
       65 
     </volume> 
     <fpage>
       10545 
     </fpage> 
     <lpage>
       10554 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16288047 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Majewski1"> 
    <label>20</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Majewski 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nogueira 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bhaskar 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Coy 
       </surname> 
       <given-names>
         PE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Skeen 
       </surname> 
       <given-names>
         JE 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2004 
     </year> 
     <article-h3>
       Hexokinase-mitochondria interaction mediated by Akt is required to inhibit apoptosis in the presence or absence of Bax and Bak. 
     </article-h3> 
     <source />Mol Cell 
     <volume>
       16 
     </volume> 
     <fpage>
       819 
     </fpage> 
     <lpage>
       830 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15574336 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Koren1"> 
    <label>21</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Koren 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Raviv 
       </surname> 
       <given-names>
         Z 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shoshan-Barmatz 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2010 
     </year> 
     <article-h3>
       Downregulation of voltage-dependent anion channel-1 expression by RNA interference prevents cancer cell growth in vivo. 
     </article-h3> 
     <source />Cancer Biology Therapy 
     <volume>
       9 
     </volume> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-AbuHamad1"> 
    <label>22</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Abu-Hamad 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sivan 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shoshan-Barmatz 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2006 
     </year> 
     <article-h3>
       The expression level of the voltage-dependent anion channel controls life and death of the cell. 
     </article-h3> 
     <source />Proceedings of the National Academy of Sciences of the United States of America 
     <volume>
       103 
     </volume> 
     <fpage>
       5787 
     </fpage> 
     <lpage>
       5792 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16585511 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Rho1"> 
    <label>23</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Rho 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kim 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jee 
       </surname> 
       <given-names>
         CD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         YM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         HE 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2007 
     </year> 
     <article-h3>
       Expression of type 2 hexokinase and mitochondria-related genes in gastric carcinoma tissues and cell lines. 
     </article-h3> 
     <source />Anticancer Res 
     <volume>
       27 
     </volume> 
     <fpage>
       251 
     </fpage> 
     <lpage>
       258 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17352240 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Winton1"> 
    <label>24</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Winton 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Livingston 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Johnson 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rigas 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Johnston 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2005 
     </year> 
     <article-h3>
       Vinorelbine plus cisplatin vs. observation in resected non-small-cell lung cancer. 
     </article-h3> 
     <source />N Engl J Med 
     <volume>
       352 
     </volume> 
     <fpage>
       2589 
     </fpage> 
     <lpage>
       2597 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15972865 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Arriagada1"> 
    <label>25</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Arriagada 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bergman 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dunant 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Le Chevalier 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pignon 
       </surname> 
       <given-names>
         JP 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2004 
     </year> 
     <article-h3>
       Cisplatin-based adjuvant chemotherapy in patients with completely resected non-small-cell lung cancer. 
     </article-h3> 
     <source />N Engl J Med 
     <volume>
       350 
     </volume> 
     <fpage>
       351 
     </fpage> 
     <lpage>
       360 
     </lpage> 
     <pub-id pub-id-type="pmid">
       14736927 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Douillard1"> 
    <label>26</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Douillard 
       </surname> 
       <given-names>
         JY 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rosell 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         De Lena 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Carpagnano 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ramlau 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2006 
     </year> 
     <article-h3>
       Adjuvant vinorelbine plus cisplatin versus observation in patients with completely resected stage IB-IIIA non-small-cell lung cancer (Adjuvant Navelbine International Trialist Association [ANITA]): a randomised controlled trial. 
     </article-h3> 
     <source />Lancet Oncol 
     <volume>
       7 
     </volume> 
     <fpage>
       719 
     </fpage> 
     <lpage>
       727 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16945766 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Ashburner1"> 
    <label>27</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ashburner 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ball 
       </surname> 
       <given-names>
         CA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Blake 
       </surname> 
       <given-names>
         JA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Botstein 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Butler 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2000 
     </year> 
     <article-h3>
       Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. 
     </article-h3> 
     <source />Nat Genet 
     <volume>
       25 
     </volume> 
     <fpage>
       25 
     </fpage> 
     <lpage>
       29 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10802651 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Yu1"> 
    <label>28</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Yu 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kim 
       </surname> 
       <given-names>
         PM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sprecher 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Trifonov 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gerstein 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2007 
     </year> 
     <article-h3>
       The importance of bottlenecks in protein networks: correlation with gene essentiality and expression dynamics. 
     </article-h3> 
     <source />PLoS computational biology 
     <volume>
       3 
     </volume> 
     <fpage>
       e59 
     </fpage> 
     <pub-id pub-id-type="pmid">
       17447836 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Boutros1"> 
    <label>29</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Boutros 
       </surname> 
       <given-names>
         PC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lau 
       </surname> 
       <given-names>
         SK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pintilie 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liu 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shepherd 
       </surname> 
       <given-names>
         FA 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Prognostic gene signatures for non-small-cell lung cancer. 
     </article-h3> 
     <source />Proc Natl Acad Sci U S A 
     <volume>
       106 
     </volume> 
     <fpage>
       2824 
     </fpage> 
     <lpage>
       2828 
     </lpage> 
     <pub-id pub-id-type="pmid">
       19196983 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Jiang1"> 
    <label>30</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Jiang 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Deng 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chen 
       </surname> 
       <given-names>
         HS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tao 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sha 
       </surname> 
       <given-names>
         Q 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2004 
     </year> 
     <article-h3>
       Joint analysis of two microarray gene-expression data sets to select lung adenocarcinoma marker genes. 
     </article-h3> 
     <source />BMC Bioinformatics 
     <volume>
       5 
     </volume> 
     <fpage>
       81 
     </fpage> 
     <pub-id pub-id-type="pmid">
       15217521 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-vanVliet1"> 
    <label>31</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         van Vliet 
       </surname> 
       <given-names>
         MH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Reyal 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Horlings 
       </surname> 
       <given-names>
         HM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         van de Vijver 
       </surname> 
       <given-names>
         MJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Reinders 
       </surname> 
       <given-names>
         MJ 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       Pooling breast cancer datasets has a synergetic effect on classification performance and improves signature stability. 
     </article-h3> 
     <source />BMC Genomics 
     <volume>
       9 
     </volume> 
     <fpage>
       375 
     </fpage> 
     <pub-id pub-id-type="pmid">
       18684329 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Rhodes1"> 
    <label>32</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Rhodes 
       </surname> 
       <given-names>
         DR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yu 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shanker 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Deshpande 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Varambally 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2004 
     </year> 
     <article-h3>
       Large-scale meta-analysis of cancer microarray data identifies common transcriptional profiles of neoplastic transformation and progression. 
     </article-h3> 
     <source />Proc Natl Acad Sci U S A 
     <volume>
       101 
     </volume> 
     <fpage>
       9309 
     </fpage> 
     <lpage>
       9314 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15184677 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Subramanian1"> 
    <label>33</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Subramanian 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tamayo 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mootha 
       </surname> 
       <given-names>
         VK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mukherjee 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ebert 
       </surname> 
       <given-names>
         BL 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2005 
     </year> 
     <article-h3>
       Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. 
     </article-h3> 
     <source />Proc Natl Acad Sci U S A 
     <volume>
       102 
     </volume> 
     <fpage>
       15545 
     </fpage> 
     <lpage>
       15550 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16199517 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Dundas1"> 
    <label>34</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Dundas 
       </surname> 
       <given-names>
         SR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lawrie 
       </surname> 
       <given-names>
         LC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rooney 
       </surname> 
       <given-names>
         PH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Murray 
       </surname> 
       <given-names>
         GI 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2005 
     </year> 
     <article-h3>
       Mortalin is over-expressed by colorectal adenocarcinomas and correlates with poor survival. 
     </article-h3> 
     <source />J Pathol 
     <volume>
       205 
     </volume> 
     <fpage>
       74 
     </fpage> 
     <lpage>
       81 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15532096 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Kaul1"> 
    <label>35</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Kaul 
       </surname> 
       <given-names>
         SC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Aida 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yaguchi 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kaur 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wadhwa 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2005 
     </year> 
     <article-h3>
       Activation of wild type p53 function by its mortalin-binding, cytoplasmically localizing carboxyl terminus peptides. 
     </article-h3> 
     <source />J Biol Chem 
     <volume>
       280 
     </volume> 
     <fpage>
       39373 
     </fpage> 
     <lpage>
       39379 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16176931 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Wadhwa1"> 
    <label>36</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Wadhwa 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yaguchi 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hasan 
       </surname> 
       <given-names>
         MK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Taira 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kaul 
       </surname> 
       <given-names>
         SC 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2003 
     </year> 
     <article-h3>
       Mortalin-MPD (mevalonate pyrophosphate decarboxylase) interactions and their role in control of cellular proliferation. 
     </article-h3> 
     <source />Biochem Biophys Res Commun 
     <volume>
       302 
     </volume> 
     <fpage>
       735 
     </fpage> 
     <lpage>
       742 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12646231 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Kim1"> 
    <label>37</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Kim 
       </surname> 
       <given-names>
         MM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wiederschain 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kennedy 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hansen 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yuan 
       </surname> 
       <given-names>
         ZM 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2007 
     </year> 
     <article-h3>
       Modulation of p53 and MDM2 activity by novel interaction with Ras-GAP binding proteins (G3BP). 
     </article-h3> 
     <source />Oncogene 
     <volume>
       26 
     </volume> 
     <fpage>
       4209 
     </fpage> 
     <lpage>
       4215 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17297477 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Zhang1"> 
    <label>38</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Zhang 
       </surname> 
       <given-names>
         HZ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liu 
       </surname> 
       <given-names>
         JG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wei 
       </surname> 
       <given-names>
         YP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wu 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cao 
       </surname> 
       <given-names>
         YK 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2007 
     </year> 
     <article-h3>
       Expression of G3BP and RhoC in esophageal squamous carcinoma and their effect on prognosis. 
     </article-h3> 
     <source />World J Gastroenterol 
     <volume>
       13 
     </volume> 
     <fpage>
       4126 
     </fpage> 
     <lpage>
       4130 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17696235 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Oc1"> 
    <label>39</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Oc 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rusch 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Talbot 
       </surname> 
       <given-names>
         SG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sarkaria 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Viale 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2004 
     </year> 
     <article-h3>
       Casein kinase II alpha subunit and C1-inhibitor are independent predictors of outcome in patients with squamous cell carcinoma of the lung. 
     </article-h3> 
     <source />Clin Cancer Res 
     <volume>
       10 
     </volume> 
     <fpage>
       5792 
     </fpage> 
     <lpage>
       5803 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15355908 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Zinszner1"> 
    <label>40</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Zinszner 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Albalat 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ron 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       1994 
     </year> 
     <article-h3>
       A novel effector domain from the RNA-binding protein TLS or EWS is required for oncogenic transformation by CHOP. 
     </article-h3> 
     <source />Genes Dev 
     <volume>
       8 
     </volume> 
     <fpage>
       2513 
     </fpage> 
     <lpage>
       2526 
     </lpage> 
     <pub-id pub-id-type="pmid">
       7958914 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Edgar1"> 
    <label>41</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Edgar 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Domrachev 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lash 
       </surname> 
       <given-names>
         AE 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2002 
     </year> 
     <article-h3>
       Gene Expression Omnibus: NCBI gene expression and hybridization array data repository. 
     </article-h3> 
     <source />Nucleic Acids Res 
     <volume>
       30 
     </volume> 
     <fpage>
       207 
     </fpage> 
     <lpage>
       210 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11752295 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-RDevelopmentCoreTeam1"> 
    <label>42</label> 
    <mixed-citation publication-type="book"> 
     <collab>
       R_Development_Core_Team 
     </collab> 
     <year>
       2010 
     </year> 
     <source />R: A Language and Environement for Statistical Computing 
     <publisher-loc>
       Vienna, Austria 
     </publisher-loc> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Gentleman1"> 
    <label>43</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Gentleman 
       </surname> 
       <given-names>
         RC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Carey 
       </surname> 
       <given-names>
         VJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bates 
       </surname> 
       <given-names>
         DM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bolstad 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dettling 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2004 
     </year> 
     <article-h3>
       Bioconductor: open software development for computational biology and bioinformatics. 
     </article-h3> 
     <source />Genome Biol 
     <volume>
       5 
     </volume> 
     <fpage>
       R80 
     </fpage> 
     <pub-id pub-id-type="pmid">
       15461798 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Irizarry1"> 
    <label>44</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Irizarry 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hobbs 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Collin 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Beazer-Barclay 
       </surname> 
       <given-names>
         YD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Antonellis 
       </surname> 
       <given-names>
         KJ 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2003 
     </year> 
     <article-h3>
       Exploration, normalization, and summaries of high density oligonucleotide array probe level data. 
     </article-h3> 
     <source />Biostatistics 
     <volume>
       4 
     </volume> 
     <fpage>
       249 
     </fpage> 
     <lpage>
       264 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12925520 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Ramasamy1"> 
    <label>45</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ramasamy 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mondry 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Holmes 
       </surname> 
       <given-names>
         CC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Altman 
       </surname> 
       <given-names>
         DG 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       Key Issues in Conducting a Meta-Analysis of Gene Expression Microarray Datasets. 
     </article-h3> 
     <source />Plos Medicine 
     <volume>
       5 
     </volume> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Saeed1"> 
    <label>46</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Saeed 
       </surname> 
       <given-names>
         AI 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sharov 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         White 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Li 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liang 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2003 
     </year> 
     <article-h3>
       TM4: a free, open-source system for microarray data management and analysis. 
     </article-h3> 
     <source />Biotechniques 
     <volume>
       34 
     </volume> 
     <fpage>
       374 
     </fpage> 
     <lpage>
       378 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12613259 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Storey1"> 
    <label>47</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Storey 
       </surname> 
       <given-names>
         JD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tibshirani 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2003 
     </year> 
     <article-h3>
       Statistical significance for genomewide studies. 
     </article-h3> 
     <source />Proceedings of the National Academy of Sciences of the United States of America 
     <volume>
       100 
     </volume> 
     <fpage>
       9440 
     </fpage> 
     <lpage>
       9445 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12883005 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Peri1"> 
    <label>48</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Peri 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Navarro 
       </surname> 
       <given-names>
         JD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Amanchy 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kristiansen 
       </surname> 
       <given-names>
         TZ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jonnalagadda 
       </surname> 
       <given-names>
         CK 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2003 
     </year> 
     <article-h3>
       Development of human protein reference database as an initial platform for approaching systems biology in humans. 
     </article-h3> 
     <source />Genome Res 
     <volume>
       13 
     </volume> 
     <fpage>
       2363 
     </fpage> 
     <lpage>
       2371 
     </lpage> 
     <pub-id pub-id-type="pmid">
       14525934 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Shannon1"> 
    <label>49</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Shannon 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Markiel 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ozier 
       </surname> 
       <given-names>
         O 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Baliga 
       </surname> 
       <given-names>
         NS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wang 
       </surname> 
       <given-names>
         JT 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2003 
     </year> 
     <article-h3>
       Cytoscape: a software environment for integrated models of biomolecular interaction networks. 
     </article-h3> 
     <source />Genome research 
     <volume>
       13 
     </volume> 
     <fpage>
       2498 
     </fpage> 
     <lpage>
       2504 
     </lpage> 
     <pub-id pub-id-type="pmid">
       14597658 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Assenov1"> 
    <label>50</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Assenov 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ramirez 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Schelhorn 
       </surname> 
       <given-names>
         SE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lengauer 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Albrecht 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       Computing topological parameters of biological networks. 
     </article-h3> 
     <source />Bioinformatics (Oxford, England) 
     <volume>
       24 
     </volume> 
     <fpage>
       282 
     </fpage> 
     <lpage>
       284 
     </lpage> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0014635-Huangda1"> 
    <label>51</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Huang da 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sherman 
       </surname> 
       <given-names>
         BT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lempicki 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. 
     </article-h3> 
     <source />Nat Protoc 
     <volume>
       4 
     </volume> 
     <fpage>
       44 
     </fpage> 
     <lpage>
       57 
     </lpage> 
     <pub-id pub-id-type="pmid">
       19131956 
     </pub-id> 
    </mixed-citation> 
   </ref> 
  </ref-list> 
 </back> 
</article>