<article article-type="research-article"> 
 <!--?properties open_acces=""?--> 
 <!-- Original-type: oa--> 
 <front> 
  <journal-meta> 
   <journal-id journal-id-type="nlm-ta">
     Br J Cancer 
   </journal-id> 
   <journal-h3>
     British Journal of Cancer 
   </journal-h3> 
   <issn pub-type="ppub">
     0007-0920 
   </issn> 
   <issn pub-type="epub">
     1532-1827 
   </issn> 
   <publisher> 
    <publisher-name>
      Nature Publishing Group 
    </publisher-name> 
   </publisher> 
  </journal-meta> 
  <article-meta> 
   <article-id pub-id-type="pmid">
     20147967 
   </article-id> 
   <article-id pub-id-type="pmc">
     2837563 
   </article-id> 
   <article-id pub-id-type="pii">
     6605534 
   </article-id> 
   <article-id pub-id-type="doi">
     10.1038/sj.bjc.6605534 
   </article-id> 
   <article-categories> 
    <subj-group subj-group-type="heading"> 
     <subject>
       Translational Therapeutics 
     </subject> 
    </subj-group> 
   </article-categories> 
   <h3-group> 
    <article-h3>
      Activating 
     <italic>
       K-Ras 
     </italic> mutations outwith ‘hotspot' codons in sporadic colorectal tumours – implications for personalised cancer medicine 
    </article-h3> 
   </h3-group> 
   <contrib-group> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Smith 
      </surname> 
      <given-names>
        G 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1">
       1 
     </xref> 
     <xref ref-type="author-notes" rid="note1">
       5 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Bounds 
      </surname> 
      <given-names>
        R 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1">
       1 
     </xref> 
     <xref ref-type="author-notes" rid="note1">
       5 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Wolf 
      </surname> 
      <given-names>
        H 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1">
       1 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Steele 
      </surname> 
      <given-names>
        R J C 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff2">
       2 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Carey 
      </surname> 
      <given-names>
        F A 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff3">
       3 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Wolf 
      </surname> 
      <given-names>
        C R 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1">
       1 
     </xref> 
     <xref ref-type="aff" rid="aff4">
       4 
     </xref> 
     <xref ref-type="corresp" rid="caf1">
       * 
     </xref> 
    </contrib> 
   </contrib-group> 
   <aff id="aff1"> 
    <label>1</label> 
    <institution>
      Biomedical Research Institute, Ninewells Hospital and Medical School 
    </institution> Dundee DD1 9SY, 
    <country>
      UK 
    </country> 
   </aff> 
   <aff id="aff2"> 
    <label>2</label> 
    <institution>
      Department of Surgery and Molecular Oncology, University of Dundee 
    </institution> Dundee DD1 9SY, 
    <country>
      UK 
    </country> 
   </aff> 
   <aff id="aff3"> 
    <label>3</label> 
    <institution>
      Department of Molecular and Cellular Pathology, Ninewells Hospital and Medical School 
    </institution> Dundee DD1 9SY, 
    <country>
      UK 
    </country> 
   </aff> 
   <aff id="aff4"> 
    <label>4</label> 
    <institution>
      CRUK Molecular Pharmacology Unit, Ninewells Hospital and Medical School 
    </institution> Dundee DD1 9SY, 
    <country>
      UK 
    </country> 
   </aff> 
   <author-notes> 
    <corresp id="caf1"> 
     <label>*</label>Author for correspondence: 
     <email xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="mailto:c.r.wolf@dundee.ac.uk">
       c.r.wolf@dundee.ac.uk 
     </email> 
    </corresp> 
    <fn fn-type="present-address" id="note1"> 
     <label>5</label> 
     <p>Joint first authors</p> 
    </fn> 
   </author-notes> 
   <pub-date pub-type="epub"> 
    <day>
      16 
    </day> 
    <month>
      02 
    </month> 
    <year>
      2010 
    </year> 
   </pub-date> 
   <pub-date pub-type="ppub"> 
    <day>
      16 
    </day> 
    <month>
      02 
    </month> 
    <year>
      2010 
    </year> 
   </pub-date> 
   <volume>
     102 
   </volume> 
   <issue>
     4 
   </issue> 
   <fpage>
     693 
   </fpage> 
   <lpage>
     703 
   </lpage> 
   <history> 
    <date date-type="rev-recd"> 
     <day>
       10 
     </day> 
     <month>
       12 
     </month> 
     <year>
       2009 
     </year> 
    </date> 
    <date date-type="accepted"> 
     <day>
       16 
     </day> 
     <month>
       12 
     </month> 
     <year>
       2009 
     </year> 
    </date> 
   </history> 
   <copyright-statement>
     Copyright 2010, Cancer Research UK 
   </copyright-statement> 
   <copyright-year>
     2010 
   </copyright-year> 
   <permissions> 
    <copyright-holder>
      Cancer Research UK 
    </copyright-holder> 
   </permissions> 
   <abstract>
    <h3>Abstract</h3> 
    <sec> 
     <h3>Background:</h3> 
     <p>Response to 
      <italic>
        EGFR 
      </italic>-targeted therapies in colorectal cancer patients has been convincingly associated with Kirsten-Ras ( 
      <italic>
        K-Ras 
      </italic>) mutation status. Current mandatory mutation testing for patient selection is limited to the 
      <italic>
        K-Ras 
      </italic> ‘hotspot' codons 12 and 13.</p> 
    </sec> 
    <sec> 
     <h3>Methods:</h3> 
     <p>Colorectal tumours ( 
      <italic>
        n 
      </italic>=106) were screened for additional 
      <italic>
        K-Ras 
      </italic> mutations, phenotypes compared in transformation and Ras GTPase activating assays and gene and pathway changes induced by individual 
      <italic>
        K-Ras 
      </italic> mutants identified by microarray analysis. Taqman-based gene copy number and FISH analyses were used to investigate 
      <italic>
        K-Ras 
      </italic> gene amplification.</p> 
    </sec> 
    <sec> 
     <h3>Results:</h3> 
     <p>Four additional 
      <italic>
        K-Ras 
      </italic> mutations (Leu<sub>19</sub>Phe (1 out of 106 tumours), Lys<sub>117</sub>Asn (1 out of 106), Ala<sub>146</sub>Thr (7 out of 106) and Arg<sub>164</sub>Gln (1 out of 106)) were identified. Lys<sub>117</sub>Asn and Ala<sub>146</sub>Thr had phenotypes similar to the hotspot mutations, whereas Leu<sub>19</sub>Phe had an attenuated phenotype and the Arg<sub>164</sub>Gln mutation was phenotypically equivalent to wt 
      <italic>
        K-Ras 
      </italic>. We additionally identified a new 
      <italic>
        K-Ras 
      </italic> gene amplification event, present in approximately 2% of tumours.</p> 
    </sec> 
    <sec> 
     <h3>Conclusions:</h3> 
     <p>The identification of mutations outwith previously described hotspot codons increases the 
      <italic>
        K-Ras 
      </italic> mutation burden in colorectal tumours by one-third. Future mutation screening to facilitate optimal patient selection for treatment with 
      <italic>
        EGFR 
      </italic>-targeted therapies should therefore be extended to codon 146, and in addition should consider the unique molecular signatures associated with individual 
      <italic>
        K-Ras 
      </italic> mutations.</p> 
    </sec> 
   </abstract> 
   <kwd-group> 
    <kwd>
      K-Ras 
    </kwd> 
    <kwd>
      mutation 
    </kwd> 
    <kwd>
      colorectal tumour 
    </kwd> 
    <kwd>
      gene amplification 
    </kwd> 
    <kwd>
      personalised medicine 
    </kwd> 
   </kwd-group> 
  </article-meta> 
 </front> 
 <p>Colorectal cancer accounts for approximately 15% of all cancers diagnosed annually in the United Kingdom, and is a major cause of death due to cancer, second only to lung and breast cancer ( 
  <ext-link ext-link-type="uri" xlink:href="http://info.cancerresearchuk.org/cancerstats/incidence">
    http://info.cancerresearchuk.org/cancerstats/incidence 
  </ext-link>).</p> 
 <p>The majority of colorectal tumours arise from adenomatous polyps (adenomas), benign precursor lesions that develop from normal colonic mucosa. However, fewer than 10% of colorectal adenomas develop into invasive cancers and a number of molecular mechanisms including epigenetic events, DNA mismatch-repair defects, chromosomal rearrangements and mutations in key oncogenes and tumour suppressor genes have been suggested to regulate progression from adenoma to adenocarcinoma ( 
  <xref ref-type="other" rid="bib10">
    de la Chapelle, 2004 
  </xref>).</p> 
 <p>A genetic model for colorectal cancer highlighted key genes, including the tumour suppressor genes adenomatous polyposis coli ( 
  <italic>
    APC 
  </italic>) and 
  <italic>
    p53 
  </italic> and the oncogene Kirsten-Ras ( 
  <italic>
    K-Ras 
  </italic>), the progressive acquisition of mutations in which was proposed to regulate the adenoma-carcinoma transition ( 
  <xref ref-type="other" rid="bib37">
    Vogelstein 
   <italic>
     et al 
   </italic>, 1988 
  </xref>). We subsequently refined this model ( 
  <xref ref-type="other" rid="bib33">
    Smith 
   <italic>
     et al 
   </italic>, 2002 
  </xref>), highlighting a significant increase in 
  <italic>
    K-Ras 
  </italic> mutation frequency in Dukes' C tumours, suggesting that 
  <italic>
    K-Ras 
  </italic> mutation status may be an important determinant of tumour progression. We additionally used comparative genomic hybridisation (CGH) analysis to identify common chromosomal aberrations in colorectal tumours, and highlighted an amplification of the region of chromosome 12p where the 
  <italic>
    K-Ras 
  </italic> gene is localised ( 
  <xref ref-type="other" rid="bib24">
    Leslie 
   <italic>
     et al 
   </italic>, 2003 
  </xref>). We and others have shown that 
  <italic>
    K-Ras 
  </italic> mutations are associated with significantly reduced survival in colorectal cancer patients ( 
  <xref ref-type="other" rid="bib2">
    Andreyev 
   <italic>
     et al 
   </italic>, 2001 
  </xref>; 
  <xref ref-type="other" rid="bib7">
    Conlin 
   <italic>
     et al 
   </italic>, 2005 
  </xref>), although previous data is not entirely consistent ( 
  <xref ref-type="other" rid="bib15">
    Etienne-Grimaldi 
   <italic>
     et al 
   </italic>, 2008 
  </xref>; 
  <xref ref-type="other" rid="bib41">
    Winder 
   <italic>
     et al 
   </italic>, 2009 
  </xref>). 
  <italic>
    K-Ras 
  </italic> mutation status has recently been convincingly associated with response to the new generation 
  <italic>
    EGFR 
  </italic> antagonists cetuximab (Erbitux) and panitumumab (Vectibix), where response is preferentially observed in wt 
  <italic>
    K-Ras 
  </italic> tumours ( 
  <xref ref-type="other" rid="bib25">
    Lievre 
   <italic>
     et al 
   </italic>, 2006 
  </xref>; 
  <xref ref-type="other" rid="bib4">
    Benvenuti 
   <italic>
     et al 
   </italic>, 2007 
  </xref>; 
  <xref ref-type="other" rid="bib22">
    Khambata-Ford 
   <italic>
     et al 
   </italic>, 2007 
  </xref>; 
  <xref ref-type="other" rid="bib16">
    Freeman 
   <italic>
     et al 
   </italic>, 2008 
  </xref>; 
  <xref ref-type="other" rid="bib20">
    Karapetis 
   <italic>
     et al 
   </italic>, 2008 
  </xref>; 
  <xref ref-type="other" rid="bib31">
    Ramos 
   <italic>
     et al 
   </italic>, 2008 
  </xref>; 
  <xref ref-type="other" rid="bib26">
    Loupakis 
   <italic>
     et al 
   </italic>, 2009 
  </xref>; 
  <xref ref-type="other" rid="bib35">
    Van Cutsem 
   <italic>
     et al 
   </italic>, 2009 
  </xref>). 
  <italic>
    K-Ras 
  </italic> mutation testing is therefore increasingly recommended to facilitate selection of the most appropriate patients for treatment with 
  <italic>
    EGFR 
  </italic> antagonists ( 
  <xref ref-type="other" rid="bib27">
    McNeill, 2008 
  </xref>; 
  <xref ref-type="other" rid="bib36">
    van Krieken and Tol, 2009 
  </xref>).</p> 
 <p> 
  <italic>
    K-Ras 
  </italic> is a member of the highly homologous family of small p21<sup>Ras</sup> G proteins ( 
  <italic>
    H-Ras 
  </italic>, 
  <italic>
    N-Ras 
  </italic> and 
  <italic>
    K-Ras 
  </italic>) which transduce signals across the plasma membrane, principally by activation of the RAS–RAF–MEK–ERK–MAPK signalling cascade ( 
  <xref ref-type="fig" rid="fig1">
    Figure 1 
  </xref>; ( 
  <xref ref-type="other" rid="bib3">
    Barbacid, 1990 
  </xref>)). Ras genes are the most frequently mutated oncogenes in human cancer, where tumour-specific mutations lead to the permanent activation of Ras signalling cascades, influencing proliferation, differentiation and apoptosis ( 
  <xref ref-type="other" rid="bib6">
    Bos, 1989 
  </xref>). Previous analyses of 
  <italic>
    K-Ras 
  </italic> mutations in human tumours have consistently focussed on single-point mutations in codons 12, 13 and 61, where mutation has been shown to result in reduced Ras GAP GTPase activity, locking the protein in the active Ras-GTP conformation ( 
  <xref ref-type="other" rid="bib14">
    Ellis and Clark, 2000 
  </xref>), although additional mutations at codons 19, 22 and 146 have been described in single colorectal tumour case reports ( 
  <xref ref-type="other" rid="bib29">
    Orita 
   <italic>
     et al 
   </italic>, 1991 
  </xref>; 
  <xref ref-type="other" rid="bib28">
    Miyakura 
   <italic>
     et al 
   </italic>, 2002 
  </xref>; 
  <xref ref-type="other" rid="bib1">
    Akagi 
   <italic>
     et al 
   </italic>, 2007 
  </xref>).</p> 
 <p>Ras proteins regulate signal transduction by activating a number of downstream effector proteins, including the cytoplasmic serine/threonine protein kinase B-Raf ( 
  <xref ref-type="fig" rid="fig1">
    Figure 1 
  </xref>). 
  <italic>
    B-Raf 
  </italic> mutations have also been identified in human cancers, including melanoma and thyroid, ovarian and colorectal tumours, although mutation frequency estimates in colorectal tumours vary from 1 to 20%, and have been particularly associated with tumours deficient in mismatch-repair activity ( 
  <xref ref-type="other" rid="bib40">
    Wellbrock 
   <italic>
     et al 
   </italic>, 2004 
  </xref>). The most common 
  <italic>
    B-Raf 
  </italic> mutation is a T to A transversion, resulting in a valine to glutamic acid substitution (V600E), present in approximately 90% of 
  <italic>
    B-Raf 
  </italic> mutant tumours ( 
  <xref ref-type="other" rid="bib9">
    Davies 
   <italic>
     et al 
   </italic>, 2002 
  </xref>), which results in a 500-fold increase in 
  <italic>
    in vitro 
  </italic> kinase activity and the induction of cell proliferation and transformation ( 
  <xref ref-type="other" rid="bib38">
    Wan 
   <italic>
     et al 
   </italic>, 2004 
  </xref>). Although 
  <italic>
    K-Ras 
  </italic> and 
  <italic>
    B-Raf 
  </italic> mutations are found in the same tumour types, they are thought to be mutually exclusive ( 
  <xref ref-type="other" rid="bib40">
    Wellbrock 
   <italic>
     et al 
   </italic>, 2004 
  </xref>), providing distinct but related mechanisms for the activation of 
  <italic>
    K-Ras 
  </italic> signalling pathways. 
  <italic>
    B-Raf 
  </italic> mutation status has also recently been associated with response to cetuximab and panitumumab where, like 
  <italic>
    K-Ras 
  </italic>, clinical response is limited to wt 
  <italic>
    B-Raf 
  </italic> tumours ( 
  <xref ref-type="other" rid="bib12">
    Di Nicolantonio 
   <italic>
     et al 
   </italic>, 2008 
  </xref>).</p> 
 <p>As 
  <italic>
    K-Ras 
  </italic> mutations in colorectal tumours are clearly important biomarkers of tumour progression and patient survival and also influence response to new generation 
  <italic>
    EGFR 
  </italic> antagonists, we have used a variety of experimental approaches to further investigate inter-individual differences in 
  <italic>
    K-Ras 
  </italic> mutation burden.</p> 
 <sec sec-type="materials|methods"> 
  <h3>Materials and methods</h3> 
  <sec> 
   <h3>Study participants</h3> 
   <p>Patients (age range 45–80 years, median age 67 years, 64 males and 42 females) undergoing surgery for colorectal cancer at Ninewells Hospital, Dundee or Perth Royal Infirmary were invited to participate in the study. All patients were Caucasian, had pre-operative pathological confirmation of diagnosis (ICD-9 classification 153.0–153.9, 154.0–154.1) and had no history of previous cancer, inflammatory bowel disease, ulcerative colitis or diverticular disease. Patient details have been described previously ( 
    <xref ref-type="other" rid="bib33">
      Smith 
     <italic>
       et al 
     </italic>, 2002 
    </xref>). The study was approved by the Tayside Committee on Medical Research Ethics and written informed consent was obtained from all study participants.</p> 
  </sec> 
  <sec> 
   <h3>DNA preparation</h3> 
   <p>Primary colorectal tumour resection specimens were brought fresh from theatre to pathology and tumours selected by an experienced pathologist and stored in liquid nitrogen before analysis. Genomic DNA for mutation analysis was extracted using the Wizard Genomic DNA Purification Kit (Promega, Southampton, UK) according to the manufacturer's instructions. All study participants provided a 10?ml venous blood sample, which was stored in EDTA blood containers at ?20°C. Genomic DNA was extracted from 200? 
    <italic>
      ? 
    </italic>l of whole blood, using a QIAamp 96 spin blood kit (Qiagen, Crawley, West Sussex, UK) according to the manufacturer's instructions.</p> 
  </sec> 
  <sec> 
   <h3> 
    <italic>
      K-Ras 
    </italic> and 
    <italic>
      B-Raf 
    </italic> mutation detection</h3> 
   <p>Methods for 
    <italic>
      K-Ras 
    </italic> and 
    <italic>
      B-Raf 
    </italic> mutation detection and restriction fragment length polymorphism (RFLP) analysis of mutation frequencies are given in 
    <xref ref-type="app" rid="app1">
      Appendix A 
    </xref>.</p> 
  </sec> 
  <sec> 
   <h3>Cell culture and transfection</h3> 
   <p>The mouse embryonic fibroblast cell line NIH3T3 was obtained from Cancer Research UK cell services. NIH3T3 cells were maintained in Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% calf serum unless otherwise stated, at 37°C in 5% CO<sub>2</sub>. Cells were transfected using Lipofectamine (Invitrogen, Paisley, UK) according to the manufacturer's instructions.</p> 
  </sec> 
  <sec> 
   <h3>Mammalian expression plasmids</h3> 
   <p>The pEF.6 plink and pEFHA wt 
    <italic>
      K-Ras 
    </italic> plasmids were kind gifts from Professor Richard Marais (Institute of Cancer Research, London, UK). Site-directed mutagenesis (SDM) was carried out on the pEFHA wt 
    <italic>
      K-Ras 
    </italic> plasmid using the QuikChange II SDM kit (Agilent, South Queensferry, Edinburgh, UK) according to the manufacturer's instructions to generate the following 
    <italic>
      K-Ras 
    </italic> mutations: G<sub>12</sub>V, G<sub>12</sub>D, G<sub>13</sub>D, Q<sub>61</sub>H, L<sub>19</sub>F, K<sub>117</sub>N, A<sub>146</sub>T and R<sub>164</sub>Q, using the oligonucleotide primers described in 
    <xref ref-type="table" rid="tbl1">
      Table 1 
    </xref>.</p> 
  </sec> 
  <sec> 
   <h3> 
    <italic>
      K-Ras 
    </italic> foci formation assays</h3> 
   <p>NIH3T3 cells were seeded at 2.5 × 10<sup>5</sup> cells in 35?mm culture dishes and allowed to adhere overnight before transfection with 250?ng of either pEF.6, pEFHA (wt 
    <italic>
      K-Ras 
    </italic>) or plasmids containing one of the following 
    <italic>
      K-Ras 
    </italic> mutants: G<sub>12</sub>V, G<sub>12</sub>D, G<sub>13</sub>D, Q<sub>61</sub>H, L<sub>19</sub>F, K<sub>117</sub>N, A<sub>146</sub>T or R<sub>164</sub>Q. After 24?h, cells were trypsinised and split (1?:?3) between two 10?cm<sup>3</sup> plates and grown in DMEM containing 5% calf serum for 21 days, then fixed with methanol and stained with 0.5% (w/v) crystal violet. Foci greater then 5?mm were counted manually. All transfections were repeated three times and average foci counts and standard deviations calculated.</p> 
  </sec> 
  <sec> 
   <h3>Ras GTPase activating assays</h3> 
   <p>Ras activity was determined using a Ras activation assay kit (Millipore, Watford, Hertfordshire, UK). Briefly, cells were harvested 48?h after transfection in an Mg-containing lysis buffer (MLB). 500? 
    <italic>
      ? 
    </italic>g of cell lysate from each sample was incubated with 10? 
    <italic>
      ? 
    </italic>g of Raf-1 RBD agarose at 4°C for 30?min with gentle rocking. After washing three times with MLB, the agarose beads were re-suspended in 40? 
    <italic>
      ? 
    </italic>l of NuPage LDS sample buffer (Invitrogen), and 20? 
    <italic>
      ? 
    </italic>l of each sample separated on SDS-PAGE. A total of 30? 
    <italic>
      ? 
    </italic>g of whole-cell lysate was additionally separated by SDS–PAGE for the analysis of total K-Ras expression.</p> 
  </sec> 
  <sec> 
   <h3>Western blot analysis</h3> 
   <p>Cells were lysed 48?h after transfection with MKK lysis buffer (20?m 
    <sc>
      M 
    </sc> Tris-acetate, 1?m 
    <sc>
      M 
    </sc> EDTA, 1% (v/v) Triton X-100, 1?m 
    <sc>
      M 
    </sc> EGTA, 0.1% (v/v) 
    <italic>
      ? 
    </italic>-mercaptoethanol, 1?m 
    <sc>
      M 
    </sc> sodium orthovanadate, 1?m 
    <sc>
      M 
    </sc> sodium pyrophosphate, 1?m 
    <sc>
      M 
    </sc> sodium 
    <italic>
      ? 
    </italic>-glycerophosphate, 5?m 
    <sc>
      M 
    </sc> sodium fluoride and protease inhibitors). Cell lysates were centrifuged at 13?000?r.p.m. for 15?min at 4°C and the soluble fraction transferred to a fresh 1.5?ml Eppendorf tube. Protein concentration was determined using the Bio-Rad protein assay (Bio-Rad, Hemel Hempstead, Hertfordshire, UK). For western blot analysis, 30? 
    <italic>
      ? 
    </italic>g of each cell lysate was separated using the NuPage electrophoresis system (12% pre-cast polyacrylamide gels, Invitrogen) and transferred to nitrocellulose membranes (Whatman, Maidstone, Kent, UK). Membranes were probed with anti-K-Ras (Merck, Nottingham, UK) and anti-actin (Santa Cruz Biotechnology, Heidelberg, Germany) antibodies overnight at 4°C. Secondary horseradish peroxidase-conjugated antibodies (anti-mouse, Dako or anti-goat, Santa Cruz Biotechnology) were applied for 1?h at room temperature before developing the membranes using enhanced chemiluminescence.</p> 
  </sec> 
  <sec> 
   <h3>RNA transcription profiling analysis</h3> 
   <p>NIH3T3 cells were plated at 2.5 × 10<sup>5</sup> cells per well in a 6-well plate and, following overnight attachment, transfected with 250?ng of pEF, wt 
    <italic>
      K-Ras 
    </italic> or one of the 
    <italic>
      K-Ras 
    </italic> mutants: G12V, G12D, G12C, G13D, Q61H, L19F, K117N, A146T and R164Q. After 48?h, cells were harvested in 1?ml of TRIzol reagent (Invitrogen) and total RNA extracted according to the manufacturer's instructions. RNA was further purified using an RNeasy mini kit (Qiagen), eluted in a final volume of 50? 
    <italic>
      ? 
    </italic>l RNAse-free dH<sub>2</sub>0 and RNA quality and concentration assessed with a Bioanalyzer 2100 using the RNA 6000 Nano LabChip Kit (Agilent). RNA labelling and microarray hybridisation was carried out in collaboration with CXR Biosciences, Dundee, UK. Briefly 1? 
    <italic>
      ? 
    </italic>g of total RNA sample was amplified to generate complementary RNA (cRNA) and labelled with cyanine 3-CTP (Cys3) using the QuickAmp labelling kit, one-colour (Agilent). Labelled cRNA was hybridised to Agilent 4 × 44K Whole Mouse Genome Oligo Microarray slides, which were scanned on an Agilent Microarray Scanner and images processed using Agilent Feature Extraction Software v9.1.</p> 
   <p>Microarray data was analysed using the open-source software Bioconducter 2.2. Lists of differentially expressed genes were generated comparing each of the 
    <italic>
      K-Ras 
    </italic> mutants with empty vector control using probes that exhibited an adjusted p value (false discovery rate (FDR)) ?0.05. Hierarchical clustering analysis was carried out using ‘Cluster' (open source clustering software) and clusters visualised using Java TreeView.</p> 
  </sec> 
  <sec> 
   <h3> 
    <italic>
      K-Ras 
    </italic> gene copy number assay</h3> 
   <p>A new Taqman gene expression assay to evaluate 
    <italic>
      K-Ras 
    </italic> gene copy number was designed, in which 
    <italic>
      K-Ras 
    </italic> copy number was compared with the endogenous control gene 
    <italic>
      RNAse P 
    </italic> (copy number 2). 
    <italic>
      K-Ras 
    </italic> was amplified from genomic DNA extracted from normal and tumour tissues ( 
    <italic>
      n 
    </italic>=96 normal/tumour pairs) from our colorectal patient series ( 
    <xref ref-type="other" rid="bib33">
      Smith 
     <italic>
       et al 
     </italic>, 2002 
    </xref>) with the oligonucleotide primers 5?-TTTAATACTTTTTATGTATTTCAGGGTGTTG-3? (300?n 
    <sc>
      M 
    </sc>) and 5?-TTACCATCTTTGCTCATCTTTTCTTTAT-3? (300?n 
    <sc>
      M 
    </sc>) and Taqman probe 5?-FAM-TGATGCCTTCTATACATTAGTTCGAGAAATTCGAAAA-TAMRA-3? (100?n 
    <sc>
      M 
    </sc>), and expression compared with RNAse P by the comparative Ct method, according to the manufacturer's instructions (Applied Biosystems, Warrington, UK).</p> 
  </sec> 
  <sec> 
   <h3>Fluorescent 
    <italic>
      in-situ 
    </italic> hybridisation (FISH) analysis</h3> 
   <p>Sections (4? 
    <italic>
      ? 
    </italic> 
    <sc>
      M 
    </sc>) of formalin-fixed paraffin-embedded tumour tissue were provided by Tayside Tissue Bank on positively charged slides, which had been baked overnight at 56°C. Sections were deparaffinised using an automated protocol on a Vysis VP2000 processor, placed into a coplin-containing pre-warmed pre-treatment solution (Vysis, Maidenhead, Kent, UK) and incubated at 80°C for 30?min. Slides were then washed at room temperature in water for 1?min, before two 5?min washes in 2 × SSC before digestion with protease buffer at 37°C for 65?min. Following digestion, slides were washed three times in 2 × SSC for 2?min, before sequential washes in 70, 85 and 100% ethanol for 2?min each at room temperature and dried on a 37°C block. Meanwhile, 1? 
    <italic>
      ? 
    </italic>l chromosome 12p BAC RP11-707G18 (Red) (BlueGnome), 1? 
    <italic>
      ? 
    </italic>l chromosome 12q BAC RP11-89H19 (Green) (BlueGnome) and 9? 
    <italic>
      ? 
    </italic>l LSI–WCP hybridisation buffer (Vysis) was mixed together for each slide and pipetted onto a 22 × 22?mm coverslip which was placed onto a region of the slide containing tumour tissue and placed onto a heat block at 75°C for 5?min to co-denature the probes and tumour tissue. Rubber cement was then applied to seal the coverslips and slides placed into a humidified hybridisation chamber in a 37°C incubator overnight. The next morning, coverslips were removed and slides placed into 0.4 × SSC at 72°C for 90?s and transferred to 2 × SSC containing 0.005% Tween for 30?s before being left to dry in the dark. In all, 20? 
    <italic>
      ? 
    </italic>l of DAPI stain was then applied to coverslips, which were then gently applied to each slide and, after removing air bubbles, sealed with clear nail varnish. Slides were analysed on an epifluorescence microscope (Olympus BX60), where probe signals from 20 different cells in three different regions of the tumour tissue were counted and the average number of probe signals and s.d. calculated.</p> 
  </sec> 
 </sec> 
 <sec> 
  <h3>Results</h3> 
  <sec> 
   <h3>Identification of new 
    <italic>
      K-Ras 
    </italic> mutations</h3> 
   <p>We have previously described 
    <italic>
      K-Ras 
    </italic> mutations at codons 12, 13 and 61 in a series of 106 unselected colorectal tumours ( 
    <xref ref-type="other" rid="bib33">
      Smith 
     <italic>
       et al 
     </italic>, 2002 
    </xref>). Our 
    <italic>
      K-Ras 
    </italic> mutation analysis was performed by direct sequencing of 
    <italic>
      K-Ras 
    </italic> exon 1 (codons 12 and 13) and exon 2 (codon 61), and would therefore have detected any additional mutations in these exons. Codon 12 mutations were detected in 23 tumours (21.7%) and codon 13 mutations in 6 tumours (5.7%), whereas no codon 61 mutations were detected. Mutations at codon 19 (G<sub>57</sub>T, Leu<sub>19</sub>Phe) and in 
    <italic>
      B-Raf 
    </italic> (V600E) were found in single tumours ( 
    <xref ref-type="table" rid="tbl2">
      Table 2 
    </xref>).</p> 
   <p>To identify additional 
    <italic>
      K-Ras 
    </italic> mutations, and to establish the relative frequencies of individual 
    <italic>
      K-Ras 
    </italic> mutations in human colorectal tumours, we used WAVE analysis followed by direct sequencing to screen the same tumour series for mutations in 
    <italic>
      K-Ras 
    </italic> exon 3 and exon 4B, the most common exon 4 splice variant. Four additional sequence changes were identified – an A to C change (Lys to Asn substitution) at codon 117, a G to A change (Ala to Thr substitution) at codon 146, a G to A change (Arg to Gln substitution) at codon 164 and a ‘silent' C to T nucleotide substitution, which did not alter the aspartic acid residue at codon 173 ( 
    <xref ref-type="table" rid="tbl2">
      Table 2 
    </xref>). Together, these 
    <italic>
      K-Ras 
    </italic> mutations and the 
    <italic>
      B-Raf 
    </italic> V600E mutation increase overall Ras pathway mutation frequency from 27.4 to 37.8%. The predicted localisation of each mutation in the functional domains of the 
    <italic>
      K-Ras 
    </italic> protein is illustrated in 
    <xref ref-type="fig" rid="fig2">
      Figure 2 
    </xref>.</p> 
   <p>To determine whether our reported sequence changes represented tumour-specific mutations or single-nucleotide polymorphisms (SNPs), genomic DNA from blood and tumour tissue was compared. Sequence changes at codons 19, 117, 146 and 164 were only detected in tumour DNA, whereas the codon 173 sequence change was detected in both blood and tumour DNA, and therefore represented a SNP ( 
    <xref ref-type="table" rid="tbl2">
      Table 2 
    </xref>).</p> 
   <p>To determine the frequencies of the codon 117, 146 and 164 
    <italic>
      K-Ras 
    </italic> mutations and the codon 173 SNP, PCR–RFLP assays were designed to permit rapid screening of blood and tumour DNA ( 
    <xref ref-type="app" rid="app1">
      Appendix A 
    </xref>). Like the codon 19 mutation, the codon 117 and codon 164 mutations were found in single tumours, whereas the codon 146 mutation was found in 7 out of 106 (6.5%) of tumours. No tumours with codon 19, 117, 146 or 146 mutations had additional 
    <italic>
      K-Ras 
    </italic> mutations in codons 12, 13 or 61. The codon 173 SNP was found in 39 individuals, predicting an allele frequency of 18.2%. As this SNP did not lead to an amino acid substitution in the 
    <italic>
      K-Ras 
    </italic> protein, it was not analysed further.</p> 
  </sec> 
  <sec> 
   <h3>Comparison of phenotypes associated with hotspot and new 
    <italic>
      K-Ras 
    </italic> mutations</h3> 
   <sec> 
    <h3>Focus formation assays</h3> 
    <p>Phenotypes associated with the various 
     <italic>
       K-Ras 
     </italic> mutations have previously not been systematically evaluated. To compare the transformation potential of the 
     <italic>
       K-Ras 
     </italic> mutants, therefore, NIH3T3 cells were transiently transfected with plasmids expressing wt 
     <italic>
       K-Ras 
     </italic> and the 
     <italic>
       K-Ras 
     </italic> mutations G12V, G12D, G13D, Q61H, L19F, K117N, A146T and R164Q. Equivalence of plasmid loading was assessed spectrophotometrically and by western blotting for 
     <italic>
       K-Ras 
     </italic> (data not shown). Cells were stained with crystal violet and foci counted 21 days after transfection, as described in Materials and Methods ( 
     <xref ref-type="fig" rid="fig3">
       Figure 3 
     </xref>). No foci were observed in untransfected cells or in cells transfected with wt 
     <italic>
       K-Ras 
     </italic>, whereas abundant foci were seen with each hotspot 
     <italic>
       K-Ras 
     </italic> mutation ( 
     <xref ref-type="fig" rid="fig3">
       Figure 3A 
     </xref>). Codon 12 mutations had slightly greater transforming potential than codon 13 mutations and consistently greater transforming potential than the codon 61 Q61H mutation ( 
     <xref ref-type="fig" rid="fig3">
       Figure 3C 
     </xref>). After transfection of additional 
     <italic>
       K-Ras 
     </italic> mutations, significant focus formation was observed for codon 117 and 146 mutations ( 
     <xref ref-type="fig" rid="fig3">
       Figure 3B 
     </xref>). In contrast, the codon 164 mutation was phenotypically equivalent to wt 
     <italic>
       K-ras 
     </italic> with no evidence of foci formation and the codon 19 mutation generated low but consistent numbers of isolated foci ( 
     <xref ref-type="fig" rid="fig3">
       Figure 3C 
     </xref>).</p> 
   </sec> 
   <sec> 
    <h3>Ras activating assays</h3> 
    <p>To assess which 
     <italic>
       K-Ras 
     </italic> mutations were in the active GTP-bound conformation, a 
     <italic>
       Raf-1 
     </italic> binding assay was carried out as described in Materials and Methods. 
     <italic>
       Raf-1 
     </italic> selectively binds GTP-bound Ras (rather than the inactive GDP-bound form) which, following immunoprecipitation, can be visualised by western blotting. 
     <xref ref-type="fig" rid="fig4">
       Figure 4 
     </xref> illustrates the results of our analysis where, consistent with our focus formation experiments, the 
     <italic>
       K-Ras 
     </italic> G12V and L19F, K117N and A146T mutations are clearly in the active GTP-bound conformation. In contrast, the R164Q mutation, like wt 
     <italic>
       K-Ras 
     </italic>, was not GTP-bound.</p> 
   </sec> 
   <sec> 
    <h3>RNA transcription-profiling experiments</h3> 
    <p>To further compare and contrast the phenotypes associated with each of the 
     <italic>
       K-Ras 
     </italic> mutants, transcription-profiling experiments were carried out as described in Materials and Methods. Hierarchical clustering analysis revealed the presence of two major gene clusters, ‘cluster 1' containing the G12V, G12C and G12D mutants and ‘cluster 2' containing G13D, A146T, K117N, R164Q and L19F ( 
     <xref ref-type="fig" rid="fig5">
       Figure 5 
     </xref>). It is interesting to note that the codon 12 mutants were most similar to wt 
     <italic>
       K-Ras 
     </italic>, whereas the K117N and A146T mutants clustered with the activating codon 13 and codon 61 hotspot mutants. The L19F and R164Q mutants formed a subcluster within cluster 2 suggesting, consistent with our foci formation data, that these mutants are phenotypically distinct from the G13D, A146T and K117N mutants. The presence of L19F in cluster 2 is consistent with our Ras GTPase assay data whereas, in contrast, our experimental data for R164Q predicted a ‘wt' Ras phenotype. The presence of R164Q in cluster 2, rather than in cluster 1 with wt 
     <italic>
       K-Ras 
     </italic> was therefore initially surprising, but suggests that the R164Q mutation also has an ‘activating' phenotype, albeit attenuated relative to the other mutations studied.</p> 
    <p>To further investigate the phenotypes associated with each of the 
     <italic>
       K-Ras 
     </italic> mutants, the expression of a diverse selection of genes associated a variety of cellular processes including signal transduction, cytoskeleton remodelling and cell adhesion was compared. Although further analysis of this complex dataset, including composite analysis of biological pathways and processes is ongoing and will be discussed in more detail in a future manuscript, representative examples of genes showing differential expression following the introduction of each 
     <italic>
       K-Ras 
     </italic> mutant are summarised in 
     <xref ref-type="table" rid="tbl3">
       Table 3 
     </xref>.</p> 
    <p>Consistent with our hierarchical clustering analysis, introduction of the R164Q mutation led to relatively few changes in gene expression or showed reduced pathway activation compared with the other mutants studied. However, there were examples of genes that were induced by all of the mutants studied, including the protein tyrosine phosphatase 
     <italic>
       Ptpre 
     </italic> and the Rho GTPase activating protein 
     <italic>
       Arhgap6 
     </italic>, and multiple examples of genes induced or repressed by all of the mutants with the exception of R164Q. These included the MAPK phosphatases 
     <italic>
       DUSP4 
     </italic> and 
     <italic>
       DUSP6 
     </italic>, 
     <italic>
       Ereg 
     </italic>, 
     <italic>
       Hbegf 
     </italic> and 
     <italic>
       Btc 
     </italic>, all involved in 
     <italic>
       EGFR 
     </italic> binding, the Rho guanine-exchange factor 
     <italic>
       NGEF 
     </italic>, cell adhesion molecule 
     <italic>
       Ceacam 1 
     </italic> and plasminogen activator inhibitor 
     <italic>
       Serpinb2 
     </italic>. Consistently, the novel A146T and K117N mutants (and, to a lesser extent, the L19F mutant) clustered with and influenced gene expression similarly to the previously described activating G13D and Q61H ‘hotspot' mutations.</p> 
    <p>Of particular interest were genes, for example, 
     <italic>
       Vegfa 
     </italic>, 
     <italic>
       Pak3 
     </italic>, 
     <italic>
       Pim1 
     </italic> and 
     <italic>
       EII2 
     </italic> which were differentially expressed by cluster 1 but not cluster 2 mutants, genes, for example, 
     <italic>
       Spry4 
     </italic>, 
     <italic>
       Igf1R 
     </italic>, 
     <italic>
       Creb1 
     </italic> and 
     <italic>
       Tcf4 
     </italic> which were differentially expressed by cluster 2 mutants, and additional genes, for example, 
     <italic>
       Jun 
     </italic>, 
     <italic>
       E2F2 
     </italic>, 
     <italic>
       Mmp3 
     </italic> and 
     <italic>
       Glut-1 
     </italic> which were differentially regulated by each mutant studied.</p> 
   </sec> 
  </sec> 
  <sec> 
   <h3> 
    <italic>
      K-Ras 
    </italic> gene amplification</h3> 
   <p>To investigate whether altered 
    <italic>
      K-Ras 
    </italic> activity could additionally result from gene amplification, a 
    <italic>
      K-Ras 
    </italic> gene copy number assay was designed and carried out as described in Materials and Methods and 
    <italic>
      K-Ras 
    </italic> copy number compared in normal and tumour tissues. In total, 2 of the 96 tumour pairs analysed (2.1%) showed tumour-specific copy number increases ( 
    <xref ref-type="fig" rid="fig6">
      Figure 6 
    </xref>), one (study code 1264, wt 
    <italic>
      K-Ras 
    </italic>) with 4 tumour copies of 
    <italic>
      K-Ras 
    </italic> and the other (study code 1233, wt 
    <italic>
      K-Ras 
    </italic>) with 27 copies of the 
    <italic>
      K-Ras 
    </italic> gene. Assay reproducibility and linearity was demonstrated from the analysis of standard curves generated by serial dilutions of a wide concentration range of input genomic DNA (data not shown).</p> 
   <p> 
    <italic>
      K-Ras 
    </italic> gene amplification was confirmed by FISH analysis as described in Materials and Methods ( 
    <xref ref-type="fig" rid="fig7">
      Figure 7 
    </xref>). 
    <xref ref-type="fig" rid="fig7">
      Figures 7A and B 
    </xref> illustrate chromosome metaphase spreads confirming FISH probe specificities for chromosomes 12p and 12q, respectively, whereas 
    <xref ref-type="fig" rid="fig7">
      Figures 7C and D 
    </xref> show representative colorectal tumour FISH analysis. 
    <xref ref-type="fig" rid="fig7">
      Figure 7C 
    </xref> illustrates a tumour sample (study code 1271) in which the 
    <italic>
      K-Ras 
    </italic> gene is not amplified (red fluorescence=green fluorescence), whereas 
    <xref ref-type="fig" rid="fig7">
      Figure 7D 
    </xref> illustrates a colorectal tumour (study code 1264) with 
    <italic>
      K-Ras 
    </italic> gene amplification (red fluorescence&gt;green fluorescence). Signals from the red 12p12.1 probe in tumour 1264 appear as a cluster of bright dots forming a line which is suggestive of a homogeneously staining region (HSR), a chromosomal region which is amplified either on the original chromosome or on another chromosome, as opposed to double minutes where the amplified gene is located in extra-chromosomal circular DNA molecules.</p> 
  </sec> 
 </sec> 
 <sec> 
  <h3>Discussion</h3> 
  <p>Recent clinical data has shown a compelling association between 
   <italic>
     K-Ras 
   </italic> and 
   <italic>
     B-Raf 
   </italic> mutation status and response to 
   <italic>
     EGFR 
   </italic> blockade by cetuximab and panitumumab in the treatment of metastatic colorectal tumours ( 
   <xref ref-type="other" rid="bib4">
     Benvenuti 
    <italic>
      et al 
    </italic>, 2007 
   </xref>; 
   <xref ref-type="other" rid="bib22">
     Khambata-Ford 
    <italic>
      et al 
    </italic>, 2007 
   </xref>; 
   <xref ref-type="other" rid="bib16">
     Freeman 
    <italic>
      et al 
    </italic>, 2008 
   </xref>; 
   <xref ref-type="other" rid="bib20">
     Karapetis 
    <italic>
      et al 
    </italic>, 2008 
   </xref>; 
   <xref ref-type="other" rid="bib31">
     Ramos 
    <italic>
      et al 
    </italic>, 2008 
   </xref>). 
   <italic>
     K-Ras 
   </italic> mutations in colorectal tumours have been extensively documented, where recent meta analysis of more than 3000 tumours estimates an average 
   <italic>
     K-Ras 
   </italic> mutation frequency of 34.8% ( 
   <xref ref-type="other" rid="bib2">
     Andreyev 
    <italic>
      et al 
    </italic>, 2001 
   </xref>). This and the majority of previous analyses were, however, limited to codons 12 and 13, whereas only a minority of studies have additionally analysed codon 61. Although additional 
   <italic>
     K-Ras 
   </italic> mutations have been reported outwith these ‘hotspot' codons, to our knowledge, there has been no systematic analysis of 
   <italic>
     K-Ras 
   </italic> mutation frequency in human colorectal tumours. Our use of ‘WAVE' denaturing HPLC (dHPLC) analysis ( 
   <xref ref-type="other" rid="bib23">
     Kuklin 
    <italic>
      et al 
    </italic>, 1997 
   </xref>) allowed us to carry out a rapid, systematic analysis of the entire 
   <italic>
     K-Ras 
   </italic> coding sequence to identify colorectal tumours with sequence changes, the nature which were subsequently confirmed by conventional dideoxy sequencing.</p> 
  <p>Our data clearly demonstrates that 
   <italic>
     K-Ras 
   </italic> mutations in human colorectal tumours are likely to be significantly underestimated if only hotspot codons are analysed. Indeed, current mutation screening tests based on the hotspot codons 12 and 13 are likely to result in the mis-classification of up to one-third of patients. Although mutations at codons 19, 117 and 164 are relatively rare events, our data suggests that each 
   <italic>
     K-Ras 
   </italic> mutation may differentially influence treatment response. We would therefore recommend the adoption of comprehensive 
   <italic>
     K-Ras 
   </italic> mutation profiling, or at least the routine inclusion of the codon 146 mutation, for patient selection for cetuximab and related therapies.</p> 
  <p>Current data from the COSMIC database, which summarises literature data and the ongoing Sanger cancer-genome project describes 29 
   <italic>
     K-Ras 
   </italic> colorectal tumour mutations outwith codons 12, 13 and 61, and more than 11?800 mutations at these hotspot codons ( 
   <ext-link ext-link-type="uri" xlink:href="http://www.sanger.ac.uk/genetics/CGP/">
     http://www.sanger.ac.uk/genetics/CGP/ 
   </ext-link>), but does not describe associated phenotypes. Consistent with our own data, codon 146 was the most frequently mutated of the mutations described. 
   <italic>
     K-Ras 
   </italic> codon 146 mutations have been described in a human colorectal tumour cell line ( 
   <xref ref-type="other" rid="bib19">
     Higashi 
    <italic>
      et al 
    </italic>, 1990 
   </xref>), in mice with thymic lymphomas ( 
   <xref ref-type="other" rid="bib32">
     Sloan 
    <italic>
      et al 
    </italic>, 1990 
   </xref>) and in a single human colorectal tumour case report ( 
   <xref ref-type="other" rid="bib29">
     Orita 
    <italic>
      et al 
    </italic>, 1991 
   </xref>). In more detailed molecular profiling studies, codon 146 mutations were described in 4% of two independent colorectal tumour series from Hong Kong and the United States of America ( 
   <xref ref-type="other" rid="bib13">
     Edkins 
    <italic>
      et al 
    </italic>, 2006 
   </xref>) and in chronic myelomonocytic leukaemia ( 
   <xref ref-type="other" rid="bib17">
     Gelsi-Boyer 
    <italic>
      et al 
    </italic>, 2008 
   </xref>; 
   <xref ref-type="other" rid="bib34">
     Tyner 
    <italic>
      et al 
    </italic>, 2009 
   </xref>). Consistent with our own data, 
   <xref ref-type="other" rid="bib34">
     Tyner 
    <italic>
      et al 
    </italic> (2009) 
   </xref> reported an oncogenic phenotype associated with the 
   <italic>
     K-Ras 
   </italic> 146 mutation in leukaemia and 
   <xref ref-type="other" rid="bib26">
     Loupakis 
    <italic>
      et al 
    </italic> (2009) 
   </xref> have recently reported that a patient with metastatic colorectal cancer with a 
   <italic>
     K-Ras 
   </italic> 146 mutation was resistant to cetuximab.</p> 
  <p>There is only a single report of 
   <italic>
     K-Ras 
   </italic> codon 19 mutations in human colorectal tumours ( 
   <xref ref-type="other" rid="bib1">
     Akagi 
    <italic>
      et al 
    </italic>, 2007 
   </xref>), in which 
   <italic>
     in vitro 
   </italic> elevation of active Ras-GTP levels, anchorage-independent growth and increased tumourigenicity in nude mice was demonstrated. A codon 19 mutation has additionally been described in 
   <italic>
     H-Ras 
   </italic> in a human pituitary carcinoma metastases ( 
   <xref ref-type="other" rid="bib30">
     Pei 
    <italic>
      et al 
    </italic>, 1994 
   </xref>). There are two recent reports of codon 117 mutations – the first in 
   <italic>
     K-Ras 
   </italic> in human colorectal tumours ( 
   <xref ref-type="other" rid="bib42">
     Wojcik 
    <italic>
      et al 
    </italic>, 2008 
   </xref>) and the second in 
   <italic>
     H-Ras 
   </italic>, associated with activation of the RAS–MAPK pathway and the mental retardation Costello syndrome ( 
   <xref ref-type="other" rid="bib11">
     Denayer 
    <italic>
      et al 
    </italic>, 2008 
   </xref>). It is interesting to note that Costello syndrome has been associated with increased cancer predisposition ( 
   <xref ref-type="other" rid="bib18">
     Gripp, 2005 
   </xref>). There are no reports of 
   <italic>
     K-Ras 
   </italic> 164 mutations, and mutations at this codon were not identified by Sanger cancer genome project investigators.</p> 
  <p>Recent clinical data convincingly associates response to anti- 
   <italic>
     EGFR 
   </italic> therapies with 
   <italic>
     K-Ras 
   </italic> mutation status in patients with metastatic colorectal cancer where, logically, response is preferentially observed in 
   <italic>
     K-Ras 
   </italic> wt tumours ( 
   <xref ref-type="other" rid="bib25">
     Lievre 
    <italic>
      et al 
    </italic>, 2006 
   </xref>; 
   <xref ref-type="other" rid="bib4">
     Benvenuti 
    <italic>
      et al 
    </italic>, 2007 
   </xref>; 
   <xref ref-type="other" rid="bib22">
     Khambata-Ford 
    <italic>
      et al 
    </italic>, 2007 
   </xref>; 
   <xref ref-type="other" rid="bib16">
     Freeman 
    <italic>
      et al 
    </italic>, 2008 
   </xref>; 
   <xref ref-type="other" rid="bib20">
     Karapetis 
    <italic>
      et al 
    </italic>, 2008 
   </xref>; 
   <xref ref-type="other" rid="bib31">
     Ramos 
    <italic>
      et al 
    </italic>, 2008 
   </xref>; 
   <xref ref-type="other" rid="bib26">
     Loupakis 
    <italic>
      et al 
    </italic>, 2009 
   </xref>; 
   <xref ref-type="other" rid="bib35">
     Van Cutsem 
    <italic>
      et al 
    </italic>, 2009 
   </xref>) which retain the ability to respond to 
   <italic>
     EGFR 
   </italic> blockade. Our description of 
   <italic>
     K-Ras 
   </italic> gene amplification in a subset of wt tumours is of particular interest in this regard as increased 
   <italic>
     K-Ras 
   </italic> gene copy number may lead to a more active ‘mutation'-like phenotype. Experiments to investigate this possibility are currently underway in our laboratory.</p> 
  <p>It is interesting to note that our transcription profiling analysis revealed some similarities but many marked differences in gene expression following the introduction of the individual 
   <italic>
     K-Ras 
   </italic> mutants. We found that the codon 117 and the common codon 146 mutations cluster with the previously described activating mutations at codons 13 and 61, highlighting differences in individual mutation phenotypes. Our analysis of 
   <italic>
     K-Ras 
   </italic> induced gene expression identified a number of genes previously associated with 
   <italic>
     K-Ras 
   </italic> signalling, for example the 
   <italic>
     MAPK 
   </italic> dual specificity phosphatases 
   <italic>
     DUSP4 
   </italic> and 
   <italic>
     DUSP6 
   </italic>. 
   <italic>
     DUSPs 
   </italic> are key regulators of the balance between kinase pathway activation and inactivation, and have previously been reported to be upregulated in an adaptive response, creating a negative feedback loop following 
   <italic>
     MAPK 
   </italic> pathway activation ( 
   <xref ref-type="other" rid="bib21">
     Keyse, 2008 
   </xref>). Consistent with our own data, 
   <italic>
     DUSP4 
   </italic> expression has previously been shown to be increased in pancreatic tumours with 
   <italic>
     K-Ras 
   </italic> mutations ( 
   <xref ref-type="other" rid="bib43">
     Yip-Schneider 
    <italic>
      et al 
    </italic>, 2001 
   </xref>) and DUSP6 expression increased in a variety of tumour types with mutations in Ras or Raf pathway genes ( 
   <xref ref-type="other" rid="bib8">
     Croonquist 
    <italic>
      et al 
    </italic>, 2003 
   </xref>; 
   <xref ref-type="other" rid="bib39">
     Warmka 
    <italic>
      et al 
    </italic>, 2004 
   </xref>; 
   <xref ref-type="other" rid="bib5">
     Bloethner 
    <italic>
      et al 
    </italic>, 2005 
   </xref>). Of particular interest, however, was our observation that certain signalling cascades, including those mediated by 
   <italic>
     Igf1 
   </italic> and 
   <italic>
     Vegf 
   </italic>, were differentially activated by cluster 1 and cluster 2 
   <italic>
     K-Ras 
   </italic> mutants, suggesting that not only the presence but the specific molecular characteristics of individual 
   <italic>
     K-Ras 
   </italic> mutations may be important determinants of both tumour progression and treatment response. For example, recent data suggests that glucose deprivation can promote the acquisition of 
   <italic>
     K-Ras 
   </italic> mutations in human tumours and reports upregulation of the glucose transporter 
   <italic>
     GLUT-1 
   </italic> in 
   <italic>
     K-Ras 
   </italic> mutant tumours ( 
   <xref ref-type="other" rid="bib44">
     Yun 
    <italic>
      et al 
    </italic>, 2009 
   </xref>). Our data confirms this hypothesis, but shows 
   <italic>
     GLUT-1 
   </italic> induction restricted to the codon 12, 13, 61 and 117 mutations, with no change in 
   <italic>
     GLUT-1 
   </italic> expression associated with the introduction of codon 19, 146 or 164 mutations, again reflecting phenotypic heterogeneity.</p> 
  <p>Current clinical data is sufficiently compelling that 
   <italic>
     K-Ras 
   </italic> mutation testing to facilitate the selection of patients most likely to benefit from anti- 
   <italic>
     EGFR 
   </italic> therapy is increasingly recommended in clinical practise ( 
   <xref ref-type="other" rid="bib27">
     McNeill, 2008 
   </xref>; 
   <xref ref-type="other" rid="bib36">
     van Krieken and Tol, 2009 
   </xref>), and has resulted in a recent formal recommendation from the National Comprehensive Cancer Network that patients with 
   <italic>
     K-Ras 
   </italic> tumour mutations should not be treated with cetuximab or panitumumab ( 
   <xref ref-type="other" rid="bib27">
     McNeill, 2008 
   </xref>). There is increasing interest in the identification of prognostic molecular markers, which may be used to select the most appropriate patients for adjuvant chemotherapy or to identify colorectal cancer patients at increased risk of disease progression. Although the clinical implications of non-hotspot 
   <italic>
     K-Ras 
   </italic> mutations requires further validation both as prognostic markers and therapeutic targets, our data suggests that future analysis of 
   <italic>
     K-Ras 
   </italic> mutations and quantitation of mutation burden in colorectal tumours should not be limited to previously described mutation hotspots and should additionally consider the unique molecular signatures associated with individual 
   <italic>
     K-Ras 
   </italic> mutations.</p> 
 </sec> 
 <back> 
  <ack> 
   <p>We thank Professor Richard Marais; Professor of Molecular Oncology, Institute of Cancer Research for technical advice and the generous gift of protocols and K-Ras expression plasmids; Dr Simon Plummer, CXR Biosciences, Dundee, UK for carrying out the transcription profiling analysis; Probir Chakravarty, Cancer Research UK for helping with bioinformatics analysis; Dr Norman Pratt, Dundee, UK for helping with FISH analysis; and Tayside Tissue Bank for helping with patient recruitment and sample processing. We acknowledge financial support from Cancer Research UK (C4639/A5661), Food Standards Agency (Contract TO1004) and the Ninewells Cancer Campaign, and funding from the MRC and Cancer Research UK for Tayside Tissue Bank.</p> 
  </ack> 
  <app-group> 
   <app id="app1"> 
    <h3>Appendix 1</h3> 
    <p>Kirsten-Ras ( 
     <italic>
       K-Ras 
     </italic>) mutation detection and restriction fragment length polymorphism (RFLP) analysis of mutation frequencies</p> 
    <sec> 
     <h3> 
      <italic>
        K-Ras 
      </italic> Exon 3 and Exon 4 mutation detection</h3> 
     <p>Mutations in exons 3 and 4 of the 
      <italic>
        K-Ras 
      </italic> gene were identified using the Transgenomic WAVE denaturing HPLC system. PCR products for WAVE analysis were generated using the Expand High Fidelity PCR system (Roche, Welingkar garden, UK) incorporating Taq polymerase with enhanced proofreading activity. A 200?bp PCR product encompassing 
      <italic>
        K-Ras 
      </italic> exon 3 was generated using the oligonucleotide primers 5?-TTCTTTCCCAGAGAACAAATTAA-3? and 5?-AGATCTGTATTTATTTCAGTGTTACTTACC-3?. Similarly, a 261?bp PCR product encompassing 
      <italic>
        K-Ras 
      </italic> Exon 4b was generated using the oligonucleotide primers 5?-TACTTTTTATGTATTTCAGGGTGTTG-3? and 5?-GCTAACAGTCTGCATGGAGC-3?.</p> 
     <p>Assay conditions (acetonitrile gradient and temperature profile) were optimised for WAVE analysis of each PCR fragment using Transgenomic WAVEMAKER 4.1 software, according to the manufacturer's instructions. 
      <italic>
        K-Ras 
      </italic> exon 3 was analysed at 55°C with an initial concentration of 51% Buffer B (0.1? 
      <sc>
        M 
      </sc> TEAA, 25% acetonitrile) and exon 4 at 55°C with an initial concentration of 54% Buffer B. All samples were denatured and cooled slowly to room temperature before WAVE analysis to maximise heteroduplex formation.</p> 
    </sec> 
    <sec> 
     <h3>Direct sequencing</h3> 
     <p>The 
      <italic>
        K-Ras 
      </italic> mutations detected by WAVE analysis were confirmed by direct sequencing of PCR products. This analysis was performed by the DNA analysis facility, Department of Molecular and Cellular Pathology, Ninewells Hospital and Medical School, Dundee DD1 9SY, UK, using 16 capillary ABI 3100 Genetic Analysers (Applied Biosystems). Before sequencing, all PCR products were purified using QIAquick PCR purification kits (Qiagen), according to the manufacturer's instructions. All sequences obtained were aligned with previously published sequences (NCBI Genbank Accession Numbers L00047 and L00049 for 
      <italic>
        K-Ras 
      </italic> exons 3 and 4b, respectively); the presence and nature of each mutation was confirmed by repeat PCR and sequencing.</p> 
    </sec> 
    <sec> 
     <h3>RFLP analysis of 
      <italic>
        K-Ras 
      </italic> codon 117 mutations</h3> 
     <p>To permit rapid identification of the 
      <italic>
        K-Ras 
      </italic> codon 117 mutation, a PCR–RFLP assay was designed where the presence of the mutation created a recognition site for the restriction enzyme 
      <italic>
        Bsr S1 
      </italic>. A 288?bp PCR product was amplified using the primers 5?-GATCTTTTGAGAGAGATACAAGGTTTC-3? and 5?-TGTTCTAGAAGGCAAATCACA-3?. Following PCR amplification, PCR products were digested with 
      <italic>
        Bsr S1 
      </italic> for 2?h at 65°C and analysed on 8% polyacrylamide gels, run in 1 × TBE buffer at 200?V for 2?h. Gels were stained with ethidium bromide and DNA bands visualised on a UV transilluminator. Samples without the 
      <italic>
        K-Ras 
      </italic> codon 117 mutation were resistant to digestion and produced a single band of 288?bp, whereas samples homozygous for the codon 117 mutation were digested to produce bands of 271 and 17?bp. Heterozygous samples produced bands of 288, 271 and 17?bp. Band sizes were determined by comparison with ?X174 DNA size markers (Promega).</p> 
    </sec> 
    <sec> 
     <h3>RFLP analysis of 
      <italic>
        K-Ras 
      </italic> codon 146 mutations</h3> 
     <p>Similarly, a PCR-RFLP assay was designed to identify the 
      <italic>
        K-Ras 
      </italic> codon 146 mutation, where the presence of the mutation created a recognition site for the restriction enzyme 
      <italic>
        Mse I 
      </italic>. A 197?bp PCR product was amplified using the primers 5? TGG AAT TCC TTT TAT TGA AAC ATC A 3? and 5? GAT TAA GAA GCA ATG CCC TCT C 3?. Following PCR amplification, PCR products were digested with 
      <italic>
        Mse 
      </italic>1 for 2?h at 37°C and analysed on 8% polyacrylamide gels as described above. Samples without the 
      <italic>
        K-Ras 
      </italic> codon 146 mutation were digested to produce bands of 101, 91 and 5?bp, whereas samples homozygous for the codon 146 mutation produced bands of 91, 78, 23 and 5?bp. Heterozygous samples produced bands of 101, 91, 78, 23 and 5?bp.</p> 
    </sec> 
    <sec> 
     <h3>RFLP analysis of 
      <italic>
        K-Ras 
      </italic> codon 164 mutations</h3> 
     <p>The presence of the codon 164 mutation was confirmed using a novel RFLP assay, wherein the presence of the mutation destroyed a recognition site for the restriction enzyme 
      <italic>
        BstB 
      </italic>I. A 261?bp PCR product was generated as described above for 
      <italic>
        K-Ras 
      </italic> exon 4B mutation detection. After PCR amplification, PCR products were digested with 
      <italic>
        BstB 
      </italic>I for 2?h at 65°C and analysed on 2% agarose gels. Samples without the codon 164 mutation were digested to produce bands of 203 and 58?bp, whereas samples homozygous for the mutation produced a single band of 261?bp. Heterozygous samples produced bands of 261, 203 and 58?bp.</p> 
    </sec> 
    <sec> 
     <h3>RFLP analysis of 
      <italic>
        K-Ras 
      </italic> codon 173 single-nucleotide polymorphism (SNP)</h3> 
     <p>The codon 173 SNP was detected using a novel RFLP assay, where the presence of the mutation destroyed a recognition site for the restriction enzyme 
      <italic>
        Bcc 
      </italic>I. A 261?bp PCR product was generated as described above for 
      <italic>
        K-Ras 
      </italic> exon 4B mutation detection. After PCR amplification, PCR products were digested with 
      <italic>
        Bcc 
      </italic>I for 2?h at 37°C and analysed on 2% agarose gels. Samples without the codon 173 SNP were digested to produce bands of 173 and 88?bp, whereas samples homozygous for the codon 173 SNP produced a single band of 261?bp. Heterozygous samples produced bands of 261, 173 and 88?bp.</p> 
    </sec> 
    <sec> 
     <h3>RFLP analysis of 
      <italic>
        B-Raf 
      </italic> V599E mutation</h3> 
     <p>The 
      <italic>
        B-Raf 
      </italic> V599E mutation was detected using a novel RFLP assay, where the presence of the mutation destroyed a recognition site for the restriction enzyme 
      <italic>
        Tsp 
      </italic>RI. A 224?bp PCR product was amplified using the primers 5?-TCATAATGCTTGCTCTGATAGGA-3? and 5?-GGCCAAAAATTTAATCAGTGGA-3? in a 50?ml reaction containing 1 × PCR buffer (Promega), 1.25?m 
      <sc>
        M 
      </sc> MgCl<sub>2,</sub> 200?m 
      <sc>
        M 
      </sc> dNTPs, 300?n 
      <sc>
        M 
      </sc> primers, 6% DMSO and 1.5?U of Taq polymerase in a PCR as described above. After PCR amplification, PCR products were digested with 
      <italic>
        Tsp 
      </italic>R1 for 2?h at 37°C and analysed on 8% polyacrylamide gels as described above. Samples without the 
      <italic>
        B-Raf 
      </italic> V599E mutation were digested to produce bands of 124, 78 and 22?bp, whereas samples homozygous for the V599E mutation produced bands of 202 and 22?bp. Heterozygous samples produced bands of 202, 124, 78 and 22?bp.</p> 
    </sec> 
   </app> 
  </app-group> 
  <ref-list> 
   <ref id="bib1"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Akagi 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Uchibori 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yamaguchi 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kurosawa 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tanaka 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kozu 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Characterization of a novel oncogenic K-ras mutation in colon cancer 
     </article-h3> 
     <source />Biochem Biophys Res Commun 
     <year>
       2007 
     </year> 
     <volume>
       352 
     </volume> 
     <fpage>
       728 
     </fpage> 
     <lpage>
       732 
     </lpage> 
     <!--PubMed citation query: 'Biochem Biophys Res Commun||352|728||bib1|'--> 
     <pub-id pub-id-type="pmid">
       17150185 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib2"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Andreyev 
       </surname> 
       <given-names>
         HJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Norman 
       </surname> 
       <given-names>
         AR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cunningham 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Oates 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dix 
       </surname> 
       <given-names>
         BR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Iacopetta 
       </surname> 
       <given-names>
         BJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Young 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Walsh 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ward 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hawkins 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Beranek 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jandik 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Benamouzig 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jullian 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Laurent-Puig 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Olschwang 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Muller 
       </surname> 
       <given-names>
         O 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hoffmann 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rabes 
       </surname> 
       <given-names>
         HM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zietz 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Troungos 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Valavanis 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yuen 
       </surname> 
       <given-names>
         ST 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ho 
       </surname> 
       <given-names>
         JW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Croke 
       </surname> 
       <given-names>
         CT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         O'Donoghue 
       </surname> 
       <given-names>
         DP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Giaretti 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rapallo 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Russo 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bazan 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tanaka 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Omura 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Azuma 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ohkusa 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fujimori 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ono 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pauly 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Faber 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Glaesener 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         de Goeij 
       </surname> 
       <given-names>
         AF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Arends 
       </surname> 
       <given-names>
         JW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Andersen 
       </surname> 
       <given-names>
         SN 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lovig 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Breivik 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gaudernack 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Clausen 
       </surname> 
       <given-names>
         OP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         De Angelis 
       </surname> 
       <given-names>
         PD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Meling 
       </surname> 
       <given-names>
         GI 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rognum 
       </surname> 
       <given-names>
         TO 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Smith 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Goh 
       </surname> 
       <given-names>
         HS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Font 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rosell 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sun 
       </surname> 
       <given-names>
         XF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zhang 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Benhattar 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Losi 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         JQ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wang 
       </surname> 
       <given-names>
         ST 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Clarke 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bell 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Quirke 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bubb 
       </surname> 
       <given-names>
         VJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Piris 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cruickshank 
       </surname> 
       <given-names>
         NR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Morton 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fox 
       </surname> 
       <given-names>
         JC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Al-Mulla 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lees 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hall 
       </surname> 
       <given-names>
         CN 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Snary 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wilkinson 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dillon 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Costa 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pricolo 
       </surname> 
       <given-names>
         VE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Finkelstein 
       </surname> 
       <given-names>
         SD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Thebo 
       </surname> 
       <given-names>
         JS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Senagore 
       </surname> 
       <given-names>
         AJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Halter 
       </surname> 
       <given-names>
         SA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wadler 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Malik 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Krtolica 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Urosevic 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Kirsten ras mutations in patients with colorectal cancer: the ‘RASCAL II' study 
     </article-h3> 
     <source />Br J Cancer 
     <year>
       2001 
     </year> 
     <volume>
       85 
     </volume> 
     <fpage>
       692 
     </fpage> 
     <lpage>
       696 
     </lpage> 
     <!--PubMed citation query: 'Br J Cancer||85|692||bib2|'--> 
     <pub-id pub-id-type="pmid">
       11531254 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib3"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Barbacid 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Ras oncogenes: their role in neoplasia 
     </article-h3> 
     <source />Eur J Clin Invest 
     <year>
       1990 
     </year> 
     <volume>
       20 
     </volume> 
     <fpage>
       225 
     </fpage> 
     <lpage>
       235 
     </lpage> 
     <!--PubMed citation query: 'Eur J Clin Invest||20|225||bib3|'--> 
     <pub-id pub-id-type="pmid">
       2114981 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib4"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Benvenuti 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sartore-Bianchi 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Di Nicolantonio 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zanon 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Moroni 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Veronese 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Siena 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bardelli 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Oncogenic activation of the RAS/RAF signaling pathway impairs the response of metastatic colorectal cancers to anti-epidermal growth factor receptor antibody therapies 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2007 
     </year> 
     <volume>
       67 
     </volume> 
     <fpage>
       2643 
     </fpage> 
     <lpage>
       2648 
     </lpage> 
     <!--PubMed citation query: 'Cancer Res||67|2643||bib4|'--> 
     <pub-id pub-id-type="pmid">
       17363584 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib5"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Bloethner 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chen 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hemminki 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Muller-Berghaus 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ugurel 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Schadendorf 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kumar 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Effect of common B-RAF and N-RAS mutations on global gene expression in melanoma cell lines 
     </article-h3> 
     <source />Carcinogenesis 
     <year>
       2005 
     </year> 
     <volume>
       26 
     </volume> 
     <fpage>
       1224 
     </fpage> 
     <lpage>
       1232 
     </lpage> 
     <!--PubMed citation query: 'Carcinogenesis||26|1224||bib5|'--> 
     <pub-id pub-id-type="pmid">
       15760917 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib6"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Bos 
       </surname> 
       <given-names>
         JL 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       ras oncogenes in human cancer: a review 
     </article-h3> 
     <source />Cancer Res 
     <year>
       1989 
     </year> 
     <volume>
       49 
     </volume> 
     <fpage>
       4682 
     </fpage> 
     <lpage>
       4689 
     </lpage> 
     <!--PubMed citation query: 'Cancer Res||49|4682||bib6|'--> 
     <pub-id pub-id-type="pmid">
       2547513 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib7"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Conlin 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Smith 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Carey 
       </surname> 
       <given-names>
         FA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wolf 
       </surname> 
       <given-names>
         CR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Steele 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The prognostic significance of K-ras, p53, and APC mutations in colorectal carcinoma 
     </article-h3> 
     <source />Gut 
     <year>
       2005 
     </year> 
     <volume>
       54 
     </volume> 
     <fpage>
       1283 
     </fpage> 
     <lpage>
       1286 
     </lpage> 
     <!--PubMed citation query: 'Gut||54|1283||bib7|'--> 
     <pub-id pub-id-type="pmid">
       15843421 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib8"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Croonquist 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Linden 
       </surname> 
       <given-names>
         MA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zhao 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Van Ness 
       </surname> 
       <given-names>
         BG 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Gene profiling of a myeloma cell line reveals similarities and unique signatures among IL-6 response, N-ras-activating mutations, and coculture with bone marrow stromal cells 
     </article-h3> 
     <source />Blood 
     <year>
       2003 
     </year> 
     <volume>
       102 
     </volume> 
     <fpage>
       2581 
     </fpage> 
     <lpage>
       2592 
     </lpage> 
     <!--PubMed citation query: 'Blood||102|2581||bib8|'--> 
     <pub-id pub-id-type="pmid">
       12791645 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib9"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Davies 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bignell 
       </surname> 
       <given-names>
         GR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cox 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stephens 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Edkins 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Clegg 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Teague 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Woffendin 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Garnett 
       </surname> 
       <given-names>
         MJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bottomley 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Davis 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dicks 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ewing 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Floyd 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gray 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hall 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hawes 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hughes 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kosmidou 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Menzies 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mould 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Parker 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stevens 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Watt 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hooper 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wilson 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jayatilake 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gusterson 
       </surname> 
       <given-names>
         BA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cooper 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shipley 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hargrave 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pritchard-Jones 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Maitland 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chenevix-Trench 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Riggins 
       </surname> 
       <given-names>
         GJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bigner 
       </surname> 
       <given-names>
         DD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Palmieri 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cossu 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Flanagan 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nicholson 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ho 
       </surname> 
       <given-names>
         JW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Leung 
       </surname> 
       <given-names>
         SY 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yuen 
       </surname> 
       <given-names>
         ST 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Weber 
       </surname> 
       <given-names>
         BL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Seigler 
       </surname> 
       <given-names>
         HF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Darrow 
       </surname> 
       <given-names>
         TL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Paterson 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Marais 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Marshall 
       </surname> 
       <given-names>
         CJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wooster 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stratton 
       </surname> 
       <given-names>
         MR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Futreal 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Mutations of the BRAF gene in human cancer 
     </article-h3> 
     <source />Nature 
     <year>
       2002 
     </year> 
     <volume>
       417 
     </volume> 
     <fpage>
       949 
     </fpage> 
     <lpage>
       954 
     </lpage> 
     <!--PubMed citation query: 'Nature||417|949||bib9|'--> 
     <pub-id pub-id-type="pmid">
       12068308 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib10"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         de la Chapelle 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Genetic predisposition to colorectal cancer 
     </article-h3> 
     <source />Nat Rev Cancer 
     <year>
       2004 
     </year> 
     <volume>
       4 
     </volume> 
     <fpage>
       769 
     </fpage> 
     <lpage>
       780 
     </lpage> 
     <!--PubMed citation query: 'Nat Rev Cancer||4|769||bib10|'--> 
     <pub-id pub-id-type="pmid">
       15510158 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib11"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Denayer 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Parret 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chmara 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Schubbert 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vogels 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Devriendt 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Frijns 
       </surname> 
       <given-names>
         JP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rybin 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         de Ravel 
       </surname> 
       <given-names>
         TJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shannon 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cools 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Scheffzek 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Legius 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Mutation analysis in Costello syndrome: functional and structural characterization of the HRAS p.Lys117Arg mutation 
     </article-h3> 
     <source />Hum Mutat 
     <year>
       2008 
     </year> 
     <volume>
       29 
     </volume> 
     <fpage>
       232 
     </fpage> 
     <lpage>
       239 
     </lpage> 
     <!--PubMed citation query: 'Hum Mutat||29|232||bib11|'--> 
     <pub-id pub-id-type="pmid">
       17979197 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib12"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Di Nicolantonio 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Martini 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Molinari 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sartore-Bianchi 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Arena 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Saletti 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         De Dosso 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mazzucchelli 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Frattini 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Siena 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bardelli 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Wild-type BRAF is required for response to panitumumab or cetuximab in metastatic colorectal cancer 
     </article-h3> 
     <source />J Clin Oncol 
     <year>
       2008 
     </year> 
     <volume>
       26 
     </volume> 
     <fpage>
       5705 
     </fpage> 
     <lpage>
       5712 
     </lpage> 
     <!--PubMed citation query: 'J Clin Oncol||26|5705||bib12|'--> 
     <pub-id pub-id-type="pmid">
       19001320 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib13"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Edkins 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         O'Meara 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Parker 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stevens 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Reis 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jones 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Greenman 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Davies 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dalgliesh 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Forbes 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hunter 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Smith 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stephens 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Goldstraw 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nicholson 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chan 
       </surname> 
       <given-names>
         TL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Velculescu 
       </surname> 
       <given-names>
         VE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yuen 
       </surname> 
       <given-names>
         ST 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Leung 
       </surname> 
       <given-names>
         SY 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stratton 
       </surname> 
       <given-names>
         MR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Futreal 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Recurrent KRAS codon 146 mutations in human colorectal cancer 
     </article-h3> 
     <source />Cancer Biol Ther 
     <year>
       2006 
     </year> 
     <volume>
       5 
     </volume> 
     <fpage>
       928 
     </fpage> 
     <lpage>
       932 
     </lpage> 
     <!--PubMed citation query: 'Cancer Biol Ther||5|928||bib13|'--> 
     <pub-id pub-id-type="pmid">
       16969076 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib14"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ellis 
       </surname> 
       <given-names>
         CA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Clark 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The importance of being K-Ras 
     </article-h3> 
     <source />Cell Signal 
     <year>
       2000 
     </year> 
     <volume>
       12 
     </volume> 
     <fpage>
       425 
     </fpage> 
     <lpage>
       434 
     </lpage> 
     <!--PubMed citation query: 'Cell Signal||12|425||bib14|'--> 
     <pub-id pub-id-type="pmid">
       10989276 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib15"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Etienne-Grimaldi 
       </surname> 
       <given-names>
         MC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Formento 
       </surname> 
       <given-names>
         JL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Francoual 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Francois 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Formento 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Renee 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Laurent-Puig 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chazal 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Benchimol 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Delpero 
       </surname> 
       <given-names>
         JR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Letoublon 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pezet 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Seitz 
       </surname> 
       <given-names>
         JF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Milano 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       K-Ras mutations and treatment outcome in colorectal cancer patients receiving exclusive fluoropyrimidine therapy 
     </article-h3> 
     <source />Clin Cancer Res 
     <year>
       2008 
     </year> 
     <volume>
       14 
     </volume> 
     <fpage>
       4830 
     </fpage> 
     <lpage>
       4835 
     </lpage> 
     <!--PubMed citation query: 'Clin Cancer Res||14|4830||bib15|'--> 
     <pub-id pub-id-type="pmid">
       18676755 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib16"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Freeman 
       </surname> 
       <given-names>
         DJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Juan 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Reiner 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hecht 
       </surname> 
       <given-names>
         JR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Meropol 
       </surname> 
       <given-names>
         NJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Berlin 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mitchell 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sarosi 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Radinsky 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Amado 
       </surname> 
       <given-names>
         RG 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Association of K-ras mutational status and clinical outcomes in patients with metastatic colorectal cancer receiving panitumumab alone 
     </article-h3> 
     <source />Clin Colorectal Cancer 
     <year>
       2008 
     </year> 
     <volume>
       7 
     </volume> 
     <fpage>
       184 
     </fpage> 
     <lpage>
       190 
     </lpage> 
     <!--PubMed citation query: 'Clin Colorectal Cancer||7|184||bib16|'--> 
     <pub-id pub-id-type="pmid">
       18621636 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib17"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Gelsi-Boyer 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Trouplin 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Adelaide 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Aceto 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Remy 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pinson 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Houdayer 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Arnoulet 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sainty 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bentires-Alj 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Olschwang 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vey 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mozziconacci 
       </surname> 
       <given-names>
         MJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Birnbaum 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chaffanet 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Genome profiling of chronic myelomonocytic leukemia: frequent alterations of RAS and RUNX1 genes 
     </article-h3> 
     <source />BMC Cancer 
     <year>
       2008 
     </year> 
     <volume>
       8 
     </volume> 
     <fpage>
       299 
     </fpage> 
     <!--PubMed citation query: 'BMC Cancer||8|299||bib17|'--> 
     <pub-id pub-id-type="pmid">
       18925961 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib18"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Gripp 
       </surname> 
       <given-names>
         KW 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Tumor predisposition in Costello syndrome 
     </article-h3> 
     <source />Am J Med Genet C Semin Med Genet 
     <year>
       2005 
     </year> 
     <volume>
       137C 
     </volume> 
     <fpage>
       72 
     </fpage> 
     <lpage>
       77 
     </lpage> 
     <!--PubMed citation query: 'Am J Med Genet C Semin Med Genet||137C|72||bib18|'--> 
     <pub-id pub-id-type="pmid">
       16010679 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib19"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Higashi 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sasai 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Suzuki 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Miyoshi 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ohuchi 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Takai 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mori 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kakunaga 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Hamster cell line suitable for transfection assay of transforming genes 
     </article-h3> 
     <source />Proc Natl Acad Sci USA 
     <year>
       1990 
     </year> 
     <volume>
       87 
     </volume> 
     <fpage>
       2409 
     </fpage> 
     <lpage>
       2413 
     </lpage> 
     <!--PubMed citation query: 'Proc Natl Acad Sci USA||87|2409||bib19|'--> 
    </citation> 
   </ref> 
   <ref id="bib20"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Karapetis 
       </surname> 
       <given-names>
         CS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Khambata-Ford 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jonker 
       </surname> 
       <given-names>
         DJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         O'Callaghan 
       </surname> 
       <given-names>
         CJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tu 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tebbutt 
       </surname> 
       <given-names>
         NC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Simes 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chalchal 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shapiro 
       </surname> 
       <given-names>
         JD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Robitaille 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Price 
       </surname> 
       <given-names>
         TJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shepherd 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Au 
       </surname> 
       <given-names>
         HJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Langer 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Moore 
       </surname> 
       <given-names>
         MJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zalcberg 
       </surname> 
       <given-names>
         JR 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       K-ras mutations and benefit from cetuximab in advanced colorectal cancer 
     </article-h3> 
     <source />N Engl J Med 
     <year>
       2008 
     </year> 
     <volume>
       359 
     </volume> 
     <fpage>
       1757 
     </fpage> 
     <lpage>
       1765 
     </lpage> 
     <!--PubMed citation query: 'N Engl J Med||359|1757||bib20|'--> 
     <pub-id pub-id-type="pmid">
       18946061 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib21"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Keyse 
       </surname> 
       <given-names>
         SM 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Dual-specificity MAP kinase phosphatases (MKPs) and cancer 
     </article-h3> 
     <source />Cancer Metastasis Rev 
     <year>
       2008 
     </year> 
     <volume>
       27 
     </volume> 
     <fpage>
       253 
     </fpage> 
     <lpage>
       261 
     </lpage> 
     <!--PubMed citation query: 'Cancer Metastasis Rev||27|253||bib21|'--> 
     <pub-id pub-id-type="pmid">
       18330678 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib22"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Khambata-Ford 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Garrett 
       </surname> 
       <given-names>
         CR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Meropol 
       </surname> 
       <given-names>
         NJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Basik 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Harbison 
       </surname> 
       <given-names>
         CT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wu 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wong 
       </surname> 
       <given-names>
         TW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Huang 
       </surname> 
       <given-names>
         X 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Takimoto 
       </surname> 
       <given-names>
         CH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Godwin 
       </surname> 
       <given-names>
         AK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tan 
       </surname> 
       <given-names>
         BR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Krishnamurthi 
       </surname> 
       <given-names>
         SS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Burris 
       </surname> 
       <given-names>
         HA 
       </given-names> 
       <suffix>
         III 
       </suffix> 
      </name> 
      <name> 
       <surname>
         Poplin 
       </surname> 
       <given-names>
         EA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hidalgo 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Baselga 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Clark 
       </surname> 
       <given-names>
         EA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mauro 
       </surname> 
       <given-names>
         DJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Expression of epiregulin and amphiregulin and K-ras mutation status predict disease control in metastatic colorectal cancer patients treated with cetuximab 
     </article-h3> 
     <source />J Clin Oncol 
     <year>
       2007 
     </year> 
     <volume>
       25 
     </volume> 
     <fpage>
       3230 
     </fpage> 
     <lpage>
       3237 
     </lpage> 
     <!--PubMed citation query: 'J Clin Oncol||25|3230||bib22|'--> 
     <pub-id pub-id-type="pmid">
       17664471 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib23"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Kuklin 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Munson 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gjerde 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Haefele 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Taylor 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Detection of single-nucleotide polymorphisms with the WAVE DNA fragment analysis system 
     </article-h3> 
     <source />Genet Test 
     <year>
       1997 
     </year> 
     <volume>
       1 
     </volume> 
     <fpage>
       201 
     </fpage> 
     <lpage>
       206 
     </lpage> 
     <!--PubMed citation query: 'Genet Test||1|201||bib23|'--> 
     <pub-id pub-id-type="pmid">
       10464646 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib24"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Leslie 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pratt 
       </surname> 
       <given-names>
         NR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gillespie 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sales 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kernohan 
       </surname> 
       <given-names>
         NM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Smith 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wolf 
       </surname> 
       <given-names>
         CR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Carey 
       </surname> 
       <given-names>
         FA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Steele 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Mutations of APC, K-ras, and p53 are associated with specific chromosomal aberrations in colorectal adenocarcinomas 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2003 
     </year> 
     <volume>
       63 
     </volume> 
     <fpage>
       4656 
     </fpage> 
     <lpage>
       4661 
     </lpage> 
     <!--PubMed citation query: 'Cancer Res||63|4656||bib24|'--> 
     <pub-id pub-id-type="pmid">
       12907646 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib25"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Lievre 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bachet 
       </surname> 
       <given-names>
         JB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Le Corre 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Boige 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Landi 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Emile 
       </surname> 
       <given-names>
         JF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cote 
       </surname> 
       <given-names>
         JF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tomasic 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Penna 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ducreux 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rougier 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Penault-Llorca 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Laurent-Puig 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       KRAS mutation status is predictive of response to cetuximab therapy in colorectal cancer 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2006 
     </year> 
     <volume>
       66 
     </volume> 
     <fpage>
       3992 
     </fpage> 
     <lpage>
       3995 
     </lpage> 
     <!--PubMed citation query: 'Cancer Res||66|3992||bib25|'--> 
     <pub-id pub-id-type="pmid">
       16618717 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib26"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Loupakis 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ruzzo 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cremolini 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vincenzi 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Salvatore 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Santini 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Masi 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stasi 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Canestrari 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rulli 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Floriani 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bencardino 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Galluccio 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Catalano 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tonini 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Magnani 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fontanini 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Basolo 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Falcone 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Graziano 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       KRAS codon 61, 146 and BRAF mutations predict resistance to cetuximab plus irinotecan in KRAS codon 12 and 13 wild-type metastatic colorectal cancer 
     </article-h3> 
     <source />Br J Cancer 
     <year>
       2009 
     </year> 
     <volume>
       101 
     </volume> 
     <fpage>
       715 
     </fpage> 
     <lpage>
       721 
     </lpage> 
     <!--PubMed citation query: 'Br J Cancer||101|715||bib26|'--> 
     <pub-id pub-id-type="pmid">
       19603018 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib27"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         McNeill 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       K-Ras mutations are changingpractice in advanced colorectal cancer 
     </article-h3> 
     <source />J Natl Cancer Inst 
     <year>
       2008 
     </year> 
     <volume>
       100 
     </volume> 
     <fpage>
       1667 
     </fpage> 
     <lpage>
       1669 
     </lpage> 
     <!--PubMed citation query: 'J Natl Cancer Inst||100|1667||bib27|'--> 
     <pub-id pub-id-type="pmid">
       19033564 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib28"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Miyakura 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sugano 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fukayama 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Konishi 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nagai 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Concurrent mutations of K-ras oncogene at codons 12 and 22 in colon cancer 
     </article-h3> 
     <source />Jpn J Clin Oncol 
     <year>
       2002 
     </year> 
     <volume>
       32 
     </volume> 
     <fpage>
       219 
     </fpage> 
     <lpage>
       221 
     </lpage> 
     <!--PubMed citation query: 'Jpn J Clin Oncol||32|219||bib28|'--> 
     <pub-id pub-id-type="pmid">
       12110640 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib29"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Orita 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Higashi 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kawasaki 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Harada 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Igarashi 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Monden 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Morimoto 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shimano 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mori 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Miyoshi 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       A novel point mutation at codon 146 of the K-ras gene in a human colorectal cancer identified by the polymerase chain reaction 
     </article-h3> 
     <source />Virus Genes 
     <year>
       1991 
     </year> 
     <volume>
       5 
     </volume> 
     <fpage>
       75 
     </fpage> 
     <lpage>
       79 
     </lpage> 
     <!--PubMed citation query: 'Virus Genes||5|75||bib29|'--> 
     <pub-id pub-id-type="pmid">
       2017878 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib30"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Pei 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Melmed 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Scheithauer 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kovacs 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Prager 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       H-ras mutations in human pituitary carcinoma metastases 
     </article-h3> 
     <source />J Clin Endocrinol Metab 
     <year>
       1994 
     </year> 
     <volume>
       78 
     </volume> 
     <fpage>
       842 
     </fpage> 
     <lpage>
       846 
     </lpage> 
     <!--PubMed citation query: 'J Clin Endocrinol Metab||78|842||bib30|'--> 
     <pub-id pub-id-type="pmid">
       8157709 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib31"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ramos 
       </surname> 
       <given-names>
         FJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Macarulla 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Capdevila 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Elez 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tabernero 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Understanding the predictive role of K-ras for epidermal growth factor receptor-targeted therapies in colorectal cancer 
     </article-h3> 
     <source />Clin Colorectal Cancer 
     <year>
       2008 
     </year> 
     <volume>
       7 
     </volume> 
     <supplement>
       Suppl 2 
     </supplement> 
     <fpage>
       S52 
     </fpage> 
     <lpage>
       S57 
     </lpage> 
     <!--PubMed citation query: 'Clin Colorectal Cancer||7|S52||bib31|'--> 
     <pub-id pub-id-type="pmid">
       19064407 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib32"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Sloan 
       </surname> 
       <given-names>
         SR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Newcomb 
       </surname> 
       <given-names>
         EW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pellicer 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Neutron radiation can activate K-ras via a point mutation in codon 146 and induces a different spectrum of ras mutations than does gamma radiation 
     </article-h3> 
     <source />Mol Cell Biol 
     <year>
       1990 
     </year> 
     <volume>
       10 
     </volume> 
     <fpage>
       405 
     </fpage> 
     <lpage>
       408 
     </lpage> 
     <!--PubMed citation query: 'Mol Cell Biol||10|405||bib32|'--> 
     <pub-id pub-id-type="pmid">
       2403644 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib33"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Smith 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Carey 
       </surname> 
       <given-names>
         FA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Beattie 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wilkie 
       </surname> 
       <given-names>
         MJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lightfoot 
       </surname> 
       <given-names>
         TJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Coxhead 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Garner 
       </surname> 
       <given-names>
         RC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Steele 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wolf 
       </surname> 
       <given-names>
         CR 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Mutations in APC, Kirsten-ras, and p53–alternative genetic pathways to colorectal cancer 
     </article-h3> 
     <source />Proc Natl Acad Sci USA 
     <year>
       2002 
     </year> 
     <volume>
       99 
     </volume> 
     <fpage>
       9433 
     </fpage> 
     <lpage>
       9438 
     </lpage> 
     <!--PubMed citation query: 'Proc Natl Acad Sci USA||99|9433||bib33|'--> 
    </citation> 
   </ref> 
   <ref id="bib34"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Tyner 
       </surname> 
       <given-names>
         JW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Erickson 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Deininger 
       </surname> 
       <given-names>
         MW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Willis 
       </surname> 
       <given-names>
         SG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Eide 
       </surname> 
       <given-names>
         CA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Levine 
       </surname> 
       <given-names>
         RL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Heinrich 
       </surname> 
       <given-names>
         MC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gattermann 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gilliland 
       </surname> 
       <given-names>
         DG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Druker 
       </surname> 
       <given-names>
         BJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Loriaux 
       </surname> 
       <given-names>
         MM 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       High-throughput sequencing screen reveals novel, transforming RAS mutations in myeloid leukemia patients 
     </article-h3> 
     <source />Blood 
     <year>
       2009 
     </year> 
     <volume>
       113 
     </volume> 
     <fpage>
       1749 
     </fpage> 
     <lpage>
       1755 
     </lpage> 
     <!--PubMed citation query: 'Blood||113|1749||bib34|'--> 
     <pub-id pub-id-type="pmid">
       19075190 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib35"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Van Cutsem 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kohne 
       </surname> 
       <given-names>
         CH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hitre 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zaluski 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chang Chien 
       </surname> 
       <given-names>
         CR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Makhson 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         D'Haens 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pinter 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lim 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bodoky 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Roh 
       </surname> 
       <given-names>
         JK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Folprecht 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ruff 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stroh 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tejpar 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Schlichting 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nippgen 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rougier 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Cetuximab and chemotherapy as initial treatment for metastatic colorectal cancer 
     </article-h3> 
     <source />N Engl J Med 
     <year>
       2009 
     </year> 
     <volume>
       360 
     </volume> 
     <fpage>
       1408 
     </fpage> 
     <lpage>
       1417 
     </lpage> 
     <!--PubMed citation query: 'N Engl J Med||360|1408||bib35|'--> 
     <pub-id pub-id-type="pmid">
       19339720 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib36"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         van Krieken 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tol 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Setting future standards for KRAS testing in colorectal cancer 
     </article-h3> 
     <source />Pharmacogenomics 
     <year>
       2009 
     </year> 
     <volume>
       10 
     </volume> 
     <fpage>
       1 
     </fpage> 
     <lpage>
       3 
     </lpage> 
     <!--PubMed citation query: 'Pharmacogenomics||10|1||bib36|'--> 
     <pub-id pub-id-type="pmid">
       19102707 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib37"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Vogelstein 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fearon 
       </surname> 
       <given-names>
         ER 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hamilton 
       </surname> 
       <given-names>
         SR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kern 
       </surname> 
       <given-names>
         SE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Preisinger 
       </surname> 
       <given-names>
         AC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Leppert 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nakamura 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         White 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Smits 
       </surname> 
       <given-names>
         AM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bos 
       </surname> 
       <given-names>
         JL 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Genetic alterations during colorectal-tumor development 
     </article-h3> 
     <source />N Engl J Med 
     <year>
       1988 
     </year> 
     <volume>
       319 
     </volume> 
     <fpage>
       525 
     </fpage> 
     <lpage>
       532 
     </lpage> 
     <!--PubMed citation query: 'N Engl J Med||319|525||bib37|'--> 
     <pub-id pub-id-type="pmid">
       2841597 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib38"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Wan 
       </surname> 
       <given-names>
         PT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Garnett 
       </surname> 
       <given-names>
         MJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Roe 
       </surname> 
       <given-names>
         SM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Niculescu-Duvaz 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Good 
       </surname> 
       <given-names>
         VM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jones 
       </surname> 
       <given-names>
         CM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Marshall 
       </surname> 
       <given-names>
         CJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Springer 
       </surname> 
       <given-names>
         CJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Barford 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Marais 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Mechanism of activation of the RAF-ERK signaling pathway by oncogenic mutations of B-RAF 
     </article-h3> 
     <source />Cell 
     <year>
       2004 
     </year> 
     <volume>
       116 
     </volume> 
     <fpage>
       855 
     </fpage> 
     <lpage>
       867 
     </lpage> 
     <!--PubMed citation query: 'Cell||116|855||bib38|'--> 
     <pub-id pub-id-type="pmid">
       15035987 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib39"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Warmka 
       </surname> 
       <given-names>
         JK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mauro 
       </surname> 
       <given-names>
         LJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wattenberg 
       </surname> 
       <given-names>
         EV 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Mitogen-activated protein kinase phosphatase-3 is a tumor promoter target in initiated cells that express oncogenic Ras 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       2004 
     </year> 
     <volume>
       279 
     </volume> 
     <fpage>
       33085 
     </fpage> 
     <lpage>
       33092 
     </lpage> 
     <!--PubMed citation query: 'J Biol Chem||279|33085||bib39|'--> 
     <pub-id pub-id-type="pmid">
       15159408 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib40"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Wellbrock 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Karasarides 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Marais 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The RAF proteins take centre stage 
     </article-h3> 
     <source />Nat Rev Mol Cell Biol 
     <year>
       2004 
     </year> 
     <volume>
       5 
     </volume> 
     <fpage>
       875 
     </fpage> 
     <lpage>
       885 
     </lpage> 
     <!--PubMed citation query: 'Nat Rev Mol Cell Biol||5|875||bib40|'--> 
     <pub-id pub-id-type="pmid">
       15520807 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib41"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Winder 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mundlein 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rhomberg 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dirschmid 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hartmann 
       </surname> 
       <given-names>
         BL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Knauer 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Drexel 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wenzl 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         De Vries 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lang 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Different types of K-Ras mutations are conversely associated with overall survival in patients with colorectal cancer 
     </article-h3> 
     <source />Oncol Rep 
     <year>
       2009 
     </year> 
     <volume>
       21 
     </volume> 
     <fpage>
       1283 
     </fpage> 
     <lpage>
       1287 
     </lpage> 
     <!--PubMed citation query: 'Oncol Rep||21|1283||bib41|'--> 
     <pub-id pub-id-type="pmid">
       19360305 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib42"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Wojcik 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kulig 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Okon 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zazula 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mozdzioch 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Niepsuj 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stachura 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       KRAS mutation profile in colorectal carcinoma and novel mutation–internal tandem duplication in KRAS 
     </article-h3> 
     <source />Pol J Pathol 
     <year>
       2008 
     </year> 
     <volume>
       59 
     </volume> 
     <fpage>
       93 
     </fpage> 
     <lpage>
       96 
     </lpage> 
     <!--PubMed citation query: 'Pol J Pathol||59|93||bib42|'--> 
     <pub-id pub-id-type="pmid">
       18669174 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib43"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Yip-Schneider 
       </surname> 
       <given-names>
         MT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lin 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Marshall 
       </surname> 
       <given-names>
         MS 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Pancreatic tumor cells with mutant K-ras suppress ERK activity by MEK-dependent induction of MAP kinase phosphatase-2 
     </article-h3> 
     <source />Biochem Biophys Res Commun 
     <year>
       2001 
     </year> 
     <volume>
       280 
     </volume> 
     <fpage>
       992 
     </fpage> 
     <lpage>
       997 
     </lpage> 
     <!--PubMed citation query: 'Biochem Biophys Res Commun||280|992||bib43|'--> 
     <pub-id pub-id-type="pmid">
       11162624 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="bib44"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Yun 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rago 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cheong 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pagliarini 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Angenendt 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rajagopalan 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Schmidt 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Willson 
       </surname> 
       <given-names>
         JK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Markowitz 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zhou 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Diaz 
       </surname> 
       <given-names>
         LA 
       </given-names> 
       <suffix>
         Jr 
       </suffix> 
      </name> 
      <name> 
       <surname>
         Velculescu 
       </surname> 
       <given-names>
         VE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lengauer 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kinzler 
       </surname> 
       <given-names>
         KW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vogelstein 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Papadopoulos 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Glucose deprivation contributes to the development of KRAS pathway mutations in tumor cells 
     </article-h3> 
     <source />Science 
     <year>
       2009 
     </year> 
     <volume>
       325 
     </volume> 
     <fpage>
       1555 
     </fpage> 
     <lpage>
       1559 
     </lpage> 
     <!--PubMed citation query: 'Science||325|1555||bib44|'--> 
     <pub-id pub-id-type="pmid">
       19661383 
     </pub-id> 
    </citation> 
   </ref> 
  </ref-list> 
 </back> 
 <floats-wrap> 
  <fig id="fig1"> 
   <label>Figure 1</label> 
   <p>The Ras cycle. Ras proteins are key components of signal transduction pathways leading from cell-surface receptors to the control of cell proliferation, differentiation or death. Active Ras, where tumour-specific mutations lock Ras in the GTP-bound conformation, stimulates the RAS–RAF–MEK–ERK–MAP kinase signalling pathway.</p> 
   <graphic mime-subtype="eps" xlink:href="6605534f1"></graphic> 
  </fig> 
  <fig id="fig2"> 
   <label>Figure 2</label> 
   <p>Location of novel and hotspot Kirsten-Ras ( 
    <italic>
      K-Ras 
    </italic>) mutations. The location of new (red) and hotspot (blue) 
    <italic>
      K-Ras 
    </italic> mutations are illustrated on a representation of the 
    <italic>
      K-Ras 
    </italic> protein sequence, together with the position of the novel 
    <italic>
      K-Ras 
    </italic> single-nucleotide polymorphism (SNP) (black). Putative GTP and effector binding sites and GAP and GEF interaction domains are highlighted.</p> 
   <graphic mime-subtype="eps" xlink:href="6605534f2"></graphic> 
  </fig> 
  <fig id="fig3"> 
   <label>Figure 3</label> 
   <p>Focus formation assays. NIH3T3 cells were transfected with pEF.6 or plasmids containing wt or mutant 
    <italic>
      K-Ras 
    </italic> and foci visualised after crystal violet staining. ( 
    <bold>
      A 
    </bold>) Cells transfected with empty vector or wt 
    <italic>
      K-Ras 
    </italic> were compared with previously described hotspot mutations. ( 
    <bold>
      B 
    </bold>) The transforming potential of L19F, K117N, A146T and R164Q mutations were compared with the hotspot mutation G12V and wt 
    <italic>
      K-Ras 
    </italic>. All experiments were performed in duplicate and each set of transfections was repeated three times. ( 
    <bold>
      C 
    </bold>) The combined results of all transfections, wherein total foci counts are presented ± SD are illustrated.</p> 
   <graphic mime-subtype="eps" xlink:href="6605534f3"></graphic> 
  </fig> 
  <fig id="fig4"> 
   <label>Figure 4</label> 
   <p>Ras GTPase activating assays. Ras GTPase activating assays were carried out for each of the novel Kirsten-Ras ( 
    <italic>
      K-Ras 
    </italic>) mutations and western blotting used to assess the expression of ( 
    <bold>
      A 
    </bold>) active GTP-bound K-Ras ( 
    <bold>
      B 
    </bold>) total 
    <italic>
      K-Ras 
    </italic> and ( 
    <bold>
      C 
    </bold>) 
    <italic>
      ? 
    </italic>-actin (loading control). The 
    <italic>
      K-Ras 
    </italic> G12V construct was included as a positive control for 
    <italic>
      K-Ras 
    </italic> protein in the active GTP-bound conformation.</p> 
   <graphic mime-subtype="eps" xlink:href="6605534f4"></graphic> 
  </fig> 
  <fig id="fig5"> 
   <label>Figure 5</label> 
   <p>Hierarchical clustering analysis. RNA transcription profiling analysis was carried out as described in Materials and Methods. Lists of differentially expressed genes were generated comparing each of the Kirsten-Ras ( 
    <italic>
      K-Ras 
    </italic>) mutants with empty vector control using probes that exhibited an adjusted 
    <italic>
      P 
    </italic>-value (false discovery rate (FDR)) ?0.05. Following hierarchical clustering analysis, gene clusters were represented on a heat map, where upregulated genes are highlighted in green and downregulated genes highlighted in red.</p> 
   <graphic mime-subtype="eps" xlink:href="6605534f5"></graphic> 
  </fig> 
  <fig id="fig6"> 
   <label>Figure 6</label> 
   <p>Kirsten-Ras ( 
    <italic>
      K-Ras 
    </italic>) gene copy number assay. A novel 
    <italic>
      K-Ras 
    </italic> gene copy number assay was designed as described in Materials and Methods and used to screen genomic DNA samples extracted from 96 paired normal (N) and colorectal tumour (T) tissues. Representative results from six normal/tumour pairs are illustrated, highlighting increased copy number in samples 1233 and 1264.</p> 
   <graphic mime-subtype="eps" xlink:href="6605534f6"></graphic> 
  </fig> 
  <fig id="fig7"> 
   <label>Figure 7</label> 
   <p>FISH analysis. Chromosome metaphase spreads were hybridised with FISH probes for ( 
    <bold>
      A 
    </bold>) chromosome 12p (red) and ( 
    <bold>
      B 
    </bold>) chromosome 12q (green) to confirm probe specificity and FISH analysis performed as described in Materials and Methods. Representative colorectal tumour sections ( 
    <bold>
      C 
    </bold>) without and ( 
    <bold>
      D 
    </bold>) with a 
    <italic>
      K-Ras 
    </italic> gene amplification are illustrated.</p> 
   <graphic mime-subtype="eps" xlink:href="6605534f7"></graphic> 
  </fig> 
  <table-wrap id="tbl1" position="float"> 
   <label>Table 1</label> 
   <p content-type="table-h3">Site-directed mutagenesis (SDM) primers for Kirsten-Ras ( 
    <italic>
      K-Ras 
    </italic>) mutation generation</p> 
   <table border="1" frame="hsides" rules="groups" width="85%"> 
    <colgroup> 
     <col align="left" /> 
     <col align="left" /> 
    </colgroup> 
    <thead valign="bottom"> 
     <tr> 
      <th align="left" charoff="50" valign="top"> 
       <bold>
         Primer name 
       </bold> </th> 
      <th align="left" charoff="50" valign="top"> 
       <bold>
         Primer sequence 
       </bold> </th> 
     </tr> 
    </thead> 
    <tbody valign="top"> 
     <tr> 
      <td align="left" charoff="50" valign="top">G12V SDM Forward</td> 
      <td align="left" charoff="50" valign="top">5?-GTGGTGGGCGCTGTTGGCGTGGGAAAGAG-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">G12V SDM Reverse</td> 
      <td align="left" charoff="50" valign="top">5?-CTCTTTCCCACGCCAACAGCGCCCACCAC-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">G12D SDM Forward</td> 
      <td align="left" charoff="50" valign="top">5?-GTGGTGGGCGCTGATGGCGTGGGAAAGAG-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">G12D SDM Reverse</td> 
      <td align="left" charoff="50" valign="top">5?-CTCTTTCCCACGCCATCAGCGCCCACCAC-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">G13D SDM Forward</td> 
      <td align="left" charoff="50" valign="top">5?-GTGGGCGCTGGAGACGTGGGAAAGAGTG-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">G13D SDM Reverse</td> 
      <td align="left" charoff="50" valign="top">5?-CACTCTTTCCCACGTCTCCAGCGCCCAC-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">Q61H SDM Forward</td> 
      <td align="left" charoff="50" valign="top">5?-CGACACAGCAGGTCATGAGGAGTACAGTGC-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">Q61H SDM Reverse</td> 
      <td align="left" charoff="50" valign="top">5?-GCACTGTACTCCTCATGACCTGCTGTGTCG-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">L19F SDM Forward</td> 
      <td align="left" charoff="50" valign="top">5?-GTGGGAAAGAGTGCCTTCACCATCCAGCTGATC-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">L19R SDM Reverse</td> 
      <td align="left" charoff="50" valign="top">5?-GATCAGCTGGATGGTGAAGGCACTCTTTCCCAC-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">K117N SDM Forward</td> 
      <td align="left" charoff="50" valign="top">5?-CCTATGGTCCTAGTAGGAAATAACTGTGATTTGCCTTC-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">K117N SDM Reverse</td> 
      <td align="left" charoff="50" valign="top">5?-GAAGGCAAATCACAGTTATTTCCTACTAGGACCATAGG-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">A146T SDM Forward</td> 
      <td align="left" charoff="50" valign="top">5?-CCTTTTATTGAAACATCAACAAAGACAAGACAGGGTGTTGATG-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">A146T SDM Reverse</td> 
      <td align="left" charoff="50" valign="top">5?-CATCAACACCCTGTCTTGTCTTTGTTGATGTTTCAATAAAAGG-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">R164Q SDM Forward</td> 
      <td align="left" charoff="50" valign="top">5?-CATTAGTTCGAGAAATTCAAAAACATAAAGAAAAGATGAGCAAAGATGG-3?</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">R164Q SDM Reverse</td> 
      <td align="left" charoff="50" valign="top">5?-CCATCTTTGCTCATCTTTTCTTTATGTTTTTGAATTTCTCGAACTAATG-3?</td> 
     </tr> 
    </tbody> 
   </table> 
  </table-wrap> 
  <table-wrap id="tbl2" position="float"> 
   <label>Table 2</label> 
   <p content-type="table-h3">Frequencies of hotspot and novel mutations in Kirsten-Ras ( 
    <italic>
      K-Ras 
    </italic>) in colorectal tumours</p> 
   <table border="1" frame="hsides" rules="groups" width="85%"> 
    <colgroup> 
     <col align="left" /> 
     <col align="left" /> 
     <col align="left" /> 
     <col align="char" char="(" /> 
    </colgroup> 
    <thead valign="bottom"> 
     <tr> 
      <th align="left" charoff="50" valign="top"> 
       <bold>
         Mutation 
       </bold> </th> 
      <th align="left" charoff="50" valign="top"> 
       <bold>
         Nucleotide change 
       </bold> </th> 
      <th align="left" charoff="50" valign="top"> 
       <bold>
         Amino acid change 
       </bold> </th> 
      <th align="center" char="(" charoff="50" valign="top"> 
       <bold>
         Frequency (%) 
       </bold> </th> 
     </tr> 
    </thead> 
    <tbody valign="top"> 
     <tr> 
      <td align="left" charoff="50" colspan="4" valign="top"> 
       <italic> 
        <bold>
          K-Ras 
        </bold> 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="4" valign="top"> 
       <italic>
         (A) Hotspot codon mutations 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="4" valign="top">?Codon</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">??12</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">23/106 (21.7%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">G-&gt;A</td> 
      <td align="left" charoff="50" valign="top">Gly<sub>12</sub>Ser</td> 
      <td align="char" char="(" charoff="50" valign="top">4/106 (3.8%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">G-&gt;C</td> 
      <td align="left" charoff="50" valign="top">Gly<sub>12</sub>Ala</td> 
      <td align="char" char="(" charoff="50" valign="top">3/106 (2.8%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">G-&gt;T</td> 
      <td align="left" charoff="50" valign="top">Gly<sub>12</sub>Cys</td> 
      <td align="char" char="(" charoff="50" valign="top">2/106 (1.9%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">G-&gt;T</td> 
      <td align="left" charoff="50" valign="top">Gly<sub>12</sub>Val</td> 
      <td align="char" char="(" charoff="50" valign="top">6/106 (5.7%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">G-&gt;C</td> 
      <td align="left" charoff="50" valign="top">Gly<sub>12</sub>Arg</td> 
      <td align="char" char="(" charoff="50" valign="top">1/106 (&lt;1%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">G-&gt;A</td> 
      <td align="left" charoff="50" valign="top">Gly<sub>12</sub>Asp</td> 
      <td align="char" char="(" charoff="50" valign="top">7/106 (6.6%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">??13</td> 
      <td align="left" charoff="50" valign="top">G-&gt;A</td> 
      <td align="left" charoff="50" valign="top">Gly<sub>13</sub>Asp</td> 
      <td align="char" char="(" charoff="50" valign="top">6/106 (5.7%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">??61</td> 
      <td align="left" charoff="50" valign="top">None detected</td> 
      <td align="left" charoff="50" valign="top">None detected</td> 
      <td align="char" char="(" charoff="50" valign="top">0/106</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">Mutation total</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">29/106 (27.4%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="4" valign="top"> 
       <italic>
         (B) Novel codon mutations 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="4" valign="top">?Codon</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">??19</td> 
      <td align="left" charoff="50" valign="top">G-&gt;T</td> 
      <td align="left" charoff="50" valign="top">Leu<sub>19</sub>Phe</td> 
      <td align="char" char="(" charoff="50" valign="top">1/106 (&lt;1%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">??117</td> 
      <td align="left" charoff="50" valign="top">A-&gt;C</td> 
      <td align="left" charoff="50" valign="top">Lys<sub>117</sub>Asn</td> 
      <td align="char" char="(" charoff="50" valign="top">1/106 (&lt;1%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">??146</td> 
      <td align="left" charoff="50" valign="top">G-&gt;A</td> 
      <td align="left" charoff="50" valign="top">Ala<sub>146</sub>Thr</td> 
      <td align="char" char="(" charoff="50" valign="top">7/106 (6.6%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">??164</td> 
      <td align="left" charoff="50" valign="top">G-&gt;A</td> 
      <td align="left" charoff="50" valign="top">Arg<sub>164</sub>Gln</td> 
      <td align="char" char="(" charoff="50" valign="top">1/106 (&lt;1%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">??173<sup>a</sup></td> 
      <td align="left" charoff="50" valign="top">T-&gt;C</td> 
      <td align="left" charoff="50" valign="top">No change</td> 
      <td align="char" char="(" charoff="50" valign="top">39/106 (36.8%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">New mutation total</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">39/106 (36.8%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="4" valign="top"> 
       <italic>
         B-Raf 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="4" valign="top">?Codon</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">??600</td> 
      <td align="left" charoff="50" valign="top">T-&gt;A</td> 
      <td align="left" charoff="50" valign="top">Val600Glu</td> 
      <td align="char" char="(" charoff="50" valign="top">1/106 (&lt;1%)</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">Final mutation total</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="(" charoff="50" valign="top">40/106 (37.8%)</td> 
     </tr> 
    </tbody> 
   </table> 
   <table-wrap-foot> 
    <fn id="t2-fn1"> 
     <label>a</label> 
     <p>The sequence change at codon 173 is a single-nucleotide polymorphism, not a tumour-specific mutation.</p> 
    </fn> 
   </table-wrap-foot> 
  </table-wrap> 
  <table-wrap id="tbl3" position="float"> 
   <label>Table 3</label> 
   <p content-type="table-h3">Summary of genes differentially expressed by the different Kirsten-Ras ( 
    <italic>
      K-Ras 
    </italic>) mutations relative to NIH3T3 cells mock transfected with pEF vector</p> 
   <table border="1" frame="hsides" rules="groups" width="85%"> 
    <colgroup> 
     <col align="left" /> 
     <col align="left" /> 
     <col align="char" char="." /> 
     <col align="char" char="." /> 
     <col align="char" char="." /> 
     <col align="char" char="." /> 
     <col align="char" char="." /> 
     <col align="char" char="." /> 
     <col align="char" char="." /> 
     <col align="char" char="." /> 
     <col align="char" char="." /> 
    </colgroup> 
    <thead valign="bottom"> 
     <tr> 
      <th align="left" charoff="50" valign="top">&nbsp;</th> 
      <th align="left" charoff="50" valign="top">&nbsp;</th> 
      <th align="center" char="." charoff="50" colspan="9" valign="top"> 
       <bold>
         Fold change 
       </bold> </th> 
     </tr> 
     <tr> 
      <th align="left" charoff="50" valign="top">&nbsp;</th> 
      <th align="left" charoff="50" valign="top">&nbsp;</th> 
      <th align="center" char="." charoff="50" colspan="3" valign="top"> 
       <bold>
         Cluster 1 
       </bold> </th> 
      <th align="center" char="." charoff="50" colspan="6" valign="top"> 
       <bold>
         Cluster 2 
       </bold> </th> 
     </tr> 
     <tr> 
      <th align="left" charoff="50" valign="top"> 
       <bold>
         Gene 
       </bold> </th> 
      <th align="left" charoff="50" valign="top"> 
       <bold>
         Gene function 
       </bold> </th> 
      <th align="center" char="." charoff="50" valign="top"> 
       <bold>
         G12V 
       </bold> </th> 
      <th align="center" char="." charoff="50" valign="top"> 
       <bold>
         G12C 
       </bold> </th> 
      <th align="center" char="." charoff="50" valign="top"> 
       <bold>
         G12D 
       </bold> </th> 
      <th align="center" char="." charoff="50" valign="top"> 
       <bold>
         G13D 
       </bold> </th> 
      <th align="center" char="." charoff="50" valign="top"> 
       <bold>
         A146T 
       </bold> </th> 
      <th align="center" char="." charoff="50" valign="top"> 
       <bold>
         K117N 
       </bold> </th> 
      <th align="center" char="." charoff="50" valign="top"> 
       <bold>
         Q61H 
       </bold> </th> 
      <th align="center" char="." charoff="50" valign="top"> 
       <bold>
         R164Q 
       </bold> </th> 
      <th align="center" char="." charoff="50" valign="top"> 
       <bold>
         L19F 
       </bold> </th> 
     </tr> 
    </thead> 
    <tbody valign="top"> 
     <tr> 
      <td align="left" charoff="50" colspan="11" valign="top"> 
       <italic>
         Signal transduction 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         DUSP4 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">MAPK phosphatase activity</td> 
      <td align="char" char="." charoff="50" valign="top">2.4</td> 
      <td align="char" char="." charoff="50" valign="top">4.3</td> 
      <td align="char" char="." charoff="50" valign="top">3.1</td> 
      <td align="char" char="." charoff="50" valign="top">3.3</td> 
      <td align="char" char="." charoff="50" valign="top">2.9</td> 
      <td align="char" char="." charoff="50" valign="top">3.2</td> 
      <td align="char" char="." charoff="50" valign="top">2.9</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">2.9</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         DUSP6 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Phosphoprotein phosphatase activity</td> 
      <td align="char" char="." charoff="50" valign="top">2.8</td> 
      <td align="char" char="." charoff="50" valign="top">3</td> 
      <td align="char" char="." charoff="50" valign="top">2.5</td> 
      <td align="char" char="." charoff="50" valign="top">2.2</td> 
      <td align="char" char="." charoff="50" valign="top">2.2</td> 
      <td align="char" char="." charoff="50" valign="top">2.6</td> 
      <td align="char" char="." charoff="50" valign="top">2.8</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">1.9</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Ptpre 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Protein tyrosine phosphatase activity</td> 
      <td align="char" char="." charoff="50" valign="top">3.2</td> 
      <td align="char" char="." charoff="50" valign="top">5.1</td> 
      <td align="char" char="." charoff="50" valign="top">3.8</td> 
      <td align="char" char="." charoff="50" valign="top">5.2</td> 
      <td align="char" char="." charoff="50" valign="top">4.0</td> 
      <td align="char" char="." charoff="50" valign="top">5.6</td> 
      <td align="char" char="." charoff="50" valign="top">4.4</td> 
      <td align="char" char="." charoff="50" valign="top">2.7</td> 
      <td align="char" char="." charoff="50" valign="top">4.5</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Spry4 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Protein binding</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">7.0</td> 
      <td align="char" char="." charoff="50" valign="top">5.9</td> 
      <td align="char" char="." charoff="50" valign="top">4.1</td> 
      <td align="char" char="." charoff="50" valign="top">5.6</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">4.4</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Ereg 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">EGFR binding</td> 
      <td align="char" char="." charoff="50" valign="top">7.5</td> 
      <td align="char" char="." charoff="50" valign="top">11.1</td> 
      <td align="char" char="." charoff="50" valign="top">7.9</td> 
      <td align="char" char="." charoff="50" valign="top">7.0</td> 
      <td align="char" char="." charoff="50" valign="top">7.0</td> 
      <td align="char" char="." charoff="50" valign="top">7.3</td> 
      <td align="char" char="." charoff="50" valign="top">8.6</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">6.2</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Hbegf 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">EGFR binding</td> 
      <td align="char" char="." charoff="50" valign="top">2.3</td> 
      <td align="char" char="." charoff="50" valign="top">3.6</td> 
      <td align="char" char="." charoff="50" valign="top">2.8</td> 
      <td align="char" char="." charoff="50" valign="top">2.6</td> 
      <td align="char" char="." charoff="50" valign="top">2.2</td> 
      <td align="char" char="." charoff="50" valign="top">2.7</td> 
      <td align="char" char="." charoff="50" valign="top">2.7</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">2.2</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Btc 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">EGFR binding</td> 
      <td align="char" char="." charoff="50" valign="top">4.2</td> 
      <td align="char" char="." charoff="50" valign="top">5.3</td> 
      <td align="char" char="." charoff="50" valign="top">5.1</td> 
      <td align="char" char="." charoff="50" valign="top">4.5</td> 
      <td align="char" char="." charoff="50" valign="top">4.5</td> 
      <td align="char" char="." charoff="50" valign="top">3.9</td> 
      <td align="char" char="." charoff="50" valign="top">5.8</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">4.1</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Vegfa 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Growth factor activity</td> 
      <td align="char" char="." charoff="50" valign="top">1.8</td> 
      <td align="char" char="." charoff="50" valign="top">2.3</td> 
      <td align="char" char="." charoff="50" valign="top">1.8</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Igf1R 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Protein tyrosine kinase activity</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">2.7</td> 
      <td align="char" char="." charoff="50" valign="top">2.3</td> 
      <td align="char" char="." charoff="50" valign="top">2.3</td> 
      <td align="char" char="." charoff="50" valign="top">2.8</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">2.7</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Jun 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Transcription factor</td> 
      <td align="char" char="." charoff="50" valign="top">2.0</td> 
      <td align="char" char="." charoff="50" valign="top">2.0</td> 
      <td align="char" char="." charoff="50" valign="top">1.8</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">1.4</td> 
      <td align="char" char="." charoff="50" valign="top">1.4</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         E2F2 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Transcription factor</td> 
      <td align="char" char="." charoff="50" valign="top">3.3</td> 
      <td align="char" char="." charoff="50" valign="top">4.2</td> 
      <td align="char" char="." charoff="50" valign="top">3.5</td> 
      <td align="char" char="." charoff="50" valign="top">2.4</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">2.3</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Creb1 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Transcription factor</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">2.3</td> 
      <td align="char" char="." charoff="50" valign="top">2.0</td> 
      <td align="char" char="." charoff="50" valign="top">2.0</td> 
      <td align="char" char="." charoff="50" valign="top">2.3</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">2.8</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Wnt5A 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Receptor binding</td> 
      <td align="char" char="." charoff="50" valign="top">2.3</td> 
      <td align="char" char="." charoff="50" valign="top">2.8</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">2.5</td> 
      <td align="char" char="." charoff="50" valign="top">2.8</td> 
      <td align="char" char="." charoff="50" valign="top">2.2</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">2.4</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         TCF4 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Transcription factor</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">2.7</td> 
      <td align="char" char="." charoff="50" valign="top">2.5</td> 
      <td align="char" char="." charoff="50" valign="top">3.2</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">2.6</td> 
      <td align="char" char="." charoff="50" valign="top">3.2</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="11" valign="top"> 
       <italic>
         Cytoskeleton remodelling 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Arhgap6 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Rho GTPase activator activity</td> 
      <td align="char" char="." charoff="50" valign="top">8.7</td> 
      <td align="char" char="." charoff="50" valign="top">10.5</td> 
      <td align="char" char="." charoff="50" valign="top">4.6</td> 
      <td align="char" char="." charoff="50" valign="top">10.7</td> 
      <td align="char" char="." charoff="50" valign="top">9.2</td> 
      <td align="char" char="." charoff="50" valign="top">9.7</td> 
      <td align="char" char="." charoff="50" valign="top">13.2</td> 
      <td align="char" char="." charoff="50" valign="top">3.0</td> 
      <td align="char" char="." charoff="50" valign="top">7.2</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         NGEF 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Rho guanine-exchange factor activity</td> 
      <td align="char" char="." charoff="50" valign="top">3.5</td> 
      <td align="char" char="." charoff="50" valign="top">9.1</td> 
      <td align="char" char="." charoff="50" valign="top">5.5</td> 
      <td align="char" char="." charoff="50" valign="top">3.9</td> 
      <td align="char" char="." charoff="50" valign="top">3.8</td> 
      <td align="char" char="." charoff="50" valign="top">5.0</td> 
      <td align="char" char="." charoff="50" valign="top">4.0</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">3.0</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="11" valign="top"> 
       <italic>
         Cell adhesion 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         MMP3 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Metalloendopeptidase activity</td> 
      <td align="char" char="." charoff="50" valign="top">3.5</td> 
      <td align="char" char="." charoff="50" valign="top">2.3</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">2.5</td> 
      <td align="char" char="." charoff="50" valign="top">2.9</td> 
      <td align="char" char="." charoff="50" valign="top">3.3</td> 
      <td align="char" char="." charoff="50" valign="top">3.4</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Ceacam1 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Cell adhesion</td> 
      <td align="char" char="." charoff="50" valign="top">17.8</td> 
      <td align="char" char="." charoff="50" valign="top">29.9</td> 
      <td align="char" char="." charoff="50" valign="top">21.1</td> 
      <td align="char" char="." charoff="50" valign="top">19.9</td> 
      <td align="char" char="." charoff="50" valign="top">20.4</td> 
      <td align="char" char="." charoff="50" valign="top">30.0</td> 
      <td align="char" char="." charoff="50" valign="top">27.5</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">15.6</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="11" valign="top"> 
       <italic>
         G-protein signalling 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         GNAQ 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Signal transducer activity</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">1.8</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">1.5</td> 
      <td align="char" char="." charoff="50" valign="top">1.7</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">1.8</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Gna11 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Signal transducer activity</td> 
      <td align="char" char="." charoff="50" valign="top">?1.5</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">?1.6</td> 
      <td align="char" char="." charoff="50" valign="top">?1.9</td> 
      <td align="char" char="." charoff="50" valign="top">?1.5</td> 
      <td align="char" char="." charoff="50" valign="top">?1.9</td> 
      <td align="char" char="." charoff="50" valign="top">?1.6</td> 
      <td align="char" char="." charoff="50" valign="top">?1.7</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="11" valign="top"> 
       <italic>
         Immune response 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Ccl2 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Cytokine</td> 
      <td align="char" char="." charoff="50" valign="top">3.2</td> 
      <td align="char" char="." charoff="50" valign="top">2.8</td> 
      <td align="char" char="." charoff="50" valign="top">2.2</td> 
      <td align="char" char="." charoff="50" valign="top">1.9</td> 
      <td align="char" char="." charoff="50" valign="top">2.0</td> 
      <td align="char" char="." charoff="50" valign="top">2.2</td> 
      <td align="char" char="." charoff="50" valign="top">2.0</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="11" valign="top"> 
       <italic>
         Protein kinases 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Abl1 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Tyrosine-protein kinase</td> 
      <td align="char" char="." charoff="50" valign="top">?1.6</td> 
      <td align="char" char="." charoff="50" valign="top">?1.5</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">?1.6</td> 
      <td align="char" char="." charoff="50" valign="top">?1.6</td> 
      <td align="char" char="." charoff="50" valign="top">?1.8</td> 
      <td align="char" char="." charoff="50" valign="top">?2.0</td> 
      <td align="char" char="." charoff="50" valign="top">?1.6</td> 
      <td align="char" char="." charoff="50" valign="top">?1.8</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Pak3 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Protein serine/threonine kinase</td> 
      <td align="char" char="." charoff="50" valign="top">?1.6</td> 
      <td align="char" char="." charoff="50" valign="top">?1.8</td> 
      <td align="char" char="." charoff="50" valign="top">?1.7</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Pim1 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Protein serine/threonine kinase</td> 
      <td align="char" char="." charoff="50" valign="top">1.9</td> 
      <td align="char" char="." charoff="50" valign="top">1.6</td> 
      <td align="char" char="." charoff="50" valign="top">1.6</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="11" valign="top"> 
       <italic>
         Cell survival 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Serpinb2 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Serine-type endopeptidase inhibitor activity</td> 
      <td align="char" char="." charoff="50" valign="top">12.4</td> 
      <td align="char" char="." charoff="50" valign="top">20.7</td> 
      <td align="char" char="." charoff="50" valign="top">11.7</td> 
      <td align="char" char="." charoff="50" valign="top">15.8</td> 
      <td align="char" char="." charoff="50" valign="top">19.1</td> 
      <td align="char" char="." charoff="50" valign="top">18.5</td> 
      <td align="char" char="." charoff="50" valign="top">18.5</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">13.7</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="11" valign="top"> 
       <italic>
         Transcription 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Ell2 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">RNA polymerase II transcription elongation factor activity</td> 
      <td align="char" char="." charoff="50" valign="top">3.1</td> 
      <td align="char" char="." charoff="50" valign="top">2.8</td> 
      <td align="char" char="." charoff="50" valign="top">2.8</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="left" charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
      <td align="char" char="." charoff="50" valign="top">&nbsp;</td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" colspan="11" valign="top"> 
       <italic>
         Metabolism/glycolysis 
       </italic> </td> 
     </tr> 
     <tr> 
      <td align="left" charoff="50" valign="top">? 
       <italic>
         Glut-1 
       </italic> </td> 
      <td align="left" charoff="50" valign="top">Glucose transport</td> 
      <td align="char" char="." charoff="50" valign="top">2.0</td> 
      <td align="char" char="." charoff="50" valign="top">1.8</td> 
      <td align="char" char="." charoff="50" valign="top">1.6</td> 
      <td align="char" char="." charoff="50" valign="top">1.8</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">1.5</td> 
      <td align="char" char="." charoff="50" valign="top">1.5</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
      <td align="char" char="." charoff="50" valign="top">ND</td> 
     </tr> 
    </tbody> 
   </table> 
   <table-wrap-foot> 
    <fn id="t3-fn1"> 
     <label></label> 
     <p>Abbreviation: ND=statistically significant fold change not detected.</p> 
    </fn> 
   </table-wrap-foot> 
  </table-wrap> 
 </floats-wrap> 
</article>