<article xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"> 
 <!--?properties open_acces=""?--> 
 <front> 
  <journal-meta> 
   <journal-id journal-id-type="nlm-ta">
     BMC Bioinformatics 
   </journal-id> 
   <journal-h3>
     BMC Bioinformatics 
   </journal-h3> 
   <issn pub-type="epub">
     1471-2105 
   </issn> 
   <publisher> 
    <publisher-name>
      BioMed Central 
    </publisher-name> 
   </publisher> 
  </journal-meta> 
  <article-meta> 
   <article-id pub-id-type="pmid">
     19828069 
   </article-id> 
   <article-id pub-id-type="pmc">
     2762058 
   </article-id> 
   <article-id pub-id-type="publisher-id">
     1471-2105-10-S12-S1 
   </article-id> 
   <article-id pub-id-type="doi">
     10.1186/1471-2105-10-S12-S1 
   </article-id> 
   <article-categories> 
    <subj-group subj-group-type="heading"> 
     <subject>
       Research 
     </subject> 
    </subj-group> 
   </article-categories> 
   <h3-group> 
    <article-h3>
      Data recovery and integration from public databases uncovers transformation-specific transcriptional downregulation of cAMP-PKA pathway-encoding genes 
    </article-h3> 
   </h3-group> 
   <contrib-group> 
    <contrib contrib-type="author" id="A1"> 
     <name> 
      <surname>
        Balestrieri 
      </surname> 
      <given-names>
        Chiara 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       c.balestrieri@campus.unimib.it 
     </email> 
    </contrib> 
    <contrib contrib-type="author" id="A2"> 
     <name> 
      <surname>
        Alberghina 
      </surname> 
      <given-names>
        Lilia 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       lilia.alberghina@unimib.it 
     </email> 
    </contrib> 
    <contrib contrib-type="author" id="A3"> 
     <name> 
      <surname>
        Vanoni 
      </surname> 
      <given-names>
        Marco 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       marco.vanoni@unimib.it 
     </email> 
    </contrib> 
    <contrib contrib-type="author" corresp="yes" id="A4"> 
     <name> 
      <surname>
        Chiaradonna 
      </surname> 
      <given-names>
        Ferdinando 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       ferdinando.chiaradonna@unimib.it 
     </email> 
    </contrib> 
   </contrib-group> 
   <aff id="I1"> 
    <label>1</label>Department of Biotechnology and Biosciences, University of Milano-Bicocca, Piazza della Scienza 2, 20126 Milano, Italy 
   </aff> 
   <pub-date pub-type="collection"> 
    <year>
      2009 
    </year> 
   </pub-date> 
   <pub-date pub-type="epub"> 
    <day>
      15 
    </day> 
    <month>
      10 
    </month> 
    <year>
      2009 
    </year> 
   </pub-date> 
   <volume>
     10 
   </volume> 
   <issue>
     Suppl 12 
   </issue> 
   <supplement> 
    <named-content content-type="supplement-h3">
      Bioinformatics Methods for Biomedical Complex System Applications 
    </named-content> 
    <named-content content-type="supplement-editor">
      Luciano Milanesi and Paolo Romano 
    </named-content> 
    <ext-link ext-link-type="uri" xlink:href="http://www.biomedcentral.com/content/pdf/1471-2105-10-S12-info.pdf">
      http://www.biomedcentral.com/content/pdf/1471-2105-10-S12-info.pdf 
    </ext-link> 
   </supplement> 
   <fpage>
     S1 
   </fpage> 
   <lpage>
     S1 
   </lpage> 
   <ext-link ext-link-type="uri" xlink:href="http://www.biomedcentral.com/1471-2105/10/S12/S1"></ext-link> 
   <permissions> 
    <copyright-statement>
      Copyright © 2009 Balestrieri et al; licensee BioMed Central Ltd. 
    </copyright-statement> 
    <copyright-year>
      2009 
    </copyright-year> 
    <copyright-holder>
      Balestrieri et al; licensee BioMed Central Ltd. 
    </copyright-holder> 
    <license license-type="open-access" xlink:href="http://creativecommons.org/licenses/by/2.0"> 
     <p>This is an open access article distributed under the terms of the Creative Commons Attribution License ( 
      <ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/2.0"></ext-link> ), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p> 
     <!--<rdf xmlns="http://web.resource.org/cc/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:dc="http://purl.org/dc/elements/1.1" xmlns:dcterms="http://purl.org/dc/terms"><Work xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dcterms="http://purl.org/dc/terms/" rdf:about=""><license rdf:resource="http://creativecommons.org/licenses/by/2.0"/><dc:type rdf:resource="http://purl.org/dc/dcmitype/Text"/><dc:author>
               Balestrieri
               Chiara
               
               c.balestrieri@campus.unimib.it
            </dc:author><dc:h3>
            Data recovery and integration from public databases uncovers transformation-specific transcriptional downregulation of cAMP-PKA pathway-encoding genes
         </dc:h3><dc:date>2009</dc:date><dcterms:bibliographicCitation>BMC Bioinformatics 10(Suppl 12): S1-. (2009)</dcterms:bibliographicCitation><dc:identifier type="sici">1471-2105(2009)10:Suppl 12&#x0003c;S1&#x0003e;</dc:identifier><dcterms:isPartOf>urn:ISSN:1471-2105</dcterms:isPartOf><License rdf:about="http://creativecommons.org/licenses/by/2.0"><permits rdf:resource="http://web.resource.org/cc/Reproduction" xmlns=""/><permits rdf:resource="http://web.resource.org/cc/Distribution" xmlns=""/><requires rdf:resource="http://web.resource.org/cc/Notice" xmlns=""/><requires rdf:resource="http://web.resource.org/cc/Attribution" xmlns=""/><permits rdf:resource="http://web.resource.org/cc/DerivativeWorks" xmlns=""/></License></Work></rdf>--> 
    </license> 
   </permissions> 
   <abstract>
    <h3>Abstract</h3> 
    <sec> 
     <h3>Background</h3> 
     <p>The integration of data from multiple genome-wide assays is essential for understanding dynamic spatio-temporal interactions within cells. Such integration, which leads to a more complete view of cellular processes, offers the opportunity to rationalize better the high amount of "omics" data freely available in several public databases.</p> 
     <p>In particular, integration of microarray-derived transcriptome data with other high-throughput analyses (genomic and mutational analysis, promoter analysis) may allow us to unravel transcriptional regulatory networks under a variety of physio-pathological situations, such as the alteration in the cross-talk between signal transduction pathways in transformed cells.</p> 
    </sec> 
    <sec> 
     <h3>Results</h3> 
     <p>Here we sequentially apply web-based and statistical tools to a case study: the role of oncogenic activation of different signal transduction pathways in the transcriptional regulation of genes encoding proteins involved in the cAMP-PKA pathway. To this end, we first re-analyzed available genome-wide expression data for genes encoding proteins of the downstream branch of the PKA pathway in normal tissues and human tumor cell lines. Then, in order to identify mutation-dependent transcriptional signatures, we classified cancer cells as a function of their mutational state. The results of such procedure were used as a starting point to analyze the structure of PKA pathway-encoding genes promoters, leading to identification of specific combinations of transcription factor binding sites, which are neatly consistent with available experimental data and help to clarify the relation between gene expression, transcriptional factors and oncogenes in our case study.</p> 
    </sec> 
    <sec> 
     <h3>Conclusions</h3> 
     <p>Genome-wide, large-scale "omics" experimental technologies give different, complementary perspectives on the structure and regulatory properties of complex systems. Even the relatively simple, integrated workflow presented here offers opportunities not only for filtering data noise intrinsic in high throughput data, but also to progressively extract novel information that would have remained hidden otherwise. In fact we have been able to detect a strong transcriptional repression of genes encoding proteins of cAMP/PKA pathway in cancer cells of different genetic origins. The basic workflow presented herein may be easily extended by incorporating other tools and can be applied even by researchers with poor bioinformatics skills.</p> 
    </sec> 
   </abstract> 
   <conference> 
    <conf-date>
      19–21 May 2008 
    </conf-date> 
    <conf-name>
      Bioinformatics Methods for Biomedical Complex Systems Applications (NETTAB2008) 
    </conf-name> 
    <conf-loc>
      Varenna, Italy 
    </conf-loc> 
   </conference> 
  </article-meta> 
 </front> 
 <sec> 
  <h3>Background</h3> 
  <p>Integration achieves one of the most important imperatives of systems biology, namely it reduces the dimensionality of global data needed to deliver useful information about the networks active in the system of interest. The integration of data from different sources provides an effective means to deal with this issue by reinforcing 
   <italic>
     bona fide 
   </italic> observations and reducing false negatives. Moreover, because different experimental technologies provide different insights into a system, the integration of multiple data types offers the greatest information about a particular cellular process [ 
   <xref ref-type="bibr" rid="B1">
     1 
   </xref>- 
   <xref ref-type="bibr" rid="B3">
     3 
   </xref>]. For example, gene perturbation experiments (e.g., knockouts or RNA interference) and microarrays analysis can reveal relationships between genes that may imply direct physical interactions or indirect logical interactions. Indeed, microarray experiments permit us to look at overall patterns of gene expression in order to understand the architecture of genetic regulatory networks, a global approach that could ultimately lead to complete description of the transcription-control mechanisms in a cell. In contrast, chromatin immunoprecipitation (ChIp) data can reveal direct protein-DNA interactions or cofactor associations with bound transcription factors. Combined together, these technologies can provide a much more detailed view of a transcriptional regulatory network than either alone. </p> 
  <p>Several recent methods have addressed the problem of heterogeneous data integration and network prediction by modeling the noise inherent in high-throughput genomic datasets, especially by using statistical methods, which can significantly improve specificity and sensitivity and allow the robust integration of datasets with heterogeneous properties [ 
   <xref ref-type="bibr" rid="B4">
     4 
   </xref>, 
   <xref ref-type="bibr" rid="B5">
     5 
   </xref>]. However, many of these methods recently developed to implement our ability to integrate and compare heterogeneous data, are often not easy to use and/or not freely accessible [ 
   <xref ref-type="bibr" rid="B6">
     6 
   </xref>, 
   <xref ref-type="bibr" rid="B7">
     7 
   </xref>]. Taking into consideration that the development of efficient methods that facilitate the biological interpretation of these data is crucial, in the present work we focus on efficient identification of regulatory mechanisms, and propose an approach for analysis and interpretation of gene expression data based on the integration of various types of related biological information.</p> 
  <p>The cAMP-PKA signalling pathway is an important regulator of cell fate that controls the activity of metabolic enzymes, transcription factors and cytoskeletal proteins and is strongly associated with the onset of several endocrine and non-endocrine tumors. A fundamental characteristic of cAMP is its ability to stimulate cell proliferation in many cell types while inhibiting in others. Such ability has been related to the fact that cAMP regulates the Ras/Raf/ERK pathway, whose role in cancer onset is well known (about 25% of human cancers have a Ras mutation). Indeed the cAMP pathway is able to suppress ERK signaling through its ability to target C-Raf and conversely, to activate ERK signaling through its ability to target B-Raf [ 
   <xref ref-type="bibr" rid="B8">
     8 
   </xref>]. The underlying inhibitory mechanism is reasonably well characterized and involves the uncoupling of Ras signaling to C-Raf. On the other hand, models to explain ERK activation by cAMP are incomplete and in addition to B-Raf the involvement of other proteins has been suggested [ 
   <xref ref-type="bibr" rid="B9">
     9 
   </xref>, 
   <xref ref-type="bibr" rid="B10">
     10 
   </xref>]. Many observations regarding the cAMP ability to inhibit/stimulate proliferation by interfering with ERKs have been collected in normal or immortalized cell lines. However, recognizing the important role of both pathways in the development of cancer, is relevant to a more specific analysis of their crosstalk network also in cancer cells [ 
   <xref ref-type="bibr" rid="B11">
     11 
   </xref>- 
   <xref ref-type="bibr" rid="B14">
     14 
   </xref>]. </p> 
  <p>As previously described, the Ras pathway is able to crosstalk with the cAMP-PKA pathway by some typical signal transduction mechanisms (i.e. protein-protein interaction, protein phosphorylation). Moreover, through its ability to regulate the activity of a large number of transcription factors [ 
   <xref ref-type="bibr" rid="B15">
     15 
   </xref>, 
   <xref ref-type="bibr" rid="B16">
     16 
   </xref>], the Ras pathway is able to control several transcriptional programs leading to proliferation, differentiation, metabolism, cytoskeletal reorganization and immune response. Such transcriptional programs are the result of ras-specific effectors stimulation [ 
   <xref ref-type="bibr" rid="B13">
     13 
   </xref>, 
   <xref ref-type="bibr" rid="B17">
     17 
   </xref>]. Until now more than ten distinct functional classes of proteins have been involved as effectors of the small GTPase Ras, but the best studied are Raf kinases, type I phosphoinositide (PI) 3-kinases, Ral-guanine nucleotide exchange factors (Ral-GEFs), the Rac exchange factor Tiam1, and phospholipase C [ 
   <xref ref-type="bibr" rid="B18">
     18 
   </xref>]. </p> 
  <p>Raf and phosphatidylinositol 3-kinase (PI3K) were the first two identified Ras effectors and the main focus of research investigating Ras functions [ 
   <xref ref-type="bibr" rid="B19">
     19 
   </xref>, 
   <xref ref-type="bibr" rid="B20">
     20 
   </xref>]. Raf promotes cell proliferation and differentiation through the MAP kinase (MAPK) pathway [ 
   <xref ref-type="bibr" rid="B21">
     21 
   </xref>], at the same time as PI3K generates anti-apoptotic signalling, directly or through Akt pathway activation [ 
   <xref ref-type="bibr" rid="B22">
     22 
   </xref>, 
   <xref ref-type="bibr" rid="B23">
     23 
   </xref>]. Both signalling pathways can activate two different signals distinct for their response timing. Indeed both MAPK and PI3K are able to activate phosphorylation cascades that lead, as primary effect, to post-translational modification of several substrates (membrane targets, cytosolic targets, cytoskeletal targets and nuclear targets), which rapidly activate functional processes. Early response to Ras signalling is quite fast: for instance in resting cells stimulated with mitogens, Ras-GTP level increases within 2 minutes from stimulation with serum [ 
   <xref ref-type="bibr" rid="B19">
     19 
   </xref>]. Raf-1 undergoes transient activation within 2-3 minutes, and rapidly activates the mitogen-activated protein kinase (MAPK) cascade whose most downstream component, ERK, rapidly moves into the nucleus. Here it phosphorylates nuclear proteins, notably transcription factors [ 
   <xref ref-type="bibr" rid="B24">
     24 
   </xref>, 
   <xref ref-type="bibr" rid="B25">
     25 
   </xref>] whose activity can be controlled by regulating their sub-cellular localization, expression, stability, ability to bind to other components of transcriptional complexes and to DNA, and their ability to remodel chromatin structure [ 
   <xref ref-type="bibr" rid="B26">
     26 
   </xref>]. Transcription factors are under the control of MAPK pathway, including members of the ETS family (i.e. Ets-1, Ets-2, PU-1), MADS box family (i.e. MEF2A, MEF2C, Sp1), Zinc Finger family (i.e. GATA-2 and GATA-4), bZip family (i.e. Fra-1, c-Jun, JunB, JunD, ATF-2, c-Fos and CREB), bHLH family (i.e. c-Myc, MITF), Nuclear Hormone Receptor (i.e. PR, GR and ER) as well as other transcription factors (i.e., SMAD1, STAT1) and coregulatory proteins (i.e., CBP, p300) [ 
   <xref ref-type="bibr" rid="B15">
     15 
   </xref>, 
   <xref ref-type="bibr" rid="B24">
     24 
   </xref>, 
   <xref ref-type="bibr" rid="B25">
     25 
   </xref>]. </p> 
  <p>Like ERKs, Akt and other targets of PI3K signalling can phosphorylate and activate transcription factors [ 
   <xref ref-type="bibr" rid="B27">
     27 
   </xref>]. Akt protein can control several transcription factors directly or indirectly. Direct targets are the forkhead box proteins, FOXO, and the cell cycle inhibitor, MIZ1, which are both inhibited upon AKT-mediated phosphorylation. AKT-dependent regulation of p53, nuclear factor B (NFkB), c-MYC, activator protein 1 (AP1) and beta-catenin is indirect [ 
   <xref ref-type="bibr" rid="B16">
     16 
   </xref>]. </p> 
  <p>Such an observation led us to re-analyze, by using a generalized workflow for data recovery and integration, available data from multiple global assays and several databases (genomics, transcriptomics, promoter analysis and literature). In particular we searched for information for genes encoding proteins of the downstream branch of the PKA pathway (starting from adenylyl cyclase and downstream) in tumor cell lines (NCI60 cells) as a function of mutational activation of different pathways (notably the Ras and PI3K pathway) in comparison with the corresponding normal tissues, with the aim to define better the connection between these pathways in cancer cells [ 
   <xref ref-type="bibr" rid="B28">
     28 
   </xref>]. </p> 
 </sec> 
 <sec> 
  <h3>Results and discussion</h3> 
  <p>Gene-expression profiling has been applied extensively in cancer research. As a first step to identify regulatory mechanisms underlining gene-expression profiles it is necessary to extract, filter, cross-reference and structure information from cancer-related data sets [ 
   <xref ref-type="bibr" rid="B29">
     29 
   </xref>]. The aim of this work has been the identification of cancer-specific specific gene expression signatures in genes encoding proteins involved in the cAMP-PKA pathway. In particular we wished to identify, if present, differences between primary normal tissues and cancer cells and search for correlation with the pathway mutationally activated in any given transformed cell line by integrating an accurate analysis of recovered data from several databases with the application of different statistical tests.</p> 
  <sec> 
   <h3>Transformation-dependent, transcriptional remodelling of the PKA pathway encoding genes in 60 human cancer cell lines (NCI60)</h3> 
   <p>The NCI60 cell collection includes cell lines derived from colorectal, renal, ovarian, breast, prostate, lung and central nervous system cancers, as well as leukaemias and melanomas (Table 
    <xref ref-type="table" rid="T1">
      1 
    </xref>), that are most commonly used in cancer research and drug screening [ 
    <xref ref-type="bibr" rid="B30">
      30 
    </xref>, 
    <xref ref-type="bibr" rid="B31">
      31 
    </xref>]. A good correlation between transcriptional profiles of the cell lines and their tumor cancer of origin [ 
    <xref ref-type="bibr" rid="B31">
      31 
    </xref>, 
    <xref ref-type="bibr" rid="B32">
      32 
    </xref>] has been found for 51 out of 59 cell lines. NCI60 transcriptional profiles are available in public databases. </p> 
   <table-wrap id="T1" position="float"> 
    <label>Table 1</label> 
    <p>The cancer cell lines in the NCI60 collection sorted by tissue of origin</p> 
    <table frame="hsides" rules="groups"> 
     <thead> 
      <tr> 
       <td align="left"> 
        <bold>
          Tumor Type 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Cell lines 
        </bold> </td> 
      </tr> 
     </thead> 
     <tbody> 
      <tr> 
       <td align="left">Breast</td> 
       <td align="left">HS578T, MDA-MB231, MD-MB435, MCF7, T47, MDA-N, BT549</td> 
      </tr> 
      <tr> 
       <td align="left">CNS</td> 
       <td align="left">SF295, SF359, SNB19, U251, SF268, SNB75</td> 
      </tr> 
      <tr> 
       <td align="left">Colon</td> 
       <td align="left">HCC2998, HCT116, HCT15, SW620, COLO205, HT29, KM12</td> 
      </tr> 
      <tr> 
       <td align="left">Leukaemia</td> 
       <td align="left">CCRF-CEM, RPMI-8226, HL60, MOLT4, K562, SR</td> 
      </tr> 
      <tr> 
       <td align="left">Melanoma</td> 
       <td align="left">SK-MEL2, LOXIMVI, M14, MALME-3M, SK-MEL28, SK-MEL5, UACC257, UACC62</td> 
      </tr> 
      <tr> 
       <td align="left">Lung</td> 
       <td align="left">A549, HOP62, NCI-H23, NCI-H460, EKVX, NCI-H226, NCI-H322, NCI-H522, HOP92</td> 
      </tr> 
      <tr> 
       <td align="left">Ovarian</td> 
       <td align="left">OVCAR5, OVCAR8, SKOW3, IGROV1, OVCAR3, OVCAR4</td> 
      </tr> 
      <tr> 
       <td align="left">Prostate</td> 
       <td align="left">PC3, DU145</td> 
      </tr> 
      <tr> 
       <td align="left">Renal</td> 
       <td align="left">786-0, RXF-393, A498, ACHN, CAKI-1, SNC12C, TK10, U031 </td> 
      </tr> 
      <tr> 
       <td align="left">Unknown</td> 
       <td align="left">ADR-RES</td> 
      </tr> 
     </tbody> 
    </table> 
   </table-wrap> 
   <p>Since the stabilized cell lines within the NCI60 collection represent a physiological model to study gene profiles in cancer cells, with features strongly similar to cancer tissues, we reviewed information present in public databases about the 60 cell lines and 21 normal tissues, in order to identify transformation-dependent transcriptional signatures for PKA pathway-encoding genes (Table 
    <xref ref-type="table" rid="T2">
      2 
    </xref>).</p> 
   <table-wrap id="T2" position="float"> 
    <label>Table 2</label> 
    <p>Gene expression profiling datasets of NCI60 cell lines and normal tissues analyzed in this study</p> 
    <table frame="hsides" rules="groups"> 
     <thead> 
      <tr> 
       <td align="left"> 
        <bold>
          Reference 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Tissue of origin 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Number of transcriptional profiles 
        </bold> </td> 
       <td align="left"> 
        <bold>
          GEO Number 
        </bold> </td> 
      </tr> 
     </thead> 
     <tbody> 
      <tr> 
       <td align="left">[ 
        <xref ref-type="bibr" rid="B31">
          31 
        </xref>]</td> 
       <td align="left">NCI60 cells</td> 
       <td align="left">60</td> 
       <td align="left">GSE5949</td> 
      </tr> 
      <tr> 
       <td align="left"></td> 
       <td align="left">Breast</td> 
       <td align="left">0</td> 
       <td align="left">-</td> 
      </tr> 
      <tr> 
       <td align="left">[ 
        <xref ref-type="bibr" rid="B106">
          106 
        </xref>]</td> 
       <td align="left">CNS</td> 
       <td align="left">2</td> 
       <td align="left">GSE96</td> 
      </tr> 
      <tr> 
       <td align="left">[ 
        <xref ref-type="bibr" rid="B107">
          107 
        </xref>]</td> 
       <td align="left">Colon</td> 
       <td align="left">4</td> 
       <td align="left">GSE6731</td> 
      </tr> 
      <tr> 
       <td align="left">[ 
        <xref ref-type="bibr" rid="B108">
          108 
        </xref>]</td> 
       <td align="left">Blood</td> 
       <td align="left">4</td> 
       <td align="left">GSE1402</td> 
      </tr> 
      <tr> 
       <td align="left">[ 
        <xref ref-type="bibr" rid="B106">
          106 
        </xref>]</td> 
       <td align="left">Lung</td> 
       <td align="left">2</td> 
       <td align="left">GSE96</td> 
      </tr> 
      <tr> 
       <td align="left"></td> 
       <td align="left">Skin</td> 
       <td align="left">0</td> 
       <td align="left">-</td> 
      </tr> 
      <tr> 
       <td align="left">[ 
        <xref ref-type="bibr" rid="B106">
          106 
        </xref>]</td> 
       <td align="left">Ovary</td> 
       <td align="left">3</td> 
       <td align="left">GSE96</td> 
      </tr> 
      <tr> 
       <td align="left">[ 
        <xref ref-type="bibr" rid="B106">
          106 
        </xref>]</td> 
       <td align="left">Prostate</td> 
       <td align="left">3</td> 
       <td align="left">GSE96</td> 
      </tr> 
      <tr> 
       <td align="left">[ 
        <xref ref-type="bibr" rid="B106">
          106 
        </xref>]</td> 
       <td align="left">Kidney</td> 
       <td align="left">3</td> 
       <td align="left">GSE96</td> 
      </tr> 
     </tbody> 
    </table> 
    <table-wrap-foot> 
     <p>Gene expression profiles retrieved from the GEO Database. Dataset A (60 profiles) is made up of the NCI60 cell lines [ 
      <xref ref-type="bibr" rid="B31">
        31 
      </xref>]. Dataset B (13 profiles) is a subset of transcriptional profiles of a diverse array of tissues, organs, and cell lines from a normal human physiological state [ 
      <xref ref-type="bibr" rid="B106">
        106 
      </xref>]. Dataset C (4 profiles) encompasses the normal human adult samples derived from colonoscopic biopsy present in a database comprising samples of patients with Crohn's disease or ulcerative colitis [ 
      <xref ref-type="bibr" rid="B107">
        107 
      </xref>]. Dataset D (4 profiles) contains normal control samples present in a database containing transcriptional profiles of peripheral blood mononuclear cells (PBMC) obtained by juvenile arthritis patients and healthy controls [ 
      <xref ref-type="bibr" rid="B108">
        108 
      </xref>].</p> 
    </table-wrap-foot> 
   </table-wrap> 
   <p>We identified and gathered the transcriptional profile for 41 genes encoding proteins involved in the PKA pathway (adenylyl cyclases -ADCY-, phosphodiesterases -PDE-, A-kinase anchor proteins -AKAP-, cAMP-dependent transcriptional factors -TF-, PKA catalytic subunits -PRKAC- and PKA regulatory subunits -PRKACR-, Table 
    <xref ref-type="table" rid="T3">
      3 
    </xref>) and compared expression profiles of cancer cell lines with those of primary normal tissues, collected from different datasets (Table 
    <xref ref-type="table" rid="T3">
      3 
    </xref>). To identify differences between normal and cancer samples, we performed an ANOVA analysis on the entire data set. As shown in Figure 
    <xref ref-type="fig" rid="F1">
      1 
    </xref>, distributions of expression values of genes encoding proteins of the cAMP/PKA pathway were statistically different between normal and transformed cells (p-value &lt;0.0001), indicating that in transformed cells the PKA pathway-related genes are differentially expressed as compared to normal cells. Namely, the box plot indicates that, overall, the distribution of expression of values of transformed cells is shifted towards lower expression values. Dispersion of the distribution in transformed cells is much reduced compared to that observed in normal tissues, as if transformation events superimpose a negative regulation that largely abrogates tissue-specific regulation (i.e., the major factor responsible for dispersion of expression in normal tissues, see next paragraph). </p> 
   <table-wrap id="T3" position="float"> 
    <label>Table 3</label> 
    <p>PKA related genes identified in all the datasets shown in Table 
     <xref ref-type="table" rid="T2">
       2 
     </xref> and used in this study</p> 
    <table frame="hsides" rules="groups"> 
     <thead> 
      <tr> 
       <td align="left"> 
        <bold>
          Probeset 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Unigene 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Symbol 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Description 
        </bold> </td> 
      </tr> 
     </thead> 
     <tbody> 
      <tr> 
       <td align="left">33353_at</td> 
       <td align="left">Hs.192215</td> 
       <td align="left">ADCY1</td> 
       <td align="left">Adenylate cyclase 1 (brain)</td> 
      </tr> 
      <tr> 
       <td align="left">34686_at</td> 
       <td align="left">Hs.481545</td> 
       <td align="left">ADCY2</td> 
       <td align="left">Adenylate cyclase 2 (brain)</td> 
      </tr> 
      <tr> 
       <td align="left">33134_at</td> 
       <td align="left">Hs.467898</td> 
       <td align="left">ADCY3</td> 
       <td align="left">Adenylate cyclase 3</td> 
      </tr> 
      <tr> 
       <td align="left">39383_at</td> 
       <td align="left">Hs.525401</td> 
       <td align="left">ADCY6</td> 
       <td align="left">Adenylate cyclase 6</td> 
      </tr> 
      <tr> 
       <td align="left">40585_at</td> 
       <td align="left">Hs.513578</td> 
       <td align="left">ADCY7</td> 
       <td align="left">Adenylate cyclase 7</td> 
      </tr> 
      <tr> 
       <td align="left">36246_at</td> 
       <td align="left">Hs.414631</td> 
       <td align="left">ADCY8</td> 
       <td align="left">Adenylate cyclase 8 (brain)</td> 
      </tr> 
      <tr> 
       <td align="left">33800_at</td> 
       <td align="left">Hs.391860</td> 
       <td align="left">ADCY9</td> 
       <td align="left">Adenylate cyclase 9</td> 
      </tr> 
      <tr> 
       <td align="left">37698_at</td> 
       <td align="left">Hs.463506</td> 
       <td align="left">AKAP1</td> 
       <td align="left">A kinase (PRKA) anchor protein 1</td> 
      </tr> 
      <tr> 
       <td align="left">36633_at</td> 
       <td align="left">Hs.462457</td> 
       <td align="left">AKAP10</td> 
       <td align="left">A kinase (PRKA) anchor protein 10</td> 
      </tr> 
      <tr> 
       <td align="left">34657_at</td> 
       <td align="left">Hs.105105</td> 
       <td align="left">AKAP11</td> 
       <td align="left">A kinase (PRKA) anchor protein 11</td> 
      </tr> 
      <tr> 
       <td align="left">37680_at</td> 
       <td align="left">Hs.371240</td> 
       <td align="left">AKAP12</td> 
       <td align="left">A kinase (PRKA) anchor protein (gravin) 12</td> 
      </tr> 
      <tr> 
       <td align="left">554_at</td> 
       <td align="left">Hs.459211</td> 
       <td align="left">AKAP13</td> 
       <td align="left">A kinase (PRKA) anchor protein 13</td> 
      </tr> 
      <tr> 
       <td align="left">41075_at</td> 
       <td align="left">Hs.98397</td> 
       <td align="left">AKAP3</td> 
       <td align="left">A kinase (PRKA) anchor protein 3</td> 
      </tr> 
      <tr> 
       <td align="left">37087_at</td> 
       <td align="left">Hs.97633</td> 
       <td align="left">AKAP4</td> 
       <td align="left">A kinase (PRKA) anchor protein 4</td> 
      </tr> 
      <tr> 
       <td align="left">32421_at</td> 
       <td align="left">Hs.532489</td> 
       <td align="left">AKAP5</td> 
       <td align="left">A kinase (PRKA) anchor protein 5</td> 
      </tr> 
      <tr> 
       <td align="left">40747_at</td> 
       <td align="left">Hs.509083</td> 
       <td align="left">AKAP6</td> 
       <td align="left">A kinase (PRKA) anchor protein 6</td> 
      </tr> 
      <tr> 
       <td align="left">41703_r_at</td> 
       <td align="left">Hs.486483</td> 
       <td align="left">AKAP7</td> 
       <td align="left">A kinase (PRKA) anchor protein 7</td> 
      </tr> 
      <tr> 
       <td align="left">35138_at</td> 
       <td align="left">Hs.199029</td> 
       <td align="left">AKAP8</td> 
       <td align="left">A kinase (PRKA) anchor protein 8</td> 
      </tr> 
      <tr> 
       <td align="left">37886_at</td> 
       <td align="left">Hs.399800</td> 
       <td align="left">AKAP8L</td> 
       <td align="left">A kinase (PRKA) anchor protein 8-like</td> 
      </tr> 
      <tr> 
       <td align="left">36506_at</td> 
       <td align="left">Hs.527348</td> 
       <td align="left">AKAP9</td> 
       <td align="left">A kinase (PRKA) anchor protein (yotiao) 9</td> 
      </tr> 
      <tr> 
       <td align="left">36297_at</td> 
       <td align="left">Hs.435267</td> 
       <td align="left">ATF1</td> 
       <td align="left">Activating transcription factor 1</td> 
      </tr> 
      <tr> 
       <td align="left">37535_at</td> 
       <td align="left">Hs.516646</td> 
       <td align="left">CREB1</td> 
       <td align="left">cAMP responsive element binding protein 1</td> 
      </tr> 
      <tr> 
       <td align="left">32066_g_at</td> 
       <td align="left">Hs.200250</td> 
       <td align="left">CREM</td> 
       <td align="left">cAMP responsive element modulator</td> 
      </tr> 
      <tr> 
       <td align="left">35522_at</td> 
       <td align="left">Hs.487129</td> 
       <td align="left">PDE10A</td> 
       <td align="left">Phosphodiesterase 10A</td> 
      </tr> 
      <tr> 
       <td align="left">36311_at</td> 
       <td align="left">Hs.416061</td> 
       <td align="left">PDE1A</td> 
       <td align="left">Phosphodiesterase 1A, calmodulin-dependent</td> 
      </tr> 
      <tr> 
       <td align="left">38921_at</td> 
       <td align="left">Hs.530871</td> 
       <td align="left">PDE1B</td> 
       <td align="left">Phosphodiesterase 1B, calmodulin-dependent</td> 
      </tr> 
      <tr> 
       <td align="left">32418_at</td> 
       <td align="left">Hs.487897</td> 
       <td align="left">PDE1C</td> 
       <td align="left">Phosphodiesterase 1C, calmodulin-dependent </td> 
      </tr> 
      <tr> 
       <td align="left">666_at</td> 
       <td align="left">Hs.89901</td> 
       <td align="left">PDE4A</td> 
       <td align="left">Phosphodiesterase 4A, cAMP-specific</td> 
      </tr> 
      <tr> 
       <td align="left">33705_at</td> 
       <td align="left">Hs.198072</td> 
       <td align="left">PDE4B</td> 
       <td align="left">Phosphodiesterase 4B, cAMP-specific</td> 
      </tr> 
      <tr> 
       <td align="left">38860_at</td> 
       <td align="left">Hs.437211</td> 
       <td align="left">PDE4C</td> 
       <td align="left">Phosphodiesterase 4C, cAMP-specific</td> 
      </tr> 
      <tr> 
       <td align="left">38526_at</td> 
       <td align="left">Hs.117545</td> 
       <td align="left">PDE4D</td> 
       <td align="left">Phosphodiesterase 4D, cAMP-specific</td> 
      </tr> 
      <tr> 
       <td align="left">37676_at</td> 
       <td align="left">Hs.9333</td> 
       <td align="left">PDE8A</td> 
       <td align="left">Phosphodiesterase 8A</td> 
      </tr> 
      <tr> 
       <td align="left">37249_at</td> 
       <td align="left">Hs.78106</td> 
       <td align="left">PDE8B</td> 
       <td align="left">Phosphodiesterase 8B</td> 
      </tr> 
      <tr> 
       <td align="left">33709_at</td> 
       <td align="left">Hs.473927</td> 
       <td align="left">PDE9A</td> 
       <td align="left">Phosphodiesterase 9A</td> 
      </tr> 
      <tr> 
       <td align="left">438_at</td> 
       <td align="left">Hs.194350</td> 
       <td align="left">PRKACA</td> 
       <td align="left">Protein kinase, cAMP-dependent, catalytic, alpha</td> 
      </tr> 
      <tr> 
       <td align="left">36215_at</td> 
       <td align="left">Hs.487325</td> 
       <td align="left">PRKACB</td> 
       <td align="left">Protein kinase, cAMP-dependent, catalytic, beta</td> 
      </tr> 
      <tr> 
       <td align="left">36359_at</td> 
       <td align="left">Hs.158029</td> 
       <td align="left">PRKACG</td> 
       <td align="left">Protein kinase, cAMP-dependent, catalytic, gamma</td> 
      </tr> 
      <tr> 
       <td align="left">226_at</td> 
       <td align="left">Hs.280342</td> 
       <td align="left">PRKAR1A</td> 
       <td align="left">Protein kinase, cAMP-dependent, regulatory, type I, alpha</td> 
      </tr> 
      <tr> 
       <td align="left">1091_at</td> 
       <td align="left">Hs.550753</td> 
       <td align="left">PRKAR1B</td> 
       <td align="left">Protein kinase, cAMP-dependent, regulatory, type I, beta</td> 
      </tr> 
      <tr> 
       <td align="left">116_at</td> 
       <td align="left">Hs.517841</td> 
       <td align="left">PRKAR2A</td> 
       <td align="left">Protein kinase, cAMP-dependent, regulatory, type II, alpha</td> 
      </tr> 
      <tr> 
       <td align="left">37221_at</td> 
       <td align="left">Hs.433068</td> 
       <td align="left">PRKAR2B</td> 
       <td align="left">Protein kinase, cAMP-dependent, regulatory, type II, beta</td> 
      </tr> 
     </tbody> 
    </table> 
   </table-wrap> 
   <fig id="F1" position="float"> 
    <label>Figure 1</label> 
    <p> 
     <bold>
       Statistical analysis of the 41 PKA pathway-encoding genes expression in normal and transformed samples. 
     </bold> 81 transcriptional profiles from normal tissues and from the NCI60 cancer cell line collection, were recovered from the GEO database. After normalization (see Methods), the expression values of 41 PKA pathway-encoding genes were used to perform an ANOVA analysis (p-value 0.0001) to evaluate the statistical significance of the differences between normal and transformed samples. IQR: Interquartile Range. Outliers are also shown.</p> 
    <graphic xlink:href="1471-2105-10-S12-S1-1"></graphic> 
   </fig> 
   <p>The same data-set was then analyzed through unsupervised hierarchical clustering (as implemented in the GeneSpring platform) that organizes genes according to the similarity or dissimilarity in expression profile, placing the cases with similar expression profiles together as neighbouring columns in the dendrogram (Figure 
    <xref ref-type="fig" rid="F2">
      2 
    </xref>). Six different classes corresponding to the main arms of the dendrogram derived from clustering according to Tissue and cell lines (classes 
    <bold>
      I 
    </bold> to 
    <bold>
      III 
    </bold> correspond to the left main branch of dendrogram, 
    <bold>
      IV 
    </bold> to 
    <bold>
      VI 
    </bold> to the right branch) were identified. Each cell line is color-coded at the bottom according to its condition (i.e., normal, blue, or transformed, red) or the tissue of origin. Notably, classes 
    <bold>
      II 
    </bold> and 
    <bold>
      V 
    </bold> contain only transformed cells, while only one transformed cell line clusters in class 
    <bold>
      VI 
    </bold>. In most cases clustering effectively separates normal and transformed cell lines of the same histological origin: for instance, normal and transformed cell lines derived from kidney cluster to class 
    <bold>
      I 
    </bold> and 
    <bold>
      III 
    </bold>, hemopoietic normal and transformed cell lines to 
    <bold>
      IV 
    </bold> and 
    <bold>
      II 
    </bold>, colon cancer cells are in class 
    <bold>
      II 
    </bold> while normal colon in class 
    <bold>
      IV 
    </bold>, respectively (Table 
    <xref ref-type="table" rid="T4">
      4 
    </xref>). Class 
    <bold>
      I 
    </bold> and 
    <bold>
      IV 
    </bold> contained cancer lines of several histological origin, while class 
    <bold>
      II 
    </bold> was enriched for cancer cells from colon and blood, class 
    <bold>
      III 
    </bold> for ovary and kidney and class 
    <bold>
      V 
    </bold> for lung, respectively (Table 
    <xref ref-type="table" rid="T4">
      4 
    </xref>). These results indicate that regulation of the PKA pathway is tissue-dependent, in keeping with the pleiotropic and tissue-specific phenotypes regulated by intracellular cAMP. They also suggest that transformation transcriptionally remodulates the PKA pathway, so that in most cases expression profiling of genes encoding proteins of the cAMP-PKA pathway is quite different in cancer cells as compared to their normal counterparts. Interestingly in class 
    <bold>
      IV 
    </bold>, which comprises all the colon and hemopoietic normal samples, we observe strong expression of few genes (AKAP9-11; PDE4D; PRKCB and PRKAR2B; CREB1- colon sample- and AKAP9-11; PDE4B and PDE8A; PRKCB and PRKAR1A; CREB1 and CREM- hemopoietic sample-) as compared to their transformed counterparts, in which the same genes appeared poorly expressed (class 
    <bold>
      II 
    </bold>). In human colon carcinoma cells it has been reported that PRKAR2B overexpression suppresses neoplastic cell growth [ 
    <xref ref-type="bibr" rid="B33">
      33 
    </xref>], consistently with the notion that abnormal expression of isoforms of PKA regulatory subunits may be involved in neoplastic transformation. Moreover in several models of hemopoietic malignancies, it has been shown that induction of cAMP/PKA pathway stimulates leukemia cell differentiation (event associated to the relapse of the disease) or lymphoma cells apoptosis [ 
    <xref ref-type="bibr" rid="B34">
      34 
    </xref>, 
    <xref ref-type="bibr" rid="B35">
      35 
    </xref>].</p> 
   <fig id="F2" position="float"> 
    <label>Figure 2</label> 
    <p> 
     <bold>
       Hierarchical clustering of the 41 PKA pathway-encoding genes analyzed in this paper. 
     </bold> Two-way (gene, column and cell line, row) hierarchical clustering (see Methods) of the same profiles analyzed in Figure 
     <xref ref-type="fig" rid="F1">
       1 
     </xref>. Normalized expression is colour-coded from green (poor expression) to red (strong expression). The name of each gene is colour-coded according to family to which it belongs.</p> 
    <p>The 6 main classes described in the text (red lines on the top of the dendrogram and roman number bottom of the dendrogram) are shown. The distance function is based on Pearson correlation and complete linkage clustering. Legends for expression, condition, gene family and tissue of origin are shown on the right of the dendrogram.</p> 
    <graphic xlink:href="1471-2105-10-S12-S1-2"></graphic> 
   </fig> 
   <table-wrap id="T4" position="float"> 
    <label>Table 4</label> 
    <p>Correlation between PKA related gene patterns and tissues</p> 
    <table frame="hsides" rules="groups"> 
     <thead> 
      <tr> 
       <td align="left"> 
        <bold>
          Class 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Normal 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Transformed 
        </bold> </td> 
      </tr> 
     </thead> 
     <tbody> 
      <tr> 
       <td align="left">I</td> 
       <td align="left"> 
        <break></break> 
        <break></break> 
        <break></break>Kidney 3/3 
        <break></break> 
        <break></break> </td> 
       <td align="left">Breast 2/8 
        <break></break>Lung 1/9 
        <break></break>CNS 1/6 
        <break></break>Kidney 2/8 
        <break></break>Prostate 1/2 
        <break></break>Ovary 2/6</td> 
      </tr> 
      <tr> 
       <td colspan="3"> 
        <hr /> </td> 
      </tr> 
      <tr> 
       <td align="left">II</td> 
       <td align="left"></td> 
       <td align="left">Colon 6/7 
        <break></break>Prostate 1/2 
        <break></break>Lung 1/9 
        <break></break>Hemopoietic 6/7</td> 
      </tr> 
      <tr> 
       <td colspan="3"> 
        <hr /> </td> 
      </tr> 
      <tr> 
       <td align="left">III</td> 
       <td align="left">Ovary 3/3 
        <break></break> 
        <break></break> </td> 
       <td align="left">Ovary 4/6 
        <break></break>Breast 1/8 
        <break></break>Kidney 5/8</td> 
      </tr> 
      <tr> 
       <td colspan="3"> 
        <hr /> </td> 
      </tr> 
      <tr> 
       <td align="left">IV</td> 
       <td align="left"> 
        <break></break> 
        <break></break> 
        <break></break>Colon 4/4 
        <break></break> 
        <break></break> 
        <break></break> Hemopoietic 4/4</td> 
       <td align="left">Breast 2/8 
        <break></break>Kidney 1/8 
        <break></break>CNS 4/6 
        <break></break> 
        <break></break>Lung 2/9 
        <break></break>Skin 1/8 
        <break></break> </td> 
      </tr> 
      <tr> 
       <td colspan="3"> 
        <hr /> </td> 
      </tr> 
      <tr> 
       <td align="left">V</td> 
       <td align="left"></td> 
       <td align="left">Breast 3/8 
        <break></break>CNS 1/6 
        <break></break>Colon 1/7 
        <break></break>Skin 7/8 
        <break></break>Lung 4/9</td> 
      </tr> 
      <tr> 
       <td colspan="3"> 
        <hr /> </td> 
      </tr> 
      <tr> 
       <td align="left">VI</td> 
       <td align="left">CNS 2/2 
        <break></break> Prostate 
        <break></break>3/3 Lung 2/2</td> 
       <td align="left"> 
        <break></break> 
        <break></break>Lung 1/9</td> 
      </tr> 
     </tbody> 
    </table> 
    <table-wrap-foot> 
     <p>Six different classes, corresponding to the main arms of the dendrogram derived from clustering according to "Tissue and cell line", were identified. The number on the right of each tissue represents the number of samples belonging to a class as compared to the total sample analyzed.</p> 
    </table-wrap-foot> 
   </table-wrap> 
  </sec> 
  <sec> 
   <h3>Analysis of mutational status of the NCI60 cell lines and correlation with tissue-specific PKA pathway gene regulation</h3> 
   <p>In the previous paragraph we have shown that a different and a tissue-specific pattern of expression of the PKA pathway encoding genes between normal and transformed samples does exist. Moreover, we observed that a similar pattern is common to different tissues, both in normal and transformed samples. While in normal tissues such a finding may be justified by a common histological origin or by the PKA pathway regulating a common intracellular process (i.e. differentiation, metabolism), in transformed samples, in which the correct regulation of the PKA pathway is lost, such similar gene regulation can suggest a transformation or a mutation-dependent gene regulation. </p> 
   <p>For this aim, we determined the mutation status of the NCI-60 panel of human cancer cell lines, identified the pathway in which such mutations were involved and correlated the mutation status and pathway altered in the transformed cells with transcriptional profiling data. The 60 cell lines were sorted according to mutational status, using the information provided by Catalogue Of Somatic Mutations In Cancer ( 
    <ext-link ext-link-type="uri" xlink:href="http://www.sanger.ac.uk/genetics/CGP/cosmic/"></ext-link>), and divided into four groups based on the carried mutation as follows (Table 
    <xref ref-type="table" rid="T5">
      5 
    </xref>):</p> 
   <p>1) Cell lines carrying mutations able to interfere with the Ras pathway (i.e., mutations in genes encoding Ras, B-Raf, ERBB2, PDGFRA, referred to as Ras), 29 cell lines;</p> 
   <p>2) Cell lines carrying mutations able to interfere with PI3K-Akt pathway (i.e., mutations in genes encoding PI3KCA, PTEN and Lkb1, referred to as PI3K), 13 cell lines.</p> 
   <p>3) Cell lines carrying no somatic mutations interfering with the two above pathways (i.e., mutations in genes encoding CDKN2A, p53, referred to as Other Mutation), 14 cell lines;</p> 
   <p>4) Cell lines for which the presence of somatic mutations interfering with the above pathways has not been searched, referred to as Not Tested), 4 cell lines.</p> 
   <table-wrap id="T5" position="float"> 
    <label>Table 5</label> 
    <p>NCI60 cell lines with predicted active pathways by mutational analysis</p> 
    <table frame="hsides" rules="groups"> 
     <thead> 
      <tr> 
       <td align="left"> 
        <bold>
          Tumor Type 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Ras 
        </bold> </td> 
       <td align="left"> 
        <bold>
          PI3K 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Other Mutation 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Not Tested 
        </bold> </td> 
      </tr> 
     </thead> 
     <tbody> 
      <tr> 
       <td align="left"> 
        <bold>
          Breast 
        </bold> </td> 
       <td align="left">HS578T, MDA-MB231, MD-MB435</td> 
       <td align="left">MCF7, T47,</td> 
       <td align="left">-</td> 
       <td align="left">MDA-N, BT549</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          CNS 
        </bold> </td> 
       <td align="left">-</td> 
       <td align="left">SF295, SF359, SNB19, U251</td> 
       <td align="left">SF268, SNB75</td> 
       <td align="left">-</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          Colon 
        </bold> </td> 
       <td align="left">HCC2998, HCT116, HCT15, SW620, COLO205, HT29</td> 
       <td align="left">KM12</td> 
       <td align="left">-</td> 
       <td align="left">-</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          Leukaemia 
        </bold> </td> 
       <td align="left">CCRF-CEM, RPMI-8226, HL60, MOLT4, K562</td> 
       <td align="left">-</td> 
       <td align="left">-</td> 
       <td align="left">SR</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          Melanoma 
        </bold> </td> 
       <td align="left">SK-MEL2, LOXIMVI, M14, MALME-3M, SK-MEL28, SK-MEL5, UACC257, UACC62</td> 
       <td align="left">-</td> 
       <td align="left">-</td> 
       <td align="left">-</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          Lung 
        </bold> </td> 
       <td align="left">A549, HOP62, NCI-H23, NCI-H460</td> 
       <td align="left">-</td> 
       <td align="left">EKVX, NCI-H226, NCI-H322, NCI-H522</td> 
       <td align="left">HOP92</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          Ovarian 
        </bold> </td> 
       <td align="left">OVCAR5, OVCAR8</td> 
       <td align="left">SKOW3, IGROV1</td> 
       <td align="left">OVCAR3, OVCAR4</td> 
       <td align="left">-</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          Prostate 
        </bold> </td> 
       <td align="left">-</td> 
       <td align="left">PC3, DU145</td> 
       <td align="left">-</td> 
       <td align="left">-</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          Renal 
        </bold> </td> 
       <td align="left">-</td> 
       <td align="left">786-0, RXF-393</td> 
       <td align="left">A498, ACHN, CAKI-1, SNC12C, TK10, U031</td> 
       <td align="left">-</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          Unknown 
        </bold> </td> 
       <td align="left">ADR-RES</td> 
       <td align="left">-</td> 
       <td align="left">-</td> 
       <td align="left">-</td> 
      </tr> 
     </tbody> 
    </table> 
    <table-wrap-foot> 
     <p>The 60 cell lines reorganized in 4 categories, as described in the text, on the basis of the most representative mutation of each cell line: </p> 
     <p>Cell lines carrying mutations able to interfere with Ras-Raf-MAPK pathway (i.e., mutations in genes encoding Ras, B-Raf, ERBB2, PDGFRA, referred to as Ras);</p> 
     <p>Cell lines carrying mutations able to interfere with PI3K-Akt pathway (i.e., mutations in genes encoding PI3KCA, PTEN and Lkb1, referred to as PI3K);</p> 
     <p>Cell lines carrying no somatic mutations interfering with the two above pathways (i.e., mutations in genes encoding for CDKN2A, p53, referred to as Other Mutation);</p> 
     <p>Cell lines for which the presence of somatic mutations interfering with the two above pathways has not been searched, referred to as Not Tested.</p> 
    </table-wrap-foot> 
   </table-wrap> 
   <p>To assess overall data quality and visualize relations and differences between the aforementioned transformed and normal samples, we applied dimensional reduction through principal component analysis (PCA). A three-dimensional PCA plot of all expression data (accounting for 91% of variance) is shown in Figure 
    <xref ref-type="fig" rid="F3">
      3A 
    </xref>. PC1 (x axis) effectively separates the normal group from the four groups of transformed cells. PC2 (y axis) effectively separates the Ras group from the others, while PC3 (z axis) best separates the Other Mutation group from the others. Overall, the Ras group appeared to segregate the most from the other groups.</p> 
   <fig id="F3" position="float"> 
    <label>Figure 3</label> 
    <p> 
     <bold>
       Identification of differentially regulated genes in normal and transformed samples. (A) 
     </bold> Samples were sorted in five groups according to mutational activation: green, normal; yellow, Ras; red, PI3K; blue, Other Mutation; cyan, Not Tested. Principal Component Analysis (PCA) performed on 41 PKA pathway-encoding genes for normal samples and the four classes of mutation-dependent samples. Each sphere represents the comparative averaging of the 41 genes for each pathway identified by mutational analysis. 
     <bold>
       (B) 
     </bold> For each of the 5 groups described in 
     <bold>
       (A) 
     </bold>, the 41 PKA-encoding genes were clustered, relative to their level of expression, in three subgroups: Strong (&gt;1, red), Average (=1, black) and Low (&lt;1, green). 
     <bold>
       (C) 
     </bold> Gene list according to expression level and mutational group of the three subgroups previously indicated, divided for each sample. Color-coding is as follows: blue, common between normal and at least one transformed sample; yellow, specific for normal samples; grey, specific for transformed samples. Percentage of regulated genes for each subgroup is shown at the bottom. 
     <bold>
       (D) 
     </bold> ANOVA analysis to evaluate the statistical significance of the differences between the five classes of samples described in 
     <bold>
       (A) 
     </bold>. The right inset shows p-value of the pair-wise comparisons. Statistically significant differences are indicated in red. IQR: Interquartile Range. Outliers are also shown.</p> 
    <graphic xlink:href="1471-2105-10-S12-S1-3"></graphic> 
   </fig> 
   <p>In Figure 
    <xref ref-type="fig" rid="F3">
      3B 
    </xref>, the 41 genes encoding proteins involved in the PKA pathway were sorted according to their relative level of expression and color-coded in the graph according to expression: strong (red, value &gt;1), average (black, value=1) and low (green, level &lt;1). These three series were crossed with the groups described above, namely Normal, Ras, PI3K, Other Mutation and Not Tested. In Normal tissues, expression of 83% of the genes was classified as Strong, a value 2-3 fold higher than those observed in the different transformed groups (27-41%). Overall, in the transformed groups, expression of most PKA pathway-encoding genes was classified as Average or Low, with the exception of the Ras group, in which only one gene was scored as low.</p> 
   <p>Expression of PKA pathway-encoding genes was further classified as follows (Figure 
    <xref ref-type="fig" rid="F3">
      3C 
    </xref>): genes with similar level of expression between normal and at least one transformed group (blue color), genes whose expression level is different between the normal and transformed groups (yellow color) and genes with similar expression level among the different transformed groups (grey color). Such a classification allowed us to pinpoint genes, such as ADCY2 and AKAP13 whose expression is strong only in the Normal group. More interestingly, expression of a few genes, such as ADCY3 and AKAP8 was strong only in members of the transformed groups, despite overall reduction in expression of the PKA pathway-encoding genes observed in transformed samples. </p> 
   <p>These results were further confirmed by pair-wise ANOVA analyses (Figure 
    <xref ref-type="fig" rid="F3">
      3D 
    </xref>), in which the distribution of expression values of genes encoding proteins of the cAMP/PKA pathway were found to be statistically different between normal and each group of transformed cells (p-value between 0.0001 and 0.0003). Notably, the difference in distribution between the Ras group and the PI3K and Other Mutation groups was also statistically significant, unlike the difference with the Not Tested group. This suggests that cells in this group may be biased for mutations within genes encoding proteins of the Ras pathway. </p> 
   <p>To reveal gene expression changes relate to mutation status of the 60 cell lines, and better interpret the results of PCA and ANOVA, a hierarchical clustering was performed. The resulting dendrogram is shown in Figure 
    <xref ref-type="fig" rid="F4">
      4 
    </xref>, in which each cell line is color-coded at the bottom according to its tissue of origin -row labeled tissue-, mutated gene -row labeled mutation-, inferred pathway activated by mutation -row labeled pathway-. A robust association between the transcriptional profiles and mutations in the Ras pathway was observed (indicated as Ras, red color). Two cell lines of the Not Tested group were interdispersed within the Ras group, indicating that these two lines are most likely responsible for the lack of statistical difference between the Ras and the Not Tested group (see above). Comparison of the Tissue and Pathway categories indicated that within the two Ras sub-clusters, some tissue-specificity is conserved. Indeed, the left cluster, comprising a total of 18 cell lines, was characterized by 6 colon cancers and 6 leukemias of which 5 on 6 were mutated in Ras pathway. Similarly the right cluster, comprising a total of 19 cell lines, was characterized by 8 melanomas and 5 lung cancers of which 7 on 8 were mutated in Ras pathway for melanoma and 4 on 5 for lung cancer. The other sub-clusters, comprising all the remaining cell lines and the other three groups of mutations and consequently of pathways, were more dispersed along the clustergram. Together, these results indicate that transformation events modulate transcriptional regulation of genes encoding proteins of the PKA pathway and that mutational activation of the Ras pathway originates a distinguishable signature, in comparison with mutational activation of the other genes studied in this report. Such a distinguishable signature is particularly noticeable in melanoma cells, in which strong expression of a gene set encoding a complete functional PKA pathway module (ADCY3; PDE4B, PDE4D and PDE8A; AKAP12; PRKAR1A and PRKAR2B; PRKACB; CREM) is observed, suggesting a deregulated cAMP signaling. Moreover, analysis of expression values for PRKAR1A and PRKAR2B genes indicated the presence in melanoma cells of a high R1/R2 ratio, that has been associated to melanocyte proliferation [ 
    <xref ref-type="bibr" rid="B36">
      36 
    </xref>]. </p> 
   <fig id="F4" position="float"> 
    <label>Figure 4</label> 
    <p> 
     <bold>
       Hierarchical clustering of the 41 PKA pathway-encoding genes in transformed samples. 
     </bold> Two-way (gene, column and cell line, row) hierarchical clustering (see Methods) of the profiles from the NCI60 collection only. Normalized expression is colour-coded from green (poor expression) to red (strong expression). The distance function is based on Pearson correlation and complete linkage clustering. The name of each gene is colour-coded according to the family to which it belongs. Legends for expression, condition, gene family and tissue of origin are shown on the right of the dendrogram. The data have been organized on the basis of the tissue of origin of the cancer (Tissue), the specific oncogenic mutations identified in each cell line (Mutation), the putative altered pathway by the specific mutations (Pathway) and the gene family.</p> 
    <graphic xlink:href="1471-2105-10-S12-S1-4"></graphic> 
   </fig> 
  </sec> 
  <sec> 
   <h3>Promoter analysis: finding correlation between oncogenic pathway, transcriptional profiles and promoter regulation</h3> 
   <p>Genes involved in the same pathway or transcriptionally co-regulated are likely to share similar promoter features. To test this hypothesis in our model, the 15 groups previously established (see Figure 
    <xref ref-type="fig" rid="F3">
      3 
    </xref>), containing coregulated genes for each group, were used for promoter identification and analysis. Using a series of biocomputing procedures and statistical processes (see Methods and the Figure), we identified Transcription Factor Binding Sites (TFBSs) conserved within the promoters (operationally defined as regions spanning 500 nt upstream and 100 nt downstream from the transcription start site) of the 41 PKA pathway-encoding genes. Genes were sorted in the 15 groups indicated in Figure 
    <xref ref-type="fig" rid="F3">
      3B 
    </xref> and 
    <xref ref-type="fig" rid="F3">
      3C 
    </xref>, and each group separately analyzed. In this first analysis (Fig. 
    <xref ref-type="fig" rid="F5">
      5A 
    </xref>), each TFBS was scored as either absent or present, regardless of the number of copies present within a given promoter. This analysis permitted the identification of 30 TFBSs enriched in the promoters of the 41 PKA pathway-encoding genes whose frequency of occurrence i.e., the ratio between the promoters that contained the specified motif (S) and the 41 promoters in our collection (T) was compared with the frequency of occurrence within vertebrate genomes (computed using the promoter Library Matrix Family of vertebrates that comprises 260.000 vertebrate promoters). Statistical analysis indicated that of these 30 TFBSs, 7 were over-represented (red color) and 9 under-represented (green color). The remaining showed the same frequency of occurrence found in the whole vertebrate genome collection. </p> 
   <fig id="F5" position="float"> 
    <label>Figure 5</label> 
    <p> 
     <bold>
       TFBS identification by using the enrichment as parameter. (A) 
     </bold> The panel shows for each TFBS, recognized as relevant (present in ? 70% of the promoters of 41 PKA pathway-encoding genes) the percentage of promoters in our collection that contain the motif as compared to Matrix Family Library on vertebrates. This percentage has been calculated by dividing the total number of promoters containing the motif (S) by the total number of promoters (T). Color-coding scheme on the right of the panel. 
     <bold>
       (B) 
     </bold> Schematic representation of the TFBSs (color-coded as shown on the right of the panel) identified in the promoters of the 15 subgroups described in the text and in Figure 
     <xref ref-type="fig" rid="F3">
       3 
     </xref>. Each cartoon represents the promoter structure resulting from the average of the TFBS identified in ? 70% of the gene promoters for each subgroup. The asterisks on the bottom of the cartoon indicate the over-represented TFBS, as scored in panel 
     <bold>
       A 
     </bold>, for all the 41 PKA pathway-encoding genes.</p> 
    <graphic xlink:href="1471-2105-10-S12-S1-5"></graphic> 
   </fig> 
   <p>A 
    <italic>
      consensus 
    </italic> representation for the promoter structure of each subgroup of coregulated genes was drawn by taking into account the 30 TFBSs present in at least 70% of the genes within each subgroup (Figure 
    <xref ref-type="fig" rid="F5">
      5B 
    </xref>). Surprisingly, the vast majority of these consensus promoters (13 out of 15) showed a common module (upper part, module), comprising 4 TFBSs: ETSF, MAZF, ZBPF and EGRF, 3 of which are over-represented in our collection (over-represented motifs are indicated by an asterisk at the bottom of the figure). This strongly suggests a functional implication of these TFBSs in expression of PKA pathway-encoding genes. Other interesting features indicated by this analysis include the identification of binding sites for PAX6 (indicated by red P) and ZF5F and NKXH (indicated by red Z and N respectively) only in 
    <italic>
      consensus 
    </italic> promoters of some genes within the normal or transformed group, respectively.</p> 
   <p>Another feature that may be critical in the identification of enriched elements is the number of copies of a given TFBS within a promoter. In fact, it has been documented that the presence of multiple copies of 
    <italic>
      cis 
    </italic>-elements in promoters, particularly when clustered, makes transcriptional activation stronger [ 
    <xref ref-type="bibr" rid="B31">
      31 
    </xref>, 
    <xref ref-type="bibr" rid="B32">
      32 
    </xref>]. For this reason, total number and frequency (number of each TFBS/promoter) of the 30 TFBSs previously identified, was scored within each of the 15 subgroups and classified by hierarchical clustering (Figure 
    <xref ref-type="fig" rid="F6">
      6A 
    </xref> and 
    <xref ref-type="fig" rid="F6">
      6B 
    </xref>, respectively). Analysis using both criteria confirmed the results reported in Figure 
    <xref ref-type="fig" rid="F5">
      5 
    </xref>: the presence in promoters of all subgroups of a TFBS module comprising ETSF, MAZF, ZBPF and EGRF. Clustering according to Regulation in Figure 
    <xref ref-type="fig" rid="F6">
      6A 
    </xref> show that all promoters of genes characterized by low expression transformed samples cluster together (class II). Promoters belonging to genes with strong expression in the Ras group cluster in a completely independent arm (lower part of the dendrogram), opposite to where cluster promoters belonging to genes with strong expression in the Normal group (class I). Additionally, clustering by frequency highlighted the specific enrichment of EKLF in genes with low expression. Clustering according to both criteria indicated that Normal samples clustered in a different way as compared to transformed samples (upper part of the dendrogram) and that the PI3K, Other Mutation and Not Tested samples were more interspersed along the dendrogram and confirmed that the Ras category showed a different promoter composition as compared to other categories, in keeping with the PCA analysis presented in Figure 
    <xref ref-type="fig" rid="F3">
      3 
    </xref>.</p> 
   <fig id="F6" position="float"> 
    <label>Figure 6</label> 
    <p> 
     <bold>
       Hierarchical clustering of TFBSs present in the promoters of the 41 PKA pathway-encoding genes, according to total number and frequency. 
     </bold> Two-way (TBFS, column and expression subgroup, see Figure 
     <xref ref-type="fig" rid="F3">
       3 
     </xref>, row) hierarchical clustering of the TFBS present within the promoters of the 41 PKA pathway-encoding genes. Clustering was run according to the total number of TBFS present in each group (panel 
     <bold>
       A 
     </bold>) or to the frequency, i.e. the total number of a given TBFS divided by the number of promoters (panel ( 
     <bold>
       B 
     </bold>). The color-coding scale is shown at the top of each panel. The distance function is based on Pearson correlation and complete linkage clustering. The two classes, corresponding to the main arms of the dendrogram, derived from clustering according to "Condition" are shown on the right of each dendrogram.</p> 
    <graphic xlink:href="1471-2105-10-S12-S1-6"></graphic> 
   </fig> 
  </sec> 
  <sec> 
   <h3>Data mining for PKA pathway-related gene promoters</h3> 
   <p>As previously described, computational analysis of our promoter collection, permitted the identification of some TFBS that are able to characterize in a specific manner normal and transformed samples. To confirm some of our computational results, we interrogated several databases and searched in the literature for studies on promoter structure of PKA pathway-encoding genes. Experimental studies, using one or more molecular approaches including EMSA, Chromatin Immunoprecipitation and transactivation assay, have been found for 16 PKA pathway-encoding genes: PRKAR1A, PRKAR1B, PRKAR2B, PRKACA, AKAP1, AKAP8, AKAP9, AKAP10, AKAP12, ADCY8, ADCY9, PDE4B, PDE4C, PDE4D, CREB and CREM. This subset of genes was re-analyzed as described above and the obtained results were compared with literature data (Table 
    <xref ref-type="table" rid="T6">
      6 
    </xref>). In total, 36 TFBSs have been experimentally identified: 20 of these (i.e. 55%) have been predicted by our computational approach and for two genes alone (AKAP9 and PRKAKA), none of the experimentally identified sites was identified by the computational approach that overall identified a much higher number of sites compared to those retrieved from literature. The biological significance of the presence of the identified TFBS and of their relationship with oncogenic mutations, notably in the Ras pathway, is proposed below. </p> 
   <table-wrap id="T6" position="float"> 
    <label>Table 6</label> 
    <p>Comparison between computational data and literature data</p> 
    <table frame="hsides" rules="groups"> 
     <thead> 
      <tr> 
       <td align="left"> 
        <bold>
          GeneName 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Computational data 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Literature data 
        </bold> </td> 
      </tr> 
     </thead> 
     <tbody> 
      <tr> 
       <td align="left"> 
        <bold>
          ADCY8 
        </bold> </td> 
       <td align="left">HOXF, ETSF, SORY </td> 
       <td align="left">CREB</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          ADCY9 
        </bold> </td> 
       <td align="left">ZF5F, SP1F, ZBPF, MAZF, EKLF, EGRF, 
        <bold>
          EBOX 
        </bold>, CDEF, WHNF, AHRR, ETSF, HESF</td> 
       <td align="left"> 
        <bold>
          c-Myc/EBOX 
        </bold> </td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          AKAP1 
        </bold> </td> 
       <td align="left">MYOD, EKLF, MAZF, EGRF, ETSF, HOXF, AP1R, 
        <bold>
          EBOX 
        </bold> </td> 
       <td align="left"> 
        <bold>
          c-Myc/EBOX 
        </bold> </td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          AKAP8 
        </bold> </td> 
       <td align="left">SP1F, EKLF, ZBPF, EBOX, ZF5F, HIFF, MAZF, EGRF, ETSF, AHRR, 
        <bold>
          CREB 
        </bold> </td> 
       <td align="left"> 
        <bold>
          CREB 
        </bold> </td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          AKAP9 
        </bold> </td> 
       <td align="left">ZBPF, EGRF, EKLF, ETSF, HOXF, SP1F, MAZF, GLIF, HOMF, NKXH, CREB</td> 
       <td align="left"> c-Myc/EBOX </td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          AKAP10 
        </bold> </td> 
       <td align="left">ETSF, SP1F, HESF, 
        <bold>
          EBOX 
        </bold>, ZBPF, MAZF, MYBL, MYOD, AP4R, EGRF, CREB, MZF1</td> 
       <td align="left"> 
        <bold>
          c-Myc/EBOX 
        </bold> </td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          AKAP12 
        </bold> </td> 
       <td align="left"> 
        <bold>
          EBOX 
        </bold>, EGRF, SORY</td> 
       <td align="left"> 
        <bold>
          c-Myc/EBOX 
        </bold>, SRF/SRFF</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          CREB1 
        </bold> </td> 
       <td align="left">SORY, 
        <bold>
          SP1F 
        </bold>, 
        <bold>
          NFKB 
        </bold>, ETSF, ZBPF, MYOD, EKLF, AP4R, MZF1, HICF</td> 
       <td align="left">Myc/EBOX, 
        <bold>
          Sp1/SP1F 
        </bold>, 
        <bold>
          NFkB/NFKB 
        </bold> </td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          CREM 
        </bold> </td> 
       <td align="left">ZBPF, EGRF, SP1F, MAZF, EKLF, ETSF, 
        <bold>
          CREB 
        </bold>, WHNF, HESF, EBOX, MYBL, HIFF, AHRR</td> 
       <td align="left"> 
        <bold>
          CREB 
        </bold> </td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          PDE4B 
        </bold> </td> 
       <td align="left">ETSF, SORY, HOXF, ZBPF, HEAT, 
        <bold>
          CREB 
        </bold> </td> 
       <td align="left"> 
        <bold>
          CREB 
        </bold> </td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          PDE4C 
        </bold> </td> 
       <td align="left">ZBPF, ETSF, 
        <bold>
          CREB 
        </bold>, GLIF, MAZF</td> 
       <td align="left">Myc/EBOX, 
        <bold>
          CREB 
        </bold> </td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          PDE4D 
        </bold> </td> 
       <td align="left">SORY, HOXF, NKXH, ETSF, 
        <bold>
          CREB 
        </bold>, GATA, MYOD</td> 
       <td align="left"> 
        <bold>
          CREB 
        </bold> </td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          PRKACA 
        </bold> </td> 
       <td align="left">EKLF, ZBPF, EGRF, MZF1, NKXH, ETSF, MAZF, SORY, HOXF</td> 
       <td align="left">USF1/EBOX, USF2/EBOX</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          PRKAR1A 
        </bold> </td> 
       <td align="left">ETSF, 
        <bold>
          CREB 
        </bold>, ZBPF, EBOX, EGRF, SORY, GATA, MYBL, MYOD, HIFF, 
        <bold>
          SP1F 
        </bold>, HESF</td> 
       <td align="left">AP1/AP1F, AP2/AP2F, 
        <bold>
          Sp1/SP1F, CREB, 
        </bold> FOXC2, FOXD, FOXD2 </td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          PRKAR1B 
        </bold> </td> 
       <td align="left">EKLF, MAZF, HESF, PLAG, ZBPF, EBOX, 
        <bold>
          EGRF 
        </bold>, MYOD, MZF1, AHRR, ETSF, HIFF, SP1F, 
        <bold>
          AP1F 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Jun/AP1F 
        </bold>, p53/P53F, Oct-1/OCT1, 
        <bold>
          Egr1/EGRF 
        </bold> Pax1/PAX1 </td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          PRKAR2B 
        </bold> </td> 
       <td align="left">ZBPF, ZF5F, EGRF, EKLF, MAZF, HESF, 
        <bold>
          SP1F 
        </bold>, CREB, 
        <bold>
          EBOX 
        </bold>, ETSF, 
        <bold>
          NF1F 
        </bold> </td> 
       <td align="left"> 
        <bold>
          Sp1/SP1F, NF-1/NF1F, Myc/EBOX, 
        </bold> CEBPbeta/CEBP 
        <bold>
          USF1, USF2/EBOX 
        </bold> </td> 
      </tr> 
     </tbody> 
    </table> 
    <table-wrap-foot> 
     <p>The table shows the transcriptional factor families identified by computational analysis and the transcriptional factors/transcriptional family identified by analysis of literature data. The factors identified by both analysis are shown in 
      <bold>
        bold 
      </bold>.</p> 
    </table-wrap-foot> 
   </table-wrap> 
   <p>PKA type I regulatory subunit A (PRKAR1A) expression has been studied in different cellular models by analyzing its mRNA expression and by using its putative promoter region. In its promoter, binding sites for activator protein-1 and 2 (AP-1 and AP-2) and Sp1 [ 
    <xref ref-type="bibr" rid="B37">
      37 
    </xref>] have been identified. Moreover, a more recent work showed a direct activity of FOX family (FOXC2, D1 and D2) transcriptional factors members in the regulation of PRKAR1A expression both at transcriptional and at post-transcriptional levels [ 
    <xref ref-type="bibr" rid="B38">
      38 
    </xref>, 
    <xref ref-type="bibr" rid="B39">
      39 
    </xref>].</p> 
   <p>The promoter of PRKAR1B has been identified and studied in human and mouse: binding sites for Jun and p53 (human) and Oct-1, Egr1 and Pax1 (mouse) have been found. These binding sites have been experimentally verified by Electrophoretic Mobility Shift Assay, functional analysis and Northern blot [ 
    <xref ref-type="bibr" rid="B40">
      40 
    </xref>, 
    <xref ref-type="bibr" rid="B41">
      41 
    </xref>].</p> 
   <p>PRKAR2B promoter has been studied in particular in Sertoli cells (human). Some reports identified binding sites for Sp1, NF-1, Myc, C/EBPbeta, able to induce the PRKAR2B promoter, USF1 and USF2. Interestingly, overexpression of USF2, but not USF1, led to inhibition of both cAMP- and C/EBPbeta-mediated induction of PRKAR2B [ 
    <xref ref-type="bibr" rid="B42">
      42 
    </xref>- 
    <xref ref-type="bibr" rid="B44">
      44 
    </xref>].</p> 
   <p>The promoter of Protein kinase, cAMP-dependent, catalytic, alpha (PRKACA) has been identified both in humans and mouse, but little information has been produced for human promoter. Indeed, one paper describes the presence of binding sites for USF1 and USF2 transcription factors [ 
    <xref ref-type="bibr" rid="B45">
      45 
    </xref>].</p> 
   <p>AKAP1, AKAP9 and AKAP10 promoters contain binding sites for c-Myc as shown by computational analysis and ChIP experiments in several human cell lines [ 
    <xref ref-type="bibr" rid="B46">
      46 
    </xref>, 
    <xref ref-type="bibr" rid="B47">
      47 
    </xref>]. Moreover, a single study indicates the presence in the promoter of AKAP12 of binding sites for Serum Response Factor transcriptional factors [ 
    <xref ref-type="bibr" rid="B48">
      48 
    </xref>] and more recently for Myc.</p> 
   <p>ADCY9 promoter contains binding sites for c-Myc as shown by an experimental approach [ 
    <xref ref-type="bibr" rid="B49">
      49 
    </xref>].</p> 
   <p>Several promoters of genes encoding phosphodiesterase proteins have been isolated and to some extent studied. All the studies have been performed on sequences of human promoters and in particular the PDE4B, PDE4C (both present in our collection of PKA pathway related genes), and PDE5A, PDE6A, PDE6B and PDE7A promoters (not present in our gene list) have been better characterized. In the PDE4B promoter, binding sites for CREB have been found [ 
    <xref ref-type="bibr" rid="B50">
      50 
    </xref>]. In PDE4C promoter, binding sites for Myc have been found [ 
    <xref ref-type="bibr" rid="B47">
      47 
    </xref>]. In the PDE5A promoter, binding sites for Jun and AP-2 have been found [ 
    <xref ref-type="bibr" rid="B51">
      51 
    </xref>, 
    <xref ref-type="bibr" rid="B52">
      52 
    </xref>]; in PDE6A and PDE6B promoters, binding sites for Sp1 [ 
    <xref ref-type="bibr" rid="B53">
      53 
    </xref>] and Sp4 [ 
    <xref ref-type="bibr" rid="B54">
      54 
    </xref>, 
    <xref ref-type="bibr" rid="B55">
      55 
    </xref>] respectively and in PDE7A promoter, Ets2 and NFkB1 binding sites [ 
    <xref ref-type="bibr" rid="B56">
      56 
    </xref>].</p> 
   <p>The cyclic AMP response element (CRE)-binding protein CREB promoter has been identified in human, mouse and rat. Analysis done on human promoter, experimentally confirmed, identified binding site for c-Myc [ 
    <xref ref-type="bibr" rid="B57">
      57 
    </xref>] and Sp1 [ 
    <xref ref-type="bibr" rid="B58">
      58 
    </xref>]. Further information about such promoter has been produced in mouse and rat cells which allowed the identification of binding site for NfkB [ 
    <xref ref-type="bibr" rid="B57">
      57 
    </xref>].</p> 
   <p>An important regulative mechanism of the PKA pathway is feedback control. Indeed as well as the cAMP produced by Adenylyl Cyclases, activate PKA kinase activity, PKA is able to inhibit the pathway, activating by phosphorylation the Phosphodiesterases, which ultimately induce hydrolysis of cAMP switching off the pathway. Moreover a huge amount of data has been published regarding the ability of PKA to activate specific transcription factors by phosphorylation: cyclic AMP response element (CRE)-binding protein CREB, the cAMP response element modulator (CREM), the activating transcription factor 1 (ATF-1) and a repressor, ICER (inducible cAMP early repressor) [ 
    <xref ref-type="bibr" rid="B59">
      59 
    </xref>] that, to a certain extent, has been shown to regulate PKA pathway-related genes transcription. Some of the promoters, already discussed above, have been shown to have CRE binding sites. Moreover, two interesting recent publications, have identified and characterized in different cellular contexts and by several approaches, through a genome-wide approach, target genes that are regulated by CREB [ 
    <xref ref-type="bibr" rid="B60">
      60 
    </xref>, 
    <xref ref-type="bibr" rid="B61">
      61 
    </xref>]. The authors have identified and proved by ChiP analysis (PRKAR1A, PDE7B) the presence of CRE site in PRKAR1A, in PDE7B, AKAP8, PDE4C and ADCY8. In the latter case they did not observe binding by Chip analysis, but another report has shown that its activation is mediated specifically via the canonical CRE site [ 
    <xref ref-type="bibr" rid="B62">
      62 
    </xref>]. Binding sites for CREB1 have been found in PDE7A [ 
    <xref ref-type="bibr" rid="B56">
      56 
    </xref>], PDE4D [ 
    <xref ref-type="bibr" rid="B63">
      63 
    </xref>], CREM [ 
    <xref ref-type="bibr" rid="B64">
      64 
    </xref>] and experimentally confirmed. Moreover analysis of the promoter of CREB gene showed the presence of several CRE binding sites [ 
    <xref ref-type="bibr" rid="B65">
      65 
    </xref>, 
    <xref ref-type="bibr" rid="B66">
      66 
    </xref>]. </p> 
   <p>Most of AP-1 (i.e. Jun), AP-2 and Sp1 transcription factors are involved in growth-related signal transduction pathways, among which Ras is a main actor, and their over-expression can have positive or negative effects on proliferation [ 
    <xref ref-type="bibr" rid="B67">
      67 
    </xref>- 
    <xref ref-type="bibr" rid="B71">
      71 
    </xref>]. Indeed Sp family has been shown to be regulated by post-translational mechanisms by Ras pathway [ 
    <xref ref-type="bibr" rid="B72">
      72 
    </xref>, 
    <xref ref-type="bibr" rid="B73">
      73 
    </xref>] as well as Ets1 and Ets2 [ 
    <xref ref-type="bibr" rid="B74">
      74 
    </xref>, 
    <xref ref-type="bibr" rid="B75">
      75 
    </xref>] and NFkB [ 
    <xref ref-type="bibr" rid="B76">
      76 
    </xref>- 
    <xref ref-type="bibr" rid="B78">
      78 
    </xref>].</p> 
   <p>Egr-1 is an early responsive gene linked to mitogenic stimulation directly regulated by MAPK pathway [ 
    <xref ref-type="bibr" rid="B79">
      79 
    </xref>- 
    <xref ref-type="bibr" rid="B82">
      82 
    </xref>]. Moreover for Myc [ 
    <xref ref-type="bibr" rid="B83">
      83 
    </xref>, 
    <xref ref-type="bibr" rid="B84">
      84 
    </xref>], C/EBPbeta [ 
    <xref ref-type="bibr" rid="B85">
      85 
    </xref>] and NF-1 [ 
    <xref ref-type="bibr" rid="B86">
      86 
    </xref>] a large amount of data about their correlation with Ras pathway has been reported. Each of these transcriptional factors has been associated with several cellular responses (proliferation, survival, apoptosis) and transformation as is the case of the PKA pathway as well. Therefore it is possible that mitogenic signal through Ras and the regulation of such transcription factors, modulates the expression of PKA pathway related genes. </p> 
   <p>An important role, in the activation of the CREB family transcription factors, is played by stimuli which are able to induce their phosphorylation and consequently their activation. In fact as reviewed in [ 
    <xref ref-type="bibr" rid="B65">
      65 
    </xref>] not only the protein kinase A is involved in this function but also several growth factors (NGF, FGF, IGF-I, PDGF, EGF), survival signals and hypoxia that often activate the Ras pathway, pointing to an essential role of the latter pathway also in gene transcriptional regulation of PKA pathway-encoding genes by transcription factors of the CREB family. </p> 
  </sec> 
 </sec> 
 <sec> 
  <h3>Conclusions</h3> 
  <p>By using a generalized workflow for data recovery and integration that combines accurate analysis of recovered data from several databases with the application of different statistical tests we have been able to correlate strong transcriptional repression of genes encoding proteins of the cAMP/PKA pathway in transformed samples of different genetic origin (i.e., bearing mutations in different pathways). This finding prompted us to compute 
   <italic>
     consensus 
   </italic> promoters, whose composition was specifically enriched for different transcription factor binding sites (TBFS). Comparison of TFBS computationally identified in the 
   <italic>
     consensus 
   </italic> promoters with TBFS experimentally identified by a variety of techniques, shows a good agreement. Indeed, by lowering the stringency used in the workflow, some of the TFBS missed by higher stringency analysis (false negatives) were recovered, in keeping with the notion that intersection of different data sets and/or techniques decreases both noise and the number of hits.</p> 
  <p>The workflow we have followed is summarized in Figure 
   <xref ref-type="fig" rid="F7">
     7 
   </xref> and detailed in Methods section. As the number of sites hosting curated transcriptional profiles increases, more and more data to be used as starting point become available. We used the GEO database to recover data from the NCI60 cell collection (cancer samples) and matching normal tissues and to which specific statistical tests (i.e. ANOVA, Hierarchical clustering) were applied. By using the COSMIC database, which gives information about the mutational status of the NCI60 collection, we could sort the NCI60 cell lines in 4 subgroups with mutational activation of genes encoding components of the Ras pathway, of the PI3K pathway, of other pathways or for which no information was available. Such a sorting allowed us to uncover an hitherto unrecognized oncogene-dependent pattern of regulation of 41 genes encoding components of the cAMP/PKA pathway (Figure 
   <xref ref-type="fig" rid="F7">
     7B 
   </xref> and 
   <xref ref-type="fig" rid="F7">
     7C 
   </xref>). The transcriptional profiles for transformed cells within one of the identified subgroups may then be used as a new query to GEO database (green arrow), in order to correlate and confirm, i.e. in cancer tissues, the oncogene-dependent pattern identified.</p> 
  <fig id="F7" position="float"> 
   <label>Figure 7</label> 
   <p> 
    <bold>
      Flowchart of our web-based and statistical strategy used to elucidate the relation between PKA encoding genes transcriptional profiles and oncogenic mutations. (A) 
    </bold> Flow chart of our web-based and statistical strategy with indication of some of the databases (Source) used, the type of data analyzed (Input), the specific program and statistical test (Tool) used and the result obtained (Output). 
    <bold>
      (B) 
    </bold> Graphical representation of the block diagram summarizing functional interconnections within the PKA pathway module with indication of the expression level (geometric mean) of each gene belonging to the network -Strong (red), Average (black) and Low (green)- as identified by our analysis both in normal (B, left) and transformed samples (B, right). 
    <bold>
      (C) 
    </bold> Boxplots of the expression of PKA pathway-encoding genes in normal (C, left) and transformed (C, right) samples, grouped for functional classes (ADCY: adenylyl cyclase; AKAP: A-kinase anchor protein; PDE: phosphodiesterase; PRKACR: PKA regulatory subunit; PRKAC: PKA catalytic subunit). The represented value is the median. 
    <bold>
      (D) 
    </bold> Schematic representation of the TFBSs (color-coded) identified in the promoters of PKA pathway-encoding genes of normal and transformed samples. Each cartoon represents the promoter structure resulting from the merge of the TFBS identified in ? 70% of the gene promoters of all normal samples and transformed samples.</p> 
   <graphic xlink:href="1471-2105-10-S12-S1-7"></graphic> 
  </fig> 
  <p>Deregulation of transcriptional programs, such as that identified for PKA pathway-encoding genes, may be considered a direct consequence of a deregulated activity of transcription factors. The TRANSFAC database was used with a high stringency threshold, to identify the regulatory sequence in co-regulated genes with high confidence, improving the deduced linkages between transcription factors and the regulated genes. Using this approach, we demonstrated that in all PKA encoding genes TFBSs for ETS, MAZ, ZBP and EGR transcription factors are present (Figure 
   <xref ref-type="fig" rid="F7">
     7D 
   </xref>) and that specific subsets of TFBS are present in the normal and transformed samples. The number of TFBS identified by computational analysis was higher than those that could be retrieved from literature as experimentally determined. This observation was to some extent expected because of limited literature reference availability, complexity to retrieve data, difficulty to analyze data from several origins, and the lack of powerful data analysis and integration tools. Under these less-than-ideal conditions, a dedicated tool such as the TFBS database, can be extremely powerful, allowing predictions that are amenable to experimental verification, should this be necessary. As discussed above, most of the false-negatives that failed to be detected by our computational approaches could be recovered by appropriately lowering the stringency of analysis. </p> 
  <p>In Figure 
   <xref ref-type="fig" rid="F7">
     7B 
   </xref> transcriptional expression of PKA pathway encoding genes is color-mapped (geometric mean, Strong expression, red, Average expression, black and Low expression green) on a block diagram summarizing functional interconnections within the PKA pathway module. A general and balanced co-regulation of both positive and negative regulators of the cAMP/PKA pathway is apparent in both normal and transformed samples. Notably, in normal cells variability in expression is maximal for genes encoding the catalytic subunit of PKA. Because of the pleiotropic role of the PKA pathway (including stimulation of growth and differentiation in many cell types, such as somatotrophs, thyrocytes, melanocytes, ovarian follicular granulosa cells, keratinocytes, nervous, muscle and blood cells and adipocyte and the important role of such pathway in the regulation of the function of tissues as kidney, ovary, brain, and prostate), strong expression in normal tissues is expected [ 
   <xref ref-type="bibr" rid="B8">
     8 
   </xref>, 
   <xref ref-type="bibr" rid="B87">
     87 
   </xref>, 
   <xref ref-type="bibr" rid="B88">
     88 
   </xref>]. It should also be remembered that cross-talk between the PKA pathway and oncogene-mediated pathways can also take place at post-transcriptional levels. For example, several Authors reported the ability of oncogenic and viral Ras proteins to either stimulate [ 
   <xref ref-type="bibr" rid="B89">
     89 
   </xref>- 
   <xref ref-type="bibr" rid="B91">
     91 
   </xref>] or inhibit [ 
   <xref ref-type="bibr" rid="B92">
     92 
   </xref>- 
   <xref ref-type="bibr" rid="B94">
     94 
   </xref>] ADCY activity in different cell lines (thyroid, epithelial, kidney, fibroblast). Moreover an involvement of MAPK or PI3K pathways in the regulation of PDE activity has been reported, suggesting that mitogenic stimulation may positively regulate PDE4 expression directly [ 
   <xref ref-type="bibr" rid="B95">
     95 
   </xref>], confirming our transcriptional results, or by post-translational mechanisms in which p42(MAPK) phosphorylation activity has a relevant role in their regulation [ 
   <xref ref-type="bibr" rid="B96">
     96 
   </xref>]. Another important post-transcriptional mechanism that links Ras or PI3K pathways to cAMP/PKA pathway is the positive and negative control of CREB activity by a phosphorylation [ 
   <xref ref-type="bibr" rid="B97">
     97 
   </xref>, 
   <xref ref-type="bibr" rid="B98">
     98 
   </xref>]. Moreover, it has been reported that cAMP is able to induce proliferation rather than growth inhibition, in several tumors where oncogenic activation of B-Raf has been identified (i.e., melanoma and thyroid cancer) [ 
   <xref ref-type="bibr" rid="B8">
     8 
   </xref>]. Nevertheless, the general and coordinated down-regulation of essentially all genes of the pathway in transformed cells (as compared with normal tissues) suggests that at least one PKA-mediated function needs to be reduced substantially in order to express the transformed phenotype. Although at this stage it is too early to propose specific hypotheses, it is intriguing to remember that PKA has been ascribed a role in activating mitochondrial respiration and decreasing ROS production [ 
   <xref ref-type="bibr" rid="B99">
     99 
   </xref>, 
   <xref ref-type="bibr" rid="B100">
     100 
   </xref>], thus effectively counteracting mitochondria dysfunction that is found associated with increased glycolysis (Warburg effect, [ 
   <xref ref-type="bibr" rid="B101">
     101 
   </xref>- 
   <xref ref-type="bibr" rid="B103">
     103 
   </xref>]) in many cancer cells. On the other hand, a reduction in oxidative phosphorylation that will decrease ATP supply, as substrate of adenylate cyclase, may result in a decreased cAMP production without relevant changes in the level of the enzymes (and possibly therefore of their gene expression). </p> 
  <p>It is expected that deeper computational integration of transcriptional data with other genome-wide findings, including -but not limited to- proteomics, interactomics and metabolomics, will allow a better extraction of hidden information. We propose that such data integration can be further applied to examine the topology of biological networks, to provide information on directionality of interactions, and create wiring diagrams that better depict the functional outcome of component-component relationships. Together, these strategies should facilitate a systems approach to modular biology.</p> 
  <p>Thus systems biology can be approached by perturbing the suspected components of a given cellular process, monitoring the responses, integrating the data and modeling the biological process in question [ 
   <xref ref-type="bibr" rid="B104">
     104 
   </xref>]. By applying a single "omic" approach, able to sample a "horizontal" slice (i.e. across all genes or gene products) of a multidimensional space, the knowledge of a system can be expanded from a single gene to a network of genes, which can be regarded as a basic model for the system. When genes or proteins in this network are systematically disrupted, responses from other parts of the network can be recorded and the data obtained can be incorporated into the basic model. However sampling a single dimension of a complex space will undoubtedly provide relevant information, but may not highlight the major regulatory features. Therefore, a wiring diagram that depicts the direction of interactions in the network and the behavior of each of these components can be constructed to better represent the relationships between the components [ 
   <xref ref-type="bibr" rid="B104">
     104 
   </xref>]. The example shown in Figure 
   <xref ref-type="fig" rid="F7">
     7 
   </xref> illustrates how our current knowledge of a biological system can be expanded and a model built based on integrated "omic" information. Ultimately, development of such computational methods and their recursive integration with genome-wide and hypothesis-driven experimental investigations that also take into account post-translational and substrate-dependent mechanisms controlling the cAMP/PKA pathway activity, should reconcile experiments from different experimental systems (cell cultures, animal models and human tumor samples) and contribute to explain at an integrative, systems level how the cAMP-PKA pathway is affected by oncogenic processes originated by mutational activation of signal transduction.</p> 
 </sec> 
 <sec sec-type="methods"> 
  <h3>Methods</h3> 
  <sec> 
   <h3>Data recovery and normalization</h3> 
   <p>Gene expression data of NCI60 cell lines and normal tissues samples were downloaded from the Gene Expression Omnibus (GEO) at the National Center for Biotechnology Information (NCBI) website ( 
    <ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/geo/"></ext-link>) [ 
    <xref ref-type="bibr" rid="B105">
      105 
    </xref>]. In particular, gene expression profiles of NCI60 cell collection (cancer samples) were recovered from GEO database ( 
    <bold>
      GSE5949, 
    </bold>[ 
    <xref ref-type="bibr" rid="B31">
      31 
    </xref>]) in which the experimental data were obtained by using the Affymetrix HG-U95Av2 oligonucleotide array platform. For the analysis only results obtained by oligonucleotide arrays were considered, because this platform uses a different method to evaluate mRNA expression as compared to cDNA array platform. Therefore, also for normal tissue samples, the data used for the comparative analysis, were recovered from transcriptional profiles produced by using U95Av2 oligonucleotide array ( 
    <bold>
      GSE96 
    </bold>[ 
    <xref ref-type="bibr" rid="B106">
      106 
    </xref>], 
    <bold>
      GSE6731 
    </bold>[ 
    <xref ref-type="bibr" rid="B107">
      107 
    </xref>] and 
    <bold>
      GSE1402 
    </bold>[ 
    <xref ref-type="bibr" rid="B108">
      108 
    </xref>]). </p> 
   <p>A total of 81 transcriptional profiles encompassing cancer cell lines with nine histological origins and samples from six normal tissues were recovered. Further details can be found in the legends of Tables 
    <xref ref-type="table" rid="T1">
      1 
    </xref> and 
    <xref ref-type="table" rid="T2">
      2 
    </xref>. All datasets were generated by downloading and processing CEL files. They were preprocessed using Robust Multichip Average (RMA) [ 
    <xref ref-type="bibr" rid="B109">
      109 
    </xref>, 
    <xref ref-type="bibr" rid="B110">
      110 
    </xref>] and then transformed from log<sub>2</sub> values to linear scale values, and normalized per gene to the median value of its level of expression across 81 samples, as implemented in GeneSpring GX 7.3.1 (Silicon Genetics - 
    <ext-link ext-link-type="uri" xlink:href="http://www.chem.agilent.com/"></ext-link>). The RMA preprocessing algorithm includes background and quantile normalization steps [ 
    <xref ref-type="bibr" rid="B109">
      109 
    </xref>, 
    <xref ref-type="bibr" rid="B111">
      111 
    </xref>]. Although background correction, as first step analysis, has been computed separately for each array, all the other procedures performed by using RMA (normalization and summarization), have been performed across all the arrays (RMA is a multiple-array method). Normalization is necessary so that multiple chips can be compared to each other, and analyzed together. The normalization procedure is aimed at making the distributions identical across arrays. RMA usually gives very accurate normalizations. Note that, RMA implemented in GeneSpring GX 7.3.1, all the arrays are used and no chip is discarded. We used RMA analysis as compared to other tools of analysis, because, as described in several papers [ 
    <xref ref-type="bibr" rid="B112">
      112 
    </xref>, 
    <xref ref-type="bibr" rid="B113">
      113 
    </xref>], it successfully reduces the variance of low abundance transcripts and better distinguishes differentially expressed transcripts from those that are unchanging, by using controlled datasets in which known quantities of specific mRNAs have been added to a common reference pool, [ 
    <xref ref-type="bibr" rid="B109">
      109 
    </xref>, 
    <xref ref-type="bibr" rid="B110">
      110 
    </xref>, 
    <xref ref-type="bibr" rid="B114">
      114 
    </xref>]. </p> 
  </sec> 
  <sec> 
   <h3>Transformation-dependent, transcriptional remodeling of the PKA pathway-encoding genes in 60 human cancer cell lines (NCI60) and 21 human normal tissues</h3> 
   <p>We identified and gathered the transcriptional profile for 41 genes encoding proteins involved in the PKA pathway (adenylyl cyclases -ADCY-, phosphodiesterases -PDE-, A-kinase anchor proteins -AKAP-, cAMP-dependent transcriptional factors -TF-, PKA catalytic subunits -PRKAC- and PKA regulatory subunits -PRKACR-, Table 
    <xref ref-type="table" rid="T3">
      3 
    </xref>).</p> 
   <p>In order to identify specific variations in the expression pattern of the selected PKA pathway-related genes both in normal and transformed samples, different tools of analysis were used.</p> 
   <p>Initially, the PKA pathway related genes expression profiles, observed in transformed samples as compared to normal samples, were evaluated by analysis of variance (ANOVA). Such statistical linear modeling procedure, that partitions the total variance into parts corresponding to various sources in the model [ 
    <xref ref-type="bibr" rid="B115">
      115 
    </xref>, 
    <xref ref-type="bibr" rid="B116">
      116 
    </xref>] have been successfully used to analyze microarray data [ 
    <xref ref-type="bibr" rid="B117">
      117 
    </xref>- 
    <xref ref-type="bibr" rid="B120">
      120 
    </xref>]. In order to model and test the hypothesis that the expression of genes of PKA pathway was different between normal tissues and transformed cell lines, the following comparisons were used: 
    <italic>
      Expression of gene 
     <sub>i</sub> (where i=i-esimo) of Normal Tissues vs. Transformed cell lines 
    </italic> (Figure 
    <xref ref-type="fig" rid="F1">
      1 
    </xref>), and a p-value &lt; 0.05.</p> 
   <p>The same data-set was then analyzed through unsupervised hierarchical clustering [ 
    <xref ref-type="bibr" rid="B121">
      121 
    </xref>] (as implemented in the GeneSpring platform). Two-way hierarchical clustering was performed on RMA-generated linear scale expression levels using the Pearson correlation coefficient as the measure of similarity and complete linkage clustering [ 
    <xref ref-type="bibr" rid="B122">
      122 
    </xref>]. The results of this process are dendrograms, in which short branches connect very similar elements, and longer branches join elements with diminishing degrees of similarity. The vectors used were sample - normal tissues and transformed cells- and expression of genes of PKA pathway-related genes and the arms were classified by different variables: 
    <italic>
      Conditions 
    </italic> and 
    <italic>
      Tissues 
    </italic>, (Figure 
    <xref ref-type="fig" rid="F2">
      2 
    </xref>). </p> 
  </sec> 
  <sec> 
   <h3>Analysis of mutational status of the NCI60 cell lines and correlation with tissue-specific PKA pathway gene regulation</h3> 
   <p>The 60 cell lines were sorted according to mutational status, using the information provided by Catalogue Of Somatic Mutations In Cancer ( 
    <ext-link ext-link-type="uri" xlink:href="http://www.sanger.ac.uk/genetics/CGP/cosmic/"></ext-link>) [ 
    <xref ref-type="bibr" rid="B123">
      123 
    </xref>]. This database holds somatic mutation data and other information related to human cancer cell lines and tissues, and can be interrogated through a series of web pages to provide a graphical or tabular view of the data along with various export options. We could sort the NCI60 cell lines in 4 subgroups presenting mutational activation of genes encoding components of the Ras pathway, of the PI3K pathway, of other pathways or for which no information was available, (Table 
    <xref ref-type="table" rid="T5">
      5 
    </xref>).</p> 
   <p>In order to identify specific variations in the expression pattern of the PKA pathway-related genes in these 4 subgroups, different tools of analysis were used.</p> 
   <p>We applied unsupervised Principal Component Analysis (PCA) [ 
    <xref ref-type="bibr" rid="B124">
      124 
    </xref>, 
    <xref ref-type="bibr" rid="B125">
      125 
    </xref>] to establish the interrelationships among the samples used in our study. PCA is a statistical method that can be used to reduce complex data sets with multiple variables into significantly smaller numbers of variables (known as components), which retain the relevant variance information used to distinguish the sample groups from another. By visualizing projections of these components in low-dimensional spaces, we were able to observe the grouping of samples reflecting underlying patterns in their gene expression profiles. PCA on the mean centered and scaling data was used to model the effects of oncogene-dependent transformation on the gene expression. The following comparisons were performed: 
    <italic>
      Expression of gene 
     <sub>i</sub> of Normal Tissues vs. PI3K mutation cell lines; vs. Ras mutation cell lines; vs. Not Tested mutation cell lines; vs. Other Mutation cell lines. 
    </italic> </p> 
   <p>Also in this case, in order to model and test the hypothesis that the expression of genes of PKA pathway was different between normal tissues and the four subgroups previously identified, we applied one-way ANOVA by using the following comparisons: 
    <italic>
      Expression of gene 
     <sub>i</sub> (where i=i-esimo) of Normal Tissues vs. PI3K mutation cell lines; vs. Ras mutation cell lines; vs. Not Tested mutation cell lines; vs. Other Mutation cell lines 
    </italic>, (Figure 
    <xref ref-type="fig" rid="F3">
      3D 
    </xref>), and a p-value &lt; 0.05.</p> 
   <p>The data-set of 41 genes was then analyzed through unsupervised hierarchical clustering (Pearson correlation coefficient and complete linkage clustering). The vectors used were sample - oncogene-dependent transformed cells - and expression of genes of PKA pathway-related genes. The results of this process are dendrograms, in which the arms were classified by different variables: 
    <italic>
      Tissue, mutation and Pathway 
    </italic> (Figure 
    <xref ref-type="fig" rid="F4">
      4 
    </xref>).</p> 
  </sec> 
  <sec> 
   <h3>Computational analysis of promoters of differentially regulated PKA pathway-encoding genes and identification of transcriptional factor binding sites</h3> 
   <p>In order to identify Transcriptional Factor Binding Sites (TFBS) present in promoters of co-regulated genes, the 41 PKA pathway-encoding genes were sorted, relative to their level of expression, in three groups: Strong (&gt;1), Average (=1) and Low (&lt;1), where 1 is the expression value calculated by RMA. Each groups was identified in each sample group, i.e., Normal Tissues, cell lines carrying mutation(s) in Ras pathway-encoding genes, cell lines carrying mutation(s) in PI3K pathway-encoding genes, cell lines carrying mutation(s) in other pathways, cell lines Not Tested for mutation, thus generating 15 subgroups. A TFBS was called present only when present in more than 70 % of promoters within each group. </p> 
   <p>Proximal promoter regions - defined as 500 nt upstream and 100 nt downstream from the transcription start site (TSS), automatically assigned to genes on the basis of 5' cap-site databases integrated into promoter identification program - were identified using Eldorado (gems launcher, Genomatix [ 
    <xref ref-type="bibr" rid="B126">
      126 
    </xref>]) and the Genomatix Promoter Database. </p> 
   <p>TFBS in the promoter regions were identified by using ModelInspector and the Genomatix Promoter Database, comprising a total of 519 matrices from 154 families (Matrix Family Library, on Vertebrates, Version 7.1, June 2008). The Matrix Family Library is based on 260,000 human, mouse, and rat promoter sequences, with an average length of 650bp. Analysis on the 41 PKA pathway-encoding genes was performed with a threshold of 1.0 for the core similarity -that is reached only when the highest conserved bases of a matrix match exactly in the sequence- and a value of 0.85 for the Optimized matrix threshold [ 
    <xref ref-type="bibr" rid="B127">
      127 
    </xref>]. Optimized matrix threshold is the optimized value defined in a way that a minimum number of matches is found in non-regulatory test sequences. This value, when is higher than 0.80, permits the reduction of false positive matches. </p> 
   <p>The total number and frequency (i.e., the ratio between the total number of TBFS and the number of promoters present within each subgroup) of each TFBS within each subgroup were calculated. The frequency of each TFBS called present in each of the 15 subgroups of PKA pathway-encoding genes was compared with the frequency of the same TFBS within the Matrix Family Library on Vertebrates. TFBS enrichment was scored based on p-value generated by hypergeometric distribution and calculated with the 2-tailed Fisher's exact test, implemented through the use of a 2 x 2 contingency table (Figure 
    <xref ref-type="fig" rid="F5">
      5 
    </xref>).</p> 
   <p>In order to identify differences between the 15 groups a two-way hierarchical clustering (by using as vectors sample and TFBS) was applied by using the total number values and the frequency values of each TFBS identified in ? 70% of the promoters in each group. The total number value was transformed in the log<sub>2</sub> and used in the hierarchical clustering by using the Pearson correlation coefficient as the measure of similarity and complete linkage clustering (Figure 
    <xref ref-type="fig" rid="F6">
      6 
    </xref>).</p> 
  </sec> 
  <sec> 
   <h3>Promoter data mining</h3> 
   <p>To identify known transcription factor binding sites in the promoter sequences of PKA pathway-encoding genes, the annotated promoter and associated information have been retrieved from Transcriptional Regulatory Element Database (TRED) [ 
    <xref ref-type="bibr" rid="B128">
      128 
    </xref>] ( 
    <ext-link ext-link-type="uri" xlink:href="http://rulai.cshl.edu/cgi-bin/TRED/tred.cgi?process=home"></ext-link>) and from NCBI ( 
    <ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/"></ext-link>). Both web sites are freely accessible. The results have been shown in the Table 
    <xref ref-type="table" rid="T6">
      6 
    </xref>.</p> 
  </sec> 
 </sec> 
 <sec> 
  <h3>Competing interests</h3> 
  <p>The authors declare that they have no competing interests.</p> 
 </sec> 
 <sec> 
  <h3>Authors' contributions</h3> 
  <p>CB, MV and FC conceived the experiments and wrote the manuscript. CB and FC carried out the data analysis. LA, MV and FC participated in data analysis. All authors read, edit and approved the final manuscript.</p> 
 </sec> 
 <back> 
  <ack> 
   <sec> 
    <h3>Acknowledgments</h3> 
    <p>This work has been supported by grants from ITALBIONET to L.A. and FAR to M.V. and F.C. The authors wish to thank Neil Campbell for language checking.</p> 
    <p>This article has been published as part of 
     <italic>
       BMC Bioinformatics 
     </italic> Volume 10 Supplement 12, 2009: Bioinformatics Methods for Biomedical Complex System Applications. The full contents of the supplement are available online at 
     <ext-link ext-link-type="uri" xlink:href="http://www.biomedcentral.com/1471-2105/10?issue=S12"></ext-link>.</p> 
   </sec> 
  </ack> 
  <ref-list> 
   <ref id="B1"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Alberghina 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chiaradonna 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vanoni 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Systems biology and the molecular circuits of cancer. 
     </article-h3> 
     <source />Chembiochem 
     <year>
       2004 
     </year> 
     <volume>
       5 
     </volume> 
     <fpage>
       1322 
     </fpage> 
     <lpage>
       1333 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15457529 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1002/cbic.200400170 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B2"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Li 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Xuan 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wang 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zhan 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Inferring regulatory networks. 
     </article-h3> 
     <source />Front Biosci 
     <year>
       2008 
     </year> 
     <volume>
       13 
     </volume> 
     <fpage>
       263 
     </fpage> 
     <lpage>
       275 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17981545 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.2741/2677 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B3"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Zhu 
       </surname> 
       <given-names>
         X 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gerstein 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Snyder 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Getting connected: analysis and principles of biological networks. 
     </article-h3> 
     <source />Genes Dev 
     <year>
       2007 
     </year> 
     <volume>
       21 
     </volume> 
     <fpage>
       1010 
     </fpage> 
     <lpage>
       1024 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17473168 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1101/gad.1528707 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B4"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Srinivasan 
       </surname> 
       <given-names>
         BS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shah 
       </surname> 
       <given-names>
         NH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Flannick 
       </surname> 
       <given-names>
         JA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Abeliuk 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Novak 
       </surname> 
       <given-names>
         AF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Batzoglou 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Current progress in network research: toward reference networks for key model organisms. 
     </article-h3> 
     <source />Briefings in bioinformatics 
     <year>
       2007 
     </year> 
     <volume>
       8 
     </volume> 
     <fpage>
       318 
     </fpage> 
     <lpage>
       332 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17728341 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1093/bib/bbm038 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B5"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Kwoh 
       </surname> 
       <given-names>
         CK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ng 
       </surname> 
       <given-names>
         PY 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Network analysis approach for biology. 
     </article-h3> 
     <source />Cell Mol Life Sci 
     <year>
       2007 
     </year> 
     <volume>
       64 
     </volume> 
     <fpage>
       1739 
     </fpage> 
     <lpage>
       1751 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17415520 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1007/s00018-007-7053-7 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B6"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Philippi 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kohler 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Addressing the problems with life-science databases for traditional uses and systems biology. 
     </article-h3> 
     <source />Nature reviews 
     <year>
       2006 
     </year> 
     <volume>
       7 
     </volume> 
     <fpage>
       482 
     </fpage> 
     <lpage>
       488 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16682980 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/nrg1872 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B7"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Romano 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kracht 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Manniello 
       </surname> 
       <given-names>
         MA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stegehuis 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fritze 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The role of informatics in the coordinated management of biological resources collections. 
     </article-h3> 
     <source />Applied bioinformatics 
     <year>
       2005 
     </year> 
     <volume>
       4 
     </volume> 
     <fpage>
       175 
     </fpage> 
     <lpage>
       186 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16231959 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B8"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Dumaz 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Marais 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Integrating signals between cAMP and the RAS/RAF/MEK/ERK signalling pathways. Based on the anniversary prize of the Gesellschaft fur Biochemie und Molekularbiologie Lecture delivered on 5 July 2003 at the Special FEBS Meeting in Brussels. 
     </article-h3> 
     <source />Febs J 
     <year>
       2005 
     </year> 
     <volume>
       272 
     </volume> 
     <fpage>
       3491 
     </fpage> 
     <lpage>
       3504 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16008550 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1111/j.1742-4658.2005.04763.x 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B9"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Mei 
       </surname> 
       <given-names>
         FC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Qiao 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tsygankova 
       </surname> 
       <given-names>
         OM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Meinkoth 
       </surname> 
       <given-names>
         JL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Quilliam 
       </surname> 
       <given-names>
         LA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cheng 
       </surname> 
       <given-names>
         X 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Differential signaling of cyclic AMP: opposing effects of exchange protein directly activated by cyclic AMP and cAMP-dependent protein kinase on protein kinase B activation. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       2002 
     </year> 
     <volume>
       277 
     </volume> 
     <fpage>
       11497 
     </fpage> 
     <lpage>
       11504 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11801596 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.M110856200 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B10"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Stork 
       </surname> 
       <given-names>
         PJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Schmitt 
       </surname> 
       <given-names>
         JM 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Crosstalk between cAMP and MAP kinase signaling in the regulation of cell proliferation. 
     </article-h3> 
     <source />Trends Cell Biol 
     <year>
       2002 
     </year> 
     <volume>
       12 
     </volume> 
     <fpage>
       258 
     </fpage> 
     <lpage>
       266 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12074885 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/S0962-8924(02)02294-8 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B11"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Bos 
       </surname> 
       <given-names>
         JL 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The ras gene family and human carcinogenesis. 
     </article-h3> 
     <source />Mutat Res 
     <year>
       1988 
     </year> 
     <volume>
       195 
     </volume> 
     <fpage>
       255 
     </fpage> 
     <lpage>
       271 
     </lpage> 
     <pub-id pub-id-type="pmid">
       3283542 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B12"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Cho-Chung 
       </surname> 
       <given-names>
         YS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nesterova 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Becker 
       </surname> 
       <given-names>
         KG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Srivastava 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Park 
       </surname> 
       <given-names>
         YG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         YN 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cho 
       </surname> 
       <given-names>
         YS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kim 
       </surname> 
       <given-names>
         MK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Neary 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cheadle 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Dissecting the circuitry of protein kinase A and cAMP signaling in cancer genesis: antisense, microarray, gene overexpression, and transcription factor decoy. 
     </article-h3> 
     <source />Ann N Y Acad Sci 
     <year>
       2002 
     </year> 
     <volume>
       968 
     </volume> 
     <fpage>
       22 
     </fpage> 
     <lpage>
       36 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12119265 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1111/j.1749-6632.2002.tb04324.x 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B13"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Malumbres 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Barbacid 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       RAS oncogenes: the first 30 years. 
     </article-h3> 
     <source />Nat Rev Cancer 
     <year>
       2003 
     </year> 
     <volume>
       3 
     </volume> 
     <fpage>
       459 
     </fpage> 
     <lpage>
       465 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12778136 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/nrc1097 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B14"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Siu 
       </surname> 
       <given-names>
         YT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jin 
       </surname> 
       <given-names>
         DY 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       CREB--a real culprit in oncogenesis. 
     </article-h3> 
     <source />Febs J 
     <year>
       2007 
     </year> 
     <volume>
       274 
     </volume> 
     <fpage>
       3224 
     </fpage> 
     <lpage>
       3232 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17565603 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1111/j.1742-4658.2007.05884.x 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B15"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Treisman 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Regulation of transcription by MAP kinase cascades. 
     </article-h3> 
     <source />Curr Opin Cell Biol 
     <year>
       1996 
     </year> 
     <volume>
       8 
     </volume> 
     <fpage>
       205 
     </fpage> 
     <lpage>
       215 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8791420 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/S0955-0674(96)80067-6 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B16"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Bader 
       </surname> 
       <given-names>
         AG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kang 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zhao 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vogt 
       </surname> 
       <given-names>
         PK 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Oncogenic PI3K deregulates transcription and translation. 
     </article-h3> 
     <source />Nat Rev Cancer 
     <year>
       2005 
     </year> 
     <volume>
       5 
     </volume> 
     <fpage>
       921 
     </fpage> 
     <lpage>
       929 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16341083 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/nrc1753 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B17"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Downward 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Signal transduction. Prelude to an anniversary for the RAS oncogene. 
     </article-h3> 
     <source />Science 
     <year>
       2006 
     </year> 
     <volume>
       314 
     </volume> 
     <fpage>
       433 
     </fpage> 
     <lpage>
       434 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17053139 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1126/science.1134727 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B18"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Downward 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Targeting RAS signalling pathways in cancer therapy. 
     </article-h3> 
     <source />Nat Rev Cancer 
     <year>
       2003 
     </year> 
     <volume>
       3 
     </volume> 
     <fpage>
       11 
     </fpage> 
     <lpage>
       22 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12509763 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/nrc969 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B19"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Marais 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Light 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mason 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Paterson 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Olson 
       </surname> 
       <given-names>
         MF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Marshall 
       </surname> 
       <given-names>
         CJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Requirement of Ras-GTP-Raf complexes for activation of Raf-1 by protein kinase C. 
     </article-h3> 
     <source />Science 
     <year>
       1998 
     </year> 
     <volume>
       280 
     </volume> 
     <fpage>
       109 
     </fpage> 
     <lpage>
       112 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9525855 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1126/science.280.5360.109 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B20"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Rodriguez-Viciana 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Marte 
       </surname> 
       <given-names>
         BM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Warne 
       </surname> 
       <given-names>
         PH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Downward 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Phosphatidylinositol 3' kinase: one of the effectors of Ras. 
     </article-h3> 
     <source />Philos Trans R Soc Lond B Biol Sci 
     <year>
       1996 
     </year> 
     <volume>
       351 
     </volume> 
     <fpage>
       225 
     </fpage> 
     <lpage>
       231 
     </lpage> 
     <comment>
       discussion 231-222. 
     </comment> 
     <pub-id pub-id-type="pmid">
       8650270 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1098/rstb.1996.0020 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B21"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         McKay 
       </surname> 
       <given-names>
         MM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Morrison 
       </surname> 
       <given-names>
         DK 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Integrating signals from RTKs to ERK/MAPK. 
     </article-h3> 
     <source />Oncogene 
     <year>
       2007 
     </year> 
     <volume>
       26 
     </volume> 
     <fpage>
       3113 
     </fpage> 
     <lpage>
       3121 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17496910 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/sj.onc.1210394 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B22"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Anderson 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Boronenkov 
       </surname> 
       <given-names>
         IV 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Doughman 
       </surname> 
       <given-names>
         SD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kunz 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Loijens 
       </surname> 
       <given-names>
         JC 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Phosphatidylinositol phosphate kinases, a multifaceted family of signaling enzymes. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       1999 
     </year> 
     <volume>
       274 
     </volume> 
     <fpage>
       9907 
     </fpage> 
     <lpage>
       9910 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10187762 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.274.15.9907 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B23"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Chang 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         JT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Navolanic 
       </surname> 
       <given-names>
         PM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Steelman 
       </surname> 
       <given-names>
         LS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shelton 
       </surname> 
       <given-names>
         JG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Blalock 
       </surname> 
       <given-names>
         WL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Franklin 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McCubrey 
       </surname> 
       <given-names>
         JA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Involvement of PI3K/Akt pathway in cell cycle progression, apoptosis, and neoplastic transformation: a target for cancer chemotherapy. 
     </article-h3> 
     <source />Leukemia 
     <year>
       2003 
     </year> 
     <volume>
       17 
     </volume> 
     <fpage>
       590 
     </fpage> 
     <lpage>
       603 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12646949 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/sj.leu.2402824 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B24"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Davis 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Transcriptional regulation by MAP kinases. 
     </article-h3> 
     <source />Mol Reprod Dev 
     <year>
       1995 
     </year> 
     <volume>
       42 
     </volume> 
     <fpage>
       459 
     </fpage> 
     <lpage>
       467 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8607977 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1002/mrd.1080420414 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B25"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Shaul 
       </surname> 
       <given-names>
         YD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Seger 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The MEK/ERK cascade: From signaling specificity to diverse functions. 
     </article-h3> 
     <source />Biochim Biophys Acta 
     <year>
       2007 
     </year> 
     <volume>
       1773 
     </volume> 
     <fpage>
       1213 
     </fpage> 
     <lpage>
       1226 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17112607 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/j.bbamcr.2006.10.005 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B26"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Hazzalin 
       </surname> 
       <given-names>
         CA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mahadevan 
       </surname> 
       <given-names>
         LC 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       MAPK-regulated transcription: a continuously variable gene switch? 
     </article-h3> 
     <source />Nat Rev Mol Cell Biol 
     <year>
       2002 
     </year> 
     <volume>
       3 
     </volume> 
     <fpage>
       30 
     </fpage> 
     <lpage>
       40 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11823796 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/nrm715 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B27"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Chang 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Steelman 
       </surname> 
       <given-names>
         LS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         JT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shelton 
       </surname> 
       <given-names>
         JG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Navolanic 
       </surname> 
       <given-names>
         PM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Blalock 
       </surname> 
       <given-names>
         WL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Franklin 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McCubrey 
       </surname> 
       <given-names>
         JA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Signal transduction mediated by the Ras/Raf/MEK/ERK pathway from cytokine receptors to transcription factors: potential targeting for therapeutic intervention. 
     </article-h3> 
     <source />Leukemia 
     <year>
       2003 
     </year> 
     <volume>
       17 
     </volume> 
     <fpage>
       1263 
     </fpage> 
     <lpage>
       1293 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12835716 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/sj.leu.2402945 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B28"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Oda 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Matsuoka 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Funahashi 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kitano 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       A comprehensive pathway map of epidermal growth factor receptor signaling. 
     </article-h3> 
     <source />Mol Syst Biol 
     <year>
       2005 
     </year> 
     <volume>
       1 
     </volume> 
     <fpage>
       2005 0010 
     </fpage> 
     <pub-id pub-id-type="pmid">
       16729045 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/msb4100014 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B29"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Lander 
       </surname> 
       <given-names>
         ES 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Array of hope. 
     </article-h3> 
     <source />Nat Genet 
     <year>
       1999 
     </year> 
     <volume>
       21 
     </volume> 
     <fpage>
       3 
     </fpage> 
     <lpage>
       4 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9915492 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/4427 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B30"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Scherf 
       </surname> 
       <given-names>
         U 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ross 
       </surname> 
       <given-names>
         DT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Waltham 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Smith 
       </surname> 
       <given-names>
         LH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         JK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tanabe 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kohn 
       </surname> 
       <given-names>
         KW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Reinhold 
       </surname> 
       <given-names>
         WC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Myers 
       </surname> 
       <given-names>
         TG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Andrews 
       </surname> 
       <given-names>
         DT 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       A gene expression database for the molecular pharmacology of cancer. 
     </article-h3> 
     <source />Nat Genet 
     <year>
       2000 
     </year> 
     <volume>
       24 
     </volume> 
     <fpage>
       236 
     </fpage> 
     <lpage>
       244 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10700175 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/73439 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B31"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Wang 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Huang 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shou 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Su 
       </surname> 
       <given-names>
         EW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Onyia 
       </surname> 
       <given-names>
         JE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liao 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Li 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Comparative analysis and integrative classification of NCI60 cell lines and primary tumors using gene expression profiling data. 
     </article-h3> 
     <source />BMC Genomics 
     <year>
       2006 
     </year> 
     <volume>
       7 
     </volume> 
     <fpage>
       166 
     </fpage> 
     <pub-id pub-id-type="pmid">
       16817967 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1186/1471-2164-7-166 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B32"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ross 
       </surname> 
       <given-names>
         DT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Scherf 
       </surname> 
       <given-names>
         U 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Eisen 
       </surname> 
       <given-names>
         MB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Perou 
       </surname> 
       <given-names>
         CM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rees 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Spellman 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Iyer 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jeffrey 
       </surname> 
       <given-names>
         SS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Van de Rijn 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Waltham 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       Systematic variation in gene expression patterns in human cancer cell lines. 
     </article-h3> 
     <source />Nat Genet 
     <year>
       2000 
     </year> 
     <volume>
       24 
     </volume> 
     <fpage>
       227 
     </fpage> 
     <lpage>
       235 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10700174 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/73432 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B33"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Nesterova 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yokozaki 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McDuffie 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cho-Chung 
       </surname> 
       <given-names>
         YS 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Overexpression of RII beta regulatory subunit of protein kinase A in human colon carcinoma cell induces growth arrest and phenotypic changes that are abolished by site-directed mutation of RII beta. 
     </article-h3> 
     <source />Eur J Biochem 
     <year>
       1996 
     </year> 
     <volume>
       235 
     </volume> 
     <fpage>
       486 
     </fpage> 
     <lpage>
       494 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8654392 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1111/j.1432-1033.1996.00486.x 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B34"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Guillemin 
       </surname> 
       <given-names>
         MC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Raffoux 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vitoux 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kogan 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Soilihi 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lallemand-Breitenbach 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zhu 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Janin 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Daniel 
       </surname> 
       <given-names>
         MT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gourmel 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       In vivo activation of cAMP signaling induces growth arrest and differentiation in acute promyelocytic leukemia. 
     </article-h3> 
     <source />J Exp Med 
     <year>
       2002 
     </year> 
     <volume>
       196 
     </volume> 
     <fpage>
       1373 
     </fpage> 
     <lpage>
       1380 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12438428 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1084/jem.20021129 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B35"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Lerner 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kim 
       </surname> 
       <given-names>
         DH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The cAMP signaling pathway as a therapeutic target in lymphoid malignancies. 
     </article-h3> 
     <source />Leuk Lymphoma 
     <year>
       2000 
     </year> 
     <volume>
       37 
     </volume> 
     <fpage>
       39 
     </fpage> 
     <lpage>
       51 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10721768 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B36"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Mantovani 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bondioni 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lania 
       </surname> 
       <given-names>
         AG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rodolfo 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Peverelli 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Polentarutti 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Veliz Rodriguez 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ferrero 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bosari 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Beck-Peccoz 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       High expression of PKA regulatory subunit 1A protein is related to proliferation of human melanoma cells. 
     </article-h3> 
     <source />Oncogene 
     <year>
       2008 
     </year> 
     <volume>
       27 
     </volume> 
     <fpage>
       1834 
     </fpage> 
     <lpage>
       1843 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17906691 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/sj.onc.1210831 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B37"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Solberg 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sandberg 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Natarajan 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Torjesen 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hansson 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jahnsen 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tasken 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The human gene for the regulatory subunit RI alpha of cyclic adenosine 3', 5'-monophosphate-dependent protein kinase: two distinct promoters provide differential regulation of alternately spliced messenger ribonucleic acids. 
     </article-h3> 
     <source />Endocrinology 
     <year>
       1997 
     </year> 
     <volume>
       138 
     </volume> 
     <fpage>
       169 
     </fpage> 
     <lpage>
       181 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8977401 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1210/en.138.1.169 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B38"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Dahle 
       </surname> 
       <given-names>
         MK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Knutsen 
       </surname> 
       <given-names>
         HK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tasken 
       </surname> 
       <given-names>
         KA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pilz 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tasken 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Cyclic AMP regulates expression of the RI alpha subunit of cAMP-dependent protein kinase through an alternatively spliced 5' UTR. 
     </article-h3> 
     <source />Eur J Biochem 
     <year>
       2001 
     </year> 
     <volume>
       268 
     </volume> 
     <fpage>
       5920 
     </fpage> 
     <lpage>
       5929 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11722580 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1046/j.0014-2956.2001.02542.x 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B39"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Dahle 
       </surname> 
       <given-names>
         MK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gronning 
       </surname> 
       <given-names>
         LM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cederberg 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Blomhoff 
       </surname> 
       <given-names>
         HK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Miura 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Enerback 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tasken 
       </surname> 
       <given-names>
         KA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tasken 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Mechanisms of FOXC2- and FOXD1-mediated regulation of the RI alpha subunit of cAMP-dependent protein kinase include release of transcriptional repression and activation by protein kinase B alpha and cAMP. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       2002 
     </year> 
     <volume>
       277 
     </volume> 
     <fpage>
       22902 
     </fpage> 
     <lpage>
       22908 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11943768 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.M200131200 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B40"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Clegg 
       </surname> 
       <given-names>
         CH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Haugen 
       </surname> 
       <given-names>
         HS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Boring 
       </surname> 
       <given-names>
         LF 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Promoter sequences in the RI beta subunit gene of cAMP-dependent protein kinase required for transgene expression in mouse brain. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       1996 
     </year> 
     <volume>
       271 
     </volume> 
     <fpage>
       1638 
     </fpage> 
     <lpage>
       1644 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8576164 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.271.3.1638 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B41"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Clegg 
       </surname> 
       <given-names>
         CH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Koeiman 
       </surname> 
       <given-names>
         NR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jenkins 
       </surname> 
       <given-names>
         NA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gilbert 
       </surname> 
       <given-names>
         DJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Copeland 
       </surname> 
       <given-names>
         NG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Neubauer 
       </surname> 
       <given-names>
         MG 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Structural features of the murine gene encoding the RI beta subunit of cAMP-dependent protein kinase. 
     </article-h3> 
     <source />Mol Cell Neurosci 
     <year>
       1994 
     </year> 
     <volume>
       5 
     </volume> 
     <fpage>
       153 
     </fpage> 
     <lpage>
       164 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8032683 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1006/mcne.1994.1017 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B42"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Dahle 
       </surname> 
       <given-names>
         MK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tasken 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tasken 
       </surname> 
       <given-names>
         KA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       USF2 inhibits C/EBP-mediated transcriptional regulation of the RIIbeta subunit of cAMP-dependent protein kinase. 
     </article-h3> 
     <source />BMC Mol Biol 
     <year>
       2002 
     </year> 
     <volume>
       3 
     </volume> 
     <fpage>
       10 
     </fpage> 
     <pub-id pub-id-type="pmid">
       12086590 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1186/1471-2199-3-10 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B43"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Knutsen 
       </surname> 
       <given-names>
         HK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tasken 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Eskild 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Richards 
       </surname> 
       <given-names>
         JS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kurten 
       </surname> 
       <given-names>
         RC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Torjesen 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jahnsen 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hansson 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Guerin 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tasken 
       </surname> 
       <given-names>
         KA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Characterization of the 5'-flanking region of the gene for the cAMP-inducible protein kinase A subunit, RIIbeta, in Sertoli cells. 
     </article-h3> 
     <source />Mol Cell Endocrinol 
     <year>
       1997 
     </year> 
     <volume>
       129 
     </volume> 
     <fpage>
       101 
     </fpage> 
     <lpage>
       114 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9175634 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/S0303-7207(97)04045-8 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B44"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Singh 
       </surname> 
       <given-names>
         IS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Luo 
       </surname> 
       <given-names>
         Z 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kozlowski 
       </surname> 
       <given-names>
         MT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Erlichman 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Association of USF and c-Myc with a helix-loop-helix-consensus motif in the core promoter of the murine type II beta regulatory subunit gene of cyclic adenosine 3', 5'-monophosphate-dependent protein kinase. 
     </article-h3> 
     <source />Mol Endocrinol 
     <year>
       1994 
     </year> 
     <volume>
       8 
     </volume> 
     <fpage>
       1163 
     </fpage> 
     <lpage>
       1174 
     </lpage> 
     <pub-id pub-id-type="pmid">
       7838149 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1210/me.8.9.1163 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B45"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Barradeau 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Imaizumi-Scherrer 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Weiss 
       </surname> 
       <given-names>
         MC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Faust 
       </surname> 
       <given-names>
         DM 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Muscle-regulated expression and determinants for neuromuscular junctional localization of the mouse RIalpha regulatory subunit of cAMP-dependent protein kinase. 
     </article-h3> 
     <source />Proc Natl Acad Sci U S A 
     <year>
       2001 
     </year> 
     <volume>
       98 
     </volume> 
     <fpage>
       5037 
     </fpage> 
     <lpage>
       5042 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11296260 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1073/pnas.081393598 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B46"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Fernandez 
       </surname> 
       <given-names>
         PC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Frank 
       </surname> 
       <given-names>
         SR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wang 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Schroeder 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liu 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Greene 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cocito 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Amati 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Genomic targets of the human c-Myc protein. 
     </article-h3> 
     <source />Genes Dev 
     <year>
       2003 
     </year> 
     <volume>
       17 
     </volume> 
     <fpage>
       1115 
     </fpage> 
     <lpage>
       1129 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12695333 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1101/gad.1067003 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B47"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Li 
       </surname> 
       <given-names>
         Z 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Van Calcar 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Qu 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cavenee 
       </surname> 
       <given-names>
         WK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zhang 
       </surname> 
       <given-names>
         MQ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ren 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       A global transcriptional regulatory role for c-Myc in Burkitt's lymphoma cells. 
     </article-h3> 
     <source />Proc Natl Acad Sci U S A 
     <year>
       2003 
     </year> 
     <volume>
       100 
     </volume> 
     <fpage>
       8164 
     </fpage> 
     <lpage>
       8169 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12808131 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1073/pnas.1332764100 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B48"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Streb 
       </surname> 
       <given-names>
         JW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Miano 
       </surname> 
       <given-names>
         JM 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       AKAP12alpha, an atypical serum response factor-dependent target gene. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       2005 
     </year> 
     <volume>
       280 
     </volume> 
     <fpage>
       4125 
     </fpage> 
     <lpage>
       4134 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15590635 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.M412466200 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B49"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Mao 
       </surname> 
       <given-names>
         DY 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Watson 
       </surname> 
       <given-names>
         JD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yan 
       </surname> 
       <given-names>
         PS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Barsyte-Lovejoy 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Khosravi 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wong 
       </surname> 
       <given-names>
         WW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Farnham 
       </surname> 
       <given-names>
         PJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Huang 
       </surname> 
       <given-names>
         TH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Penn 
       </surname> 
       <given-names>
         LZ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Analysis of Myc bound loci identified by CpG island arrays shows that Max is essential for Myc-dependent repression. 
     </article-h3> 
     <source />Curr Biol 
     <year>
       2003 
     </year> 
     <volume>
       13 
     </volume> 
     <fpage>
       882 
     </fpage> 
     <lpage>
       886 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12747840 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/S0960-9822(03)00297-5 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B50"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         D'Sa 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tolbert 
       </surname> 
       <given-names>
         LM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Conti 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Duman 
       </surname> 
       <given-names>
         RS 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Regulation of cAMP-specific phosphodiesterases type 4B and 4D (PDE4) splice variants by cAMP signaling in primary cortical neurons. 
     </article-h3> 
     <source />J Neurochem 
     <year>
       2002 
     </year> 
     <volume>
       81 
     </volume> 
     <fpage>
       745 
     </fpage> 
     <lpage>
       757 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12065634 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1046/j.1471-4159.2002.00878.x 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B51"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Lin 
       </surname> 
       <given-names>
         CS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chow 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lau 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tu 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lue 
       </surname> 
       <given-names>
         TF 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Identification and regulation of human PDE5A gene promoter. 
     </article-h3> 
     <source />Biochem Biophys Res Commun 
     <year>
       2001 
     </year> 
     <volume>
       280 
     </volume> 
     <fpage>
       684 
     </fpage> 
     <lpage>
       692 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11162575 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1006/bbrc.2000.4220 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B52"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Lin 
       </surname> 
       <given-names>
         CS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chow 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lau 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tu 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lue 
       </surname> 
       <given-names>
         TF 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Regulation of human PDE5A2 intronic promoter by cAMP and cGMP: identification of a critical Sp1-binding site. 
     </article-h3> 
     <source />Biochem Biophys Res Commun 
     <year>
       2001 
     </year> 
     <volume>
       280 
     </volume> 
     <fpage>
       693 
     </fpage> 
     <lpage>
       699 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11162576 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1006/bbrc.2000.4221 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B53"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Mohamed 
       </surname> 
       <given-names>
         MK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Taylor 
       </surname> 
       <given-names>
         RE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Feinstein 
       </surname> 
       <given-names>
         DS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Huang 
       </surname> 
       <given-names>
         X 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pittler 
       </surname> 
       <given-names>
         SJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Structure and upstream region characterization of the human gene encoding rod photoreceptor cGMP phosphodiesterase alpha-subunit. 
     </article-h3> 
     <source />J Mol Neurosci 
     <year>
       1998 
     </year> 
     <volume>
       10 
     </volume> 
     <fpage>
       235 
     </fpage> 
     <lpage>
       250 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9770645 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1007/BF02761777 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B54"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Lerner 
       </surname> 
       <given-names>
         LE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gribanova 
       </surname> 
       <given-names>
         YE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Whitaker 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Knox 
       </surname> 
       <given-names>
         BE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Farber 
       </surname> 
       <given-names>
         DB 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The rod cGMP-phosphodiesterase beta-subunit promoter is a specific target for Sp4 and is not activated by other Sp proteins or CRX. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       2002 
     </year> 
     <volume>
       277 
     </volume> 
     <fpage>
       25877 
     </fpage> 
     <lpage>
       25883 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11943774 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.M201407200 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B55"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Lerner 
       </surname> 
       <given-names>
         LE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gribanova 
       </surname> 
       <given-names>
         YE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ji 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Knox 
       </surname> 
       <given-names>
         BE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Farber 
       </surname> 
       <given-names>
         DB 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Nrl and Sp nuclear proteins mediate transcription of rod-specific cGMP-phosphodiesterase beta-subunit gene: involvement of multiple response elements. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       2001 
     </year> 
     <volume>
       276 
     </volume> 
     <fpage>
       34999 
     </fpage> 
     <lpage>
       35007 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11438531 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.M103301200 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B56"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Torras-Llort 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Azorin 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Functional characterization of the human phosphodiesterase 7A1 promoter. 
     </article-h3> 
     <source />Biochem J 
     <year>
       2003 
     </year> 
     <volume>
       373 
     </volume> 
     <fpage>
       835 
     </fpage> 
     <lpage>
       843 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12737631 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1042/BJ20021829 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B57"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Delfino 
       </surname> 
       <given-names>
         FJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Walker 
       </surname> 
       <given-names>
         WH 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       NF-kappaB induces cAMP-response element-binding protein gene transcription in sertoli cells. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       1999 
     </year> 
     <volume>
       274 
     </volume> 
     <fpage>
       35607 
     </fpage> 
     <lpage>
       35613 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10585438 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.274.50.35607 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B58"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Shell 
       </surname> 
       <given-names>
         SA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fix 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Olejniczak 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gram-Humphrey 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Walker 
       </surname> 
       <given-names>
         WH 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Regulation of cyclic adenosine 3',5'-monophosphate response element binding protein (CREB) expression by Sp1 in the mammalian testis. 
     </article-h3> 
     <source />Biol Reprod 
     <year>
       2002 
     </year> 
     <volume>
       66 
     </volume> 
     <fpage>
       659 
     </fpage> 
     <lpage>
       666 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11870072 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1095/biolreprod66.3.659 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B59"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Mayr 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Montminy 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Transcriptional regulation by the phosphorylation-dependent factor CREB. 
     </article-h3> 
     <source />Nat Rev Mol Cell Biol 
     <year>
       2001 
     </year> 
     <volume>
       2 
     </volume> 
     <fpage>
       599 
     </fpage> 
     <lpage>
       609 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11483993 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/35085068 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B60"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Impey 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McCorkle 
       </surname> 
       <given-names>
         SR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cha-Molstad 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dwyer 
       </surname> 
       <given-names>
         JM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yochum 
       </surname> 
       <given-names>
         GS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Boss 
       </surname> 
       <given-names>
         JM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McWeeney 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dunn 
       </surname> 
       <given-names>
         JJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mandel 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Goodman 
       </surname> 
       <given-names>
         RH 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Defining the CREB regulon: a genome-wide analysis of transcription factor regulatory regions. 
     </article-h3> 
     <source />Cell 
     <year>
       2004 
     </year> 
     <volume>
       119 
     </volume> 
     <fpage>
       1041 
     </fpage> 
     <lpage>
       1054 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15620361 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B61"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Zhang 
       </surname> 
       <given-names>
         X 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Odom 
       </surname> 
       <given-names>
         DT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Koo 
       </surname> 
       <given-names>
         SH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Conkright 
       </surname> 
       <given-names>
         MD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Canettieri 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Best 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chen 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jenner 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Herbolsheimer 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jacobsen 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       Genome-wide analysis of cAMP-response element binding protein occupancy, phosphorylation, and target gene activation in human tissues. 
     </article-h3> 
     <source />Proc Natl Acad Sci U S A 
     <year>
       2005 
     </year> 
     <volume>
       102 
     </volume> 
     <fpage>
       4459 
     </fpage> 
     <lpage>
       4464 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15753290 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1073/pnas.0501076102 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B62"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Chao 
       </surname> 
       <given-names>
         JR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ni 
       </surname> 
       <given-names>
         YG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bolanos 
       </surname> 
       <given-names>
         CA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rahman 
       </surname> 
       <given-names>
         Z 
       </given-names> 
      </name> 
      <name> 
       <surname>
         DiLeone 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nestler 
       </surname> 
       <given-names>
         EJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Characterization of the mouse adenylyl cyclase type VIII gene promoter: regulation by cAMP and CREB. 
     </article-h3> 
     <source />Eur J Neurosci 
     <year>
       2002 
     </year> 
     <volume>
       16 
     </volume> 
     <fpage>
       1284 
     </fpage> 
     <lpage>
       1294 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12405989 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1046/j.1460-9568.2002.02186.x 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B63"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Wang 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Deng 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bugaj-Gaweda 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kwan 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gunwaldsen 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Leonard 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Xin 
       </surname> 
       <given-names>
         X 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hu 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Unterbeck 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         De Vivo 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Cloning and characterization of novel PDE4D isoforms PDE4D6 and PDE4D7. 
     </article-h3> 
     <source />Cell Signal 
     <year>
       2003 
     </year> 
     <volume>
       15 
     </volume> 
     <fpage>
       883 
     </fpage> 
     <lpage>
       891 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12834813 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/S0898-6568(03)00042-1 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B64"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Walker 
       </surname> 
       <given-names>
         WH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Habener 
       </surname> 
       <given-names>
         JF 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Role of transcription factors CREB and CREM in cAMP-regulated transcription during spermatogenesis. 
     </article-h3> 
     <source />Trends Endocrinol Metab 
     <year>
       1996 
     </year> 
     <volume>
       7 
     </volume> 
     <fpage>
       133 
     </fpage> 
     <lpage>
       138 
     </lpage> 
     <pub-id pub-id-type="pmid">
       18406739 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/1043-2760(96)00035-5 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B65"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Meyer 
       </surname> 
       <given-names>
         TE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Waeber 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lin 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Beckmann 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Habener 
       </surname> 
       <given-names>
         JF 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The promoter of the gene encoding 3',5'-cyclic adenosine monophosphate (cAMP) response element binding protein contains cAMP response elements: evidence for positive autoregulation of gene transcription. 
     </article-h3> 
     <source />Endocrinology 
     <year>
       1993 
     </year> 
     <volume>
       132 
     </volume> 
     <fpage>
       770 
     </fpage> 
     <lpage>
       780 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8381074 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1210/en.132.2.770 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B66"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Coven 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ni 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Widnell 
       </surname> 
       <given-names>
         KL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chen 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Walker 
       </surname> 
       <given-names>
         WH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Habener 
       </surname> 
       <given-names>
         JF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nestler 
       </surname> 
       <given-names>
         EJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Cell type-specific regulation of CREB gene expression: mutational analysis of CREB promoter activity. 
     </article-h3> 
     <source />J Neurochem 
     <year>
       1998 
     </year> 
     <volume>
       71 
     </volume> 
     <fpage>
       1865 
     </fpage> 
     <lpage>
       1874 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9798910 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B67"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Black 
       </surname> 
       <given-names>
         AR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Black 
       </surname> 
       <given-names>
         JD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Azizkhan-Clifford 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Sp1 and kruppel-like factor family of transcription factors in cell growth regulation and cancer. 
     </article-h3> 
     <source />J Cell Physiol 
     <year>
       2001 
     </year> 
     <volume>
       188 
     </volume> 
     <fpage>
       143 
     </fpage> 
     <lpage>
       160 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11424081 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1002/jcp.1111 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B68"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Merchant 
       </surname> 
       <given-names>
         JL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Du 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Todisco 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Sp1 phosphorylation by Erk 2 stimulates DNA binding. 
     </article-h3> 
     <source />Biochem Biophys Res Commun 
     <year>
       1999 
     </year> 
     <volume>
       254 
     </volume> 
     <fpage>
       454 
     </fpage> 
     <lpage>
       461 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9918860 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1006/bbrc.1998.9964 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B69"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Milde-Langosch 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The Fos family of transcription factors and their role in tumourigenesis. 
     </article-h3> 
     <source />Eur J Cancer 
     <year>
       2005 
     </year> 
     <volume>
       41 
     </volume> 
     <fpage>
       2449 
     </fpage> 
     <lpage>
       2461 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16199154 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/j.ejca.2005.08.008 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B70"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Milde-Langosch 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Roder 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Andritzky 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Aslan 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hemminger 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Brinkmann 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bamberger 
       </surname> 
       <given-names>
         CM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Loning 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bamberger 
       </surname> 
       <given-names>
         AM 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The role of the AP-1 transcription factors c-Fos, FosB, Fra-1 and Fra-2 in the invasion process of mammary carcinomas. 
     </article-h3> 
     <source />Breast Cancer Res Treat 
     <year>
       2004 
     </year> 
     <volume>
       86 
     </volume> 
     <fpage>
       139 
     </fpage> 
     <lpage>
       152 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15319566 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1023/B:BREA.0000032982.49024.71 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B71"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Maurer 
       </surname> 
       <given-names>
         GD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Leupold 
       </surname> 
       <given-names>
         JH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Schewe 
       </surname> 
       <given-names>
         DM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Biller 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kates 
       </surname> 
       <given-names>
         RE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hornung 
       </surname> 
       <given-names>
         HM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lau-Werner 
       </surname> 
       <given-names>
         U 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Post 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Allgayer 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Analysis of specific transcriptional regulators as early predictors of independent prognostic relevance in resected colorectal cancer. 
     </article-h3> 
     <source />Clin Cancer Res 
     <year>
       2007 
     </year> 
     <volume>
       13 
     </volume> 
     <fpage>
       1123 
     </fpage> 
     <lpage>
       1132 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17317820 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1158/1078-0432.CCR-06-1668 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B72"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Pore 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liu 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shu 
       </surname> 
       <given-names>
         HK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Li 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Haas-Kogan 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stokoe 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Milanini-Mongiat 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pages 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         O'Rourke 
       </surname> 
       <given-names>
         DM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bernhard 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       Sp1 is involved in Akt-mediated induction of VEGF expression through an HIF-1-independent mechanism. 
     </article-h3> 
     <source />Mol Biol Cell 
     <year>
       2004 
     </year> 
     <volume>
       15 
     </volume> 
     <fpage>
       4841 
     </fpage> 
     <lpage>
       4853 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15342781 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1091/mbc.E04-05-0374 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B73"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Milanini-Mongiat 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pouyssegur 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pages 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Identification of two Sp1 phosphorylation sites for p42/p44 mitogen-activated protein kinases: their implication in vascular endothelial growth factor gene transcription. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       2002 
     </year> 
     <volume>
       277 
     </volume> 
     <fpage>
       20631 
     </fpage> 
     <lpage>
       20639 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11904305 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.M201753200 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B74"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Foulds 
       </surname> 
       <given-names>
         CE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nelson 
       </surname> 
       <given-names>
         ML 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Blaszczak 
       </surname> 
       <given-names>
         AG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Graves 
       </surname> 
       <given-names>
         BJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Ras/mitogen-activated protein kinase signaling activates Ets-1 and Ets-2 by CBP/p300 recruitment. 
     </article-h3> 
     <source />Mol Cell Biol 
     <year>
       2004 
     </year> 
     <volume>
       24 
     </volume> 
     <fpage>
       10954 
     </fpage> 
     <lpage>
       10964 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15572696 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1128/MCB.24.24.10954-10964.2004 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B75"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Yang 
       </surname> 
       <given-names>
         BS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hauser 
       </surname> 
       <given-names>
         CA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Henkel 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Colman 
       </surname> 
       <given-names>
         MS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Van Beveren 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stacey 
       </surname> 
       <given-names>
         KJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hume 
       </surname> 
       <given-names>
         DA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Maki 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ostrowski 
       </surname> 
       <given-names>
         MC 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Ras-mediated phosphorylation of a conserved threonine residue enhances the transactivation activities of c-Ets1 and c-Ets2. 
     </article-h3> 
     <source />Mol Cell Biol 
     <year>
       1996 
     </year> 
     <volume>
       16 
     </volume> 
     <fpage>
       538 
     </fpage> 
     <lpage>
       547 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8552081 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B76"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         KW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kim 
       </surname> 
       <given-names>
         MS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kang 
       </surname> 
       <given-names>
         NJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kim 
       </surname> 
       <given-names>
         DH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Surh 
       </surname> 
       <given-names>
         YJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         HJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Moon 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       H-Ras selectively up-regulates MMP-9 and COX-2 through activation of ERK1/2 and NF-kappaB: an implication for invasive phenotype in rat liver epithelial cells. 
     </article-h3> 
     <source />Int J Cancer 
     <year>
       2006 
     </year> 
     <volume>
       119 
     </volume> 
     <fpage>
       1767 
     </fpage> 
     <lpage>
       1775 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16721810 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1002/ijc.22056 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B77"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Kim 
       </surname> 
       <given-names>
         BY 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gaynor 
       </surname> 
       <given-names>
         RB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Song 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dritschilo 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jung 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Constitutive activation of NF-kappaB in Ki-ras-transformed prostate epithelial cells. 
     </article-h3> 
     <source />Oncogene 
     <year>
       2002 
     </year> 
     <volume>
       21 
     </volume> 
     <fpage>
       4490 
     </fpage> 
     <lpage>
       4497 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12085227 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/sj.onc.1205547 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B78"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Finco 
       </surname> 
       <given-names>
         TS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Baldwin 
       </surname> 
       <given-names>
         AS 
       </given-names> 
       <suffix>
         Jr. 
       </suffix> 
      </name> 
     </person-group> 
     <article-h3>
       Kappa B site-dependent induction of gene expression by diverse inducers of nuclear factor kappa B requires Raf-1. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       1993 
     </year> 
     <volume>
       268 
     </volume> 
     <fpage>
       17676 
     </fpage> 
     <lpage>
       17679 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8349650 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B79"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Wong 
       </surname> 
       <given-names>
         WK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ou 
       </surname> 
       <given-names>
         XM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chen 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shih 
       </surname> 
       <given-names>
         JC 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Activation of human monoamine oxidase B gene expression by a protein kinase C MAPK signal transduction pathway involves c-Jun and Egr-1. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       2002 
     </year> 
     <volume>
       277 
     </volume> 
     <fpage>
       22222 
     </fpage> 
     <lpage>
       22230 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11956220 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.M202844200 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B80"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Rim 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Qureshi 
       </surname> 
       <given-names>
         SA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gius 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nho 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sukhatme 
       </surname> 
       <given-names>
         VP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Foster 
       </surname> 
       <given-names>
         DA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Evidence that activation of the Egr-1 promoter by v-Raf involves serum response elements. 
     </article-h3> 
     <source />Oncogene 
     <year>
       1992 
     </year> 
     <volume>
       7 
     </volume> 
     <fpage>
       2065 
     </fpage> 
     <lpage>
       2068 
     </lpage> 
     <pub-id pub-id-type="pmid">
       1408148 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B81"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Hodge 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liao 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stofega 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Guan 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Carter-Su 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Schwartz 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Growth hormone stimulates phosphorylation and activation of elk-1 and expression of c-fos, egr-1, and junB through activation of extracellular signal-regulated kinases 1 and 2. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       1998 
     </year> 
     <volume>
       273 
     </volume> 
     <fpage>
       31327 
     </fpage> 
     <lpage>
       31336 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9813041 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.273.47.31327 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B82"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Guha 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         O'Connell 
       </surname> 
       <given-names>
         MA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pawlinski 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hollis 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McGovern 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yan 
       </surname> 
       <given-names>
         SF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stern 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mackman 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Lipopolysaccharide activation of the MEK-ERK1/2 pathway in human monocytic cells mediates tissue factor and tumor necrosis factor alpha expression by inducing Elk-1 phosphorylation and Egr-1 expression. 
     </article-h3> 
     <source />Blood 
     <year>
       2001 
     </year> 
     <volume>
       98 
     </volume> 
     <fpage>
       1429 
     </fpage> 
     <lpage>
       1439 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11520792 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1182/blood.V98.5.1429 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B83"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Sears 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nuckolls 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Haura 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Taya 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tamai 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nevins 
       </surname> 
       <given-names>
         JR 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Multiple Ras-dependent phosphorylation pathways regulate Myc protein stability. 
     </article-h3> 
     <source />Genes Dev 
     <year>
       2000 
     </year> 
     <volume>
       14 
     </volume> 
     <fpage>
       2501 
     </fpage> 
     <lpage>
       2514 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11018017 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1101/gad.836800 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B84"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Sears 
       </surname> 
       <given-names>
         RC 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The life cycle of C-myc: from synthesis to degradation. 
     </article-h3> 
     <source />Cell Cycle 
     <year>
       2004 
     </year> 
     <volume>
       3 
     </volume> 
     <fpage>
       1133 
     </fpage> 
     <lpage>
       1137 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15467447 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B85"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Mo 
       </surname> 
       <given-names>
         X 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kowenz-Leutz 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Xu 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Leutz 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Ras induces mediator complex exchange on C/EBP beta. 
     </article-h3> 
     <source />Mol Cell 
     <year>
       2004 
     </year> 
     <volume>
       13 
     </volume> 
     <fpage>
       241 
     </fpage> 
     <lpage>
       250 
     </lpage> 
     <pub-id pub-id-type="pmid">
       14759369 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/S1097-2765(03)00521-5 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B86"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Nebl 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mermod 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cato 
       </surname> 
       <given-names>
         AC 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Post-transcriptional down-regulation of expression of transcription factor NF1 by Ha-ras oncogene. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       1994 
     </year> 
     <volume>
       269 
     </volume> 
     <fpage>
       7371 
     </fpage> 
     <lpage>
       7378 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8125955 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B87"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Cho-Chung 
       </surname> 
       <given-names>
         YS 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Role of cyclic AMP receptor proteins in growth, differentiation, and suppression of malignancy: new approaches to therapy. 
     </article-h3> 
     <source />Cancer Res 
     <year>
       1990 
     </year> 
     <volume>
       50 
     </volume> 
     <fpage>
       7093 
     </fpage> 
     <lpage>
       7100 
     </lpage> 
     <pub-id pub-id-type="pmid">
       2224844 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B88"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Cho-Chung 
       </surname> 
       <given-names>
         YS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pepe 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Clair 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Budillon 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nesterova 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       cAMP-dependent protein kinase: role in normal and malignant growth. 
     </article-h3> 
     <source />Crit Rev Oncol Hematol 
     <year>
       1995 
     </year> 
     <volume>
       21 
     </volume> 
     <fpage>
       33 
     </fpage> 
     <lpage>
       61 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8822496 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/1040-8428(94)00166-9 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B89"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Spina 
       </surname> 
       <given-names>
         AM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chiosi 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Naviglio 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Valente 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Marchese 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Furgi 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Metafora 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Illiano 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       ras oncogene-induced transformation of a rat seminal vesicle epithelial cell line produces a marked increase of adenylate cyclase and protein kinase C activities. 
     </article-h3> 
     <source />FEBS Lett 
     <year>
       1993 
     </year> 
     <volume>
       331 
     </volume> 
     <fpage>
       150 
     </fpage> 
     <lpage>
       154 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8405395 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/0014-5793(93)80315-L 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B90"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Franks 
       </surname> 
       <given-names>
         DJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Whitfield 
       </surname> 
       <given-names>
         JF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Durkin 
       </surname> 
       <given-names>
         JP 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       A viral K-RAS protein increases the stimulability of adenylate cyclase by cholera toxin in NRK cells. 
     </article-h3> 
     <source />Biochem Biophys Res Commun 
     <year>
       1987 
     </year> 
     <volume>
       147 
     </volume> 
     <fpage>
       596 
     </fpage> 
     <lpage>
       601 
     </lpage> 
     <pub-id pub-id-type="pmid">
       2443137 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/0006-291X(87)90972-7 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B91"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Franks 
       </surname> 
       <given-names>
         DJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Whitfield 
       </surname> 
       <given-names>
         JF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Durkin 
       </surname> 
       <given-names>
         JP 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Viral p21 Ki-RAS protein: a potent intracellular mitogen that stimulates adenylate cyclase activity in early G1 phase of cultured rat cells. 
     </article-h3> 
     <source />J Cell Biochem 
     <year>
       1987 
     </year> 
     <volume>
       33 
     </volume> 
     <fpage>
       87 
     </fpage> 
     <lpage>
       94 
     </lpage> 
     <pub-id pub-id-type="pmid">
       3553214 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1002/jcb.240330203 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B92"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Beckner 
       </surname> 
       <given-names>
         SK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hattori 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Shih 
       </surname> 
       <given-names>
         TY 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The ras oncogene product p21 is not a regulatory component of adenylate cyclase. 
     </article-h3> 
     <source />Nature 
     <year>
       1985 
     </year> 
     <volume>
       317 
     </volume> 
     <fpage>
       71 
     </fpage> 
     <lpage>
       72 
     </lpage> 
     <pub-id pub-id-type="pmid">
       3929144 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/317071a0 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B93"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Colletta 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Corda 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Schettini 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cirafici 
       </surname> 
       <given-names>
         AM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kohn 
       </surname> 
       <given-names>
         LD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Consiglio 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Adenylate cyclase activity of v-ras-k transformed rat epithelial thyroid cells. 
     </article-h3> 
     <source />FEBS Lett 
     <year>
       1988 
     </year> 
     <volume>
       228 
     </volume> 
     <fpage>
       37 
     </fpage> 
     <lpage>
       41 
     </lpage> 
     <pub-id pub-id-type="pmid">
       3125065 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/0014-5793(88)80579-9 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B94"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Levitzki 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rudick 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pastan 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vass 
       </surname> 
       <given-names>
         WC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lowy 
       </surname> 
       <given-names>
         DR 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Adenylate cyclase activity of NIH 3T3 cells morphologically transformed by ras genes. 
     </article-h3> 
     <source />FEBS Lett 
     <year>
       1986 
     </year> 
     <volume>
       197 
     </volume> 
     <fpage>
       134 
     </fpage> 
     <lpage>
       138 
     </lpage> 
     <pub-id pub-id-type="pmid">
       3081369 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/0014-5793(86)80313-1 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B95"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Liu 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Palmer 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jimmo 
       </surname> 
       <given-names>
         SL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tilley 
       </surname> 
       <given-names>
         DG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dunkerley 
       </surname> 
       <given-names>
         HA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pang 
       </surname> 
       <given-names>
         SC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Maurice 
       </surname> 
       <given-names>
         DH 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Expression of phosphodiesterase 4D (PDE4D) is regulated by both the cyclic AMP-dependent protein kinase and mitogen-activated protein kinase signaling pathways. A potential mechanism allowing for the coordinated regulation of PDE4D activity and expression in cells. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       2000 
     </year> 
     <volume>
       275 
     </volume> 
     <fpage>
       26615 
     </fpage> 
     <lpage>
       26624 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10851231 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.M001634200 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B96"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Houslay 
       </surname> 
       <given-names>
         MD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Baillie 
       </surname> 
       <given-names>
         GS 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       The role of ERK2 docking and phosphorylation of PDE4 cAMP phosphodiesterase isoforms in mediating cross-talk between the cAMP and ERK signalling pathways. 
     </article-h3> 
     <source />Biochem Soc Trans 
     <year>
       2003 
     </year> 
     <volume>
       31 
     </volume> 
     <fpage>
       1186 
     </fpage> 
     <lpage>
       1190 
     </lpage> 
     <pub-id pub-id-type="pmid">
       14641023 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1042/BST0311186 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B97"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Salas 
       </surname> 
       <given-names>
         TR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Reddy 
       </surname> 
       <given-names>
         SA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Clifford 
       </surname> 
       <given-names>
         JL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Davis 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kikuchi 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lippman 
       </surname> 
       <given-names>
         SM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Menter 
       </surname> 
       <given-names>
         DG 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Alleviating the suppression of glycogen synthase kinase-3beta by Akt leads to the phosphorylation of cAMP-response element-binding protein and its transactivation in intact cell nuclei. 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       2003 
     </year> 
     <volume>
       278 
     </volume> 
     <fpage>
       41338 
     </fpage> 
     <lpage>
       41346 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12900420 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.M302972200 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B98"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Xing 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ginty 
       </surname> 
       <given-names>
         DD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Greenberg 
       </surname> 
       <given-names>
         ME 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Coupling of the RAS-MAPK pathway to gene activation by RSK2, a growth factor-regulated CREB kinase. 
     </article-h3> 
     <source />Science 
     <year>
       1996 
     </year> 
     <volume>
       273 
     </volume> 
     <fpage>
       959 
     </fpage> 
     <lpage>
       963 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8688081 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1126/science.273.5277.959 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B99"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Piccoli 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Scacco 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bellomo 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Signorile 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Iuso 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Boffoli 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Scrima 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Capitanio 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Papa 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       cAMP controls oxygen metabolism in mammalian cells. 
     </article-h3> 
     <source />FEBS Lett 
     <year>
       2006 
     </year> 
     <volume>
       580 
     </volume> 
     <fpage>
       4539 
     </fpage> 
     <lpage>
       4543 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16870178 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/j.febslet.2006.06.085 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B100"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Raha 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Myint 
       </surname> 
       <given-names>
         AT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Johnstone 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Robinson 
       </surname> 
       <given-names>
         BH 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Control of oxygen free radical formation from mitochondrial complex I: roles for protein kinase A and pyruvate dehydrogenase kinase. 
     </article-h3> 
     <source />Free Radic Biol Med 
     <year>
       2002 
     </year> 
     <volume>
       32 
     </volume> 
     <fpage>
       421 
     </fpage> 
     <lpage>
       430 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11864782 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/S0891-5849(01)00816-4 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B101"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Warburg 
       </surname> 
       <given-names>
         O 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       On the origin of cancer cells. 
     </article-h3> 
     <source />Science 
     <year>
       1956 
     </year> 
     <volume>
       123 
     </volume> 
     <fpage>
       309 
     </fpage> 
     <lpage>
       314 
     </lpage> 
     <pub-id pub-id-type="pmid">
       13298683 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1126/science.123.3191.309 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B102"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Chiaradonna 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sacco 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Manzoni 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Giorgio 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vanoni 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Alberghina 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Ras-dependent carbon metabolism and transformation in mouse fibroblasts. 
     </article-h3> 
     <source />Oncogene 
     <year>
       2006 
     </year> 
     <volume>
       25 
     </volume> 
     <fpage>
       5391 
     </fpage> 
     <lpage>
       5404 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16607279 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/sj.onc.1209528 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B103"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Chiaradonna 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gaglio 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vanoni 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Alberghina 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Expression of transforming K-Ras oncogene affects mitochondrial function and morphology in mouse fibroblasts. 
     </article-h3> 
     <source />Biochim Biophys Acta 
     <year>
       2006 
     </year> 
     <volume>
       1757 
     </volume> 
     <fpage>
       1338 
     </fpage> 
     <lpage>
       1356 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16987493 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1016/j.bbabio.2006.08.001 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B104"> 
    <citation citation-type="book"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Alberghina 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hohmann 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Westerhoff 
       </surname> 
       <given-names>
         HV 
       </given-names> 
      </name> 
     </person-group> 
     <person-group person-group-type="editor"> 
      <name> 
       <surname>
         Alberghina L, Westerhoff HV 
       </surname> 
      </name> 
     </person-group> 
     <article-h3>
       Systems Biology: necessary developments and trends 
     </article-h3> 
     <source />Systems Biology: Definitions and Perspectives 
     <year>
       2005 
     </year> 
     <publisher-name>
       Springer-Verlag Berlin Heidelberg 
     </publisher-name> 
     <fpage>
       389 
     </fpage> 
     <lpage>
       402 
     </lpage> 
     <comment>
       [Hohmann S (Series Editor):Topics in Current Genetics, vol 13.] 
     </comment> 
    </citation> 
   </ref> 
   <ref id="B105"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Barrett 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Suzek 
       </surname> 
       <given-names>
         TO 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Troup 
       </surname> 
       <given-names>
         DB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wilhite 
       </surname> 
       <given-names>
         SE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ngau 
       </surname> 
       <given-names>
         WC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ledoux 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rudnev 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lash 
       </surname> 
       <given-names>
         AE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fujibuchi 
       </surname> 
       <given-names>
         W 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Edgar 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       NCBI GEO: mining millions of expression profiles--database and tools. 
     </article-h3> 
     <source />Nucleic Acids Res 
     <year>
       2005 
     </year> 
     <volume>
       33 
     </volume> 
     <fpage>
       D562 
     </fpage> 
     <lpage>
       566 
     </lpage> 
     <comment>
       Database issue 
     </comment> 
     <pub-id pub-id-type="pmid">
       15608262 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1093/nar/gki022 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B106"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Su 
       </surname> 
       <given-names>
         AI 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cooke 
       </surname> 
       <given-names>
         MP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ching 
       </surname> 
       <given-names>
         KA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hakak 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Walker 
       </surname> 
       <given-names>
         JR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wiltshire 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Orth 
       </surname> 
       <given-names>
         AP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vega 
       </surname> 
       <given-names>
         RG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sapinoso 
       </surname> 
       <given-names>
         LM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Moqrich 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       Large-scale analysis of the human and mouse transcriptomes. 
     </article-h3> 
     <source />Proc Natl Acad Sci U S A 
     <year>
       2002 
     </year> 
     <volume>
       99 
     </volume> 
     <fpage>
       4465 
     </fpage> 
     <lpage>
       4470 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11904358 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1073/pnas.012025199 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B107"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Wu 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dassopoulos 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cope 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Maitra 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Brant 
       </surname> 
       <given-names>
         SR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Harris 
       </surname> 
       <given-names>
         ML 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bayless 
       </surname> 
       <given-names>
         TM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Parmigiani 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chakravarti 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Genome-wide gene expression differences in Crohn's disease and ulcerative colitis from endoscopic pinch biopsies: insights into distinctive pathogenesis. 
     </article-h3> 
     <source />Inflamm Bowel Dis 
     <year>
       2007 
     </year> 
     <volume>
       13 
     </volume> 
     <fpage>
       807 
     </fpage> 
     <lpage>
       821 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17262812 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1002/ibd.20110 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B108"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Barnes 
       </surname> 
       <given-names>
         MG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Aronow 
       </surname> 
       <given-names>
         BJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Luyrink 
       </surname> 
       <given-names>
         LK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Moroldo 
       </surname> 
       <given-names>
         MB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pavlidis 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Passo 
       </surname> 
       <given-names>
         MH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Grom 
       </surname> 
       <given-names>
         AA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hirsch 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Giannini 
       </surname> 
       <given-names>
         EH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Colbert 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       Gene expression in juvenile arthritis and spondyloarthropathy: pro-angiogenic ELR+ chemokine genes relate to course of arthritis. 
     </article-h3> 
     <source />Rheumatology (Oxford) 
     <year>
       2004 
     </year> 
     <volume>
       43 
     </volume> 
     <fpage>
       973 
     </fpage> 
     <lpage>
       979 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15150433 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1093/rheumatology/keh224 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B109"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Irizarry 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hobbs 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Collin 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Beazer-Barclay 
       </surname> 
       <given-names>
         YD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Antonellis 
       </surname> 
       <given-names>
         KJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Scherf 
       </surname> 
       <given-names>
         U 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Speed 
       </surname> 
       <given-names>
         TP 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Exploration, normalization, and summaries of high density oligonucleotide array probe level data. 
     </article-h3> 
     <source />Biostatistics (Oxford, England) 
     <year>
       2003 
     </year> 
     <volume>
       4 
     </volume> 
     <fpage>
       249 
     </fpage> 
     <lpage>
       264 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12925520 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B110"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Cope 
       </surname> 
       <given-names>
         LM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Irizarry 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jaffee 
       </surname> 
       <given-names>
         HA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wu 
       </surname> 
       <given-names>
         Z 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Speed 
       </surname> 
       <given-names>
         TP 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       A benchmark for Affymetrix GeneChip expression measures. 
     </article-h3> 
     <source />Bioinformatics 
     <year>
       2004 
     </year> 
     <volume>
       20 
     </volume> 
     <fpage>
       323 
     </fpage> 
     <lpage>
       331 
     </lpage> 
     <pub-id pub-id-type="pmid">
       14960458 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1093/bioinformatics/btg410 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B111"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Allison 
       </surname> 
       <given-names>
         DB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cui 
       </surname> 
       <given-names>
         X 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Page 
       </surname> 
       <given-names>
         GP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sabripour 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Microarray data analysis: from disarray to consolidation and consensus. 
     </article-h3> 
     <source />Nature reviews 
     <year>
       2006 
     </year> 
     <volume>
       7 
     </volume> 
     <fpage>
       55 
     </fpage> 
     <lpage>
       65 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16369572 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/nrg1749 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B112"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Oudes 
       </surname> 
       <given-names>
         AJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Campbell 
       </surname> 
       <given-names>
         DS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sorensen 
       </surname> 
       <given-names>
         CM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Walashek 
       </surname> 
       <given-names>
         LS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         True 
       </surname> 
       <given-names>
         LD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liu 
       </surname> 
       <given-names>
         AY 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Transcriptomes of human prostate cells. 
     </article-h3> 
     <source />BMC Genomics 
     <year>
       2006 
     </year> 
     <volume>
       7 
     </volume> 
     <fpage>
       92 
     </fpage> 
     <pub-id pub-id-type="pmid">
       16638148 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1186/1471-2164-7-92 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B113"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Casorelli 
       </surname> 
       <given-names>
         I 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tenedini 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tagliafico 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Blasi 
       </surname> 
       <given-names>
         MF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Giuliani 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Crescenzi 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pelosi 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Testa 
       </surname> 
       <given-names>
         U 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Peschle 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mele 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <article-h3>
       Identification of a molecular signature for leukemic promyelocytes and their normal counterparts: Focus on DNA repair genes. 
     </article-h3> 
     <source />Leukemia 
     <year>
       2006 
     </year> 
     <volume>
       20 
     </volume> 
     <fpage>
       1978 
     </fpage> 
     <lpage>
       1988 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16990782 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1038/sj.leu.2404376 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B114"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Irizarry 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bolstad 
       </surname> 
       <given-names>
         BM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Collin 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cope 
       </surname> 
       <given-names>
         LM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hobbs 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Speed 
       </surname> 
       <given-names>
         TP 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Summaries of Affymetrix GeneChip probe level data. 
     </article-h3> 
     <source />Nucleic Acids Res 
     <year>
       2003 
     </year> 
     <volume>
       31 
     </volume> 
     <fpage>
       e15 
     </fpage> 
     <pub-id pub-id-type="pmid">
       12582260 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1093/nar/gng015 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B115"> 
    <citation citation-type="book"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Fisher 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
     </person-group> 
     <source />Statistical Methods for Research Workers 
     <year>
       1925 
     </year> 
     <publisher-name>
       Edinburgh: Oliver and Boyd 
     </publisher-name> 
     <pub-id pub-id-type="pmid">
       17246289 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B116"> 
    <citation citation-type="book"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Fisher 
       </surname> 
       <given-names>
         LD vBG 
       </given-names> 
      </name> 
     </person-group> 
     <source />Biostatistics: a methodology for the health sciences. 
     <year>
       1993 
     </year> 
     <publisher-name>
       New York: John Wiley &amp; Sons Inc. 
     </publisher-name> 
    </citation> 
   </ref> 
   <ref id="B117"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Kerr 
       </surname> 
       <given-names>
         MK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Martin 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Churchill 
       </surname> 
       <given-names>
         GA 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Analysis of variance for gene expression microarray data. 
     </article-h3> 
     <source />J Comput Biol 
     <year>
       2000 
     </year> 
     <volume>
       7 
     </volume> 
     <fpage>
       819 
     </fpage> 
     <lpage>
       837 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11382364 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1089/10665270050514954 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B118"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Coombes 
       </surname> 
       <given-names>
         KR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Highsmith 
       </surname> 
       <given-names>
         WE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Krogmann 
       </surname> 
       <given-names>
         TA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Baggerly 
       </surname> 
       <given-names>
         KA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stivers 
       </surname> 
       <given-names>
         DN 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Abruzzo 
       </surname> 
       <given-names>
         LV 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Identifying and quantifying sources of variation in microarray data using high-density cDNA membrane arrays. 
     </article-h3> 
     <source />J Comput Biol 
     <year>
       2002 
     </year> 
     <volume>
       9 
     </volume> 
     <fpage>
       655 
     </fpage> 
     <lpage>
       669 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12323099 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1089/106652702760277372 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B119"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Pritchard 
       </surname> 
       <given-names>
         CC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hsu 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Delrow 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nelson 
       </surname> 
       <given-names>
         PS 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Project normal: defining normal variance in mouse gene expression. 
     </article-h3> 
     <source />Proc Natl Acad Sci U S A 
     <year>
       2001 
     </year> 
     <volume>
       98 
     </volume> 
     <fpage>
       13266 
     </fpage> 
     <lpage>
       13271 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11698685 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1073/pnas.221465998 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B120"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Pavlidis 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Noble 
       </surname> 
       <given-names>
         WS 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Analysis of strain and regional variation in gene expression in mouse brain. 
     </article-h3> 
     <source />Genome Biol 
     <year>
       2001 
     </year> 
     <volume>
       2 
     </volume> 
     <fpage>
       RESEARCH0042 
     </fpage> 
     <pub-id pub-id-type="pmid">
       11597334 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1186/gb-2001-2-10-research0042 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B121"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Johnson 
       </surname> 
       <given-names>
         SC 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Hierarchical clustering schemes. 
     </article-h3> 
     <source />Psychometrika 
     <year>
       1967 
     </year> 
     <volume>
       32 
     </volume> 
     <fpage>
       241 
     </fpage> 
     <lpage>
       254 
     </lpage> 
     <pub-id pub-id-type="pmid">
       5234703 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1007/BF02289588 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B122"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Eisen 
       </surname> 
       <given-names>
         MB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Spellman 
       </surname> 
       <given-names>
         PT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Brown 
       </surname> 
       <given-names>
         PO 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Botstein 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Cluster analysis and display of genome-wide expression patterns. 
     </article-h3> 
     <source />Proc Natl Acad Sci U S A 
     <year>
       1998 
     </year> 
     <volume>
       95 
     </volume> 
     <fpage>
       14863 
     </fpage> 
     <lpage>
       14868 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9843981 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1073/pnas.95.25.14863 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B123"> 
    <citation citation-type="book"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Forbes 
       </surname> 
       <given-names>
         SA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bhamra 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bamford 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dawson 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kok 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Clements 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Menzies 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Teague 
       </surname> 
       <given-names>
         JW 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Futreal 
       </surname> 
       <given-names>
         PA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stratton 
       </surname> 
       <given-names>
         MR 
       </given-names> 
      </name> 
     </person-group> 
     <person-group person-group-type="editor"> 
      <name> 
       <surname>
         Jonathan L Haines et al 
       </surname> 
      </name> 
     </person-group> 
     <article-h3>
       The Catalogue of Somatic Mutations in Cancer (COSMIC). 
     </article-h3> 
     <source />Current protocols in human genetics / editorial board 
     <year>
       2008 
     </year> 
     <comment>
       Chapter 10: Unit 10 11. 
     </comment> 
    </citation> 
   </ref> 
   <ref id="B124"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Joliffe 
       </surname> 
       <given-names>
         IT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Morgan 
       </surname> 
       <given-names>
         BJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Principal component analysis and exploratory factor analysis. 
     </article-h3> 
     <source />Statistical methods in medical research 
     <year>
       1992 
     </year> 
     <volume>
       1 
     </volume> 
     <fpage>
       69 
     </fpage> 
     <lpage>
       95 
     </lpage> 
     <pub-id pub-id-type="pmid">
       1341653 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1177/096228029200100105 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B125"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Raychaudhuri 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Stuart 
       </surname> 
       <given-names>
         JM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Altman 
       </surname> 
       <given-names>
         RB 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Principal components analysis to summarize microarray experiments: application to sporulation time series. 
     </article-h3> 
     <source />Pacific Symposium on Biocomputing 
     <year>
       2000 
     </year> 
     <fpage>
       455 
     </fpage> 
     <lpage>
       466 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10902193 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B126"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Frech 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Danescu-Mayer 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Werner 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       A novel method to develop highly specific models for regulatory units detects a new LTR in GenBank which contains a functional promoter. 
     </article-h3> 
     <source />J Mol Biol 
     <year>
       1997 
     </year> 
     <volume>
       270 
     </volume> 
     <fpage>
       674 
     </fpage> 
     <lpage>
       687 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9245596 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1006/jmbi.1997.1140 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B127"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Cartharius 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Frech 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Grote 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Klocke 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Haltmeier 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Klingenhoff 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Frisch 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bayerlein 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Werner 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       MatInspector and beyond: promoter analysis based on transcription factor binding sites. 
     </article-h3> 
     <source />Bioinformatics 
     <year>
       2005 
     </year> 
     <volume>
       21 
     </volume> 
     <fpage>
       2933 
     </fpage> 
     <lpage>
       2942 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15860560 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1093/bioinformatics/bti473 
     </pub-id> 
    </citation> 
   </ref> 
   <ref id="B128"> 
    <citation citation-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Zhao 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Xuan 
       </surname> 
       <given-names>
         Z 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liu 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zhang 
       </surname> 
       <given-names>
         MQ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       TRED: a Transcriptional Regulatory Element Database and a platform for in silico gene regulation studies. 
     </article-h3> 
     <source />Nucleic acids research 
     <year>
       2005 
     </year> 
     <volume>
       33 
     </volume> 
     <fpage>
       D103 
     </fpage> 
     <lpage>
       107 
     </lpage> 
     <comment>
       (Database Issue) 
     </comment> 
     <pub-id pub-id-type="pmid">
       15608156 
     </pub-id> 
     <pub-id pub-id-type="doi">
       10.1093/nar/gki004 
     </pub-id> 
    </citation> 
   </ref> 
  </ref-list> 
 </back> 
</article>