<article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"> 
 <!--?properties open_acces=""?--> 
 <front> 
  <journal-meta> 
   <journal-id journal-id-type="nlm-ta">
     BMC Cancer 
   </journal-id> 
   <journal-h3-group> 
    <journal-h3>
      BMC Cancer 
    </journal-h3> 
   </journal-h3-group> 
   <issn pub-type="epub">
     1471-2407 
   </issn> 
   <publisher> 
    <publisher-name>
      BioMed Central 
    </publisher-name> 
   </publisher> 
  </journal-meta> 
  <article-meta> 
   <article-id pub-id-type="pmid">
     20233444 
   </article-id> 
   <article-id pub-id-type="pmc">
     2845115 
   </article-id> 
   <article-id pub-id-type="publisher-id">
     1471-2407-10-101 
   </article-id> 
   <article-id pub-id-type="doi">
     10.1186/1471-2407-10-101 
   </article-id> 
   <article-categories> 
    <subj-group subj-group-type="heading"> 
     <subject>
       Research Article 
     </subject> 
    </subj-group> 
   </article-categories> 
   <h3-group> 
    <article-h3>
      A rapid, sensitive, reproducible and cost-effective method for mutation profiling of colon cancer and metastatic lymph nodes 
    </article-h3> 
   </h3-group> 
   <contrib-group> 
    <contrib contrib-type="author" equal-contrib="yes" id="A1"> 
     <name> 
      <surname>
        Fumagalli 
      </surname> 
      <given-names>
        Debora 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       debora.fumagalli@gmail.com 
     </email> 
    </contrib> 
    <contrib contrib-type="author" equal-contrib="yes" id="A2"> 
     <name> 
      <surname>
        Gavin 
      </surname> 
      <given-names>
        Patrick G 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       pgavin@nsabp.org 
     </email> 
    </contrib> 
    <contrib contrib-type="author" id="A3"> 
     <name> 
      <surname>
        Taniyama 
      </surname> 
      <given-names>
        Yusuke 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       yusuketaniyama@gmail.com 
     </email> 
    </contrib> 
    <contrib contrib-type="author" id="A4"> 
     <name> 
      <surname>
        Kim 
      </surname> 
      <given-names>
        Seung-Il 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       skim@nsabp.org 
     </email> 
    </contrib> 
    <contrib contrib-type="author" id="A5"> 
     <name> 
      <surname>
        Choi 
      </surname> 
      <given-names>
        Hyun-Joo 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       chj0103@catholic.ac.Kr 
     </email> 
    </contrib> 
    <contrib contrib-type="author" id="A6"> 
     <name> 
      <surname>
        Paik 
      </surname> 
      <given-names>
        Soonmyung 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       soon.paik@nsabp.org 
     </email> 
    </contrib> 
    <contrib contrib-type="author" corresp="yes" id="A7"> 
     <name> 
      <surname>
        Pogue-Geile 
      </surname> 
      <given-names>
        Katherine L 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="I1">
       1 
     </xref> 
     <email>
       kay.pogue@nsabp.org 
     </email> 
    </contrib> 
   </contrib-group> 
   <aff id="I1"> 
    <label>1</label>Department of Pathology, National Surgical Adjuvant Breast and Bowel Project (NSABP), 1307 Federal St, Pittsburgh, PA 15212, USA 
   </aff> 
   <pub-date pub-type="collection"> 
    <year>
      2010 
    </year> 
   </pub-date> 
   <pub-date pub-type="epub"> 
    <day>
      16 
    </day> 
    <month>
      3 
    </month> 
    <year>
      2010 
    </year> 
   </pub-date> 
   <volume>
     10 
   </volume> 
   <fpage>
     101 
   </fpage> 
   <lpage>
     101 
   </lpage> 
   <history> 
    <date date-type="received"> 
     <day>
       12 
     </day> 
     <month>
       8 
     </month> 
     <year>
       2009 
     </year> 
    </date> 
    <date date-type="accepted"> 
     <day>
       16 
     </day> 
     <month>
       3 
     </month> 
     <year>
       2010 
     </year> 
    </date> 
   </history> 
   <permissions> 
    <copyright-statement>
      Copyright ©2010 Fumagalli et al; licensee BioMed Central Ltd. 
    </copyright-statement> 
    <copyright-year>
      2010 
    </copyright-year> 
    <copyright-holder>
      Fumagalli et al; licensee BioMed Central Ltd. 
    </copyright-holder> 
    <license license-type="open-access" xlink:href="http://creativecommons.org/licenses/by/2.0"> 
     <license-p>
       This is an Open Access article distributed under the terms of the Creative Commons Attribution License ( 
      <ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/2.0">
        http://creativecommons.org/licenses/by/2.0 
      </ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. 
     </license-p> 
    </license> 
   </permissions> 
   <self-uri xlink:href="http://www.biomedcentral.com/1471-2407/10/101"></self-uri> 
   <abstract>
    <h3>Abstract</h3> 
    <sec> 
     <h3>Background</h3> 
     <p>An increasing number of studies show that genetic markers can aid in refining prognostic information and predicting the benefit from systemic therapy. Our goal was to develop a high throughput, cost-effective and simple methodology for the detection of clinically relevant hot spot mutations in colon cancer.</p> 
    </sec> 
    <sec> 
     <h3>Methods</h3> 
     <p>The Maldi-Tof mass spectrometry platform and OncoCarta panel from Sequenom were used to profile 239 colon cancers and 39 metastatic lymph nodes from NSABP clinical trial C-07 utilizing routinely processed FFPET (formalin-fixed paraffin-embedded tissue).</p> 
    </sec> 
    <sec> 
     <h3>Results</h3> 
     <p>Among the 238 common hot-spot cancer mutations in 19 genes interrogated by the OncoCarta panel, mutations were detected in 7 different genes at 26 different nucleotide positions in our colon cancer samples. Twenty-four assays that detected mutations in more than 1% of the samples were reconfigured into a new multiplexed panel, termed here as ColoCarta. Mutation profiling was repeated on 32 mutant samples using ColoCarta and the results were identical to results with OncoCarta, demonstrating that this methodology was reproducible. Further evidence demonstrating the validity of the data was the fact that the mutation frequencies of the most common colon cancer mutations were similar to the COSMIC (Catalog of Somatic Mutations in Cancer) database. The frequencies were 43.5% for 
      <italic>
        KRAS 
      </italic>, 20.1% for 
      <italic>
        PIK3CA 
      </italic>, and 12.1% for 
      <italic>
        BRAF 
      </italic>. In addition, infrequent mutations in 
      <italic>
        NRAS 
      </italic>, 
      <italic>
        AKT1 
      </italic>, 
      <italic>
        ABL1 
      </italic>, and 
      <italic>
        MET 
      </italic>were detected. Mutation profiling of metastatic lymph nodes and their corresponding primary tumors showed that they were 89.7% concordant. All mutations found in the lymph nodes were also found in the corresponding primary tumors, but in 4 cases a mutation was present in the primary tumor only.</p> 
    </sec> 
    <sec> 
     <h3>Conclusions</h3> 
     <p>This study describes a high throughput technology that can be used to interrogate DNAs isolated from routinely processed FFPET and identifies the specific mutations that are common to colon cancer. The development of this technology and the ColoCarta panel may provide a mechanism for rapid screening of mutations in clinically relevant genes like 
      <italic>
        KRAS 
      </italic>, 
      <italic>
        PIK3CA 
      </italic>, and 
      <italic>
        BRAF 
      </italic>.</p> 
    </sec> 
    <sec> 
     <h3>Trial Registration</h3> 
     <p> 
      <bold>
        ClinicalTrials.gov: 
      </bold>NSABP C-07: NCT00004931</p> 
    </sec> 
   </abstract> 
  </article-meta> 
 </front> 
 <sec> 
  <h3>Background</h3> 
  <p>Recent evidence suggests that mutation profiling can assist in the prognosis and prediction for colon cancer. 
   <italic>
     KRAS 
   </italic>, 
   <italic>
     PIK3CA 
   </italic>and 
   <italic>
     BRAF 
   </italic>mutations are frequent in tumors of the colon and have been associated with poor prognosis [ 
   <xref ref-type="bibr" rid="B1">
     1 
   </xref>- 
   <xref ref-type="bibr" rid="B6">
     6 
   </xref>]. However, these results remain controversial because other studies have shown that mutations in these genes are not prognostic. A large study, a meta analysis, of 
   <italic>
     KRAS 
   </italic>mutations, found that only 
   <italic>
     KRAS 
   </italic>G12V was a bad prognostic marker; other 
   <italic>
     KRAS 
   </italic>mutations were not associated with bad prognosis [ 
   <xref ref-type="bibr" rid="B2">
     2 
   </xref>]. Evidence has also demonstrated that 
   <italic>
     KRAS 
   </italic>mutations are potential markers for prediction because tumors with 
   <italic>
     KRAS 
   </italic>mutations are significantly associated with resistance to EGFR antibody based therapies [ 
   <xref ref-type="bibr" rid="B7">
     7 
   </xref>- 
   <xref ref-type="bibr" rid="B11">
     11 
   </xref>]. Publications have reported the same phenomenon with 
   <italic>
     BRAF 
   </italic>and 
   <italic>
     PIK3CA 
   </italic>mutations, although these observations are still not well established [ 
   <xref ref-type="bibr" rid="B12">
     12 
   </xref>, 
   <xref ref-type="bibr" rid="B13">
     13 
   </xref>]. The published study suggesting that 
   <italic>
     BRAF 
   </italic>mutant tumors were resistant to EGFR therapies was a small study [ 
   <xref ref-type="bibr" rid="B13">
     13 
   </xref>]. Predictive value of 
   <italic>
     PIK3CA 
   </italic>mutations remains controversial in that other publications have shown that these mutations have no predictive value [ 
   <xref ref-type="bibr" rid="B14">
     14 
   </xref>, 
   <xref ref-type="bibr" rid="B15">
     15 
   </xref>]. These inconsistencies, together with two other factors, have limited the impact of mutation profiling for prognosis and prediction in standard care of colon cancer. A large sample size is required to establish that a gene mutation has a significant impact for patient prediction or prognosis. Another limitation is that until recently conducting such large studies with the standard sequencing technologies was too time consuming and too expensive to be practical for clinical studies.</p> 
  <p>Moreover, while the high frequency of 
   <italic>
     KRAS 
   </italic>, 
   <italic>
     BRAF 
   </italic>and 
   <italic>
     PIK3CA 
   </italic>mutations in colon cancer is well documented, other potentially important mutations have not been profiled with a large number of clinical samples. Whole genome sequencing of a small number of colon samples demonstrated that somatic cancer mutations consist of a few genes that occur frequently and many more mutations that occur very infrequently in many different genes [ 
   <xref ref-type="bibr" rid="B16">
     16 
   </xref>, 
   <xref ref-type="bibr" rid="B17">
     17 
   </xref>]. Mutations in these infrequently mutated genes could have a similar effect or synergize with mutations in 
   <italic>
     KRAS 
   </italic>, 
   <italic>
     PIK3CA 
   </italic>, and 
   <italic>
     BRAF 
   </italic>.</p> 
  <p>Given these considerations, our goal was to find a cost-effective and high throughput methodology that would detect frequent and infrequent cancer mutations genes in a large number of samples. Furthermore, it was essential that the methodology would work with degraded DNAs isolated from FFPET.</p> 
  <p>The mass spectrometric SNP genotyping technology based on the Sequenom platform provided an ideal choice for mutation profiling to address these several criteria. It has been shown to work with small amounts of degraded DNAs (5 ng), and the high multiplexing capacity minimizes the use of irreplaceable clinical samples. In addition, a variety of studies have demonstrated that the sensitivity of mass spectrometric methods exceeds that of traditional Sanger sequencing and is highly concordant with Sanger sequencing, Pyrosequencing, and allele-specific PCR [ 
   <xref ref-type="bibr" rid="B16">
     16 
   </xref>, 
   <xref ref-type="bibr" rid="B18">
     18 
   </xref>, 
   <xref ref-type="bibr" rid="B19">
     19 
   </xref>]. Furthermore, Sequenom has recently developed the OncoCarta Panel, an oncogene panel that offers a rapid and parallel analysis of 238 simple and complex cancer mutations across 19 genes.</p> 
  <p>The OncoCarta panel includes assays for most colon cancer mutations in the clinically relevant genes, 
   <italic>
     BRAF 
   </italic>(99%), 
   <italic>
     KRAS 
   </italic>(98%), and 
   <italic>
     PIK3CA 
   </italic>(78%), and in addition contains assays for other cancer mutations in genes that intersect with the same pathways as that of 
   <italic>
     KRAS 
   </italic>, 
   <italic>
     BRAF 
   </italic>, and 
   <italic>
     PIK3CA 
   </italic>, such as 
   <italic>
     AKT1 
   </italic>, 
   <italic>
     EGFR 
   </italic>, 
   <italic>
     HRAS 
   </italic>, 
   <italic>
     NRAS 
   </italic>, 
   <italic>
     MET 
   </italic>and others. The frequency of 
   <italic>
     KRAS 
   </italic>, 
   <italic>
     PIK3CA 
   </italic>, and 
   <italic>
     BRAF 
   </italic>mutations in the National Surgical Adjuvant Breast and Bowel Project (NSABP) trial C-07 were similar to the frequencies for colon samples listed in the COSMIC data base. This observation provides evidence that the mutation data obtained with the Sequenom platform is accurate. Our results also demonstrate that a majority of colon cancer samples have aberrant 
   <italic>
     PIK(3) 
   </italic>- 
   <italic>
     RAS/RAF 
   </italic>network; similar results have been seen previously [ 
   <xref ref-type="bibr" rid="B6">
     6 
   </xref>].</p> 
  <p>Mutations in 
   <italic>
     ABL1 
   </italic>and 
   <italic>
     MET 
   </italic>, not previously identified in colon cancer, were identified, and 13 other genes were screened and found not to be mutated in hot spot locations. Furthermore, this study identified the most frequent colon cancer mutations from OncoCarta, providing the necessary information to reduce the number of assays from 187 to 24, creating a smaller, more specific and economical panel, requiring less DNA, and thus conserving precious clinical samples.</p> 
 </sec> 
 <sec sec-type="methods"> 
  <h3>Methods</h3> 
  <sec> 
   <h3>Clinical samples and histological evaluation</h3> 
   <p>Samples used in this study were from NSABP clinical trial C-07. This trial enrolled patients between 02/2000 and 11/2002 to compare oxaliplatin and bolus 5-FU/LV to bolus 5FU/LV alone for resected stage II and III colon cancer [ 
    <xref ref-type="bibr" rid="B20">
      20 
    </xref>]. Tissue samples were obtained at surgery before the patients had received any treatment and were routinely processed with FFPE. C-07 was approved by an institutional review board, and informed consent was obtained from each subject. A pathologist categorized tumors into poor, moderate, well differentiated and signet cells carcinoma, according to the World Health Organization (WHO) criteria. Samples were graded for mucinous character, based on the amount of mucin retained within the tumor (1 = no mucin, 2 = &lt; 50% mucinous volume/total tumor volume, 3 = &gt;50% mucinous volume/total tumor volume). Only grade 3 tumors were considered mucinous tumors carcinoma, which is in accordance with WHO criteria.</p> 
  </sec> 
  <sec> 
   <h3>DNA isolation</h3> 
   <p>DNA was isolated from FFPE tumor blocks collected from patients participating in NSABP clinical trial C-07. FFPE tumor blocks were cut, and sections of the slide containing the most tumor cells were defined by a pathologist and isolated by macrodissection. Genomic DNA was extracted from 4 five ?m unstained sections. After attempting several extraction procedures from a variety of manufacturers (Machery Negel, Qiagen, Ambion), it was determined that the Ambion RecoverAll™ Total Nucleic Acid isolation kit (Applied Biosystem, Foster City, CA) yielded the best DNA based on the quality, quantity, and the performance on the mass spectrometer (data not shown). The extraction was performed as recommended by the manufacturer, with two exceptions; the protease digestion was extended overnight and the elution volume was increased to 150 ul to maximize the total amount of DNA recovered. Additional protease was added to samples incompletely digested after the overnight treatment. DNA was measured with fluorescence, using the Quant-iT ™ PicoGreen<sup>® </sup>dsDNA Assay Kit (Invitrogen, Carlsbad, CA) and the InfiniteF200 fluorometer (Tecan, Mannedorf, Switzerland).</p> 
   <p>As a positive control for known mutations and to test the performance of the platform, annotated cell line DNAs (A2058, HS578T, HL60, MCF7, MDAMB231, NCI-H1299, NCI-H1395, UACC-893) were purchased from American Type Culture Collection (ATCC, Manassas, VA, US). Two cell lines, SKBR3 and MCF-7, were grown in culture and cell pellets were fixed in formalin, andembedded in paraffin and DNAs were isolated as described for the clinical samples.</p> 
  </sec> 
  <sec> 
   <h3>Mass Spec Type Plex Technology and the OncoCarta Panel</h3> 
   <p>For mutation detection, the Sequenom platform and the OncoCarta mutation panel were used and the protocol provided by Sequenom (San Diego, CA) was followed with minor modifications. A schematic of the procedure is shown in Fig. 
    <xref ref-type="fig" rid="F1">
      1 
    </xref>. A Tecan Evo liquid handler was used to normalize the DNA samples and to set up the PCR reactions. The amount of DNA added to the PCR was reduced to 15 ng or less. DNAs were amplified using the OncoCarta PCR primer pools, unincorporated nucleotides were inactivated by shrimp alkaline phosphatase (SAP), and a single base extension reaction was performed using extension primers that hybridize immediately adjacent to the mutations and a custom mixture of nucleotides. Salts were removed by the addition of a cation exchange resin. Multiplexed reactions were spotted onto the SpectroChipII, and mutations, if present, were resolved by MALDI-TOF on the Compact Mass Spectrometer (Sequenom, San Diego, CA).</p> 
   <fig id="F1" position="float"> 
    <label>Figure 1</label> 
    <p> 
     <bold>
       Methodology for mutation detection 
     </bold>. Genomic DNA from the samples is amplified by PCR, resulting in copies of both mutant and wildtype alleles. Shrimp Alkaline Phosphatase removed excess nucleotides from the sample wells. Primer extension was performed using terminator nucleotides A, C, T, G, each with distinct masses. This linear amplification results in sequences proportional to the alleles that can be distinguished by mass spec (Maldi-Tof Separation).</p> 
    <graphic xlink:href="1471-2407-10-101-1"></graphic> 
   </fig> 
   <p>The OncoCarta™ Panel v1.0 (Sequenom, San Diego, CA) consists of 24 pools of primer pairs and 24 pools of extension primers, and has the capacity to detect 238 mutations in 19 genes, listed in Table 
    <xref ref-type="table" rid="T1">
      1 
    </xref>. Each pool consists of 5-9 primer pairs in the PCR reaction. Two types of assays have been designed in the OncoCarta panel, referred to as simple and complex. The simple assays are those in which a single assay is able to detect the amino acid changes at that codon. The complex assays are those that require more than one assay to identify codon changes or deletions and insertion, and thus are able to detect multiple different amino acid substitutions or deletions. An example of a complex assay is 
    <italic>
      KRAS_ 
    </italic>1 and 
    <italic>
      KRAS 
    </italic>_2, which interrogate 2 different nucleotide positions within codon 12 and together identify all codon 12 amino acid changes. Much more complex assays are included in OncoCarta, which interrogate insertions and deletions within the 
    <italic>
      EGFR 
    </italic>gene.</p> 
   <table-wrap id="T1" position="float"> 
    <label>Table 1</label> 
    <p>Mutations detected with OncoCarta</p> 
    <table frame="hsides" rules="groups"> 
     <tbody> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-G250E</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-L747_E749del, A750P</td> 
       <td align="left">KIT-P585P</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-Q252H</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-E746_A750del</td> 
       <td align="left">KIT-D579del</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-Y253H</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-L747_E749del, A750P</td> 
       <td align="left">KIT-K642E</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-Y253F</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-L747_S752del, P753S</td> 
       <td align="left">KIT-D816V</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-E255K</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-E746_T751del, V ins</td> 
       <td align="left">KIT-D816H/D816Y</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-E255V</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-L747_S752del, Q ins</td> 
       <td align="left">KIT-V825A</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-D276G</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-L747_S752del, Q ins</td> 
       <td align="left">KIT-E839K</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-F311L</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-E746_T751del, S752D/SNP C2255T</td> 
       <td align="left">KIT-M552L</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-T315I</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-D770_N771&gt;AGG/V769_D770insASV/V769_D770insASV</td> 
       <td align="left">KIT-Y568D</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-F317L</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-D770_N771insG</td> 
       <td align="left">KIT-F584S</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-M351T</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-L747_T750del, P ins</td> 
       <td align="left">KIT-P551_V555del</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-E355G</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-E746_A750del</td> 
       <td align="left">KIT-P551_V555del</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-F359V</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-E746_T751del, I ins</td> 
       <td align="left">KIT-Y553_Q556del</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-H396R</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-L747_T751del</td> 
       <td align="left">KIT-Y553_Q556del</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          AKT1 
        </italic>-rs11555435</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-L747_T751del</td> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-G12V/A/D/C/S/R/F</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          AKT1 
        </italic>-rs11555431</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-E746_A750del, V ins</td> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-G13C/S/V/D</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          AKT1 
        </italic>-rs11555432</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-E746_A750del, V ins</td> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-A59T</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          AKT1 
        </italic>-rs12881616</td> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-S752_I759del</td> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-Q61E/K/L/R/P/H</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          AKT1 
        </italic>-rs11555433</td> 
       <td align="left">ERBB2-L755P</td> 
       <td align="left"> 
        <italic>
          MET 
        </italic>-R970C</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          AKT1 
        </italic>-rs11555436</td> 
       <td align="left">ERBB2-G776S/G776LC</td> 
       <td align="left"> 
        <italic>
          MET 
        </italic>-T992I</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          AKT1 
        </italic>-rs34409589</td> 
       <td align="left">ERBB2-G776VC</td> 
       <td align="left"> 
        <italic>
          MET 
        </italic>-Y1230C</td> 
      </tr> 
      <tr> 
       <td align="left">AKT2-S302G</td> 
       <td align="left">ERBB2-G776VC/G776VC</td> 
       <td align="left"> 
        <italic>
          MET 
        </italic>-Y1235D</td> 
      </tr> 
      <tr> 
       <td align="left">AKT2-R371H</td> 
       <td align="left">ERBB2-M774_A775insYVMA</td> 
       <td align="left"> 
        <italic>
          MET 
        </italic>-M1250T</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-G464R</td> 
       <td align="left">ERBB2-A775_G776insYVMA</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-G12V/G12A/G12D</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-G464V/G464E</td> 
       <td align="left">ERBB2-P780_Y781insGSP</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-G12C/G12R/G12S</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-G466V/G466G/G466E</td> 
       <td align="left">ERBB2-P780_Y781insGSP</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-G13V/G13A/G13D</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-G466R</td> 
       <td align="left">ERBB2-S779_P780insVGS</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-G13C/G13R/G13S</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-F468C</td> 
       <td align="left">FGFR1-S125L</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-A18T</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-G469S/E/A/V/R</td> 
       <td align="left">FGFR1-P252T</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-Q61L/Q61R/Q61P</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-D594V| G</td> 
       <td align="left">FGFR3-R248C</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-Q61H</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-F595L</td> 
       <td align="left">FGFR3-S249C</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-Q61E/Q61K</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-G596R</td> 
       <td align="left">FGFR3-G370C</td> 
       <td align="left">PDGFRA-V561D</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-L597S/R/Q/V</td> 
       <td align="left">FGFR3-Y373C</td> 
       <td align="left">PDGFRA-T674I</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-T599I</td> 
       <td align="left">FGFR3-A391E</td> 
       <td align="left">PDGFRA-F808L</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-V600E/K/R/L</td> 
       <td align="left">FGFR3-K650Q/E</td> 
       <td align="left">PDGFRA-D846Y</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-K601N/E</td> 
       <td align="left">FGFR3-K650T/M</td> 
       <td align="left">PDGFRA-N870S</td> 
      </tr> 
      <tr> 
       <td align="left">CDK-R24C/H</td> 
       <td align="left">FLT3-I836del</td> 
       <td align="left">PDGFRA-D1071N</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-R108K</td> 
       <td align="left">FLT3_2</td> 
       <td align="left">PDGFRA-D842_H845del</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-T263P</td> 
       <td align="left">FLT3_3</td> 
       <td align="left">PDGFRA-I843_D846del</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-A289V</td> 
       <td align="left">FLT3-D835H/D835Y</td> 
       <td align="left">PDGFRA-S566_E571&gt;K</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-G598V</td> 
       <td align="left"> 
        <italic>
          HRAS 
        </italic>-G12V/D</td> 
       <td align="left">PDGFRA-I843_S847&gt;T</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-E709K/E709H</td> 
       <td align="left"> 
        <italic>
          HRAS 
        </italic>-G13C/R/S</td> 
       <td align="left">PDGFRA-D842V</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-E709A/E709G/E709V</td> 
       <td align="left"> 
        <italic>
          HRAS 
        </italic>-G13V/D</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-R88Q</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-G719S/G719C</td> 
       <td align="left"> 
        <italic>
          HRAS 
        </italic>-Q61H</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-N345K</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-G719A</td> 
       <td align="left"> 
        <italic>
          HRAS 
        </italic>-Q61H/L/R/P/K</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-C420R</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-M766_A767insAI</td> 
       <td align="left">JAK2-V617F</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-P539R</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-S768I</td> 
       <td align="left">KIT-D52N</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-E542K</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-V769_D770insASV</td> 
       <td align="left">KIT-Y503_F504insAY</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-E545K</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-V769_D770insCV</td> 
       <td align="left">KIT-W557R/W557R/W557G</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-Q546K</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-D770_N771&gt;AGG/V769_D770insASV/V769_D770insASV</td> 
       <td align="left">KIT-V559D/V559A/V559G</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-H701P</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-D770_N771insG</td> 
       <td align="left">KIT-V559I</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-H1047R/H1047L</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-N771_P772&gt;SVDNR</td> 
       <td align="left">KIT-V560D/V560G</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-H1047Y</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-P772_H773insV</td> 
       <td align="left">KIT-K550_K558del</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-G1049R</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-H773&gt;NPY</td> 
       <td align="left">KIT-K558_V560del</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-R38H</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-H773_V774insNPH/H773_V774insPH/H773_V774insH</td> 
       <td align="left">KIT-K558_E562del</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-C901F</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-V774_C775insHV</td> 
       <td align="left">KIT-V559del</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-M1043I/M1043I</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-T790 M</td> 
       <td align="left">KIT-V559_V560del</td> 
       <td align="left">RET-C634R</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-L858R</td> 
       <td align="left">KIT-V560del</td> 
       <td align="left">RET-C634W/Y</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-L861Q</td> 
       <td align="left">KIT-Y570_L576del</td> 
       <td align="left">RET-E632_L633del</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-L747_T750del, P ins/E746_A750del, T751A</td> 
       <td align="left">KIT-E561K</td> 
       <td align="left">RET-M918T</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          EGFR 
        </italic>-E746_T751del, I ins/S752_I759del</td> 
       <td align="left">KIT-L576P</td> 
       <td align="left">RET-A664D</td> 
      </tr> 
     </tbody> 
    </table> 
   </table-wrap> 
  </sec> 
  <sec> 
   <h3>Data analysis</h3> 
   <p>Data analysis was performed using MassArray Typer Analyzer software 4.0.4.20 (Sequenom), which facilitates visualization of data patterns as well as the raw spectra. Mutations were identified in two different ways. Typer automates the identification of mutants by comparing ratios of the wild type peak to that of all suspected mutants and generates an Onco Mutation report detailing specific mutations and the ratios of wild type and mutation peaks. In addition, raw data was exported to Excel and an in-house macro was used to duplicate the analysis. The area under the peaks allows for quantification for each allele, giving a direct evaluation of the proportion of mutated and wildtype (wt) allele in the sample [ 
    <xref ref-type="bibr" rid="B18">
      18 
    </xref>].</p> 
   <p>All mutations from both the Onco mutation report and the in-house Excel report were reviewed manually by 3 investigators (DF, PGG, KPG). Manual review of mutations was necessary to identify "real" mutant peaks from salt peaks or other background peaks. Selected reviewed mutations from the Onco Mutation Report and from the in-house macro were compared and were concordant.</p> 
  </sec> 
 </sec> 
 <sec> 
  <h3>Results</h3> 
  <sec> 
   <h3>Mutations were detected in control DNAs from intact and FFPETsamples</h3> 
   <p>Previously described mutations in control cell lines were detected. 
    <italic>
      BRAF 
    </italic>_V600E, 
    <italic>
      HRAS 
    </italic>_G12D, 
    <italic>
      NRAS 
    </italic>_Q61L, 
    <italic>
      PIK3CA 
    </italic>_E545K, 
    <italic>
      KRAS 
    </italic>_G13D, 
    <italic>
      NRAS 
    </italic>_Q61K, 
    <italic>
      EGFR 
    </italic>1_S125L, and 
    <italic>
      PIK3CA 
    </italic>_H1047R were detected in the appropriate cell lines (A2058, HS578T, HL60, MCF7, MDAMB231, NCI-H1299, NCI-H1395, and UACC-893, respectively). The appropriate mutation was found in MCF-7 ( 
    <italic>
      PIK3CA 
    </italic>_E545K) from both intact DNA and DNA isolated from FFPET. DNAs from clinical samples, control cell lines, and cell lines formalin-fixed, paraffin-embedded cell lines showed the same rates of primer extension and performance on mass spectrometer.</p> 
   <p>The proportion of the mutated alleles in each cell line, as observed from the area under the mutant peak on the spectra, ranged from 0.4-0.6, as expected for a pure clonal population with a heterozygote mutation. Spectra for cell line UACC-893 had equal fractions of mutant and wt alleles (Fig. 
    <xref ref-type="fig" rid="F2">
      2A 
    </xref>). One exception to this distribution among cell lines was seen in A2058, which showed spectra consistent with 2 copies of the WT allele and one mutant 
    <italic>
      BRAF 
    </italic>mutant allele (Fig. 
    <xref ref-type="fig" rid="F2">
      2B 
    </xref>). The 3 alleles of 
    <italic>
      BRAF 
    </italic>in A2058 are consistent with the observation that there are 3 copies of chromosome 7 in this cell line (COSMIC in the SNP Array Based LOH and Copy Number Analysis data base) [ 
    <xref ref-type="bibr" rid="B21">
      21 
    </xref>].</p> 
   <fig id="F2" position="float"> 
    <label>Figure 2</label> 
    <p> 
     <bold>
       Spectra for cell lines UACC-893 and A2058 
     </bold>. The expected positions for the unexteneded primer (UEP), and the extension products (Mutant and WT) from assays 
     <italic>
       PIK3CA 
     </italic>_9 and 
     <italic>
       BRAF 
     </italic>_15 in cell lines UACC-893 and A2058, respectively, are indicated with red dashed lines. The proportion of peak areas and the specific base is also shown. Assays 
     <italic>
       PIK3CA 
     </italic>_9 and 
     <italic>
       BRAF 
     </italic>_15 detected mutations in 
     <italic>
       PIK3CA 
     </italic>at amino acid position 1047 and in 
     <italic>
       BRAF 
     </italic>at amino acid position 600, respectively. Other peaks included in these spectra as result of multiplexing but not part of the designated assays are indicated as grey dashed lines.</p> 
    <graphic xlink:href="1471-2407-10-101-2"></graphic> 
   </fig> 
  </sec> 
  <sec> 
   <h3>The Sequenom platform was sensitive and quantitative</h3> 
   <p>Pilot studies demonstrated that the assays worked with as little as 1 ng of DNA (Fig. 
    <xref ref-type="fig" rid="F3">
      3 
    </xref>). The fraction of unextended primer was .09 even when the input DNA was between 1-3 ng, When concentrations of the amount of DNA was between 3-14 ng, the fraction of unextended primer was similar, .07. Thus, the assays worked well even when only 1 ng of DNA was used.</p> 
   <fig id="F3" position="float"> 
    <label>Figure 3</label> 
    <p> 
     <bold>
       Fractional unextended primer versus input DNA 
     </bold>. The range and the average for the percent of unextended primer for different amounts of input DNA into the PCR reactions are shown. The number of samples used in each category was 4 for 1-3 ng, 9 for 3-9 ng, 13 for 9-13 and 210 for 14 ng.</p> 
    <graphic xlink:href="1471-2407-10-101-3"></graphic> 
   </fig> 
   <p>In clinical samples with some assays it was possible to detect mutations that only represented 5% of the total 2 peak areas. The spectra in Fig. 
    <xref ref-type="fig" rid="F4">
      4 
    </xref> show a small but clear peak at the expected size for a 
    <italic>
      PIK3CA 
    </italic>1047R mutation in a lymph node. We also were able to demonstrate the sensitivity of the platform by performing a cell mixing experiment. Mutation analysis was done using MCF-7 cell line DNA alone or mixed with SKBR3 at various percentages. MCF-7 cells contain a PIK3CA mutation, and SKBR3 cells do not. Fig. 
    <xref ref-type="fig" rid="F5">
      5 
    </xref> demonstrates that the mutation was detectable even when the MCF-7 cells represented only 5 to 10% of the total DNA and only 5 to 2.5% of the alleles. This sensitivity is important for mutation detection in clinical cancer samples, which usually contain some amount of normal tissue, which dilutes the number of tumor cells. This is of particular concern when profiling lymph nodes, which may contain a minority of tumor cells.</p> 
   <fig id="F4" position="float"> 
    <label>Figure 4</label> 
    <p> 
     <bold>
       Sensitive detection of mutations in clinical FFPE samples with the Sequenom platform 
     </bold>. Small mutant but definitive peak illustrating a 
     <italic>
       PIK3CA 
     </italic>-1047R mutation in approximately 5% of the sample DNA is shown.</p> 
    <graphic xlink:href="1471-2407-10-101-4"></graphic> 
   </fig> 
   <fig id="F5" position="float"> 
    <label>Figure 5</label> 
    <p> 
     <bold>
       Quantification of the sensitivity with a cell line mixing experiment 
     </bold>. Spectra of MCF-7 cells (mutant) alone or mixed with SKBR3 cells (WT) are shown. Percents are based on the ng amounts of DNA. This assay detects an E545K mutation in PIK3CA.</p> 
    <graphic xlink:href="1471-2407-10-101-5"></graphic> 
   </fig> 
  </sec> 
  <sec> 
   <h3>Frequencies of C-07 mutations in 
    <italic>
      KRAS 
    </italic>, 
    <italic>
      NRAS 
    </italic>, 
    <italic>
      PIK3CA 
    </italic>, and 
    <italic>
      BRAF 
    </italic>detected with OncoCarta and the Sequenom platform were similar to previous reports</h3> 
   <p>In this preliminary assessment of the feasibility of using the Sequenom platform to do large-scale mutation profiling of colon cancer samples isolated from FFPET, it was essential to determine if our data yielded frequencies typical of what has been seen previously. Table 
    <xref ref-type="table" rid="T2">
      2 
    </xref> shows the mutation frequencies obtained here and from the COSMIC (Catalog of Somatic Mutations in Cancer) database [ 
    <xref ref-type="bibr" rid="B21">
      21 
    </xref>]. The COSMIC frequencies seen in Table 
    <xref ref-type="table" rid="T2">
      2 
    </xref> are based only on those mutations that are detectable with OncoCarta. OncoCarta assays interrogate 99%, 98%, and 78% of the known colon cancer mutations in 
    <italic>
      BRAF 
    </italic>, 
    <italic>
      KRAS 
    </italic>, and 
    <italic>
      PIK3CA 
    </italic>, respectively, based on a large number of colon cancer samples that have been sequenced in 
    <italic>
      BRAF 
    </italic>(n = 4628), 
    <italic>
      KRAS 
    </italic>(n = 858) and 
    <italic>
      PIK3CA 
    </italic>(n = 247). The OncoCarta panel found that the most frequent mutations in C-07 were 
    <italic>
      KRAS 
    </italic>(43.5%), 
    <italic>
      PIK3CA 
    </italic>(20.1%), and 
    <italic>
      BRAF 
    </italic>(12.1%), which are similar to what is seen in COSMIC. 
    <italic>
      NRAS 
    </italic>mutations, while infrequent, were detected in codons 12, 13 and 61 and represent a sizable minority of the C0-7 samples (3.8%). These data suggest that FFPET samples can be interrogated with the technology described here and yield accurate data.</p> 
   <table-wrap id="T2" position="float"> 
    <label>Table 2</label> 
    <p>Frequency of colon cancer mutations</p> 
    <table frame="hsides" rules="groups"> 
     <thead> 
      <tr> 
       <th align="left">Mutation</th> 
       <th align="center">No. Mutated Samples</th> 
       <th align="center">Frequency in Primary Tumor*</th> 
       <th align="center">in COSMIC<sup>†</sup></th> 
       <th align="center">Multiple Mutations<sup>‡</sup></th> 
      </tr> 
     </thead> 
     <tbody> 
      <tr> 
       <td align="left"> 
        <italic>
          ABL1 
        </italic>-F359V</td> 
       <td align="center">1</td> 
       <td align="center">0.40%</td> 
       <td align="center">0/66<sup>§</sup></td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left">? 
        <bold>
          ABL1 Total 
        </bold> </td> 
       <td align="center"> 
        <bold>
          1 
        </bold> </td> 
       <td align="center"> 
        <bold>
          0.40% 
        </bold> </td> 
       <td align="center">NF(0/66)<sup>§</sup></td> 
       <td align="center">100%</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          AKT1 
        </italic>-E17K</td> 
       <td align="center">1</td> 
       <td align="center">0.40%</td> 
       <td align="center">0/31<sup>§</sup></td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left">? 
        <bold>
          AKT1 Total 
        </bold> </td> 
       <td align="center"> 
        <bold>
          1 
        </bold> </td> 
       <td align="center"> 
        <bold>
          0.40% 
        </bold> </td> 
       <td align="center">NF(0/31)<sup>§</sup></td> 
       <td align="center">100%</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-D594V| G</td> 
       <td align="center">1</td> 
       <td align="center">0.40%</td> 
       <td align="center">NF (0/3179)</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-V600E</td> 
       <td align="center">28</td> 
       <td align="center">11.70%</td> 
       <td align="center">14.60%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left">? 
        <bold>
          BRAF Total 
        </bold> </td> 
       <td align="center"> 
        <bold>
          29 
        </bold> </td> 
       <td align="center"> 
        <bold>
          12.10% 
        </bold> </td> 
       <td align="center">14.70%</td> 
       <td align="center">24%</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-G12A</td> 
       <td align="center">2</td> 
       <td align="center">0.80%</td> 
       <td align="center">1.80%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-G12C</td> 
       <td align="center">10</td> 
       <td align="center">4.20%</td> 
       <td align="center">3.60%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-G12D</td> 
       <td align="center">40</td> 
       <td align="center">16.70%</td> 
       <td align="center">13.20%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-G12R</td> 
       <td align="center">3</td> 
       <td align="center">1.30%</td> 
       <td align="center">0.40%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-G12S</td> 
       <td align="center">3</td> 
       <td align="center">1.30%</td> 
       <td align="center">4.20%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-G12V</td> 
       <td align="center">20</td> 
       <td align="center">8.40%</td> 
       <td align="center">7.40%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-G13D</td> 
       <td align="center">23</td> 
       <td align="center">9.60%</td> 
       <td align="center">5.20%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-A59T</td> 
       <td align="center">1</td> 
       <td align="center">0.40%</td> 
       <td align="center">0.10%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-Q61L</td> 
       <td align="center">1</td> 
       <td align="center">0.40%</td> 
       <td align="center">0.20%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-Q61R</td> 
       <td align="center">1</td> 
       <td align="center">0.40%</td> 
       <td align="center">NF (0/1927)</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left">? 
        <bold>
          KRAS Total 
        </bold> </td> 
       <td align="center"> 
        <bold>
          104 
        </bold> </td> 
       <td align="center"> 
        <bold>
          43.50% 
        </bold> </td> 
       <td align="center">36.10%</td> 
       <td align="center">34%</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          MET 
        </italic>-R970C</td> 
       <td align="center">2</td> 
       <td align="center">0.80%</td> 
       <td align="center">NF (0/77)</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          MET 
        </italic>-T992I</td> 
       <td align="center">6</td> 
       <td align="center">2.50%</td> 
       <td align="center">NF (0/77)</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left">? 
        <bold>
          MET Total 
        </bold> </td> 
       <td align="center"> 
        <bold>
          8 
        </bold> </td> 
       <td align="center"> 
        <bold>
          3.30% 
        </bold> </td> 
       <td align="center">0%</td> 
       <td align="center">50%</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-G12C</td> 
       <td align="center">1</td> 
       <td align="center">0.40%</td> 
       <td align="center">NF (0/46)</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-G12D</td> 
       <td align="center">4</td> 
       <td align="center">1.70%</td> 
       <td align="center">NF (0/46)</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-G13R</td> 
       <td align="center">1</td> 
       <td align="center">0.40%</td> 
       <td align="center">NF (0/46)</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-G13V</td> 
       <td align="center">1</td> 
       <td align="center">0.40%</td> 
       <td align="center">NF (0/46)</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-Q61H</td> 
       <td align="center">1</td> 
       <td align="center">0.40%</td> 
       <td align="center">NF (0/46)</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-Q61K</td> 
       <td align="center">1</td> 
       <td align="center">0.40%</td> 
       <td align="center">NF (0/46)</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left">? 
        <bold>
          NRAS Total 
        </bold> </td> 
       <td align="center"> 
        <bold>
          9 
        </bold> </td> 
       <td align="center"> 
        <bold>
          3.80% 
        </bold> </td> 
       <td align="center">2.2%</td> 
       <td align="center">38%</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-R88Q</td> 
       <td align="center">5</td> 
       <td align="center">2.10%</td> 
       <td align="center">NF (0/171)</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-C420R</td> 
       <td align="center">2</td> 
       <td align="center">0.80%</td> 
       <td align="center">NF (0/171)</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-E542K</td> 
       <td align="center">9</td> 
       <td align="center">3.80%</td> 
       <td align="center">4.10%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-E545K</td> 
       <td align="center">12</td> 
       <td align="center">5.00%</td> 
       <td align="center">4.10%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-Q546K</td> 
       <td align="center">4</td> 
       <td align="center">1.70%</td> 
       <td align="center">1.20%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-H701P</td> 
       <td align="center">1</td> 
       <td align="center">0.40%</td> 
       <td align="center">NF (0/171)</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-H1047L</td> 
       <td align="center">1</td> 
       <td align="center">0.40%</td> 
       <td align="center">1.80%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-H1047R</td> 
       <td align="center">14</td> 
       <td align="center">5.90%</td> 
       <td align="center">5.30%</td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left">? 
        <bold>
          PIK3CA Total 
        </bold> </td> 
       <td align="center"> 
        <bold>
          48 
        </bold> </td> 
       <td align="center"> 
        <bold>
          20.10% 
        </bold> </td> 
       <td align="center">16.40%</td> 
       <td align="center">80%</td> 
      </tr> 
     </tbody> 
    </table> 
    <table-wrap-foot> 
     <p>*% of C-07 samples with this mutation.</p> 
     <p> <sup>†</sup>Data from COSMIC for colon adenocarcinoma limited to the same mutations interrogated with OncoCarta. The mutations listed are only the ones found in C-07. Some COSMIC amino acid changes are not shown here if they were not mutated in C-07.</p> 
     <p> <sup>‡</sup>% of samples with a mutation in the gene shown and at least one other mutation in C-07 samples.</p> 
     <p> <sup>§</sup>COSMIC data are from large intestine, not specific to colon.</p> 
    </table-wrap-foot> 
   </table-wrap> 
   <p>While most of the specific amino acid mutations mirror what is seen on the COSMIC database, some unique colon cancer gene mutations were found, which include 
    <italic>
      ABL1 
    </italic>-F359V, 
    <italic>
      AKT1- 
    </italic>E17K, 
    <italic>
      MET- 
    </italic>R970C, and 
    <italic>
      MET 
    </italic>-T992I. Other amino acid changes that were not in the COSMIC database were amino acid changes R88Q, H701P, and C420R in 
    <italic>
      PIK3CA 
    </italic>, 
    <italic>
      BRAF 
    </italic>-594V/G, and 
    <italic>
      KRAS 
    </italic>-Q61R, and several in 
    <italic>
      NRAS 
    </italic>, including G12C, G12D, G13R, G13V, Q61H and Q61K (Table 
    <xref ref-type="table" rid="T2">
      2 
    </xref>).</p> 
  </sec> 
  <sec> 
   <h3>MET mutations were found in C0-7 and amplified in sometumors</h3> 
   <p> 
    <italic>
      MET 
    </italic>mutations were found in 3.3% of C-07 samples. Interestingly, these mutations were not only unexpected in their appearance within the colon cancer population but also the frequency within the samples was unexpected. In four of the eight samples with 
    <italic>
      MET 
    </italic>mutations, the mutant alleles were present at 58-70%, suggesting an amplification of the mutant allele or a loss of the wt gene (Fig. 
    <xref ref-type="fig" rid="F6">
      6 
    </xref>). Amplification may represent the best explanation, in that amplification of the 
    <italic>
      MET 
    </italic>genomic region, 7q31, has been observed in the Progenetix CGH Database in 23% of colorectal cancers [ 
    <xref ref-type="bibr" rid="B22">
      22 
    </xref>]</p> 
   <fig id="F6" position="float"> 
    <label>Figure 6</label> 
    <p> 
     <bold>
       MET mutation is amplified 
     </bold>. The proportion and position (blue dashed lines) of mutant and wt alleles are shown. The 
     <italic>
       MET 
     </italic>_1 assay detects R970C mutations in 
     <italic>
       MET 
     </italic>.</p> 
    <graphic xlink:href="1471-2407-10-101-6"></graphic> 
   </fig> 
  </sec> 
  <sec> 
   <h3>Sequenom data was reproducible</h3> 
   <p>Most of the assays in the OncoCarta panel did not detect mutations or the frequency of mutations was very low (below 1%) in our colon cancer samples. OncoCarta assays interrogate mutations in these 19 genes listed in Table 
    <xref ref-type="table" rid="T1">
      1 
    </xref>. To reduce the cost, time and the amount of DNA required for profiling, only 24 assays, which detected mutations at a frequency of 1% or greater in C-07, were selected, resorted in 6 pools and included in a new panel, termed ColoCarta (Table 
    <xref ref-type="table" rid="T3">
      3 
    </xref>). Mutation profiles of 32 mutant samples with 41 mutations were repeated with the ColoCarta. The mutations detected by the 2 panels (OncoCarta and ColoCarta) were identical, demonstrating the reproducibility of the methodology.</p> 
   <table-wrap id="T3" position="float"> 
    <label>Table 3</label> 
    <p>ColoCarta panel</p> 
    <table frame="hsides" rules="groups"> 
     <thead> 
      <tr> 
       <th align="left">Sequenom's Assay Name</th> 
       <th align="left">Amino Acid Change</th> 
      </tr> 
     </thead> 
     <tbody> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>_15 &amp;16</td> 
       <td align="left">V600E/K/R/L</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>_9</td> 
       <td align="left"> 
        <italic>
          BRAF 
        </italic>-D594V</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          HRAS 
        </italic>_6*</td> 
       <td align="left"> 
        <italic>
          HRAS 
        </italic>-Q61L</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>_1 &amp; 2</td> 
       <td align="left">G12V/A/D/C/S/R/F</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>_4</td> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-G13D</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>_5</td> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-A59T</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>_7</td> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>-Q61L</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          KRAS 
        </italic>_8</td> 
       <td align="left">Q61H/Q61H</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          MET 
        </italic>_1</td> 
       <td align="left"> 
        <italic>
          MET 
        </italic>-R970C</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          MET 
        </italic>_2</td> 
       <td align="left"> 
        <italic>
          MET 
        </italic>-T992I</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>_1</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-G12V</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>_2</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-G12C</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>_3</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-G13V</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>_4</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-G13C</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>_7</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-Q61H</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>_8</td> 
       <td align="left"> 
        <italic>
          NRAS 
        </italic>-Q61E</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>_1</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-R88Q</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>_3</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-C420R</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>_5</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-E542K</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>_6</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-E545K</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>_7</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-Q546K</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>_8</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-H701P</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>_9</td> 
       <td align="left"> 
        <italic>
          PIK3CA 
        </italic>-H1047R</td> 
      </tr> 
     </tbody> 
    </table> 
    <table-wrap-foot> 
     <p>* 
      <italic>
        HRAS 
      </italic>_6 was included in panel but occurred in &lt; 0.1% of samples.</p> 
    </table-wrap-foot> 
   </table-wrap> 
  </sec> 
  <sec> 
   <h3>Multiple mutation frequencies suggest an order to the acquisition of different mutations</h3> 
   <p>A majority of the tumors (64%) contained at least one or more mutations in the following genes: 
    <italic>
      BRAF 
    </italic>, 
    <italic>
      KRAS 
    </italic>, 
    <italic>
      NRAS 
    </italic>, 
    <italic>
      MET 
    </italic>, or 
    <italic>
      PIK3CA 
    </italic>, and 18% had 2 or more mutations. The most common double mutation was in 
    <italic>
      KRAS 
    </italic>and 
    <italic>
      PIK3CA 
    </italic>, followed by 
    <italic>
      PIK3CA 
    </italic>and 
    <italic>
      BRAF 
    </italic>(Table 
    <xref ref-type="table" rid="T4">
      4 
    </xref>). Most samples with 
    <italic>
      PIK3CA 
    </italic>mutations (80%) also had mutations in other genes, the most frequent of which was 
    <italic>
      KRAS 
    </italic>; other mutated genes were 
    <italic>
      BRAF 
    </italic>, 
    <italic>
      MET 
    </italic>, 
    <italic>
      NRAS 
    </italic>, and a second 
    <italic>
      PIK3CA 
    </italic>mutation (Table 
    <xref ref-type="table" rid="T2">
      2 
    </xref>, last column). Tumors with 
    <italic>
      MET 
    </italic>and 
    <italic>
      NRAS 
    </italic>mutations also have an unexpectedly high frequency of co-occurring mutations, which suggests that they occur as a second mutation and perhaps later in the etiology of the tumor. Many tumors contain only a 
    <italic>
      KRAS 
    </italic>or 
    <italic>
      BRAF 
    </italic>mutation, which is consistent with previous reports finding these mutations in earlier stages of colon cancer [ 
    <xref ref-type="bibr" rid="B23">
      23 
    </xref>, 
    <xref ref-type="bibr" rid="B24">
      24 
    </xref>]. The multiple mutation frequencies for tumors with 
    <italic>
      KRAS 
    </italic>and 
    <italic>
      PIK3CA 
    </italic>or with 
    <italic>
      PIK3CA 
    </italic>and 
    <italic>
      BRAF 
    </italic>were slightly higher and lower, respectively, than expected based on their individual frequencies (Table 
    <xref ref-type="table" rid="T4">
      4 
    </xref>). Conversely, the expected double mutation frequency of 
    <italic>
      BRAF 
    </italic>and 
    <italic>
      KRAS 
    </italic>would be 5.1%, based on our data, but this combination was not found, also in agreement with previous reports [ 
    <xref ref-type="bibr" rid="B24">
      24 
    </xref>] (Table 
    <xref ref-type="table" rid="T4">
      4 
    </xref>).</p> 
   <table-wrap id="T4" position="float"> 
    <label>Table 4</label> 
    <p>Single and double mutations in C-07</p> 
    <table frame="hsides" rules="groups"> 
     <thead> 
      <tr> 
       <th></th> 
       <th align="center" colspan="7">Double Mutation Frequencies</th> 
      </tr> 
      <tr> 
       <th></th> 
       <th></th> 
       <th align="center" colspan="2">KRAS</th> 
       <th align="center" colspan="2">PIK3CA</th> 
       <th align="center" colspan="2">All other</th> 
      </tr> 
      <tr> 
       <th></th> 
       <th align="right">Single</th> 
       <th align="center">Actual</th> 
       <th align="center">Expected</th> 
       <th align="center">Actual</th> 
       <th align="center">Expected</th> 
       <th align="center">Actual</th> 
       <th align="center">Expected</th> 
      </tr> 
     </thead> 
     <tbody> 
      <tr> 
       <td align="left">KRAS</td> 
       <td align="right">43.70%</td> 
       <td align="center">NA</td> 
       <td align="center">NA</td> 
       <td align="center">10.40%</td> 
       <td align="center">8.70%</td> 
       <td align="center">14.60%</td> 
       <td align="center">7.21%</td> 
      </tr> 
      <tr> 
       <td align="left">PIK3CA</td> 
       <td align="right">20.10%</td> 
       <td align="center">10.40%</td> 
       <td align="center">8.70%</td> 
       <td align="center">NA</td> 
       <td align="center">NA</td> 
       <td align="center">15.50%</td> 
       <td align="center">8.06%</td> 
      </tr> 
      <tr> 
       <td align="left">BRAF</td> 
       <td align="right">11.80%</td> 
       <td align="center">0</td> 
       <td align="center">5.10%</td> 
       <td align="center">1.80%</td> 
       <td align="center">2.40%</td> 
       <td align="center">2.50%</td> 
       <td align="center">5.71%</td> 
      </tr> 
      <tr> 
       <td align="left">MET</td> 
       <td align="right">3.30%</td> 
       <td align="center">1.67%</td> 
       <td align="center">1.44%</td> 
       <td align="center">0</td> 
       <td align="center">0.66%</td> 
       <td align="center">1.67%</td> 
       <td align="center">2%</td> 
      </tr> 
      <tr> 
       <td align="left">NRAS</td> 
       <td align="right">3.80%</td> 
       <td align="center">0</td> 
       <td align="center">1.66%</td> 
       <td align="center">0.42%</td> 
       <td align="center">0.40%</td> 
       <td align="center">1.30%</td> 
       <td align="center">2.14%</td> 
      </tr> 
      <tr> 
       <td align="left">All Mutations</td> 
       <td align="right">60.20%</td> 
       <td></td> 
       <td></td> 
       <td></td> 
       <td></td> 
       <td></td> 
       <td></td> 
      </tr> 
     </tbody> 
    </table> 
   </table-wrap> 
  </sec> 
  <sec> 
   <h3>Primary tumors with KRAS and PIK3CA mutations vary with respect to the frequency of these mutant alleles</h3> 
   <p>In the samples with co-occurring mutations, the ratios of 
    <italic>
      KRAS 
    </italic>mutation ratio ( 
    <italic>
      KRAS 
    </italic>mutation peak area/total peak area) to the 
    <italic>
      PIK3CA 
    </italic>mutation ratio ( 
    <italic>
      PIK3CA 
    </italic>mutation peak area/total peak area) was determined. Twenty-two out of 31 samples (71%) had 
    <italic>
      KRAS 
    </italic>/ 
    <italic>
      PIK3CA 
    </italic>ratios above 1.25 (Table 
    <xref ref-type="table" rid="T5">
      5 
    </xref>). 
    <italic>
      PIK3CA 
    </italic>mutations were more prevalent in only 2 out of 31 samples. These differences demonstrate that in a majority of primary tumors with double mutations in 
    <italic>
      KRAS 
    </italic>and 
    <italic>
      PIK3CA 
    </italic>, the 
    <italic>
      KRAS 
    </italic>mutations are more prevalent than the 
    <italic>
      PIK3CA 
    </italic>. This unequal distribution of mutant alleles within a tumor may be due to the fact that a majority of the tumor cells have only the 
    <italic>
      KRAS 
    </italic>mutation, and cells with a 
    <italic>
      PIK3CA 
    </italic>mutation are in the minority, or it could be due to copy number variations in the 
    <italic>
      KRAS 
    </italic>and 
    <italic>
      PIK3CA 
    </italic>loci.</p> 
   <table-wrap id="T5" position="float"> 
    <label>Table 5</label> 
    <p>KRAS/PIK3CA ratio mutation frequencies within primary tumors</p> 
    <table frame="hsides" rules="groups"> 
     <thead> 
      <tr> 
       <th align="left">No of Samples</th> 
       <th align="center">KRAS/PIK3CA</th> 
      </tr> 
     </thead> 
     <tbody> 
      <tr> 
       <td align="left">22</td> 
       <td align="center">1.25-3.22</td> 
      </tr> 
      <tr> 
       <td align="left">7</td> 
       <td align="center">0.93-1.13</td> 
      </tr> 
      <tr> 
       <td align="left">2</td> 
       <td align="center">0.81-.42</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          Average 
        </bold> </td> 
       <td align="center">1.67</td> 
      </tr> 
      <tr> 
       <td align="left"> 
        <bold>
          Median 
        </bold> </td> 
       <td align="center">1.6</td> 
      </tr> 
     </tbody> 
    </table> 
   </table-wrap> 
  </sec> 
  <sec> 
   <h3>BRAF mutations were correlated with poorly differentiated tumors and with mucinous tumors</h3> 
   <p>The frequency of mutations for 
    <italic>
      KRAS 
    </italic>, 
    <italic>
      PIK3CA 
    </italic>, and 
    <italic>
      BRAF 
    </italic>were tested for correlation to the degree of differentiation and to the prevalence of mucin in the tumor. 
    <italic>
      BRAF 
    </italic>mutations were found in 26.2% of the poorly differentiated tumors and in 8.2% of the moderate and well differentiated. These frequencies were significantly different by Chi square test (p value = 0.001). 
    <italic>
      BRAF 
    </italic>mutations were also associated with mucinous tumors: 
    <italic>
      BRAF 
    </italic>mutations occurred in 28% of grade 3 mucinous tumors (&gt;50% mucinous tumor cells) but in only 9.4% of the non-mucinous tumors (grade 1 and 2). This was significant by the Chi square test at p value = 0.006. Similar data have been reported previously [ 
    <xref ref-type="bibr" rid="B25">
      25 
    </xref>, 
    <xref ref-type="bibr" rid="B26">
      26 
    </xref>]. 
    <italic>
      KRAS 
    </italic>and 
    <italic>
      PIK3CA 
    </italic>mutations did not correlate with either the degree of differentiation or with prevalence of mucinous cells.</p> 
  </sec> 
  <sec> 
   <h3>Mutation profiling demonstrated a majority of primary and lymph node samples were concordant but differences were detected</h3> 
   <p>Lymph node metastases were not routinely collected in C-07 but as a pilot study to determine the feasibility of using lymph nodes for mutation profiling was conducted. We isolated DNA from 39 lymph nodes containing tumor cells and their corresponding primary tumors. These primary and lymph nodes samples were profiled with the entire OncoCarta panel. The majority of lymph nodes and their corresponding primary tumors (89.7%) were concordant. A total of 26 mutations were detected in lymph nodes, including 
    <italic>
      KRAS 
    </italic>, 
    <italic>
      BRAF 
    </italic>, 
    <italic>
      PIK3CA 
    </italic>, and 
    <italic>
      NRAS 
    </italic>. Thirty-five out of 39 lymph nodes had identical mutation profiles, but in 4 cases mutations in the primary tumors were not found in the corresponding lymph nodes ( 
    <italic>
      BRAF 
    </italic>[ 
    <xref ref-type="bibr" rid="B2">
      2 
    </xref>], 
    <italic>
      PIK3CA 
    </italic>[ 
    <xref ref-type="bibr" rid="B1">
      1 
    </xref>] and 
    <italic>
      KRAS 
    </italic>[ 
    <xref ref-type="bibr" rid="B1">
      1 
    </xref>]).</p> 
  </sec> 
  <sec> 
   <h3>Mutation profiles demonstrate that tumor cell populations may be different in lymph nodes and in the primary tumors</h3> 
   <p>Peak area evaluation of tumors that had 2 mutations and for which a metastatic lymph node was available demonstrated differences between the primary and lymph node samples. Table 
    <xref ref-type="table" rid="T6">
      6 
    </xref> details the frequency of mutant and wt alleles based on the peak areas for 5 such samples.</p> 
   <table-wrap id="T6" position="float"> 
    <label>Table 6</label> 
    <p>Allele frequencies for primary tumors with two mutations and metastatic lymph nodes</p> 
    <table frame="hsides" rules="groups"> 
     <thead> 
      <tr> 
       <th align="left">Sample</th> 
       <th align="left">Mutations</th> 
       <th align="center" colspan="2">Mutant Allele Frequency</th> 
       <th align="center" colspan="2">Mutation Ratios</th> 
      </tr> 
      <tr> 
       <th></th> 
       <th></th> 
       <th align="center">Primary</th> 
       <th align="center">Lymph Node</th> 
       <th align="center">Primary M1/M2</th> 
       <th align="center">Lymph node M1/M2</th> 
      </tr> 
     </thead> 
     <tbody> 
      <tr> 
       <td align="left">C07-0388</td> 
       <td align="left">M1. 
        <italic>
          KRAS 
        </italic>-G12D</td> 
       <td align="center">0.45</td> 
       <td align="center">0.44</td> 
       <td align="center">1.88</td> 
       <td align="center">1.63</td> 
      </tr> 
      <tr> 
       <td></td> 
       <td align="left">M2. 
        <italic>
          PIK3CA 
        </italic>-H1047R</td> 
       <td align="center">0.24</td> 
       <td align="center">0.27</td> 
       <td></td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left">C07-0717</td> 
       <td align="left">M1. 
        <italic>
          KRAS 
        </italic>-G13D</td> 
       <td align="center">0.08</td> 
       <td align="center">0.09</td> 
       <td align="center">1.14</td> 
       <td align="center">0.90</td> 
      </tr> 
      <tr> 
       <td></td> 
       <td align="left">M2. 
        <italic>
          PIK3CA 
        </italic>-H1047R</td> 
       <td align="center">0.07</td> 
       <td align="center">0.1</td> 
       <td></td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left">C07-0940</td> 
       <td align="left">M1. 
        <italic>
          KRAS 
        </italic>-G12D</td> 
       <td align="center">0.42</td> 
       <td align="center">0.34</td> 
       <td align="center">1.91</td> 
       <td align="center">1.10</td> 
      </tr> 
      <tr> 
       <td></td> 
       <td align="left">M2. 
        <italic>
          PIK3CA 
        </italic>-E542K</td> 
       <td align="center">0.22</td> 
       <td align="center">0.31</td> 
       <td></td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left">C07-2244</td> 
       <td align="left">M1. 
        <italic>
          KRAS 
        </italic>-G12C</td> 
       <td align="center">0.37</td> 
       <td align="center">0.15</td> 
       <td align="center">1.76</td> 
       <td align="center">_</td> 
      </tr> 
      <tr> 
       <td></td> 
       <td align="left">M2. 
        <italic>
          PIK3CA 
        </italic>-H1047R</td> 
       <td align="center">0.21</td> 
       <td align="center">0</td> 
       <td></td> 
       <td></td> 
      </tr> 
      <tr> 
       <td align="left">C07-1837</td> 
       <td align="left">M1. 
        <italic>
          BRAF 
        </italic>-V600E</td> 
       <td align="center">0.2</td> 
       <td align="center">0.22</td> 
       <td align="center">1.67</td> 
       <td align="center">4.40</td> 
      </tr> 
      <tr> 
       <td></td> 
       <td align="left">M2. 
        <italic>
          PIK3CA 
        </italic>-H1047R</td> 
       <td align="center">0.12</td> 
       <td align="center">0.05</td> 
       <td></td> 
       <td></td> 
      </tr> 
     </tbody> 
    </table> 
   </table-wrap> 
   <p> 
    <italic>
      KRAS 
    </italic>to 
    <italic>
      PIK3CA 
    </italic>ratios demonstrated that there were more 
    <italic>
      KRAS 
    </italic>mutations than 
    <italic>
      PIK3CA 
    </italic>mutations in 4 of 4 primary samples, and in 3 of the 4 lymph node samples. However, it is also notable that the ratio of 
    <italic>
      KRAS 
    </italic>/PIK3A was lower in the lymph node compared to their primary tumor in 3 out of 4 samples. In sample 0940, the 
    <italic>
      KRAS 
    </italic>/ 
    <italic>
      PIK3CA 
    </italic>mutation decreased by almost 1/2 in the lymph node tumor compared to the primary. Thus, in these samples there is either a loss of 
    <italic>
      KRAS 
    </italic>mutations or an accumulation of 
    <italic>
      PIK3CA 
    </italic>mutations, suggesting that 
    <italic>
      PIK3CA 
    </italic>mutations may impart a selective advantage in the lymph node.</p> 
   <p>In contrast, two other samples have a less frequent occurrence of their 
    <italic>
      PIK3CA 
    </italic>mutation in the lymph node than in the primary tumor. In sample 2244, the 
    <italic>
      PIK3CA 
    </italic>mutation was undetectable in the lymph node (Table 
    <xref ref-type="table" rid="T6">
      6 
    </xref>). In fact, if there was a selection for both mutations in the lymph node, then the 
    <italic>
      PIK3CA 
    </italic>mutation frequency would have been the same as that of the 
    <italic>
      KRAS 
    </italic>mutation (0.15). On the other hand, if the 
    <italic>
      PIK3CA 
    </italic>/ 
    <italic>
      KRAS 
    </italic>ratio were the same in the primary and lymph node tumor, then the 
    <italic>
      PIK3CA 
    </italic>mutation frequency would have been .08, which is still detectable with this technology (Fig. 
    <xref ref-type="fig" rid="F3">
      3 
    </xref>). Thus, in sample 2244 there were fewer 
    <italic>
      PIK3CA 
    </italic>mutant alleles in the lymph node than in the primary tumor. In sample 1837, mutations in both 
    <italic>
      BRAF 
    </italic>and 
    <italic>
      PIK3CA 
    </italic>were detected and the 
    <italic>
      BRAF 
    </italic>/ 
    <italic>
      PIK3CA 
    </italic>ratio was 1.67, but increased to 4.4 in the metastatic lymph node.</p> 
  </sec> 
 </sec> 
 <sec> 
  <h3>Discussion</h3> 
  <p>The Sequenom platform provides a superior technology for the screening of many hot spot mutations in cancer samples. Sanger sequencing would require amplification of at least 60 different fragments per sample, and many reactions would require optimization, thus adding considerable time and expense. Multiplexing and the use of the OncoCarta panel allowed us to skip this time consuming step. Thus, conservatively, Sanger sequencing would be 40 times more expensive, and require at least 2 times more DNA. Other sequencing technologies, which employ differential melting of mutant and wt sequences, such as HRMA, still require that the PCR product be sequenced. This would add significant cost and time to the procedure because 60% of the colon samples contained one or more mutations. In addition the Sequenom platform is more sensitive than Sanger sequencing in that it was able to detect mutations that represented only 5% of the DNA. Pyrosequencing represented a potential alternative to the Sequenom platform, but in our hands assays needed to be optimized, and the lack of multiplexing made the procedure more time consuming and demanded more DNA. The Sequenom methodology also focuses on only those nucleotides that are known to be cancer mutations and thus makes review of the sequence information considerably faster than Sanger. Next-Generation sequencing was cost prohibitive and has not been shown to work with DNAs isolated from FFPET. Thus, the Sequenom platform and the OncoCarta Panel provided the simplest, most rapid, sensitive and cost-effective method for detecting hot spot cancer mutations in degraded DNAs isolated from archival and routinely processed FFPET. The ColoCarta panel provides a more specific panel for colon cancer mutation detection and greatly reduces the amount of DNA needed for mutation profiling.</p> 
  <p>The frequencies and specific amino acid mutations detected here were similar to the COSMIC database and other publications [ 
   <xref ref-type="bibr" rid="B6">
     6 
   </xref>]. The small variation in frequency between our data and other reports may be attributed to differences in the stage of the samples analyzed, the number of samples considered, and the sensitivity of the technology [ 
   <xref ref-type="bibr" rid="B18">
     18 
   </xref>]. These observations, combined with the perfect match that we obtained between the expected and the detected mutations in our control cell lines, both fresh and FFPE, and the fact that mutations detected with OncoCarta and ColoCarta were identical, suggest that the technology is reliable and reproducible in DNAs isolated from FFPE samples.</p> 
  <p>In our study, the majority of tumors (60.3%) had one or more mutations in 
   <italic>
     KRAS 
   </italic>, 
   <italic>
     PIK3CA 
   </italic>, and 
   <italic>
     BRAF 
   </italic>. Mutations in these genes are likely to perturb many different and overlapping signaling pathways, including 
   <italic>
     PI3K/AKT, ERK/MAPK, SAPK/JNK, NFK? 
   </italic>, and others. We were also able to detect other less frequent mutations that are likely to perturb the same pathways and these may cause resistance to EGFR-targeted therapies, as reported for 
   <italic>
     KRAS 
   </italic>, 
   <italic>
     PIK3CA 
   </italic>and 
   <italic>
     BRAF 
   </italic>. For example, 
   <italic>
     AKT1 
   </italic>and 
   <italic>
     NRAS 
   </italic>are molecules that are downstream mediators of the EGFR signaling pathway, and mutations in these genes are likely to affect the response to drugs that target EGFR.</p> 
  <p>Mutations in ABL, 
   <italic>
     AKT1 
   </italic>, and 
   <italic>
     MET 
   </italic>were detected here but were not listed in COSMIC, probably due to the small number of samples analyzed. The 
   <italic>
     AKT1 
   </italic>-E17K mutation was initially identified as a SNP, rs34409589, but in a recent publication it was found to be a somatic mutation and was found in 3 of 51 colon cancers [ 
   <xref ref-type="bibr" rid="B27">
     27 
   </xref>]. The frequency of these mutations in this small study (51 samples) was 6% and is much greater than in the C0-7 samples (0.4%). This difference in frequencies may be because the Carpten et al [ 
   <xref ref-type="bibr" rid="B27">
     27 
   </xref>] samples were from more advanced stages than those from the C-07 trial. Moreover, they selected large tumors (&gt;100 mg) and containing more than 60% tumor cells. No such selection was done for our study, and samples were from stages II and III exclusively. The significance of 
   <italic>
     ABL1 
   </italic>and 
   <italic>
     AKT1 
   </italic>mutations for patient prediction and prognosis in our study is questionable given that they each were found in only in one sample and represented only 0.4% of the cases.</p> 
  <p>To our knowledge, this is the first report of 
   <italic>
     MET 
   </italic>mutations in the primary colon cancer, but a different 
   <italic>
     MET 
   </italic>mutation (N1118Y) was found in a lung metastasis of the large intestine [ 
   <xref ref-type="bibr" rid="B28">
     28 
   </xref>]. The 
   <italic>
     MET 
   </italic>mutations, R970C and T992I, were detected in 8 out of 239 C-07 colon cancers. These mutations correspond to 
   <italic>
     MET 
   </italic>-R988C and 
   <italic>
     MET 
   </italic>-T1010I, respectively, in the long form of MET which is the isoform referred to in the COSMIC database [ 
   <xref ref-type="bibr" rid="B29">
     29 
   </xref>]. The R970C and T992I mutations are located in the juxtamembrane segment of the protein and were detected in lung carcinoma [ 
   <xref ref-type="bibr" rid="B30">
     30 
   </xref>]. These mutations, when introduced into a lung cell line, increased focus formation, formation of colonies in soft agar, cell motility, and migration. These mutations also resulted in constitutive tyrosine phosphorylation on several cellular proteins including paxillin at key tyrosine residues and may account for the increased motility of cells with this mutation. Another critical amino acid in this location is a Ser 985, which, when phosphorylated, has been found to diminish 
   <italic>
     MET 
   </italic>signaling [ 
   <xref ref-type="bibr" rid="B31">
     31 
   </xref>]. If phosphorylation at Thr residue 992 (1010) reduces signaling, then the R992I mutation would inhibit this negative feedback and may result in constitutive signaling [ 
   <xref ref-type="bibr" rid="B30">
     30 
   </xref>].</p> 
  <p>If 
   <italic>
     MET 
   </italic>mutations confer an alternative activated signaling pathway, then these mutations could also confer resistance to anti-EGFR-based therapies or provide a new target for directed therapies. Therapeutic drugs have been developed to specifically target 
   <italic>
     MET 
   </italic>, including small molecule kinase inhibitors, anti- 
   <italic>
     MET 
   </italic>monoclonal antibodies, and inhibitors of HGF, the 
   <italic>
     MET 
   </italic>ligand. Invitro assays have demonstrated that a number of 
   <italic>
     MET 
   </italic>targeted therapies were able to prevent 
   <italic>
     MET 
   </italic>signaling, decrease cell viability, and limit cell motility and migration in vitro [ 
   <xref ref-type="bibr" rid="B32">
     32 
   </xref>]. The small molecule ARQ 197, a kinase inhibitor, has entered phase II clinical trials so may represent a possible therapeutic strategy for some colon tumors.</p> 
  <p>To our knowledge, this is also one of the most exhaustive analyses of mutation profiling of metastatic lymph nodes and their corresponding primary colon tumors. Our analysis showed that a majority of samples were concordant (89.7%) but in a few samples mutations were detected only in the primary tumor and not in the metastatic lymph node. Also in samples with 2 co-occurring mutations, the ratio of the double mutations varied in primary and lymph node tumors. Discordance in the genetic profile between primary tumors and the metastatic lymph nodes has been observed [ 
   <xref ref-type="bibr" rid="B33">
     33 
   </xref>]. Such data may indicate that tumor cell migration selects different cell populations from the one in the primary tumor. However, it is also possible that these mutational differences between the lymph node and the primary tumor are a result of tumor heterogeneity.</p> 
  <p>Another interesting observation in our study was that 
   <italic>
     BRAF 
   </italic>mutations were significantly correlated with poorly differentiated tumors and the prevalence of mucin; similar observations have been reported [ 
   <xref ref-type="bibr" rid="B25">
     25 
   </xref>, 
   <xref ref-type="bibr" rid="B26">
     26 
   </xref>]. These characteristics are both associated with a worse prognosis and are consistent with other reports associating 
   <italic>
     BRAF 
   </italic>mutations with a bad prognosis [ 
   <xref ref-type="bibr" rid="B3">
     3 
   </xref>]. However, in our study we found that there were 2 metastatic lymph nodes that did not maintain the 
   <italic>
     BRAF 
   </italic>mutation present in the corresponding primary tumor, suggesting that 
   <italic>
     BRAF 
   </italic>mutations are not essential for metastatic spread to the lymph node in all tumors. Clearly, additional studies would be required to understand these apparent inconsistencies; additional lymph node samples are not currently available but could be the subject of further studies when samples become available [ 
   <xref ref-type="bibr" rid="B3">
     3 
   </xref>].</p> 
 </sec> 
 <sec> 
  <h3>Conclusions</h3> 
  <p>The Sequenom platform provided a superior technology for the screening of 238 common hot spot cancer mutations in 19 genes. The frequent occurrence of 
   <italic>
     KRAS 
   </italic>, 
   <italic>
     PIK3CA 
   </italic>, and 
   <italic>
     BRAF 
   </italic>was confirmed, and mutations not detected before in colon cancer were found in 
   <italic>
     MET 
   </italic>and 
   <italic>
     ABL1 
   </italic>. Twenty-five assays from the OncoCarta were replexed to form a new panel, termed ColoCarta, which will be used to screen an additional 800 tumors from NSABP clinical trial C0-7 with the purpose of identifying prognostic or predictive markers for stage II and III colon cancer.</p> 
  <p> 
   <bold>
     Note Added in Proof: 
   </bold>Although MET-R988C and MET-T1010I mutations were listed in COSMIC as somatic cancer mutations, these nucleotide changes correspond to SNPs rs34589476 and rs56391007 in the NCBI SNP data base, respectively. The frequency for these SNPs is unknown so whether these nucleotide changes are associated with cancer is unknown.</p> 
 </sec> 
 <sec> 
  <h3>Competing interests</h3> 
  <p>The authors declare that they have no competing interests.</p> 
 </sec> 
 <sec> 
  <h3>Authors' contributions</h3> 
  <p>DF and PGG designed and carried out the experiments and participated in the drafting of the manuscript. YT graded tumors with regard to the degree of differentiation, mucin content, and defined tumor regions, S-IK carried out experiments. H-JC defined the tumor regions; SP participated in the coordination of the study; KLP-G designed and coordinated the study and drafted the manuscript. All authors have given final approval of the version to be published.</p> 
 </sec> 
 <sec> 
  <h3>Pre-publication history</h3> 
  <p>The pre-publication history for this paper can be accessed here:</p> 
  <p> 
   <ext-link ext-link-type="uri" xlink:href="http://www.biomedcentral.com/1471-2407/10/101/prepub">
     http://www.biomedcentral.com/1471-2407/10/101/prepub 
   </ext-link> </p> 
 </sec> 
 <back> 
  <sec> 
   <h3>Acknowledgements</h3> 
   <p>The authors would like to thank Melanie Finnigan, Bill Hiller, and Theresa Oeler for help in cutting and cataloging slides, and Hema Liyanage from Sequenom for replexing the assays. This study was supported by Public Health Service grants U10-CA-37377, U10-CA-69974, U10-CA-12027, and U10-CA-69651 from the National Cancer Institute, National Institutes of Health, and Department of Health and Human Services. This project is funded, in part, under a grant with the Pennsylvania Department of Health. The Department specifically disclaims responsibility for any analyses, interpretations or conclusions. The authors retain the right to provide a copy of the final manuscript to the NIH upon acceptance for journal publication, for public archiving in PubMed Central as soon as possible but no later than 12 months after publication by the journal.</p> 
  </sec> 
  <ref-list> 
   <ref id="B1"> 
    <mixed-citation publication-type="other"> 
     <name> 
      <surname>
        Ogino 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Nosho 
      </surname> 
      <given-names>
        K 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kirkner 
      </surname> 
      <given-names>
        GJ 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Shima 
      </surname> 
      <given-names>
        K 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Irahara 
      </surname> 
      <given-names>
        N 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kure 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Chan 
      </surname> 
      <given-names>
        AT 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Engelman 
      </surname> 
      <given-names>
        JA 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kraft 
      </surname> 
      <given-names>
        P 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Cantley 
      </surname> 
      <given-names>
        LC 
      </given-names> 
     </name> 
     <article-h3>
       PIK3CA Mutation Is Associated With Poor Prognosis Among Patients With Curatively Resected Colon Cancer 
     </article-h3> 
     <source />J Clin Oncol 
     <year>
       2009 
     </year> 
    </mixed-citation> 
   </ref> 
   <ref id="B2"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Andreyev 
      </surname> 
      <given-names>
        HJ 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Norman 
      </surname> 
      <given-names>
        AR 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Cunningham 
      </surname> 
      <given-names>
        D 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Oates 
      </surname> 
      <given-names>
        J 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Dix 
      </surname> 
      <given-names>
        BR 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Iacopetta 
      </surname> 
      <given-names>
        BJ 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Young 
      </surname> 
      <given-names>
        J 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Walsh 
      </surname> 
      <given-names>
        T 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Ward 
      </surname> 
      <given-names>
        R 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Hawkins 
      </surname> 
      <given-names>
        N 
      </given-names> 
     </name> 
     <article-h3>
       Kirsten ras mutations in patients with colorectal cancer: the 'RASCAL II' study 
     </article-h3> 
     <source />Br J Cancer 
     <year>
       2001 
     </year> 
     <volume>
       85 
     </volume> 
     <issue>
       5 
     </issue> 
     <fpage>
       692 
     </fpage> 
     <lpage>
       696 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1054/bjoc.2001.1964 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       11531254 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B3"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Samowitz 
      </surname> 
      <given-names>
        WS 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Sweeney 
      </surname> 
      <given-names>
        C 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Herrick 
      </surname> 
      <given-names>
        J 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Albertsen 
      </surname> 
      <given-names>
        H 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Levin 
      </surname> 
      <given-names>
        TR 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Murtaugh 
      </surname> 
      <given-names>
        MA 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Wolff 
      </surname> 
      <given-names>
        RK 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Slattery 
      </surname> 
      <given-names>
        ML 
      </given-names> 
     </name> 
     <article-h3>
       Poor Survival Associated with the BRAF V600E Mutation in Microsatellite-Stable Colon Cancers 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2005 
     </year> 
     <volume>
       65 
     </volume> 
     <issue>
       14 
     </issue> 
     <fpage>
       6063 
     </fpage> 
     <lpage>
       6069 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1158/0008-5472.CAN-05-0404 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       16024606 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B4"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Ogino 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Nosho 
      </surname> 
      <given-names>
        K 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kirkner 
      </surname> 
      <given-names>
        GJ 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kawasaki 
      </surname> 
      <given-names>
        T 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Meyerhardt 
      </surname> 
      <given-names>
        JA 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Loda 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Giovannucci 
      </surname> 
      <given-names>
        EL 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Fuchs 
      </surname> 
      <given-names>
        CS 
      </given-names> 
     </name> 
     <article-h3>
       CpG island methylator phenotype, microsatellite instability, BRAF mutation and clinical outcome in colon cancer 
     </article-h3> 
     <source />Gut 
     <year>
       2009 
     </year> 
     <volume>
       58 
     </volume> 
     <issue>
       1 
     </issue> 
     <fpage>
       90 
     </fpage> 
     <lpage>
       96 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1136/gut.2008.155473 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       18832519 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B5"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Conlin 
      </surname> 
      <given-names>
        A 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Smith 
      </surname> 
      <given-names>
        G 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Carey 
      </surname> 
      <given-names>
        FA 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Wolf 
      </surname> 
      <given-names>
        CR 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Steele 
      </surname> 
      <given-names>
        RJC 
      </given-names> 
     </name> 
     <article-h3>
       The prognostic significance of K-ras, p53, and APC mutations in colorectal carcinoma 
     </article-h3> 
     <source />Gut 
     <year>
       2005 
     </year> 
     <volume>
       54 
     </volume> 
     <issue>
       9 
     </issue> 
     <fpage>
       1283 
     </fpage> 
     <lpage>
       1286 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1136/gut.2005.066514 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       15843421 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B6"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Barault 
      </surname> 
      <given-names>
        L 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Veyrie 
      </surname> 
      <given-names>
        N 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Jooste 
      </surname> 
      <given-names>
        V 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Lecorre 
      </surname> 
      <given-names>
        D 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Chapusot 
      </surname> 
      <given-names>
        C 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Ferraz 
      </surname> 
      <given-names>
        JM 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Lievre 
      </surname> 
      <given-names>
        A 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Cortet 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Bouvier 
      </surname> 
      <given-names>
        AM 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Rat 
      </surname> 
      <given-names>
        P 
      </given-names> 
     </name> 
     <article-h3>
       Mutations in the RAS-MAPK, PI(3)K (phosphatidylinositol-3-OH kinase) signaling network correlate with poor survival in a population-based series of colon cancers 
     </article-h3> 
     <source />Int J Cancer 
     <year>
       2008 
     </year> 
     <volume>
       122 
     </volume> 
     <issue>
       10 
     </issue> 
     <fpage>
       2255 
     </fpage> 
     <lpage>
       2259 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1002/ijc.23388 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       18224685 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B7"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Khambata-Ford 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Garrett 
      </surname> 
      <given-names>
        CR 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Meropol 
      </surname> 
      <given-names>
        NJ 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Basik 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Harbison 
      </surname> 
      <given-names>
        CT 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Wu 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Wong 
      </surname> 
      <given-names>
        TW 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Huang 
      </surname> 
      <given-names>
        X 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Takimoto 
      </surname> 
      <given-names>
        CH 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Godwin 
      </surname> 
      <given-names>
        AK 
      </given-names> 
     </name> 
     <article-h3>
       Expression of epiregulin and amphiregulin and K-ras mutation status predict disease control in metastatic colorectal cancer patients treated with cetuximab 
     </article-h3> 
     <source />J Clin Oncol 
     <year>
       2007 
     </year> 
     <volume>
       25 
     </volume> 
     <issue>
       22 
     </issue> 
     <fpage>
       3230 
     </fpage> 
     <lpage>
       3237 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1200/JCO.2006.10.5437 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       17664471 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B8"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Linardou 
      </surname> 
      <given-names>
        H 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Dahabreh 
      </surname> 
      <given-names>
        IJ 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kanaloupiti 
      </surname> 
      <given-names>
        D 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Siannis 
      </surname> 
      <given-names>
        F 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Bafaloukos 
      </surname> 
      <given-names>
        D 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kosmidis 
      </surname> 
      <given-names>
        P 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Papadimitriou 
      </surname> 
      <given-names>
        CA 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Murray 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <article-h3>
       Assessment of somatic k-RAS mutations as a mechanism associated with resistance to EGFR-targeted agents: a systematic review and meta-analysis of studies in advanced non-small-cell lung cancer and metastatic colorectal cancer 
     </article-h3> 
     <source />The Lancet Oncology 
     <year>
       2008 
     </year> 
     <volume>
       9 
     </volume> 
     <issue>
       10 
     </issue> 
     <fpage>
       962 
     </fpage> 
     <lpage>
       972 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1016/S1470-2045(08)70206-7 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       18804418 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B9"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Lievre 
      </surname> 
      <given-names>
        A 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Bachet 
      </surname> 
      <given-names>
        JB 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Boige 
      </surname> 
      <given-names>
        V 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Cayre 
      </surname> 
      <given-names>
        A 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Le Corre 
      </surname> 
      <given-names>
        D 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Buc 
      </surname> 
      <given-names>
        E 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Ychou 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Bouche 
      </surname> 
      <given-names>
        O 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Landi 
      </surname> 
      <given-names>
        B 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Louvet 
      </surname> 
      <given-names>
        C 
      </given-names> 
     </name> 
     <article-h3>
       KRAS mutations as an independent prognostic factor in patients with advanced colorectal cancer treated with cetuximab 
     </article-h3> 
     <source />J Clin Oncol 
     <year>
       2008 
     </year> 
     <volume>
       26 
     </volume> 
     <issue>
       3 
     </issue> 
     <fpage>
       374 
     </fpage> 
     <lpage>
       379 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1200/JCO.2007.12.5906 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       18202412 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B10"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Amado 
      </surname> 
      <given-names>
        RG 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Wolf 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Peeters 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Van Cutsem 
      </surname> 
      <given-names>
        E 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Siena 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Freeman 
      </surname> 
      <given-names>
        DJ 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Juan 
      </surname> 
      <given-names>
        T 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Sikorski 
      </surname> 
      <given-names>
        R 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Suggs 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Radinsky 
      </surname> 
      <given-names>
        R 
      </given-names> 
     </name> 
     <article-h3>
       Wild-type KRAS is required for panitumumab efficacy in patients with metastatic colorectal cancer 
     </article-h3> 
     <source />J Clin Oncol 
     <year>
       2008 
     </year> 
     <volume>
       26 
     </volume> 
     <issue>
       10 
     </issue> 
     <fpage>
       1626 
     </fpage> 
     <lpage>
       1634 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1200/JCO.2007.14.7116 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       18316791 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B11"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Baselga 
      </surname> 
      <given-names>
        J 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Rosen 
      </surname> 
      <given-names>
        N 
      </given-names> 
     </name> 
     <article-h3>
       Determinants of RASistance to anti-epidermal growth factor receptor agents 
     </article-h3> 
     <source />J Clin Oncol 
     <year>
       2008 
     </year> 
     <volume>
       26 
     </volume> 
     <issue>
       10 
     </issue> 
     <fpage>
       1582 
     </fpage> 
     <lpage>
       1584 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1200/JCO.2007.15.3700 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       18316790 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B12"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Sartore-Bianchi 
      </surname> 
      <given-names>
        A 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Martini 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Molinari 
      </surname> 
      <given-names>
        F 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Veronese 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Nichelatti 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Artale 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Di Nicolantonio 
      </surname> 
      <given-names>
        F 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Saletti 
      </surname> 
      <given-names>
        P 
      </given-names> 
     </name> 
     <name> 
      <surname>
        De Dosso 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Mazzucchelli 
      </surname> 
      <given-names>
        L 
      </given-names> 
     </name> 
     <article-h3>
       PIK3CA Mutations in Colorectal Cancer Are Associated with Clinical Resistance to EGFR-Targeted Monoclonal Antibodies 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2009 
     </year> 
     <volume>
       69 
     </volume> 
     <issue>
       5 
     </issue> 
     <fpage>
       1851 
     </fpage> 
     <lpage>
       1857 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1158/0008-5472.CAN-08-2466 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       19223544 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B13"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Di Nicolantonio 
      </surname> 
      <given-names>
        F 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Martini 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Molinari 
      </surname> 
      <given-names>
        F 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Sartore-Bianchi 
      </surname> 
      <given-names>
        A 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Arena 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Saletti 
      </surname> 
      <given-names>
        P 
      </given-names> 
     </name> 
     <name> 
      <surname>
        De Dosso 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Mazzucchelli 
      </surname> 
      <given-names>
        L 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Frattini 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Siena 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <article-h3>
       Wild-type BRAF is required for response to panitumumab or cetuximab in metastatic colorectal cancer 
     </article-h3> 
     <source />J Clin Oncol 
     <year>
       2008 
     </year> 
     <volume>
       26 
     </volume> 
     <issue>
       35 
     </issue> 
     <fpage>
       5705 
     </fpage> 
     <lpage>
       5712 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1200/JCO.2008.18.0786 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       19001320 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B14"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Lambrechts 
      </surname> 
      <given-names>
        D 
      </given-names> 
     </name> 
     <name> 
      <surname>
        De Roock 
      </surname> 
      <given-names>
        W 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Prenen 
      </surname> 
      <given-names>
        H 
      </given-names> 
     </name> 
     <name> 
      <surname>
        De Schutter 
      </surname> 
      <given-names>
        J 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Jacobs 
      </surname> 
      <given-names>
        B 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Biesmans 
      </surname> 
      <given-names>
        B 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Claes 
      </surname> 
      <given-names>
        B 
      </given-names> 
     </name> 
     <name> 
      <surname>
        De Hertogh 
      </surname> 
      <given-names>
        G 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Van Cutsem 
      </surname> 
      <given-names>
        E 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Tejpar 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <article-h3>
       The role of KRAS, BRAF, NRAS, and PIK3CA mutations as markers of resistance to cetuximab in chemorefractory metastatic colorectal cancer 
     </article-h3> 
     <source />J Clin Oncol (Meeting Abstracts) 
     <year>
       2009 
     </year> 
     <volume>
       27 
     </volume> 
     <issue>
       15S 
     </issue> 
     <fpage>
       4020 
     </fpage> 
    </mixed-citation> 
   </ref> 
   <ref id="B15"> 
    <mixed-citation publication-type="other"> 
     <name> 
      <surname>
        Prenen 
      </surname> 
      <given-names>
        H 
      </given-names> 
     </name> 
     <name> 
      <surname>
        De Schutter 
      </surname> 
      <given-names>
        J 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Jacobs 
      </surname> 
      <given-names>
        B 
      </given-names> 
     </name> 
     <name> 
      <surname>
        De Roock 
      </surname> 
      <given-names>
        W 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Biesmans 
      </surname> 
      <given-names>
        B 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Claes 
      </surname> 
      <given-names>
        B 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Lambrechts 
      </surname> 
      <given-names>
        D 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Van Cutsem 
      </surname> 
      <given-names>
        E 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Tejpar 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <article-h3>
       PIK3CA Mutations Are Not a Major Determinant of Resistance to the Epidermal Growth Factor Receptor Inhibitor Cetuximab in Metastatic Colorectal Cancer 
     </article-h3> 
     <source />Clin Cancer Res 
     <year>
       2009 
     </year> 
     <pub-id pub-id-type="pmid">
       19366826 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B16"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Thomas 
      </surname> 
      <given-names>
        RK 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Baker 
      </surname> 
      <given-names>
        AC 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Debiasi 
      </surname> 
      <given-names>
        RM 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Winckler 
      </surname> 
      <given-names>
        W 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Laframboise 
      </surname> 
      <given-names>
        T 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Lin 
      </surname> 
      <given-names>
        WM 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Wang 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Feng 
      </surname> 
      <given-names>
        W 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Zander 
      </surname> 
      <given-names>
        T 
      </given-names> 
     </name> 
     <name> 
      <surname>
        MacConaill 
      </surname> 
      <given-names>
        L 
      </given-names> 
     </name> 
     <article-h3>
       High-throughput oncogene mutation profiling in human cancer 
     </article-h3> 
     <source />Nat Genet 
     <year>
       2007 
     </year> 
     <volume>
       39 
     </volume> 
     <issue>
       3 
     </issue> 
     <fpage>
       347 
     </fpage> 
     <lpage>
       351 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1038/ng1975 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       17293865 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B17"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Wood 
      </surname> 
      <given-names>
        LD 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Parsons 
      </surname> 
      <given-names>
        DW 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Jones 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Lin 
      </surname> 
      <given-names>
        J 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Sjoblom 
      </surname> 
      <given-names>
        T 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Leary 
      </surname> 
      <given-names>
        RJ 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Shen 
      </surname> 
      <given-names>
        D 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Boca 
      </surname> 
      <given-names>
        SM 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Barber 
      </surname> 
      <given-names>
        T 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Ptak 
      </surname> 
      <given-names>
        J 
      </given-names> 
     </name> 
     <article-h3>
       The genomic landscapes of human breast and colorectal cancers 
     </article-h3> 
     <source />Science 
     <year>
       2007 
     </year> 
     <volume>
       318 
     </volume> 
     <issue>
       5853 
     </issue> 
     <fpage>
       1108 
     </fpage> 
     <lpage>
       1113 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1126/science.1145720 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       17932254 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B18"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Vivante 
      </surname> 
      <given-names>
        A 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Amariglio 
      </surname> 
      <given-names>
        N 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Koren-Michowitz 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Ashur-Fabian 
      </surname> 
      <given-names>
        O 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Nagler 
      </surname> 
      <given-names>
        A 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Rechavi 
      </surname> 
      <given-names>
        G 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Cohen 
      </surname> 
      <given-names>
        Y 
      </given-names> 
     </name> 
     <article-h3>
       High-throughput, sensitive and quantitative assay for the detection of BCR-ABL kinase domain mutations 
     </article-h3> 
     <source />Leukemia 
     <year>
       2007 
     </year> 
     <volume>
       21 
     </volume> 
     <issue>
       6 
     </issue> 
     <fpage>
       1318 
     </fpage> 
     <lpage>
       1321 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1038/sj.leu.2404635 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       17330094 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B19"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        van Puijenbroek 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Dierssen 
      </surname> 
      <given-names>
        JW 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Stanssens 
      </surname> 
      <given-names>
        P 
      </given-names> 
     </name> 
     <name> 
      <surname>
        van Eijk 
      </surname> 
      <given-names>
        R 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Cleton-Jansen 
      </surname> 
      <given-names>
        AM 
      </given-names> 
     </name> 
     <name> 
      <surname>
        van Wezel 
      </surname> 
      <given-names>
        T 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Morreau 
      </surname> 
      <given-names>
        H 
      </given-names> 
     </name> 
     <article-h3>
       Mass spectrometry-based loss of heterozygosity analysis of single-nucleotide polymorphism loci in paraffin embedded tumors using the MassEXTEND assay: single-nucleotide polymorphism loss of heterozygosity analysis of the protein tyrosine phosphatase receptor type J in familial colorectal cancer 
     </article-h3> 
     <source />J Mol Diagn 
     <year>
       2005 
     </year> 
     <volume>
       7 
     </volume> 
     <issue>
       5 
     </issue> 
     <fpage>
       623 
     </fpage> 
     <lpage>
       630 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16258161 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B20"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Kuebler 
      </surname> 
      <given-names>
        JP 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Wieand 
      </surname> 
      <given-names>
        HS 
      </given-names> 
     </name> 
     <name> 
      <surname>
        O'Connell 
      </surname> 
      <given-names>
        MJ 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Smith 
      </surname> 
      <given-names>
        RE 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Colangelo 
      </surname> 
      <given-names>
        LH 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Yothers 
      </surname> 
      <given-names>
        G 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Petrelli 
      </surname> 
      <given-names>
        NJ 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Findlay 
      </surname> 
      <given-names>
        MP 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Seay 
      </surname> 
      <given-names>
        TE 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Atkins 
      </surname> 
      <given-names>
        JN 
      </given-names> 
     </name> 
     <article-h3>
       Oxaliplatin combined with weekly bolus fluorouracil and leucovorin as surgical adjuvant chemotherapy for stage II and III colon cancer: results from NSABP C-07 
     </article-h3> 
     <source />J Clin Oncol 
     <year>
       2007 
     </year> 
     <volume>
       25 
     </volume> 
     <issue>
       16 
     </issue> 
     <fpage>
       2198 
     </fpage> 
     <lpage>
       2204 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1200/JCO.2006.08.2974 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       17470851 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B21"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Forbes 
      </surname> 
      <given-names>
        SA 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Bhamra 
      </surname> 
      <given-names>
        G 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Bamford 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Dawson 
      </surname> 
      <given-names>
        E 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kok 
      </surname> 
      <given-names>
        C 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Clements 
      </surname> 
      <given-names>
        J 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Menzies 
      </surname> 
      <given-names>
        A 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Teague 
      </surname> 
      <given-names>
        JW 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Futreal 
      </surname> 
      <given-names>
        PA 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Stratton 
      </surname> 
      <given-names>
        MR 
      </given-names> 
     </name> 
     <article-h3>
       The Catalogue of Somatic Mutations in Cancer (COSMIC) 
     </article-h3> 
     <source />Curr Protoc Hum Genet 
     <year>
       2008 
     </year> 
     <volume>
       Chapter 10 
     </volume> 
     <issue>
       Unit 10 
     </issue> 
     <fpage>
       11 
     </fpage> 
     <pub-id pub-id-type="pmid">
       18428421 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B22"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Baudis 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Cleary 
      </surname> 
      <given-names>
        ML 
      </given-names> 
     </name> 
     <article-h3>
       Progenetix.net: an online repository for molecular cytogenetic aberration data 
     </article-h3> 
     <source />Bioinformatics 
     <year>
       2001 
     </year> 
     <volume>
       17 
     </volume> 
     <issue>
       12 
     </issue> 
     <fpage>
       1228 
     </fpage> 
     <lpage>
       1229 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1093/bioinformatics/17.12.1228 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       11751233 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B23"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Pratilas 
      </surname> 
      <given-names>
        C 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Solit 
      </surname> 
      <given-names>
        D 
      </given-names> 
     </name> 
     <article-h3>
       Therapeutic strategies for targeting BRAF in human cancer 
     </article-h3> 
     <source />Rev Recent Clin Trials 
     <year>
       2007 
     </year> 
     <volume>
       2 
     </volume> 
     <issue>
       2 
     </issue> 
     <fpage>
       121 
     </fpage> 
     <lpage>
       134 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.2174/157488707780599393 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       18473997 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B24"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Yuen 
      </surname> 
      <given-names>
        ST 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Davies 
      </surname> 
      <given-names>
        H 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Chan 
      </surname> 
      <given-names>
        TL 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Ho 
      </surname> 
      <given-names>
        JW 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Bignell 
      </surname> 
      <given-names>
        GR 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Cox 
      </surname> 
      <given-names>
        C 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Stephens 
      </surname> 
      <given-names>
        P 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Edkins 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Tsui 
      </surname> 
      <given-names>
        WW 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Chan 
      </surname> 
      <given-names>
        AS 
      </given-names> 
     </name> 
     <article-h3>
       Similarity of the phenotypic patterns associated with BRAF and KRAS mutations in colorectal neoplasia 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2002 
     </year> 
     <volume>
       62 
     </volume> 
     <issue>
       22 
     </issue> 
     <fpage>
       6451 
     </fpage> 
     <lpage>
       6455 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12438234 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B25"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Tanaka 
      </surname> 
      <given-names>
        H 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Deng 
      </surname> 
      <given-names>
        G 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Matsuzaki 
      </surname> 
      <given-names>
        K 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kakar 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kim 
      </surname> 
      <given-names>
        GE 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Miura 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Sleisenger 
      </surname> 
      <given-names>
        MH 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kim 
      </surname> 
      <given-names>
        YS 
      </given-names> 
     </name> 
     <article-h3>
       BRAF mutation, CpG island methylator phenotype and microsatellite instability occur more frequently and concordantly in mucinous than non-mucinous colorectal cancer 
     </article-h3> 
     <source />Int J Cancer 
     <year>
       2006 
     </year> 
     <volume>
       118 
     </volume> 
     <issue>
       11 
     </issue> 
     <fpage>
       2765 
     </fpage> 
     <lpage>
       2771 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1002/ijc.21701 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       16381005 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B26"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Li 
      </surname> 
      <given-names>
        WQ 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kawakami 
      </surname> 
      <given-names>
        K 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Ruszkiewicz 
      </surname> 
      <given-names>
        A 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Bennett 
      </surname> 
      <given-names>
        G 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Moore 
      </surname> 
      <given-names>
        J 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Iacopetta 
      </surname> 
      <given-names>
        B 
      </given-names> 
     </name> 
     <article-h3>
       BRAF mutations are associated with distinctive clinical, pathological and molecular features of colorectal cancer independently of microsatellite instability status 
     </article-h3> 
     <source />Mol Cancer 
     <year>
       2006 
     </year> 
     <volume>
       5 
     </volume> 
     <fpage>
       2 
     </fpage> 
     <pub-id pub-id-type="doi">
       10.1186/1476-4598-5-2 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       16403224 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B27"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Carpten 
      </surname> 
      <given-names>
        JD 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Faber 
      </surname> 
      <given-names>
        AL 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Horn 
      </surname> 
      <given-names>
        C 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Donoho 
      </surname> 
      <given-names>
        GP 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Briggs 
      </surname> 
      <given-names>
        SL 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Robbins 
      </surname> 
      <given-names>
        CM 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Hostetter 
      </surname> 
      <given-names>
        G 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Boguslawski 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Moses 
      </surname> 
      <given-names>
        TY 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Savage 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <article-h3>
       A transforming mutation in the pleckstrin homology domain of AKT1 in cancer 
     </article-h3> 
     <source />Nature 
     <year>
       2007 
     </year> 
     <volume>
       448 
     </volume> 
     <issue>
       7152 
     </issue> 
     <fpage>
       439 
     </fpage> 
     <lpage>
       444 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1038/nature05933 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       17611497 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B28"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Lorenzato 
      </surname> 
      <given-names>
        A 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Olivero 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Patane 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Rosso 
      </surname> 
      <given-names>
        E 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Oliaro 
      </surname> 
      <given-names>
        A 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Comoglio 
      </surname> 
      <given-names>
        PM 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Di Renzo 
      </surname> 
      <given-names>
        MF 
      </given-names> 
     </name> 
     <article-h3>
       Novel Somatic Mutations of the MET Oncogene in Human Carcinoma Metastases Activating Cell Motility and Invasion 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2002 
     </year> 
     <volume>
       62 
     </volume> 
     <issue>
       23 
     </issue> 
     <fpage>
       7025 
     </fpage> 
     <lpage>
       7030 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12460923 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B29"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Loriaux 
      </surname> 
      <given-names>
        MM 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Levine 
      </surname> 
      <given-names>
        RL 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Tyner 
      </surname> 
      <given-names>
        JW 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Frohling 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Scholl 
      </surname> 
      <given-names>
        C 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Stoffregen 
      </surname> 
      <given-names>
        EP 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Wernig 
      </surname> 
      <given-names>
        G 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Erickson 
      </surname> 
      <given-names>
        H 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Eide 
      </surname> 
      <given-names>
        CA 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Berger 
      </surname> 
      <given-names>
        R 
      </given-names> 
     </name> 
     <article-h3>
       High-throughput sequence analysis of the tyrosine kinome in acute myeloid leukemia 
     </article-h3> 
     <source />Blood 
     <year>
       2008 
     </year> 
     <volume>
       111 
     </volume> 
     <issue>
       9 
     </issue> 
     <fpage>
       4788 
     </fpage> 
     <lpage>
       4796 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1182/blood-2007-07-101394 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       18252861 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B30"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Ma 
      </surname> 
      <given-names>
        PC 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kijima 
      </surname> 
      <given-names>
        T 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Maulik 
      </surname> 
      <given-names>
        G 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Fox 
      </surname> 
      <given-names>
        EA 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Sattler 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Griffin 
      </surname> 
      <given-names>
        JD 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Johnson 
      </surname> 
      <given-names>
        BE 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Salgia 
      </surname> 
      <given-names>
        R 
      </given-names> 
     </name> 
     <article-h3>
       c-MET Mutational Analysis in Small Cell Lung Cancer: Novel Juxtamembrane Domain Mutations Regulating Cytoskeletal Functions 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2003 
     </year> 
     <volume>
       63 
     </volume> 
     <issue>
       19 
     </issue> 
     <fpage>
       6272 
     </fpage> 
     <lpage>
       6281 
     </lpage> 
     <pub-id pub-id-type="pmid">
       14559814 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B31"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Hashigasako 
      </surname> 
      <given-names>
        A 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Machide 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Nakamura 
      </surname> 
      <given-names>
        T 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Matsumoto 
      </surname> 
      <given-names>
        K 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Nakamura 
      </surname> 
      <given-names>
        T 
      </given-names> 
     </name> 
     <article-h3>
       Bi-directional Regulation of Ser-985 Phosphorylation of c-Met via Protein Kinase C and Protein Phosphatase 2A Involves c-Met Activation and Cellular Responsiveness to Hepatocyte Growth Factor 
     </article-h3> 
     <source />J Biol Chem 
     <year>
       2004 
     </year> 
     <volume>
       279 
     </volume> 
     <issue>
       25 
     </issue> 
     <fpage>
       26445 
     </fpage> 
     <lpage>
       26452 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1074/jbc.M314254200 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       15075332 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B32"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Seiwert 
      </surname> 
      <given-names>
        TY 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Jagadeeswaran 
      </surname> 
      <given-names>
        R 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Faoro 
      </surname> 
      <given-names>
        L 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Janamanchi 
      </surname> 
      <given-names>
        V 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Nallasura 
      </surname> 
      <given-names>
        V 
      </given-names> 
     </name> 
     <name> 
      <surname>
        El Dinali 
      </surname> 
      <given-names>
        M 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Yala 
      </surname> 
      <given-names>
        S 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kanteti 
      </surname> 
      <given-names>
        R 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Cohen 
      </surname> 
      <given-names>
        EE 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Lingen 
      </surname> 
      <given-names>
        MW 
      </given-names> 
     </name> 
     <article-h3>
       The MET receptor tyrosine kinase is a potential novel therapeutic target for head and neck squamous cell carcinoma 
     </article-h3> 
     <source />Cancer Res 
     <year>
       2009 
     </year> 
     <volume>
       69 
     </volume> 
     <issue>
       7 
     </issue> 
     <fpage>
       3021 
     </fpage> 
     <lpage>
       3031 
     </lpage> 
     <pub-id pub-id-type="doi">
       10.1158/0008-5472.CAN-08-2881 
     </pub-id> 
     <pub-id pub-id-type="pmid">
       19318576 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="B33"> 
    <mixed-citation publication-type="journal"> 
     <name> 
      <surname>
        Gamblin 
      </surname> 
      <given-names>
        TC 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Finkelstein 
      </surname> 
      <given-names>
        SD 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Upsal 
      </surname> 
      <given-names>
        N 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Kaye 
      </surname> 
      <given-names>
        JD 
      </given-names> 
     </name> 
     <name> 
      <surname>
        Blumberg 
      </surname> 
      <given-names>
        D 
      </given-names> 
     </name> 
     <article-h3>
       Microdissection-based allelotyping: a novel technique to determine the temporal sequence and biological aggressiveness of colorectal cancer 
     </article-h3> 
     <source />Am Surg 
     <year>
       2006 
     </year> 
     <volume>
       72 
     </volume> 
     <issue>
       5 
     </issue> 
     <fpage>
       445 
     </fpage> 
     <lpage>
       453 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16719202 
     </pub-id> 
    </mixed-citation> 
   </ref> 
  </ref-list> 
 </back> 
</article>