<article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"> 
 <!--?properties open_acces=""?--> 
 <front> 
  <journal-meta> 
   <journal-id journal-id-type="nlm-ta">
     PLoS One 
   </journal-id> 
   <journal-id journal-id-type="publisher-id">
     plos 
   </journal-id> 
   <journal-id journal-id-type="pmc">
     plosone 
   </journal-id> 
   <journal-h3-group> 
    <journal-h3>
      PLoS ONE 
    </journal-h3> 
   </journal-h3-group> 
   <issn pub-type="epub">
     1932-6203 
   </issn> 
   <publisher> 
    <publisher-name>
      Public Library of Science 
    </publisher-name> 
    <publisher-loc>
      San Francisco, USA 
    </publisher-loc> 
   </publisher> 
  </journal-meta> 
  <article-meta> 
   <article-id pub-id-type="pmid">
     20856805 
   </article-id> 
   <article-id pub-id-type="pmc">
     2939879 
   </article-id> 
   <article-id pub-id-type="publisher-id">
     10-PONE-RA-18406R1 
   </article-id> 
   <article-id pub-id-type="doi">
     10.1371/journal.pone.0012737 
   </article-id> 
   <article-categories> 
    <subj-group subj-group-type="heading"> 
     <subject>
       Research Article 
     </subject> 
    </subj-group> 
    <subj-group subj-group-type="Discipline"> 
     <subject>
       Genetics and Genomics/Functional Genomics 
     </subject> 
     <subject>
       Genetics and Genomics/Gene Expression 
     </subject> 
     <subject>
       Molecular Biology/Bioinformatics 
     </subject> 
     <subject>
       Physiology/Genomics 
     </subject> 
     <subject>
       Physiology/Reproductive Physiology 
     </subject> 
     <subject>
       Marine and Aquatic Sciences/Bioinformatics 
     </subject> 
     <subject>
       Marine and Aquatic Sciences/Fisheries 
     </subject> 
     <subject>
       Marine and Aquatic Sciences/Genetics, Genomics, and Barcoding 
     </subject> 
    </subj-group> 
   </article-categories> 
   <h3-group> 
    <article-h3>
      Queen Conch ( 
     <italic>
       Strombus gigas 
     </italic>) Testis Regresses during the Reproductive Season at Nearshore Sites in the Florida Keys 
    </article-h3> 
    <alt-h3 alt-h3-type="running-head">
      Queen Conch Testis Regression 
    </alt-h3> 
   </h3-group> 
   <contrib-group> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Spade 
      </surname> 
      <given-names>
        Daniel J. 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1"> 
      <sup>1</sup> 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Griffitt 
      </surname> 
      <given-names>
        Robert J. 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1"> 
      <sup>1</sup> 
     </xref> 
     <xref ref-type="author-notes" rid="fn1"> 
      <sup>¤</sup> 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Liu 
      </surname> 
      <given-names>
        Li 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff2"> 
      <sup>2</sup> 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Brown-Peterson 
      </surname> 
      <given-names>
        Nancy J. 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff3"> 
      <sup>3</sup> 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Kroll 
      </surname> 
      <given-names>
        Kevin J. 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1"> 
      <sup>1</sup> 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Feswick 
      </surname> 
      <given-names>
        April 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1"> 
      <sup>1</sup> 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Glazer 
      </surname> 
      <given-names>
        Robert A. 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff4"> 
      <sup>4</sup> 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Barber 
      </surname> 
      <given-names>
        David S. 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1"> 
      <sup>1</sup> 
     </xref> 
    </contrib> 
    <contrib contrib-type="author"> 
     <name> 
      <surname>
        Denslow 
      </surname> 
      <given-names>
        Nancy D. 
      </given-names> 
     </name> 
     <xref ref-type="aff" rid="aff1"> 
      <sup>1</sup> 
     </xref> 
     <xref ref-type="corresp" rid="cor1"> 
      <sup>*</sup> 
     </xref> 
    </contrib> 
   </contrib-group> 
   <aff id="aff1"> 
    <label>1</label> 
    <addr-line>
      Department of Physiological Sciences and Center for Environmental and Human Toxicology, University of Florida, Gainesville, Florida, United States of America 
    </addr-line> 
   </aff> 
   <aff id="aff2"> 
    <label>2</label> 
    <addr-line>
      Interdisciplinary Center for Biotechnology Research, University of Florida, Gainesville, Florida, United States of America 
    </addr-line> 
   </aff> 
   <aff id="aff3"> 
    <label>3</label> 
    <addr-line>
      Department of Coastal Sciences, University of Southern Mississippi, Ocean Springs, Mississippi, United States of America 
    </addr-line> 
   </aff> 
   <aff id="aff4"> 
    <label>4</label> 
    <addr-line>
      Fish and Wildlife Research Institute, Florida Fish and Wildlife Conservation Commission, Marathon, Florida, United States of America 
    </addr-line> 
   </aff> 
   <contrib-group> 
    <contrib contrib-type="editor"> 
     <name> 
      <surname>
        Orban 
      </surname> 
      <given-names>
        Laszlo 
      </given-names> 
     </name> 
     <role>
       Editor 
     </role> 
     <xref ref-type="aff" rid="edit1"></xref> 
    </contrib> 
   </contrib-group> 
   <aff id="edit1">
     Temasek Life Sciences Laboratory, Singapore 
   </aff> 
   <author-notes> 
    <corresp id="cor1">
      * E-mail: 
     <email>
       ndenslow@ufl.edu 
     </email> 
    </corresp> 
    <fn fn-type="con"> 
     <p>Conceived and designed the experiments: DJS RJG KK RAG DB ND. Performed the experiments: DJS RJG NJBP KK AF RAG. Analyzed the data: DJS RJG LL NJBP AF. Contributed reagents/materials/analysis tools: RAG DB ND. Wrote the paper: DJS RJG NJBP AF RAG DB ND.</p> 
    </fn> 
    <fn fn-type="current-aff" id="fn1"> 
     <label>¤</label> 
     <p>Current address: Department of Coastal Sciences, University of Southern Mississippi, Ocean Springs, Mississippi, United States of America</p> 
    </fn> 
   </author-notes> 
   <pub-date pub-type="collection"> 
    <year>
      2010 
    </year> 
   </pub-date> 
   <pub-date pub-type="epub"> 
    <day>
      15 
    </day> 
    <month>
      9 
    </month> 
    <year>
      2010 
    </year> 
   </pub-date> 
   <volume>
     5 
   </volume> 
   <issue>
     9 
   </issue> 
   <elocation-id>
     e12737 
   </elocation-id> 
   <history> 
    <date date-type="received"> 
     <day>
       28 
     </day> 
     <month>
       4 
     </month> 
     <year>
       2010 
     </year> 
    </date> 
    <date date-type="accepted"> 
     <day>
       2 
     </day> 
     <month>
       8 
     </month> 
     <year>
       2010 
     </year> 
    </date> 
   </history> 
   <permissions> 
    <copyright-statement>
      Spade et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited. 
    </copyright-statement> 
   </permissions> 
   <abstract>
    <h3>Abstract</h3> 
    <sec> 
     <h3>Background</h3> 
     <p>Queen conch ( 
      <italic>
        Strombus gigas 
      </italic>) reproduction is inhibited in nearshore areas of the Florida Keys, relative to the offshore environment where conchs reproduce successfully. Nearshore reproductive failure is possibly a result of exposure to environmental factors, including heavy metals, which are likely to accumulate close to shore. Metals such as Cu and Zn are detrimental to reproduction in many mollusks.</p> 
    </sec> 
    <sec> 
     <h3>Methodology/Principal Findings</h3> 
     <p>Histology shows gonadal atrophy in nearshore conchs as compared to reproductively healthy offshore conchs. In order to determine molecular mechanisms leading to tissue changes and reproductive failure, a microarray was developed. A normalized cDNA library for queen conch was constructed and sequenced using the 454 Life Sciences GS-FLX pyrosequencer, producing 27,723 assembled contigs and 7,740 annotated transcript sequences. The resulting sequences were used to design the microarray. Microarray analysis of conch testis indicated differential regulation of 255 genes (p&lt;0.01) in nearshore conch, relative to offshore. Changes in expression for three of four transcripts of interest were confirmed using real-time reverse transcription polymerase chain reaction. Gene Ontology enrichment analysis indicated changes in biological processes: respiratory chain (GO:0015992), spermatogenesis (GO:0007283), small GTPase-mediated signal transduction (GO:0007264), and others. Inductively coupled plasma-mass spectrometry analysis indicated that Zn and possibly Cu were elevated in some nearshore conch tissues.</p> 
    </sec> 
    <sec> 
     <h3>Conclusions/Significance</h3> 
     <p>Congruence between testis histology and microarray data suggests that nearshore conch testes regress during the reproductive season, while offshore conch testes develop normally. Possible mechanisms underlying the testis regression observed in queen conch in the nearshore Florida Keys include a disruption of small GTPase (Ras)-mediated signaling in testis development. Additionally, elevated tissue levels of Cu (34.77 ng/mg in testis) and Zn (831.85 ng/mg in digestive gland, 83.96 ng/mg in testis) nearshore are similar to reported levels resulting in reproductive inhibition in other gastropods, indicating that these metals possibly contribute to NS conch reproductive failure.</p> 
    </sec> 
   </abstract> 
   <counts> 
    <page-count count="14"></page-count> 
   </counts> 
  </article-meta> 
 </front> 
 <sec id="s1"> 
  <h3>Introduction</h3> 
  <p>Queen conch ( 
   <italic>
     Strombus gigas 
   </italic>) is a species of significant ecological and economic importance throughout its range. For example, the estimated economic value of the annual conch fishery in the Bahamas is approximately $4.457 million, representing 9,800 seasonal jobs 
   <xref ref-type="bibr" rid="pone.0012737-Chakalall1">
     [1] 
   </xref>. The queen conch is also a large benthic invertebrate associated with coral reef ecosystems, and therefore could serve as an indicator species for toxic effects contributing to the decline of the Florida coral reef ecosystem. As a result of the queen conch population decline in Florida, a complete moratorium on the Florida conch fishery was declared in 1986 
   <xref ref-type="bibr" rid="pone.0012737-Delgado1">
     [2] 
   </xref>, 
   <xref ref-type="bibr" rid="pone.0012737-Glazer1">
     [3] 
   </xref>. The queen conch was listed under the Convention on International Trade in Endangered Species' (CITES) Appendix II in 1992 
   <xref ref-type="bibr" rid="pone.0012737-Davis1">
     [4] 
   </xref>. However, recovery of adult conchs in spawning aggregations within the Florida Keys has been modest. In 2001, the number of adult conchs in offshore (OS) spawning aggregations was estimated at 27,000, up from a lowest observed estimate of 5,750 in 1992, according to transect data collected by the Florida Fish and Wildlife Conservation Commission 
   <xref ref-type="bibr" rid="pone.0012737-Glazer1">
     [3] 
   </xref>. It is believed that little or no reproduction occurs in nearshore (NS) aggregations, and that this might contribute to the slow recovery of the population 
   <xref ref-type="bibr" rid="pone.0012737-Delgado1">
     [2] 
   </xref>, 
   <xref ref-type="bibr" rid="pone.0012737-Glazer1">
     [3] 
   </xref>. A study of conch reproduction found that NS conchs failed to develop adequate gonad tissue for reproduction, but that translocation of NS conchs to the OS environment resulted in development of normal gonad tissue and reproductive activity within three months 
   <xref ref-type="bibr" rid="pone.0012737-Delgado1">
     [2] 
   </xref>. However, the causes of reproductive failure of NS conchs remain unknown.</p> 
  <p>Human impacts on coastal marine ecosystems are ever-increasing, and threats include inputs of nutrients, organic contaminants, and metals, as well as changes in temperature, decreasing ocean pH, and deoxygenation 
   <xref ref-type="bibr" rid="pone.0012737-Doney1">
     [5] 
   </xref>. While many of these factors can theoretically affect reproduction, one plausible cause for reproductive failure in a NS marine gastropod is heavy metal exposure. A number of gastropod studies have related heavy metal exposure, in particular exposure to Cu 
   <xref ref-type="bibr" rid="pone.0012737-Laskowski1">
     [6] 
   </xref>- 
   <xref ref-type="bibr" rid="pone.0012737-Snyman1">
     [8] 
   </xref> and Zn 
   <xref ref-type="bibr" rid="pone.0012737-Laskowski1">
     [6] 
   </xref>, 
   <xref ref-type="bibr" rid="pone.0012737-Coeurdassier1">
     [9] 
   </xref>, 
   <xref ref-type="bibr" rid="pone.0012737-Ducrot1">
     [10] 
   </xref>, to reduced fecundity – reproductive output usually measured in terms of egg laying. Despite the link between exposure to Cu and Zn and decreased reproductive output in gastropods, past studies consider mostly female-mediated effects at the individual level. In the Florida Keys, both male and female reproductive development is inhibited NS 
   <xref ref-type="bibr" rid="pone.0012737-Delgado1">
     [2] 
   </xref>, 
   <xref ref-type="bibr" rid="pone.0012737-Glazer1">
     [3] 
   </xref>. Given that heavy metals are known to inhibit gastropod egg laying, and that general and point sources for metal contamination exist close to shore in the Florida Keys 
   <xref ref-type="bibr" rid="pone.0012737-Carnahan1">
     [11] 
   </xref>– 
   <xref ref-type="bibr" rid="pone.0012737-Lewis1">
     [13] 
   </xref>, our general hypothesis is that heavy metals are likely to contribute to the reproductive failure observed NS. For the present study, our specific hypotheses were:</p> 
  <list list-type="order"> 
   <list-item> 
    <p>that testis transcriptional data would identify candidate gene expression pathways affected by NS environmental stressors, and</p> 
   </list-item> 
   <list-item> 
    <p>that tissue concentrations of heavy metals in NS conchs would exceed those of OS conchs.</p> 
   </list-item> 
  </list> 
  <p>For this study we developed and used a microarray to identify gene expression differences between the testes of NS and OS conchs. Gene expression data was anchored in histopathology to provide a more complete understanding of the dysfunction in testis development in NS conchs. Additionally, we used inductively coupled plasma-mass spectrometry to quantify nine metals, including Cu and Zn, in conch tissues and to determine whether their concentrations correlate with histological and gene expression evidence of NS reproductive dysfunction.</p> 
 </sec> 
 <sec id="s2" sec-type="methods"> 
  <h3>Methods</h3> 
  <sec id="s2a"> 
   <h3>Field collections</h3> 
   <p>For this study, all conchs were collected in the Florida Keys ( 
    <xref ref-type="fig" rid="pone-0012737-g001">
      Figure 1 
    </xref>). Tissue samples used for queen conch cDNA library construction were collected from Sombrero Reef on 9 June 2004 and from East Sister's Rock and Eastern Sambo on 15 March 2005. Tissue samples used for the comparison of reproductive (OS) versus non-reproductive (NS) queen conchs by histology, real-time reverse-transcription polymerase chain reaction (real-time RT-PCR), and metals analysis were collected from Pelican Shoal (OS) and Tingler Island (NS) on 15 February 2007 and from Eastern Sambo (OS) and East Sisters' Rock (NS) on 7 June–9 June 2007. Microarray analysis was conducted using 15 February 2007 samples from Pelican Shoal and Tingler Island. Sampling was conducted by free diving or SCUBA diving. Only adult queen conchs, identified by a fully flared lip 
    <xref ref-type="bibr" rid="pone.0012737-Davis1">
      [4] 
    </xref>, were collected.</p> 
   <fig id="pone-0012737-g001" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0012737.g001 
    </object-id> 
    <label>Figure 1</label> 
    <h3>Nearshore (NS) and offshore (OS) queen conch sampling sites in the Florida Keys.</h3> 
    <p>Microarray construction: 
     <bold>
       ESR 
     </bold>, East Sister's Rock (NS); 
     <bold>
       ES 
     </bold>, Eastern Sambo (OS); 
     <bold>
       SR 
     </bold>, Sombrero Reef (OS). Microarray and real-time reverse-transcription polymerase chain-reaction experiments: 
     <bold>
       PS 
     </bold>, Pelican Shoal (OS); 
     <bold>
       TI 
     </bold>, Tingler Island (NS). Validation of 18S rRNA for real-time RT-PCR: 
     <bold>
       PS 
     </bold>; 
     <bold>
       TI 
     </bold>; 
     <bold>
       ES 
     </bold>; 
     <bold>
       ESR 
     </bold>; 
     <bold>
       DS 
     </bold>, Delta Shoal (OS). Inductively coupled plasma-mass spectrometry and histology: 
     <bold>
       PS 
     </bold>; 
     <bold>
       TI 
     </bold>; 
     <bold>
       ESR 
     </bold>, 
     <bold>
       ES 
     </bold>.</p> 
    <graphic xlink:href="pone.0012737.g001"></graphic> 
   </fig> 
   <p>In all cases, adult male conchs were collected live and transported immediately to the Florida Fish and Wildlife Conservation Commission's Fish and Wildlife Research Institute (FWRI) laboratory in Marathon, FL. Conchs were then euthanized and tissues were harvested. For molecular assays and determination of tissue metal burdens, gonad, digestive gland, neural ganglia, blood, and foot muscle samples were frozen immediately in liquid nitrogen. Frozen tissue samples were maintained at ?80°C until further analysis. For histology, a piece of testis tissue approximately 1 cm<sup>3</sup> in size from the middle of each conch's testis was placed in an individually labeled cassette and fixed in 10% neutral buffered formalin for a minimum of 7 days and retained for further processing.</p> 
  </sec> 
  <sec id="s2b"> 
   <h3>Histopathology</h3> 
   <p>Histological examinations of the testes of experimental conchs were conducted as described in Delgado et al. 
    <xref ref-type="bibr" rid="pone.0012737-Delgado1">
      [2] 
    </xref>. Briefly, fixed tissue samples were rinsed in running tap water overnight, dehydrated in a graded series of ethanols, cleared, embedded in Paraplast (Leica Microsystems, Wetzlar, Germany), sectioned at 4 µm and stained with hematoxylin and eosin following standard histological techniques.</p> 
   <p>Testicular development was assessed histologically to determine reproductive capability of conchs. Conchs were classified into reproductive phases based on a maturity scale presented in Delgado et al. 
    <xref ref-type="bibr" rid="pone.0012737-Delgado1">
      [2] 
    </xref> ( 
    <xref ref-type="table" rid="pone-0012737-t001">
      Table 1 
    </xref>). In order to quantify the amount of testicular tissue present, a series of photomicrographs were taken to enable visualization of all gonadal tissue, resulting in 6–20 photographs for each specimen. Three photographs were randomly selected for each specimen, and the area of each testicular lobule in these photographs was determined using ImageJ software. A sperm scale based on the presence of developing sperm in the testis (spermatogonia ?=?1, spermatocytes ?=?2, spermatids ?=?3, spermatozoa ?=?4, vas deferens with spermatozoa ?=?5) was recorded for each lobule. A spermatogenic index (SI) was calculated following methodology similar to that reported in Kofoed et al. 
    <xref ref-type="bibr" rid="pone.0012737-1">
      [14] 
    </xref>; the area of each lobule (mm<sup>2</sup>) was multiplied by the sperm scale for that lobule. A total SI was determined for each specimen by summing the SI for each lobule in the three photographic views.</p> 
   <table-wrap id="pone-0012737-t001" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0012737.t001 
    </object-id> 
    <label>Table 1</label> 
    <h3>Reproductive phases used to describe testicular development in queen conch.</h3> 
    <alternatives> 
     <graphic id="pone-0012737-t001-1" xlink:href="pone.0012737.t001"></graphic> 
     <table frame="hsides" rules="groups"> 
      <colgroup span="1"> 
       <col align="left" span="1" /> 
       <col align="center" span="1" /> 
      </colgroup> 
      <thead> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Reproductive Phase</td> 
        <td align="left" colspan="1" rowspan="1">Description</td> 
       </tr> 
      </thead> 
      <tbody> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Early Developing</td> 
        <td align="left" colspan="1" rowspan="1">Only spermatogonia and spermatocytes present</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Developing</td> 
        <td align="left" colspan="1" rowspan="1">All stages of spermatogenesis present; no spermatozoa in vas deferens</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Spawning Capable</td> 
        <td align="left" colspan="1" rowspan="1">All stages of spermatogenesis present, spermatozoa present in vas deferens</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Regressing</td> 
        <td align="left" colspan="1" rowspan="1">Empty lobules, resorption of spermatozoa. Some spermatogenesis occurring</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Atretic</td> 
        <td align="left" colspan="1" rowspan="1">Lobules degenerating, resorption of spermatozoa, no active spermatogenesis</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Regenerating</td> 
        <td align="left" colspan="1" rowspan="1">Only spermatogonia present</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">No Tissue</td> 
        <td align="left" colspan="1" rowspan="1">No spermatogenesis occurring and no spermatogonia present; this is an abnormal condition in adult males.</td> 
       </tr> 
      </tbody> 
     </table> 
    </alternatives> 
    <table-wrap-foot> 
     <fn id="nt101"> 
      <label></label> 
      <p>Modified from Delgado et al. 
       <xref ref-type="bibr" rid="pone.0012737-Delgado1">
         [2] 
       </xref>.</p> 
     </fn> 
    </table-wrap-foot> 
   </table-wrap> 
  </sec> 
  <sec id="s2c"> 
   <h3>Construction, sequencing, and annotation of a normalized cDNA library</h3> 
   <p>cDNA library construction was carried out following the method of Garcia-Reyero et al. 
    <xref ref-type="bibr" rid="pone.0012737-GarciaReyero1">
      [15] 
    </xref>. Briefly, total RNA was isolated from 12 individual conch tissue samples (neural ganglia, gonad, and digestive gland from each of four conchs, one female and one male from NS and one female and one male from OS) using TRIzol (Invitrogen, Carlsbad, CA, USA) according to the manufacturer's protocol. RNA quality and integrity were assessed using the NanoDrop ND-1000 spectrophotometer (NanoDrop Technologies, Wilmington, DE, USA) and Agilent 2100 Bioanalyzer (Agilent Technologies, Santa Clara, CA, USA), respectively. 200 ng of each RNA sample was pooled and used as the template for the cDNA library construction and normalization using the SMART (Clontech/Takara Bio USA, Madison, WI, USA) and Trimmer (Evrogen Joint Stock Company, Moscow, Russia) kits, respectively. The normalized queen conch cDNA library was sequenced using a 454 GS-FLX pyrosequencer (Roche/454 Life Sciences, Branford, CT, USA) 
    <xref ref-type="bibr" rid="pone.0012737-Droege1">
      [16] 
    </xref>, at the University of Florida Interdisciplinary Center for Biotechnology Research (ICBR). Sequencing was conducted as described in Garcia-Reyero et al. 
    <xref ref-type="bibr" rid="pone.0012737-GarciaReyero1">
      [15] 
    </xref>. Sequence assembly and annotation were performed as described in Farmerie et al. 
    <xref ref-type="bibr" rid="pone.0012737-Farmerie1">
      [17] 
    </xref> using Newbler v1.1.03.24 and Paracel TranscriptAssembler. Minimum confidence for a match was set at e&lt;1×10<sup>?4</sup>. Each sequence and its top 100 hits from BLAST search were retained in ICBR's BlastQuest database 
    <xref ref-type="bibr" rid="pone.0012737-Farmerie1">
      [17] 
    </xref> and annotated with Gene Ontology (GO) information and with human and zebrafish ( 
    <italic>
      Danio rerio 
    </italic>) homologues by referring to the NCBI Gene database. A queen conch oligonucleotide microarray was designed in an 8×15 k format consisting of 15,208 60-mer user probes and 536 internal control probes using the eArray service from Agilent. The microarray was submitted as a platform dataset to NCBI's Gene Expression Omnibus (GEO) database (GPL8934).</p> 
  </sec> 
  <sec id="s2d"> 
   <h3>Extraction of RNA from experimental samples</h3> 
   <p>For microarray and cloning procedures, four biological replicate conch RNA samples were isolated from testis of male conchs collected in February, 2007, at Pelican Shoal (OS) and Tingler Island (NS) ( 
    <xref ref-type="fig" rid="pone-0012737-g001">
      Figure 1 
    </xref>). One array from each group was determined to be of insufficient quality during the quality control process, and so the final sample size for microarrays was three individuals from each location, OS and NS. The OS individuals are considered to be the reference group, as they exhibit successful sexual reproduction, while NS conchs do not. Total RNA was extracted using the RNA STAT-60 reagent (Tel-Test, Friendswood, TX, USA), reconstituted in RNAsecure (Ambion), and DNase-treated with 
    <italic>
      Turbo 
    </italic> DNA-free (Ambion). In all cases, manufacturers' protocols were followed. RNA samples used for real-time RT-PCR assays were extracted using a more rigorous procedure, in which STAT-60 extraction of RNA was followed by gradient centrifugation through a 1.4 mL layer of 5.7 M CsCl, 0.5 M EDTA in the Beckman Optima-TLX ultracentrifuge and TLA120.2 rotor. RNA cleanup was performed using the Qiagen RNEasy Mini Kit. RNA quantity was assessed on the NanoDrop ND-1000 and quality was assessed using the Agilent 2100 Bioanalyzer. We have observed that un-degraded conch total RNA samples show no 28S rRNA band; therefore, no RIN can be calculated to quantify RNA integrity for these samples. This is likely due to the “hidden break” in invertebrate 28S rRNA that has been described by Ishikawa 
    <xref ref-type="bibr" rid="pone.0012737-Ishikawa1">
      [18] 
    </xref> and that is present in gastropods including 
    <italic>
      Haliotis rufescens 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Groppe1">
      [19] 
    </xref>. Despite the impossibility of calculating RIN, only high-quality samples for which Bioanalyzer profiles appeared to be un-degraded were used for further analysis.</p> 
  </sec> 
  <sec id="s2e"> 
   <h3>Microarray processing</h3> 
   <p>RNA samples were labeled with Cy3-CTP and hybridized to the microarray following the Agilent Protocol h3d One Color Microarray-Based Gene Expression Analysis v5.5 (publication no. G4140-90040). Specific activity of Cy3 label was at least 10.04 pmol/?L in each sample, and averaged 13.14 pmol/?L. Microarray scanning and feature extraction was performed at ICBR using an Agilent G2505B Microarray Scanner and Agilent Feature Extraction Software v9.5. All microarray data here reported are MIAME compliant; raw and normalized microarray data have been submitted to the GEO database (GSE17379), according to MIAME standards 
    <xref ref-type="bibr" rid="pone.0012737-Brazma1">
      [20] 
    </xref>.</p> 
  </sec> 
  <sec id="s2f"> 
   <h3>Cloning of 
    <italic>
      S. gigas 
    </italic> 18S ribosomal RNA</h3> 
   <p>3 ?g of each conch RNA sample was reverse transcribed to produce cDNA using Invitrogen SuperScript II Reverse Transcriptase and random primers, per the manufacturer's protocol. 18S rRNA was cloned using primers designed in the program Primer3 
    <xref ref-type="bibr" rid="pone.0012737-Rozen1">
      [21] 
    </xref> based on alignment of 18S rRNA from the gastropod 
    <italic>
      Bursa rana 
    </italic> (X94269.1) and the bivalve 
    <italic>
      Nucula sulcata 
    </italic> (AF207642.1) ( 
    <xref ref-type="table" rid="pone-0012737-t002">
      Table 2 
    </xref>). 18S rRNA primers were used in a PCR reaction with Invitrogen Taq polymerase, according to the manufacturer's protocol. PCR products were cloned in the pGEM-T Easy vector (Sigma-Aldrich, St. Louis, MO, USA) and Invitrogen One-shot Top10 chemically competent 
    <italic>
      E. coli 
    </italic> cells, per the manufacturer's protocols. The sequence of the cloned 18S rRNA fragment was confirmed by Sanger sequencing at ICBR (GU198749).</p> 
   <table-wrap id="pone-0012737-t002" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0012737.t002 
    </object-id> 
    <label>Table 2</label> 
    <h3>Primers for 18S rRNA cloning and for real-time RT-PCR.</h3> 
    <alternatives> 
     <graphic id="pone-0012737-t002-2" xlink:href="pone.0012737.t002"></graphic> 
     <table frame="hsides" rules="groups"> 
      <colgroup span="1"> 
       <col align="left" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
      </colgroup> 
      <thead> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Sequence</td> 
        <td align="left" colspan="1" rowspan="1">Purpose</td> 
        <td align="left" colspan="1" rowspan="1">Direction</td> 
        <td align="left" colspan="1" rowspan="1">Primer Sequence (5?–3?)</td> 
        <td align="left" colspan="1" rowspan="1">Amplicon Size (bp)</td> 
        <td align="left" colspan="1" rowspan="1">Accession Number</td> 
       </tr> 
      </thead> 
      <tbody> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">18S rRNA</td> 
        <td align="left" colspan="1" rowspan="1">Cloning</td> 
        <td align="left" colspan="1" rowspan="1">Forward</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <named-content content-type="gene">
           GTT TCC CAT CCT ACG CTT CC 
         </named-content> </td> 
        <td align="left" colspan="1" rowspan="1">636</td> 
        <td align="left" colspan="1" rowspan="1">GU198749</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1">Reverse</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <named-content content-type="gene">
           AGA CAA ATC GCT CCA CCA AC 
         </named-content> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">18S rRNA</td> 
        <td align="left" colspan="1" rowspan="1">Real-time</td> 
        <td align="left" colspan="1" rowspan="1">Forward</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <named-content content-type="gene">
           TCG GTC TTA TTT TGC TGG TTT 
         </named-content> </td> 
        <td align="left" colspan="1" rowspan="1">226</td> 
        <td align="left" colspan="1" rowspan="1">GU198749</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1">Reverse</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <named-content content-type="gene">
           ATC GCT AGT TGG CAT CGT TT 
         </named-content> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Ctr1c</td> 
        <td align="left" colspan="1" rowspan="1">Real-time</td> 
        <td align="left" colspan="1" rowspan="1">Forward</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <named-content content-type="gene">
           ACA AGG GCG GAA GAA GAA GT 
         </named-content> </td> 
        <td align="left" colspan="1" rowspan="1">158</td> 
        <td align="left" colspan="1" rowspan="1">UF_Sgi_AF_100593</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1">Reverse</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <named-content content-type="gene">
           GGC TTT CAG TAC CCA AAC GA 
         </named-content> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">TepII</td> 
        <td align="left" colspan="1" rowspan="1">Real-time</td> 
        <td align="left" colspan="1" rowspan="1">Forward</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <named-content content-type="gene">
           GTC ACG GCT GAC TCC TTC TC 
         </named-content> </td> 
        <td align="left" colspan="1" rowspan="1">151</td> 
        <td align="left" colspan="1" rowspan="1">UF_Sgi_AF_105314</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1">Reverse</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <named-content content-type="gene">
           TAA AGA ACA CGC CGA TCT CC 
         </named-content> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">GST</td> 
        <td align="left" colspan="1" rowspan="1">Real-time</td> 
        <td align="left" colspan="1" rowspan="1">Forward</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <named-content content-type="gene">
           TAT GGC AAG ACC AAC ATG GA 
         </named-content> </td> 
        <td align="left" colspan="1" rowspan="1">174</td> 
        <td align="left" colspan="1" rowspan="1">UF_Sgi_AF_101461</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1">Reverse</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <named-content content-type="gene">
           ATT CGC GTA AAA GCC AAA GA 
         </named-content> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Stard7</td> 
        <td align="left" colspan="1" rowspan="1">Real-time</td> 
        <td align="left" colspan="1" rowspan="1">Forward</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <named-content content-type="gene">
           GCG CTG TTG CTG AAC ATA AA 
         </named-content> </td> 
        <td align="left" colspan="1" rowspan="1">183</td> 
        <td align="left" colspan="1" rowspan="1">UF_Sgi_AF_103703</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1">Reverse</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <named-content content-type="gene">
           CTT CTT GCA CAC CAT CTC GTT 
         </named-content> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
      </tbody> 
     </table> 
    </alternatives> 
    <table-wrap-foot> 
     <fn id="nt102"> 
      <label></label> 
      <p>“Accession Number” refers to the accession for the sequence in NCBI (18S rRNA) or the UF ProbeName for the corresponding microarray probe. Transcripts: Ctr1c, copper transporter 1c; TepII, thiolester containing protein II; GST, Similar to Glutathione S-transferase; Stard7, StAR-related lipid transfer (START) domain containing 7.</p> 
     </fn> 
    </table-wrap-foot> 
   </table-wrap> 
  </sec> 
  <sec id="s2g"> 
   <h3>Real-Time RT-PCR</h3> 
   <p>Copper transporter 1c (Ctr1c), thiolester-containing protein II (TepII), Similar to Glutathione S-transferase (GST), and Start domain-containing protein 7 (Stard7) were evaluated by real-time RT-PCR. Primers for transcripts of interest ( 
    <xref ref-type="table" rid="pone-0012737-t002">
      Table 2 
    </xref>) were developed from 454-derived cDNA library sequences using Primer3. All primer sets were verified using the same cloning and sequencing methods as in the section 
    <italic>
      Cloning of 
    </italic> S. gigas 
    <italic>
      18S ribosomal RNA 
    </italic>, above. The 638 bp clone of 18S rRNA was used to design a second set of primers that was more optimal for real-time RT-PCR. For all other clones, cloning primers were also used for real-time RT-PCR. Plasmids containing each cloned sequence were used to create a standard curve consisting of eight points in a serial dilution from 1×10<sup>2</sup> through 1×10<sup>9</sup> copies/reaction. Real-time RT-PCR was performed as a two step process. In step one, 2 ?g DNase-treated RNA was reverse transcribed using Invitrogen's SuperScript II reverse transcriptase and random primers. In step two, real-time PCR reactions were run using SYBR Green Supermix (Bio-Rad, Hercules, CA, USA), per the manufacturer's protocol, on the Bio-Rad iCycler real-time PCR thermal cycler, using a two-step protocol with an initial denaturation at 95°C, followed by 40 cycles of denaturation at 95°C and annealing and extension at 58°C, during which real-time quantification was enabled. Following amplification, a dissociation curve was run beginning at 55°C and increasing to 95°C at 0.5°C intervals every 10s. Standards and experimental samples were run in duplicate, along with two negative controls for each gene: a “no reverse transcriptase (-RT)” control, in which DNAse-treated RNA samples were pooled and water was used in place of reverse transcriptase during the reverse transcription reaction, and a “no template control (NTC),” in which water was used in place of template cDNA during the real-time PCR reaction.</p> 
   <p>The use of 18S rRNA as a reference gene for RT-PCR was validated by measuring 18S rRNA by the method described above for 23 conch testis samples, collected in February, 2007, June, 2007, and March, 2009 ( 
    <xref ref-type="fig" rid="pone-0012737-g001">
      Figure 1 
    </xref>). Initial quantity for each sample was calculated as 18S rRNA copy number/ng total RNA. Data were analyzed in JMP v8 using a two-way ANOVA on the factors “collection (date)” and “location (OS vs. NS).” This analysis showed nearly identical mean 18S levels between OS and NS, with no statistically significant difference according to ANOVA ( 
    <xref ref-type="supplementary-material" rid="pone.0012737.s001">
      Table S1 
    </xref>). While no reference gene is perfect 
    <xref ref-type="bibr" rid="pone.0012737-Bustin1">
      [22] 
    </xref>, 
    <xref ref-type="bibr" rid="pone.0012737-Bustin2">
      [23] 
    </xref>, 18S rRNA appears to be the best internal reference for this experiment. 18S rRNA expression does vary across some sites and collection times, but is remarkably consistent for the samples here presented, collected in February, 2007. Moreover, at least one commonly used reference gene, ?-actin, was differentially regulated between NS and OS in the microarray study ( 
    <xref ref-type="supplementary-material" rid="pone.0012737.s002">
      Table S2 
    </xref>, ProbeName UF_Sgi_AF_101275), making it a poor candidate for our internal reference in real-time RT-PCR.</p> 
  </sec> 
  <sec id="s2h"> 
   <h3>Metal analysis by inductively coupled plasma-mass spectrometry (ICP-MS)</h3> 
   <p>ICP-MS was used to determine levels of <sup>58</sup>Ni, <sup>65</sup>Cu, <sup>66</sup>Zn, <sup>88</sup>Sr, <sup>107</sup>Ag,<sup> 111</sup>Cd, <sup>118</sup>Sn, <sup>202</sup>Hg, and <sup>238</sup>U in blood, digestive gland, foot, neural ganglia, and testis for male conchs collected in February and June 2007 field collections (n?=?2-8, varying by sample type). Weighed tissue samples (approximately 50–200 mg) were acid digested to completion in 0.5 mL 67–70% optima grade HNO<sub>3</sub> at 140° C for 2 hours. This was followed by addition of 0.5 mL 30% ultrapure H<sub>2</sub>O<sub>2</sub> and further digested at 110°C until almost dry. The sample was quantitatively diluted to 5 mL using ultrapure water for a final concentration of 2% HNO<sub>3</sub> and filtered through a 0.22 ?m nylon syringe filter. The reconstituted samples were analyzed for total metal content using an XSeries 2 ICP-MS (Thermo Electron Corporation, Winsford, Cheshire, UK) with <sup>115</sup>In as an internal standard. Samples were quantified against a seven point standard curve with standard concentrations 1, 5, 10, 50, 100, 500, and 1000 ppb each analyte. The lower limit of detectability for this assay was set at 0.5 ppb analyte in the digested sample.</p> 
  </sec> 
  <sec id="s2i"> 
   <h3>Statistical Analysis</h3> 
   <p>Histological data were analyzed using JMP v8 (SAS, Cary, NC, USA). Differences in SI were analyzed using ANOVA with month nested within location. A student's t-test based on least square means of the nested ANOVA was calculated in JMP for all month and location combinations. SI was tested for homogeneity of variance (Levene's test) and normality of distribution (Kolmogorov-Smirnov test), and values were log<sub>10</sub> transformed if assumptions were violated. Raw microarray data were imported into JMP Genomics v3.1 and analyzed as follows: Non-uniform spots were flagged and removed from the dataset. Next, 590 rows not containing at least two data points for each treatment group (OS and NS) were deleted, leaving 15,154 rows in the analysis. Array data were then median-centered prior to performing one-way ANOVA on the factor location (OS/NS) to identify differentially regulated transcripts (p&lt;0.01 or p&lt;0.05 for further analyses, FDR?=?5%). Hierarchical clustering analysis of significant transcripts (p&lt;0.01) was performed using the program Cluster 
    <xref ref-type="bibr" rid="pone.0012737-Eisen1">
      [24] 
    </xref> and visualized in the program Java TreeView 
    <xref ref-type="bibr" rid="pone.0012737-Saldanha1">
      [25] 
    </xref>. Data were median-centered by gene and clustering was based on centered correlation and complete linkage. Real-time RT-PCR data were analyzed using the Kruskall-Wallis non-parametric test calculator available at 
    <ext-link ext-link-type="uri" xlink:href="http://elegans.wsmed.edu/~leon/stats/utest.html">
      http://elegans.wsmed.edu/~leon/stats/utest.html 
    </ext-link>. ICP-MS data were imported into JMP v7 and analyzed for difference of means using two-way ANOVA, with the two factors being tissue and location; this analysis was followed by the 
    <italic>
      post hoc 
    </italic> Tukey-Kramer HSD test for multiple comparisons (p&lt;0.05). For non-parametric correlation analysis, Spearman's ? was calculated in JMP.</p> 
  </sec> 
  <sec id="s2j"> 
   <h3>Gene Ontology and Pathway Analysis</h3> 
   <p>For microarray data, functional enrichment analysis of Gene Ontology terms was performed by Fisher's exact test using the FatiGO tool within the Babelomics suite 
    <xref ref-type="bibr" rid="pone.0012737-AlShahrour1">
      [26] 
    </xref>. All terms with a nominal p-value of p&lt;0.05 (no 
    <italic>
      post hoc 
    </italic> correction) were considered to be enriched. Finally, Pathway Studio 7 (Ariadne Genomics, Rockville, MD, USA), operating on the ResNet 7.0 mammalian database updated with zebrafish annotation, was used to identify all shortest paths between genes falling under significantly enriched terms and cellular processes, in order to illustrate important connections within these biological processes, based on human and zebrafish ( 
    <italic>
      Danio rerio 
    </italic>) homologs.</p> 
  </sec> 
 </sec> 
 <sec id="s3"> 
  <h3>Results</h3> 
  <sec id="s3a"> 
   <h3>Histopathology</h3> 
   <p>Testis tissue from eight conchs (four OS, four NS) in February 2007 and seven conchs in June 2007 (two OS, five NS) was analyzed by histopathology ( 
    <xref ref-type="table" rid="pone-0012737-t003">
      Table 3 
    </xref>). All OS conchs from both months had normal, healthy testicular tissue present in &gt;75% of each histological section ( 
    <xref ref-type="fig" rid="pone-0012737-g002">
      Figure 2A 
    </xref>). In contrast, the amount of testicular tissue present in NS conchs was lower than OS conchs in both February and June ( 
    <xref ref-type="fig" rid="pone-0012737-g002">
      Figure 2B, C 
    </xref>). Differences in gonadal development based on reproductive phase were not marked in February between NS and OS conchs. All conchs from both locations had testicular tissue undergoing active spermatogenesis in February, although a higher percentage of OS conchs (75%) were Spawning Capable compared to NS conchs (50%; 
    <xref ref-type="table" rid="pone-0012737-t003">
      Table 3 
    </xref>). However, there was a significant difference in the SI between OS and NS conchs in February (t<sub>11</sub>?=?2.606, p?=?0.024), with OS conchs having a SI value nearly 7 times greater than the NS value ( 
    <xref ref-type="table" rid="pone-0012737-t003">
      Table 3 
    </xref>).</p> 
   <fig id="pone-0012737-g002" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0012737.g002 
    </object-id> 
    <label>Figure 2</label> 
    <h3>Histological sections of testis tissue from queen conchs captured in the Florida Keys in 2007.</h3> 
    <p> 
     <bold>
       A 
     </bold>. Testicular tissue from an offshore (OS), Spawning Capable male in February. All OS conchs captured in June had a similar appearance. 
     <bold>
       B 
     </bold>. Testicular tissue from a nearshore (NS), Developing male in February. 
     <bold>
       C 
     </bold>. Testicular tissue from a NS male in June showing little spermatogenic tissue or spermatogenesis. ST—spermatogenic tissue; VD—vas deferens.</p> 
    <graphic xlink:href="pone.0012737.g002"></graphic> 
   </fig> 
   <table-wrap id="pone-0012737-t003" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0012737.t003 
    </object-id> 
    <label>Table 3</label> 
    <h3>Summary of testicular development observed in queen conchs collected from offshore (OS) and nearshore (NS) sites in the Florida Keys in February and June 2007.</h3> 
    <alternatives> 
     <graphic id="pone-0012737-t003-3" xlink:href="pone.0012737.t003"></graphic> 
     <table frame="hsides" rules="groups"> 
      <colgroup span="1"> 
       <col align="left" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
      </colgroup> 
      <thead> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Gonadal Development</td> 
        <td align="left" colspan="1" rowspan="1">OS February (n?=?4)</td> 
        <td align="left" colspan="1" rowspan="1">NS February (n?=?4)</td> 
        <td align="left" colspan="1" rowspan="1">OS June (n?=?2)</td> 
        <td align="left" colspan="1" rowspan="1">NS June (n?=?5)</td> 
       </tr> 
      </thead> 
      <tbody> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Early Developing</td> 
        <td align="left" colspan="1" rowspan="1">0</td> 
        <td align="left" colspan="1" rowspan="1">0</td> 
        <td align="left" colspan="1" rowspan="1">0</td> 
        <td align="left" colspan="1" rowspan="1">20</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Developing</td> 
        <td align="left" colspan="1" rowspan="1">25</td> 
        <td align="left" colspan="1" rowspan="1">50</td> 
        <td align="left" colspan="1" rowspan="1">0</td> 
        <td align="left" colspan="1" rowspan="1">20</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Spawning Capable</td> 
        <td align="left" colspan="1" rowspan="1">75</td> 
        <td align="left" colspan="1" rowspan="1">50</td> 
        <td align="left" colspan="1" rowspan="1">100</td> 
        <td align="left" colspan="1" rowspan="1">20</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Regenerating</td> 
        <td align="left" colspan="1" rowspan="1">0</td> 
        <td align="left" colspan="1" rowspan="1">0</td> 
        <td align="left" colspan="1" rowspan="1">0</td> 
        <td align="left" colspan="1" rowspan="1">20</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">No gonadal tissue</td> 
        <td align="left" colspan="1" rowspan="1">0</td> 
        <td align="left" colspan="1" rowspan="1">0</td> 
        <td align="left" colspan="1" rowspan="1">0</td> 
        <td align="left" colspan="1" rowspan="1">20</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">SI</td> 
        <td align="left" colspan="1" rowspan="1">21.29±2.57<sup>a</sup></td> 
        <td align="left" colspan="1" rowspan="1">3.74±0.72<sup>b</sup></td> 
        <td align="left" colspan="1" rowspan="1">26.03±4.51<sup>a</sup></td> 
        <td align="left" colspan="1" rowspan="1">0.60±0.47<sup>c</sup></td> 
       </tr> 
      </tbody> 
     </table> 
    </alternatives> 
    <table-wrap-foot> 
     <fn id="nt103"> 
      <label></label> 
      <p>Data for reproductive phases presented as percentage. Data for spermatogenic index (SI) presented as mean ± SE. Superscripts indicate differences in SI (nested ANOVA, p&lt;0.05).</p> 
     </fn> 
    </table-wrap-foot> 
   </table-wrap> 
   <p>In contrast, by June all OS conchs were Spawning Capable while only 60% of NS conchs exhibited active spermatogenesis, with only one NS individual Spawning Capable ( 
    <xref ref-type="table" rid="pone-0012737-t003">
      Table 3 
    </xref>). The SI was nearly 45 times higher in OS conchs compared to NS conchs in June (t<sub>11</sub>?=?6.05, p&lt;0.001; 
    <xref ref-type="table" rid="pone-0012737-t003">
      Table 3 
    </xref>). Overall, in both February and June the SI was significantly higher for OS conchs when compared to NS conchs. There was no difference in SI between months for OS conch (t<sub>11</sub>?=?0.245, p?=?0.81), suggesting OS conchs were reproductively active from February through June. Interestingly, for NS conchs, the SI was significantly higher in February than in June (t<sub>11</sub>?=??4.49, p?=?0.0009; 
    <xref ref-type="table" rid="pone-0012737-t003">
      Table 3 
    </xref>) suggesting that the NS conchs were unable to maintain spermatogenic tissue during the reproductive season.</p> 
  </sec> 
  <sec id="s3b"> 
   <h3>Normalized cDNA library sequencing and assembly</h3> 
   <p>Sequencing produced a total of 64,794,458 bases across 286,933 reads (average read length ?=?225.8 bases) ( 
    <xref ref-type="table" rid="pone-0012737-t004">
      Table 4 
    </xref>). Sequences were submitted to the NCBI Sequence Read Archive (SRA) (experiment SRX017250). These sequences were assembled into 28,010 contigs, of which 7,740 matched to NCBI NR or NT database with e&lt;1×10<sup>?4</sup>. Of 7,740 annotated sequences, at least one GO term was assigned to 3,971 (51.3 percent), human homologues to 2,688 (34.7 percent), and zebrafish homologues to 2,681 (34.6 percent). The microarray was designed with probes corresponding to all 7,740 annotated sequences and to 7,468 additional un-annotated sequences for a total of 15,208 user-defined elements.</p> 
   <table-wrap id="pone-0012737-t004" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0012737.t004 
    </object-id> 
    <label>Table 4</label> 
    <h3>Information on conch transcriptome assembly.</h3> 
    <alternatives> 
     <graphic id="pone-0012737-t004-4" xlink:href="pone.0012737.t004"></graphic> 
     <table frame="hsides" rules="groups"> 
      <colgroup span="1"> 
       <col align="left" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
      </colgroup> 
      <thead> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Run</td> 
        <td align="left" colspan="1" rowspan="1">Bases</td> 
        <td align="left" colspan="1" rowspan="1">Reads</td> 
        <td align="left" colspan="1" rowspan="1">Average Length</td> 
        <td align="left" colspan="1" rowspan="1">Alias</td> 
        <td align="left" colspan="1" rowspan="1">SRA Accession</td> 
       </tr> 
      </thead> 
      <tbody> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Titration</td> 
        <td align="left" colspan="1" rowspan="1">1,286,141</td> 
        <td align="left" colspan="1" rowspan="1">5,354</td> 
        <td align="left" colspan="1" rowspan="1">240.22</td> 
        <td align="left" colspan="1" rowspan="1">ERXZ1SM</td> 
        <td align="left" colspan="1" rowspan="1">SRR037030.3</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Production-1</td> 
        <td align="left" colspan="1" rowspan="1">37,762,116</td> 
        <td align="left" colspan="1" rowspan="1">161,610</td> 
        <td align="left" colspan="1" rowspan="1">233.66</td> 
        <td align="left" colspan="1" rowspan="1">ES8A9FR</td> 
        <td align="left" colspan="1" rowspan="1">SRR037031.2</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Production-2</td> 
        <td align="left" colspan="1" rowspan="1">15,882,529</td> 
        <td align="left" colspan="1" rowspan="1">67,779</td> 
        <td align="left" colspan="1" rowspan="1">234.33</td> 
        <td align="left" colspan="1" rowspan="1">ETMYUMG</td> 
        <td align="left" colspan="1" rowspan="1">SRR037032.2</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Production-3</td> 
        <td align="left" colspan="1" rowspan="1">9,863,672</td> 
        <td align="left" colspan="1" rowspan="1">52,190</td> 
        <td align="left" colspan="1" rowspan="1">189.00</td> 
        <td align="left" colspan="1" rowspan="1">EU6CYYIF</td> 
        <td align="left" colspan="1" rowspan="1">SRR037033.2</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Total</td> 
        <td align="left" colspan="1" rowspan="1">64,794,458</td> 
        <td align="left" colspan="1" rowspan="1">286,933</td> 
        <td align="left" colspan="1" rowspan="1">225.82</td> 
        <td align="left" colspan="1" rowspan="1">S. gigas 454</td> 
        <td align="left" colspan="1" rowspan="1">SRX017250.1</td> 
       </tr> 
      </tbody> 
     </table> 
    </alternatives> 
    <table-wrap-foot> 
     <fn id="nt104"> 
      <label></label> 
      <p>“Alias” refers to the run alias listed in the SRA database.</p> 
     </fn> 
    </table-wrap-foot> 
   </table-wrap> 
  </sec> 
  <sec id="s3c"> 
   <h3>Microarray analysis of testicular transcription</h3> 
   <p>255 differentially-regulated probes (58 up and 197 down in NS with respect to OS conchs) were identified by ANOVA (n?=?3, p&lt;0.01, FDR?=?5%) ( 
    <xref ref-type="fig" rid="pone-0012737-g003">
      Figure 3 
    </xref>, 
    <xref ref-type="supplementary-material" rid="pone.0012737.s002">
      Table S2 
    </xref>). At a less stringent p-value, 1147 differentially-regulated probes (341 up and 806 down) were identified (p&lt;0.05, FDR?=?5%) ( 
    <xref ref-type="supplementary-material" rid="pone.0012737.s002">
      Table S2 
    </xref>). Based on differentially-regulated probes, all OS and NS individuals clustered separately from one another, indicating that the identified set of transcripts show a clear difference between these two presumably outbred groups of wild conchs ( 
    <xref ref-type="fig" rid="pone-0012737-g003">
      Figure 3 
    </xref>). Differentially regulated genes were predominantly down-regulated in this experiment; at a cutoff of p&lt;0.01, the proportion of differentially regulated transcripts that are down-regulated was 77.4 percent. The two most up-regulated probes with annotation (p&lt;0.01) were Similar to Glutathione S-transferase (GST, 15.24-fold up-regulated NS) and Collagen 1, Alpha 1 (COL1A1, 10.26-fold up-regulated NS). The two most down-regulated probes with annotation (p&lt;0.01) were RIKEN CDNA F730014I05 Gene, a mouse genome sequence (13.83-fold down-regulated NS), and Dolichyl-phosphate Mannosyltransferase Polypeptide 2 Regulatory Subunit (Dpm2, 4.92-fold down-regulted NS).</p> 
   <fig id="pone-0012737-g003" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0012737.g003 
    </object-id> 
    <label>Figure 3</label> 
    <h3>Hierarchical clustering of significantly differentially regulated genes in conch testis.</h3> 
    <p>Red color represents expression of a gene at a level greater than the row (gene) average, and blue color represents expression lower than the row average. The map shows a clear distinction between nearshore (NS) and offshore (OS) testis samples based on the 256 differentially-regulated transcripts. Approximately one-fourth of the regulated transcripts are up-regulated in NS relative to OS; the majority are down-regulated.</p> 
    <graphic xlink:href="pone.0012737.g003"></graphic> 
   </fig> 
   <p>Functional enrichment analysis based on GO terms for biological process identified 11 significantly enriched terms in the differentially-regulated gene list ( 
    <xref ref-type="table" rid="pone-0012737-t005">
      Table 5 
    </xref>). The most significantly enriched term was “proton transport,” under which all but one of seven differentially-regulated genes was down-regulated. Another notable term was “small GTPase-mediated signal transduction.” “Spermatogenesis” was the twelfth term on the list (p?=?0.052). Pathway analysis (Pathway Studio) further illustrated the results of the enrichment analysis ( 
    <xref ref-type="fig" rid="pone-0012737-g004">
      Figure 4 
    </xref>): most affected transcripts were down-regulated (blue color), many of these transcripts are found in the mitochondria, and there were a large number of associations with the cell processes “respiratory chain,” “cell proliferation,” and “spermatogenesis,” among others.</p> 
   <fig id="pone-0012737-g004" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0012737.g004 
    </object-id> 
    <label>Figure 4</label> 
    <h3>Pathway analysis of differentially regulated genes from conch testis microarray study.</h3> 
    <p>Pathway Studio (Ariadne Genomics) was used to find all shortest paths between human homologues of genes falling under significantly enriched GO Biological Processes in the testis. Red color represents up-regulation nearshore (NS); blue color represents down-regulation NS. Genes: ARL1, zgc:92883 (ADP-ribosylation factor-like 1); ATP5B, ATP synthase, H+ transporting, mitochondrial F1 complex, beta polypeptide; ATP5C1, ATP synthase, H+ transporting, mitochondrial F1 complex, gamma polypeptide 1; ATP5G2, ATP synthase, H+ transporting, mitochondrial F0 complex, subunit C2 (subunit 9); ATP5H, ATP synthase, H+ transporting, mitochondrial F0 complex, subunit d; CRP, si:ch211-234p6.13 ( 
     <italic>
       Danio rerio 
     </italic> hypothetical protein); DEGS1, im:6909319 (degenerative spermatocyte homolog, lipid desaturase); DGKZ, hypothetical LOC571856 (similar to diacylglycerol kinase, iota); DNAI2, dynein, axonemal, intermediate chain 2; DNAJB13, DnaJ (Hsp40) related, subfamily B, member 13; GSTM4, glutathione S-transferase mu 4; MOV10, si:dkeyp-38g6.3 (Moloney leukemia virus 10); NAPA, N-ethylmaleimide sensitive fusion protein attachment protein alpha; PGDS, prostaglandin D2 synthase, hematopoietic; PICK1, hypothetical protein LOC791503; PPME1, zgc:56239 (protein phosphatase methylesterase 1); PSME4, hypothetical LOC561538 (proteasome (prosome, macropain) activator subunit 4); RAB1A, RAB1A member RAS oncogene family; RABL4, RAB, member of RAS oncogene family-like 4; RAP1A, RAP1A, member of RAS oncogene family; SARS2, seryl-tRNA synthetase 2, mitochondrial; SLMO2, slowmo homolog 2 (Drosophila) (similer to kiser); VAPA (VAMP (vesicle-associated membrane protein)-associated protein A, 33 kDa. Organelles, clockwise from top center: mitochondrion, endoplasmic reticulum, Golgi complex, nucleus.</p> 
    <graphic xlink:href="pone.0012737.g004"></graphic> 
   </fig> 
   <table-wrap id="pone-0012737-t005" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0012737.t005 
    </object-id> 
    <label>Table 5</label> 
    <h3>Functional enrichment analysis based on Gene Ontology (GO) biological process terms.</h3> 
    <alternatives> 
     <graphic id="pone-0012737-t005-5" xlink:href="pone.0012737.t005"></graphic> 
     <table frame="hsides" rules="groups"> 
      <colgroup span="1"> 
       <col align="left" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
      </colgroup> 
      <thead> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Biological Process</td> 
        <td align="left" colspan="1" rowspan="1">GO Term ID</td> 
        <td align="left" colspan="1" rowspan="1">% of DR</td> 
        <td align="left" colspan="1" rowspan="1">% of other</td> 
        <td align="left" colspan="1" rowspan="1">p-value</td> 
       </tr> 
      </thead> 
      <tbody> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">proton transport</td> 
        <td align="left" colspan="1" rowspan="1">GO:0015992</td> 
        <td align="left" colspan="1" rowspan="1">2.90</td> 
        <td align="left" colspan="1" rowspan="1">0.75</td> 
        <td align="left" colspan="1" rowspan="1">0.005</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">membrane fusion</td> 
        <td align="left" colspan="1" rowspan="1">GO:0006944</td> 
        <td align="left" colspan="1" rowspan="1">0.83</td> 
        <td align="left" colspan="1" rowspan="1">0.00</td> 
        <td align="left" colspan="1" rowspan="1">0.005</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">virus induced gene silencing</td> 
        <td align="left" colspan="1" rowspan="1">GO:0009616</td> 
        <td align="left" colspan="1" rowspan="1">0.83</td> 
        <td align="left" colspan="1" rowspan="1">0.00</td> 
        <td align="left" colspan="1" rowspan="1">0.005</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">receptor clustering</td> 
        <td align="left" colspan="1" rowspan="1">GO:0043113</td> 
        <td align="left" colspan="1" rowspan="1">0.83</td> 
        <td align="left" colspan="1" rowspan="1">0.00</td> 
        <td align="left" colspan="1" rowspan="1">0.005</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">aromatic compound metabolic process</td> 
        <td align="left" colspan="1" rowspan="1">GO:0006725</td> 
        <td align="left" colspan="1" rowspan="1">1.24</td> 
        <td align="left" colspan="1" rowspan="1">0.13</td> 
        <td align="left" colspan="1" rowspan="1">0.011</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">seryl-tRNA aminoacylation</td> 
        <td align="left" colspan="1" rowspan="1">GO:0006434</td> 
        <td align="left" colspan="1" rowspan="1">0.83</td> 
        <td align="left" colspan="1" rowspan="1">0.03</td> 
        <td align="left" colspan="1" rowspan="1">0.015</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">cilium biogenesis</td> 
        <td align="left" colspan="1" rowspan="1">GO:0042384</td> 
        <td align="left" colspan="1" rowspan="1">0.83</td> 
        <td align="left" colspan="1" rowspan="1">0.03</td> 
        <td align="left" colspan="1" rowspan="1">0.015</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">small GTPase mediated signal transduction</td> 
        <td align="left" colspan="1" rowspan="1">GO:0007264</td> 
        <td align="left" colspan="1" rowspan="1">2.90</td> 
        <td align="left" colspan="1" rowspan="1">1.07</td> 
        <td align="left" colspan="1" rowspan="1">0.023</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">prostaglandin biosynthetic process</td> 
        <td align="left" colspan="1" rowspan="1">GO:0001516</td> 
        <td align="left" colspan="1" rowspan="1">0.83</td> 
        <td align="left" colspan="1" rowspan="1">0.07</td> 
        <td align="left" colspan="1" rowspan="1">0.029</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">protein kinase C activation</td> 
        <td align="left" colspan="1" rowspan="1">GO:0007205</td> 
        <td align="left" colspan="1" rowspan="1">0.83</td> 
        <td align="left" colspan="1" rowspan="1">0.07</td> 
        <td align="left" colspan="1" rowspan="1">0.029</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">neuron differentiation</td> 
        <td align="left" colspan="1" rowspan="1">GO:0030182</td> 
        <td align="left" colspan="1" rowspan="1">0.83</td> 
        <td align="left" colspan="1" rowspan="1">0.10</td> 
        <td align="left" colspan="1" rowspan="1">0.045</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">spermatogenesis</td> 
        <td align="left" colspan="1" rowspan="1">GO:0007283</td> 
        <td align="left" colspan="1" rowspan="1">1.66</td> 
        <td align="left" colspan="1" rowspan="1">0.52</td> 
        <td align="left" colspan="1" rowspan="1">0.052</td> 
       </tr> 
      </tbody> 
     </table> 
    </alternatives> 
    <table-wrap-foot> 
     <fn id="nt105"> 
      <label></label> 
      <p>“% of DR” refers to the percent of differentially regulated transcripts falling under the term; “% of other” refers to the percent of all other transcripts with GO annotation that fall under the term. P-value is the raw (nominal) p-value from Fisher's exact test.</p> 
     </fn> 
    </table-wrap-foot> 
   </table-wrap> 
  </sec> 
  <sec id="s3d"> 
   <h3>Real-time RT-PCR</h3> 
   <p>Efficiencies of the real-time RT-PCR assays here reported ranged from 92.5 percent to 108.6 percent ( 
    <xref ref-type="table" rid="pone-0012737-t006">
      Table 6 
    </xref>), and their correlation coefficients ranged from 0.988 to 0.999. The difference between threshold cycles of the last experimental sample to amplify and the first negative control well to amplify in any reaction was at least 6.49 cycles and 9.83 cycles for –RT and NTC controls, respectively. For each assay, the dissociation curve indicated that a single amplicon was produced. By real-time RT-PCR (n?=?4), two of the four genes, Stard7 and TepII, were significantly differentially regulated (p?=?0.029 and 0.014, respectively); the direction of regulation was the same as determined by microarray. The fold-change was similar to that determined by microarray for Stard7(1.87 by real-time RT-PCR compared to 2.31 by microarray), but smaller for TepII (5.66 by real-time RT-PCR compared to 29.66 by microarray). For GST, the fold-change was smaller, but the direction of regulation (4.71-fold up-regulated NS) was similar to that determined by microarray (15.24-fold up-regulated NS). This difference in real-time RT-PCR was not significant according to the Kruskall-Wallis test (p?=?0.100). For one gene, Ctr1c, the direction of regulation determined by real-time RT-PCR was opposite that determined by microarray, though the difference was essentially zero (1.03-fold down NS by RT-PCR compared to 1.74-fold up NS by microarray). This change was not significant by Kruskall-Wallis (p?=?0.443). Therefore, RT-PCR results were similar to microarray, though each transcript's fold change and statistical significance was reduced when measured by RT-PCR, compared to microarray.</p> 
   <table-wrap id="pone-0012737-t006" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0012737.t006 
    </object-id> 
    <label>Table 6</label> 
    <h3>Comparison of microarray and real-time RT-PCR results.</h3> 
    <alternatives> 
     <graphic id="pone-0012737-t006-6" xlink:href="pone.0012737.t006"></graphic> 
     <table frame="hsides" rules="groups"> 
      <colgroup span="1"> 
       <col align="left" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
      </colgroup> 
      <thead> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="3" rowspan="1">Microarray</td> 
        <td align="left" colspan="4" rowspan="1">Real-Time RT-PCR</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Gene</td> 
        <td align="left" colspan="1" rowspan="1">Fold Change</td> 
        <td align="left" colspan="1" rowspan="1">Direction</td> 
        <td align="left" colspan="1" rowspan="1">p&gt;F</td> 
        <td align="left" colspan="1" rowspan="1">Fold Change</td> 
        <td align="left" colspan="1" rowspan="1">Direction</td> 
        <td align="left" colspan="1" rowspan="1">p&gt;t</td> 
        <td align="left" colspan="1" rowspan="1">Efficiency</td> 
       </tr> 
      </thead> 
      <tbody> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Ctr1c</td> 
        <td align="left" colspan="1" rowspan="1">1.75</td> 
        <td align="left" colspan="1" rowspan="1">up</td> 
        <td align="left" colspan="1" rowspan="1">0.029</td> 
        <td align="left" colspan="1" rowspan="1">1.03</td> 
        <td align="left" colspan="1" rowspan="1">down</td> 
        <td align="left" colspan="1" rowspan="1">0.443</td> 
        <td align="left" colspan="1" rowspan="1">108.2%</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">TepII</td> 
        <td align="left" colspan="1" rowspan="1">29.66</td> 
        <td align="left" colspan="1" rowspan="1">up</td> 
        <td align="left" colspan="1" rowspan="1">0.020</td> 
        <td align="left" colspan="1" rowspan="1">5.66</td> 
        <td align="left" colspan="1" rowspan="1">up</td> 
        <td align="left" colspan="1" rowspan="1">0.014</td> 
        <td align="left" colspan="1" rowspan="1">92.5%</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">GST</td> 
        <td align="left" colspan="1" rowspan="1">15.24</td> 
        <td align="left" colspan="1" rowspan="1">up</td> 
        <td align="left" colspan="1" rowspan="1">0.009</td> 
        <td align="left" colspan="1" rowspan="1">4.71</td> 
        <td align="left" colspan="1" rowspan="1">up</td> 
        <td align="left" colspan="1" rowspan="1">0.100</td> 
        <td align="left" colspan="1" rowspan="1">95.1%</td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Stard7</td> 
        <td align="left" colspan="1" rowspan="1">2.32</td> 
        <td align="left" colspan="1" rowspan="1">down</td> 
        <td align="left" colspan="1" rowspan="1">0.024</td> 
        <td align="left" colspan="1" rowspan="1">1.89</td> 
        <td align="left" colspan="1" rowspan="1">down</td> 
        <td align="left" colspan="1" rowspan="1">0.029</td> 
        <td align="left" colspan="1" rowspan="1">100.7%</td> 
       </tr> 
      </tbody> 
     </table> 
    </alternatives> 
    <table-wrap-foot> 
     <fn id="nt106"> 
      <label></label> 
      <p>Real-time RT-PCR values were normalized to 18S rRNA (18S rRNA efficiency ?=?108.6%). “Direction” of regulation is given for nearshore samples, with respect to offshore. Transcripts: Ctr1c, copper transporter 1c; TepII, thiolester containing protein II; GST, Similar to Glutathione S-transferase; Stard7, StAR-related lipid transfer (START) domain containing 7.</p> 
     </fn> 
    </table-wrap-foot> 
   </table-wrap> 
  </sec> 
  <sec id="s3e"> 
   <h3>Tissue metal burdens</h3> 
   <p>ICP-MS results for all nine analytes are given in 
    <xref ref-type="supplementary-material" rid="pone.0012737.s003">
      Table S3 
    </xref> (sample size varies: n?=?2–8/group, specifically enumerated in 
    <xref ref-type="supplementary-material" rid="pone.0012737.s003">
      Table S3 
    </xref>). <sup>66</sup>Zn was present at a significantly higher level in the digestive gland of NS conchs (831.85 ng/mg) than OS conch digestive gland (84.53 ng/mg), or any other tissue at either site ( 
    <xref ref-type="fig" rid="pone-0012737-g005">
      Figure 5A 
    </xref>). In addition, although not statistically significant, the concentration of Zn in the NS testis (83.96 ng/mg) was approximately 15-fold higher than in the OS testis (5.43 ng/mg) ( 
    <xref ref-type="fig" rid="pone-0012737-g005">
      Figure 5A 
    </xref>). <sup>65</sup>Cu, conversely, was not significantly higher in any of the NS tissue means compared to the corresponding OS means. However, there was a non-significant (p?=?0.65), approximately five-fold difference between <sup>65</sup>Cu levels in NS (34.77 ng/mg) and OS (6.60 ng/mg) gonad ( 
    <xref ref-type="fig" rid="pone-0012737-g005">
      Figure 5B 
    </xref>, 
    <xref ref-type="supplementary-material" rid="pone.0012737.s003">
      Table S3 
    </xref>). In the tissue term of the two-way ANOVA, concentrations of <sup>58</sup>Ni, <sup>66</sup>Zn, <sup>111</sup>Cd, and <sup>238</sup>U were significantly higher in digestive gland than any other tissue. <sup>118</sup>Sn, despite being detected only at very low concentrations in these samples, was found at its highest concentrations in the neural ganglia. <sup>65</sup>Cu levels were highest in the blood, which in molluscs contains a copper-based hemocyanin pigment 
    <xref ref-type="bibr" rid="pone.0012737-vanHolde1">
      [27] 
    </xref>.</p> 
   <fig id="pone-0012737-g005" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0012737.g005 
    </object-id> 
    <label>Figure 5</label> 
    <h3>Tissue distribution of Zn (A) and Cu (B) in offshore (OS) and nearshore (NS) conchs.</h3> 
    <p>Letters indicate significant difference in 2-way ANOVA, with the two factors tissue and location, followed by Tukey-Kramer HSD (p&lt;0.05). Note different y-axis for Cu and Zn. Break in Zn data (A) omits 150–800 ng/mg. DG ?=? digestive gland; NG ?=? neural ganglia.</p> 
    <graphic xlink:href="pone.0012737.g005"></graphic> 
   </fig> 
  </sec> 
  <sec id="s3f"> 
   <h3>Correlations among microarray, histology, and metal data</h3> 
   <p>Correlation analysis was based on testis histological conditions (n?=?7–8), metal concentrations in testis and digestive gland (n?=?3–8), and expression levels of differentially-regulated transcripts under the GO biological processes spermatogenesis and small GTPase-mediated signal transduction as determined by microarray (n?=?3) ( 
    <xref ref-type="table" rid="pone-0012737-t007">
      Table 7 
    </xref>). SI was significantly inversely correlated with digestive gland Zn (??=??0.655), and inversely correlated with digestive gland Cu (??=??0.509), though this was not statistically significant (p?=?0.110). Digestive gland Zn was also significantly and inversely correlated with four of the 11 transcripts included in the analysis; gonad Zn was correlated with two of the 11. SI was significantly correlated with six of the 11 genes in the analysis.</p> 
   <table-wrap id="pone-0012737-t007" position="float"> 
    <object-id pub-id-type="doi">
      10.1371/journal.pone.0012737.t007 
    </object-id> 
    <label>Table 7</label> 
    <h3>Non-parametric correlations among Spermatogenic Index, metal concentrations, and transcript expression levels.</h3> 
    <alternatives> 
     <graphic id="pone-0012737-t007-7" xlink:href="pone.0012737.t007"></graphic> 
     <table frame="hsides" rules="groups"> 
      <colgroup span="1"> 
       <col align="left" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
       <col align="center" span="1" /> 
      </colgroup> 
      <thead> 
       <tr> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1">SI</td> 
        <td align="left" colspan="1" rowspan="1">DG Zn</td> 
        <td align="left" colspan="1" rowspan="1">Testis Zn</td> 
        <td align="left" colspan="1" rowspan="1">DG Cu</td> 
        <td align="left" colspan="1" rowspan="1">Testis Cu</td> 
        <td align="left" colspan="1" rowspan="1">PSME4</td> 
        <td align="left" colspan="1" rowspan="1">KISER</td> 
        <td align="left" colspan="1" rowspan="1">DNAJB13</td> 
        <td align="left" colspan="1" rowspan="1">DEGS1</td> 
        <td align="left" colspan="1" rowspan="1">RRAS</td> 
        <td align="left" colspan="1" rowspan="1">RAB1B</td> 
        <td align="left" colspan="1" rowspan="1">RAB1A</td> 
        <td align="left" colspan="1" rowspan="1">TIAM1</td> 
        <td align="left" colspan="1" rowspan="1">RABL4</td> 
        <td align="left" colspan="1" rowspan="1">ARL1</td> 
        <td align="left" colspan="1" rowspan="1">4R79.2</td> 
       </tr> 
      </thead> 
      <tbody> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">SI</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">DG Zn</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           ?0.655 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Testis Zn</td> 
        <td align="left" colspan="1" rowspan="1">?0.382</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.733 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">DG Cu</td> 
        <td align="left" colspan="1" rowspan="1">?0.509</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.836 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.661 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">Testis Cu</td> 
        <td align="left" colspan="1" rowspan="1">?0.345</td> 
        <td align="left" colspan="1" rowspan="1">0.333</td> 
        <td align="left" colspan="1" rowspan="1">?0.164</td> 
        <td align="left" colspan="1" rowspan="1">0.491</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">PSME4</td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1">?0.600</td> 
        <td align="left" colspan="1" rowspan="1">?0.800</td> 
        <td align="left" colspan="1" rowspan="1">?0.429</td> 
        <td align="left" colspan="1" rowspan="1">0.100</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">KISER</td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1">?0.771</td> 
        <td align="left" colspan="1" rowspan="1">?0.700</td> 
        <td align="left" colspan="1" rowspan="1">?0.543</td> 
        <td align="left" colspan="1" rowspan="1">?0.100</td> 
        <td align="left" colspan="1" rowspan="1">0.714</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">DNAJB13</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.886 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">?0.543</td> 
        <td align="left" colspan="1" rowspan="1">?0.600</td> 
        <td align="left" colspan="1" rowspan="1">?0.314</td> 
        <td align="left" colspan="1" rowspan="1">?0.314</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.943 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">DEGS1</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.829 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           ?0.829 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">?0.600</td> 
        <td align="left" colspan="1" rowspan="1">?0.600</td> 
        <td align="left" colspan="1" rowspan="1">?0.600</td> 
        <td align="left" colspan="1" rowspan="1">0.600</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.943 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.714</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">RRAS</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.829 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">?0.657</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           ?0.900 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">?0.371</td> 
        <td align="left" colspan="1" rowspan="1">0.200</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.943 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1">0.657</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">RAB1B</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.886 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           ?0.886 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">?0.700</td> 
        <td align="left" colspan="1" rowspan="1">?0.600</td> 
        <td align="left" colspan="1" rowspan="1">?0.600</td> 
        <td align="left" colspan="1" rowspan="1">0.600</td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.886 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.714</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">RAB1A</td> 
        <td align="left" colspan="1" rowspan="1">0.714</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           ?0.886 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           ?1.000 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">?0.657</td> 
        <td align="left" colspan="1" rowspan="1">0.100</td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1">0.657</td> 
        <td align="left" colspan="1" rowspan="1">0.714</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.886 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.829 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">TIAM1</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           ?0.943 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1">0.600</td> 
        <td align="left" colspan="1" rowspan="1">0.429</td> 
        <td align="left" colspan="1" rowspan="1">0.500</td> 
        <td align="left" colspan="1" rowspan="1">?0.657</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           ?0.886 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">?0.771</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           ?0.943 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">?0.771</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           ?0.943 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">?0.771</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">RABL4</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.829 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           ?0.829 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">?0.600</td> 
        <td align="left" colspan="1" rowspan="1">?0.600</td> 
        <td align="left" colspan="1" rowspan="1">?0.500</td> 
        <td align="left" colspan="1" rowspan="1">0.600</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.943 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.714</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           1.000 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.657</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.886 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.714</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           ?0.943 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">ARL1</td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1">?0.771</td> 
        <td align="left" colspan="1" rowspan="1">?0.700</td> 
        <td align="left" colspan="1" rowspan="1">?0.543</td> 
        <td align="left" colspan="1" rowspan="1">?0.100</td> 
        <td align="left" colspan="1" rowspan="1">0.714</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           1.000 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.943 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           ?0.886 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.943 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1"></td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
       <tr> 
        <td align="left" colspan="1" rowspan="1">4R79.2</td> 
        <td align="left" colspan="1" rowspan="1">0.657</td> 
        <td align="left" colspan="1" rowspan="1">?0.657</td> 
        <td align="left" colspan="1" rowspan="1">?0.800</td> 
        <td align="left" colspan="1" rowspan="1">?0.600</td> 
        <td align="left" colspan="1" rowspan="1">0.100</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.943 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.886 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.657</td> 
        <td align="left" colspan="1" rowspan="1"> 
         <bold>
           0.829 
         </bold> </td> 
        <td align="left" colspan="1" rowspan="1">0.543</td> 
        <td align="left" colspan="1" rowspan="1">0.714</td> 
        <td align="left" colspan="1" rowspan="1">-0.600</td> 
        <td align="left" colspan="1" rowspan="1">0.657</td> 
        <td align="left" colspan="1" rowspan="1">0.771</td> 
        <td align="left" colspan="1" rowspan="1"></td> 
       </tr> 
      </tbody> 
     </table> 
    </alternatives> 
    <table-wrap-foot> 
     <fn id="nt107"> 
      <label></label> 
      <p>“SI” refers to Spermatogenic Index, determined histologically. “DG” refers to digestive gland. Transcripts: PSME4, proteasome activator subunit 4; KISER, similar to kiser; DNAJB13, DnaJ related subfamily B member 13; DEGS1, degenerative spermatocyte homolog 1 lipid desaturase; RRAS, related Ras viral oncogene homolog; RAB1B, Ras-related protein 1B; RAB1A, RAB1A member Ras oncogene family; TIAM1, T-cell lymphoma invasion and metastasis 1; RABL4, Rab member of Ras oncogene family-like 4; ARL1, ADP ribosylation factor-like 1; 4R79.2, 4R79.2 hypothetical protein. Boldface values indicate significance (p&lt;0.05).</p> 
     </fn> 
    </table-wrap-foot> 
   </table-wrap> 
  </sec> 
 </sec> 
 <sec id="s4"> 
  <h3>Discussion</h3> 
  <p>The testis histological data gathered in this study show a strong difference between NS and OS conchs in terms of development throughout the reproductive season ( 
   <xref ref-type="table" rid="pone-0012737-t003">
     Table 3 
   </xref>, 
   <xref ref-type="fig" rid="pone-0012737-g002">
     Figure 2 
   </xref>). Queen conchs from NS sites had dramatically less spermatogenic tissue in relation to OS conchs in both February and June. Additionally, spermatogenesis was somewhat reduced in February and markedly reduced in June in NS conchs, during the peak of the conch reproductive season. The SI values give a clear picture of the reduction in spermatogenic capability of NS conchs during both February and June, and highlight the decrease in spermatogenic capability of NS conchs between February and June. Although all NS conchs were undergoing active spermatogenesis in February, there is evidence of developmental delay in February, as a lower percentage of NS conchs were Spawning Capable compared to OS conchs, as well as evidence of significant regression from February to June. The histological data from June collections show that NS conchs are unable to maintain reproductive capability throughout the spawning season. While all OS conchs collected in June had high SI values and were Spawning Capable, only 20% of the NS conchs were Spawning Capable, and all had very low SI values. The current study only presents this histological data as a physiological anchor for gene expression at two sites in the Florida Keys. While we acknowledge that site-specific effects may play a large role in testis development, these observations mirror results from conchs collected at similar NS and OS areas of the Florida Keys in 1999 
   <xref ref-type="bibr" rid="pone.0012737-Delgado1">
     [2] 
   </xref>, suggesting that NS conchs show a persistent, long-term reduction in reproductive capability. Moreover, the histology here reported showed a more dramatic reduction than that reported for 1999.</p> 
  <p>These results complement the results of our microarray and ICP-MS experiments. NS conch testis transcription differed from OS in the GO biological processes proton transport (GO:0015992), spermatogenesis (GO:0007283), small GTPase-mediated signal transduction (GO:0007264), and others ( 
   <xref ref-type="table" rid="pone-0012737-t005">
     Table 5 
   </xref>, 
   <xref ref-type="fig" rid="pone-0012737-g004">
     Figure 4 
   </xref>). This supports specific hypothesis (1), and also suggests that inhibition of small GTPase (Ras)-mediated signaling in NS testis contributes to NS reproductive failure. ICP-MS analysis indicated that Cu and Zn were elevated in some NS conch tissues, providing preliminary support for specific hypothesis (2), and creating the hypothesis that Cu and Zn may be a causative factor in reproductive failure of NS conchs in the Florida Keys. It is important to note that site-specific differences in metal concentrations and gene expression surely exist. Future studies will incorporate metal and gene expression data from additional sites to determine whether differences in these parameters are as consistent as the histological differences observed throughout the NS and OS Florida Keys.</p> 
  <sec id="s4a"> 
   <h3>Conch testis gene expression</h3> 
   <p>The gene expression analysis in the conch testis reveals, logically, that spermatogenesis-associated transcripts are down-regulated NS. Correspondingly, mitochondrial transcripts are significantly down-regulated in NS testes. The effects on proton transport identified by the GO enrichment analysis could be either a cause or a result of the observed reduction in spermatogenesis in NS testes, given the important role of mitochondria in spermatozoa and in sperm maturation 
    <xref ref-type="bibr" rid="pone.0012737-Pena1">
      [28] 
    </xref>– 
    <xref ref-type="bibr" rid="pone.0012737-Yaffe1">
      [30] 
    </xref>. Our finding is likely the result of the reduction in mature spermatozoa, and consequent numeric reduction in mitochondria, in NS testes as opposed to OS.</p> 
   <p>Under the Biological Process GO:0007283, spermatogenesis, we identified differentially regulated transcripts with major roles in spermatogenesis in species ranging from 
    <italic>
      Drosophila 
    </italic> to humans, including degenerative spermatocyte homolog 1 (DEGS1) 
    <xref ref-type="bibr" rid="pone.0012737-Ternes1">
      [31] 
    </xref>; Similar to Kiser (homologous to slowmo) 
    <xref ref-type="bibr" rid="pone.0012737-Reeve1">
      [32] 
    </xref>; proteasome activator subunit 4 (PSME4/PA200) 
    <xref ref-type="bibr" rid="pone.0012737-Khor1">
      [33] 
    </xref>; DnaJ related, subfamily B, member 13 (DNAJB13) 
    <xref ref-type="bibr" rid="pone.0012737-Guan1">
      [34] 
    </xref>, 
    <xref ref-type="bibr" rid="pone.0012737-Guan2">
      [35] 
    </xref>, which is also related to the TSARG genes in rats 
    <xref ref-type="bibr" rid="pone.0012737-Yang1">
      [36] 
    </xref> and mice 
    <xref ref-type="bibr" rid="pone.0012737-Li1">
      [37] 
    </xref>; and nuclear autoantigenic sperm protein (histone-binding) (NASP) 
    <xref ref-type="bibr" rid="pone.0012737-Alekseev1">
      [38] 
    </xref>. These genes, important for the process of spermatogenesis in a wide range of species, appear to be conserved in queen conch, and were all down-regulated NS in the present study.</p> 
   <p>A surprising result of the GO enrichment analysis was the enrichment of the term “small GTPase-mediated signal transduction.” Most of the genes under this term are related to Ras-GTPases, proto-oncogenes involved in mammalian tumor formation and developmental disorders 
    <xref ref-type="bibr" rid="pone.0012737-Karnoub1">
      [39] 
    </xref>. Seven genes that fall under this GO term were differentially regulated in our experiment, including related Ras viral oncogene homolog (Rras); Ras related protein 1b (Rap1b); RAB1A member of Ras oncogene family; T-cell lymphoma invasion and metastasis 1 (TIAM1); RAB member of ras oncogene family 4-like (RABL4); ADP ribosylation factor-like 1 (ARL1); and 4R79.2, a hypothetical GTP-binding protein identified in 
    <italic>
      Caenorhabditis elegans 
    </italic>. All of these genes are down-regulated with the exception of TIAM1 ( 
    <xref ref-type="supplementary-material" rid="pone.0012737.s002">
      Table S2 
    </xref>). Ras function has been described in invertebrates including ascidians, for which Ras signaling is involved in embryonic tissue development 
    <xref ref-type="bibr" rid="pone.0012737-Nishida1">
      [40] 
    </xref>, and 
    <italic>
      Drosophila 
    </italic>, for which Rap1 is involved in cell adhesion and polarity during epidermal growth factor receptor-mediated tissue growth 
    <xref ref-type="bibr" rid="pone.0012737-OKeefe1">
      [41] 
    </xref>. Ras genes are also known to be involved in vertebrate and invertebrate testis development. The Ras-cyclin D2 pathway is involved in mouse spermatogonial stem cell development 
    <italic>
      in vitro 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Lee1">
      [42] 
    </xref>. MAPK and Rap-GEF signaling pathways are also involved in testis development and renewal in 
    <italic>
      Drosophila 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Singh1">
      [43] 
    </xref>. Therefore, Ras-GTPase signaling may play a major role in conch testis tissue growth and differentiation. Histological SI was correlated with six of the 11 differentially regulated transcripts involved in spermatogenesis or small GTPase-mediated signaling ( 
    <xref ref-type="table" rid="pone-0012737-t007">
      Table 7 
    </xref>). This suggests that transcription of these genes is indicative of the overall maturation of the testis tissue in queen conchs, and that perturbation of normal transcription of these genes is detrimental to spermatogenesis.</p> 
   <p>Transcripts evaluated by real-time RT-PCR were selected based on their differential regulation between NS and OS, according to the microarray study ( 
    <xref ref-type="supplementary-material" rid="pone.0012737.s002">
      Table S2 
    </xref>) and their varied and interesting biological functions. GO biological processes of these gene products include: Ctr1c, copper transmembrane transport; TepII, antibacterial humoral response; GST, glutathione metabolic process; Stard7, no biological process (but related to steroidogenic acute regulatory (StAR) protein). The results of our real-time RT-PCR assays were largely successful in validating the changes observed in the microarray study. TepII, GST, and Stard7 were confirmed by real-time RT-PCR, though the GST result was not statistically significant. Ctr1c, however, was essentially unchanged between NS and OS samples in real-time RT-PCR, with a 1.03-fold change in the direction opposite that determined by microarray. The difference in results between platforms is possibly due in part to the small sample size (n?=?4) used for both assays; increased sample size would lend power to the analyses. Unfortunately, permitting regulations limit sample size for a protected species such as 
    <italic>
      S. gigas 
    </italic>. It is also possible that for Ctr1c our probe was designed to a region with homology to other proteins or isoforms in the SLC31 family of copper transporters, causing the lack of consistency between microarray and real-time RT-PCR. Changes in TepII, GST, and Stard7 may indicate that stressors affecting NS conchs cause changes in immune response, xenobiotic metabolism/redox balance, and steroidogenesis, respectively. However, these are single gene changes, and so should be interpreted carefully.</p> 
  </sec> 
  <sec id="s4b"> 
   <h3>Potential role of metals as a reproductive stressor</h3> 
   <p>Our ICP-MS data indicated that Cu and Zn, two known reproductive toxicants in gastropods, were elevated in some NS conch tissues. Our study also included other analytes with known toxic effects, including Ni, Ag, Cd, Sn, Hg, and U. Sr was included due to its role in shell-building; it is known to be physiologically beneficial in gastropods at low doses, but toxic at high levels 
    <xref ref-type="bibr" rid="pone.0012737-Buchardt1">
      [44] 
    </xref>. However, few differences were observed for the latter seven analytes. The effects of Cu and Zn on gastropod reproductive output have been well-documented, although most examples relate to females. In laboratory exposures, Cu has resulted in reduced fecundity in 
    <italic>
      Helix aspersa 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Laskowski1">
      [6] 
    </xref>, reduced egg-laying and a dose-dependent reduction in hatching in 
    <italic>
      Pomacea palludosa 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Rogevich1">
      [7] 
    </xref>, and, as copper oxychloride, reduced oocyte number in the ovotestis of 
    <italic>
      Helix aspersa 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Snyman1">
      [8] 
    </xref>. Zn exposures, likewise have impacted reproduction in numerous studies, resulting in reduced fecundity and population growth rate in 
    <italic>
      Valvata piscinalis 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Ducrot1">
      [10] 
    </xref>, reduced fecundity in 
    <italic>
      Helix aspersa 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Laskowski1">
      [6] 
    </xref>, and, as an effluent containing Zn, Cd, and Fe, mortality and reduced egg laying in 
    <italic>
      Lymnaea palustris 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Coeurdassier1">
      [9] 
    </xref>.</p> 
   <p>General and point sources of heavy metals in south Florida include storm water runoff, roadway contaminants, septic system leachate, and boats, which may be responsible for high levels of Hg, Pb, Zn, and Cu in waterways 
    <xref ref-type="bibr" rid="pone.0012737-Kruczynski1">
      [12] 
    </xref>. Elevated Cu, Zn, Cr, Hg, Pb, and Ni levels have been identified in Biscayne Bay, adjacent to the city of Miami, as well as at the outflow of canals 
    <xref ref-type="bibr" rid="pone.0012737-Carnahan1">
      [11] 
    </xref>. Additionally, heavy metals including Cu and Zn have been detected in sediments and seagrass beds, both habitats occupied by conchs, as well as in surface waters at multiple sites throughout south Florida, with Cu sometimes exceeding guidelines for aquatic life and sediment quality 
    <xref ref-type="bibr" rid="pone.0012737-Lewis1">
      [13] 
    </xref>. Taken together, this information suggests that potential sources of Cu and Zn contamination exist in the Florida Keys and are likely to be primarily on land or close to shore, further supporting the plausibility of these metals interfering with NS testis development.</p> 
   <p>In the present study, Zn was elevated in the digestive gland, and possibly in the gonad, of NS conchs ( 
    <xref ref-type="fig" rid="pone-0012737-g005">
      Figure 5A 
    </xref>). Coupled with the knowledge that Zn causes reduced fecundity in other gastropod species 
    <xref ref-type="bibr" rid="pone.0012737-Laskowski1">
      [6] 
    </xref>, 
    <xref ref-type="bibr" rid="pone.0012737-Coeurdassier1">
      [9] 
    </xref>, this finding suggests that Zn may contribute to the reproductive failure of NS conchs. The observed NS digestive gland mean concentration of 831.85 ng Zn/mg tissue is similar to the body burden observed (approx. 200–500 µg Zn/g tissue) in effluent treatments resulting in mortality and reduced fecundity in 
    <italic>
      Lymnaea palustris 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Coeurdassier1">
      [9] 
    </xref>. While available data in the literature focus on female-mediated reproductive inhibition measured as reduced fecundity, studies of fecundity may miss mechanistic effects in both males and females. Further, while the gonad is the apparent site of action for any potential toxicant, accumulation of Zn in the digestive gland in the present study is also likely to be a significant finding. The digestive gland is adjacent to the gonad and is believed to be a site of metal accumulation and detoxification in gastropods 
    <xref ref-type="bibr" rid="pone.0012737-Gros1">
      [45] 
    </xref>– 
    <xref ref-type="bibr" rid="pone.0012737-Nott1">
      [48] 
    </xref>. While a recent study indicates that Zn concentrations in the testis of the Japanese eel 
    <italic>
      Anguilla japonica 
    </italic> track the progression of spermatogenesis 
    <xref ref-type="bibr" rid="pone.0012737-Yamaguchi1">
      [49] 
    </xref>, it is important to note that an excess of Zn from external sources could still have a deleterious effect, as is possible in the present study. The relationship between Zn and spermatogenesis is likely complex, and should be the subject of further study.</p> 
   <p>No significant differences in Cu concentrations within any tissue were found between NS and OS. The mean concentration of 34.77 ng Cu/mg tissue observed in NS conch testis in this study is still only a fraction of the toxic levels accumulated in studies by Rogevich et al. 
    <xref ref-type="bibr" rid="pone.0012737-Rogevich1">
      [7] 
    </xref> (396.60 ng Cu/mg tissue) and Snyman et al. 
    <xref ref-type="bibr" rid="pone.0012737-Snyman1">
      [8] 
    </xref> (260.47 ng Cu/mg tissue), but is approximately five times the OS mean of 6.60 ng Cu/mg tissue. Further, the aforementioned studies measured whole body Cu rather than tissue-specific accumulation. Blood levels of Cu in our study (40.18 ng Cu/mg tissue NS, 58.90 ng Cu/mg tissue OS) were the highest of any tissue, and it would be difficult to separate the Cu contribution of hemocyanin in a tissue to the amount actually bound up in cells. In other words, blood Cu bound in hemocyanin might obscure differences between tissues. Therefore, Cu might still be a factor in NS reproductive failure, and future studies will attempt to test this possibility. It should also be noted that many environmental factors could be considered stressors in a complex environmental mixture, and as with all real-world situations, multiple stressors are likely involved at our NS sites. The inverse correlations between Cu and Zn concentrations in the digestive gland and SI ( 
    <xref ref-type="table" rid="pone-0012737-t007">
      Table 7 
    </xref>) provide support for the argument that accumulation of metals, including Zn and possibly Cu, in the conch digestive gland affects development of the conch testis. These hypotheses will be examined in future studies.</p> 
  </sec> 
  <sec id="s4c"> 
   <h3>High-throughput sequencing for gastropod transcriptomics</h3> 
   <p>The approximately 60,000 extant gastropods make up the largest class within the 100,000-member phylum Mollusca, the second-largest animal phylum 
    <xref ref-type="bibr" rid="pone.0012737-Ruppert1">
      [50] 
    </xref>. However, very little work has been done in the area of gastropod genomics. A PubMed search for “gastropod microarray” on 16 July 2010 yielded only 14 results, one of which was non-germane. Two of the remaining 13 papers discussed toxicogenomics as a tool for understanding endocrine disruption in invertebrates 
    <xref ref-type="bibr" rid="pone.0012737-Iguchi1">
      [51] 
    </xref>, 
    <xref ref-type="bibr" rid="pone.0012737-Iguchi2">
      [52] 
    </xref>. The remaining 11 papers applied to only five genera of gastropods: 
    <italic>
      Helix 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Guiller1">
      [53] 
    </xref>, 
    <italic>
      Lymnaea 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Azami1">
      [54] 
    </xref>, 
    <italic>
      Haliotis 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-vanRensburg1">
      [55] 
    </xref>, 
    <italic>
      Aplysia 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Lee2">
      [56] 
    </xref>, and 
    <italic>
      Biomphalaria 
    </italic> 
    <xref ref-type="bibr" rid="pone.0012737-Adema1">
      [57] 
    </xref>, 
    <xref ref-type="bibr" rid="pone.0012737-Lockyer1">
      [58] 
    </xref>, or to schistosomes that use both humans and gastropods as hosts 
    <xref ref-type="bibr" rid="pone.0012737-Gobert1">
      [59] 
    </xref>– 
    <xref ref-type="bibr" rid="pone.0012737-Dillon1">
      [63] 
    </xref>. A fielded search for “gastropoda[organism]” on GEO yielded only 13 results, consisting of the two submissions here reported, in addition to two platforms (GPL3635 and GPL3636) and two gene expression datasets (GSE4628 and GSE18783) for 
    <italic>
      Aplysia californica 
    </italic>, one platform (GPL7421) and one gene expression dataset (GSE13039) for 
    <italic>
      Haliotis asinina 
    </italic>, and two platforms (GPL9129 and GPL9483) and two gene expression datasets (GSE16596, GSE18705, and GSE22037) for 
    <italic>
      Biomphalaria glabrata 
    </italic>. The use of high-throughput sequencing allowed us to make a significant contribution to this growing field. Still, aside from several heavily studied genera, one of which ( 
    <italic>
      Biomphalaria 
    </italic>) has direct importance for human health, the entire realm of gastropod genomics remains to be developed.</p> 
  </sec> 
  <sec id="s4d"> 
   <h3>Conclusions</h3> 
   <p>This study has provided new information regarding the reproductive failure of NS conchs in the Florida Keys. The major findings of this study include the following: first, that failure of NS conchs to reproduce is coupled with a reduction in NS testis development, as previously reported 
    <xref ref-type="bibr" rid="pone.0012737-Delgado1">
      [2] 
    </xref>, and premature regression of NS testis. Second, the microarray results indicate that reduced testis tissue in NS male conchs is concurrent with a decrease in the expression of many genes related to spermatogenesis and mitochondrial function. Transcription of small GTPase-related signaling genes is clearly affected, and this may contribute to the lack of testis tissue development, but this requires further study. Finally, this study supports the hypothesis that heavy metals may contribute to the reproductive failure of NS conchs. Zn and possibly Cu are elevated in the NS conch digestive gland, and Zn may be elevated in the testis. Given that Zn and Cu are known to reduce gastropod fecundity, the possibility that these same metals may also inhibit gametogenesis in both males and females merits further consideration.</p> 
   <p>Note that this study characterized effects of the NS environment on reproductive tissue of male conchs. While many gastropod reproduction studies rely on egg-laying (i.e. female-mediated effects) as the measure of average reproductive success 
    <xref ref-type="bibr" rid="pone.0012737-Laskowski1">
      [6] 
    </xref>– 
    <xref ref-type="bibr" rid="pone.0012737-Ducrot1">
      [10] 
    </xref>, the phenomenon observed in the NS Florida Keys affects both males and females 
    <xref ref-type="bibr" rid="pone.0012737-Delgado1">
      [2] 
    </xref>, 
    <xref ref-type="bibr" rid="pone.0012737-Glazer1">
      [3] 
    </xref>. Conchs rely on mate-pairing and copulation 
    <xref ref-type="bibr" rid="pone.0012737-Davis1">
      [4] 
    </xref>, rather than broadcast spawning or other mating strategies that would require fewer reproductive males. Logically, this lack of male reproductive maturity could have a significant impact on the conch population. Future studies will aim to assess transcriptional effects on the ovaries of affected NS females, in addition to males. Although the testicular regression in NS conchs appears to be a persistent problem in the Florida Keys, it is apparently reversible at the level of the individual, as many NS conchs transplanted to OS areas become Spawning Capable 
    <xref ref-type="bibr" rid="pone.0012737-Delgado1">
      [2] 
    </xref>. This suggests that transcriptional effects, which can immediately and transiently respond to environmental factors, can play an important role in understanding the disparity in conch reproduction from NS to OS, as well as identifying responsible factors. Therefore, the combination of microarray studies with more traditional approaches will yield useful information for managers as they work to facilitate the recovery of NS queen conch populations in the Florida Keys.</p> 
  </sec> 
 </sec> 
 <sec id="s5" sec-type="supplementary-material"> 
  <h3>Supporting Information</h3> 
  <supplementary-material content-type="local-data" id="pone.0012737.s001"> 
   <label>Table S1</label> 
   <p>Validation of 18S rRNA as a reference gene for real-time RT-PCR. “Tukey” denotes whether interaction term tissue*OS/NS is significantly different by ANOVA (only if p&lt;0.05) followed by Tukey-Kramer HSD for multiple comparisons. Within each analyte, values not connected by the same letter are significantly different. *NS samples for 06/2007 were contaminated with digestive gland. Microarray and real-time RT-PCR reported in the present study was conducted with 02/2007 samples.</p> 
   <p>(0.03 MB DOC)</p> 
   <media mime-subtype="msword" mimetype="application" xlink:href="pone.0012737.s001.doc"> 
    <p>Click here for additional data file.</p> 
   </media> 
  </supplementary-material> 
  <supplementary-material content-type="local-data" id="pone.0012737.s002"> 
   <label>Table S2</label> 
   <p>List of all differentially regulated probes from the microarray experiment (p&lt;0.05, FDR?=?5%). No gene h3 is given for probes with insufficient annotation. “Diff of treatment ?=? (NS)-(OS)” gives the log2-transformed fold change with respect to NS (OS as control). P-value as determined by one-way ANOVA (FDR?=?5%).</p> 
   <p>(0.15 MB XLS)</p> 
   <media mime-subtype="vnd.ms-excel" mimetype="application" xlink:href="pone.0012737.s002.xls"> 
    <p>Click here for additional data file.</p> 
   </media> 
  </supplementary-material> 
  <supplementary-material content-type="local-data" id="pone.0012737.s003"> 
   <label>Table S3</label> 
   <p>ICP-MS analysis. “Tukey” denotes whether interaction term tissue*OS/NS is significantly different by ANOVA (only if p&lt;0.05) followed by Tukey-Kramer HSD for multiple comparisons. Within each analyte, values not connected by the same letter are significantly different.</p> 
   <p>(0.11 MB DOC)</p> 
   <media mime-subtype="msword" mimetype="application" xlink:href="pone.0012737.s003.doc"> 
    <p>Click here for additional data file.</p> 
   </media> 
  </supplementary-material> 
 </sec> 
 <back> 
  <ack> 
   <p>The authors would like to acknowledge Dejerianne Ostrow, William G. Farmerie, and Christopher Martyniuk from UF, who aided in sequencing and microarray experiments, as well as Gabriel Delgado, from FWRI, who has contributed to sampling efforts and discussions throughout this project. We thank the Louisiana State University Veterinary Pathology laboratory for histological processing.</p> 
  </ack> 
  <fn-group> 
   <fn fn-type="conflict"> 
    <p> 
     <bold>
       Competing Interests: 
     </bold>The authors have declared that no competing interests exist.</p> 
   </fn> 
   <fn fn-type="financial-disclosure"> 
    <p> 
     <bold>
       Funding: 
     </bold>This project was funded by the following sources: The United States Environmental Protection Agency ( 
     <ext-link ext-link-type="uri" xlink:href="http://www.epa.gov">
       www.epa.gov 
     </ext-link>; #X7974799-03), Florida Fish and Wildlife Conservation Commission ( 
     <ext-link ext-link-type="uri" xlink:href="http://www.myfwc.com">
       www.myfwc.com 
     </ext-link>; #F2410 and #NG06-106) and the University of Florida College of Veterinary Medicine ( 
     <ext-link ext-link-type="uri" xlink:href="http://www.vetmed.ufl.edu">
       www.vetmed.ufl.edu 
     </ext-link>). Publication of this article was funded in part by the University of Florida Open-Access Publishing Fund. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p> 
   </fn> 
  </fn-group> 
  <ref-list> 
   <h3>References</h3> 
   <ref id="pone.0012737-Chakalall1"> 
    <label>1</label> 
    <mixed-citation publication-type="other"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Chakalall 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Crispoldi 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Garibaldi 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lupin 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mateo 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2007 
     </year> 
     <comment>
       World markets and industry of selected commercially-exploited aquatic species: Caribbean queen conch ( 
      <italic>
        Strombus gigas 
      </italic>). Food and Agriculture Organization of the United Nations. 
      <ext-link ext-link-type="uri" xlink:href="http://www.fao.org/docrep/006/Y5261E/y5261e07.htm">
        http://www.fao.org/docrep/006/Y5261E/y5261e07.htm 
      </ext-link> 
     </comment> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Delgado1"> 
    <label>2</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Delgado 
       </surname> 
       <given-names>
         GA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bartels 
       </surname> 
       <given-names>
         CT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Glazer 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Brown-Peterson 
       </surname> 
       <given-names>
         NJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McCarthy 
       </surname> 
       <given-names>
         KJ 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2004 
     </year> 
     <article-h3>
       Translocation as a strategy to rehabilitate the queen conch ( 
      <italic>
        Strombus gigas 
      </italic>) population in the Florida Keys. 
     </article-h3> 
     <source />Fish Bull 
     <volume>
       102 
     </volume> 
     <fpage>
       278 
     </fpage> 
     <lpage>
       288 
     </lpage> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Glazer1"> 
    <label>3</label> 
    <mixed-citation publication-type="book"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Glazer 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Delgado 
       </surname> 
       <given-names>
         GA 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2003 
     </year> 
     <article-h3>
       Towards a holistic strategy to managing Florida's queen conch ( 
      <italic>
        Strombus gigas 
      </italic>) population. 
     </article-h3> 
     <person-group person-group-type="editor"> 
      <name> 
       <surname>
         Aldana-Aranda 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
     </person-group> 
     <source />El caracol 
     <italic>
       Strombus gigas 
     </italic>: conocimiento integral para su manejo sustenable en el Caribe 
     <publisher-loc>
       Yúcatan, México 
     </publisher-loc> 
     <publisher-name>
       CYTED. Programa Iberoamerican de Ciencia y Tecnologia para Desarolllo 
     </publisher-name> 
     <fpage>
       73 
     </fpage> 
     <lpage>
       80 
     </lpage> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Davis1"> 
    <label>4</label> 
    <mixed-citation publication-type="book"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Davis 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2000 
     </year> 
     <article-h3>
       Queen conch ( 
      <italic>
        Strombus gigas 
      </italic>) culture techniques for research, stock enhancement and growout markets. 
     </article-h3> 
     <person-group person-group-type="editor"> 
      <name> 
       <surname>
         Fingerman 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nagabhushanam 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
     </person-group> 
     <source />Recent Advances in Marine Biotechnology, Volume 4 Aquaculture, Part A Seaweeds and Invertebrates 
     <publisher-loc>
       Enfield, NH 
     </publisher-loc> 
     <publisher-name>
       Science Publishers, Inc 
     </publisher-name> 
     <fpage>
       27 
     </fpage> 
     <lpage>
       59 
     </lpage> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Doney1"> 
    <label>5</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Doney 
       </surname> 
       <given-names>
         SC 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2010 
     </year> 
     <article-h3>
       The growing human footprint on coastal and open-ocean biogeochemistry. 
     </article-h3> 
     <source />Science 
     <volume>
       328 
     </volume> 
     <fpage>
       1512 
     </fpage> 
     <lpage>
       1516 
     </lpage> 
     <pub-id pub-id-type="pmid">
       20558706 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Laskowski1"> 
    <label>6</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Laskowski 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hopkin 
       </surname> 
       <given-names>
         SP 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       1996 
     </year> 
     <article-h3>
       Effect of Zn, Cu, Pb, and Cd on fitness in snails ( 
      <italic>
        Helix aspersa 
      </italic>). 
     </article-h3> 
     <source />Ecotoxicol Environ Saf 
     <volume>
       34 
     </volume> 
     <fpage>
       59 
     </fpage> 
     <lpage>
       69 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8793321 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Rogevich1"> 
    <label>7</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Rogevich 
       </surname> 
       <given-names>
         EC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hoang 
       </surname> 
       <given-names>
         TC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rand 
       </surname> 
       <given-names>
         GM 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Effects of sublethal chronic copper exposure on the growth and reproductive success of the Florida apple snail ( 
      <italic>
        Pomacea paludosa 
      </italic>). 
     </article-h3> 
     <source />Arch Environ Contam Toxicol 
     <volume>
       56 
     </volume> 
     <fpage>
       450 
     </fpage> 
     <lpage>
       458 
     </lpage> 
     <pub-id pub-id-type="pmid">
       18846313 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Snyman1"> 
    <label>8</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Snyman 
       </surname> 
       <given-names>
         RG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Reinecke 
       </surname> 
       <given-names>
         AJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Reinecke 
       </surname> 
       <given-names>
         SA 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2004 
     </year> 
     <article-h3>
       Changes in oocyte numbers in the ovotestis of 
      <italic>
        Helix aspersa 
      </italic>, after experimental exposure to the fungicide copper oxychloride. 
     </article-h3> 
     <source />Bull Environ Contam Toxicol 
     <volume>
       73 
     </volume> 
     <fpage>
       398 
     </fpage> 
     <lpage>
       403 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15386058 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Coeurdassier1"> 
    <label>9</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Coeurdassier 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         de Vaufleury 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Crini 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Scheifler 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Badot 
       </surname> 
       <given-names>
         PM 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2005 
     </year> 
     <article-h3>
       Assessment of whole effluent toxicity on aquatic snails: bioaccumulation of Cr, Zn, and Fe, and individual effects in bioassays. 
     </article-h3> 
     <source />Environ Toxicol Chem 
     <volume>
       24 
     </volume> 
     <fpage>
       198 
     </fpage> 
     <lpage>
       204 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15683184 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Ducrot1"> 
    <label>10</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ducrot 
       </surname> 
       <given-names>
         V 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Pery 
       </surname> 
       <given-names>
         AR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Mons 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Queau 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Charles 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2007 
     </year> 
     <article-h3>
       Dynamic energy budget as a basis to model population-level effects of zinc-spiked sediments in the gastropod 
      <italic>
        Valvata piscinalis 
      </italic>. 
     </article-h3> 
     <source />Environ Toxicol Chem 
     <volume>
       26 
     </volume> 
     <fpage>
       1774 
     </fpage> 
     <lpage>
       1783 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17702355 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Carnahan1"> 
    <label>11</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Carnahan 
       </surname> 
       <given-names>
         EA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hoare 
       </surname> 
       <given-names>
         AA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hallock 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lidz 
       </surname> 
       <given-names>
         BH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Reich 
       </surname> 
       <given-names>
         CD 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       Distribution of heavy metals and foraminiferal assemblages in sediments of Biscayne Bay, Florida, USA. 
     </article-h3> 
     <source />J Coastal Res 
     <volume>
       24 
     </volume> 
     <fpage>
       159 
     </fpage> 
     <lpage>
       169 
     </lpage> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Kruczynski1"> 
    <label>12</label> 
    <mixed-citation publication-type="book"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Kruczynski 
       </surname> 
       <given-names>
         WL 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       1999 
     </year> 
     <article-h3>
       Water quality concerns in the Florida Keys: Sources, effects, and solutions. 
     </article-h3> 
     <publisher-name>
       Florida Keys National marine Sanctuary Water Quality Protection Program 
     </publisher-name> 
     <size units="page"></size> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Lewis1"> 
    <label>13</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Lewis 
       </surname> 
       <given-names>
         MA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dantin 
       </surname> 
       <given-names>
         DD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Chancy 
       </surname> 
       <given-names>
         CA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Abel 
       </surname> 
       <given-names>
         KC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lewis 
       </surname> 
       <given-names>
         CG 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2007 
     </year> 
     <article-h3>
       Florida seagrass habitat evaluation: a comparative survey for chemical quality. 
     </article-h3> 
     <source />Environ Pollut 
     <volume>
       146 
     </volume> 
     <fpage>
       206 
     </fpage> 
     <lpage>
       218 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17049464 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-1"> 
    <label>14</label> 
    <mixed-citation publication-type="other"> 
     <person-group person-group-type="editor"> 
      <name> 
       <surname>
         Wyanski 
       </surname> 
       <given-names>
         DS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Brown-Peterson 
       </surname> 
       <given-names>
         NJ 
       </given-names> 
      </name> 
     </person-group> 
     <article-h3>
       Kofoed TMN, Tomkiewicz J, Pederson JS. Histological study of hormonally induced spermatogenesis in European eel ( 
      <italic>
        Anguilla anguilla 
      </italic>). 
     </article-h3> 
     <fpage>
       83 
     </fpage> 
     <lpage>
       86 
     </lpage> 
     <comment>
       4 
      <sup>th</sup> Workshop on Gonadal Histology of Fishes; 2010; El Puerto de Santa Maria, Spain. 
      <ext-link ext-link-type="uri" xlink:href="http://hdl.handle.net/10261/24937">
        http://hdl.handle.net/10261/24937 
      </ext-link> 
     </comment> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-GarciaReyero1"> 
    <label>15</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Garcia-Reyero 
       </surname> 
       <given-names>
         N 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Griffitt 
       </surname> 
       <given-names>
         RJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liu 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kroll 
       </surname> 
       <given-names>
         KJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Farmerie 
       </surname> 
       <given-names>
         WG 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       Construction of a robust microarray from a non-model species (largemouth bass) using pyrosequencing technology. 
     </article-h3> 
     <source />J Fish Biol 
     <volume>
       72 
     </volume> 
     <fpage>
       2354 
     </fpage> 
     <lpage>
       2376 
     </lpage> 
     <pub-id pub-id-type="pmid">
       19936325 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Droege1"> 
    <label>16</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Droege 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hill 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       The Genome Sequencer FLX System–longer reads, more applications, straight forward bioinformatics and more complete data sets. 
     </article-h3> 
     <source />J Biotechnol 
     <volume>
       136 
     </volume> 
     <fpage>
       3 
     </fpage> 
     <lpage>
       10 
     </lpage> 
     <pub-id pub-id-type="pmid">
       18616967 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Farmerie1"> 
    <label>17</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Farmerie 
       </surname> 
       <given-names>
         WG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hammer 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liu 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sahni 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Schneider 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2005 
     </year> 
     <article-h3>
       Biological workflow with BlastQuest. 
     </article-h3> 
     <source />Data Knowl Eng 
     <volume>
       53 
     </volume> 
     <fpage>
       75 
     </fpage> 
     <lpage>
       97 
     </lpage> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Ishikawa1"> 
    <label>18</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ishikawa 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       1977 
     </year> 
     <article-h3>
       Evolution of ribosomal RNA. 
     </article-h3> 
     <source />Comp Biochem Physiol B 
     <volume>
       58 
     </volume> 
     <fpage>
       1 
     </fpage> 
     <lpage>
       7 
     </lpage> 
     <pub-id pub-id-type="pmid">
       400949 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Groppe1"> 
    <label>19</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Groppe 
       </surname> 
       <given-names>
         JC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Morse 
       </surname> 
       <given-names>
         DE 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       1993 
     </year> 
     <article-h3>
       Isolation of full-length RNA templates for reverse transcription from tissues rich in RNase and proteoglycans. 
     </article-h3> 
     <source />Anal Biochem 
     <volume>
       210 
     </volume> 
     <fpage>
       337 
     </fpage> 
     <lpage>
       343 
     </lpage> 
     <pub-id pub-id-type="pmid">
       7685567 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Brazma1"> 
    <label>20</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Brazma 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hingamp 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Quackenbush 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sherlock 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Spellman 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2001 
     </year> 
     <article-h3>
       Minimum information about a microarray experiment (MIAME)-toward standards for microarray data. 
     </article-h3> 
     <source />Nat Genet 
     <volume>
       29 
     </volume> 
     <fpage>
       365 
     </fpage> 
     <lpage>
       371 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11726920 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Rozen1"> 
    <label>21</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Rozen 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Skaletsky 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2000 
     </year> 
     <article-h3>
       Primer3 on the WWW for general users and for biologist programmers. 
     </article-h3> 
     <source />Methods Mol Biol 
     <volume>
       132 
     </volume> 
     <fpage>
       365 
     </fpage> 
     <lpage>
       386 
     </lpage> 
     <pub-id pub-id-type="pmid">
       10547847 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Bustin1"> 
    <label>22</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Bustin 
       </surname> 
       <given-names>
         SA 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2000 
     </year> 
     <article-h3>
       Absolute quantification of mRNA using real-time reverse transcription polymerase chain reaction assays. 
     </article-h3> 
     <source />J Mol Endocrinol 
     <volume>
       25 
     </volume> 
     <fpage>
       169 
     </fpage> 
     <lpage>
       193 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11013345 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Bustin2"> 
    <label>23</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Bustin 
       </surname> 
       <given-names>
         SA 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2002 
     </year> 
     <article-h3>
       Quantification of mRNA using real-time reverse transcription PCR (RT-PCR): trends and problems. 
     </article-h3> 
     <source />J Mol Endocrinol 
     <volume>
       29 
     </volume> 
     <fpage>
       23 
     </fpage> 
     <lpage>
       39 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12200227 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Eisen1"> 
    <label>24</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Eisen 
       </surname> 
       <given-names>
         MB 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Spellman 
       </surname> 
       <given-names>
         PT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Brown 
       </surname> 
       <given-names>
         PO 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Botstein 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       1998 
     </year> 
     <article-h3>
       Cluster analysis and display of genome-wide expression patterns. 
     </article-h3> 
     <source />Proc Natl Acad Sci U S A 
     <volume>
       95 
     </volume> 
     <fpage>
       14863 
     </fpage> 
     <lpage>
       14868 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9843981 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Saldanha1"> 
    <label>25</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Saldanha 
       </surname> 
       <given-names>
         AJ 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2004 
     </year> 
     <article-h3>
       Java Treeview–extensible visualization of microarray data. 
     </article-h3> 
     <source />Bioinformatics 
     <volume>
       20 
     </volume> 
     <fpage>
       3246 
     </fpage> 
     <lpage>
       3248 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15180930 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-AlShahrour1"> 
    <label>26</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Al-Shahrour 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Minguez 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vaquerizas 
       </surname> 
       <given-names>
         JM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Conde 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dopazo 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2005 
     </year> 
     <article-h3>
       BABELOMICS: a suite of web tools for functional annotation and analysis of groups of genes in high-throughput experiments. 
     </article-h3> 
     <source />Nucleic Acids Res 
     <volume>
       33 
     </volume> 
     <fpage>
       W460 
     </fpage> 
     <lpage>
       464 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15980512 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-vanHolde1"> 
    <label>27</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         van Holde 
       </surname> 
       <given-names>
         KE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Miller 
       </surname> 
       <given-names>
         KI 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       1995 
     </year> 
     <article-h3>
       Hemocyanins. 
     </article-h3> 
     <source />Adv Protein Chem 
     <volume>
       47 
     </volume> 
     <fpage>
       1 
     </fpage> 
     <lpage>
       81 
     </lpage> 
     <pub-id pub-id-type="pmid">
       8561049 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Pena1"> 
    <label>28</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Pena 
       </surname> 
       <given-names>
         FJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rodriguez Martinez 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tapia 
       </surname> 
       <given-names>
         JA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Ortega Ferrusola 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gonzalez Fernandez 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Mitochondria in mammalian sperm physiology and pathology: a review. 
     </article-h3> 
     <source />Reprod Domest Anim 
     <volume>
       44 
     </volume> 
     <fpage>
       345 
     </fpage> 
     <lpage>
       349 
     </lpage> 
     <pub-id pub-id-type="pmid">
       19144010 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Erkan1"> 
    <label>29</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Erkan 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sousa 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2002 
     </year> 
     <article-h3>
       Fine structural study of the spermatogenic cycle in 
      <italic>
        Pitar rudis 
      </italic> and 
      <italic>
        Chamelea gallina 
      </italic> (Mollusca, Bivalvia, Veneridae). 
     </article-h3> 
     <source />Tissue Cell 
     <volume>
       34 
     </volume> 
     <fpage>
       262 
     </fpage> 
     <lpage>
       272 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12176309 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Yaffe1"> 
    <label>30</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Yaffe 
       </surname> 
       <given-names>
         MP 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       1997 
     </year> 
     <article-h3>
       Mitochondrial morphogenesis: fusion factor for fly fertility. 
     </article-h3> 
     <source />Curr Biol 
     <volume>
       7 
     </volume> 
     <fpage>
       R782 
     </fpage> 
     <lpage>
       783 
     </lpage> 
     <pub-id pub-id-type="pmid">
       9382824 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Ternes1"> 
    <label>31</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ternes 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Franke 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Zahringer 
       </surname> 
       <given-names>
         U 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sperling 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Heinz 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2002 
     </year> 
     <article-h3>
       Identification and characterization of a sphingolipid delta 4-desaturase family. 
     </article-h3> 
     <source />J Biol Chem 
     <volume>
       277 
     </volume> 
     <fpage>
       25512 
     </fpage> 
     <lpage>
       25518 
     </lpage> 
     <pub-id pub-id-type="pmid">
       11937514 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Reeve1"> 
    <label>32</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Reeve 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Carhan 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dee 
       </surname> 
       <given-names>
         CT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Moffat 
       </surname> 
       <given-names>
         KG 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2007 
     </year> 
     <article-h3>
       Slowmo is required for 
      <italic>
        Drosophila 
      </italic> germline proliferation. 
     </article-h3> 
     <source />Genesis 
     <volume>
       45 
     </volume> 
     <fpage>
       66 
     </fpage> 
     <lpage>
       75 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17245774 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Khor1"> 
    <label>33</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Khor 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bredemeyer 
       </surname> 
       <given-names>
         AL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Huang 
       </surname> 
       <given-names>
         CY 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Turnbull 
       </surname> 
       <given-names>
         IR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Evans 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2006 
     </year> 
     <article-h3>
       Proteasome activator PA200 is required for normal spermatogenesis. 
     </article-h3> 
     <source />Mol Cell Biol 
     <volume>
       26 
     </volume> 
     <fpage>
       2999 
     </fpage> 
     <lpage>
       3007 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16581775 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Guan1"> 
    <label>34</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Guan 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kinoshita 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yuan 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Spatiotemporal association of DNAJB13 with the annulus during mouse sperm flagellum development. 
     </article-h3> 
     <source />BMC Dev Biol 
     <volume>
       9 
     </volume> 
     <fpage>
       23 
     </fpage> 
     <pub-id pub-id-type="pmid">
       19298648 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Guan2"> 
    <label>35</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Guan 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Yuan 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       A heat-shock protein 40, DNAJB13, is an axoneme-associated component in mouse spermatozoa. 
     </article-h3> 
     <source />Mol Reprod Dev 
     <volume>
       75 
     </volume> 
     <fpage>
       1379 
     </fpage> 
     <lpage>
       1386 
     </lpage> 
     <pub-id pub-id-type="pmid">
       18247331 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Yang1"> 
    <label>36</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Yang 
       </surname> 
       <given-names>
         HM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liu 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nie 
       </surname> 
       <given-names>
         ZY 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nie 
       </surname> 
       <given-names>
         DS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Deng 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2005 
     </year> 
     <article-h3>
       Molecular cloning of a novel rat gene Tsarg1, a member of the DnaJ/HSP40 protein family. 
     </article-h3> 
     <source />DNA Seq 
     <volume>
       16 
     </volume> 
     <fpage>
       166 
     </fpage> 
     <lpage>
       172 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16147871 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Li1"> 
    <label>37</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Li 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Liu 
       </surname> 
       <given-names>
         G 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fu 
       </surname> 
       <given-names>
         JJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Li 
       </surname> 
       <given-names>
         LY 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Tan 
       </surname> 
       <given-names>
         XJ 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Molecular cloning and characterization of a novel transcript variant of Mtsarg1 gene. 
     </article-h3> 
     <source />Mol Biol Rep 
     <volume>
       36 
     </volume> 
     <fpage>
       1023 
     </fpage> 
     <lpage>
       1032 
     </lpage> 
     <pub-id pub-id-type="pmid">
       18551385 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Alekseev1"> 
    <label>38</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Alekseev 
       </surname> 
       <given-names>
         OM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Widgren 
       </surname> 
       <given-names>
         EE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Richardson 
       </surname> 
       <given-names>
         RT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         O'Rand 
       </surname> 
       <given-names>
         MG 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2005 
     </year> 
     <article-h3>
       Association of NASP with HSP90 in mouse spermatogenic cells: stimulation of ATPase activity and transport of linker histones into nuclei. 
     </article-h3> 
     <source />J Biol Chem 
     <volume>
       280 
     </volume> 
     <fpage>
       2904 
     </fpage> 
     <lpage>
       2911 
     </lpage> 
     <pub-id pub-id-type="pmid">
       15533935 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Karnoub1"> 
    <label>39</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Karnoub 
       </surname> 
       <given-names>
         AE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Weinberg 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       Ras oncogenes: split personalities. 
     </article-h3> 
     <source />Nat Rev Mol Cell Biol 
     <volume>
       9 
     </volume> 
     <fpage>
       517 
     </fpage> 
     <lpage>
       531 
     </lpage> 
     <pub-id pub-id-type="pmid">
       18568040 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Nishida1"> 
    <label>40</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Nishida 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2002 
     </year> 
     <article-h3>
       Patterning the marginal zone of early ascidian embryos: localized maternal mRNA and inductive interactions. 
     </article-h3> 
     <source />Bioessays 
     <volume>
       24 
     </volume> 
     <fpage>
       613 
     </fpage> 
     <lpage>
       624 
     </lpage> 
     <pub-id pub-id-type="pmid">
       12111722 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-OKeefe1"> 
    <label>41</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         O'Keefe 
       </surname> 
       <given-names>
         DD 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Gonzalez-Nino 
       </surname> 
       <given-names>
         E 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Burnett 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Dylla 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lambeth 
       </surname> 
       <given-names>
         SM 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Rap1 maintains adhesion between cells to affect Egfr signaling and planar cell polarity in 
      <italic>
        Drosophila 
      </italic>. 
     </article-h3> 
     <source />Dev Biol 
     <volume>
       333 
     </volume> 
     <fpage>
       143 
     </fpage> 
     <lpage>
       160 
     </lpage> 
     <pub-id pub-id-type="pmid">
       19576205 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Lee1"> 
    <label>42</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kanatsu-Shinohara 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Morimoto 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kazuki 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Takashima 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Genetic reconstruction of mouse spermatogonial stem cell self-renewal in vitro by Ras-cyclin D2 activation. 
     </article-h3> 
     <source />Cell Stem Cell 
     <volume>
       5 
     </volume> 
     <fpage>
       76 
     </fpage> 
     <lpage>
       86 
     </lpage> 
     <pub-id pub-id-type="pmid">
       19570516 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Singh1"> 
    <label>43</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Singh 
       </surname> 
       <given-names>
         SR 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hou 
       </surname> 
       <given-names>
         SX 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       Immunohistological techniques for studying the 
      <italic>
        Drosophila 
      </italic> male germline stem cell. 
     </article-h3> 
     <source />Methods Mol Biol 
     <volume>
       450 
     </volume> 
     <fpage>
       45 
     </fpage> 
     <lpage>
       59 
     </lpage> 
     <pub-id pub-id-type="pmid">
       18370050 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Buchardt1"> 
    <label>44</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Buchardt 
       </surname> 
       <given-names>
         B 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fritz 
       </surname> 
       <given-names>
         P 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       1978 
     </year> 
     <article-h3>
       Strontium uptake in shell aragonite from the freshwater gastropod 
      <italic>
        Limnaea stagnalis 
      </italic>. 
     </article-h3> 
     <source />Science 
     <volume>
       199 
     </volume> 
     <fpage>
       291 
     </fpage> 
     <lpage>
       292 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17759661 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Gros1"> 
    <label>45</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Gros 
       </surname> 
       <given-names>
         O 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Frenkiel 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Aranda 
       </surname> 
       <given-names>
         DA 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Structural analysis of the digestive gland of the queen conch 
      <italic>
        Strombus gigas 
      </italic> Linnaeus, 1758 and its intracellular parasites. 
     </article-h3> 
     <source />J Mollusc Stud 
     <volume>
       75 
     </volume> 
     <fpage>
       59 
     </fpage> 
     <lpage>
       68 
     </lpage> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Desouky1"> 
    <label>46</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Desouky 
       </surname> 
       <given-names>
         MM 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2006 
     </year> 
     <article-h3>
       Tissue distribution and subcellular localization of trace metals in the pond snail 
      <italic>
        Lymnaea stagnalis 
      </italic> with special reference to the role of lysosomal granules in metal sequestration. 
     </article-h3> 
     <source />Aquat Toxicol 
     <volume>
       77 
     </volume> 
     <fpage>
       143 
     </fpage> 
     <lpage>
       152 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16413619 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Gimbert1"> 
    <label>47</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Gimbert 
       </surname> 
       <given-names>
         F 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Vijver 
       </surname> 
       <given-names>
         MG 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Coeurdassier 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Scheifler 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Peijnenburg 
       </surname> 
       <given-names>
         WJ 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       How subcellular partitioning can help to understand heavy metal accumulation and elimination kinetics in snails. 
     </article-h3> 
     <source />Environ Toxicol Chem 
     <volume>
       27 
     </volume> 
     <fpage>
       1284 
     </fpage> 
     <lpage>
       1292 
     </lpage> 
     <pub-id pub-id-type="pmid">
       18229974 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Nott1"> 
    <label>48</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Nott 
       </surname> 
       <given-names>
         JA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nicolaidou 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       1990 
     </year> 
     <article-h3>
       Transfer of Metal Detoxification Along Marine Food-Chains. 
     </article-h3> 
     <source />Journal of the Marine Biological Association of the United Kingdom 
     <volume>
       70 
     </volume> 
     <fpage>
       905 
     </fpage> 
     <lpage>
       912 
     </lpage> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Yamaguchi1"> 
    <label>49</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Yamaguchi 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Miura 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kikuchi 
       </surname> 
       <given-names>
         K 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Celino 
       </surname> 
       <given-names>
         FT 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Agusa 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Zinc is an essential trace element for spermatogenesis. 
     </article-h3> 
     <source />Proc Natl Acad Sci U S A 
     <volume>
       106 
     </volume> 
     <fpage>
       10859 
     </fpage> 
     <lpage>
       10864 
     </lpage> 
     <pub-id pub-id-type="pmid">
       19541612 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Ruppert1"> 
    <label>50</label> 
    <mixed-citation publication-type="book"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Ruppert 
       </surname> 
       <given-names>
         EE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fox 
       </surname> 
       <given-names>
         RS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Barnes 
       </surname> 
       <given-names>
         RD 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2004 
     </year> 
     <article-h3>
       Invertebrate Zoology: A Functional Evolutionary Approach. 
     </article-h3> 
     <publisher-loc>
       Belmont, CA 
     </publisher-loc> 
     <publisher-name>
       Brooks/Cole 
     </publisher-name> 
     <comment>
       xiv+963+926 
     </comment> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Iguchi1"> 
    <label>51</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Iguchi 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Watanabe 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Katsu 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2007 
     </year> 
     <article-h3>
       Toxicogenomics and ecotoxicogenomics for studying endocrine disruption and basic biology. 
     </article-h3> 
     <source />Gen Comp Endocrinol 
     <volume>
       153 
     </volume> 
     <fpage>
       25 
     </fpage> 
     <lpage>
       29 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17324418 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Iguchi2"> 
    <label>52</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Iguchi 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Watanabe 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Katsu 
       </surname> 
       <given-names>
         Y 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2006 
     </year> 
     <article-h3>
       Application of ecotoxicogenomics for studying endocrine disruption in vertebrates and invertebrates. 
     </article-h3> 
     <source />Environ Health Perspect 
     <volume>
       114 
     </volume> 
     <issue>
       Suppl 1 
     </issue> 
     <fpage>
       101 
     </fpage> 
     <lpage>
       105 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16818254 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Guiller1"> 
    <label>53</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Guiller 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Bellido 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Coutelle 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Madec 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2006 
     </year> 
     <article-h3>
       Spatial genetic pattern in the land mollusc 
      <italic>
        Helix aspersa 
      </italic> inferred from a ‘centre-based clustering’ procedure. 
     </article-h3> 
     <source />Genet Res 
     <volume>
       88 
     </volume> 
     <fpage>
       27 
     </fpage> 
     <lpage>
       44 
     </lpage> 
     <pub-id pub-id-type="pmid">
       17014742 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Azami1"> 
    <label>54</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Azami 
       </surname> 
       <given-names>
         S 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wagatsuma 
       </surname> 
       <given-names>
         A 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Sadamoto 
       </surname> 
       <given-names>
         H 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hatakeyama 
       </surname> 
       <given-names>
         D 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Usami 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2006 
     </year> 
     <article-h3>
       Altered gene activity correlated with long-term memory formation of conditioned taste aversion in 
      <italic>
        Lymnaea 
      </italic>. 
     </article-h3> 
     <source />J Neurosci Res 
     <volume>
       84 
     </volume> 
     <fpage>
       1610 
     </fpage> 
     <lpage>
       1620 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16941636 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-vanRensburg1"> 
    <label>55</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         van Rensburg 
       </surname> 
       <given-names>
         MJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Coyne 
       </surname> 
       <given-names>
         VE 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       The role of electron transport in the defence response of the South African abalone, 
      <italic>
        Haliotis midae 
      </italic>. 
     </article-h3> 
     <source />Fish Shellfish Immunol 
     <volume>
       26 
     </volume> 
     <fpage>
       171 
     </fpage> 
     <lpage>
       176 
     </lpage> 
     <pub-id pub-id-type="pmid">
       18977446 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Lee2"> 
    <label>56</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         YS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Choi 
       </surname> 
       <given-names>
         SL 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kim 
       </surname> 
       <given-names>
         TH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lee 
       </surname> 
       <given-names>
         JA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kim 
       </surname> 
       <given-names>
         HK 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       Transcriptome analysis and identification of regulators for long-term plasticity in 
      <italic>
        Aplysia kurodai 
      </italic>. 
     </article-h3> 
     <source />Proc Natl Acad Sci U S A 
     <volume>
       105 
     </volume> 
     <fpage>
       18602 
     </fpage> 
     <lpage>
       18607 
     </lpage> 
     <pub-id pub-id-type="pmid">
       19017802 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Adema1"> 
    <label>57</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Adema 
       </surname> 
       <given-names>
         CM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hanington 
       </surname> 
       <given-names>
         PC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lun 
       </surname> 
       <given-names>
         CM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Rosenberg 
       </surname> 
       <given-names>
         GH 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Aragon 
       </surname> 
       <given-names>
         AD 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Differential transcriptomic responses of 
      <italic>
        Biomphalaria glabrata 
      </italic> (Gastropoda, Mollusca) to bacteria and metazoan parasites, 
      <italic>
        Schistosoma mansoni 
      </italic> and 
      <italic>
        Echinostoma paraensei 
      </italic> (Digenea, Platyhelminthes). 
     </article-h3> 
     <source />Mol Immunol 
     <volume>
       47 
     </volume> 
     <fpage>
       849 
     </fpage> 
     <lpage>
       860 
     </lpage> 
     <pub-id pub-id-type="pmid">
       19962194 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Lockyer1"> 
    <label>58</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Lockyer 
       </surname> 
       <given-names>
         AE 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Spinks 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Kane 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Hoffmann 
       </surname> 
       <given-names>
         KF 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Fitzpatrick 
       </surname> 
       <given-names>
         JM 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3> 
      <italic>
        Biomphalaria glabrata 
      </italic> transcriptome: cDNA microarray profiling identifies resistant- and susceptible-specific gene expression in haemocytes from snail strains exposed to 
      <italic>
        Schistosoma mansoni 
      </italic>. 
     </article-h3> 
     <source />BMC Genomics 
     <volume>
       9 
     </volume> 
     <fpage>
       634 
     </fpage> 
     <pub-id pub-id-type="pmid">
       19114004 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Gobert1"> 
    <label>59</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Gobert 
       </surname> 
       <given-names>
         GN 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Moertel 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Brindley 
       </surname> 
       <given-names>
         PJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McManus 
       </surname> 
       <given-names>
         DP 
       </given-names> 
      </name> 
     </person-group> 
     <year>
       2009 
     </year> 
     <article-h3>
       Developmental gene expression profiles of the human pathogen 
      <italic>
        Schistosoma japonicum 
      </italic>. 
     </article-h3> 
     <source />BMC Genomics 
     <volume>
       10 
     </volume> 
     <fpage>
       128 
     </fpage> 
     <pub-id pub-id-type="pmid">
       19320991 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Waisberg1"> 
    <label>60</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Waisberg 
       </surname> 
       <given-names>
         M 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Lobo 
       </surname> 
       <given-names>
         FP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Cerqueira 
       </surname> 
       <given-names>
         GC 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Passos 
       </surname> 
       <given-names>
         LK 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Carvalho 
       </surname> 
       <given-names>
         OS 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3> 
      <italic>
        Schistosoma mansoni 
      </italic>: Microarray analysis of gene expression induced by host sex. 
     </article-h3> 
     <source />Exp Parasitol 
     <volume>
       120 
     </volume> 
     <fpage>
       357 
     </fpage> 
     <lpage>
       363 
     </lpage> 
     <pub-id pub-id-type="pmid">
       18822286 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Gobert2"> 
    <label>61</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Gobert 
       </surname> 
       <given-names>
         GN 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McInnes 
       </surname> 
       <given-names>
         R 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Moertel 
       </surname> 
       <given-names>
         L 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Nelson 
       </surname> 
       <given-names>
         C 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Jones 
       </surname> 
       <given-names>
         MK 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2006 
     </year> 
     <article-h3>
       Transcriptomics tool for the human 
      <italic>
        Schistosoma 
      </italic> blood flukes using microarray gene expression profiling. 
     </article-h3> 
     <source />Exp Parasitol 
     <volume>
       114 
     </volume> 
     <fpage>
       160 
     </fpage> 
     <lpage>
       172 
     </lpage> 
     <pub-id pub-id-type="pmid">
       16631746 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Fitzpatrick1"> 
    <label>62</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Fitzpatrick 
       </surname> 
       <given-names>
         JM 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Protasio 
       </surname> 
       <given-names>
         AV 
       </given-names> 
      </name> 
      <name> 
       <surname>
         McArdle 
       </surname> 
       <given-names>
         AJ 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Williams 
       </surname> 
       <given-names>
         GA 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Johnston 
       </surname> 
       <given-names>
         DA 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       Use of genomic DNA as an indirect reference for identifying gender-associated transcripts in morphologically identical, but chromosomally distinct, 
      <italic>
        Schistosoma mansoni 
      </italic> cercariae. 
     </article-h3> 
     <source />PLoS Negl Trop Dis 
     <volume>
       2 
     </volume> 
     <fpage>
       e323 
     </fpage> 
     <pub-id pub-id-type="pmid">
       18941520 
     </pub-id> 
    </mixed-citation> 
   </ref> 
   <ref id="pone.0012737-Dillon1"> 
    <label>63</label> 
    <mixed-citation publication-type="journal"> 
     <person-group person-group-type="author"> 
      <name> 
       <surname>
         Dillon 
       </surname> 
       <given-names>
         GP 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Feltwell 
       </surname> 
       <given-names>
         T 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Skelton 
       </surname> 
       <given-names>
         J 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Coulson 
       </surname> 
       <given-names>
         PS 
       </given-names> 
      </name> 
      <name> 
       <surname>
         Wilson 
       </surname> 
       <given-names>
         RA 
       </given-names> 
      </name> 
      <etal></etal> 
     </person-group> 
     <year>
       2008 
     </year> 
     <article-h3>
       Altered patterns of gene expression underlying the enhanced immunogenicity of radiation-attenuated schistosomes. 
     </article-h3> 
     <source />PLoS Negl Trop Dis 
     <volume>
       2 
     </volume> 
     <fpage>
       e240 
     </fpage> 
     <pub-id pub-id-type="pmid">
       18493602 
     </pub-id> 
    </mixed-citation> 
   </ref> 
  </ref-list> 
 </back> 
</article>